{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "016d2660-7383-4626-917b-3a49f474197d",
   "metadata": {},
   "outputs": [],
   "source": [
    "import json"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "a5cfa7ef-318a-4fb6-b998-002e8dc56ea0",
   "metadata": {},
   "outputs": [],
   "source": [
    "with open(\"trec-covid.jsonl\") as f: # retrieval top-100 result\n",
    "    results = [json.loads(i) for i in f.readlines()][:50]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "8cfe2435-d8f8-4515-bd91-71a7f504dc74",
   "metadata": {},
   "outputs": [],
   "source": [
    "with open(\"/gallery_louvre/dayoon.ko/research/sds/src/datasets/trec-covid/corpus.jsonl\") as f:\n",
    "    corpus = [json.loads(i) for i in f.readlines()]\n",
    "    corpus_dict = {}\n",
    "    for c in corpus:\n",
    "        corpus_dict[c[\"_id\"]] = c[\"text\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "71b0dbfe-1f9b-4982-9be4-09a35c5dcf45",
   "metadata": {},
   "outputs": [],
   "source": [
    "seq_num_to_id = {}\n",
    "for i, doc in zip(range(1, len(corpus) + 1), corpus):\n",
    "    seq_num_to_id[i] = doc[\"_id\"] "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "58aa6401-3ffa-40ef-85a2-da66daf6b95f",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Add document id to each retrieved document\n",
    "for res in results:\n",
    "    for retrieval in res[\"retrieval\"]:\n",
    "        seq_num = retrieval[\"seq_num\"]\n",
    "        retrieval[\"_id\"] = seq_num_to_id[seq_num]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "6be8ae72-093d-4e93-86cb-a95e0607b541",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Get retrieval results as corpus id\n",
    "retrieval_results = {}\n",
    "for res in results:\n",
    "    qid = res[\"_id\"]\n",
    "    for retrieval in res[\"retrieval\"]:\n",
    "        doc_id = retrieval[\"_id\"]\n",
    "        if doc_id in retrieval_results:\n",
    "            retrieval_results[doc_id].append(qid)\n",
    "        else:\n",
    "            retrieval_results[doc_id] = [qid]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "bbe70196-b593-47e9-8e65-a51dec7a38d6",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'n15i01tn': ['1', '9', '43', '48'],\n",
       " 'hmvo5b0q': ['1'],\n",
       " 'dv9m19yk': ['1'],\n",
       " 'm4y8tf6u': ['1'],\n",
       " 'utsr0zv7': ['1', '33'],\n",
       " 'k9lcpjyo': ['1'],\n",
       " 'jbtrdvhe': ['1'],\n",
       " 'r2ynbnxx': ['1'],\n",
       " 'jxbk30gh': ['1'],\n",
       " 'rtw8m00s': ['1', '4'],\n",
       " 'juz9jnfk': ['1', '26'],\n",
       " 'lj2iu7z0': ['1'],\n",
       " 't1iagum7': ['1'],\n",
       " '4dtk1kyh': ['1'],\n",
       " 'jwxt4ygt': ['1'],\n",
       " 'wco27nop': ['1'],\n",
       " 'dnxhtbxn': ['1'],\n",
       " '8qfwat2v': ['1'],\n",
       " 'deee71uw': ['1'],\n",
       " 'gjigvjgv': ['1', '4', '8', '9', '26'],\n",
       " 'xndojxqk': ['1'],\n",
       " 'ccxj4s6j': ['1'],\n",
       " 'rugijb30': ['1'],\n",
       " 'p3681yit': ['1'],\n",
       " '2lxs9laj': ['1'],\n",
       " '1ybj2p1n': ['1'],\n",
       " '80c471y3': ['1', '35'],\n",
       " 'w3avnbpu': ['1'],\n",
       " 'fsy00ngb': ['1', '15', '18', '48'],\n",
       " '75773gwg': ['1'],\n",
       " 'ptizke03': ['1'],\n",
       " 'bbae1qyx': ['1'],\n",
       " 'k9yus2sv': ['1'],\n",
       " 'qp4efhwq': ['1'],\n",
       " '7v5aln90': ['1'],\n",
       " '6ck2ntid': ['1'],\n",
       " '8a6flxl6': ['1'],\n",
       " 'f6p9i951': ['1'],\n",
       " '8vi60e0a': ['1'],\n",
       " 'can1e8ro': ['1'],\n",
       " '82sc928x': ['1'],\n",
       " 'dtv7to3l': ['1'],\n",
       " 'hqzkzupi': ['1', '4', '39'],\n",
       " 'sqrn6kjy': ['1'],\n",
       " '9griuata': ['1'],\n",
       " 'pfv7q4v6': ['1'],\n",
       " '7qtlj9t8': ['1'],\n",
       " 'i14gkdw0': ['1'],\n",
       " '1x0tvnho': ['1'],\n",
       " 'nph01vl6': ['1'],\n",
       " '0nh58odf': ['1'],\n",
       " 'drzk1uq6': ['1', '4'],\n",
       " 'nk7w2ola': ['1'],\n",
       " 'tfkhptdg': ['1'],\n",
       " 'p3gswh23': ['1'],\n",
       " '475nei28': ['1'],\n",
       " 'u7tz6xzz': ['1', '37'],\n",
       " 'aura0th0': ['1'],\n",
       " 'v9ftmdlv': ['1'],\n",
       " 'okmpsv47': ['1'],\n",
       " 'uv3djven': ['1'],\n",
       " '6k5ac3f2': ['1'],\n",
       " 'gmxhmzki': ['1'],\n",
       " 'jb6o4v6p': ['1'],\n",
       " 'sigbm70i': ['1'],\n",
       " '431buxxr': ['1'],\n",
       " 'xibub2in': ['1'],\n",
       " 'ectly1v4': ['1', '37'],\n",
       " 'm0vqst1q': ['1'],\n",
       " 'c2bdmgj7': ['1', '4'],\n",
       " 'sajlyf4t': ['1'],\n",
       " 'xo7l2iei': ['1'],\n",
       " 'dy79bxsq': ['1', '9'],\n",
       " '3zy5dgxz': ['1'],\n",
       " 't6gqa48n': ['1', '3', '38'],\n",
       " 'qvz63m93': ['1'],\n",
       " 'iudq5jdu': ['1'],\n",
       " 'b5287qd4': ['1'],\n",
       " '59492sjb': ['1', '40'],\n",
       " '7nla9aji': ['1'],\n",
       " '1mjaycee': ['1'],\n",
       " '9xueqdri': ['1'],\n",
       " 'vembiw2k': ['1'],\n",
       " '0wm6u10a': ['1'],\n",
       " 'cgvj10r2': ['1'],\n",
       " '8t35z4gl': ['1'],\n",
       " '8iyynbup': ['1'],\n",
       " 'wyjzax2y': ['1', '26'],\n",
       " '245s0y7l': ['1', '26'],\n",
       " 'shvsnkxd': ['1'],\n",
       " 'xuczplaf': ['1'],\n",
       " 'on70zzn0': ['1'],\n",
       " 'dp2xzul1': ['1'],\n",
       " '6bzfptbs': ['1'],\n",
       " '5ybt4p0u': ['1'],\n",
       " 'mwpbrbqu': ['1', '4', '9', '14'],\n",
       " 'pfyqjrwk': ['1', '4', '9', '14'],\n",
       " '2y452utz': ['1'],\n",
       " 'rvrgcugn': ['1', '4', '39'],\n",
       " 'k6hh7yft': ['1'],\n",
       " 'rbzl0txp': ['2', '16', '32'],\n",
       " '1ckrnxom': ['2'],\n",
       " '4819g00y': ['2', '15', '16'],\n",
       " '04awj06g': ['2', '15', '16'],\n",
       " 'u19e9j2w': ['2'],\n",
       " '959w9sln': ['2', '12', '13', '15', '16', '31', '44'],\n",
       " 'y1rlk8th': ['2'],\n",
       " 'vj3wk150': ['2'],\n",
       " 'tzv0pxnp': ['2', '15'],\n",
       " '4068kzp0': ['2', '13', '15', '31'],\n",
       " 'r9yrr45q': ['2'],\n",
       " 'al80kebv': ['2'],\n",
       " '1npav6m4': ['2'],\n",
       " '5dfg0f5d': ['2'],\n",
       " '8rv8zk52': ['2'],\n",
       " '7gytw8j4': ['2'],\n",
       " 'krb1eidw': ['2'],\n",
       " 'aiwxlxzt': ['2'],\n",
       " 'azcr8jf3': ['2'],\n",
       " 'ax87r0bj': ['2', '31'],\n",
       " 'xrgnt6l5': ['2'],\n",
       " 'bnrmh1qs': ['2', '14'],\n",
       " 'jwflooop': ['2'],\n",
       " 'tmg7dnk4': ['2'],\n",
       " 'zxx7tikz': ['2'],\n",
       " 'w7ycc07b': ['2'],\n",
       " '5c4n4tsf': ['2'],\n",
       " 'niq3hosc': ['2'],\n",
       " 'gdl4l9xo': ['2'],\n",
       " 'foha7ozb': ['2'],\n",
       " 'c4pcztdc': ['2'],\n",
       " '8hau76g7': ['2', '31'],\n",
       " 'mkhkcqyn': ['2', '15'],\n",
       " 'zespmk29': ['2'],\n",
       " 'bhq4t850': ['2'],\n",
       " 'sb1n3fra': ['2'],\n",
       " '7t976vq8': ['2'],\n",
       " 'b7j6bwv7': ['2'],\n",
       " 'jlop0lui': ['2'],\n",
       " '92l5sm9m': ['2'],\n",
       " 'zvngy7zz': ['2'],\n",
       " '6imghzg1': ['2', '44'],\n",
       " 'dkxi8mgw': ['2', '31'],\n",
       " 'qffg6r91': ['2'],\n",
       " 'p8p6e72k': ['2'],\n",
       " 'ww1rgcds': ['2'],\n",
       " 'zfhpij93': ['2', '13', '31'],\n",
       " '20zs4unc': ['2', '13'],\n",
       " '147yc66p': ['2'],\n",
       " '6exmxw6r': ['2'],\n",
       " 't7c0drft': ['2'],\n",
       " 'nsp53lv6': ['2'],\n",
       " 'jg387csa': ['2', '13', '16', '31'],\n",
       " 'bpv7p7zo': ['2'],\n",
       " 'jxa9h27b': ['2'],\n",
       " 'kvh60zd5': ['2'],\n",
       " 'zvvwwc0r': ['2'],\n",
       " 'sgiuqs6z': ['2'],\n",
       " 'p4eekani': ['2'],\n",
       " '3svnvozz': ['2'],\n",
       " 't4kdxnhy': ['2'],\n",
       " 'fq7ud25y': ['2'],\n",
       " 'hadnxjeo': ['2'],\n",
       " '46zui1sl': ['2'],\n",
       " 'akb96git': ['2'],\n",
       " 'f53i4n02': ['2'],\n",
       " 'wm6yey1v': ['2'],\n",
       " 'hgau3922': ['2', '15', '16', '31'],\n",
       " 'qwub35cd': ['2'],\n",
       " 'oea5wvzy': ['2'],\n",
       " '7426dyn6': ['2'],\n",
       " 'ns0o86mw': ['2'],\n",
       " 'ypokfutq': ['2'],\n",
       " '0dac26xk': ['2', '13', '16'],\n",
       " 'o6q2rkv7': ['2'],\n",
       " '4j47etsu': ['2'],\n",
       " 'psnqrdo2': ['2'],\n",
       " 'g9umdcn2': ['2'],\n",
       " 'gmgat6f7': ['2', '14'],\n",
       " 'yr678u8c': ['2', '31'],\n",
       " 'hiw576og': ['2', '16', '31'],\n",
       " 'h8cemq2n': ['2'],\n",
       " 'epa6m6nq': ['2'],\n",
       " '8bw843ek': ['2', '40'],\n",
       " 'wic7n6ia': ['2'],\n",
       " 'qc1mpzli': ['2'],\n",
       " 'eiek6olk': ['2', '14'],\n",
       " 'ws03xsho': ['2'],\n",
       " '703febui': ['2'],\n",
       " '4lm663f1': ['2'],\n",
       " 'ildyxrce': ['2'],\n",
       " 'k260c04b': ['2'],\n",
       " 'ig0rnbqb': ['2', '40'],\n",
       " 'cvp10nx0': ['2', '31'],\n",
       " 'tyhtdawb': ['2'],\n",
       " 'pvbbk8cg': ['2'],\n",
       " 'l7riy8ox': ['2', '3', '32', '50'],\n",
       " 'buhu5c41': ['2'],\n",
       " '0vlzwksu': ['2'],\n",
       " '03s9spbi': ['2'],\n",
       " 'buwz6lu3': ['3'],\n",
       " 'u5nxm9tu': ['3', '49'],\n",
       " '8i1u1a9t': ['3'],\n",
       " 'car394ou': ['3', '49'],\n",
       " 't3sjv4hv': ['3', '49'],\n",
       " 'yzffm05r': ['3'],\n",
       " 'ukz73rp2': ['3'],\n",
       " '5x30jj1s': ['3', '34', '49'],\n",
       " '61fmul8c': ['3', '38'],\n",
       " 'vi1jl4po': ['3'],\n",
       " '72fokkad': ['3'],\n",
       " 'mxafg0t9': ['3'],\n",
       " 'fi1ldwe8': ['3'],\n",
       " '7jwhypgs': ['3'],\n",
       " '2jeb1vcs': ['3', '50'],\n",
       " 'g5ovvjc6': ['3'],\n",
       " 'hpbjdytv': ['3'],\n",
       " 'egj0aym6': ['3'],\n",
       " 'kh54yctx': ['3', '50'],\n",
       " 'w66gtl3b': ['3', '38'],\n",
       " 'mtq6yh25': ['3'],\n",
       " 'glwaeqbh': ['3'],\n",
       " 'xetzg7gp': ['3'],\n",
       " '6n0n5l4f': ['3', '50'],\n",
       " 's6v4bgev': ['3', '38', '50'],\n",
       " 'dqs21e0q': ['3'],\n",
       " '6dwyv4qo': ['3'],\n",
       " 'mqnz9gts': ['3'],\n",
       " 'ebalebqq': ['3', '38'],\n",
       " 'wfftfkam': ['3'],\n",
       " 'x5jfr9u0': ['3'],\n",
       " 'itu39mln': ['3', '49'],\n",
       " 'rfgogqmu': ['3'],\n",
       " 'dptgg05n': ['3', '49'],\n",
       " '7ro1up5b': ['3'],\n",
       " '1309xyi4': ['3', '38'],\n",
       " 'di8lljoi': ['3'],\n",
       " 'l11epnl4': ['3', '49'],\n",
       " 'xzl23c52': ['3', '36'],\n",
       " '2s1io2fg': ['3'],\n",
       " 'el7mdliy': ['3'],\n",
       " 'glmni128': ['3', '46'],\n",
       " 'u7nebgen': ['3'],\n",
       " '92hyxezl': ['3'],\n",
       " 'm7yii8el': ['3', '33', '50'],\n",
       " 'pc7efh1v': ['3', '33', '50'],\n",
       " '023h20vk': ['3', '32', '50'],\n",
       " 'rs79r7kc': ['3', '49'],\n",
       " 'oj4ue0ur': ['3', '32'],\n",
       " '7lk8h93q': ['3', '33', '50'],\n",
       " '99cgvtlu': ['3', '49'],\n",
       " 'e1mw9lx1': ['3', '38'],\n",
       " 's33brsk6': ['3'],\n",
       " '38mhmxvd': ['3', '49'],\n",
       " 'fkisgm2j': ['3'],\n",
       " '06t0s4ii': ['3'],\n",
       " 'y8fmls6v': ['3'],\n",
       " 'ygi1f5oy': ['3'],\n",
       " '7n6pva1l': ['3', '49'],\n",
       " 'bcqdm2b1': ['3', '38'],\n",
       " 'ipw96f03': ['3'],\n",
       " 'fj2rjmop': ['3'],\n",
       " 'v155jl20': ['3'],\n",
       " 'chaa3swx': ['3'],\n",
       " 'i15rb8d2': ['3', '5'],\n",
       " 'yrrz7oef': ['3', '50'],\n",
       " '8earvduw': ['3'],\n",
       " 'xbgmch1q': ['3', '50'],\n",
       " 'p2lhpbnq': ['3', '49'],\n",
       " '6n26bw7v': ['3'],\n",
       " 'hrgb0e03': ['3'],\n",
       " 'h554lbyf': ['3'],\n",
       " '9ldwlix3': ['3'],\n",
       " 'to6b17cb': ['3'],\n",
       " 'xyp7oyca': ['3'],\n",
       " 'rsz7ch2a': ['3'],\n",
       " 'a9h25l7j': ['3'],\n",
       " 'mlo8zypy': ['3'],\n",
       " 'tdvb0fhv': ['3', '33', '50'],\n",
       " 'fqo4leif': ['3', '6'],\n",
       " 'x4zmk05d': ['3'],\n",
       " 'lv5xjfk4': ['3'],\n",
       " 'g4qak0bu': ['3'],\n",
       " 'vuym41xv': ['3'],\n",
       " 'kxsy16bc': ['3'],\n",
       " 'r72my1p7': ['3', '32'],\n",
       " 'nhq0oq8y': ['3'],\n",
       " '8a351qol': ['3'],\n",
       " '97tb8eas': ['3'],\n",
       " 'xbnp6tgz': ['3'],\n",
       " 'vtvdk8qj': ['3'],\n",
       " 'ehe1uj7i': ['3'],\n",
       " '4e3cn50u': ['3', '40'],\n",
       " '5wakh15h': ['3'],\n",
       " '5onj6s3n': ['3'],\n",
       " 'layzzqf4': ['3'],\n",
       " 'wh9vvgv2': ['3'],\n",
       " 'zlro6tel': ['3'],\n",
       " '9yb9a9vz': ['4'],\n",
       " 'u9omhsgu': ['4', '21', '22'],\n",
       " '378cfb23': ['4', '21', '22'],\n",
       " 'vznb3puk': ['4'],\n",
       " 'd44coxr1': ['4', '38', '39'],\n",
       " 'f7cy5oad': ['4'],\n",
       " 'qfcfm40m': ['4'],\n",
       " 'gsqzmuj5': ['4'],\n",
       " 'svwkcuxf': ['4'],\n",
       " 'yxps5psk': ['4'],\n",
       " 'dtlwjndn': ['4'],\n",
       " '22vz79nz': ['4', '24'],\n",
       " 'kfw8ulm3': ['4'],\n",
       " 'oirh39hs': ['4'],\n",
       " 'yi1s6xap': ['4'],\n",
       " 'pbdotx5y': ['4'],\n",
       " 'qvhwypjt': ['4'],\n",
       " '7ksh8c5v': ['4'],\n",
       " 'j6v0zaa7': ['4', '21'],\n",
       " 'cvxni7cd': ['4'],\n",
       " 'y07xjnpe': ['4', '41'],\n",
       " 'lntg6yb8': ['4', '24'],\n",
       " 'rz0kbuqm': ['4'],\n",
       " 'igw288oj': ['4', '9', '19', '26', '27', '34', '41', '43', '44'],\n",
       " 'r63sg73c': ['4'],\n",
       " 'v7edybz4': ['4'],\n",
       " 'i8kwhhwt': ['4'],\n",
       " '7k36owrf': ['4'],\n",
       " 'q6wp6c08': ['4'],\n",
       " 'oqalb61j': ['4'],\n",
       " 'w94b4aft': ['4', '41'],\n",
       " 'g461ig4q': ['4'],\n",
       " 'o0j2wexx': ['4'],\n",
       " 'j8rtwy6l': ['4'],\n",
       " '5wazfkey': ['4'],\n",
       " '8efdzlc0': ['4'],\n",
       " 'w0ex7ynw': ['4', '38', '43'],\n",
       " 'vdpxhdfr': ['4'],\n",
       " '4nmw69co': ['4', '21'],\n",
       " 'cs2mgj70': ['4', '21'],\n",
       " 'ofuk5k9f': ['4'],\n",
       " 'pqzu10vu': ['4', '14', '23', '34'],\n",
       " 'tirw6l1c': ['4'],\n",
       " 'jxu3056n': ['4'],\n",
       " 'xeo2m4ij': ['4'],\n",
       " 'fcmzdcuh': ['4'],\n",
       " '68bzztwx': ['4', '14'],\n",
       " 'dumf55yg': ['4'],\n",
       " 'rnklbip9': ['4'],\n",
       " '8noxjjuh': ['4', '26', '43'],\n",
       " 'mnta55wf': ['4'],\n",
       " 'vzet2glz': ['4'],\n",
       " 'tr5whji0': ['4'],\n",
       " '9pdakwmb': ['4'],\n",
       " 'osbp6eau': ['4', '13'],\n",
       " 'tp6qq2pu': ['4'],\n",
       " '2v2r0dxs': ['4'],\n",
       " 'hj4wd5lc': ['4'],\n",
       " 'e2jdhkjp': ['4'],\n",
       " 'xsaj0fzu': ['4', '42'],\n",
       " 'bvrnjlly': ['4'],\n",
       " 'sct14hzu': ['4'],\n",
       " '8k6boijg': ['4'],\n",
       " 'nacessce': ['4'],\n",
       " '9i024s08': ['4', '31'],\n",
       " 'tj7z3ocr': ['4', '20'],\n",
       " 'pt1rxibl': ['4'],\n",
       " 'c2jm0g88': ['4', '24'],\n",
       " 'im9v9ezp': ['4'],\n",
       " 'jzc4ol7w': ['4'],\n",
       " 'cgjmyi5q': ['4'],\n",
       " 'ndke01xy': ['4'],\n",
       " '9819n3zf': ['4'],\n",
       " 'bu17n7zf': ['4', '31'],\n",
       " 'u4thvebq': ['4', '21'],\n",
       " 'dt0b87me': ['4', '21', '27'],\n",
       " 'yrv94oqw': ['4', '27'],\n",
       " '8om7z5cc': ['4'],\n",
       " '96tvnecq': ['4'],\n",
       " '1vcpq06y': ['4', '22'],\n",
       " '37i62atc': ['4'],\n",
       " '4di1p5v0': ['4', '27'],\n",
       " 'hf3paocz': ['4', '38'],\n",
       " 'hhd5f823': ['4', '26'],\n",
       " 'bzefannc': ['4'],\n",
       " 'hsxwz798': ['4', '34'],\n",
       " 'i6eqt6z8': ['4'],\n",
       " 'b8ce35l4': ['4'],\n",
       " 'ws1ysmae': ['4', '23'],\n",
       " '0udqwhx8': ['4'],\n",
       " 'tl1q4noo': ['4'],\n",
       " 'npk92gra': ['4', '42'],\n",
       " 'vyeescn8': ['5', '17', '33'],\n",
       " 'jd1b7asr': ['5', '17'],\n",
       " 'w8ta2qgh': ['5'],\n",
       " 'jbc74lcu': ['5'],\n",
       " 'fho2kble': ['5'],\n",
       " 'u5dajiqg': ['5', '19', '46'],\n",
       " 'er8qrkz3': ['5'],\n",
       " 'ai2zcke5': ['5', '17', '18', '19', '30'],\n",
       " 'kksrzvuk': ['5', '19', '30', '46'],\n",
       " 'fo07nxvi': ['5'],\n",
       " 'de45x8q4': ['5'],\n",
       " 'n3cg3w9v': ['5', '17', '19', '28', '46'],\n",
       " 'px2o98cv': ['5', '17', '19', '28', '46'],\n",
       " 'pnp8flc3': ['5'],\n",
       " '27f9241x': ['5'],\n",
       " 'kcyao9i9': ['5'],\n",
       " 'agc89fqv': ['5'],\n",
       " 'a2hd6d7s': ['5'],\n",
       " 'm5oq6ygu': ['5'],\n",
       " 'u3xxshqw': ['5', '29', '50'],\n",
       " '050fdtm8': ['5', '29', '50'],\n",
       " '3hgmuyn4': ['5'],\n",
       " '7joyaz7q': ['5'],\n",
       " '40fz5r90': ['5', '29'],\n",
       " 'ydywrk62': ['5'],\n",
       " 'kfal7g2v': ['5'],\n",
       " 'gpr86lxe': ['5'],\n",
       " 'zb434ve3': ['5', '29'],\n",
       " 'qilamdlp': ['5'],\n",
       " '44d5fsof': ['5'],\n",
       " 'hn12194u': ['5'],\n",
       " 'aexlwq79': ['5'],\n",
       " 'rirzes0m': ['5'],\n",
       " 'ywaefpe8': ['5'],\n",
       " 'yn7x9uit': ['5'],\n",
       " 'urhh03mn': ['5'],\n",
       " '2fs8oui2': ['5', '19'],\n",
       " '1ln02cyq': ['5'],\n",
       " '2kfmgr7k': ['5', '29', '33'],\n",
       " '5oyca0xl': ['5', '29'],\n",
       " 'qm0clntz': ['5'],\n",
       " 'f6xhrpi8': ['5', '29', '33'],\n",
       " 'fad1yyqx': ['5', '32'],\n",
       " 'sijzdlv5': ['5', '32'],\n",
       " 'pld0clpd': ['5'],\n",
       " 'howmcdh6': ['5', '29'],\n",
       " 'fcxjt7qf': ['5'],\n",
       " '0jr31q5g': ['5'],\n",
       " 'f2e4d3bz': ['5'],\n",
       " 'fgbilulc': ['5', '17'],\n",
       " 'm7dwbheg': ['5', '29'],\n",
       " 'c74zdtnn': ['5', '30'],\n",
       " 'yv2gjjzg': ['5', '29'],\n",
       " '458smf3g': ['5'],\n",
       " 'ij8wk06v': ['5'],\n",
       " 'h450b2l5': ['5'],\n",
       " 'wrzh32lh': ['5'],\n",
       " 'ibmpqwc6': ['5'],\n",
       " 'jgtbp5r6': ['5'],\n",
       " '7qd8z5e7': ['5', '50'],\n",
       " 'zzn74cjr': ['5'],\n",
       " 'mbv8vb2r': ['5'],\n",
       " 'untuy1e6': ['5'],\n",
       " '78vo1jkc': ['5'],\n",
       " 'jxha9b65': ['5', '29'],\n",
       " '7xp3szhz': ['5'],\n",
       " '64bxnvdg': ['5'],\n",
       " 'u87lx38h': ['5', '28'],\n",
       " 'kc2r034w': ['5', '19', '29'],\n",
       " 'eje3i558': ['5', '19', '29', '50'],\n",
       " 'bgm4dlwz': ['5'],\n",
       " 'knpctp4p': ['5'],\n",
       " 'hchioraj': ['5'],\n",
       " 'fd5rr87z': ['5'],\n",
       " '6nh28ntv': ['5'],\n",
       " '30tftimd': ['5', '50'],\n",
       " 'sj6ts2vo': ['5'],\n",
       " '5w66fkmi': ['5'],\n",
       " '3pcclaql': ['5'],\n",
       " '201ac32t': ['5'],\n",
       " '2qugltxu': ['5', '33'],\n",
       " '99v6mi9t': ['5'],\n",
       " '8ruux4sw': ['5', '33'],\n",
       " 'ey939w17': ['5', '33'],\n",
       " '7e8zlt3t': ['5'],\n",
       " 'ygygnmgm': ['5'],\n",
       " 'xxkagj6e': ['5', '33'],\n",
       " 'vzyrcmu4': ['5'],\n",
       " 'le3xa2p8': ['5', '33'],\n",
       " '3pxc5wot': ['5'],\n",
       " '5w1q57v2': ['5', '17', '28', '46'],\n",
       " '92jf137s': ['5', '32', '40', '50'],\n",
       " 'witgquy6': ['5'],\n",
       " 'd59dck5t': ['5'],\n",
       " 'aku5atqh': ['5', '29'],\n",
       " '60516gav': ['5'],\n",
       " '8gmn0tae': ['5'],\n",
       " '78zch9yh': ['5'],\n",
       " 'u3ow62fn': ['5'],\n",
       " 'x7v4y1ru': ['6'],\n",
       " 'r7glmi7p': ['6'],\n",
       " 'vzv0d5ry': ['6'],\n",
       " '0oak9ggm': ['6'],\n",
       " 'l6v2u8mj': ['6', '8'],\n",
       " 'icpwfyss': ['6'],\n",
       " 'qexn0nuy': ['6'],\n",
       " 'zyrzfm40': ['6'],\n",
       " '3taykdr1': ['6'],\n",
       " '1dr4r3n4': ['6'],\n",
       " 'qjma4rsp': ['6', '7'],\n",
       " 'uv06nfvz': ['6'],\n",
       " '6kpgt70s': ['6'],\n",
       " 'b7a2w4v9': ['6'],\n",
       " 'ikt8r2o1': ['6'],\n",
       " 'ij4ibrrx': ['6'],\n",
       " 'bqbj660w': ['6'],\n",
       " 'kjol6sgh': ['6'],\n",
       " '8y0v6d2i': ['6'],\n",
       " 'ertno6a5': ['6'],\n",
       " '401tisd3': ['6'],\n",
       " 'nswmhpzf': ['6'],\n",
       " 'e54b5cbs': ['6'],\n",
       " 'dci4jbyd': ['6'],\n",
       " '3ea1ngo2': ['6'],\n",
       " 'd5d10lwr': ['6'],\n",
       " 'hgc4ymok': ['6', '7'],\n",
       " '8hrjgcas': ['6'],\n",
       " 'q2b4ig1h': ['6'],\n",
       " 'k1adcls8': ['6'],\n",
       " 'knc0ruou': ['6'],\n",
       " 'tujxlve3': ['6', '8'],\n",
       " 'd7xq7x5g': ['6'],\n",
       " 'tb1zsuw4': ['6'],\n",
       " '4ipuy7su': ['6', '8'],\n",
       " '30umy2ai': ['6'],\n",
       " 'ey34e59f': ['6', '7'],\n",
       " '22kgwbz4': ['6'],\n",
       " 'onhiptn7': ['6'],\n",
       " 'hrhmtyfl': ['6'],\n",
       " 'tht1jixl': ['6'],\n",
       " '52pw2vwx': ['6'],\n",
       " 'h2mloxuh': ['6'],\n",
       " 'nm8a3jxt': ['6'],\n",
       " 'ccx61owl': ['6'],\n",
       " 'b3wp314u': ['6'],\n",
       " 'lqio7l8k': ['6'],\n",
       " 'yripn8gc': ['6'],\n",
       " '5enmq7do': ['6'],\n",
       " 'kcgfcnpu': ['6'],\n",
       " '8cppx8vs': ['6'],\n",
       " 'r3lgpdqq': ['6'],\n",
       " '1wcltcpr': ['6'],\n",
       " 'opdva99w': ['6', '8'],\n",
       " 'xa1v5t63': ['6'],\n",
       " '7ur8hr23': ['6'],\n",
       " 'kuk0scmr': ['6'],\n",
       " 'sy91rnse': ['6'],\n",
       " 'n642wlsx': ['6'],\n",
       " '56tmg38b': ['6'],\n",
       " 'n9ju3flv': ['6'],\n",
       " 'rk8pract': ['6'],\n",
       " 'p21ysly7': ['6'],\n",
       " '50cir0l6': ['6'],\n",
       " 'lul3udhv': ['6'],\n",
       " 'rtywrjo1': ['6'],\n",
       " 'e1oinu71': ['6'],\n",
       " 'wt5o9sgm': ['6'],\n",
       " '77jpm4o0': ['6'],\n",
       " 'hfdh571v': ['6'],\n",
       " '8vp57c1o': ['6'],\n",
       " 'jeufhpkq': ['6', '8'],\n",
       " 'vy0x9hvr': ['6'],\n",
       " 'r34d0rx7': ['6'],\n",
       " 'qdhw9g3h': ['6'],\n",
       " '6201nhwx': ['6'],\n",
       " 'zzslm90b': ['6'],\n",
       " 'qyp629f6': ['6'],\n",
       " 'zcmzpat5': ['6'],\n",
       " 'afcd4wc9': ['6'],\n",
       " 'heywm7kn': ['6'],\n",
       " 'jcr27h1c': ['6'],\n",
       " '5mf1r24m': ['6'],\n",
       " 'gzrjca1l': ['6', '8'],\n",
       " 'vn0xu1cn': ['6', '8'],\n",
       " 't3y1w9ef': ['6'],\n",
       " 'kx0u8w10': ['6'],\n",
       " '846t2t8g': ['6'],\n",
       " 'afdmegq0': ['6'],\n",
       " '0hxan9rw': ['6'],\n",
       " '89asskfx': ['6'],\n",
       " 'm95bmi9t': ['6'],\n",
       " 'tnnhx51r': ['6'],\n",
       " 'sdde8bgd': ['6'],\n",
       " '28ezisog': ['6'],\n",
       " 'ino9srb6': ['6'],\n",
       " 'cxq9v218': ['6'],\n",
       " 'tmgmqtjq': ['6'],\n",
       " 'us5ufqeg': ['6'],\n",
       " '77rcr30x': ['7'],\n",
       " 'vwy25qah': ['7'],\n",
       " 'fn9t38as': ['7'],\n",
       " '01mo6yo9': ['7'],\n",
       " 'yy4bnb4o': ['7'],\n",
       " 'hn69l5mq': ['7'],\n",
       " 'ofd2ipvs': ['7'],\n",
       " 'm9yv4qkm': ['7'],\n",
       " 'w0ceb715': ['7'],\n",
       " 'bt4oi1xf': ['7'],\n",
       " 'blrfn8qh': ['7'],\n",
       " 'q7mm4nwm': ['7'],\n",
       " 'cu52j6tq': ['7'],\n",
       " '5najee33': ['7'],\n",
       " 'cfdfjwdz': ['7'],\n",
       " '1o8qlqln': ['7'],\n",
       " 'e28phe9y': ['7', '15', '16'],\n",
       " '8dm8k3zs': ['7'],\n",
       " 'z38ywg1m': ['7'],\n",
       " 'm400cal3': ['7'],\n",
       " 'mcgpm9ll': ['7'],\n",
       " '15rcq4sh': ['7'],\n",
       " 'o0y0xiq6': ['7', '31'],\n",
       " '83odb66l': ['7'],\n",
       " '9595vm0k': ['7'],\n",
       " 'wgc6ch47': ['7'],\n",
       " '91pb9nq3': ['7'],\n",
       " 'rgls0xgn': ['7'],\n",
       " 'hnbxfbeo': ['7'],\n",
       " 'mryazbnq': ['7'],\n",
       " '1vd4zcad': ['7'],\n",
       " 'u79trvqf': ['7'],\n",
       " 'tfovpo7h': ['7'],\n",
       " 'qfsxuyow': ['7'],\n",
       " '79bzd4nl': ['7'],\n",
       " '2r9jlejw': ['7'],\n",
       " 'j5ag12zr': ['7'],\n",
       " 'hxial3gi': ['7'],\n",
       " 'pn9voawj': ['7'],\n",
       " 'f6rmxcib': ['7'],\n",
       " 'pncedv8d': ['7'],\n",
       " 'scd3f8vk': ['7'],\n",
       " 'lxwm71jp': ['7'],\n",
       " 'pjyfz0pm': ['7'],\n",
       " 'dza80e8o': ['7'],\n",
       " 'lsb1s8y7': ['7'],\n",
       " 'r8vgj7m5': ['7'],\n",
       " '5ig5upot': ['7'],\n",
       " 'pvo61ay8': ['7'],\n",
       " 'nrtrhq1f': ['7'],\n",
       " 'nxy21x7k': ['7'],\n",
       " '40fvjskj': ['7', '49'],\n",
       " 'zw1f1s16': ['7', '49'],\n",
       " '9skvbk8m': ['7'],\n",
       " '2hv5ohsf': ['7'],\n",
       " 'ir3k848j': ['7'],\n",
       " '86kslari': ['7'],\n",
       " '576xiuz8': ['7'],\n",
       " '80qftge0': ['7'],\n",
       " 'zqjiiy4l': ['7'],\n",
       " 'oihyz5v7': ['7'],\n",
       " '0jl6qu0i': ['7'],\n",
       " '0yj3xp9s': ['7'],\n",
       " '7e79o2gy': ['7'],\n",
       " '5xm7cwjz': ['7'],\n",
       " 'wf5cozst': ['7'],\n",
       " 'ydbi0km2': ['7'],\n",
       " 'd0y3pr9a': ['7'],\n",
       " 'b1qnfl41': ['7'],\n",
       " '9mvk86q6': ['7'],\n",
       " 'pfqkygtq': ['7'],\n",
       " 'lw12h047': ['7'],\n",
       " '7gfk554m': ['7'],\n",
       " '4070r4fc': ['7'],\n",
       " 'es2zdqf5': ['7'],\n",
       " 'peq6kyyz': ['7'],\n",
       " 'dyutsd8t': ['7'],\n",
       " 'uxeaaski': ['7'],\n",
       " '95v0nfme': ['7'],\n",
       " 'qud2syw4': ['7'],\n",
       " '3g67hjk8': ['7'],\n",
       " 'q2o8dlqw': ['7'],\n",
       " '0w7tq79d': ['7'],\n",
       " '8b5e0s5x': ['7'],\n",
       " '8cg5yj20': ['7'],\n",
       " '84yjdlab': ['7'],\n",
       " '82iy2prw': ['7'],\n",
       " 'ejfp8gqz': ['7'],\n",
       " '83pf7qwm': ['7'],\n",
       " 'cv2dg9tg': ['7'],\n",
       " 'czxm7so2': ['7'],\n",
       " '72r0kg17': ['7'],\n",
       " '6r0gfdma': ['7'],\n",
       " 'd04e7d1x': ['7'],\n",
       " 'nlb9up9i': ['7'],\n",
       " '8fuj9td2': ['7'],\n",
       " 'y7h2qm83': ['7'],\n",
       " 'szvb2gsf': ['8'],\n",
       " 'qcd9jl7m': ['8'],\n",
       " '4kkzi70v': ['8'],\n",
       " '3hazdvbl': ['8'],\n",
       " 'l8bv5t3o': ['8'],\n",
       " 'li8kvzdh': ['8', '27'],\n",
       " '8wg27hcu': ['8'],\n",
       " 'ywvk6gu6': ['8'],\n",
       " '5k55lmfm': ['8'],\n",
       " '8cw3bjxh': ['8'],\n",
       " '63ihwrpe': ['8'],\n",
       " '5u41um2o': ['8'],\n",
       " 'goup6xtg': ['8'],\n",
       " '3nipr212': ['8'],\n",
       " 'cdo1z1tc': ['8', '14', '42', '44'],\n",
       " 'kzgjnpz9': ['8'],\n",
       " 'elieuuuy': ['8'],\n",
       " 'gunn55f9': ['8', '14'],\n",
       " 't2utj1a6': ['8'],\n",
       " 'hijwxpig': ['8'],\n",
       " '90tccg03': ['8'],\n",
       " 'e2zeh8cl': ['8'],\n",
       " '6g7mijbz': ['8'],\n",
       " 'vvm1gfku': ['8'],\n",
       " 'cxoom0tu': ['8'],\n",
       " 'gm0bc40a': ['8'],\n",
       " 'z19ri377': ['8'],\n",
       " 'ooy27g9n': ['8'],\n",
       " '32h1o70a': ['8'],\n",
       " '0bm3bimr': ['8'],\n",
       " 'qziln7b6': ['8'],\n",
       " 'eerqrqwk': ['8', '27'],\n",
       " '2jttlljm': ['8'],\n",
       " 'qg4xl5w8': ['8'],\n",
       " '4c9sa9g3': ['8'],\n",
       " 'cdlcv1c9': ['8'],\n",
       " 'ky5gt3pu': ['8'],\n",
       " '693zeq60': ['8'],\n",
       " 'ole70vk0': ['8'],\n",
       " 'gabqrgk5': ['8'],\n",
       " 'x6j9wvgt': ['8'],\n",
       " 'xpizewde': ['8'],\n",
       " '17oac3bg': ['8'],\n",
       " 'mo66jgmb': ['8', '19'],\n",
       " 'jfui45ya': ['8', '19'],\n",
       " '1xjgtj0t': ['8'],\n",
       " 'hzi3kvw5': ['8'],\n",
       " '7bvsf5dk': ['8'],\n",
       " 'mskkahwi': ['8'],\n",
       " 'g8h6trql': ['8'],\n",
       " 'pusz88xy': ['8'],\n",
       " '3p5cxv4k': ['8'],\n",
       " '23f96fwj': ['8'],\n",
       " 'zjnr7vwm': ['8'],\n",
       " '0u7j42i2': ['8'],\n",
       " 'ymlfitp0': ['8', '35'],\n",
       " 'eiofivki': ['8'],\n",
       " 'nzbopcxg': ['8'],\n",
       " 'cb5ebiiv': ['8', '10'],\n",
       " 'dbzpcy5v': ['8'],\n",
       " 'ybpdwbja': ['8'],\n",
       " 'f5u6lrjz': ['8'],\n",
       " 'kk7qyn4w': ['8'],\n",
       " 'lbtrtzwn': ['8'],\n",
       " 'wkd9vivw': ['8'],\n",
       " 'urapl8c6': ['8'],\n",
       " 'plbtnzej': ['8'],\n",
       " 'bm7q56mp': ['8'],\n",
       " '0l2dhz7v': ['8'],\n",
       " 'npx45ldn': ['8'],\n",
       " '13pru6p8': ['8'],\n",
       " 'xe62fehm': ['8'],\n",
       " '6ocajatf': ['8'],\n",
       " '1xnfkbxu': ['8'],\n",
       " '6ob3u2e2': ['8'],\n",
       " 'ztc4wvlw': ['8'],\n",
       " 'ylxh8vi2': ['8'],\n",
       " 'fdwll5r5': ['8'],\n",
       " '72vrtuea': ['8'],\n",
       " '50ml630e': ['8', '48'],\n",
       " '7gsl7d3f': ['8'],\n",
       " '5nb02qlf': ['8'],\n",
       " 'kiq6xb6k': ['8'],\n",
       " '9edp04ud': ['8'],\n",
       " 'sflu2was': ['8'],\n",
       " 'kkknp1e6': ['8'],\n",
       " 'ax9btc74': ['8'],\n",
       " 'rcmo5ixl': ['8'],\n",
       " 'jpc9bx1v': ['8'],\n",
       " 'kusqyumn': ['8'],\n",
       " 'qtkwciym': ['8'],\n",
       " 'orh8fd1c': ['8'],\n",
       " 'js77ymx4': ['9'],\n",
       " '4crdebgq': ['9'],\n",
       " '6ngsyo8o': ['9', '45'],\n",
       " '4q1j6g44': ['9', '45'],\n",
       " 'qunw5hhy': ['9'],\n",
       " '06v5bf01': ['9'],\n",
       " 'yuv2ki4o': ['9'],\n",
       " 'u8hogdyf': ['9'],\n",
       " 'lf5xg8p3': ['9', '45'],\n",
       " 'in2edn29': ['9'],\n",
       " 'u54kja4g': ['9'],\n",
       " 'qy66ih4m': ['9'],\n",
       " 'lz0chxab': ['9'],\n",
       " 'eo887olu': ['9'],\n",
       " 'no8y1ior': ['9'],\n",
       " 'xtudcufl': ['9'],\n",
       " 'dw2kljya': ['9'],\n",
       " 'xm9y4ovl': ['9'],\n",
       " 'ukommd1e': ['9', '43', '45'],\n",
       " '90t0jmwf': ['9', '45'],\n",
       " 'el11o2cg': ['9'],\n",
       " 'kshgjhzh': ['9'],\n",
       " 'gpxxdebo': ['9'],\n",
       " '2pmpa7n0': ['9'],\n",
       " 'h588i68g': ['9'],\n",
       " 'v93mde6l': ['9'],\n",
       " '7g12596l': ['9'],\n",
       " '7g2kg2tz': ['9'],\n",
       " 'o4uusbix': ['9'],\n",
       " '51g3vhcx': ['9', '10'],\n",
       " '1gjo837f': ['9'],\n",
       " 'wmfcey6f': ['9'],\n",
       " 'oek9ud99': ['9'],\n",
       " 'v346cpkl': ['9'],\n",
       " '4kojv5pv': ['9'],\n",
       " 'cn6e4vjg': ['9'],\n",
       " 'dg6o8hz7': ['9'],\n",
       " 'dn2jkkec': ['9'],\n",
       " 'fbq90100': ['9'],\n",
       " 'glah2z2m': ['9'],\n",
       " '12eny1vl': ['9'],\n",
       " 'vx5rd8hs': ['9'],\n",
       " 'n22oerlx': ['9'],\n",
       " '9mvn6bqu': ['9'],\n",
       " 'pigamjae': ['9'],\n",
       " '3eysv9sl': ['9'],\n",
       " '9hbgn66l': ['9'],\n",
       " 'ulav1vcf': ['9'],\n",
       " 'scjui91o': ['9'],\n",
       " 'h6g65f9w': ['9'],\n",
       " 'rku20ctp': ['9'],\n",
       " 'w1hzhxv6': ['9'],\n",
       " 'ux0kjt9l': ['9'],\n",
       " 'j8x384vp': ['9'],\n",
       " '8wb6u7dc': ['9'],\n",
       " 'tz5jajeb': ['9'],\n",
       " 'mdt11ba5': ['9'],\n",
       " 'u2upcaod': ['9'],\n",
       " '0u84hs2g': ['9'],\n",
       " 'iwtf71qc': ['9', '43'],\n",
       " '8obqdbh9': ['9'],\n",
       " '8nvrt5y7': ['9'],\n",
       " 'm8a98j99': ['9'],\n",
       " '3sx073k9': ['9'],\n",
       " 'y8ntig41': ['9'],\n",
       " '4l467nxw': ['9'],\n",
       " 'rkhrusdc': ['9', '35'],\n",
       " 'fjy2exp6': ['9'],\n",
       " '4v0qgfat': ['9'],\n",
       " '7dhx8f4x': ['9'],\n",
       " '9ffvdgon': ['9'],\n",
       " 'ucxin4ke': ['9'],\n",
       " 'daalqip2': ['9'],\n",
       " 'e4pr78n0': ['9'],\n",
       " 'avr88y6e': ['9'],\n",
       " 'vlzbri33': ['9'],\n",
       " 'mh23h29f': ['9'],\n",
       " '5cm6122n': ['9'],\n",
       " 'scbteel5': ['9'],\n",
       " 'itefsyv4': ['9', '45'],\n",
       " 'lg0th1gl': ['9'],\n",
       " 'y1gfz1ub': ['9'],\n",
       " '54obt430': ['9'],\n",
       " '7mvrxr6a': ['9'],\n",
       " 'n8omnki4': ['9'],\n",
       " 's623p0n6': ['9'],\n",
       " 'r0ach2jn': ['9'],\n",
       " '68spkk3y': ['9'],\n",
       " 'memywn0w': ['9'],\n",
       " '678df7th': ['9'],\n",
       " 'xh9x9qam': ['9'],\n",
       " 'ms2edk3w': ['9'],\n",
       " '33uqbusb': ['9', '41'],\n",
       " 'm3ikl4da': ['9'],\n",
       " 'trn2deod': ['10'],\n",
       " 'all7ocnd': ['10'],\n",
       " 'pn02p843': ['10'],\n",
       " '71b6ai77': ['10'],\n",
       " 'vdrd74nz': ['10', '44'],\n",
       " 'c0i11t0a': ['10'],\n",
       " '3nanf73b': ['10'],\n",
       " 'qm2qvcwk': ['10'],\n",
       " 'z82gogvs': ['10'],\n",
       " 'noyx53ej': ['10'],\n",
       " 'qekxxsfy': ['10'],\n",
       " '883y61q5': ['10', '44'],\n",
       " 'vfnpeuzr': ['10', '48'],\n",
       " '5tx03o6h': ['10', '18', '44'],\n",
       " 'tnecig63': ['10'],\n",
       " 'n4kke717': ['10'],\n",
       " 'f6rx4h3r': ['10'],\n",
       " 'zkjiixuq': ['10', '48'],\n",
       " 'gilpb3oy': ['10'],\n",
       " 'rzh7ja6a': ['10'],\n",
       " 'zukjh1hr': ['10', '44'],\n",
       " 'c7l88gjn': ['10', '45'],\n",
       " 'rziqtdp0': ['10'],\n",
       " '3yi44xfe': ['10', '44'],\n",
       " 'cm41gpx1': ['10'],\n",
       " '80d9p4j8': ['10'],\n",
       " 'qpufvt3o': ['10'],\n",
       " 'xcxjc3po': ['10'],\n",
       " 'rmv5kg3w': ['10'],\n",
       " 'k208rpth': ['10'],\n",
       " 'qpzg8lam': ['10'],\n",
       " 'c5lankxq': ['10'],\n",
       " 'v941u1t1': ['10'],\n",
       " 'gkkvu8d5': ['10'],\n",
       " 'hipujblu': ['10'],\n",
       " 'x9zg7ulr': ['10'],\n",
       " '3mk5a5u6': ['10'],\n",
       " 'ocwetgv6': ['10'],\n",
       " 'esa5700v': ['10'],\n",
       " 'u9fmokcf': ['10'],\n",
       " 't7p2j504': ['10'],\n",
       " 'i1naum3h': ['10'],\n",
       " '43w659t1': ['10'],\n",
       " 'j5jr3cvw': ['10'],\n",
       " 'o8k66pes': ['10'],\n",
       " 'x58m0e2t': ['10'],\n",
       " '4ilkb96l': ['10'],\n",
       " 'k8eqzr6w': ['10'],\n",
       " 'aiah8kkn': ['10'],\n",
       " 'wolrseqc': ['10'],\n",
       " '4u7qtoz7': ['10'],\n",
       " 'yu2qzp18': ['10'],\n",
       " 'li3slzw6': ['10'],\n",
       " 'qqsefagq': ['10'],\n",
       " 'gz18sxzc': ['10'],\n",
       " 'gwh3vc9y': ['10'],\n",
       " '4je9wl1o': ['10'],\n",
       " 'v8l1caqh': ['10'],\n",
       " 'c5t4lm4d': ['10'],\n",
       " '0b6dsdct': ['10'],\n",
       " 'ls408b2b': ['10'],\n",
       " 'ekw3nlw3': ['10'],\n",
       " 'ild68h52': ['10'],\n",
       " 's74jgknw': ['10'],\n",
       " '0o79m08j': ['10'],\n",
       " '45k6tp0b': ['10', '48'],\n",
       " 'mhmno6vb': ['10'],\n",
       " 'ghh16h43': ['10'],\n",
       " 'eaywkg26': ['10'],\n",
       " 'maxvppn8': ['10'],\n",
       " 'kdbw5k7w': ['10'],\n",
       " 'jmhexmet': ['10'],\n",
       " 'hs38s0vc': ['10'],\n",
       " 'in3pyyue': ['10'],\n",
       " 'swwrokfm': ['10'],\n",
       " 'omygp8bu': ['10'],\n",
       " '3cke9x69': ['10'],\n",
       " 'xfjexm5b': ['10', '44'],\n",
       " '4lgy9te8': ['10'],\n",
       " 'zzumi9h5': ['10', '18', '44'],\n",
       " 'jhavycsq': ['10'],\n",
       " '3aaw7toy': ['10'],\n",
       " '77ebl84a': ['10'],\n",
       " 'xcfk4efo': ['10'],\n",
       " 'df64x2eu': ['10'],\n",
       " '8ko7mmu3': ['10', '18', '44'],\n",
       " '0a49okho': ['10'],\n",
       " 'b8f4a7o3': ['10'],\n",
       " 'f8w5q86d': ['10'],\n",
       " 'h62xj47p': ['10'],\n",
       " 'k574jppq': ['10'],\n",
       " 'ldeqvajo': ['10'],\n",
       " 'fac4ic30': ['10'],\n",
       " 'rqotfwm3': ['10'],\n",
       " '9bi6pobg': ['10'],\n",
       " 'f1ckv4bk': ['10'],\n",
       " '2fdtwu8o': ['10'],\n",
       " 'cqrldx58': ['10'],\n",
       " 'x0me00m0': ['11'],\n",
       " 'mu0g857x': ['11'],\n",
       " 'caahtt3b': ['11', '12'],\n",
       " 'eazqdlds': ['11'],\n",
       " 'yomkl4pp': ['11'],\n",
       " '8c1s89dc': ['11'],\n",
       " 'qskqikk9': ['11'],\n",
       " '1cpk2p9g': ['11', '18'],\n",
       " '8tbxto22': ['11', '18'],\n",
       " 'ro20ibaq': ['11'],\n",
       " 'qjd31m8l': ['11'],\n",
       " 'io7ozfpi': ['11'],\n",
       " '4snfwx8r': ['11'],\n",
       " 'gyghpk64': ['11'],\n",
       " '37zwfexp': ['11'],\n",
       " 'f2c3dfve': ['11'],\n",
       " 'qsxsrhun': ['11', '12'],\n",
       " 'jv3xx09f': ['11'],\n",
       " '4jag7lzb': ['11'],\n",
       " 'gv9q7sf1': ['11'],\n",
       " '7jxwn2y6': ['11'],\n",
       " 'wydt1ykq': ['11'],\n",
       " 'wm2qibmr': ['11'],\n",
       " 'hp6lf65q': ['11'],\n",
       " 'ee7ojrrf': ['11', '12'],\n",
       " 'b6f6hek8': ['11'],\n",
       " '26jxvscc': ['11'],\n",
       " 'lrd0jm9i': ['11'],\n",
       " 'e16jvk6a': ['11'],\n",
       " 'tft6pybe': ['11'],\n",
       " '1i9z1agk': ['11'],\n",
       " ...}"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "retrieval_results # document가 top-100에 retrieve된 query"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0b95fd5f-96db-41e3-971b-0c0cf00ddbbf",
   "metadata": {},
   "source": [
    "## Load Qrels"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "83853f86-171f-49a4-87c9-91282ff41d54",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "qrels = pd.read_csv(\"/gallery_louvre/dayoon.ko/research/sds/src/datasets/trec-covid/qrels/test.tsv\", sep=\"\\t\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "91b04ccf-a581-4ff6-9553-37fbeaeb8801",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>query-id</th>\n",
       "      <th>corpus-id</th>\n",
       "      <th>score</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>005b2j4b</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>00fmeepz</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>1</td>\n",
       "      <td>g7dhmyyo</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>1</td>\n",
       "      <td>0194oljo</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>1</td>\n",
       "      <td>021q9884</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>1</td>\n",
       "      <td>02f0opkr</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>1</td>\n",
       "      <td>047xpt2c</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>1</td>\n",
       "      <td>04ftw7k9</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>1</td>\n",
       "      <td>pl9ht0d0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>1</td>\n",
       "      <td>05vx82oo</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   query-id corpus-id  score\n",
       "0         1  005b2j4b      2\n",
       "1         1  00fmeepz      1\n",
       "2         1  g7dhmyyo      2\n",
       "3         1  0194oljo      1\n",
       "4         1  021q9884      1\n",
       "5         1  02f0opkr      1\n",
       "6         1  047xpt2c      0\n",
       "7         1  04ftw7k9      0\n",
       "8         1  pl9ht0d0      0\n",
       "9         1  05vx82oo      0"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "qrels.head(10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "2e70089c-38be-46e7-9262-1fe5387ffa8e",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>query-id</th>\n",
       "      <th>corpus-id</th>\n",
       "      <th>score</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>005b2j4b</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>61089</th>\n",
       "      <td>45</td>\n",
       "      <td>006k39tj</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43576</th>\n",
       "      <td>31</td>\n",
       "      <td>00fmeepz</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>53369</th>\n",
       "      <td>38</td>\n",
       "      <td>00qk10im</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18656</th>\n",
       "      <td>13</td>\n",
       "      <td>00rq0ggi</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>51077</th>\n",
       "      <td>36</td>\n",
       "      <td>00z7x46i</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17083</th>\n",
       "      <td>12</td>\n",
       "      <td>011k6mm0</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37288</th>\n",
       "      <td>26</td>\n",
       "      <td>011k6mm0</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>63298</th>\n",
       "      <td>47</td>\n",
       "      <td>012f0g4y</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>59950</th>\n",
       "      <td>44</td>\n",
       "      <td>019lj813</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "       query-id corpus-id  score\n",
       "0             1  005b2j4b      2\n",
       "61089        45  006k39tj      2\n",
       "43576        31  00fmeepz      2\n",
       "53369        38  00qk10im      2\n",
       "18656        13  00rq0ggi      2\n",
       "51077        36  00z7x46i      2\n",
       "17083        12  011k6mm0      2\n",
       "37288        26  011k6mm0      2\n",
       "63298        47  012f0g4y      2\n",
       "59950        44  019lj813      2"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# qrels = qrels[qrels[\"score\"] > 0].sort_values(\"corpus-id\")\n",
    "qrels = qrels[qrels[\"score\"] == 2].sort_values(\"corpus-id\")\n",
    "qrels.head(10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "fa283f65-267e-4566-92ab-e1750e01d969",
   "metadata": {},
   "outputs": [],
   "source": [
    "gt = {}\n",
    "for k in qrels[\"corpus-id\"].unique():\n",
    "    qrels_ = qrels[qrels[\"corpus-id\"] == k]\n",
    "    v = qrels_[\"query-id\"].tolist()\n",
    "    gt[k] = v"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "324895b3-0728-4e35-8558-6135c6ea942f",
   "metadata": {},
   "outputs": [],
   "source": [
    "# retrieval_results :  # document가 top-100에 retrieve된 query\n",
    "# gt : document가 실제 evidence text인 query"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "a4cd2fda-0c13-4497-be8f-8e6e9dffc084",
   "metadata": {},
   "outputs": [],
   "source": [
    "corpus_ids = []\n",
    "num_gts = []\n",
    "tp_over_t = [] # retrieval 돼야하는데 안된 경우\n",
    "tp_over_p = [] # retrieval 안돼야하는데 된 경우\n",
    "for k, v in retrieval_results.items():\n",
    "    try:\n",
    "        gt_v = [str(i) for i in gt[k]]\n",
    "        # gt_v: document 가 retrieval 되어야하는 모든 쿼리 / v: document 가 실제 retrieval 된 쿼리\n",
    "        tp_t = len(set(gt_v).intersection(set(v))) / len(gt_v) # True Positive / # True\n",
    "        tp_p = len(set(gt_v).intersection(set(v))) / len(v) # True Positive / # Positive\n",
    "        corpus_ids.append(k)\n",
    "        num_gts.append(len(gt_v))\n",
    "        tp_over_t.append(tp_t)\n",
    "        tp_over_p.append(tp_p)\n",
    "    except:\n",
    "        continue"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "ccf11455-8ba0-46d5-8022-8f1c7644d643",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>corpus-id</th>\n",
       "      <th>num-positive-query-gt</th>\n",
       "      <th>tp_over_t</th>\n",
       "      <th>tp_over_p</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>n15i01tn</td>\n",
       "      <td>1</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>k9lcpjyo</td>\n",
       "      <td>2</td>\n",
       "      <td>0.5</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>jbtrdvhe</td>\n",
       "      <td>2</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>juz9jnfk</td>\n",
       "      <td>2</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>t1iagum7</td>\n",
       "      <td>1</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  corpus-id  num-positive-query-gt  tp_over_t  tp_over_p\n",
       "0  n15i01tn                      1        0.0        0.0\n",
       "1  k9lcpjyo                      2        0.5        1.0\n",
       "2  jbtrdvhe                      2        0.0        0.0\n",
       "3  juz9jnfk                      2        1.0        1.0\n",
       "4  t1iagum7                      1        1.0        1.0"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data = {\n",
    "        \"corpus-id\": corpus_ids, \n",
    "        \"num-positive-query-gt\": num_gts,\n",
    "        \"tp_over_t\": tp_over_t,\n",
    "        \"tp_over_p\": tp_over_p\n",
    "       }\n",
    "df = pd.DataFrame(data)\n",
    "df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "2b1825e8-32d5-4202-a7ed-82c0b6964b7e",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "15\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>corpus-id</th>\n",
       "      <th>num-positive-query-gt</th>\n",
       "      <th>tp_over_t</th>\n",
       "      <th>tp_over_p</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>rbzl0txp</td>\n",
       "      <td>3</td>\n",
       "      <td>0.666667</td>\n",
       "      <td>0.666667</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>124</th>\n",
       "      <td>dptgg05n</td>\n",
       "      <td>3</td>\n",
       "      <td>0.666667</td>\n",
       "      <td>1.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>657</th>\n",
       "      <td>k9xhphpl</td>\n",
       "      <td>3</td>\n",
       "      <td>0.666667</td>\n",
       "      <td>1.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>794</th>\n",
       "      <td>gey0nidn</td>\n",
       "      <td>3</td>\n",
       "      <td>0.666667</td>\n",
       "      <td>1.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>801</th>\n",
       "      <td>5wsj003j</td>\n",
       "      <td>3</td>\n",
       "      <td>0.666667</td>\n",
       "      <td>1.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>872</th>\n",
       "      <td>n4dgqo73</td>\n",
       "      <td>3</td>\n",
       "      <td>0.666667</td>\n",
       "      <td>1.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1505</th>\n",
       "      <td>1sbnewog</td>\n",
       "      <td>3</td>\n",
       "      <td>0.666667</td>\n",
       "      <td>1.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1509</th>\n",
       "      <td>iljmhdd1</td>\n",
       "      <td>3</td>\n",
       "      <td>0.666667</td>\n",
       "      <td>1.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1519</th>\n",
       "      <td>02cfyuf4</td>\n",
       "      <td>3</td>\n",
       "      <td>0.666667</td>\n",
       "      <td>1.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1527</th>\n",
       "      <td>qopcs6jy</td>\n",
       "      <td>3</td>\n",
       "      <td>0.666667</td>\n",
       "      <td>0.666667</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "     corpus-id  num-positive-query-gt  tp_over_t  tp_over_p\n",
       "47    rbzl0txp                      3   0.666667   0.666667\n",
       "124   dptgg05n                      3   0.666667   1.000000\n",
       "657   k9xhphpl                      3   0.666667   1.000000\n",
       "794   gey0nidn                      3   0.666667   1.000000\n",
       "801   5wsj003j                      3   0.666667   1.000000\n",
       "872   n4dgqo73                      3   0.666667   1.000000\n",
       "1505  1sbnewog                      3   0.666667   1.000000\n",
       "1509  iljmhdd1                      3   0.666667   1.000000\n",
       "1519  02cfyuf4                      3   0.666667   1.000000\n",
       "1527  qopcs6jy                      3   0.666667   0.666667"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_suc = df[df[\"tp_over_t\"] > 0.5]\n",
    "df_suc = df_suc[df_suc[\"num-positive-query-gt\"] > 2]\n",
    "df_suc = df_suc.sort_values(\"tp_over_t\")\n",
    "print(len(df_suc))\n",
    "df_suc.head(10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "a0788d21-8c69-471e-8059-147902bb4977",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'SARS-CoV-2 has caused tens of thousands of infections and more than one thousand deaths. There are currently no registered therapies for treating coronavirus infections. Because of time consuming process of new drug development, drug repositioning may be the only solution to the epidemic of sudden infectious diseases. We systematically analyzed all the proteins encoded by SARS-CoV-2 genes, compared them with proteins from other coronaviruses, predicted their structures, and built 19 structures that could be done by homology modeling. By performing target-based virtual ligand screening, a total of 21 targets (including two human targets) were screened against compound libraries including ZINC drug database and our own database of natural products. Structure and screening results of important targets such as 3-chymotrypsin-like protease (3CLpro), Spike, RNA-dependent RNA polymerase (RdRp), and papain like protease (PLpro) were discussed in detail. In addition, a database of 78 commonly used anti-viral drugs including those currently on the market and undergoing clinical trials for SARS-CoV-2 was constructed. Possible targets of these compounds and potential drugs acting on a certain target were predicted. This study will provide new lead compounds and targets for further in vitro and in vivo studies of SARS-CoV-2, new insights for those drugs currently ongoing clinical studies, and also possible new strategies for drug repositioning to treat SARS-CoV-2 infections.'"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "corpus_dict[\"i1lyno9g\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "1dc835ef-3595-462d-bf55-b210a4842e20",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "21\n"
     ]
    }
   ],
   "source": [
    "df_fail = df[df[\"num-positive-query-gt\"] > 2]\n",
    "df_fail = df_fail[df_fail[\"tp_over_t\"] < 0.01]\n",
    "df_fail = df_fail.sort_values(\"tp_over_t\")\n",
    "print(len(df_fail))\n",
    "# df_fail.tail(38)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "f2fc90ac-81f0-4ef1-b265-366bb5376bd5",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "import random\n",
    "fail_id_list = random.sample(df_fail['corpus-id'].tolist(), 5)\n",
    "# fail_id_list\n",
    "succ_id_list = random.sample(df_suc['corpus-id'].tolist(), 5)\n",
    "# succ_id_list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "bc9abce9-3a61-4eba-9364-8ed934916f35",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Failure list\n",
      "lnys6iuu\n",
      "The ongoing global pandemic of infection disease COVID-19 caused by the 2019 novel coronavirus (SARS-COV-2, formerly 2019-nCoV) presents critical threats to public health and the economy since it was identified in China, December 2019. The genome of SARS-CoV-2 had been sequenced and structurally annotated, yet little is known of the intrinsic organization and evolution of the genome. To this end, we present a mathematical method for the genomic spectrum, a kind of barcode, of SARS-CoV-2 and common human coronaviruses. The genomic spectrum is constructed according to the periodic distributions of nucleotides, and therefore reflects the unique characteristics of the genome. The results demonstrate that coronavirus SARS-CoV-2 exhibits dinucleotide TT islands in the non-structural proteins 3, 4, 5, and 6. Further analysis of the dinucleotide regions suggests that the dinucleotide repeats are increased during evolution and may confer the evolutionary fitness of the virus. The special dinucleotide regions in the SARS-CoV-2 genome identified in this study may become diagnostic and pharmaceutical targets in monitoring and curing the COVID-19 disease.\n",
      "\n",
      "qvz63m93\n",
      "Background COVID-19 is a disease with global public health emergency that have shook the world since its’ first detection in China in December, 2019. Severe acute respiratory syndrome Coronavirus 2 (SARS-CoV-2) is the pathogen responsible behind this pandemic. The lethality of different viral strains is found to vary in different geographical locations but the molecular mechanism is yet to be known. Methods Available data of whole genome sequencing of different viral strains published by different countries were retrieved and then analysed using Multiple Sequence Alignment and Pair-wise Sequence Alignment leading to Phylogenetic tree construction. Each location and the corresponding genetic variations were screened in depth. Then the variations are analysed at protein level giving special emphasis on Non Synonymous amino acid substitutions. The fatality rates in different countries were matched against the mutation number, rarity of the nucleotide alterations and functional impact of the Non Synonymous changes at protein level, separately and in combination. Findings All the viral strains have been found to evolve from the viral strain of Taiwan (MT192759) which is 100% identical with the ancestor SARS-CoV-2 sequences of Wuhan (NC 045512.2; submitted on 5th Jan, 2020). Transition from C to T (C>T) is the most frequent mutation in this viral genome and mutations A>T, G>A, T>A are the rarest ones, found in countries with maximum fatality rate i.e Italy, Spain and Sweden. 20 Non Synonymous mutations are located in viral genome spanning Orf1ab polyprotein, Surface glycoprotein, Nucleocapsid protein etc. The functional effect on the structure and function of the protein can favourably or unfavourably interact with the host body. Interpretation The fatality outcome depends on three important factors (a) number of mutation (b) rarity of the allelic variation and (c) functional consequence of the mutation at protein level. The molecular divergence, evolved from the ancestral strain (S) lead to extremely lethal (E), lethal(L) and non lethal (N) strains with the involvement of an Intermediate strain(I).\n",
      "\n",
      "bu17n7zf\n",
      "The outbreak of Coronavirus disease 2019 (COVID-19), caused by severe acute respiratory syndrome (SARS) coronavirus 2 (SARS-CoV-2), has thus far killed over 3,000 people and infected over 80,000 in China and elsewhere in the world, resulting in catastrophe for humans. Similar to its homologous virus, SARS-CoV, which caused SARS in thousands of people in 2003, SARS-CoV-2 might also be transmitted from the bats and causes similar symptoms through a similar mechanism. However, COVID-19 has lower severity and mortality than SARS but is much more transmissive and affects more elderly individuals than youth and more men than women. In response to the rapidly increasing number of publications on the emerging disease, this article attempts to provide a timely and comprehensive review of the swiftly developing research subject. We will cover the basics about the epidemiology, etiology, virology, diagnosis, treatment, prognosis, and prevention of the disease. Although many questions still require answers, we hope that this review helps in the understanding and eradication of the threatening disease.\n",
      "\n",
      "dtwstwbe\n",
      "Since 2002, beta coronaviruses (CoV) have caused three zoonotic outbreaks, SARS-CoV in 2002-2003, MERS-CoV in 2012, and the newly emerged SARS-CoV-2 in late 2019. However, little is currently known about the biology of SARS-CoV-2. Here, using SARS-CoV-2 S protein pseudovirus system, we confirm that human angiotensin converting enzyme 2 (hACE2) is the receptor for SARS-CoV-2, find that SARS-CoV-2 enters 293/hACE2 cells mainly through endocytosis, that PIKfyve, TPC2, and cathepsin L are critical for entry, and that SARS-CoV-2 S protein is less stable than SARS-CoV S. Polyclonal anti-SARS S1 antibodies T62 inhibit entry of SARS-CoV S but not SARS-CoV-2 S pseudovirions. Further studies using recovered SARS and COVID-19 patients' sera show limited cross-neutralization, suggesting that recovery from one infection might not protect against the other. Our results present potential targets for development of drugs and vaccines for SARS-CoV-2.\n",
      "\n",
      "cn3bpmwj\n",
      "BACKGROUND AND AIMS: The outbreak of the new coronavirus, SARS-CoV-2, causes a respiratory disease and individuals with pre-existing cardiometabolic disorders display worse prognosis through the infection course. The aim of this minireview is to present epidemiological data related to metabolic comorbidities in association with the SARS-CoV-2. METHODS: This is a narrative mini-review with Pubmed search until April 23, 2020 using the keywords COVID-19, SARS-CoV-2, treatment of coronavirus and following terms: diabetes mellitus, obesity, arterial hypertension, ACE-inhibitors, cytokine storm, immune response and vitamin D. RESULTS: Studies indicate that obese individuals are more likely to develop infections, and that adipose tissue serves as a pathogen reservoir. In diabetic individuals higher rate of inflammatory processes is seen due to constant glucose recognition by C type lectin receptors. Hypertensive individuals, usually grouped with other conditions, are treated with drugs to reduce blood pressure mostly through ACEi and ARB, that leads to increased ACE2 expression, used by SARS-CoV-2 for human's cell entry. Until now, the studies have shown that individuals with those conditions and affected by COVID-19 present an uncontrolled release of pro-inflammatory cytokines and an unbalanced immune response, leading to the cytokine storm phenomenon. Vitamin D is highlighted as a potential therapeutic target, because in addition to acting on the immune system, it plays an important role in the control of cardiometabolic diseases. CONCLUSION: Currently, since there is no proven and effective antiviral therapy for SARS-CoV-2, the efforts should focus on controlling inflammatory response and reduce the risks of associated complications.\n",
      "\n",
      "Success list\n",
      "t8q99tlq\n",
      "Monitoring the mutation dynamics of SARS-CoV-2 is critical for the development of effective approaches to contain the pathogen. By analyzing 106 SARS-CoV-2 and 39 SARS genome sequences, we provided direct genetic evidence that SARS-CoV-2 has a much lower mutation rate than SARS. Minimum Evolution phylogeny analysis revealed the putative original status of SARS-CoV-2 and the early-stage spread history. The discrepant phylogenies for the spike protein and its receptor binding domain proved a previously reported structural rearrangement prior to the emergence of SARS-CoV-2. Despite that we found the spike glycoprotein of SARS-CoV-2 is particularly more conserved, we identified a mutation that leads to weaker receptor binding capability, which concerns a SARS-CoV-2 sample collected on 27th January 2020 from India. This represents the first report of a significant SARS-CoV-2 mutant, and raises the alarm that the ongoing vaccine development may become futile in future epidemic if more mutations were identified. Highlights Based on the currently available genome sequence data, we proved that SARS-COV-2 genome has a much lower mutation rate and genetic diversity than SARS during the 2002-2003 outbreak. The spike (S) protein encoding gene of SARS-COV-2 is found relatively more conserved than other protein-encoding genes, which is a good indication for the ongoing antiviral drug and vaccine development. Minimum Evolution phylogeny analysis revealed the putative original status of SARS-CoV-2 and the early-stage spread history. We confirmed a previously reported rearrangement in the S protein arrangement of SARS-COV-2, and propose that this rearrangement should have occurred between human SARS-CoV and a bat SARS-CoV, at a time point much earlier before SARS-COV-2 transmission to human. We provided first evidence that a mutated SARS-COV-2 with reduced human ACE2 receptor binding affinity have emerged in India based on a sample collected on 27th January 2020.\n",
      "\n",
      "rbzl0txp\n",
      "Stability of SARS-CoV-2 in different environmental conditions.\n",
      "\n",
      "5wsj003j\n",
      "Background The reasons for the large differences between countries in the sizes of their SARS CoV2 epidemics is unknown. Individual level studies have found that the use of face masks was protective for the acquisition and transmission of a range of respiratory viruses including SARS CoV1. We hypothesized that population level usage of face masks may be negatively associated SARS CoV2 spread. Methods At a country level, linear regression was used to assess the association between COVID19 diagnoses per inhabitant and the national promotion of face masks in public (coded as a binary variable), controlling for the age of the COVID19 epidemic and testing intensity. Results Eight of the 49 countries with available data advocated wearing face masks in public: China, Czechia, Hong Kong, Japan, Singapore, South Korea, Thailand and Malaysia. In multivariate analysis face mask use was negatively associated with number of COVID19 cases/inhabitant (coef. -326, 95% CI -601- -51, P=0.021). Testing intensity was positively associated with COVID-19 cases (coef. 0.07, 95% CI 0.05-0.08, P<0.001). Conclusion Whilst these results are susceptible to residual confounding, they do provide ecological level support to the individual level studies that found face mask usage to reduce the transmission and acquisition of respiratory viral infections.\n",
      "\n",
      "02cfyuf4\n",
      "Direct massively parallel sequencing of SARS-CoV-2 genome was undertaken from nasopharyngeal and oropharyngeal swab samples of infected individuals in Eastern India. Seven of the isolates belonged to the A2a clade, while one belonged to the B4 clade. Specific mutations, characteristic of the A2a clade, were also detected, which included the P323L in RNA-dependent RNA polymerase and D614G in the Spike glycoprotein. Further, our data revealed emergence of novel subclones harbouring nonsynonymous mutations, viz. G1124V in Spike (S) protein, R203K, and G204R in the nucleocapsid (N) protein. The N protein mutations reside in the SR-rich region involved in viral capsid formation and the S protein mutation is in the S2 domain, which is involved in triggering viral fusion with the host cell membrane. Interesting correlation was observed between these mutations and travel or contact history of COVID-19 positive cases. Consequent alterations of miRNA binding and structure were also predicted for these mutations. More importantly, the possible implications of mutation D614G (in SD domain) and G1124V (in S2 subunit) on the structural stability of S protein have also been discussed. Results report for the first time a bird's eye view on the accumulation of mutations in SARS-CoV-2 genome in Eastern India.\n",
      "\n",
      "zm8hpuer\n",
      "In the Wuhan Province of China, in December 2019, the novel coronavirus 2019 (COVID-19) has caused a severe involvement of the lower respiratory tract leading to an acute respiratory syndrome. Subsequently, coronavirus 2 (SARS-CoV-2) provoked a pandemic which is considered a life-threatening disease. The SARS-CoV-2, a family member of betacoronaviruses, possesses single-stranded positive-sense RNA with typical structural proteins, involving the envelope, membrane, nucleocapsid and spike proteins that are responsible for the viral infectivity, and nonstructural proteins. The effectual host immune response including innate and adaptive immunity against SARS-Cov-2 seems crucial to control and resolve the viral infection. However, the severity and outcome of the COVID-19 might be associated with the excessive production of proinflammatory cytokines \"cytokine storm\" leading to an acute respiratory distress syndrome. Regretfully, the exact pathophysiology and treatment, especially for the severe COVID-19, is still uncertain. The results of preliminary studies have shown that immune-modulatory or immune-suppressive treatments such as hydroxychloroquine, interleukin (IL)-6 and IL-1 antagonists, commonly used in rheumatology, might be considered as treatment choices for COVID-19, particularly in severe disease. In this review, to gain better information about appropriate anti-inflammatory treatments, mostly used in rheumatology for COVID-19, we have focused the attention on the structural features of SARS-CoV-2, the host immune response against SARS-CoV-2 and its association with the cytokine storm.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "print(\"Failure list\")\n",
    "for id_ in fail_id_list:\n",
    "    print(id_)\n",
    "    print(corpus_dict[id_])\n",
    "    print()\n",
    "print(\"Success list\")\n",
    "for id_ in succ_id_list:\n",
    "    print(id_)\n",
    "    print(corpus_dict[id_])\n",
    "    print()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "0052fb42-dc94-4ac9-bc54-aed090a72157",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'lnys6iuu'"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "fail_id_list[0]"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "68f05d63-02ca-4fe6-8001-bab60aadba5b",
   "metadata": {},
   "source": [
    "# Check Queries"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "b611c3cc-0bfa-4c4e-80aa-19c4b9581206",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?', 'what is the origin of COVID-19', 'What is the protein structure of the SARS-CoV-2 spike?']\n",
      "\n",
      "['Does SARS-CoV-2 have any subtypes, and if so what are they?', 'What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?', 'What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?']\n",
      "\n",
      "['what are the initial symptoms of Covid-19?', 'which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?', 'what are the transmission routes of coronavirus?', 'what is the origin of COVID-19']\n",
      "\n",
      "['What is the protein structure of the SARS-CoV-2 spike?', 'will SARS-CoV2 infected people develop immunity? Is cross protection possible?', 'which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?']\n",
      "\n",
      "['What is the mechanism of cytokine storm syndrome on the COVID-19?', 'what kinds of complications related to COVID-19 are associated with diabetes', 'What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?']\n",
      "\n",
      "---------\n",
      "['What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?', 'What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?', 'What is the protein structure of the SARS-CoV-2 spike?']\n",
      "\n",
      "['how does the coronavirus respond to changes in the weather', 'how long does coronavirus remain stable  on surfaces?', 'how long can the coronavirus live outside the body']\n",
      "\n",
      "['what are the transmission routes of coronavirus?', 'what are the best masks for preventing infection by Covid-19?', 'How much impact do masks have on preventing the spread of the COVID-19?']\n",
      "\n",
      "['What is the protein structure of the SARS-CoV-2 spike?', 'Does SARS-CoV-2 have any subtypes, and if so what are they?', 'What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?']\n",
      "\n",
      "['What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?', 'which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?', 'What is the mechanism of cytokine storm syndrome on the COVID-19?']\n",
      "\n"
     ]
    }
   ],
   "source": [
    "def check_matching_queries(_document_id):\n",
    "    queries = gt[_document_id]\n",
    "    questions = [results[i-1]['question'] for i in queries]\n",
    "    # print(queries)\n",
    "    print(questions)\n",
    "\n",
    "for __id in fail_id_list:\n",
    "    # print(corpus_dict[__id])\n",
    "    check_matching_queries(__id)\n",
    "    print()\n",
    "\n",
    "print('---------')\n",
    "for __id in succ_id_list:\n",
    "    # print(corpus_dict[__id])\n",
    "    check_matching_queries(__id)\n",
    "    print()\n",
    "\n",
    "# check_matching_queries(fail_id_list[0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 75,
   "id": "20f28b30-a935-493e-9478-e786c0e2f956",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'50'"
      ]
     },
     "execution_count": 75,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# results[0]['_id']\n",
    "# results[0]['question']\n",
    "\n",
    "results[49]['_id']"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "22b5e046-05ee-4aef-9a71-e24f7db06d8a",
   "metadata": {},
   "source": [
    "# Check retrieved queries"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b474eb8c-0956-4a20-8b6f-abd5f70bd267",
   "metadata": {},
   "outputs": [],
   "source": [
    "# hotpotqa: 5233329\n",
    "# trec-covid: 173???\n",
    "# np: 2681468"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "71c5a0d3-d8d6-40d3-abc7-96b31c2b1970",
   "metadata": {},
   "outputs": [],
   "source": [
    "import json\n",
    "\n",
    "def load_jsonl(path):\n",
    "    with open(path) as f:\n",
    "        return [json.loads(i) for i in f.readlines()]\n",
    "\n",
    "def get_matching_queries(document_id):\n",
    "    queries = gt[document_id]\n",
    "    questions = [results[i-1]['question'] for i in queries]\n",
    "    return questions\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "0acd68b1-7e33-442b-9f6e-3b025b5fef1b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'_id': 'i1lyno9g',\n",
       " 'title': 'Analysis of therapeutic targets for SARS-CoV-2 and discovery of potential drugs by computational methods',\n",
       " 'text': 'SARS-CoV-2 has caused tens of thousands of infections and more than one thousand deaths. There are currently no registered therapies for treating coronavirus infections. Because of time consuming process of new drug development, drug repositioning may be the only solution to the epidemic of sudden infectious diseases. We systematically analyzed all the proteins encoded by SARS-CoV-2 genes, compared them with proteins from other coronaviruses, predicted their structures, and built 19 structures that could be done by homology modeling. By performing target-based virtual ligand screening, a total of 21 targets (including two human targets) were screened against compound libraries including ZINC drug database and our own database of natural products. Structure and screening results of important targets such as 3-chymotrypsin-like protease (3CLpro), Spike, RNA-dependent RNA polymerase (RdRp), and papain like protease (PLpro) were discussed in detail. In addition, a database of 78 commonly used anti-viral drugs including those currently on the market and undergoing clinical trials for SARS-CoV-2 was constructed. Possible targets of these compounds and potential drugs acting on a certain target were predicted. This study will provide new lead compounds and targets for further in vitro and in vivo studies of SARS-CoV-2, new insights for those drugs currently ongoing clinical studies, and also possible new strategies for drug repositioning to treat SARS-CoV-2 infections.',\n",
       " 'metadata': {'url': '', 'pubmed_id': ''},\n",
       " 'retrieval': [{'source': '/gallery_louvre/dayoon.ko/research/sds/src/datasets/trec-covid/queries.jsonl',\n",
       "   'seq_num': 29,\n",
       "   'document': 'which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?'},\n",
       "  {'source': '/gallery_louvre/dayoon.ko/research/sds/src/datasets/trec-covid/queries.jsonl',\n",
       "   'seq_num': 5,\n",
       "   'document': 'what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?'},\n",
       "  {'source': '/gallery_louvre/dayoon.ko/research/sds/src/datasets/trec-covid/queries.jsonl',\n",
       "   'seq_num': 36,\n",
       "   'document': 'What is the protein structure of the SARS-CoV-2 spike?'},\n",
       "  {'source': '/gallery_louvre/dayoon.ko/research/sds/src/datasets/trec-covid/queries.jsonl',\n",
       "   'seq_num': 37,\n",
       "   'document': 'What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?'},\n",
       "  {'source': '/gallery_louvre/dayoon.ko/research/sds/src/datasets/trec-covid/queries.jsonl',\n",
       "   'seq_num': 28,\n",
       "   'document': 'what evidence is there for the value of hydroxychloroquine in treating Covid-19?'},\n",
       "  {'source': '/gallery_louvre/dayoon.ko/research/sds/src/datasets/trec-covid/queries.jsonl',\n",
       "   'seq_num': 25,\n",
       "   'document': 'which biomarkers predict the severe clinical course of 2019-nCOV infection?'},\n",
       "  {'source': '/gallery_louvre/dayoon.ko/research/sds/src/datasets/trec-covid/queries.jsonl',\n",
       "   'seq_num': 46,\n",
       "   'document': 'what evidence is there for dexamethasone as a treatment for COVID-19?'},\n",
       "  {'source': '/gallery_louvre/dayoon.ko/research/sds/src/datasets/trec-covid/queries.jsonl',\n",
       "   'seq_num': 17,\n",
       "   'document': 'are there any clinical trials available for the coronavirus'},\n",
       "  {'source': '/gallery_louvre/dayoon.ko/research/sds/src/datasets/trec-covid/queries.jsonl',\n",
       "   'seq_num': 32,\n",
       "   'document': 'Does SARS-CoV-2 have any subtypes, and if so what are they?'},\n",
       "  {'source': '/gallery_louvre/dayoon.ko/research/sds/src/datasets/trec-covid/queries.jsonl',\n",
       "   'seq_num': 3,\n",
       "   'document': 'will SARS-CoV2 infected people develop immunity? Is cross protection possible?'}]}"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ret_suc = load_jsonl(\"/gallery_louvre/dayoon.ko/research/sds/retrieval/results/trec-covid-suc-query-retrieval.jsonl\")\n",
    "ret_fail = load_jsonl(\"/gallery_louvre/dayoon.ko/research/sds/retrieval/results/trec-covid-fail-query-retrieval.jsonl\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "id": "241cbdbd-8b74-4209-bb12-14a8ed42ee0c",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ID: i1lyno9g\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "are there any clinical trials available for the coronavirus\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: xpbcoipf\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: gmk4qo4i\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "What new public datasets are available related to COVID-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "are there any clinical trials available for the coronavirus\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: r0gr0bhl\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What new public datasets are available related to COVID-19?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "What new public datasets are available related to COVID-19?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "what is the origin of COVID-19\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 79yna07e\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What vaccine candidates are being tested for Covid-19?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "What vaccine candidates are being tested for Covid-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 915srotp\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "what is the origin of COVID-19\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what is the origin of COVID-19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: suhqgmlo\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "what is the origin of COVID-19\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 9ryu9ady\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "are there any clinical trials available for the coronavirus\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: b518n9dx\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "are there any clinical trials available for the coronavirus\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "are there any clinical trials available for the coronavirus\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "are there serological tests that detect antibodies to coronavirus?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: o3nfoqgs\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 0u4ar3b5\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "are there any clinical trials available for the coronavirus\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "are there any clinical trials available for the coronavirus\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "Does Vitamin D impact COVID-19 prevention and treatment?\n",
      "what type of hand sanitizer is needed to destroy Covid-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 5o9bbspc\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "are there any clinical trials available for the coronavirus\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 0lk8eujq\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "Does Vitamin D impact COVID-19 prevention and treatment?\n",
      "are there any clinical trials available for the coronavirus\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 1ew0p6x7\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the mortality rates overall and in specific populations\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what is known about those infected with Covid-19 but are asymptomatic?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "how long can the coronavirus live outside the body\n",
      "how has lack of testing availability led to underreporting of true incidence of Covid-19?\n",
      "what are the initial symptoms of Covid-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: bjyzrmsb\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what are the initial symptoms of Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what are the initial symptoms of Covid-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "what is the origin of COVID-19\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "what causes death from Covid-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: o8df8szx\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what causes death from Covid-19?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "what are the initial symptoms of Covid-19?\n",
      "what is known about those infected with Covid-19 but are asymptomatic?\n",
      "what are the guidelines for triaging patients infected with coronavirus?\n",
      "What are the longer-term complications of those who recover from COVID-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 4fqsx5y5\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the mortality rates overall and in specific populations\n",
      "what causes death from Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what causes death from Covid-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: u6uqroi0\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "what are the mortality rates overall and in specific populations\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "Does Vitamin D impact COVID-19 prevention and treatment?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "what are the health outcomes for children who contract COVID-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 1vhxcbx7\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what causes death from Covid-19?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "has social distancing had an impact on slowing the spread of COVID-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 2qljx9cq\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "What vaccine candidates are being tested for Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "are there any clinical trials available for the coronavirus\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: jsjxdmi2\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the initial symptoms of Covid-19?\n",
      "what is known about those infected with Covid-19 but are asymptomatic?\n",
      "what are the health outcomes for children who contract COVID-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what are the health outcomes for children who contract COVID-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what is known about those infected with Covid-19 but are asymptomatic?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "what are the initial symptoms of Covid-19?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 2705en59\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: hl956i02\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "How much impact do masks have on preventing the spread of the COVID-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "has social distancing had an impact on slowing the spread of COVID-19?\n",
      "How much impact do masks have on preventing the spread of the COVID-19?\n",
      "what are the best masks for preventing infection by Covid-19?\n",
      "how has lack of testing availability led to underreporting of true incidence of Covid-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "what type of hand sanitizer is needed to destroy Covid-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: ve34oyz0\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "How much impact do masks have on preventing the spread of the COVID-19?\n",
      "what are the transmission routes of coronavirus?\n",
      "what are the best masks for preventing infection by Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "How much impact do masks have on preventing the spread of the COVID-19?\n",
      "how long does coronavirus remain stable  on surfaces?\n",
      "has social distancing had an impact on slowing the spread of COVID-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "how long can the coronavirus live outside the body\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "what are the best masks for preventing infection by Covid-19?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 40xsypzt\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: yop76n7z\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 4hmecfi0\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: t996fbad\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "what are the initial symptoms of Covid-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: f37cp6j3\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "what is known about those infected with Covid-19 but are asymptomatic?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 3g06zxmx\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "Does Vitamin D impact COVID-19 prevention and treatment?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 5h7qyn1g\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: ltar1aa5\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "what is the origin of COVID-19\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what is the origin of COVID-19\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "what are the transmission routes of coronavirus?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: fofy6whl\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "what is the origin of COVID-19\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "what is the origin of COVID-19\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: ayhsbfd1\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "what is the origin of COVID-19\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "what is the origin of COVID-19\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "are there serological tests that detect antibodies to coronavirus?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: qw05apnf\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what is the origin of COVID-19\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "what evidence is there related to COVID-19 super spreaders\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "what is the origin of COVID-19\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "what evidence is there related to COVID-19 super spreaders\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: bu1bc2bi\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: m2cu5iof\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "what is the origin of COVID-19\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: m95bmi9t\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what types of rapid testing for Covid-19 have been developed?\n",
      "are there serological tests that detect antibodies to coronavirus?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what types of rapid testing for Covid-19 have been developed?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "are there any clinical trials available for the coronavirus\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "What vaccine candidates are being tested for Covid-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: a47onmje\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the transmission routes of coronavirus?\n",
      "how long can the coronavirus live outside the body\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "how long does coronavirus remain stable  on surfaces?\n",
      "has social distancing had an impact on slowing the spread of COVID-19?\n",
      "how long can the coronavirus live outside the body\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "what are the transmission routes of coronavirus?\n",
      "what is the origin of COVID-19\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "what type of hand sanitizer is needed to destroy Covid-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: f7gr98eb\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the transmission routes of coronavirus?\n",
      "how long does coronavirus remain stable  on surfaces?\n",
      "how long can the coronavirus live outside the body\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "how long does coronavirus remain stable  on surfaces?\n",
      "what type of hand sanitizer is needed to destroy Covid-19?\n",
      "what is the origin of COVID-19\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "how long can the coronavirus live outside the body\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 7kw9lws0\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "Does Vitamin D impact COVID-19 prevention and treatment?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: nadzy6lm\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what is known about those infected with Covid-19 but are asymptomatic?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what is known about those infected with Covid-19 but are asymptomatic?\n",
      "what evidence is there related to COVID-19 super spreaders\n",
      "how has lack of testing availability led to underreporting of true incidence of Covid-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what are the initial symptoms of Covid-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: kugr54if\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are best practices in hospitals and at home in maintaining quarantine?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what are the guidelines for triaging patients infected with coronavirus?\n",
      "what are best practices in hospitals and at home in maintaining quarantine?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "has social distancing had an impact on slowing the spread of COVID-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: ax9btc74\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what types of rapid testing for Covid-19 have been developed?\n",
      "are there serological tests that detect antibodies to coronavirus?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "are there serological tests that detect antibodies to coronavirus?\n",
      "how has lack of testing availability led to underreporting of true incidence of Covid-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: oihyz5v7\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "are there serological tests that detect antibodies to coronavirus?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "are there serological tests that detect antibodies to coronavirus?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what types of rapid testing for Covid-19 have been developed?\n",
      "are there any clinical trials available for the coronavirus\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 9595vm0k\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "are there serological tests that detect antibodies to coronavirus?\n",
      "what types of rapid testing for Covid-19 have been developed?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "are there serological tests that detect antibodies to coronavirus?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "are there any clinical trials available for the coronavirus\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 8ruux4sw\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "are there any clinical trials available for the coronavirus\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What vaccine candidates are being tested for Covid-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "are there any clinical trials available for the coronavirus\n",
      "what types of rapid testing for Covid-19 have been developed?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: bgm4dlwz\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: agc89fqv\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "What vaccine candidates are being tested for Covid-19?\n",
      "are there any clinical trials available for the coronavirus\n",
      "what type of hand sanitizer is needed to destroy Covid-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: fcmzdcuh\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the initial symptoms of Covid-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what causes death from Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "what causes death from Covid-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "how has lack of testing availability led to underreporting of true incidence of Covid-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: qvhwypjt\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what causes death from Covid-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "what causes death from Covid-19?\n",
      "what are the initial symptoms of Covid-19?\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "what are the guidelines for triaging patients infected with coronavirus?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "what is known about those infected with Covid-19 but are asymptomatic?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 2y452utz\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what is the origin of COVID-19\n",
      "what are the initial symptoms of Covid-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 6k5ac3f2\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "what is the origin of COVID-19\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What new public datasets are available related to COVID-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: qp4efhwq\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what is the origin of COVID-19\n",
      "how long can the coronavirus live outside the body\n",
      "What vaccine candidates are being tested for Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what is the origin of COVID-19\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what are the initial symptoms of Covid-19?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "Does Vitamin D impact COVID-19 prevention and treatment?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 0kss5r7u\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "what causes death from Covid-19?\n",
      "What are the longer-term complications of those who recover from COVID-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "what are the initial symptoms of Covid-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: jg6v644y\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "What are the longer-term complications of those who recover from COVID-19?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what is known about those infected with Covid-19 but are asymptomatic?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: fzmrvjtl\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what are the guidelines for triaging patients infected with coronavirus?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "What are the longer-term complications of those who recover from COVID-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "what causes death from Covid-19?\n",
      "what are the health outcomes for children who contract COVID-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: yr1dq258\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the transmission routes of coronavirus?\n",
      "what type of hand sanitizer is needed to destroy Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "how long does coronavirus remain stable  on surfaces?\n",
      "what type of hand sanitizer is needed to destroy Covid-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what are best practices in hospitals and at home in maintaining quarantine?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "how long can the coronavirus live outside the body\n",
      "has social distancing had an impact on slowing the spread of COVID-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: qwngx01h\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the best masks for preventing infection by Covid-19?\n",
      "How much impact do masks have on preventing the spread of the COVID-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what are the best masks for preventing infection by Covid-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what are the guidelines for triaging patients infected with coronavirus?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "How much impact do masks have on preventing the spread of the COVID-19?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "has social distancing had an impact on slowing the spread of COVID-19?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: jc9ugexn\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "are there any clinical trials available for the coronavirus\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "are there any clinical trials available for the coronavirus\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "Does Vitamin D impact COVID-19 prevention and treatment?\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: vxqdfiel\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "are there any clinical trials available for the coronavirus\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "are there any clinical trials available for the coronavirus\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "are there serological tests that detect antibodies to coronavirus?\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "what are the guidelines for triaging patients infected with coronavirus?\n",
      "What vaccine candidates are being tested for Covid-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 7y87ktmi\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "are there any clinical trials available for the coronavirus\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "are there any clinical trials available for the coronavirus\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "are there serological tests that detect antibodies to coronavirus?\n",
      "what are the guidelines for triaging patients infected with coronavirus?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: eitnkdi9\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "are there any clinical trials available for the coronavirus\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "What vaccine candidates are being tested for Covid-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what type of hand sanitizer is needed to destroy Covid-19?\n",
      "what types of rapid testing for Covid-19 have been developed?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: iymmf4k6\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "are there any clinical trials available for the coronavirus\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "What vaccine candidates are being tested for Covid-19?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: yth9hh5q\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "are there any clinical trials available for the coronavirus\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: juz9jnfk\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what is the origin of COVID-19\n",
      "what are the initial symptoms of Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what is the origin of COVID-19\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "what are the initial symptoms of Covid-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 7ivw72l0\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: t8q99tlq\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: w5ytp1q7\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 8zzi6gu9\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What vaccine candidates are being tested for Covid-19?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What vaccine candidates are being tested for Covid-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: vf32wxkx\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 8cvjsisw\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "are there any clinical trials available for the coronavirus\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "Does Vitamin D impact COVID-19 prevention and treatment?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: nwu4tdgh\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: khpc9f98\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "how does the coronavirus respond to changes in the weather\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "how long does coronavirus remain stable  on surfaces?\n",
      "how long can the coronavirus live outside the body\n",
      "what are the transmission routes of coronavirus?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "how does the coronavirus respond to changes in the weather\n",
      "are there serological tests that detect antibodies to coronavirus?\n",
      "are there any clinical trials available for the coronavirus\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: vjg2auh7\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "How much impact do masks have on preventing the spread of the COVID-19?\n",
      "what are the transmission routes of coronavirus?\n",
      "what are the best masks for preventing infection by Covid-19?\n",
      "are there any clinical trials available for the coronavirus\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "are there any clinical trials available for the coronavirus\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "How much impact do masks have on preventing the spread of the COVID-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "what are the best masks for preventing infection by Covid-19?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what are the transmission routes of coronavirus?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: zb434ve3\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "are there any clinical trials available for the coronavirus\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: k9xhphpl\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "how long can the coronavirus live outside the body\n",
      "what are the transmission routes of coronavirus?\n",
      "how long does coronavirus remain stable  on surfaces?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "how long does coronavirus remain stable  on surfaces?\n",
      "how long can the coronavirus live outside the body\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what type of hand sanitizer is needed to destroy Covid-19?\n",
      "what are the transmission routes of coronavirus?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "are there any clinical trials available for the coronavirus\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: ws1ysmae\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the mortality rates overall and in specific populations\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "what causes death from Covid-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "Does Vitamin D impact COVID-19 prevention and treatment?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: mbbnk3la\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "what are the health outcomes for children who contract COVID-19?\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "what causes death from Covid-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: bsypo08l\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What new public datasets are available related to COVID-19?\n",
      "what is the origin of COVID-19\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "what is the origin of COVID-19\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: rs79r7kc\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what are the initial symptoms of Covid-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "are there serological tests that detect antibodies to coronavirus?\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: xly61tfw\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "what is the origin of COVID-19\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "are there serological tests that detect antibodies to coronavirus?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 9uymi395\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "Does Vitamin D impact COVID-19 prevention and treatment?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "What are the longer-term complications of those who recover from COVID-19?\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: i1lyno9g\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "SARS-CoV-2 has caused tens of thousands of infections and more than one thousand deaths. There are currently no registered therapies for treating coronavirus infections. Because of time consuming process of new drug development, drug repositioning may be the only solution to the epidemic of sudden infectious diseases. We systematically analyzed all the proteins encoded by SARS-CoV-2 genes, compared them with proteins from other coronaviruses, predicted their structures, and built 19 structures that could be done by homology modeling. By performing target-based virtual ligand screening, a total of 21 targets (including two human targets) were screened against compound libraries including ZINC drug database and our own database of natural products. Structure and screening results of important targets such as 3-chymotrypsin-like protease (3CLpro), Spike, RNA-dependent RNA polymerase (RdRp), and papain like protease (PLpro) were discussed in detail. In addition, a database of 78 commonly used anti-viral drugs including those currently on the market and undergoing clinical trials for SARS-CoV-2 was constructed. Possible targets of these compounds and potential drugs acting on a certain target were predicted. This study will provide new lead compounds and targets for further in vitro and in vivo studies of SARS-CoV-2, new insights for those drugs currently ongoing clinical studies, and also possible new strategies for drug repositioning to treat SARS-CoV-2 infections.\n",
      "PURPOSE: Recently, the world has been dealing with a new type of coronavirus called COVID-19 that in terms of symptoms is similar to the SARS coronavirus. Unfortunately, researchers could not find a registered therapy to treat the infection related to the virus yet. Regarding the fact that drug repurposing is a good strategy for epidemic viral infection, we applied the drug repurposing strategy using virtual screening to identify therapeutic options for COVID-19. For this purpose, five proteins of COVID-19 (3-chymotrypsin-like protease (3CLpro), Papain-Like protease (PLpro), cleavage site, HR1 and RBD in Spike protein) were selected as target proteins for drug repositioning. METHODS: First, five proteins of COVID-19 were built by homology modeling. Then FDA-approved drugs (2471 drugs) were screened against cleavage site and RBD in Spike protein via virtual screening. One hundred and twenty-eight FDA-approved drugs with the most favorable free-binding energy were attached to the cleavage site and RBD in Spike protein. Of these 128 drugs, 18 drugs have either been used currently as antiviral or have been reported to possess antiviral effects. Virtual screening was then performed for the 18 selected drugs with ACE2, 3CLpro and PLpro and HR1 and TMPRSS2. RESULTS: According to the results, glecaprevir, paritaprevir, simeprevir, ledipasvir, glycyrrhizic acid, TMC-310911, and hesperidin showed highly favorably free binding energies with all tested target proteins. CONCLUSION: The above-mentioned drugs can be regarded as candidates to treat COVID-19 infections, but further study on the efficiency of these drugs is also necessary.\n",
      "The outbreak of a novel human coronavirus (SARS-CoV-2) has evolved into global health emergency, infecting hundreds of thousands of people worldwide. We have identified experimental data on the inhibitory activity of compounds tested against closely related (96% sequence identity, 100% active site conservation) protease of SARS-CoV and employed this data to build QSAR models for this dataset. We employed these models for virtual screening of all drugs from DrugBank, including compounds in clinical trials. Molecular docking and similarity search approaches were explored in parallel with QSAR modeling, but molecular docking failed to correctly discriminate between experimentally active and inactive compounds. As a result of our studies, we recommended 41 approved, experimental, or investigational drugs as potential agents against SARS-CoV-2 acting as putative inhibitors of Mpro. Ten compounds with feasible prices were purchased and are awaiting the experimental validation..\n",
      "The novelty of new human coronavirus COVID-19/SARS-CoV-2 and the lack of effective drugs and vaccines gave rise to a wide variety of strategies employed to fight this worldwide pandemic. Many of these strategies rely on the repositioning of existing drugs that could shorten the time and reduce the cost compared to de novo drug discovery. In this study, we presented a new network-based algorithm for drug repositioning, called SAveRUNNER (Searching off-lAbel dRUg aNd NEtwoRk), which predicts drug-disease associations by quantifying the interplay between the drug targets and the disease-specific proteins in the human interactome via a novel network-based similarity measure that prioritizes associations between drugs and diseases locating in the same network neighborhoods. Specifically, we applied SAveRUNNER on a panel of 14 selected diseases with a consolidated knowledge about their disease-causing genes and that have been found to be related to COVID-19 for genetic similarity, comorbidity, or for their association to drugs tentatively repurposed to treat COVID-19. Focusing specifically on SARS subnetwork, we identified 282 repurposable drugs, including some the most rumored off-label drugs for COVID-19 treatments, as well as a new combination therapy of 5 drugs, actually used in clinical practice. Furthermore, to maximize the efficiency of putative downstream validation experiments, we prioritized 24 potential anti-SARS-CoV repurposable drugs based on their network-based similarity values. These top-ranked drugs include ACE-inhibitors, monoclonal antibodies, and thrombin inhibitors. Finally, our findings were in-silico validated by performing a gene set enrichment analysis, which confirmed that most of the network-predicted repurposable drugs may have a potential treatment effect against human coronavirus infections.\n",
      "The rapid outbreak of the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in China followed by its spread around the world poses a serious global concern for public health with almost 90000 people infected and thousands of fatalities To conquer viral infections, the inhibition of proteases essential for proteolytic processing of viral polyproteins is a conventional therapeutic strategy To this date, no specific drugs or vaccines are available to treat SARS-CoV-2 despite its close relation to the SARS-CoV-1 virus that caused a similar epidemic in 2003 Thus, there remains an urgent need for the development of specific antiviral therapeutics to conquer SARS-CoV-2 In order to find novel inhibitors, we computationally screened a compound library of over 687 million compounds for binding at the recently solved crystal structure of the main protease of SARS-CoV-2 A screening of such a vast chemical space for SARS-CoV-2 protease inhibitors has not been reported before After shape screening, two docking protocols were applied followed by the determination of pharmacokinetically relevant molecular descriptors to narrow down the initial hits Next, molecular dynamics simulations were conducted to validate the stability of docked binding modes and comprehensively quantify ligand binding energies After evaluation of off-target binding, we report a list of 11 drug-like compounds with improved binding free energy to the target protease in contrast to the cocrystallized peptidomimetic lead compound that suffers from poor pharmacokinetic properties Furthermore, we identified one potent binder with comparable properties from the natural compound library\n",
      "The recent outbreak of coronavirus disease-19 (COVID-19) continues to drastically affect healthcare throughout the world. To date, no approved treatment regimen or vaccine is available to effectively attenuate or prevent the infection. Therefore, collective and multidisciplinary efforts are needed to identify new therapeutics or to explore effectiveness of existing drugs and drug-like small molecules against SARS-CoV-2 for lead identification and repurposing prospects. This study addresses the identification of small molecules that specifically bind to any of the three essential proteins (RdRp, 3CL-protease and helicase) of SARS-CoV-2. By applying computational approaches we screened a library of 4574 compounds also containing FDA-approved drugs against these viral proteins. Shortlisted hits from initial screening were subjected to iterative docking with the respective proteins. Ranking score on the basis of binding energy, clustering score, shape complementarity and functional significance of the binding pocket was applied to identify the binding compounds. Finally, to minimize chances of false positives, we performed docking of the identified molecules with 100 irrelevant proteins of diverse classes thereby ruling out the non-specific binding. Three FDA-approved drugs showed binding to 3CL-protease either at the catalytic pocket or at an allosteric site related to functionally important dimer formation. A drug-like molecule showed binding to RdRp in its catalytic pocket blocking the key catalytic residues. Two other drug-like molecules showed specific interactions with helicase at a key domain involved in catalysis. This study provides lead drugs or drug-like molecules for further in vitro and clinical investigation for drug repurposing and new drug development prospects.\n",
      "The coronavirus disease 2019 (COVID-19) pandemic caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has infected over 7.1 million people and led to over 0.4 million deaths. Currently, there is no specific anti-SARS-CoV-2 medication. New drug discovery typically takes more than 10 years. Drug repositioning becomes one of the most feasible approaches for combating COVID-19. This work curates the largest available experimental data set for SARS-CoV-2 or SARS-CoV 3CL (main) protease inhibitors. On the basis of this data set, we develop validated machine learning models with relatively low root-mean-square error to screen 1553 FDA-approved drugs as well as another 7012 investigational or off-market drugs in DrugBank. We found that many existing drugs might be potentially potent to SARS-CoV-2. The druggability of many potent SARS-CoV-2 3CL protease inhibitors is analyzed. This work offers a foundation for further experimental studies of COVID-19 drug repositioning.\n",
      "The coronavirus disease 2019 (COVID-19) pandemic caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has infected near 5 million people and led to over 0.3 million deaths. Currently, there is no specific anti-SARS-CoV-2 medication. New drug discovery typically takes more than ten years. Drug repositioning becomes one of the most feasible approaches for combating COVID-19. This work curates the largest available experimental dataset for SARS-CoV-2 or SARS-CoV main protease inhibitors. Based on this dataset, we develop validated machine learning models with relatively low root mean square error to screen 1553 FDA-approved drugs as well as other 7012 investigational or off-market drugs in DrugBank. We found that many existing drugs might be potentially potent to SARS-CoV-2. The druggability of many potent SARS-CoV-2 main protease inhibitors is analyzed. This work offers a foundation for further experimental studies of COVID-19 drug repositioning.\n",
      "A new coronavirus, known as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), is the aetiological agent responsible for the 2019-2020 viral pneumonia outbreak of coronavirus disease 2019 (COVID-19)1-4. Currently, there are no targeted therapeutic agents for the treatment of this disease, and effective treatment options remain very limited. Here we describe the results of a programme that aimed to rapidly discover lead compounds for clinical use, by combining structure-assisted drug design, virtual drug screening and high-throughput screening. This programme focused on identifying drug leads that target main protease (Mpro) of SARS-CoV-2: Mpro is a key enzyme of coronaviruses and has a pivotal role in mediating viral replication and transcription, making it an attractive drug target for SARS-CoV-25,6. We identified a mechanism-based inhibitor (N3) by computer-aided drug design, and then determined the crystal structure of Mpro of SARS-CoV-2 in complex with this compound. Through a combination of structure-based virtual and high-throughput screening, we assayed more than 10,000 compounds-including approved drugs, drug candidates in clinical trials and other pharmacologically active compounds-as inhibitors of Mpro. Six of these compounds inhibited Mpro, showing half-maximal inhibitory concentration values that ranged from 0.67 to 21.4 µM. One of these compounds (ebselen) also exhibited promising antiviral activity in cell-based assays. Our results demonstrate the efficacy of our screening strategy, which can lead to the rapid discovery of drug leads with clinical potential in response to new infectious diseases for which no specific drugs or vaccines are available.\n",
      "Global coronavirus disease pandemic (COVID-19) caused by newly identified SARS- CoV-2 coronavirus continues to claim the lives of thousands of people worldwide. The unavailability of specific medications to treat COVID-19 has led to drug repositioning efforts using various approaches, including computational analyses. Such analyses mostly rely on molecular docking and require the 3D structure of the target protein to be available. In this study, we utilized a set of machine learning algorithms and trained them on a dataset of RNA-dependent RNA polymerase (RdRp) inhibitors to run inference analyses on antiviral and anti-inflammatory drugs solely based on the ligand information. We also performed virtual screening analysis of the drug candidates predicted by machine learning models and docked them against the active site of SARS- CoV-2 RdRp, a key component of the virus replication machinery. Based on the ligand information of RdRp inhibitors, the machine learning models were able to identify candidates such as remdesivir and baloxavir marboxil, molecules with documented activity against RdRp of the novel coronavirus. Among the other identified drug candidates were beclabuvir, a non-nucleoside inhibitor of the hepatitis C virus (HCV) RdRp enzyme, and HCV protease inhibitors paritaprevir and faldaprevir. Further analysis of these candidates using molecular docking against the SARS-CoV-2 RdRp revealed low binding energies against the enzyme active site. Our approach also identified anti-inflammatory drugs lupeol, lifitegrast, antrafenine, betulinic acid, and ursolic acid to have potential activity against SARS-CoV-2 RdRp. We propose that the results of this study are considered for further validation as potential therapeutic options against COVID-19.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: xpbcoipf\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "COVID-19, since emerged in Wuhan, China, has been a major concern due to its high infection rate, leaving more than one million infected people around the world. Huge number of studies tried to reveal the structure of the SARS-CoV-2 compared to the SARS-CoV-1, in order to suppress this high infection rate. Some of these studies showed that the mutations in the SARS-CoV-1 Spike protein might be responsible for its higher affinity to the ACE2 human cell receptor. In this work, we used molecular dynamics simulations and Monte Carlo sampling to compare the binding affinities of the spike proteins of SARS-CoV and SARS-CoV-2 to the ACE2. We found that the SARS-CoV-2 binds to ACE2 stronger than SARS-CoV by 7 kcal/mol, due to enhanced electrostatic interactions. The major contributions to the electrostatic binding energies are resulting from the salt-bridges formed between R426 and ACE2-E329 in case of SARS-CoV and K417 and ACE2-D30 for SARS-CoV2. In addition, there is no significant contribution from a single mutant to the binding energies. However, these mutations induce sophisticated structural changes that enhance the binding energies. Our results also indicate that the SARS-CoV-2 is unlikely a lab engineered virus.\n",
      "SARS-CoV-2, since emerging in Wuhan, China, has been a major concern because of its high infection rate and has left more than six million infected people around the world. Many studies endeavored to reveal the structure of the SARS-CoV-2 compared to the SARS-CoV, in order to find solutions to suppress this high infection rate. Some of these studies showed that the mutations in the SARS-CoV spike (S) protein might be responsible for its higher affinity to the ACE2 human cell receptor. In this work, we used molecular dynamics simulations and Monte Carlo sampling to compare the binding affinities of the S proteins of SARS-CoV and SARS-CoV-2 to the ACE2. Our results show that the protein surface of the ACE2 at the receptor binding domain (RBD) exhibits negative electrostatic potential, while a positive potential is observed for the S proteins of SARS-CoV/SARS-CoV-2. In addition, the binding energies at the interface are slightly higher for SARS-CoV-2 because of enhanced electrostatic interactions. The major contributions to the electrostatic binding energies result from the salt bridges forming between R426 and ACE-2-E329 in the case of SARS-CoV and K417 and ACE2-D30 in the SARS-CoV-2. In addition, our results indicate that the enhancement in the binding energy is not due to a single mutant but rather because of the sophisticated structural changes induced by all these mutations together. This finding suggests that it is implausible for the SARS-CoV-2 to be a lab-engineered virus.\n",
      "[Image: see text] SARS-CoV-2, since emerging in Wuhan, China, has been a major concern because of its high infection rate and has left more than six million infected people around the world. Many studies endeavored to reveal the structure of the SARS-CoV-2 compared to the SARS-CoV, in order to find solutions to suppress this high infection rate. Some of these studies showed that the mutations in the SARS-CoV spike (S) protein might be responsible for its higher affinity to the ACE2 human cell receptor. In this work, we used molecular dynamics simulations and Monte Carlo sampling to compare the binding affinities of the S proteins of SARS-CoV and SARS-CoV-2 to the ACE2. Our results show that the protein surface of the ACE2 at the receptor binding domain (RBD) exhibits negative electrostatic potential, while a positive potential is observed for the S proteins of SARS-CoV/SARS-CoV-2. In addition, the binding energies at the interface are slightly higher for SARS-CoV-2 because of enhanced electrostatic interactions. The major contributions to the electrostatic binding energies result from the salt bridges forming between R426 and ACE-2-E329 in the case of SARS-CoV and K417 and ACE2-D30 in the SARS-CoV-2. In addition, our results indicate that the enhancement in the binding energy is not due to a single mutant but rather because of the sophisticated structural changes induced by all these mutations together. This finding suggests that it is implausible for the SARS-CoV-2 to be a lab-engineered virus.\n",
      "The COVID-19 disease has plagued over 200 countries with over three million cases and has resulted in over 200,000 deaths within 3 months. To gain insight into the high infection rate of the SARS-CoV-2 virus, we compare the interaction between the human ACE2 receptor and the SARS-CoV-2 spike protein with that of other pathogenic coronaviruses using molecular dynamics simulations. SARS-CoV, SARS-CoV-2, and HCoV-NL63 recognize ACE2 as the natural receptor but present a distinct binding interface to ACE2 and a different network of residue-residue contacts. SARS-CoV and SARS-CoV-2 have comparable binding affinities achieved by balancing energetics and dynamics. The SARS-CoV-2-ACE2 complex contains a higher number of contacts, a larger interface area, and decreased interface residue fluctuations relative to the SARS-CoV-ACE2 complex. These findings expose an exceptional evolutionary exploration exerted by coronaviruses toward host recognition. We postulate that the versatility of cell receptor binding strategies has immediate implications for therapeutic strategies.\n",
      "The COVID-19 disease has plagued over 200 countries with over three million cases and has resulted in over 200,000 deaths within 3 months. To gain insight into the high infection rate of the SARS-CoV-2 virus, we compare the interaction between the human ACE2 receptor and the SARS-CoV-2 spike protein with that of other pathogenic coronaviruses using molecular dynamics simulations. SARS-CoV, SARS-CoV-2, and HCoV-NL63 recognize ACE2 as the natural receptor but present a distinct binding interface to ACE2 and a different network of residue–residue contacts. SARS-CoV and SARS-CoV-2 have comparable binding affinities achieved by balancing energetics and dynamics. The SARS-CoV-2–ACE2 complex contains a higher number of contacts, a larger interface area, and decreased interface residue fluctuations relative to the SARS-CoV–ACE2 complex. These findings expose an exceptional evolutionary exploration exerted by coronaviruses toward host recognition. We postulate that the versatility of cell receptor binding strategies has immediate implications for therapeutic strategies.\n",
      "The outbreak of a novel coronavirus, which was later formally named the severe acute respiratory coronavirus 2 (SARS-CoV-2), has caused a worldwide public health crisis. Previous studies showed that SARS-CoV-2 is highly homologous to SARS-CoV and infects humans through the binding of the spike protein to ACE2. Here, we have systematically studied the molecular mechanisms of human infection with SARS-CoV-2 and SARS-CoV by protein-protein docking and MD simulations. It was found that SARS-CoV-2 binds ACE2 with a higher affinity than SARS-CoV, which may partly explain that SARS-CoV-2 is much more infectious than SARS-CoV. In addition, the spike protein of SARS-CoV-2 has a significantly lower free energy than that of SARS-CoV, suggesting that SARS-CoV-2 is more stable and may survive a higher temperature than SARS-CoV. This provides insights into the evolution of SARS-CoV-2 because SARS-like coronaviruses have originated in bats. Our computation also suggested that the RBD-ACE2 binding for SARS-CoV-2 is much more temperature-sensitive than that for SARS-CoV. Thus, it is expected that SARS-CoV-2 would decrease its infection ability much faster than SARS-CoV when the temperature rises. These findings would be beneficial for the disease prevention and drug/vaccine development of SARS-CoV-2.\n",
      "The coronavirus disease 2019 (COVID-19) pandemic, caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), is a major public health concern. A handful of static structures now provide molecular insights into how SARS-CoV-2 and SARS-CoV interact with its host target, which is the angiotensin converting enzyme 2 (ACE2). Molecular recognition, binding and function are dynamic processes. To evaluate this, multiple all atom molecular dynamics simulations of at least 500 ns each were performed to better understand the structural stability and interfacial interactions between the receptor binding domain of the spike protein of SARS-CoV-2 and SARS-CoV bound to ACE2. Several contacts were observed to form, break and reform in the interface during the simulations. Our results indicate that SARS-CoV and SARS-CoV-2 utilizes unique strategies to achieve stable binding to ACE2. Several differences were observed between the residues of SARS-CoV-2 and SARS-CoV that consistently interacted with ACE2. Notably, a stable salt bridge between Lys417 of SARS-CoV-2 spike protein and Asp30 of ACE2 as well as three stable hydrogen bonds between Tyr449, Gln493, and Gln498 of SARS-CoV-2 and Asp38, Glu35, and Lys353 of ACE2 were observed, which were absent in the SARS-CoV-ACE2 interface. Some previously reported residues, which were suggested to enhance the binding affinity of SARS-CoV-2, were not observed to form stable interactions in these simulations. Stable binding to the host receptor is crucial for virus entry. Therefore, special consideration should be given to these stable interactions while designing potential drugs and treatment modalities to target or disrupt this interface.\n",
      "The recent pandemic of COVID-19, caused by SARS-CoV-2, is unarguably the most fearsome compared with the earlier outbreaks caused by other coronaviruses, SARS-CoV and MERS-CoV. Human ACE2 is now established as a receptor for the SARS-CoV-2 spike protein. Where variations in the viral spike protein, in turn, lead to the cross-species transmission of the virus, genetic variations in the host receptor ACE2 may also contribute to the susceptibility and/or resistance against the viral infection. This study aims to explore the binding of the proteins encoded by different human ACE2 allelic variants with SARS-CoV-2 spike protein. Briefly, coding variants of ACE2 corresponding to the reported binding sites for its attachment with coronavirus spike protein were selected and molecular models of these variants were constructed by homology modeling. The models were then superimposed over the native ACE2 and ACE2-spike protein complex, to observe structural changes in the ACE2 variants and their intermolecular interactions with SARS-CoV-2 spike protein, respectively. Despite strong overall structural similarities, the spatial orientation of the key interacting residues varies in the ACE2 variants compared with the wild-type molecule. Most ACE2 variants showed a similar binding affinity for SARS-CoV-2 spike protein as observed in the complex structure of wild-type ACE2 and SARS-CoV-2 spike protein. However, ACE2 alleles, rs73635825 (S19P) and rs143936283 (E329G) showed noticeable variations in their intermolecular interactions with the viral spike protein. In summary, our data provide a structural basis of potential resistance against SARS-CoV-2 infection driven by ACE2 allelic variants.\n",
      "The COVID-19 disease has plagued over 110 countries and has resulted in over 4,000 deaths within 10 weeks. We compare the interaction between the human ACE2 receptor and the SARS-CoV-2 spike protein with that of other pathogenic coronaviruses using molecular dynamics simulations. SARS-CoV, SARS-CoV-2, and HCoV-NL63 recognize ACE2 as the natural receptor but present a distinct binding interface to ACE2 and a different network of residue-residue contacts. SARS-CoV and SARS-CoV-2 have comparable binding affinities achieved by balancing energetics and dynamics. The SARS-CoV-2–ACE2 complex contains a higher number of contacts, a larger interface area, and decreased interface residue fluctuations relative to SARS-CoV. These findings expose an exceptional evolutionary exploration exerted by coronaviruses toward host recognition. We postulate that the versatility of cell receptor binding strategies has immediate implications on therapeutic strategies. One Sentence Summary Molecular dynamics simulations reveal a temporal dimension of coronaviruses interactions with the host receptor.\n",
      "The recent emergence of a novel coronavirus (SARS-CoV-2) in China has caused significant public health concerns. Recently, ACE2 was reported as an entry receptor for SARS-CoV-2. In this study, we present the crystal structure of the C-terminal domain of SARS-CoV-2 (SARS-CoV-2-CTD) spike (S) protein in complex with human ACE2 (hACE2), which reveals a hACE2-binding mode similar overall to that observed for SARS-CoV. However, atomic details at the binding interface demonstrate that key residue substitutions in SARS-CoV-2-CTD slightly strengthen the interaction and lead to higher affinity for receptor binding than SARS-RBD. Additionally, a panel of murine monoclonal antibodies (mAbs) and polyclonal antibodies (pAbs) against SARS-CoV-S1/receptor-binding domain (RBD) were unable to interact with the SARS-CoV-2 S protein, indicating notable differences in antigenicity between SARS-CoV and SARS-CoV-2. These findings shed light on the viral pathogenesis and provide important structural information regarding development of therapeutic countermeasures against the emerging virus.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: gmk4qo4i\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Background Researchers are generating molecular data pertaining to the SARS-CoV-2 RNA genome and its proteins at an unprecedented rate during the COVID-19 pandemic. As a result, there is a critical need for rapid and continuously updated access to the latest molecular data in a format in which all data can be quickly cross-referenced and compared. We adapted our genome browser visualization tool to the viral genome for this purpose. Molecular data, curated from published studies or from database submissions, are mapped to the viral genome and grouped together into “annotation tracks” where they can be visualized along the linear map of the viral genome sequence and programmatically downloaded in standard format for analysis. Results The UCSC Genome Browser for SARS-CoV-2 (https://genome.ucsc.edu/covid19.html) provides continuously updated access to the mutations in the many thousands of SARS-CoV-2 genomes deposited in GISAID and the international nucleotide sequencing databases, displayed alongside phylogenetic trees. These data are augmented with alignments of bat, pangolin, and other animal and human coronavirus genomes, including per-base evolutionary rate analysis. All available annotations are cross-referenced on the virus genome, including those from major databases (PDB, RFAM, IEDB, UniProt) as well as up-to-date individual results from preprints. Annotated data include predicted and validated immune epitopes, promising antibodies, RT-PCR and sequencing primers, CRISPR guides (from research, diagnostics, vaccines, and therapies), and points of interaction between human and viral genes. As a community resource, any user can add manual annotations which are quality checked and shared publicly on the browser the next day. Conclusions We invite all investigators to contribute additional data and annotations to this resource to accelerate research and development activities globally. Contact us at genome-www@soe.ucsc.edu with data suggestions or requests for support for adding data. Rapid sharing of data will accelerate SARS-CoV-2 research, especially when researchers take time to integrate their data with those from other labs on a widely-used community browser platform with standardized machine-readable data formats, such as the SARS-CoV-2 Genome Browser.\n",
      "The SARS-CoV-2 pandemic has led to unprecedented, nearly real-time genetic tracing due to the rapid community sequencing response. Researchers immediately leveraged these data to infer the evolutionary relationships among viral samples and to study key biological questions, including whether host viral genome editing and recombination are features of SARS-CoV-2 evolution. This global sequencing effort is inherently decentralized and must rely on data collected by many labs using a wide variety of molecular and bioinformatic techniques. There is thus a strong possibility that systematic errors associated with lab-specific practices affect some sequences in the repositories. We find that some recurrent mutations in reported SARS-CoV-2 genome sequences have been observed predominantly or exclusively by single labs, co-localize with commonly used primer binding sites and are more likely to affect the protein coding sequences than other similarly recurrent mutations. We show that their inclusion can affect phylogenetic inference on scales relevant to local lineage tracing, and make it appear as though there has been an excess of recurrent mutation and/or recombination among viral lineages. We suggest how samples can be screened and problematic mutations removed. We also develop tools for comparing and visualizing differences among phylogenies and we show that consistent clade- and tree-based comparisons can be made between phylogenies produced by different groups. These will facilitate evolutionary inferences and comparisons among phylogenies produced for a wide array of purposes. Building on the SARS-CoV-2 Genome Browser at UCSC, we present a toolkit to compare, analyze and combine SARS-CoV-2 phylogenies, find and remove potential sequencing errors and establish a widely shared, stable clade structure for a more accurate scientific inference and discourse. Foreword We wish to thank all groups that responded rapidly by producing these invaluable and essential sequence data. Their contributions have enabled an unprecedented, lightning-fast process of scientific discovery---truly an incredible benefit for humanity and for the scientific community. We emphasize that most lab groups with whom we associate specific suspicious alleles are also those who have produced the most sequence data at a time when it was urgently needed. We commend their efforts. We have already contacted each group and many have updated their sequences. Our goal with this work is not to highlight potential errors, but to understand the impacts of these and other kinds of highly recurrent mutations so as to identify commonalities among the suspicious examples that can improve sequence quality and analysis going forward.\n",
      "Background SARS-CoV-2 most likely evolved from a bat beta-coronavirus and started infecting humans in December 2019. Since then it has rapidly infected people around the world, with more than 4.5 million confirmed cases by the middle of May 2020. Early genome sequencing of the virus has enabled the development of molecular diagnostics and the commencement of therapy and vaccine development. The analysis of the early sequences showed relatively few evolutionary selection pressures. However, with the rapid worldwide expansion into diverse human populations, significant genetic variations are becoming increasingly likely. The current limitations on social movement between countries also offers the opportunity for these viral variants to become distinct strains with potential implications for diagnostics, therapies and vaccines. Methods We used the current sequencing archives (NCBI and GISAID) to investigate 15,487 whole genomes, looking for evidence of strain diversification and selective pressure. Results We used 6,294 SNPs to build a phylogenetic tree of SARS-CoV-2 diversity and noted strong evidence for the existence of two major clades and six sub-clades, unevenly distributed across the world. We also noted that convergent evolution has potentially occurred across several locations in the genome, showing selection pressures, including on the spike glycoprotein where we noted a potentially critical mutation that could affect its binding to the ACE2 receptor. We also report on mutations that could prevent current molecular diagnostics from detecting some of the sub-clades. Conclusion The worldwide whole genome sequencing effort is revealing the challenge of developing SARS-CoV-2 containment tools suitable for everyone and the need for data to be continually evaluated to ensure accuracy in outbreak estimations.\n",
      "We gratefully acknowledge the UK COVID-19 Genomics Consortium (COG UK) for funding, and Public Health Wales / Cardiff University and MRC-University of Glasgow Centre for Virus Research for making their COG-UK sequence data publicly available. COG-UK is supported by funding from the Medical Research Council (MRC) part of UK Research & Innovation (UKRI), the National Institute of Health Research (NIHR) and Genome Research Limited, operating as the Wellcome Sanger Institute. The research was supported by the Wellcome Trust Core Award Grant Number 203141/Z/16/Z with funding from the NIHR Oxford BRC. The views expressed are those of the author(s) and not necessarily those of the NHS, the NIHR or the Department of Health. We are deeply grateful to Robert Esnouf and the BMRC Research Computing team for unfailing assistance with computational infrastructure. We also thank Benjamin Carpenter and James Docker for assistance in the laboratory, and Lorne Lonie, Maria Lopopolo, Chris Allen, John Broxholme and the WHG high-throughput genomics team for sequencing and quality control. The HIV clone p92BR025.8 was obtained through the Centre For AIDS Reagents from Drs Beatrice Hahn and Feng Gao, and the UNAIDS Virus Network (courtesy of the NIH AIDS Research and Reference Reagent Program). KAL is supported by The Wellcome Trust and The Royal Society (107652/Z/15/Z). MH, LF, MdC, GMC, NO, LAD, DB, CF and TG are supported by Li Ka Shing Foundation funding awarded to CF. PS is supported by a Wellcome Investigator Award (WT103767MA). Summary SARS-CoV-2, the causative agent of COVID-19, emerged in late 2019 causing a global pandemic, with the United Kingdom (UK) one of the hardest hit countries. Rapid sequencing and publication of consensus genomes have enabled phylogenetic analysis of the virus, demonstrating SARS-CoV-2 evolves relatively slowly1, but with multiple sites in the genome that appear inconsistent with the overall consensus phylogeny2. To understand these discrepancies, we used veSEQ3, a targeted RNA-seq approach, to quantify minor allele frequencies in 413 clinical samples from two UK locations. We show that SARS-CoV-2 infections are characterised by extensive within-host diversity, which is frequently shared among infected individuals with patterns consistent with geographical structure. These results were reproducible in data from two other sequencing locations in the UK, where we find evidence of mixed infection by major circulating lineages with patterns that cannot readily be explained by artefacts in the data. We conclude that SARS-CoV-2 diversity is transmissible, and propose that geographic patterns are generated by transient co-circulation of distinct viral populations. Co-transmission of mixed populations could open opportunities for resolving clusters of transmission and understanding pathogenesis.\n",
      "COVID-19 is a newly communicable disease with a catastrophe outbreak that affects all over the world. We retrieved about 8,781 nucleotide fragments and complete genomes of SARS-CoV-2 reported from sixty-four countries. The CoV-2 reference genome was obtained from the National Genomics Data Center (NGDC), GISAID, and NCBI Genbank. All the sequences were aligned against reference genomes using Clustal Omega and variants were called using in-house built Python script. We intend to establish a user-friendly online resource to visualize the variants in the viral genome along with the Primer Infopedia. After analyzing and filtering the data globally, it was made available to the public. The detail of data available to the public includes mutations from 5688 SARS-CoV-2 sequences curated from 91 regions. This database incorporated 39920 mutations over 3990 unique positions. According to the translational impact, these mutations include 11829 synonymous mutations including 681 synonymous frameshifts and 21701 nonsynonymous mutations including 10 nonsynonymous frameshifts. Development of SARS-CoV-2 mutation genome browsers is a fundamental step obliging towards the virus surveillance, viral detection, and development of vaccine and therapeutic drugs. The SARS-COV-2 mutation browser is available at http://covid-19.dnageography.com.\n",
      "The novel human coronavirus (SARS-CoV-2) causes the coronavirus disease 2019 (COVID-19) pandemic worldwide. The increasing sequencing data have shown abundant single nucleotide variations in SARS-CoV-2 genome. However, it is difficult to quickly analyze genomic variation and screen key mutations of SARS-CoV-2. In this study, we developed a visual program, named BioAider, for quick and convenient sequence annotation and mutation analysis on multiple genome-sequencing data. Using BioAider, we conducted a comprehensive genome variation analysis on 3,240 sequences of SARS-CoV-2 genome. Herein, we detected 14 substitution hotspots within SARS-CoV-2 genome, including 10 non-synonymous and 4 synonymous ones. Among these hotspots, NSP13-Y541C was predicted to be a crucial substitution which might affect the unwinding activity of NSP13, a key protein for viral replication. Besides, we also found 3 groups of potentially linked substitution hotspots which were worth further study. In particular, we discovered a SR-rich region (aa 184-204) on the N protein of SARS-CoV-2 distinct from SARS-CoV, indicating more complex replication mechanism and unique N-M interaction of SARS-CoV-2. Interestingly, the quantity of SRXX repeat fragments in the SR-rich region well reflected the evolutionary relationship among SARS-CoV-2 and SARS-CoV-2 related animal coronaviruses, providing further evidence of its animal origin. Overall, we developed an efficient tool for rapid identification of mutations, identified substitution hotspots in SARS-CoV-2 genomes, and detected a distinctive polymorphism SR-rich region in N protein. This tool and the detected hotspots could facilitate the viral genomic study and may contribute for screening antiviral target sites.\n",
      "Background The COVID-19 pandemic is causing a major global health and socio-economic burden, instigating the mobilisation of resources into the development of control tools, such as diagnostics and vaccines. The poor performance of some diagnostic serological tools has emphasised the need for up to date immune-informatic analyses to inform the selection of viable targets for further study. This requires the integration and analysis of genetic and immunological data for SARS-CoV-2 and its homology with other human coronavirus species to understand cross-reactivity. Methods We have developed an online tool for SARS-CoV-2 research, which combines an extensive epitope mapping and prediction meta-analysis, with an updated variant database (55,944 non-synonymous mutations) based on 16,087 whole genome sequences, and an analysis of human coronavirus homology. To demonstrate its utility, we present an integrated analysis of the SARS-CoV-2 spike and nucleocapsid proteins, which are candidate vaccine and serological diagnostic targets. Results Our analysis reveals that the nucleocapsid protein in its native form appears to be a sub-optimal target for use in serological diagnostic platforms. Whilst, a further analysis suggests that orf3a proteins may be a suitable alternative target for diagnostic assays. Conclusions The tool can be accessed online (http://genomics.lshtm.ac.uk/immuno) and will serve as a useful tool for biological discovery in the fight against SARS-CoV-2. Further, it may be adapted to inform on biological targets in future outbreaks, including new human coronaviruses that spill over from animal hosts.\n",
      "Abstract SARS-CoV-2 is a SARS-like coronavirus of likely zoonotic origin first identified in December 2019 in Wuhan, the capital of China's Hubei province. The virus has since spread globally, resulting in the currently ongoing COVID-19 pandemic. The first whole genome sequence was published on January 52,020, and thousands of genomes have been sequenced since this date. This resource allows unprecedented insights into the past demography of SARS-CoV-2 but also monitoring of how the virus is adapting to its novel human host, providing information to direct drug and vaccine design. We curated a dataset of 7666 public genome assemblies and analysed the emergence of genomic diversity over time. Our results are in line with previous estimates and point to all sequences sharing a common ancestor towards the end of 2019, supporting this as the period when SARS-CoV-2 jumped into its human host. Due to extensive transmission, the genetic diversity of the virus in several countries recapitulates a large fraction of its worldwide genetic diversity. We identify regions of the SARS-CoV-2 genome that have remained largely invariant to date, and others that have already accumulated diversity. By focusing on mutations which have emerged independently multiple times (homoplasies), we identify 198 filtered recurrent mutations in the SARS-CoV-2 genome. Nearly 80% of the recurrent mutations produced non-synonymous changes at the protein level, suggesting possible ongoing adaptation of SARS-CoV-2. Three sites in Orf1ab in the regions encoding Nsp6, Nsp11, Nsp13, and one in the Spike protein are characterised by a particularly large number of recurrent mutations (>15 events) which may signpost convergent evolution and are of particular interest in the context of adaptation of SARS-CoV-2 to the human host. We additionally provide an interactive user-friendly web-application to query the alignment of the 7666 SARS-CoV-2 genomes.\n",
      "Background This is a comprehensive analysis of 46 Indian SARS-CoV-2 genome sequences available from the NCBI and GISAID repository during early 2020. Evolutionary dynamics, gene-specific phylogeny and emergence of the novel co-evolving mutations in nine structural and non-structural genes among circulating SARS-CoV-2 strains in ten states of India have been assessed. Materials and methods 46 SARS-CoV-2 nucleotide sequences submitted from India were downloaded from the GISAID (39/46) or from NCBI (7/46) database. Phylogenetic study and analyses of mutation were based on the nine structural and non-structural genes of SARS-CoV-2 strains. Secondary structure of RdRP/NSP12 protein was predicted with respect to the novel A97V mutation. Results Phylogenetic analyses revealed the evolution of “genome-type clusters” and adaptive selection of “L” type SARS-CoV-2 strains with genetic closeness to the bat SARS-like coronaviruses than pangolin or MERS-CoVs. With regards to the novel co-evolving mutations, 2 groups are seen to circulate in India at present: the “major group” (52.2%) and the “minor group” (30.4%), harboring four and five co-existing mutations, respectively. The “major group” mutations fall in the A2a clade. All the minor group mutations, except 11083G>T (L37F, NSP6) were unique to the Indian isolates. Conclusion The study highlights rapidly evolving SARS-CoV-2 virus and co-circulation of multiple clades and sub-clades, driving this pandemic worldwide. This comprehensive study is a potential resource for monitoring the novel mutations in the viral genome, changes in viral pathogenesis, for designing vaccines and other therapeutics.\n",
      "We present an approach to investigate SARS-CoV-2 virus sequences based on alignment-free methods for RNA sequence comparison. In particular, we verify a given clustering result for the GISAID data set, which was obtained analyzing the molecular differences in coronavirus populations by phylogenetic trees. For this purpose, we use alignment-free dissimilarity measures for sequences and combine them with learning vector quantization classifiers for virus type discriminant analysis and classification. Those vector quantizers belong to the class of interpretable machine learning methods, which, on the one hand side provide additional knowledge about the classification decisions like discriminant feature correlations, and on the other hand can be equipped with a reject option. This option gives the model the property of self controlled evidence if applied to new data, i.e. the models refuses to make a classification decision, if the model evidence for the presented data is not given. After training such a classifier for the GISAID data set, we apply the obtained classifier model to another but unlabeled SARS-CoV-2 virus data set. On the one hand side, this allows us to assign new sequences to already known virus types and, on the other hand, the rejected sequences allow speculations about new virus types with respect to nucleotide base mutations in the viral sequences. Author summary The currently emerging global disease COVID-19 caused by novel SARS-CoV-2 viruses requires all scientific effort to investigate the development of the viral epidemy, the properties of the virus and its types. Investigations of the virus sequence are of special interest. Frequently, those are based on mathematical/statistical analysis. However, machine learning methods represent a promising alternative, if one focuses on interpretable models, i.e. those that do not act as black-boxes. Doing so, we apply variants of Learning Vector Quantizers to analyze the SARS-CoV-2 sequences. We encoded the sequences and compared them in their numerical representations to avoid the computationally costly comparison based on sequence alignments. Our resulting model is interpretable, robust, efficient, and has a self-controlling mechanism regarding the applicability to data. This framework was applied to two data sets concerning SARS-CoV-2. We were able to verify previously published virus type findings for one of the data sets by training our model to accurately identify the virus type of sequences. For sequences without virus type information (second data set), our trained model can predict them. Thereby, we observe a new scattered spreading of the sequences in the data space which probably is caused by mutations in the viral sequences.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: r0gr0bhl\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What new public datasets are available related to COVID-19?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "COVID-19 is a newly communicable disease with a catastrophe outbreak that affects all over the world. We retrieved about 8,781 nucleotide fragments and complete genomes of SARS-CoV-2 reported from sixty-four countries. The CoV-2 reference genome was obtained from the National Genomics Data Center (NGDC), GISAID, and NCBI Genbank. All the sequences were aligned against reference genomes using Clustal Omega and variants were called using in-house built Python script. We intend to establish a user-friendly online resource to visualize the variants in the viral genome along with the Primer Infopedia. After analyzing and filtering the data globally, it was made available to the public. The detail of data available to the public includes mutations from 5688 SARS-CoV-2 sequences curated from 91 regions. This database incorporated 39920 mutations over 3990 unique positions. According to the translational impact, these mutations include 11829 synonymous mutations including 681 synonymous frameshifts and 21701 nonsynonymous mutations including 10 nonsynonymous frameshifts. Development of SARS-CoV-2 mutation genome browsers is a fundamental step obliging towards the virus surveillance, viral detection, and development of vaccine and therapeutic drugs. The SARS-COV-2 mutation browser is available at http://covid-19.dnageography.com.\n",
      "In late December 2019, an emerging viral infection COVID-19 was identified in Wuhan, China, and became a global pandemic. Characterization of the genetic variants of SARS-CoV-2 is crucial in following and evaluating it spread across countries. In this study, we collected and analyzed 3,067 SARS-CoV-2 genomes isolated from 55 countries during the first three months after the onset of this virus. Using comparative genomics analysis, we traced the profiles of the whole-genome mutations and compared the frequency of each mutation in the studied population. The accumulation of mutations during the epidemic period with their geographic locations was also monitored. The results showed 782 variant sites, of which 512 (65.47%) had a non-synonymous effect. Frequencies of mutated alleles revealed the presence of 38 recurrent non-synonymous mutations, including ten hotspot mutations with a prevalence higher than 0.10 in this population and distributed in six SARS-CoV-2 genes. The distribution of these recurrent mutations on the world map revealed certain genotypes specific to the geographic location. We also found co-occurring mutations resulting in the presence of several haplotypes. Moreover, evolution over time has shown a mechanism of mutation co-accumulation which might affect the severity and spread of the SARS-CoV-2. On the other hand, analysis of the selective pressure revealed the presence of negatively selected residues that could be taken into considerations as therapeutic targets We have also created an inclusive unified database (http://genoma.ma/covid-19/) that lists all of the genetic variants of the SARS-CoV-2 genomes found in this study with phylogeographic analysis around the world.\n",
      "A new severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) associated with human to human transmission and extreme human sickness has been as of late announced from the city of Wuhan in China. Our objectives were to mutation analysis between recently reported genomes at various times and locations and to characterize the genomic structure of SARS-CoV-2 using bioinformatics programs. Information on the variation of viruses is of considerable medical and biological impacts on the prevention, diagnosis, and therapy of infectious diseases. To understand the genomic structure and variations of the SARS-CoV-2. The study analyzed 95 SARS-CoV-2 complete genome sequences available in GenBank, National MicrobiologyData Center (NMDC) and NGDC Genome Warehouse from December-2019 until 05 of April-2020. The genomic signature analysis demonstrates that a strong association between the time of sample collection, location of sample and accumulation of genetic diversity. We found 116 mutations, the three most common mutations were 8782C>T in ORF1ab gene, 28144T>C in ORF8 gene and 29095C>T in the N gene. The mutations might affect the severity and spread of the SARS-CoV-2. The finding heavily supports an intense requirement for additional prompt, inclusive investigations that combine genomic detail, epidemiological information and graph records of the clinical features of patients with COVID-19.\n",
      "The novel human coronavirus (SARS-CoV-2) causes the coronavirus disease 2019 (COVID-19) pandemic worldwide. The increasing sequencing data have shown abundant single nucleotide variations in SARS-CoV-2 genome. However, it is difficult to quickly analyze genomic variation and screen key mutations of SARS-CoV-2. In this study, we developed a visual program, named BioAider, for quick and convenient sequence annotation and mutation analysis on multiple genome-sequencing data. Using BioAider, we conducted a comprehensive genome variation analysis on 3,240 sequences of SARS-CoV-2 genome. Herein, we detected 14 substitution hotspots within SARS-CoV-2 genome, including 10 non-synonymous and 4 synonymous ones. Among these hotspots, NSP13-Y541C was predicted to be a crucial substitution which might affect the unwinding activity of NSP13, a key protein for viral replication. Besides, we also found 3 groups of potentially linked substitution hotspots which were worth further study. In particular, we discovered a SR-rich region (aa 184-204) on the N protein of SARS-CoV-2 distinct from SARS-CoV, indicating more complex replication mechanism and unique N-M interaction of SARS-CoV-2. Interestingly, the quantity of SRXX repeat fragments in the SR-rich region well reflected the evolutionary relationship among SARS-CoV-2 and SARS-CoV-2 related animal coronaviruses, providing further evidence of its animal origin. Overall, we developed an efficient tool for rapid identification of mutations, identified substitution hotspots in SARS-CoV-2 genomes, and detected a distinctive polymorphism SR-rich region in N protein. This tool and the detected hotspots could facilitate the viral genomic study and may contribute for screening antiviral target sites.\n",
      "The COVID-19 pandemic is the most significant public health issue in recent history. Its causal agent, SARS-CoV-2, has evolved rapidly since its first emergence in December 2019. Mutations in the viral genome have critical impacts on the adaptation of viral strains to the local environment, and may alter the characteristics of viral transmission, disease manifestation, and the efficacy of treatment and vaccination. Using the complete sequences of 1,932 SARS-CoV-2 genomes, we examined the genomic, geographic and temporal distributions of aged, new, and frequent mutations of SARS-CoV-2, and identified six phylogenetic clusters of the strains, which also exhibit a geographic preference in different continents. Mutations in the form of single nucleotide variations (SNVs) provide a direct interpretation for the six phylogenetic clusters. Linkage disequilibrium, haplotype structure, evolutionary process, global distribution of mutations unveiled a sketch of the mutational history. Additionally, we found a positive correlation between the average mutation count and case fatality, and this correlation had strengthened with time, suggesting an important role of SNVs on disease outcomes. This study suggests that SNVs may become an important consideration in virus detection, clinical treatment, drug design, and vaccine development to avoid target shifting, and that continued isolation and sequencing is a crucial component in the fight against this pandemic. Significance Statement Mutation is the driving force of evolution for viruses like SARS-CoV-2, the causal agent of COVID-19. In this study, we discovered that the genome of SARS-CoV-2 is changing rapidly from the originally isolated form. These mutations have been spreading around the world and caused more than 2.5 million of infected cases and 170 thousands of deaths. We found that fourteen frequent mutations identified in this study can characterize the six main clusters of SARS-CoV-2 strains. In addition, we found the mutation burden is positively correlated with the fatality of COVID-19 patients. Understanding mutations in the SARS-CoV-2 genome will provide useful insight for the design of treatment and vaccination.\n",
      "Background Researchers are generating molecular data pertaining to the SARS-CoV-2 RNA genome and its proteins at an unprecedented rate during the COVID-19 pandemic. As a result, there is a critical need for rapid and continuously updated access to the latest molecular data in a format in which all data can be quickly cross-referenced and compared. We adapted our genome browser visualization tool to the viral genome for this purpose. Molecular data, curated from published studies or from database submissions, are mapped to the viral genome and grouped together into “annotation tracks” where they can be visualized along the linear map of the viral genome sequence and programmatically downloaded in standard format for analysis. Results The UCSC Genome Browser for SARS-CoV-2 (https://genome.ucsc.edu/covid19.html) provides continuously updated access to the mutations in the many thousands of SARS-CoV-2 genomes deposited in GISAID and the international nucleotide sequencing databases, displayed alongside phylogenetic trees. These data are augmented with alignments of bat, pangolin, and other animal and human coronavirus genomes, including per-base evolutionary rate analysis. All available annotations are cross-referenced on the virus genome, including those from major databases (PDB, RFAM, IEDB, UniProt) as well as up-to-date individual results from preprints. Annotated data include predicted and validated immune epitopes, promising antibodies, RT-PCR and sequencing primers, CRISPR guides (from research, diagnostics, vaccines, and therapies), and points of interaction between human and viral genes. As a community resource, any user can add manual annotations which are quality checked and shared publicly on the browser the next day. Conclusions We invite all investigators to contribute additional data and annotations to this resource to accelerate research and development activities globally. Contact us at genome-www@soe.ucsc.edu with data suggestions or requests for support for adding data. Rapid sharing of data will accelerate SARS-CoV-2 research, especially when researchers take time to integrate their data with those from other labs on a widely-used community browser platform with standardized machine-readable data formats, such as the SARS-CoV-2 Genome Browser.\n",
      "A novel pathogen, named SARS-CoV-2, has caused an unprecedented worldwide pandemic in the first half of 2020. As the SARS-CoV-2 genome sequences have become available, one of the important focus of scientists has become tracking variations in the viral genome. In this study, 30366 SARS-CoV-2 isolate genomes were aligned using the software developed by our group (ODOTool) and 11 variations in SARS-CoV-2 genome over 10% of whole isolates were discussed. Results indicated that, frequency rates of these 11 variations change between 3.56%–88.44 % and these rates differ greatly depending on the continents they have been reported. Despite some variations being in low frequency rate in some continents, C14408T and A23403G variations on Nsp12 and S protein, respectively, observed to be the most prominent variations all over the world, in general, and both cause missense mutations. It is also notable that most of isolates carry C14408T and A23403 variations simultaneously and also nearly all isolates carrying the G25563T variation on ORF3a, also carry C14408T and A23403 variations, although their location distributions are not similar. All these data should be considered towards development of vaccine and antiviral treatment strategies as well as tracing diversity of virus in all over the world.\n",
      "Abstract SARS-CoV-2 is a SARS-like coronavirus of likely zoonotic origin first identified in December 2019 in Wuhan, the capital of China's Hubei province. The virus has since spread globally, resulting in the currently ongoing COVID-19 pandemic. The first whole genome sequence was published on January 52,020, and thousands of genomes have been sequenced since this date. This resource allows unprecedented insights into the past demography of SARS-CoV-2 but also monitoring of how the virus is adapting to its novel human host, providing information to direct drug and vaccine design. We curated a dataset of 7666 public genome assemblies and analysed the emergence of genomic diversity over time. Our results are in line with previous estimates and point to all sequences sharing a common ancestor towards the end of 2019, supporting this as the period when SARS-CoV-2 jumped into its human host. Due to extensive transmission, the genetic diversity of the virus in several countries recapitulates a large fraction of its worldwide genetic diversity. We identify regions of the SARS-CoV-2 genome that have remained largely invariant to date, and others that have already accumulated diversity. By focusing on mutations which have emerged independently multiple times (homoplasies), we identify 198 filtered recurrent mutations in the SARS-CoV-2 genome. Nearly 80% of the recurrent mutations produced non-synonymous changes at the protein level, suggesting possible ongoing adaptation of SARS-CoV-2. Three sites in Orf1ab in the regions encoding Nsp6, Nsp11, Nsp13, and one in the Spike protein are characterised by a particularly large number of recurrent mutations (>15 events) which may signpost convergent evolution and are of particular interest in the context of adaptation of SARS-CoV-2 to the human host. We additionally provide an interactive user-friendly web-application to query the alignment of the 7666 SARS-CoV-2 genomes.\n",
      "The recent pandemic of SARS-CoV-2 infection has affected more than 3.0 million people worldwide with more than 200 thousand reported deaths. The SARS-CoV-2 genome has the capability of gaining rapid mutations as the virus spreads. Whole-genome sequencing data offers a wide range of opportunities to study mutation dynamics. The advantage of an increasing amount of whole-genome sequence data of SARS-CoV-2 intrigued us to explore the mutation profile across the genome, to check the genome diversity, and to investigate the implications of those mutations in protein stability and viral transmission. We have identified frequently mutated residues by aligning ~660 SARS-CoV-2 genomes and validated in 10,000 datasets available in GISAID Nextstrain. We further evaluated the potential of these frequently mutated residues in protein structure stability of spike glycoprotein and their possible functional consequences in other proteins. Among the 11 genes, surface glycoprotein, nucleocapsid, ORF1ab, and ORF8 showed frequent mutations, while envelop, membrane, ORF6, ORF7a and ORF7b showed conservation in terms of amino acid substitutions. Combined analysis with the frequently mutated residues identified 20 viral variants, among which 12 specific combinations comprised more than 97% of the isolates considered for the analysis. Some of the mutations across different proteins showed co-occurrences, suggesting their structural and/or functional interaction among different SARS-COV-2 proteins, and their involvement in adaptability and viral transmission. Analysis of protein structure stability of surface glycoprotein mutants indicated the viability of specific variants and are more prone to be temporally and spatially distributed across the globe. A similar empirical analysis of other proteins indicated the existence of important functional implications of several variants. Identification of frequently mutated variants among COVID-19 patients might be useful for better clinical management, contact tracing, and containment of the disease.\n",
      "The COVID-19 pandemic has spread across the globe at an alarming rate in the last four months. However, unlike any of the previous global outbreaks the availability of hundreds of SARS-CoV-2 sequences provides us with a unique opportunity to understand viral evolution in real time. We analysed 480 full-length (>29000 nt) sequences from the 1575 SARS-CoV-2 sequences available and identified 37 single-nucleotide substitutions occurring in >1% of the genomes. Majority of the substitutions were C to T or G to A. We identify C/Gs with an upstream TTT trinucleotide motif as hotspots for mutations in the SARS-CoV-2 genome. Interestingly, two of the 37 substitutions occur within highly conserved secondary structures in the 5’ and 3’ untranslated regions that are critical for the virus life cycle. Furthermore, clustering analysis revealed unique geographical distribution of SARS-CoV-2 variants defined by their mutation profile. Of note, we observed several co-occurring mutations that almost never occur individually. We define 3 mutually exclusive lineages (A1, B1 and C1) of SARS-CoV-2 which account for about three quarters of the genomes analysed. We identify lineage-defining leading mutations in the SARS-CoV-2 genome which precede the occurrence of sub-lineage defining trailing mutations. The identification of mutually exclusive lineage-defining mutations with geographically restricted patterns of distribution has potential implications for diagnosis, pathogenesis and vaccine design. Our work provides novel insights on the temporal evolution of SARS-CoV-2.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 79yna07e\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What vaccine candidates are being tested for Covid-19?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "The ongoing coronavirus disease 2019 (COVID-19) pandemic is a serious threat to global public health, and imposes severe burdens on the entire human society. The severe acute respiratory syndrome (SARS) coronavirus-2 (SARS-CoV-2) can cause severe respiratory illness and death. Currently, there are no specific antiviral drugs that can treat COVID-19. Several vaccines against SARS-CoV-2 are being actively developed by research groups around the world. The surface S (spike) protein and the highly expressed internal N (nucleocapsid) protein of SARS-CoV-2 are widely considered as promising candidates for vaccines. In order to guide the design of an effective vaccine, we need experimental data on these potential epitope candidates. In this study, we mapped the immunodominant (ID) sites of S protein using sera samples collected from recently discharged COVID-19 patients. The SARS-CoV-2 S protein-specific antibody levels in the sera of recovered COVID-19 patients were strongly correlated with the neutralising antibody titres. We used epitope mapping to determine the landscape of ID sites of S protein, which identified nine linearized B cell ID sites. Four out of the nine ID sites were found in the receptor-binding domain (RBD). Further analysis showed that these ID sites are potential high-affinity SARS-CoV-2 antibody binding sites. Peptides containing two out of the nine sites were tested as vaccine candidates against SARS-CoV-2 in a mouse model. We detected epitope-specific antibodies and SARS-CoV-2-neutralising activity in the immunised mice. This study for the first time provides human serological data for the design of vaccines against COVID-19.\n",
      "The 2019 novel severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) outbreak has caused a large number of deaths, with thousands of confirmed cases worldwide. The present study followed computational approaches to identify B- and T-cell epitopes for the spike (S) glycoprotein of SARS-CoV-2 by its interactions with the human leukocyte antigen alleles. We identified 24 peptide stretches on the SARS-CoV-2 S protein that are well conserved among the reported strains. The S protein structure further validated the presence of predicted peptides on the surface, of which 20 are surface exposed and predicted to have reasonable epitope binding efficiency. The work could be useful for understanding the immunodominant regions in the surface protein of SARS-CoV-2 and could potentially help in designing some peptide-based diagnostics. Also, identified T-cell epitopes might be considered for incorporation in vaccine designs.\n",
      "Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which first occurred in Wuhan (China) in December 2019, is a novel virus that causes a severe acute respiratory disease. The virus spike glycoproteins and nucleocapsid proteins are the main targets for the development of vaccines and antiviral drugs, to control the disease spread. We herein study the structural order-disorder propensity and the rates of evolution of these two proteins to characterize their B cell and T cell epitopes, previously suggested to contribute to immune response caused by SARS-CoV-2 infections. We first analyzed the rates of evolution along the sequences of spike and nucleocapsid proteins in relation to the spatial locations of their epitopes. For this purpose, we compared orthologs from seven human coronaviruses: SARS-CoV-2, SARS-CoV, MERS-CoV, HCoV-229E, HCoV-OC43, HCoV-NL63, and HCoV-HKU1. We then focus on the local, structural order-disorder propensities of the protein regions where the SARS-CoV-2 epitopes are located. We show that the vast majority of nucleocapsid protein epitopes overlap the RNA-binding and dimerization domains and some of them are characterized by low rates of evolutions. Similarly, spike protein epitopes are preferentially located in regions that are predicted to be ordered and well-conserved, in correspondence of the heptad repeats 1 and 2. Interestingly, both the receptor-binding motif to ACE2 and the fusion peptide of spike protein are characterized by high rates of evolution, probably to overcome host immunity. In conclusion, our results provide evidence for conserved epitopes that may help to develop long-lasting, broad-spectrum SARS-CoV-2 vaccines.\n",
      "The pandemic of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) presents a global public health threat. Most research on therapeutics against SARS-CoV-2 focused on the receptor binding domain (RBD) of the Spike (S) protein, whereas the vulnerable epitopes and functional mechanism of non-RBD regions are poorly understood. Here we isolated and characterized monoclonal antibodies (mAbs) derived from convalescent COVID-19 patients. An mAb targeting the N-terminal domain (NTD) of the SARS-CoV-2 S protein, named 4A8, exhibits high neutralization potency against both authentic and pseudotyped SARS-CoV-2, although it does not block the interaction between angiotensin-converting enzyme 2 (ACE2) receptor and S protein. The cryo-EM structure of the SARS-CoV-2 S protein in complex with 4A8 has been determined to an overall resolution of 3.1 Angstrom and local resolution of 3.4 Angstrom for the 4A8-NTD interface, revealing detailed interactions between the NTD and 4A8. Our functional and structural characterizations discover a new vulnerable epitope of the S protein and identify promising neutralizing mAbs as potential clinical therapy for COVID-19.\n",
      "The 2019 novel severe respiratory syndrome coronavirus-2 (SARS-CoV-2) outbreak has caused a large number of deaths with thousands of confirmed cases worldwide. The present study followed computational approaches to identify B- and T-cell epitopes for spike glycoprotein of SARS-CoV-2 by its interactions with the human leukocyte antigen alleles. We identified twenty-four peptide stretches on the SARS-CoV-2 spike protein that are well conserved among the reported strains. The S protein structure further validated the presence of predicted peptides on the surface. Out of which twenty are surface exposed and predicted to have reasonable epitope binding efficiency. The work could be useful for understanding the immunodominant regions in the surface protein of SARS-CoV-2 and could potentially help in designing some peptide-based diagnostics.\n",
      "The ongoing pandemic of the coronavirus disease 2019 (COVID-19) is an infectious disease caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV2). We have performed an integrated sequence-based analysis of SARS-CoV2 genomes from different geographical locations in order to identify its unique features absent in SARS-CoV and other related coronavirus family genomes, conferring unique infection, facilitation of transmission, virulence and immunogenic features to the virus. The phylogeny of the genomes yields some interesting results. Systematic gene level mutational analysis of the genomes has enabled us to identify several unique features of the SARS-CoV2 genome, which includes a unique mutation in the spike surface glycoprotein (A930V (24351C>T)) in the Indian SARS-CoV2, absent in other strains studied here. We have also predicted the impact of the mutations in the spike glycoprotein function and stability, using computational approach. To gain further insights into host responses to viral infection, we predict that antiviral host-miRNAs may be controlling the viral pathogenesis. Our analysis reveals nine host miRNAs which can potentially target SARS-CoV2 genes. Interestingly, the nine miRNAs do not have targets in SARS and MERS genomes. Also, hsa-miR-27b is the only unique miRNA which has a target gene in the Indian SARS-CoV2 genome. We also predicted immune epitopes in the genomes\n",
      "The outbreak of the 2019 novel coronavirus (SARS-CoV-2) has infected thousands of people with a large number of deaths across 26 countries. The sudden appearance of the virus leads to the limited existing therapies for SARS-CoV-2. Therefore, vaccines and antiviral medicines are in desperate need. This study took immune-informatics approaches to identify B- and T-cell epitopes for surface glycoprotein (S) of SARS-CoV-2, followed by estimating their antigenicity and interactions with the human leukocyte antigen (HLA) alleles. We identified four B cell epitopes, two MHC class-I and nine MHC class-II binding T-cell epitopes, which showed highly antigenic features. Allergenicity, toxicity and physiochemical properties analysis confirmed the specificity and selectivity of epitopes. The stability and safety of epitopes were confirmed by digestion analysis. No mutations were observed in all the selected B- and T-cell epitopes across all isolates from different locations worldwide. Epitopes were thus identified and some of them can be potential candidates for vaccine development.\n",
      "Recently emerged severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the pathogen responsible for the ongoing coronavirus disease 2019 (COVID-19) pandemic. Currently, there is no vaccine available for preventing SARS-CoV-2 infection. Like closely related severe acute respiratory syndrome coronavirus (SARS-CoV), SARS-CoV-2 also uses its receptor-binding domain (RBD) on the spike (S) protein to engage the host receptor, human angiotensin-converting enzyme 2 (ACE2), facilitating subsequent viral entry. Here we report the immunogenicity and vaccine potential of SARS-CoV-2 RBD (SARS2-RBD)-based recombinant proteins. Immunization with SARS2-RBD recombinant proteins potently induced a multi-functional antibody response in mice. The resulting antisera could efficiently block the interaction between SARS2-RBD and ACE2, inhibit S-mediated cell-cell fusion, and neutralize both SARS-CoV-2 pseudovirus entry and authentic SARS-CoV-2 infection. In addition, the anti-RBD sera also exhibited cross binding, ACE2-blockade, and neutralization effects towards SARS-CoV. More importantly, we found that the anti-RBD sera did not promote antibody-dependent enhancement of either SARS-CoV-2 pseudovirus entry or authentic virus infection of Fc receptor-bearing cells. These findings provide a solid foundation for developing RBD-based subunit vaccines for SARS-CoV2.\n",
      "Coronavirus disease 2019 (COVID-19) is a global health crisis caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), and there is a critical need to produce large quantities of high-quality SARS-CoV-2 Spike (S) protein for use in both clinical and basic science settings. To address this need, we have evaluated the expression and purification of two previously reported S protein constructs in Expi293F™ and ExpiCHO-S™ cells, two different cell lines selected for increased expression of secreted glycoproteins. We show that ExpiCHO-S™ cells produce enhanced yields of both SARS-CoV-2 S proteins. Biochemical, biophysical, and structural (cryo-EM) characterization of the SARS-CoV-2 S proteins produced in both cell lines demonstrate that the reported purification strategy yields high quality S protein (non-aggregated, uniform material with appropriate biochemical and biophysical properties). Importantly, we show that multiple preparations of these two recombinant S proteins from either cell line exhibit identical behavior in two different serology assays. We also evaluate the specificity of S protein-mediated host cell binding by examining interactions with proposed binding partners in the human secretome. In addition, the antigenicity of these proteins is demonstrated by standard ELISAs, and in a flexible protein microarray format. Collectively, we establish an array of metrics for ensuring the production of high-quality S protein to support clinical, biological, biochemical, structural and mechanistic studies to combat the global pandemic caused by SARS-CoV-2.\n",
      "A novel coronavirus (SARS-CoV-2) emerged from China in late 2019 and rapidly spread across the globe, infecting millions of people and generating societal disruption on a level not seen since the 1918 influenza pandemic. A safe and effective vaccine is desperately needed to prevent the continued spread of SARS-CoV-2; yet, rational vaccine design efforts are currently hampered by the lack of knowledge regarding viral epitopes targeted during an immune response, and the need for more in-depth knowledge on betacoronavirus immunology. To that end, we developed a computational workflow using a series of open-source algorithms and webtools to analyze the proteome of SARS-CoV-2 and identify putative T cell and B cell epitopes. Using increasingly stringent selection criteria to select peptides with significant HLA promiscuity and predicted antigenicity, we identified 41 potential T cell epitopes (5 HLA class I, 36 HLA class II) and 6 potential B cell epitopes, respectively. Docking analysis and binding predictions demonstrated enrichment for peptide binding to HLA-B (class I) and HLA-DRB1 (class II) molecules. Overlays of predicted B cell epitopes with the structure of the viral spike (S) glycoprotein revealed that 4 of 6 epitopes were located in the receptor-binding domain of the S protein. To our knowledge, this is the first study to comprehensively analyze all 10 (structural, non-structural and accessory) proteins from SARS-CoV-2 using predictive algorithms to identify potential targets for vaccine development. Significance Statement The novel coronavirus SARS-CoV-2 recently emerged from China, rapidly spreading and ushering in a global pandemic. Despite intensive research efforts, our knowledge of SARS-CoV-2 immunology and the proteins targeted by the immune response remains relatively limited, making it difficult to rationally design candidate vaccines. We employed a suite of bioinformatic tools, computational algorithms, and structural modeling to comprehensively analyze the entire SARS-CoV-2 proteome for potential T cell and B cell epitopes. Utilizing a set of stringent selection criteria to filter peptide epitopes, we identified 41 T cell epitopes (5 HLA class I, 36 HLA class II) and 6 B cell epitopes that could serve as promising targets for peptide-based vaccine development against this emerging global pathogen.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 915srotp\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "what is the origin of COVID-19\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Background This is a comprehensive analysis of 46 Indian SARS-CoV-2 genome sequences available from the NCBI and GISAID repository during early 2020. Evolutionary dynamics, gene-specific phylogeny and emergence of the novel co-evolving mutations in nine structural and non-structural genes among circulating SARS-CoV-2 strains in ten states of India have been assessed. Materials and methods 46 SARS-CoV-2 nucleotide sequences submitted from India were downloaded from the GISAID (39/46) or from NCBI (7/46) database. Phylogenetic study and analyses of mutation were based on the nine structural and non-structural genes of SARS-CoV-2 strains. Secondary structure of RdRP/NSP12 protein was predicted with respect to the novel A97V mutation. Results Phylogenetic analyses revealed the evolution of “genome-type clusters” and adaptive selection of “L” type SARS-CoV-2 strains with genetic closeness to the bat SARS-like coronaviruses than pangolin or MERS-CoVs. With regards to the novel co-evolving mutations, 2 groups are seen to circulate in India at present: the “major group” (52.2%) and the “minor group” (30.4%), harboring four and five co-existing mutations, respectively. The “major group” mutations fall in the A2a clade. All the minor group mutations, except 11083G>T (L37F, NSP6) were unique to the Indian isolates. Conclusion The study highlights rapidly evolving SARS-CoV-2 virus and co-circulation of multiple clades and sub-clades, driving this pandemic worldwide. This comprehensive study is a potential resource for monitoring the novel mutations in the viral genome, changes in viral pathogenesis, for designing vaccines and other therapeutics.\n",
      "Background & objectives SARS-CoV-2 (Severe acute respiratory syndrome coronavirus-2) is evolving with the progression of the pandemic. This study was aimed to investigate the diversity and evolution of the coronavirus SARS-CoV-2 with progression of the pandemic over time and to identify similarities and differences of viral diversity and evolution across geographical regions (countries). Methods Publicly available data on type definitions based on whole-genome sequences of the SARS-CoV-2 sampled during December and March 2020 from 3636 infected patients spread over 55 countries were collected. Phylodynamic analyses were performed and the temporal and spatial evolution of the virus was examined. Results It was found that (i) temporal variation in frequencies of types of the coronavirus was significant; ancestral viruses of type O were replaced by evolved viruses belonging to type A2a; (ii) spatial variation was not significant; with the spread of SARS-CoV-2, the dominant virus was the A2a type virus in every geographical region; (iii) within a geographical region, there was significant micro-level variation in the frequencies of the different viral types, and (iv) the evolved coronavirus of type A2a swept rapidly across all continents. Interpretation & conclusions SARS-CoV-2 belonging to the A2a type possesses a non-synomymous variant (D614G) that possibly eases the entry of the virus into the lung cells of the host. This may be the reason why the A2a type has an advantage to infect and survive and as a result has rapidly swept all geographical regions. Therefore, large-scale sequencing of coronavirus genomes and, as required, of host genomes should be undertaken in India to identify regional and ethnic variation in viral composition and its interaction with host genomes. Further, careful collection of clinical and immunological data of the host can provide deep learning in relation to infection and transmission of the types of coronavirus genomes.\n",
      "Background & objectives: SARS-CoV-2 (Severe acute respiratory syndrome coronavirus-2) is evolving with the progression of the pandemic. This study was aimed to investigate the diversity and evolution of the coronavirus SARS-CoV-2 with progression of the pandemic over time and to identify similarities and differences of viral diversity and evolution across geographical regions (countries). Methods: Publicly available data on type definitions based on whole-genome sequences of the SARS-CoV-2 sampled during December and March 2020 from 3636 infected patients spread over 55 countries were collected. Phylodynamic analyses were performed and the temporal and spatial evolution of the virus was examined. Results: It was found that (i) temporal variation in frequencies of types of the coronavirus was significant; ancestral viruses of type O were replaced by evolved viruses belonging to type A2a; (ii) spatial variation was not significant; with the spread of SARS-CoV-2, the dominant virus was the A2a type virus in every geographical region; (iii) within a geographical region, there was significant micro-level variation in the frequencies of the different viral types, and (iv) the evolved coronavirus of type A2a swept rapidly across all continents. Interpretation & conclusions: SARS-CoV-2 belonging to the A2a type possesses a non-synomymous variant (D614G) that possibly eases the entry of the virus into the lung cells of the host. This may be the reason why the A2a type has an advantage to infect and survive and as a result has rapidly swept all geographical regions. Therefore, large-scale sequencing of coronavirus genomes and, as required, of host genomes should be undertaken in India to identify regional and ethnic variation in viral composition and its interaction with host genomes. Further, careful collection of clinical and immunological data of the host can provide deep learning in relation to infection and transmission of the types of coronavirus genomes.\n",
      "Monitoring the mutation dynamics of SARS-CoV-2 is critical for the development of effective approaches to contain the pathogen. By analyzing 106 SARS-CoV-2 and 39 SARS genome sequences, we provided direct genetic evidence that SARS-CoV-2 has a much lower mutation rate than SARS. Minimum Evolution phylogeny analysis revealed the putative original status of SARS-CoV-2 and the early-stage spread history. The discrepant phylogenies for the spike protein and its receptor binding domain proved a previously reported structural rearrangement prior to the emergence of SARS-CoV-2. Despite that we found the spike glycoprotein of SARS-CoV-2 is particularly more conserved, we identified a mutation that leads to weaker receptor binding capability, which concerns a SARS-CoV-2 sample collected on 27th January 2020 from India. This represents the first report of a significant SARS-CoV-2 mutant, and raises the alarm that the ongoing vaccine development may become futile in future epidemic if more mutations were identified. Highlights Based on the currently available genome sequence data, we proved that SARS-COV-2 genome has a much lower mutation rate and genetic diversity than SARS during the 2002-2003 outbreak. The spike (S) protein encoding gene of SARS-COV-2 is found relatively more conserved than other protein-encoding genes, which is a good indication for the ongoing antiviral drug and vaccine development. Minimum Evolution phylogeny analysis revealed the putative original status of SARS-CoV-2 and the early-stage spread history. We confirmed a previously reported rearrangement in the S protein arrangement of SARS-COV-2, and propose that this rearrangement should have occurred between human SARS-CoV and a bat SARS-CoV, at a time point much earlier before SARS-COV-2 transmission to human. We provided first evidence that a mutated SARS-COV-2 with reduced human ACE2 receptor binding affinity have emerged in India based on a sample collected on 27th January 2020.\n",
      "Direct massively parallel sequencing of SARS-CoV-2 genome was undertaken from nasopharyngeal and oropharyngeal swab samples of infected individuals in Eastern India. Seven of the isolates belonged to the A2a clade, while one belonged to the B4 clade. Specific mutations, characteristic of the A2a clade, were also detected, which included the P323L in RNA-dependent RNA polymerase and D614G in the Spike glycoprotein. Further, our data revealed emergence of novel subclones harbouring nonsynonymous mutations, viz. G1124V in Spike (S) protein, R203K, and G204R in the nucleocapsid (N) protein. The N protein mutations reside in the SR-rich region involved in viral capsid formation and the S protein mutation is in the S(2) domain, which is involved in triggering viral fusion with the host cell membrane. Interesting correlation was observed between these mutations and travel or contact history of COVID-19 positive cases. Consequent alterations of miRNA binding and structure were also predicted for these mutations. More importantly, the possible implications of mutation D614G (in S(D) domain) and G1124V (in S(2) subunit) on the structural stability of S protein have also been discussed. Results report for the first time a bird’s eye view on the accumulation of mutations in SARS-CoV-2 genome in Eastern India. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (10.1007/s12038-020-00046-1) contains supplementary material, which is available to authorized users.\n",
      "OBJECTIVES: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan, China, in December 2019 and has been rapidly spreading worldwide. Although the causal relationship among mutations and the features of SARS-CoV-2 such as rapid transmission, pathogenicity, and tropism, remains unclear, our results of genomic mutations in SARS-CoV-2 may help to interpret the interaction between genomic characterization in SARS-CoV-2 and infectivity with the host. METHODS: A total of 4,254 genomic sequences of SARS-CoV-2 were collected from the Global Initiative on Sharing all Influenza Data (GISAID). Multiple sequence alignment for phylogenetic analysis and comparative genomic approach for mutation analysis were conducted using Molecular Evolutionary Genetics Analysis (MEGA), and an in-house program based on Perl language, respectively. RESULTS: Phylogenetic analysis of SARS-CoV-2 strains indicated that there were 3 major clades including S, V, and G, and 2 subclades (G.1 and G.2). There were 767 types of synonymous and 1,352 types of non-synonymous mutation. ORF1a, ORF1b, S, and N genes were detected at high frequency, whereas ORF7b and E genes exhibited low frequency. In the receptor-binding domain (RBD) of the S gene, 11 non-synonymous mutations were observed in the region adjacent to the angiotensin converting enzyme 2 (ACE2) binding site. CONCLUSION: It has been reported that the rapid infectivity and transmission of SARS-CoV-2 associated with host receptor affinity are derived from several mutations in its genes. Without these genetic mutations to enhance evolutionary adaptation, species recognition, host receptor affinity, and pathogenicity, it would not survive. It is expected that our results could provide an important clue in understanding the genomic characteristics of SARS-CoV-2.\n",
      "From an isolated epidemic, COVID-19 has now emerged as a global pandemic. The availability of genomes in the public domain following the epidemic provides a unique opportunity to understand the evolution and spread of the SARS-CoV-2 virus across the globe. The availability of whole genomes from multiple states in India prompted us to analyse the phylogenetic clusters of genomes in India. We performed whole-genome sequencing for 64 genomes making a total of 361 genomes from India, followed by phylogenetic clustering, substitution analysis, and dating of the different phylogenetic clusters of viral genomes. We describe a distinct phylogenetic cluster (Clade I / A3i) of SARS-CoV-2 genomes from India, which encompasses 41% of all genomes sequenced and deposited in the public domain from multiple states in India. Globally 3.5% of genomes, which till date could not be mapped to any distinct known cluster fall in this newly defined clade. The cluster is characterized by a core set of shared genetic variants – C6312A (T2016K), C13730T (A88V/A97V), C23929T, and C28311T (P13L). Further, the cluster is also characterized by a nucleotide substitution rate of 1.4 × 10−3 variants per site per year, lower than the prevalent A2a cluster, and predominantly driven by variants in the E and N genes and relative sparing of the S gene. Epidemiological assessments suggest that the common ancestor emerged in the month of February 2020 and possibly resulted in an outbreak followed by countrywide spread, as evidenced by the low divergence of the genomes from across the country. To the best of our knowledge, this is the first comprehensive study characterizing the distinct and predominant cluster of SARS-CoV-2 in India.\n",
      "Direct massively parallel sequencing of SARS-CoV-2 genome was undertaken from nasopharyngeal and oropharyngeal swab samples of infected individuals in Eastern India. Seven of the isolates belonged to the A2a clade, while one belonged to the B4 clade. Specific mutations, characteristic of the A2a clade, were also detected, which included the P323L in RNA-dependent RNA polymerase and D614G in the Spike glycoprotein. Further, our data revealed emergence of novel subclones harbouring nonsynonymous mutations, viz. G1124V in Spike (S) protein, R203K, and G204R in the nucleocapsid (N) protein. The N protein mutations reside in the SR-rich region involved in viral capsid formation and the S protein mutation is in the S2 domain, which is involved in triggering viral fusion with the host cell membrane. Interesting correlation was observed between these mutations and travel or contact history of COVID-19 positive cases. Consequent alterations of miRNA binding and structure were also predicted for these mutations. More importantly, the possible implications of mutation D614G (in SD domain) and G1124V (in S2 subunit) on the structural stability of S protein have also been discussed. Results report for the first time a bird's eye view on the accumulation of mutations in SARS-CoV-2 genome in Eastern India.\n",
      "India has recently started sequencing SARS-CoV2 genome from clinical isolates. Currently only few sequences are available from three states in India. Kerala was the first state to deposit complete sequence from two isolates followed by one from Gujarat. On April 27, 2020, the first five sequences from the state of West Bengal (Eastern India) were deposited on GISAID, a global initiative for sharing avian flu data. In this study, we have analysed the spike protein sequences from all five isolates and also compared their similarities or differences with other sequences reported in India and with isolates of Wuhan origin. We report one unique mutation at position 723 and another at 1124 in the S2 domain of spike protein of the isolates from West Bengal only. There was one mutation downstream of the receptor binding domain at position 614 in S1 domain which was common with the sequence from Gujarat (a state of western India). Mutation in the S2 domain showed changes in the secondary structure of the spike protein at region of the mutation. We also studied molecular dynamics using normal mode analyses and found that this mutation decreases the flexibility of S2 domain. Since both S1 and S2 are important in receptor binding followed by entry in the host cells, such mutations may define the affinity or avidity of receptor binding.\n",
      "OBJECTIVE To analyze the evolution and variation of SARS-CoV-2 during the epidemic starting at the end of 2019. METHODS We downloaded the full-length genome sequence of SARS-CoV-2 from the databases of GISAID and NCBI. Using the software for bioinformatics including MEGA-X, BEAST, and TempEst, we constructed the genomic evolution tree, inferred the time evolution signal of the virus, calculated the tMRCA time of the virus and analyzed the selection pressure of the virus during evolution. RESULTS The phylogenetic tree showed that SARS-CoV-2 belonged to the Sarbecovirus subgenus of β Coronavirus genus together with bat coronavirus BetaCoV/bat/Yunnan/RaTG13/2013, bat-SL-CoVZC45, bat-SL-CoVZXC21 and SARS-CoV. The genomic sequences of SARS-CoV-2 isolated from the ongoing epidemic showed a weak time evolution signal with an average tMRCA time of 73 days (95% CI: 38.9-119.3 days). No positive time evolution signal was found between SARS-CoV-2 and BetaCoV/bat/Yunnan/RaTG13/2013, but the former virus had a strong positive temporal evolution relationship with bat-SL-CoVZC45 and SARS-CoV. The major cause for mutations of SARS-CoV-2 was the pressure of purification selection during the epidemic. CONCLUSIONS SARS-CoV-2 may have emerged as early as November, 2019, originating most likely from bat-associated coronavirus. This finding may provide evidence for tracing the sources and evolution of the virus.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: suhqgmlo\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "what is the origin of COVID-19\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "We found, in our 788 confirmed COVID-19 patients, the decreased rate of severe/critical type, increased liver/kidney damage and prolonged period of nuclear acid positivity during virus dissemination, when compared with Wuhan. To investigate the underlining mechanism, we isolated one strain of SARS- CoV-2 (ZJ01) in mild COVID-19 patient and found the existence of 35 specific gene mutation by gene alignment. Further phylogenetic analysis and RSCU heat map results suggested that ZJ01 may be a potential evolutionary branch of SARS-CoV-2. We classified 54 strains of viruses worldwide (C/T type) based on the base (C or T) at positions 8824 and 28247. ZJ01 were both T at these two sites, becoming the only TT type currently identified in the world. The prediction of Furin cleavage site (FCS) and the sequence alignment of virus family indicated that FCS may be an important site of coronavirus evolution. ZJ01 had mutations near FCS (F1-2), which caused changes in the structure and the electrostatic distribution of the S protein surface, further affecting the binding capacity of Furin. Single cell sequencing and ACE2-Furin co-expression results confirmed that Furin level was higher in the whole body, especially in glands, liver, kidney and colon while FCS may help SARS-CoV-2 infect these organs. The evolutionary pattern of SARS-CoV-2 towards FCS formation may result in its clinical symptom becoming closer to HKU-1 and OC43 (the source of FCS sequence-PRRA) caused influenza, further showing potential in differentiating into mild COVID-19 subtypes.\n",
      "The mutations in the SARS-CoV-2 virus genome during the spread of COVID-19 have been unclear. In 788 COVID-19 patients from Zhejiang province, we observed decreased rate of severe/critical cases, increased liver/kidney damage, and prolonged period of nuclear acid positivity, compared with patients in Wuhan, China. To investigate the underlying mechanisms, we isolated one strain of SARS-CoV-2 (ZJ01) from a mild COVID-19 patient. Thirty-five specific gene mutations were identified by gene alignment. Further phylogenetic analysis and relative synonymous codon usage heat map results suggested that ZJ01 may be a potential evolutionary branch of SARS-CoV-2. We classified 54 virus strains collected globally based on the base (C or T) at positions 8824 and 28247. ZJ01 has T at both sites and is currently the only known TT type. The prediction of the Furin cleavage site (FCS) and sequence alignment of the virus family indicated that the FCS may be an important site of coronavirus evolution. ZJ01 mutations identified near the FCS (F1-2) caused changes in the structure and electrostatic distribution of the S surface protein, further affecting the binding capacity of Furin. Single-cell sequencing and ACE2-Furin co-expression results confirmed that the Furin expression was higher in the whole body, especially in glands, liver, kidneys, and colon. Thus, FCS may help SARS-CoV-2 infect these organs. The evolutionary pattern of SARS-CoV-2 towards FCS formation may result in its clinical symptom becoming closer to HKU-1 and OC43 (the source of FCS sequence-PRRA) caused mild flu-like symptoms, further showing potential in differentiating into mild COVID-19 subtypes.\n",
      "The mutations in the SARS-CoV-2 virus genome during COVID-19 dissemination are unclear. In 788 COVID-19 patients from Zhejiang province, we observed decreased rate of severe/critical cases compared with patients in Wuhan. For mechanisms exploration, we isolated one strain of SARS-CoV-2 (ZJ01) from a mild COVID-19 patient. Thirty-five specific gene mutations were identified. Phylogenetic and relative synonymous codon usage analysis suggested that ZJ01 may be a potential evolutionary branch of SARS-CoV-2. We classified 54 global virus strains based on the base (C or T) at positions 8824 and 28247 while ZJ01 has T at both sites. The prediction of the Furin cleavage site (FCS) and sequence alignment indicated that the FCS may be an important site of coronavirus evolution. ZJ01 mutations identified near the FCS (F1-2) caused changes in the structure and electrostatic distribution of the S surface protein, further affecting the binding capacity of Furin. Single-cell sequencing and ACE2-Furin co-expression results confirmed that the Furin expression was especially higher in glands, liver, kidneys, and colon. The evolutionary pattern of SARS-CoV-2 towards FCS formation may result in its clinical symptom becoming closer to HKU-1 and OC43 caused mild flu-like symptoms, further showing its potential in differentiating into mild COVID-19 subtypes.\n",
      "The 2019 novel coronavirus (2019-nCoV or SARS-CoV-2) has spread more rapidly than any other betacoronavirus including SARS-CoV and MERS-CoV. However, the mechanisms responsible for infection and molecular evolution of this virus remained unclear. We collected and analyzed 120 genomic sequences of 2019-nCoV including 11 novel genomes from patients in China. Through comprehensive analysis of the available genome sequences of 2019-nCoV strains, we have tracked multiple inheritable SNPs and determined the evolution of 2019-nCoV relative to other coronaviruses. Systematic analysis of 120 genomic sequences of 2019-nCoV revealed co-circulation of two genetic subgroups with distinct SNPs markers, which can be used to trace the 2019-nCoV spreading pathways to different regions and countries. Although 2019-nCoV, human and bat SARS-CoV share high homologous in overall genome structures, they evolved into two distinct groups with different receptor entry specificities through potential recombination in the receptor binding regions. In addition, 2019-nCoV has a unique four amino acid insertion between S1 and S2 domains of the spike protein, which created a potential furin or TMPRSS2 cleavage site. Our studies provided comprehensive insights into the evolution and spread of the 2019-nCoV. Our results provided evidence suggesting that 2019-nCoV may increase its infectivity through the receptor binding domain recombination and a cleavage site insertion. Novel 2019-nCoV sequences revealed the evolution and specificity of betacoronavirus with possible mechanisms of enhanced infectivity.\n",
      "Severe COVID-19 patients can show respiratory failure, T-cell reduction, and cytokine release syndrome (CRS), which can be fatal in both young and aged patients and is a major concern of the pandemic. However, the pathogenetic mechanisms of CRS in COVID-19 are poorly understood. Here we show single cell-level mechanisms for T-cell dysregulation in severe SARS-CoV-2 infection, and thereby demonstrate the mechanisms underlying T-cell hyperactivation and paralysis in severe COVID-19 patients. By in silico sorting CD4+ T-cells from a single cell RNA-seq dataset, we found that CD4+ T-cells were highly activated and showed unique differentiation pathways in the lung of severe COVID-19 patients. Notably, those T-cells in severe COVID-19 patients highly expressed immunoregulatory receptors and CD25, whilst repressing the expression of the transcription factor FOXP3 and interestingly, both the differentiation of regulatory T-cells (Tregs) and Th17 was inhibited. Meanwhile, highly activated CD4+ T-cells express PD-1 alongside macrophages that express PD-1 ligands in severe patients, suggesting that PD-1-mediated immunoregulation was partially operating. Furthermore, we show that CD25+ hyperactivated T-cells differentiate into multiple helper T-cell lineages, showing multifaceted effector T-cells with Th1 and Th2 characteristics. Lastly, we show that CD4+ T-cells, particularly CD25-expressing hyperactivated T-cells, produce the protease Furin, which facilitates the viral entry of SARS-CoV-2. Collectively, CD4+ T-cells from severe COVID-19 patients are hyperactivated and FOXP3-mediated negative feedback mechanisms are impaired in the lung, while activated CD4+ T-cells continue to promote further viral infection through the production of Furin. Therefore, our study proposes a new model of T-cell hyperactivation and paralysis that drives pulmonary damage, systemic CRS and organ failure in severe COVID-19 patients.\n",
      "The newly identified SARS-CoV-2 has now been reported from around 185 countries with more than a million confirmed human cases including more than 120,000 deaths. The genomes of SARS-COV-2 strains isolated from different parts of the world are now available and the unique features of constituent genes and proteins need to be explored to understand the biology of the virus. Spike glycoprotein is one of the major targets to be explored because of its role during the entry of coronaviruses into host cells. We analyzed 320 whole-genome sequences and 320 spike protein sequences of SARS-CoV-2 using multiple sequence alignment. In this study, 483 unique variations have been identified among the genomes of SARS-CoV-2 including 25 nonsynonymous mutations and one deletion in the spike (S) protein. Among the 26 variations detected in S, 12 variations were located at the N-terminal domain (NTD) and 6 variations at the receptor-binding domain (RBD) which might alter the interaction of S protein with the host receptor angiotensin-converting enzyme 2 (ACE2). Besides, 22 amino acid insertions were identified in the spike protein of SARS-CoV-2 in comparison with that of SARS-CoV. Phylogenetic analyses of spike protein revealed that Bat coronavirus have a close evolutionary relationship with circulating SARS-CoV-2. The genetic variation analysis data presented in this study can help a better understanding of SARS-CoV-2 pathogenesis. Based on results reported herein, potential inhibitors against S protein can be designed by considering these variations and their impact on protein structure.\n",
      "Severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) is a member of the betacoronavirus family, which causes COVID-19 disease. SARS-CoV-2 pathogenicity in humans leads to increased mortality rates due to alterations of significant pathways, including some resulting in exacerbated inflammatory responses linked to the \"cytokine storm\" and extensive lung pathology, as well as being linked to a number of comorbidities. Our current study compared five SARS-CoV-2 sequences from different geographical regions to those from SARS, MERS and two cold viruses, OC43 and 229E, to identify the presence of miR-like sequences. We identified seven key miRs, which highlight considerable differences between the SARS-CoV-2 sequences, compared with the other viruses. The level of conservation between the five SARS-CoV-2 sequences was identical but poor compared with the other sequences, with SARS showing the highest degree of conservation. This decrease in similarity could result in reduced levels of transcriptional control, as well as a change in the physiological effect of the virus and associated host-pathogen responses. MERS and the milder symptom viruses showed greater differences and even significant sequence gaps. This divergence away from the SARS-CoV-2 sequences broadly mirrors the phylogenetic relationships obtained from the whole-genome alignments. Therefore, patterns of mutation, occurring during sequence divergence from the longer established human viruses to the more recent ones, may have led to the emergence of sequence motifs that can be related directly to the pathogenicity of SARS-CoV-2. Importantly, we identified 7 key-microRNAs (miRs 8066, 5197, 3611, 3934-3p, 1307-3p, 3691-3p, 1468-5p) with significant links to KEGG pathways linked to viral pathogenicity and host responses. According to Bioproject data (PRJNA615032), SARS-CoV-2 mediated transcriptomic alterations were similar to the target pathways of the selected 7 miRs identified in our study. This mechanism could have considerable significance in determining the symptom spectrum of future potential pandemics. KEGG pathway analysis revealed a number of critical pathways linked to the seven identified miRs that may provide insight into the interplay between the virus and comorbidities. Based on our reported findings, miRNAs may constitute potential and effective therapeutic approaches in COVID-19 and its pathological consequences.\n",
      "Severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) is a member of the betacoronavirus family, which causes COVID-19 disease. SARS-CoV-2 pathogenicity in humans leads to increased mortality rates due to alterations of significant pathways, including some resulting in exacerbated inflammatory responses linked to the \"cytokine storm\" and extensive lung pathology, as well as being linked to a number of comorbidities. Our current study compared five SARS-CoV-2 sequences from different geographical regions to those from SARS, MERS and two cold viruses, OC43 and 229E, to identify the presence of miR-like sequences. We identified seven key miRs, which highlight considerable differences between the SARS-CoV-2 sequences, compared with the other viruses. The level of conservation between the five SARS-CoV-2 sequences was identical but poor compared with the other sequences, with SARS showing the highest degree of conservation. This decrease in similarity could result in reduced levels of transcriptional control, as well as a change in the physiological effect of the virus and associated host-pathogen responses. MERS and the milder symptom viruses showed greater differences and even significant sequence gaps. This divergence away from the SARS-CoV-2 sequences broadly mirrors the phylogenetic relationships obtained from the whole-genome alignments. Therefore, patterns of mutation, occurring during sequence divergence from the longer established human viruses to the more recent ones, may have led to the emergence of sequence motifs that can be related directly to the pathogenicity of SARS-CoV-2. Importantly, we identified 7 key-microRNAs (miRs 8066, 5197, 3611, 3934-3p, 1307-3p, 3691-3p, 1468-5p) with significant links to KEGG pathways linked to viral pathogenicity and host responses. According to Bioproject data (PRJNA615032), SARS-CoV-2 mediated transcriptomic alterations were similar to the target pathways of the selected 7 miRs identified in our study. This mechanism could have considerable significance in determining the symptom spectrum of future potential pandemics. KEGG pathway analysis revealed a number of critical pathways linked to the seven identified miRs that may provide insight into the interplay between the virus and comorbidities. Based on our reported findings, miRNAs may constitute potential and effective therapeutic approaches in COVID-19 and its pathological consequences.\n",
      "The novel coronavirus SARS-CoV-2 (2019-nCoV) is a member of the family coronaviridae and contains a single-stranded RNA genome with positive-polarity. To reveal the evolution mechanism of SARS-CoV-2 genome, we performed comprehensive genomic analysis with newly sequenced SARS-CoV-2 strains and 20 closely related coronavirus strains. Among 98 nucleotide mutations at 93 sites of the genome among different SARS-CoV-2 strains, 58 of them caused amino acid change, indicating a result of neutral evolution. However, the ratio of nucleotide substitutions to amino acid substitutions of spike gene (9.07) between SARS-CoV-2 WIV04 and Bat-SARSr-CoV RaTG13 was extensively higher than those from comparisons between other coronaviruses (range 1.29 - 4.81). The elevated synonymous mutations between SARS-CoV-2 and RaTG13, suggesting they underwent stronger purifying selection. Moreover, their nucleotide substitutions are enriched with T:C transition, which is consistent with the mutation signature caused by deactivity of RNA 3’-to-5’ exoribonuclease (ExoN). The codon usage was similar between SARS-CoV-2 and other strains in beta-coronavirus lineage B, suggesting it had small impact on the mutation pattern. In comparison of SARS-CoV-2 WIV04 with Bat-SARSr-CoV RaTG13, the ratios of non-synonymous to synonymous substitution rates (dN/dS) was the lowest among all performed comparisons, reconfirming the evolution of SARS-CoV-2 under stringent selective pressure. Moreover, some sites of spike protein might be subjected to positive selection. Therefore, our results will help understanding the evolutionary mechanisms contribute to viral pathogenicity and its adaptation with hosts.\n",
      "The newly identified SARS-CoV-2 has now been reported from around 185 countries with more than a million confirmed human cases including more than 120,000 deaths. The genomes of SARS-COV-2 strains isolated from different parts of the world are now available and the unique features of constituent genes and proteins need to be explored to understand the biology of the virus. Spike glycoprotein is one of the major targets to be explored because of its role during the entry of coronaviruses into host cells. We analyzed 320 whole-genome sequences and 320 spike protein sequences of SARS-CoV-2 using multiple sequence alignment. In this study, 483 unique variations have been identified among the genomes of SARS-CoV-2 including 25 nonsynonymous mutations and one deletion in the spike (S) protein. Among the 26 variations detected, 12 variations were located at the N-terminal domain and 6 variations at the receptor-binding domain (RBD) which might alter the interaction of S protein with the host receptor angiotensin converting enzyme-2 (ACE2). Besides, 22 amino acid insertions were identified in the spike protein of SARS-CoV-2 in comparison with that of SARS-CoV. Phylogenetic analyses of spike protein revealed that Bat coronavirus have a close evolutionary relationship with circulating SARS-CoV-2. The genetic variation analysis data presented in this study can help a better understanding of SARS-CoV-2 pathogenesis. Based on results reported herein, potential inhibitors against S protein can be designed by considering these variations and their impact on protein structure.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 9ryu9ady\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "INTRODUCTION: The COVID-19 pandemic has been particularly challenging due to a lack of established therapies and treatment guidelines. With the rapid transmission of disease, even the off-label use of available therapies has been impeded by limited availability. Several antivirals, antimalarials, and biologics are being considered for treatment at this time. The purpose of this literature review is to synthesize the available information regarding treatment options for COVID-19 and serve as a resource for health care professionals. OBJECTIVES: This narrative review was conducted to summarize the effectiveness of current therapy options for COVID-19 and address the controversial use of non-steroidal anti-inflammatory drugs (NSAIDs), angiotensin converting enzyme (ACE) inhibitors, and angiotensin receptor blockers (ARBs). PubMed and SCOPUS were queried using a combination of the keywords “COVID 19,” “SARS-CoV-2,” and “treatment.” All types of studies were evaluated including systematic reviews, case-studies, and clinical guidelines. DISCUSSION: There are currently no therapeutic drugs available that are directly active against SARS-CoV-2; however, several antivirals (remdesivir, favipiravir) and antimalarials (chloroquine, hydroxychloroquine) have emerged as potential therapies. Current guidelines recommend combination treatment with hydroxychloroquine/azithromycin or chloroquine, if hydroxychloroquine is unavailable, in patients with moderate disease, although these recommendations are based on limited evidence. Remdesivir and convalescent plasma may be considered in critical patients with respiratory failure; however, access to these therapies may be limited. Interleukin-6 (IL-6) antagonists may be used in patients who develop evidence of cytokine release syndrome (CRS). Glucocorticoids should be avoided unless there is evidence of refractory septic shock, acute respiratory distress syndrome (ARDS), or another compelling indication for their use. ACE inhibitors and ARBs should not be discontinued at this time and ibuprofen may be used for fever. CONCLUSION: There are several ongoing clinical trials that are testing the efficacy of single and combination treatments with the drugs mentioned in this review and new agents are under development. Until the results of these trials become available, we must use the best available evidence for the prevention and treatment of COVID-19. Additionally, we can learn from the experiences of healthcare providers around the world to combat this pandemic.\n",
      "INTRODUCTION: The COVID-19 pandemic has been particularly challenging due to a lack of established therapies and treatment guidelines. With the rapid transmission of disease, even the off-label use of available therapies has been impeded by limited availability. Several antivirals, antimalarials, and biologics are being considered for treatment at this time. The purpose of this literature review is to synthesize the available information regarding treatment options for COVID-19 and serve as a resource for health care professionals. OBJECTIVES: This narrative review was conducted to summarize the effectiveness of current therapy options for COVID-19 and address the controversial use of non-steroidal anti-inflammatory drugs (NSAIDs), angiotensin converting enzyme (ACE) inhibitors, and angiotensin receptor blockers (ARBs). PubMed and SCOPUS were queried using a combination of the keywords \"COVID 19,\" \"SARS-CoV-2,\" and \"treatment.\" All types of studies were evaluated including systematic reviews, case-studies, and clinical guidelines. DISCUSSION: There are currently no therapeutic drugs available that are directly active against SARS-CoV-2; however, several antivirals (remdesivir, favipiravir) and antimalarials (chloroquine, hydroxychloroquine) have emerged as potential therapies. Current guidelines recommend combination treatment with hydroxychloroquine/azithromycin or chloroquine, if hydroxychloroquine is unavailable, in patients with moderate disease, although these recommendations are based on limited evidence. Remdesivir and convalescent plasma may be considered in critical patients with respiratory failure; however, access to these therapies may be limited. Interleukin-6 (IL-6) antagonists may be used in patients who develop evidence of cytokine release syndrome (CRS). Corticosteroids should be avoided unless there is evidence of refractory septic shock, acute respiratory distress syndrome (ARDS), or another compelling indication for their use. ACE inhibitors and ARBs should not be discontinued at this time and ibuprofen may be used for fever. CONCLUSION: There are several ongoing clinical trials that are testing the efficacy of single and combination treatments with the drugs mentioned in this review and new agents are under development. Until the results of these trials become available, we must use the best available evidence for the prevention and treatment of COVID-19. Additionally, we can learn from the experiences of healthcare providers around the world to combat this pandemic.\n",
      "BACKGROUND AND AIMS: Multiple issues in management of COVID have emerged, but confusion persists regarding rational interpretation. Aim of this brief review is to review these issues based on current literature. METHODS: This is a narrative review with Pubmed and Google Scholar search till 23 March 2020. Search terms were, COVID-19, treatment of coronavirus, COVID 19 and following terms; chloroquine, hydroxychloroquine, ibuprofen, ACE-inhibitors or angiotensin receptor blockers, cardiovascular disease, diarrhoea, liver, testis and gastrointestinal disease. RESULTS: We discuss evidence regarding role of chloroquine and hydroxychloroquine in treatment and prophylaxis, use of inhibitors of the renin angiotensin system, safety of ibuprofen, unusual clinical features like gastrointestinal symptoms and interpretation of tests for cardiac enzymes and biomarkers. CONCLUSIONS: While our conclusions on management of COVID-19 patients with co-morbidities are based on current evidence, however, data is limited and there is immediate need for fast track research.\n",
      "OBJECTIVES/BACKGROUND AND OBJECTIVES: Prior epidemics of high-mortality human coronaviruses, such as the acute respiratory syndrome coronavirus (SARS-CoV or SARS-1) in 2003, have driven the characterization of compounds that could be possibly active against the currently emerging novel coronavirus SARS-CoV-2 (COVID-19). Presently, no approved treatment or prophylaxis is available for COVID-19. We comment on the existing COVID-19 research methodologies in general and the published reporting. Given the media attention and claims of effectiveness, we chose chloroquine and hydroxychloroquine, in combination with azithromycin, as an area of COVID-19 research to examine. METHODS/STUDY DESIGN AND SETTING: MEDLINE and EMBASE electronic databases were searched from 2019 to present (April 3rd, 2020) using a mix of keywords such as COVID-19 and chloroquine and hydroxychloroquine. We also searched the largest clinical medicine preprint repository, medRxiv.org. RESULTS: We found 6 studies, 3 randomized control trials and 3 observational studies, focusing on chloroquine and hydroxychloroquine (with azithromycin). We critically appraised the evidence. CONCLUSION: We found that the COVID-19 research methodology is very poor in the area of chloroquine/hydroxychloroquine research. In screening the literature, we observed the same across COVID-19 research in relation to potential treatments. The reporting is very poor and sparse, and patient-important outcomes needed to discern decision-making priorities are not reported. We do understand the barriers to perform rigorous research in health care settings overwhelmed by a novel deadly disease. However, this emergency pandemic situation does not transform flawed methods and data into credible results. The adequately powered, comparative, and robust clinical research that is needed for optimal evidence-informed decision-making remains absent in COVID-19.\n",
      "BACKGROUND: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), termed coronavirus disease 2019 (COVID-19) by the World Health Organization, is a newly emerging zoonotic agent that emerged in China in December 2019. No specific treatment for COVID-19 is currently available. Usual palliative treatment includes maintaining hydration and nutrition and controlling fever and cough. The clinical severity and extent of transmission need to be determined, and therapeutic options need to be developed and optimized. METHODS: The present review discusses the recent repurposing of drugs for COVID-19 treatment. RESULTS: Several compounds, including remdesivir, lopinavir, ritonavir, interferon-ß, ribavirin, chloroquine/hydroxychloroquine, azithromycin, tocilizumab, and ivermectin, have emerged as promising alternatives. They block the virus from entering host cells, prevent viral replication, and attenuate exacerbation of the host's immune response. CONCLUSION: Although some evidence indicates the positive actions of different classes of compounds for the treatment of COVID-19, few clinical assays have been established to definitively demonstrate their therapeutic value in humans. Multicenter clinical studies are urgently needed to validate and standardize therapeutic regimens that involve these agents. Although science has not yet presented us with a specific drug against COVID-19, the repurposing of drugs appears to be promising in our fight against this devastating disease.\n",
      "Abstract The novel severe acute respiratory syndrome coronavirus 2 is causing a worldwide pandemic that may lead to a highly morbid and potentially fatal coronavirus disease-19 (COVID-19). There is currently no drug that has been proven as an effective therapy for COVID-19. Several candidate drugs are being considered and evaluated for treatment. This includes clinically-available drugs, such as chloroquine, hydroxychloroquine, and lopinavir/ritonavir, which are being repurposed for the treatment of COVID-19. Novel experimental therapies, such as remdesivir and favipiravir, are also actively being investigated for antiviral efficacy. Clinically-available and investigational immunomodulators, such as the IL-6 inhibitors tocilizumab and sarilumab and the anti-GMCSF lenzilumab, are being tested for their anticipated effect in counteracting the pro-inflammatory cytokine environment that characterizes severe and critical COVID-19. This review article examines the evidence behind the potential use of these leading drug candidates for the treatment of COVID-19. The authors conclude, based on this review, that there is still no high-quality evidence to support any of these proposed drug therapies. The authors, therefore, encourage the enrollment of eligible patients to multiple ongoing clinical trials that assess the efficacy and safety of these candidate therapies. Until the results of controlled trials are available, none of the suggested therapeutics is clinically proven as an effective therapy for COVID-19.\n",
      "Importance The pandemic of coronavirus disease 2019 (COVID-19) caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) presents an unprecedented challenge to identify effective drugs for prevention and treatment. Given the rapid pace of scientific discovery and clinical data generated by the large number of people rapidly infected by SARS-CoV-2, clinicians need accurate evidence regarding effective medical treatments for this infection. Observations No proven effective therapies for this virus currently exist. The rapidly expanding knowledge regarding SARS-CoV-2 virology provides a significant number of potential drug targets. The most promising therapy is remdesivir. Remdesivir has potent in vitro activity against SARS-CoV-2, but it is not US Food and Drug Administration approved and currently is being tested in ongoing randomized trials. Oseltamivir has not been shown to have efficacy, and corticosteroids are currently not recommended. Current clinical evidence does not support stopping angiotensin-converting enzyme inhibitors or angiotensin receptor blockers in patients with COVID-19. Conclusions and Relevance The COVID-19 pandemic represents the greatest global public health crisis of this generation and, potentially, since the pandemic influenza outbreak of 1918. The speed and volume of clinical trials launched to investigate potential therapies for COVID-19 highlight both the need and capability to produce high-quality evidence even in the middle of a pandemic. No therapies have been shown effective to date.\n",
      "BACKGROUND AND OBJECTIVE: The world is currently experiencing the Coronavirus Disease-19 (COVID-19) pandemic. There is no approved drug for the definitive treatment of the disease. Various drugs are being tried for the treatment of COVID-19, including hydroxychloroquine (HCQ). This study was performed to systematically review the therapeutic role of HCQ in COVID-19 from the available literature. METHODS: PubMed, Embase, ClinicalTrials.gov, ICTRP (WHO), Cochrane Library databases, and two pre-print servers (medRxiv.org and Research Square) were searched for clinical studies that evaluated the therapeutic role of HCQ on COVID-19 until 10 May 2020. The available studies were critically analyzed and the data were extracted. RESULTS: A total of 663 articles were screened and 12 clinical studies (seven peer-reviewed and published studies and five non-peer-reviewed studies from pre-print servers) with a total sample size of 3543 patients were included. Some of the clinical studies demonstrated good virological and clinical outcomes with HCQ alone or in combination with azithromycin in COVID-19 patients, although the studies had major methodological limitations. Some of the other studies showed negative results with HCQ therapy along with the risk of adverse reactions. CONCLUSION: The results of efficacy and safety of HCQ in COVID-19, as obtained from the clinical studies, are not satisfactory, although many of these studies had major methodological limitations. Stronger evidence from well-designed robust randomized clinical trials is required before conclusively determining the role of HCQ in the treatment of COVID-19. Clinical prudence is required in advocating HCQ as a therapeutic armamentarium in COVID-19.\n",
      "Abstract Background and aims Multiple issues in management of COVID have emerged, but confusion persists regarding rational interpretation. Aim of this brief review is to review these issues based on current literature. Methods This is a narrative review with Pubmed and Google Scholar search till 23 March 2020. Search terms were, COVID-19, treatment of coronavirus, COVID 19 and following terms; chloroquine, hydroxychloroquine, ibuprofen, ACE-inhibitors or angiotensin receptor blockers, cardiovascular disease, diarrhoea, liver, testis and gastrointestinal disease. Results We discuss evidence regarding role of chloroquine and hydroxychloroquine in treatment and prophylaxis, use of inhibitors of the renin angiotensin system, safety of ibuprofen, unusual clinical features like gastrointestinal symptoms and interpretation of tests for cardiac enzymes and biomarkers. Conclusions While our conclusions on management of COVID-19 patients with co-morbidities are based on current evidence, however, data is limited and there is immediate need for fast track research.\n",
      "PURPOSE OF REVIEW: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the virus responsible for the aggressive coronavirus disease (COVID-19) pandemic. Recently, investigators have stipulated that COVID-19 patients receiving angiotensin-converting-enzyme inhibitors (ACEI) may be subject to poorer outcomes. This editorial presents the available evidence to guide treatment practices during this pandemic. RECENT FINDINGS: Recent studies from Wuhan cohorts provide valuable information about COVID-19. A cohort with 52 critically ill patients revealed cardiac injury in 12% of patients. Worse outcomes appear to be more prevalent in patients with hypertension and diabetes mellitus (DM), possibly due to overexpression of angiotensin-converting enzyme 2 (ACE2) receptor in airway alveolar epithelial cells. Investigators suspect that SARS-CoV-2 uses the ACE2 receptor to enter the lungs in a mechanism similar to SARS-CoV. Several hypotheses have been proposed to date regarding the net effect of ACEI/ARB on COVID-19 infections. Positive effects include ACE2 receptor blockade, disabling viral entry into the heart and lungs, and an overall decrease in inflammation secondary to ACEI/ARB. Negative effects include a possible retrograde feedback mechanism, by which ACE2 receptors are upregulated. Even though physiological models of SARS-CoV infection show a theoretical benefit of ACEI/ARB, these findings cannot be extrapolated to SARS-CoV-2 causing COVID-19. Major cardiology scientific associations, including ACC, HFSA, AHA, and ESC Hypertension Council, have rejected these correlation hypotheses. After an extensive literature review, we conclude that there is no significant evidence to support an association for now, but given the rapid evolvement of this pandemic, findings may change.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: b518n9dx\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "are there any clinical trials available for the coronavirus\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Abstract The novel coronavirus infection that initially found at the end of 2019 has attracted great attention. So far, the number of infectious cases has increased globally to more than 100 thousand and the outbreak has been defined as a pandemic situation, but there are still no “specific drug” available. Relevant reports have pointed out the novel coronavirus has 80% homology with SARS. In the difficulty where new synthesized drug cannot be applied immediately to patients, “conventional drug in new use” becomes a feasible solution. The first medication experience of the recovered patients in the US has led remdesivir to be the “specific drug”. China has also taken immediate action to put remdesivir into clinical trials with the purpose of applying it into clinical therapeutics for Corona Virus Disease 2019 (COVID-19). We started from the structure, immunogenicity, and pathogenesis of coronavirus infections of the novel coronavirus. Further, we analyzed the pharmacological actions and previous trials of remdesivir to identify the feasibility of conducting experiments on COVID-19.\n",
      "The novel coronavirus infection that initially found at the end of 2019 has attracted great attention. So far, the number of infectious cases has increased globally to more than 100 thousand and the outbreak has been defined as a pandemic situation, but there are still no \"specific drug\" available. Relevant reports have pointed out the novel coronavirus has 80% homology with SARS. In the difficulty where new synthesized drug cannot be applied immediately to patients, \"conventional drug in new use\" becomes a feasible solution. The first medication experience of the recovered patients in the US has led remdesivir to be the \"specific drug\". China has also taken immediate action to put remdesivir into clinical trials with the purpose of applying it into clinical therapeutics for Corona Virus Disease 2019 (COVID-19). We started from the structure, immunogenicity, and pathogenesis of coronavirus infections of the novel coronavirus. Further, we analyzed the pharmacological actions and previous trials of remdesivir to identify the feasibility of conducting experiments on COVID-19.\n",
      "Since December 2019, numerous cases of coronavirus disease 2019 (COVID-19) caused by the infection of the novel coronavirus (2019-nCoV) have been confirmed in Wuhan, China. The outbreak of 2019-nCoV in China embodied a significant and urgent threat to global health. 2019-nCoV was a new, highly contagious coronavirus discovered following the outbreak of SARS coronavirus (SARS-CoV) and MERS coronavirus (MERS-CoV). The novel coronavirus can cause severe respiratory disease and even death. However, no specific therapeutic drugs have been developed clinically thus far. This article examines the potential of therapeutic drugs by assessing the structure of 2019-nCoV, its mechanism in invading host cells, and the anti-viral mechanism of the human autoimmune system. We also review the latest research regarding the progress of potential therapeutic drugs and provide references for new drug developments of COVID-19.\n",
      "Since December 2019, numerous cases of coronavirus disease 2019 (COVID-19) caused by the infection of the novel coronavirus (2019-nCoV) have been confirmed in Wuhan, China. The outbreak of 2019-nCoV in China embodied a significant and urgent threat to global health. 2019-nCoV was a new, highly contagious coronavirus discovered following the outbreak of SARS coronavirus (SARS-CoV) and MERS coronavirus (MERS-CoV). The novel coronavirus can cause severe respiratory disease and even death. However, no specific therapeutic drugs have been developed clinically thus far. This article examines the potential of therapeutic drugs by assessing the structure of 2019-nCoV, its mechanism in invading host cells, and the anti-viral mechanism of the human autoimmune system. We also review the latest research regarding the progress of potential therapeutic drugs and provide references for new drug developments of COVID-19.\n",
      "Importance The pandemic of coronavirus disease 2019 (COVID-19) caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) presents an unprecedented challenge to identify effective drugs for prevention and treatment. Given the rapid pace of scientific discovery and clinical data generated by the large number of people rapidly infected by SARS-CoV-2, clinicians need accurate evidence regarding effective medical treatments for this infection. Observations No proven effective therapies for this virus currently exist. The rapidly expanding knowledge regarding SARS-CoV-2 virology provides a significant number of potential drug targets. The most promising therapy is remdesivir. Remdesivir has potent in vitro activity against SARS-CoV-2, but it is not US Food and Drug Administration approved and currently is being tested in ongoing randomized trials. Oseltamivir has not been shown to have efficacy, and corticosteroids are currently not recommended. Current clinical evidence does not support stopping angiotensin-converting enzyme inhibitors or angiotensin receptor blockers in patients with COVID-19. Conclusions and Relevance The COVID-19 pandemic represents the greatest global public health crisis of this generation and, potentially, since the pandemic influenza outbreak of 1918. The speed and volume of clinical trials launched to investigate potential therapies for COVID-19 highlight both the need and capability to produce high-quality evidence even in the middle of a pandemic. No therapies have been shown effective to date.\n",
      "Importance: The pandemic of coronavirus disease 2019 (COVID-19) caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) presents an unprecedented challenge to identify effective drugs for prevention and treatment. Given the rapid pace of scientific discovery and clinical data generated by the large number of people rapidly infected by SARS-CoV-2, clinicians need accurate evidence regarding effective medical treatments for this infection. Observations: No proven effective therapies for this virus currently exist. The rapidly expanding knowledge regarding SARS-CoV-2 virology provides a significant number of potential drug targets. The most promising therapy is remdesivir. Remdesivir has potent in vitro activity against SARS-CoV-2, but it is not US Food and Drug Administration approved and currently is being tested in ongoing randomized trials. Oseltamivir has not been shown to have efficacy, and corticosteroids are currently not recommended. Current clinical evidence does not support stopping angiotensin-converting enzyme inhibitors or angiotensin receptor blockers in patients with COVID-19. Conclusions and Relevance: The COVID-19 pandemic represents the greatest global public health crisis of this generation and, potentially, since the pandemic influenza outbreak of 1918. The speed and volume of clinical trials launched to investigate potential therapies for COVID-19 highlight both the need and capability to produce high-quality evidence even in the middle of a pandemic. No therapies have been shown effective to date.\n",
      "Abstract The COVID-19 pandemic, after it was reported in December 2019, is a highly contagious and now spreading to over 190 countries, causing a severe public health burden. Currently, there is no vaccine or specific drug to treat COVID-19, which is caused by a novel coronavirus, SARS-2-CoV. For this emergency, the FDA has approved Remdesivir and Hydroxychloroquine for treatment of COVID-19 as Emergency Use Authorization. However, even after this pandemic, COVID-19 may still have a chance to come back. Therefore, we need to come out with new strategies for drug discovery for combating COVID-19 in the future.\n",
      "Abstract At the end of the year 2019, the novel coronavirus (2019-nCoV) was spreading in Wuhan, China, and the outbreak process has a high speed. It was recognized as a pandemic by the World Health Organization (WHO) on 11 March 2020. Coronaviruses are enveloped and single-stranded RNA that have several families including Severe Acute Respiratory Syndrome (SARS) and Middle East Respiratory Syndrome (MERS). The pathogenesis mechanism and disease outcomes of SARS and MERS are now clear to some extent, but little information is available for 2019-nCoV. This newly identified corona virus infection represents flu-like symptoms, but usually the first symptoms are fever and dry cough. There has been no specific treatment against 2019-nCoV up to now, and physicians only apply supportive therapy. In the present article, we made an attempt to review the behavior of the virus around the world, epidemiology, a pathway for influx into the host cells, clinical presentation, as well as the treatments currently in use and future approaches; nitazoxanide may be our dream drug. We hope that this review has a positive impact on public knowledge for helping to deal with the 2019-nCoV and move one step forward toward its treatment in the near future.\n",
      "On December 31, 2019 a pneumonia outbreak caused by a new coronavirus (SARS-CoV-2) was detected in the city of Wuhan (China) Due to the high capacity of diffusion and human infection it has become a new zoonotic pandemic The absence of a vaccine has determined the search for antiviral drugs with the capacity to inhibit the replication of the new virus Among them, remdesivir, an analogue of adenosine, is what seems to have a more promising future This drug has shown in vitro and in animals a high capacity to block infection and viral replication with attainable concentrations in human plasma Although all studies have been carried out with SARS-CoV and MERS-CoV, it seems that by virological and functional analogy, remdesivir is one of the few antiviral drugs with proven efficacy However, studies and clinical trials in humans are required to know the result of their application in them\n",
      "The global pandemic of novel coronavirus disease 2019 (COVID‐19) caused by severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2) has created an urgent need for effective antivirals. Remdesivir (formerly GS‐5734) is a nucleoside analogue pro‐drug currently being evaluated in COVID‐19 clinical trials. Its unique structural features allow high concentrations of the active triphosphate metabolite to be delivered intracellularly and it evades proofreading to successfully inhibit viral RNA synthesis. In pre‐clinical models, remdesivir has demonstrated potent antiviral activity against diverse human and zoonotic β‐coronaviruses, including SARS‐CoV‐2. In this article we critically review available data on remdesivir with an emphasis on biochemistry, pharmacology, pharmacokinetics and in vitro activity against coronaviruses as well as clinical experience and current progress in COVID‐19 clinical trials.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: o3nfoqgs\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Since 2002, β coronaviruses (CoVs) have caused three zoonotic outbreaks, SARS-CoV in 2002, MERS-CoV in 2012, and the recent outbreak of SARS-CoV-2 late in 2019 (also named as COVID-19 or novel coronavirus 2019 or nCoV2019). Spike (S) protein, one of the structural proteins of this virus plays key role in receptor (ACE2) binding and thus virus entry. Thus, this protein has attracted scientists for detailed study and therapeutic targeting. As the nCoV2019 takes its course throughout the world, more and more sequence analyses are being done and genome sequences are being deposited in various databases. From India, two clinical isolates have been sequenced and the full genome has been deposited in GenBank. We have performed sequence analyses of the Spike protein of the Indian isolates and compared with that of the Wuhan, China (where the outbreak was first reported). While all the sequences of Wuhan isolates are identical, we found point mutations in the Indian isolates. Out of the two isolates, one was found to harbor a mutation in its receptor-binding domain (RBD) at position 407. At this site, arginine (a positively charged amino acid) was replaced by isoleucine (a hydrophobic amino acid that is also a C-β branched amino acid). This mutation has been seen to change the secondary structure of the protein at that region and this can potentially alter receptor binding of the virus. Although this finding needs further validation and more sequencing, the information might be useful in rational drug designing and vaccine engineering.\n",
      "Since 2002, beta coronaviruses (CoV) have caused three zoonotic outbreaks, SARS-CoV in 2002, MERS-CoV in 2012, and the recent outbreak of SARS-CoV-2 late in 2019 (also named as COVID-19 or novel coronavirus 2019 or nCoV2019. Spike(S) protein, one of the structural proteins of this virus plays key role in receptor (ACE2) binding and thus virus entry. Thus, this protein has attracted scientists for detailed study and therapeutic targeting. As the 2019 novel coronavirus takes its course throughout the world, more and more sequence analyses are been done and genome sequences getting deposited in various databases. From India two clinical isolates have been sequenced and the full genome deposited in GenBank. We have performed sequence analyses of the spike protein of the Indian isolates and compared with that of the Wuhan, China (where the outbreak was first reported). While all the sequences of Wuhan isolates are identical, we found point mutations in the Indian isolates. Out of the two isolates one was found to harbour a mutation in its Receptor binding domain (RBD) at position 407. At this site arginine (a positively charged amino acid) was replaced by isoleucine (a hydrophobic amino acid that is also a C-beta branched amino acid). This mutation has been seen to change the secondary structure of the protein at that region and this can potentially alter receptor ding of the virus. Although this finding needs further validation and more sequencing, the information might be useful in rational drug designing and vaccine engineering.\n",
      "Since 2002, ß coronaviruses (CoVs) have caused three zoonotic outbreaks, SARS-CoV in 2002, MERS-CoV in 2012, and the recent outbreak of SARS-CoV-2 late in 2019 (also named as COVID-19 or novel coronavirus 2019 or nCoV2019). Spike (S) protein, one of the structural proteins of this virus plays key role in receptor (ACE2) binding and thus virus entry. Thus, this protein has attracted scientists for detailed study and therapeutic targeting. As the nCoV2019 takes its course throughout the world, more and more sequence analyses are being done and genome sequences are being deposited in various databases. From India, two clinical isolates have been sequenced and the full genome has been deposited in GenBank. We have performed sequence analyses of the Spike protein of the Indian isolates and compared with that of the Wuhan, China (where the outbreak was first reported). While all the sequences of Wuhan isolates are identical, we found point mutations in the Indian isolates. Out of the two isolates, one was found to harbor a mutation in its receptor-binding domain (RBD) at position 407. At this site, arginine (a positively charged amino acid) was replaced by isoleucine (a hydrophobic amino acid that is also a C-ß branched amino acid). This mutation has been seen to change the secondary structure of the protein at that region and this can potentially alter receptor binding of the virus. Although this finding needs further validation and more sequencing, the information might be useful in rational drug designing and vaccine engineering.\n",
      "Since 2002, beta coronaviruses (CoV) have caused three zoonotic outbreaks, SARS-CoV in 2002-2003, MERS-CoV in 2012, and the newly emerged SARS-CoV-2 in late 2019. However, little is currently known about the biology of SARS-CoV-2. Here, using SARS-CoV-2 S protein pseudovirus system, we confirm that human angiotensin converting enzyme 2 (hACE2) is the receptor for SARS-CoV-2, find that SARS-CoV-2 enters 293/hACE2 cells mainly through endocytosis, that PIKfyve, TPC2, and cathepsin L are critical for entry, and that SARS-CoV-2 S protein is less stable than SARS-CoV S. Polyclonal anti-SARS S1 antibodies T62 inhibit entry of SARS-CoV S but not SARS-CoV-2 S pseudovirions. Further studies using recovered SARS and COVID-19 patients' sera show limited cross-neutralization, suggesting that recovery from one infection might not protect against the other. Our results present potential targets for development of drugs and vaccines for SARS-CoV-2.\n",
      "Since 2002, beta coronaviruses (CoV) have caused three zoonotic outbreaks, SARS-CoV in 2002–2003, MERS-CoV in 2012, and the newly emerged SARS-CoV-2 in late 2019. However, little is currently known about the biology of SARS-CoV-2. Here, using SARS-CoV-2 S protein pseudovirus system, we confirm that human angiotensin converting enzyme 2 (hACE2) is the receptor for SARS-CoV-2, find that SARS-CoV-2 enters 293/hACE2 cells mainly through endocytosis, that PIKfyve, TPC2, and cathepsin L are critical for entry, and that SARS-CoV-2 S protein is less stable than SARS-CoV S. Polyclonal anti-SARS S1 antibodies T62 inhibit entry of SARS-CoV S but not SARS-CoV-2 S pseudovirions. Further studies using recovered SARS and COVID-19 patients’ sera show limited cross-neutralization, suggesting that recovery from one infection might not protect against the other. Our results present potential targets for development of drugs and vaccines for SARS-CoV-2.\n",
      "The recent outbreak of a new coronavirus (SARS-CoV-2) in Wuhan, China, underscores the need for understanding the evolutionary processes that drive the emergence and adaptation of zoonotic viruses in humans. Here, we show that recombination in betacoronaviruses, including human-infecting viruses like SARS-CoV and MERS-CoV, frequently encompasses the Receptor Binding Domain (RBD) in the Spike gene. We find that this common process likely led to a recombination event at least 11 years ago in an ancestor of the SARS-CoV-2 involving the RBD. As a result of this recombination event, SARS-CoV and SARS-CoV-2 share a similar genotype in RBD, including two insertions (positions 432-436 and 460-472), and alleles 427N and 436Y. Both 427N and 436Y belong to a helix that interacts with the human ACE2 receptor. Ancestral state analyses revealed that SARS-CoV-2 differentiated from its most recent common ancestor with RaTG13 by accumulating a significant number of amino acid changes in the RBD. In sum, we propose a two-hit scenario in the emergence of the SARS-CoV-2 virus whereby the SARS-CoV-2 ancestors in bats first acquired genetic characteristics of SARS-CoV by incorporation of a SARS-like RBD through recombination before 2009, and subsequently, the lineage that led to SARS-CoV-2 accumulated further, unique changes specifically in the RBD.\n",
      "The 2019 novel coronavirus (2019-nCoV) is currently causing a widespread outbreak centered on Hubei province, China and is a major public health concern. Taxonomically 2019-nCoV is closely related to SARS-CoV and SARS-related bat coronaviruses, and it appears to share a common receptor with SARS-CoV (ACE-2). Here, we perform structural modeling of the 2019-nCoV spike glycoprotein. Our data provide support for the similar receptor utilization between 2019-nCoV and SARS-CoV, despite a relatively low amino acid similarity in the receptor binding module. Compared to SARS-CoV, we identify an extended structural loop containing basic amino acids at the interface of the receptor binding (S1) and fusion (S2) domains, which we predict to be proteolytically-sensitive. We suggest this loop confers fusion activation and entry properties more in line with MERS-CoV and other coronaviruses, and that the presence of this structural loop in 2019-nCoV may affect virus stability and transmission.\n",
      "The 2019 novel coronavirus (2019-nCoV) is currently causing a widespread outbreak centered on Hubei province, China and is a major public health concern. Taxonomically 2019-nCoV is closely related to SARS-CoV and SARS-related bat coronaviruses, and it appears to share a common receptor with SARS-CoV (ACE-2). Here, we perform structural modeling of the 2019-nCoV spike glycoprotein. Our data provide support for the similar receptor utilization between 2019-nCoV and SARS-CoV, despite a relatively low amino acid similarity in the receptor binding module. Compared to SARS-CoV, we identify an extended structural loop containing basic amino acids at the interface of the receptor binding (S1) and fusion (S2) domains, which we predict to be proteolytically-sensitive. We suggest this loop confers fusion activation and entry properties more in line with MERS-CoV and other coronaviruses, and that the presence of this structural loop in 2019-nCoV may affect virus stability and transmission.\n",
      "A novel and highly pathogenic coronavirus (2019-nCoV) has caused an outbreak in Wuhan city, Hubei province of China since December 2019, and soon spread nationwide and spilled over to other countries around the world. To better understand the initial step of infection at atomic-level, we determined the crystal structure of the 2019-nCoV spike receptor-binding domain (RBD) bound with the cell receptor ACE2 at 2.45 Å resolution. The overall ACE2-binding mode of the 2019-nCoV RBD is nearly identical to that of the SARS-CoV RBD, which also utilizes ACE2 as the cell receptor. Structural analysis identified residues in 2019-nCoV RBD critical for ACE2 binding, and majority of which are either highly conserved or shared similar side chain properties with those in the SARS-CoV RBD. Such similarity in structure and sequence strongly argue for a convergent evolution between 2019-nCoV and SARS-CoV RBD for improved binding to ACE2 despite of being segregated in different genetic lineages in the betacoronavirus genus. The epitopes of two SARS-CoV antibodies targeting the RBD are also analyzed with the 2019-nCoV RBD, providing insights into future identification of cross-reactive antibodies.\n",
      "The 2019 novel coronavirus (2019-nCoV/SARS-CoV-2) originally arose as part of a major outbreak of respiratory disease centered on Hubei province, China. It is now a global pandemic and is a major public health concern. Taxonomically, SARS-CoV-2 was shown to be a Betacoronavirus (lineage B) closely related to SARS-CoV and SARS-related bat coronaviruses, and it has been reported to share a common receptor with SARS-CoV (ACE-2). Subsequently, betacoronaviruses from pangolins were identified as close relatives to SARS-CoV-2. Here, we perform structural modeling of the SARS-CoV-2 spike glycoprotein. Our data provide support for the similar receptor utilization between SARS-CoV-2 and SARS-CoV, despite a relatively low amino acid similarity in the receptor binding module. Compared to SARS-CoV and all other coronaviruses in Betacoronavirus lineage B, we identify an extended structural loop containing basic amino acids at the interface of the receptor binding (S1) and fusion (S2) domains. We suggest this loop confers fusion activation and entry properties more in line with betacoronaviruses in lineages A and C, and be a key component in the evolution of SARS-CoV-2 with this structural loop affecting virus stability and transmission.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 0u4ar3b5\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "are there any clinical trials available for the coronavirus\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Purpose: The COVID-19 Pandemic has literally left the world breathless in the chase for Pharmacotherapy. With the vaccine approval likely more than a year away and novel drugs in early clinical trials, repurposing of existing drugs takes the center stage. A potential drug discussed both in geopolitical and global scientific community is hydroxychloroquine (HCQ). We intend to systematically weigh and analyze the existing evidence of HCQ in the light of published and pre-print data available so far. Methods: PubMed Ovid MEDLINE, EMBASE, Google scholar databases and official clinical trial Registries of the United States, China, WHO ICTRP were electronically searched for studies for the use of HCQ in patients with COVID-19. Pre-proof article repositories like MedRxiv, BioRxiv, and ChemRxiv were also included in the search. The literature was critically appraised. Results: Total 71 articles were available as of 15 th April of which articles of relevance (three invitro studies, two open label non-randomized trials, two open label randomized control trials, one follow-up study, three reviews, ten short communications) and 88 clinical trials registered in three clinical trial registries were analyzed. HCQ seems to be efficient in inhibiting of SARS CoV-2 in in-vitro cell lines; there is lack of strong evidence from human studies. Conclusions: The in-vitro cell culture based data of viral inhibition does not suffice for the use of hydroxychloroquine in the patients with COVID-19. Currently literature shows inadequate, low level evidence in human studies. Scarcity of safety and efficacy data warrants medical communities, health care agencies and governments across the world against the widespread use of HCQ in COVID-19 prophylaxis and treatment, until robust evidence becomes available. Keywords: Hydroxychloroquine, SARS-CoV-2, COVID-19, Corona virus, nCov2, systematic review\n",
      "OBJECTIVE: The pandemic coronavirus disease-19 (COVID-19) has pushed the global healthcare system to a crisis and amounted to a huge economic burden. Different drugs for prophylaxis against COVID-19 including chloroquine (CQ) or hydroxychloroquine (HCQ) have been tried. This study was performed to systematically review the role of CQ and HCQ in preventing the spread of COVID-19. METHODS: PubMed, EMBASE, ClinicalTrials.gov, International Clinical Trials Registry Platform and Cochrane Library databases were searched for studies that evaluated the prophylactic role of CQ or HCQ on SARS-CoV-2 (pre-clinical studies) or COVID-19 (clinical studies) until 30 March 2020. The available literature was critically appraised. RESULTS: A total of 45 articles were screened and 5 (3 in vitro pre-clinical studies and 2 clinical opinions) were included. The pre-clinical studies showed the prophylactic effects of CQ and HCQ against SARS-CoV-2. On the other hand, the clinical opinions advocated the prophylactic use of CQ and HCQ against COVID-19. However, no original clinical studies on the prophylactic role of CQ or HCQ on COVID-19 were available. CONCLUSION: Although pre-clinical results are promising, to date there is a dearth of evidence to support the efficacy of CQ or HCQ in preventing COVID-19. Considering potential safety issues and the likelihood of imparting a false sense of security, prophylaxis with CQ or HCQ against COVID-19 needs to be thoroughly evaluated in observational studies or high-quality randomized controlled studies.\n",
      "OBJECTIVE: The pandemic coronavirus disease‐19 (COVID‐19) has pushed the global healthcare system to a crisis and amounted to a huge economic burden. Different drugs for prophylaxis against COVID‐19 including chloroquine (CQ) or hydroxychloroquine (HCQ) have been tried. This study was performed to systematically review the role of CQ and HCQ in preventing the spread of COVID‐19. METHODS: PubMed, EMBASE, ClinicalTrials.gov, International Clinical Trials Registry Platform and Cochrane Library databases were searched for studies that evaluated the prophylactic role of CQ or HCQ on SARS‐CoV‐2 (pre‐clinical studies) or COVID‐19 (clinical studies) until 30 March 2020. The available literature was critically appraised. RESULTS: A total of 45 articles were screened and 5 (3 in vitro pre‐clinical studies and 2 clinical opinions) were included. The pre‐clinical studies showed the prophylactic effects of CQ and HCQ against SARS‐CoV‐2. On the other hand, the clinical opinions advocated the prophylactic use of CQ and HCQ against COVID‐19. However, no original clinical studies on the prophylactic role of CQ or HCQ on COVID‐19 were available. CONCLUSION: Although pre‐clinical results are promising, to date there is a dearth of evidence to support the efficacy of CQ or HCQ in preventing COVID‐19. Considering potential safety issues and the likelihood of imparting a false sense of security, prophylaxis with CQ or HCQ against COVID‐19 needs to be thoroughly evaluated in observational studies or high‐quality randomized controlled studies.\n",
      "Purpose: This study aims to critically assess the published studies of Chloroquine (CQ) and hydroxychloroquine (HCQ) for the treatment of COVID-19 and provide recommendations for future clinical trials for the COVID-19 pandemic. Method: A rapid systematic review was conducted by searching the PubMed, Embase, and China National Knowledge Infrastructure databases on April 13, 2020. Three clinical trial registry platforms, including ClinicalTrials.gov, the EU Clinical Trials Register, and the Chinese Clinical Trial Register were also complementarily searched. Results: A total of 10 clinical studies were identified, including 3 randomized controlled trials (RCTs), 1 comparative nonrandomized trial, 5 single-arm trials, and 1 interim analysis. The heterogeneity among studies of the baseline disease severity and reported endpoints made a pooled analysis impossible. CQ and HCQ (with or without azithromycin) showed significant therapeutic benefit in terms of virologic clearance rate, improvement in symptoms and imaging findings, time to clinical recovery, and length of hospital stay in 1 RCT, 4 single-arm trials, and the interim analysis, whereas no treatment benefit of CQ or HCQ was observed in the remaining 4 studies. Limitations of the included studies ranged from small sample size, to insufficient information concerning baseline patient characteristics, to potential for selection bias without detailing the rationale for exclusion, and presence of confounding factors. Conclusion: Based on the studies evaluated, there still lacked solid evidence supporting the efficacy and safety of HCQ and CQ as a treatment for COVID-19 with or without azithromycin. This emphasized the importance of robust RCTs investing HCQ/CQ to address the evidence uncertainties. Keywords: COVID-19, Systematic review, Clinical trial, Chloroquine, Hydroxychloroquine\n",
      "OBJECTIVE: The novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), also called COVID-19, has caused a pandemic which has swiftly involved the entire world and raised great public health concerns. The scientific community is actively exploring treatments that would potentially be effective in combating COVID-19. Hydroxychloroquine has been demonstrated to limit the replication of SARS-CoV-2 virus in vitro. In malarial pandemic countries, chloroquine is widely used to treat malaria. In malarial non-pandemic nations, chloroquine is not widely used. Chloroquine and hydroxychloroquine share similar chemical structures and mechanisms of action. The aim of this study was to indirectly investigate the efficacy of chloroquine and hydroxychloroquine for the treatment of COVID-19 by determining the prevalence of COVID-19 in malaria pandemic and non-pandemic nations. We sought evidence to support or refute the hypothesis that these drugs could show efficacy in the treatment of COVID-19. MATERIALS AND METHODS: We reviewed in vitro studies, in vivo studies, original studies, clinical trials, and consensus reports, that were conducted to evaluate the antiviral activities of chloroquine and hydroxychloroquine. The studies on \"COVID-19 and its allied treatment were found from World Health Organization (WHO), ISI-Web of Science, PubMed, EMBASE, Scopus, Google Scholar, and clinical trial registries. The search was based on keywords: antiviral drugs, chloroquine, hydroxychloroquine, COVID-19, COVID-19 treatment modalities, and coronavirus. In addition, we analyzed the prevalence of COVID-19 in malaria pandemic and non-pandemic countries. The review and analyses were performed on March 28, 2020. RESULTS: For this study, we identified a total of 09 published articles: 03 clinical trials with sample size 150; 03 in vitro studies and 03 expert consensus reports. These studies were all suggestive that chloroquine and hydroxychloroquine can successfully treat COVID-19 infections. We found that COVID-19 infections are highly pandemic in countries where malaria is least pandemic and are least pandemic in nations where malaria is highly pandemic. CONCLUSIONS: Chloroquine and hydroxychloroquine have antiviral characteristics in vitro. The findings support the hypothesis that these drugs have efficacy in the treatment of COVID-19. People are currently using these drugs for malaria. It is reasonable, given the hypothetical benefit of these two drugs, that they are now being tested in clinical trials to assess their effectiveness to combat this global health crisis.\n",
      "OBJECTIVE The novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), also called COVID-19, has caused a pandemic which has swiftly involved the entire world and raised great public health concerns. The scientific community is actively exploring treatments that would potentially be effective in combating COVID-19. Hydroxychloroquine has been demonstrated to limit the replication of SARS-CoV-2 virus in vitro. In malarial pandemic countries, chloroquine is widely used to treat malaria. In malarial non-pandemic nations, chloroquine is not widely used. Chloroquine and hydroxychloroquine share similar chemical structures and mechanisms of action. The aim of this study was to indirectly investigate the efficacy of chloroquine and hydroxychloroquine for the treatment of COVID-19 by determining the prevalence of COVID-19 in malaria pandemic and non-pandemic nations. We sought evidence to support or refute the hypothesis that these drugs could show efficacy in the treatment of COVID-19. MATERIALS AND METHODS We reviewed in vitro studies, in vivo studies, original studies, clinical trials, and consensus reports, that were conducted to evaluate the antiviral activities of chloroquine and hydroxychloroquine. The studies on \"COVID-19 and its allied treatment were found from World Health Organization (WHO), ISI-Web of Science, PubMed, EMBASE, Scopus, Google Scholar, and clinical trial registries. The search was based on keywords: antiviral drugs, chloroquine, hydroxychloroquine, COVID-19, COVID-19 treatment modalities, and coronavirus. In addition, we analyzed the prevalence of COVID-19 in malaria pandemic and non-pandemic countries. The review and analyses were performed on March 28, 2020. RESULTS For this study, we identified a total of 09 published articles: 03 clinical trials with sample size 150; 03 in vitro studies and 03 expert consensus reports. These studies were all suggestive that chloroquine and hydroxychloroquine can successfully treat COVID-19 infections. We found that COVID-19 infections are highly pandemic in countries where malaria is least pandemic and are least pandemic in nations where malaria is highly pandemic. CONCLUSIONS Chloroquine and hydroxychloroquine have antiviral characteristics in vitro. The findings support the hypothesis that these drugs have efficacy in the treatment of COVID-19. People are currently using these drugs for malaria. It is reasonable, given the hypothetical benefit of these two drugs, that they are now being tested in clinical trials to assess their effectiveness to combat this global health crisis.\n",
      "BACKGROUND AND OBJECTIVE: The world is currently experiencing the Coronavirus Disease-19 (COVID-19) pandemic. There is no approved drug for the definitive treatment of the disease. Various drugs are being tried for the treatment of COVID-19, including hydroxychloroquine (HCQ). This study was performed to systematically review the therapeutic role of HCQ in COVID-19 from the available literature. METHODS: PubMed, Embase, ClinicalTrials.gov, ICTRP (WHO), Cochrane Library databases, and two pre-print servers (medRxiv.org and Research Square) were searched for clinical studies that evaluated the therapeutic role of HCQ on COVID-19 until 10 May 2020. The available studies were critically analyzed and the data were extracted. RESULTS: A total of 663 articles were screened and 12 clinical studies (seven peer-reviewed and published studies and five non-peer-reviewed studies from pre-print servers) with a total sample size of 3543 patients were included. Some of the clinical studies demonstrated good virological and clinical outcomes with HCQ alone or in combination with azithromycin in COVID-19 patients, although the studies had major methodological limitations. Some of the other studies showed negative results with HCQ therapy along with the risk of adverse reactions. CONCLUSION: The results of efficacy and safety of HCQ in COVID-19, as obtained from the clinical studies, are not satisfactory, although many of these studies had major methodological limitations. Stronger evidence from well-designed robust randomized clinical trials is required before conclusively determining the role of HCQ in the treatment of COVID-19. Clinical prudence is required in advocating HCQ as a therapeutic armamentarium in COVID-19.\n",
      "In the absence of a vaccine the medical and scientific community is looking intensely at utilizing a pre or post exposure drug that could decrease viremia. The search for a medication that could reduce risk of serious disease, and ideally of any manifestation of disease from SARS-CoV2, and of asymptomatic shedding of SARS-CoV2 is of urgent interest. Repurposing existing pharmaceuticals is among the approaches to achieve these ends. We performed a systematic review of all interventional studies registered in ClinicalTrials.gov with a focus on one repurposed drug, Hydroxychloroquine (HCQ). The detailed analysis of these studies, some of them already recruiting, provide an overall picture of HCQ use as a COVID-19 prophylaxis around the world. Among the included studies, all but three were randomized and parallel and most of them (74%, 23/31) were double-blinded to quadruple-blinded studies. We found a great diversity in dosing and nearly all the possible scientifically reasonable regimens are under evaluation. This diversity offers benefits as well as challenges. Importantly, the final analysis of these trials should be done through an extensive reading of the results in regard to the clinical design, it will be crucial to carefully read and evaluate the results of each study in regards to the clinical design rather than quickly glancing a 140 characters-based social media message announcing the failure or success of a drug against a disease.\n",
      "BACKGROUND: As the number of severe acute respiratory syndrome coronavirus 2 (SARS-COV-2) infected people is skyrocketing worldwide, the international medical situation becomes very serious. Potential therapeutic drugs, vaccine and stem cell replacement methods are emerging, it is urgent to find specific therapeutic drugs and the best treatment regimens. After the publication of hydroxychloroquine (HCQ) with anti-SARS-COV-2 activity in vitro, a small, non-randomized, open-label clinical trial showed that HCQ treatment was significantly associated with reduced viral load in patients with coronavirus disease-19 (COVID-19). Meanwhile, a large prophylaxis study of HCQ sulfate for COVID-19 has been initiated in the United States. HCQ offered a promising efficacy in the treatment of COVID-19, but the optimal administration is still being explored. METHODS: Use the keyword 'Hydroxychloroquine' to conduct a literature search in PubMed to collect relevant literature on the mechanism of action of HCQ, clinical efficacy and safety, pharmacokinetic characteristics, precautions for clinical use and drug interactions, and extract and organize information. RESULTS: This paper reviews the mechanism, clinical efficacy and safety, pharmacokinetic characteristics, exposure-response relationship and precautions and drug interactions of HCQ, and summarized dosage recommendations for HCQ sulfate. CONCLUSION: It has been proved that HCQ, which has an established safety profile, is effective against SARS-CoV-2 with sufficient pre-clinical rationale and evidence. Data from high-quality clinical trials are urgently needed worldwide.\n",
      "Hydroxychloroquine (HCQ) is a promising candidate for Coronavirus disease of 2019 (COVID-19) treatment. The optimal dosing of HCQ is unknown. Our goal was to integrate historic and emerging pharmacological and toxicity data to understand safe and efficacious HCQ dosing strategies for COVID-19 treatment. The data sources included were (i) longitudinal clinical, pharmacokinetic (PK), and virologic data from patients with severe acute respiratory syndrome-2 (SARS-CoV-2) infection who received HCQ with or without azithromycin (n = 116), (ii) in vitro viral replication data and SARS-CoV-2 viral load inhibition by HCQ, (iii) a population PK model of HCQ, and (iv) a model relating chloroquine PKs to corrected QT (QTc) prolongation. A mechanistic PK/virologic/QTc model for HCQ was developed and externally validated to predict SARS-CoV-2 rate of viral decline and QTc prolongation. SARS-CoV-2 viral decline was associated with HCQ PKs (P < 0.001). The extrapolated patient half-maximal effective concentration (EC50 ) was 4.7 µM, comparable to the reported in vitro EC50s . HCQ doses > 400 mg b.i.d. for &#8805;5 days were predicted to rapidly decrease viral loads, reduce the proportion of patients with detectable SARS-CoV-2 infection, and shorten treatment courses, compared with lower dose (&#8804; 400 mg daily) regimens. However, HCQ doses > 600 mg b.i.d. were also predicted to prolong QTc intervals. This prolongation may have clinical implications warranting further safety assessment. Due to COVID-19's variable natural history, lower dose HCQ regimens may be indistinguishable from controls. Evaluation of higher HCQ doses is needed to ensure adequate safety and efficacy.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 5o9bbspc\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "are there any clinical trials available for the coronavirus\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Abstract Background Chloroquine and hydroxychloroquine have been found to be efficient on SARS-CoV-2, and reported to be efficient in Chinese COV-19 patients. We evaluate the role of hydroxychloroquine on respiratory viral loads. Patients and methods French Confirmed COVID-19 patients were included in a single arm protocol from early March to March 16th, to receive 600mg of hydroxychloroquine daily and their viral load in nasopharyngeal swabs was tested daily in a hospital setting. Depending on their clinical presentation, azithromycin was added to the treatment. Untreated patients from another center and cases refusing the protocol were included as negative controls. Presence and absence of virus at Day6-post inclusion was considered the end point. Results Six patients were asymptomatic, 22 had upper respiratory tract infection symptoms and eight had lower respiratory tract infection symptoms. Twenty cases were treated in this study and showed a significant reduction of the viral carriage at D6-post inclusion compared to controls, and much lower average carrying duration than reported of untreated patients in the literature. Azithromycin added to hydroxychloroquine was significantly more efficient for virus elimination. Conclusion Despite its small sample size our survey shows that hydroxychloroquine treatment is significantly associated with viral load reduction/disappearance in COVID-19 patients and its effect is reinforced by azithromycin.\n",
      "BACKGROUND: Chloroquine and hydroxychloroquine have been found to be efficient on SARS-CoV-2, and reported to be efficient in Chinese COV-19 patients. We evaluate the role of hydroxychloroquine on respiratory viral loads. PATIENTS AND METHODS: French Confirmed COVID-19 patients were included in a single arm protocol from early March to March 16th, to receive 600mg of hydroxychloroquine daily and their viral load in nasopharyngeal swabs was tested daily in a hospital setting. Depending on their clinical presentation, azithromycin was added to the treatment. Untreated patients from another center and cases refusing the protocol were included as negative controls. Presence and absence of virus at Day6-post inclusion was considered the end point. RESULTS: Six patients were asymptomatic, 22 had upper respiratory tract infection symptoms and eight had lower respiratory tract infection symptoms. Twenty cases were treated in this study and showed a significant reduction of the viral carriage at D6-post inclusion compared to controls, and much lower average carrying duration than reported of untreated patients in the literature. Azithromycin added to hydroxychloroquine was significantly more efficient for virus elimination. CONCLUSION: Despite its small sample size our survey shows that hydroxychloroquine treatment is significantly associated with viral load reduction/disappearance in COVID-19 patients and its effect is reinforced by azithromycin.\n",
      "Background Chloroquine and Hydroxychloroquine have been found to be efficient on COV-19, and reported to be efficient in Chinese patients infected by this virus. We evaluate the role of Hydroxychloroquine on respiratory viral loads. Patients and methods Patients were included in a single arm protocol to receive 600mg of hydroxychloroquine daily and their viral load in nasal swabs was tested daily. Depending on their clinical presentation azithromycin was added to the treatment. Untreated patients from another center and cases refusing the protocol were included as negative control. Presence and absence of virus at Day-6 was considered the end point. Results Twenty cases were treated in this study and showed a significant reduction of the viral carriage at D-6 compared to controls, and much lower than reported average carrying duration of untreated patients in the literature. Azithromycin added to Hydroxychloroquine was significantly more efficient for virus elimination. Conclusion : Hydroxychloroquine is significantly associated with viral load reduction/disappearance in patients with COVID-19 and its effect is reinforced by Azithromycin.\n",
      "Background Effective therapies are urgently needed for the SARS-CoV-2 pandemic. Chloroquine has been proved to have antiviral effect against coronavirus in vitro. In this study, we aimed to assess the efficacy and safety of chloroquine with different doses in COVID-19. Method In this multicenter prospective observational study, we enrolled patients older than 18 years old with confirmed SARS-CoV-2 infection excluding critical cases from 12 hospitals in Guangdong and Hubei Provinces. Eligible patients received chloroquine phosphate 500mg, orally, once (half dose) or twice (full dose) daily. Patients treated with non-chloroquine therapy were included as historical controls. The primary endpoint is the time to undetectable viral RNA. Secondary outcomes include the proportion of patients with undetectable viral RNA by day 10 and 14, hospitalization time, duration of fever, and adverse events. Results A total of 197 patients completed chloroquine treatment, and 176 patients were included as historical controls. The median time to achieve an undetectable viral RNA was shorter in chloroquine than in non-chloroquine (absolute difference in medians -6.0 days; 95% CI -6.0 to -4.0). The duration of fever is shorter in chloroquine (geometric mean ratio 0.6; 95% CI 0.5 to 0.8). No serious adverse events were observed in the chloroquine group. Patients treated with half dose experienced lower rate of adverse events than with full dose. Conclusions Although randomised trials are needed for further evaluation, this study provides evidence for safety and efficacy of chloroquine in COVID-19 and suggests that chloroquine can be a cost-effective therapy for combating the COVID-19 pandemic.\n",
      "Background: Some studies have shown that hydroxychloroquine (HCQ) is an effective drug in reducing the in vitro replication of SARS-CoV-2. However, the in vivo effect of HCQ still unclear. This study aims to evaluate viral load clearance in patients with COVID-19 who underwent HCQ treatment in comparison with a control group that did not receive the drug. Methods: This prospective study comprised consecutive viral load measurements in patients with COVID-19 hospitalized with a moderate illness. Patients received 400 mg of HCQ every 12 hours for 10 days according to the medical decision. Nasal swab samples were collected at the 1st, 7th, and 14th days of the admission. Results: 155 samples were collected from 66 patients with COVID-19 (60% female), with a median age of 58 years. The viral load between studied groups, assumed as a semiquantitative measure of cycle threshold (Ct) values, presented no significant difference within the three consecutive measures ({Delta}Ct) (p>0.05). We also analyzed the {Delta}Ct viral load at different intervals of sample collection ({Delta}t <7; 7-12 and >12 days) without significant differences at any {Delta}Ct (p>0.05). Conclusion: In this study, we did not observe any change in viral load in vivo with the use of HCQ.\n",
      "Backgrounds. Since COVID-19 outbreak, various agents have been tested but no proven effective therapies have been identified. This has led to a lot of controversies among associated researches. Hence, in order to address the issue of using hydroxychloroquine in treating COVID-19 patients, we conducted a systematic review and meta-analysis. Methods. A thorough search was carried out to find relevant studies in MEDLINE, medRxiv, PubMed, Cochrane Database, China Academic Journals Full-text Database and Web of Science. Two investigators independently reviewed 274 abstracts and 23 articles. The trials which evaluated hydroxychloroquine for treatment of COVID-19 were included for this systematic review. Two investigators assessed quality of the studies and data extraction was done by one reviewer and cross checked by the other. Results. Five trials involving 677 patients were included while conducting the meta-analysis. Compared with the control group, hydroxychloroquine with or without azithromycin showed benefits in positive-to-negative conversion of SARS-CoV-2 (odds ratio [OR], 1.95 [95% CI,0.19 to 19.73] and a reduction in progression rate (OR, 0.89 [95% CI, 0.58 to 1.37]), but without demonstrating any statistical significance. This systematic review has also suggested a possible synergistic effect of the combination therapy which included hydroxychloroquine and azithromycin. However, the use of hydroxychloroquine alone was associated with increased mortality in COVID-19 patients. Conclusion. The use of hydroxychloroquine with or without azithromycin for treatment of COVID-19 patients, seems to be effective. The combination of hydroxychloroquine and azithromycin has shown synergic effects. However, mortality rate was increased when the treatment was conducted with hydroxychloroquine.\n",
      "BACKGROUND: Effective therapies are urgently needed for the SARS-CoV-2 pandemic. Chloroquine has been proved to have antiviral effect against coronavirus in vitro. In this study, we aimed to assess the efficacy and safety of chloroquine with different doses in COVID-19. METHOD: In this multicenter prospective observational study, we enrolled patients older than 18 years old with confirmed SARS-CoV-2 infection excluding critical cases from 12 hospitals in Guangdong and Hubei Provinces. Eligible patients received chloroquine phosphate 500 mg, orally, once (half dose) or twice (full dose) daily. Patients treated with non-chloroquine therapy were included as historical controls. The primary endpoint is the time to undetectable viral RNA. Secondary outcomes include the proportion of patients with undetectable viral RNA by day 10 and 14, hospitalization time, duration of fever, and adverse events. RESULTS: A total of 197 patients completed chloroquine treatment, and 176 patients were included as historical controls. The median time to achieve an undetectable viral RNA was shorter in chloroquine than in non-chloroquine (absolute difference in medians -6.0 days; 95% CI -6.0 to -4.0). The duration of fever is shorter in chloroquine (geometric mean ratio 0.6; 95% CI 0.5 to 0.8). No serious adverse events were observed in the chloroquine group. Patients treated with half dose experienced lower rate of adverse events than with full dose. CONCLUSIONS: Although randomised trials are needed for further evaluation, this study provides evidence for safety and efficacy of chloroquine in COVID-19 and suggests that chloroquine can be a cost-effective therapy for combating the COVID-19 pandemic.\n",
      "OBJECTIVE: To assess the effectiveness of hydroxychloroquine in patients admitted to hospital with coronavirus disease 2019 (covid-19) pneumonia who require oxygen. DESIGN: Comparative observational study using data collected from routine care. SETTING: Four French tertiary care centres providing care to patients with covid-19 pneumonia between 12 March and 31 March 2020. PARTICIPANTS: 181 patients aged 18-80 years with documented severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) pneumonia who required oxygen but not intensive care. INTERVENTIONS: Hydroxychloroquine at a dose of 600 mg/day within 48 hours of admission to hospital (treatment group) versus standard care without hydroxychloroquine (control group). MAIN OUTCOME MEASURES: The primary outcome was survival without transfer to the intensive care unit at day 21. Secondary outcomes were overall survival, survival without acute respiratory distress syndrome, weaning from oxygen, and discharge from hospital to home or rehabilitation (all at day 21). Analyses were adjusted for confounding factors by inverse probability of treatment weighting. RESULTS: In the main analysis, 84 patients who received hydroxychloroquine within 48 hours of admission to hospital (treatment group) were compared with 89 patients who did not receive hydroxychloroquine (control group). Eight additional patients received hydroxychloroquine more than 48 hours after admission. In the weighted analyses, the survival rate without transfer to the intensive care unit at day 21 was 76% in the treatment group and 75% in the control group (weighted hazard ratio 0.9, 95% confidence interval 0.4 to 2.1). Overall survival at day 21 was 89% in the treatment group and 91% in the control group (1.2, 0.4 to 3.3). Survival without acute respiratory distress syndrome at day 21 was 69% in the treatment group compared with 74% in the control group (1.3, 0.7 to 2.6). At day 21, 82% of patients in the treatment group had been weaned from oxygen compared with 76% in the control group (weighted risk ratio 1.1, 95% confidence interval 0.9 to 1.3). Eight patients in the treatment group (10%) experienced electrocardiographic modifications that required discontinuation of treatment. CONCLUSIONS: Hydroxychloroquine has received worldwide attention as a potential treatment for covid-19 because of positive results from small studies. However, the results of this study do not support its use in patients admitted to hospital with covid-19 who require oxygen.\n",
      "BACKGROUND: We need an effective treatment to cure COVID-19 patients and to decrease virus carriage duration. METHODS: We conducted an uncontrolled, non-comparative, observational study in a cohort of 80 relatively mildly infected inpatients treated with a combination of hydroxychloroquine and azithromycin over a period of at least three days, with three main measurements: clinical outcome, contagiousness as assessed by PCR and culture, and length of stay in infectious disease unit (IDU). RESULTS: All patients improved clinically except one 86 year-old patient who died, and one 74 year-old patient still in intensive care. A rapid fall of nasopharyngeal viral load was noted, with 83% negative at Day7, and 93% at Day8. Virus cultures from patient respiratory samples were negative in 97.5% of patients at Day5. Consequently patients were able to be rapidly discharged from IDU with a mean length of stay of five days. CONCLUSION: We believe there is urgency to evaluate the effectiveness of this potentially-life saving therapeutic strategy at a larger scale, both to treat and cure patients at an early stage before irreversible severe respiratory complications take hold and to decrease duration of carriage and avoid the spread of the disease. Furthermore, the cost of treatment is negligible.\n",
      "Hydroxychloroquine (HCQ) is a promising candidate for Coronavirus disease of 2019 (COVID-19) treatment. The optimal dosing of HCQ is unknown. Our goal was to integrate historic and emerging pharmacological and toxicity data to understand safe and efficacious HCQ dosing strategies for COVID-19 treatment. The data sources included were (i) longitudinal clinical, pharmacokinetic (PK), and virologic data from patients with severe acute respiratory syndrome-2 (SARS-CoV-2) infection who received HCQ with or without azithromycin (n = 116), (ii) in vitro viral replication data and SARS-CoV-2 viral load inhibition by HCQ, (iii) a population PK model of HCQ, and (iv) a model relating chloroquine PKs to corrected QT (QTc) prolongation. A mechanistic PK/virologic/QTc model for HCQ was developed and externally validated to predict SARS-CoV-2 rate of viral decline and QTc prolongation. SARS-CoV-2 viral decline was associated with HCQ PKs (P < 0.001). The extrapolated patient half-maximal effective concentration (EC50 ) was 4.7 µM, comparable to the reported in vitro EC50s . HCQ doses > 400 mg b.i.d. for &#8805;5 days were predicted to rapidly decrease viral loads, reduce the proportion of patients with detectable SARS-CoV-2 infection, and shorten treatment courses, compared with lower dose (&#8804; 400 mg daily) regimens. However, HCQ doses > 600 mg b.i.d. were also predicted to prolong QTc intervals. This prolongation may have clinical implications warranting further safety assessment. Due to COVID-19's variable natural history, lower dose HCQ regimens may be indistinguishable from controls. Evaluation of higher HCQ doses is needed to ensure adequate safety and efficacy.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 0lk8eujq\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Background: Coronavirus pandemic is currently a global public health emergency. At present, no pharmacological treatment is known to treat this condition, and there is a need to review the available treatments. Objective: While there have been studies to describe the role of chloroquine and hydroxychloroquine in various viral conditions, there is limited information about the use of them in COVID-19. This systematic review aims to summarize the available evidence regarding the role of chloroquine in treating coronavirus infection. Methods: The preferred reporting items for systematic reviews and meta-analyses (PRISMA) guidelines were used for this review. A literature search was performed using PUBMED & Google Scholar to find articles about the role of CQ in COVID-19 patients. Results: We included 19 publications (Five published articles, three letters/correspondence, one commentary, five pre-proofs of accepted articles, one abstract of yet to be published article, and four were pre-prints (not yet peer-reviewed) articles) in this systematic review. All the articles mentioned about the role of chloroquine and /or hydroxychloroquine in limiting the infection with SARS-CoV-2 (the virus causing COVID-19). Conclusions: There is theoretical, experimental, preclinical and clinical evidence of the effectiveness of chloroquine in patients affected with COVID-19. There is adequate evidence of drug safety from the long-time clinical use of chloroquine and hydroxychloroquine in other indications. More data from ongoing and future trials will add more insight into the role of chloroquine and hydroxychloroquine in COVID-19 infection.\n",
      "PURPOSE: COVID-19 (coronavirus disease 2019) is a public health emergency of international concern. As of this time, there is no known effective pharmaceutical treatment, although it is much needed for patient contracting the severe form of the disease. The aim of this systematic review was to summarize the evidence regarding chloroquine for the treatment of COVID-19. METHODS: PubMed, EMBASE, and three trial Registries were searched for studies on the use of chloroquine in patients with COVID-19. RESULTS: We included six articles (one narrative letter, one in-vitro study, one editorial, expert consensus paper, two national guideline documents) and 23 ongoing clinical trials in China. Chloroquine seems to be effective in limiting the replication of SARS-CoV-2 (virus causing COVID-19) in vitro. CONCLUSIONS: There is rationale, pre-clinical evidence of effectiveness and evidence of safety from long-time clinical use for other indications to justify clinical research on chloroquine in patients with COVID-19. However, clinical use should either adhere to the Monitored Emergency Use of Unregistered Interventions (MEURI) framework or be ethically approved as a trial as stated by the World Health Organization. Safety data and data from high-quality clinical trials are urgently needed.\n",
      "OBJECTIVE The novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), also called COVID-19, has caused a pandemic which has swiftly involved the entire world and raised great public health concerns. The scientific community is actively exploring treatments that would potentially be effective in combating COVID-19. Hydroxychloroquine has been demonstrated to limit the replication of SARS-CoV-2 virus in vitro. In malarial pandemic countries, chloroquine is widely used to treat malaria. In malarial non-pandemic nations, chloroquine is not widely used. Chloroquine and hydroxychloroquine share similar chemical structures and mechanisms of action. The aim of this study was to indirectly investigate the efficacy of chloroquine and hydroxychloroquine for the treatment of COVID-19 by determining the prevalence of COVID-19 in malaria pandemic and non-pandemic nations. We sought evidence to support or refute the hypothesis that these drugs could show efficacy in the treatment of COVID-19. MATERIALS AND METHODS We reviewed in vitro studies, in vivo studies, original studies, clinical trials, and consensus reports, that were conducted to evaluate the antiviral activities of chloroquine and hydroxychloroquine. The studies on \"COVID-19 and its allied treatment were found from World Health Organization (WHO), ISI-Web of Science, PubMed, EMBASE, Scopus, Google Scholar, and clinical trial registries. The search was based on keywords: antiviral drugs, chloroquine, hydroxychloroquine, COVID-19, COVID-19 treatment modalities, and coronavirus. In addition, we analyzed the prevalence of COVID-19 in malaria pandemic and non-pandemic countries. The review and analyses were performed on March 28, 2020. RESULTS For this study, we identified a total of 09 published articles: 03 clinical trials with sample size 150; 03 in vitro studies and 03 expert consensus reports. These studies were all suggestive that chloroquine and hydroxychloroquine can successfully treat COVID-19 infections. We found that COVID-19 infections are highly pandemic in countries where malaria is least pandemic and are least pandemic in nations where malaria is highly pandemic. CONCLUSIONS Chloroquine and hydroxychloroquine have antiviral characteristics in vitro. The findings support the hypothesis that these drugs have efficacy in the treatment of COVID-19. People are currently using these drugs for malaria. It is reasonable, given the hypothetical benefit of these two drugs, that they are now being tested in clinical trials to assess their effectiveness to combat this global health crisis.\n",
      "BACKGROUND AND OBJECTIVE: The world is currently experiencing the Coronavirus Disease-19 (COVID-19) pandemic. There is no approved drug for the definitive treatment of the disease. Various drugs are being tried for the treatment of COVID-19, including hydroxychloroquine (HCQ). This study was performed to systematically review the therapeutic role of HCQ in COVID-19 from the available literature. METHODS: PubMed, Embase, ClinicalTrials.gov, ICTRP (WHO), Cochrane Library databases, and two pre-print servers (medRxiv.org and Research Square) were searched for clinical studies that evaluated the therapeutic role of HCQ on COVID-19 until 10 May 2020. The available studies were critically analyzed and the data were extracted. RESULTS: A total of 663 articles were screened and 12 clinical studies (seven peer-reviewed and published studies and five non-peer-reviewed studies from pre-print servers) with a total sample size of 3543 patients were included. Some of the clinical studies demonstrated good virological and clinical outcomes with HCQ alone or in combination with azithromycin in COVID-19 patients, although the studies had major methodological limitations. Some of the other studies showed negative results with HCQ therapy along with the risk of adverse reactions. CONCLUSION: The results of efficacy and safety of HCQ in COVID-19, as obtained from the clinical studies, are not satisfactory, although many of these studies had major methodological limitations. Stronger evidence from well-designed robust randomized clinical trials is required before conclusively determining the role of HCQ in the treatment of COVID-19. Clinical prudence is required in advocating HCQ as a therapeutic armamentarium in COVID-19.\n",
      "Background: There is no effective therapy for COVID-19. Hydroxychloroquine (HCQ) and chloroquine (CQ) have been used for its treatment but their safety and efficacy remain uncertain. Objective: We performed a systematic review to synthesize the available data on the efficacy and safety of CQ and HCQ for the treatment of COVID-19. Methods: Two reviewers searched for published and pre-published relevant articles between December 2019 to 8th June 2020. The data from the selected studies were abstracted and analyzed for efficacy and safety outcomes. Critical appraisal of the evidence was done by Cochrane risk of bias tool and Newcastle Ottawa scale. The quality of evidence was graded as per the GRADE approach. Results: We reviewed 12 observational and 3 randomized trials which included 10659 patients of whom 5713 received CQ/HCQ and 4966 received only standard of care. The efficacy of CQ/HCQ for COVID-19 was inconsistent across the studies. Meta-analysis of included studies revealed no significant reduction in mortality with HCQ use [RR 0.98 95% CI 0.66-1.46] , time to fever resolution [mean difference -0.54 days (-1.19-011)] or clinical deterioration/development of ARDS with HCQ [RR 0.90 95% CI 0.47-1.71]. There was a higher risk of ECG abnormalities/arrhythmia with HCQ/CQ [RR 1.46 95% CI 1.04 to 2.06]. The quality of evidence was graded as very low for these outcomes. Conclusions: The available evidence suggests that CQ or HCQ does not improve clinical outcomes in COVID-19. Well-designed randomized trials are required for assessing the efficacy and safety of HCQ and CQ for COVID-19.\n",
      "BACKGROUND AND OBJECTIVE: The world is currently experiencing the Coronavirus Disease-19 (COVID-19) pandemic. There is no approved drug for the definitive treatment of the disease. Various drugs are being tried for the treatment of COVID-19, including hydroxychloroquine (HCQ). This study was performed to systematically review the therapeutic role of HCQ in COVID-19 from the available literature. METHODS: PubMed, Embase, ClinicalTrials.gov, ICTRP (WHO), Cochrane Library databases, and two pre-print servers (medRxiv.org and Research Square) were searched for clinical studies that evaluated the therapeutic role of HCQ on COVID-19 until 10 May 2020. The available studies were critically analyzed and the data were extracted. RESULTS: A total of 663 articles were screened and 12 clinical studies (seven peer-reviewed and published studies and five non-peer-reviewed studies from pre-print servers) with a total sample size of 3543 patients were included. Some of the clinical studies demonstrated good virological and clinical outcomes with HCQ alone or in combination with azithromycin in COVID-19 patients, although the studies had major methodological limitations. Some of the other studies showed negative results with HCQ therapy along with the risk of adverse reactions. CONCLUSION: The results of efficacy and safety of HCQ in COVID-19, as obtained from the clinical studies, are not satisfactory, although many of these studies had major methodological limitations. Stronger evidence from well-designed robust randomized clinical trials is required before conclusively determining the role of HCQ in the treatment of COVID-19. Clinical prudence is required in advocating HCQ as a therapeutic armamentarium in COVID-19. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (10.1007/s40261-020-00927-1) contains supplementary material, which is available to authorized users.\n",
      "OBJECTIVE: The novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), also called COVID-19, has caused a pandemic which has swiftly involved the entire world and raised great public health concerns. The scientific community is actively exploring treatments that would potentially be effective in combating COVID-19. Hydroxychloroquine has been demonstrated to limit the replication of SARS-CoV-2 virus in vitro. In malarial pandemic countries, chloroquine is widely used to treat malaria. In malarial non-pandemic nations, chloroquine is not widely used. Chloroquine and hydroxychloroquine share similar chemical structures and mechanisms of action. The aim of this study was to indirectly investigate the efficacy of chloroquine and hydroxychloroquine for the treatment of COVID-19 by determining the prevalence of COVID-19 in malaria pandemic and non-pandemic nations. We sought evidence to support or refute the hypothesis that these drugs could show efficacy in the treatment of COVID-19. MATERIALS AND METHODS: We reviewed in vitro studies, in vivo studies, original studies, clinical trials, and consensus reports, that were conducted to evaluate the antiviral activities of chloroquine and hydroxychloroquine. The studies on \"COVID-19 and its allied treatment were found from World Health Organization (WHO), ISI-Web of Science, PubMed, EMBASE, Scopus, Google Scholar, and clinical trial registries. The search was based on keywords: antiviral drugs, chloroquine, hydroxychloroquine, COVID-19, COVID-19 treatment modalities, and coronavirus. In addition, we analyzed the prevalence of COVID-19 in malaria pandemic and non-pandemic countries. The review and analyses were performed on March 28, 2020. RESULTS: For this study, we identified a total of 09 published articles: 03 clinical trials with sample size 150; 03 in vitro studies and 03 expert consensus reports. These studies were all suggestive that chloroquine and hydroxychloroquine can successfully treat COVID-19 infections. We found that COVID-19 infections are highly pandemic in countries where malaria is least pandemic and are least pandemic in nations where malaria is highly pandemic. CONCLUSIONS: Chloroquine and hydroxychloroquine have antiviral characteristics in vitro. The findings support the hypothesis that these drugs have efficacy in the treatment of COVID-19. People are currently using these drugs for malaria. It is reasonable, given the hypothetical benefit of these two drugs, that they are now being tested in clinical trials to assess their effectiveness to combat this global health crisis.\n",
      "OBJECTIVE: The pandemic coronavirus disease‐19 (COVID‐19) has pushed the global healthcare system to a crisis and amounted to a huge economic burden. Different drugs for prophylaxis against COVID‐19 including chloroquine (CQ) or hydroxychloroquine (HCQ) have been tried. This study was performed to systematically review the role of CQ and HCQ in preventing the spread of COVID‐19. METHODS: PubMed, EMBASE, ClinicalTrials.gov, International Clinical Trials Registry Platform and Cochrane Library databases were searched for studies that evaluated the prophylactic role of CQ or HCQ on SARS‐CoV‐2 (pre‐clinical studies) or COVID‐19 (clinical studies) until 30 March 2020. The available literature was critically appraised. RESULTS: A total of 45 articles were screened and 5 (3 in vitro pre‐clinical studies and 2 clinical opinions) were included. The pre‐clinical studies showed the prophylactic effects of CQ and HCQ against SARS‐CoV‐2. On the other hand, the clinical opinions advocated the prophylactic use of CQ and HCQ against COVID‐19. However, no original clinical studies on the prophylactic role of CQ or HCQ on COVID‐19 were available. CONCLUSION: Although pre‐clinical results are promising, to date there is a dearth of evidence to support the efficacy of CQ or HCQ in preventing COVID‐19. Considering potential safety issues and the likelihood of imparting a false sense of security, prophylaxis with CQ or HCQ against COVID‐19 needs to be thoroughly evaluated in observational studies or high‐quality randomized controlled studies.\n",
      "BACKGROUND AND AIMS: No drugs are currently approved for Coronavirus Disease-2019 (COVID-19), although some have been tried. In view of recent studies and discussion on chloroquine and hydroxychloroquine (HCQ), we aimed to review existing literature and relevant websites regarding these drugs and COVID-19, adverse effects related to drugs, and related guidelines. AIMS AND METHODS: We systematically searched the PubMed database up till March 21, 2020 and retrieved all the articles published on chloroquine and HCQ and COVID-19. RESULTS: Two small human studies have been conducted with both these drugs in COVID-19, and have shown significant improvement in some parameters in patients with COVID-19. CONCLUSION: Considering minimal risk upon use, a long experience of use in other diseases, cost-effectiveness and easy availability across India, we propose that both these drugs are worthy of fast track clinical trial for treatment, and may be carefully considered for clinical use as experimental drugs. Since HCQ has been approved for treatment of diabetes in India, it should be further researched in diabetes and COVID-19, a subgroup where significant mortality has been shown.\n",
      "OBJECTIVE: The pandemic coronavirus disease-19 (COVID-19) has pushed the global healthcare system to a crisis and amounted to a huge economic burden. Different drugs for prophylaxis against COVID-19 including chloroquine (CQ) or hydroxychloroquine (HCQ) have been tried. This study was performed to systematically review the role of CQ and HCQ in preventing the spread of COVID-19. METHODS: PubMed, EMBASE, ClinicalTrials.gov, International Clinical Trials Registry Platform and Cochrane Library databases were searched for studies that evaluated the prophylactic role of CQ or HCQ on SARS-CoV-2 (pre-clinical studies) or COVID-19 (clinical studies) until 30 March 2020. The available literature was critically appraised. RESULTS: A total of 45 articles were screened and 5 (3 in vitro pre-clinical studies and 2 clinical opinions) were included. The pre-clinical studies showed the prophylactic effects of CQ and HCQ against SARS-CoV-2. On the other hand, the clinical opinions advocated the prophylactic use of CQ and HCQ against COVID-19. However, no original clinical studies on the prophylactic role of CQ or HCQ on COVID-19 were available. CONCLUSION: Although pre-clinical results are promising, to date there is a dearth of evidence to support the efficacy of CQ or HCQ in preventing COVID-19. Considering potential safety issues and the likelihood of imparting a false sense of security, prophylaxis with CQ or HCQ against COVID-19 needs to be thoroughly evaluated in observational studies or high-quality randomized controlled studies.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 1ew0p6x7\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the mortality rates overall and in specific populations\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "The coronavirus disease 2019 (COVID-19) has been found to be caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). However, comprehensive knowledge of COVID-19 remains incomplete and many important features are still unknown. This manuscripts conduct a meta-analysis and a sensitivity study to answer the questions: What is the basic reproduction number? How long is the incubation time of the disease on average? What portion of infections are asymptomatic? And ultimately, what is the case fatality rate? Our studies estimate the basic reproduction number to be 3.15 with the 95% interval (2.41, 3.90), the average incubation time to be 5.08 days with the 95% confidence interval (4.77, 5.39) (in day), the asymptomatic infection rate to be 46% with the 95% confidence interval (18.48%, 73.60%), and the case fatality rate to be 2.72% with 95% confidence interval (1.29%, 4.16%) where asymptomatic infections are accounted for.\n",
      "The coronavirus disease-2019 (COVID-19) has been found to be caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). However, comprehensive knowledge of COVID-19 remains incomplete and many important features are still unknown. This manuscript conducts a meta-analysis and a sensitivity study to answer the questions: What is the basic reproduction number? How long is the incubation time of the disease on average? What portion of infections are asymptomatic? And ultimately, what is the case fatality rate? Our studies estimate the basic reproduction number to be 3.15 with the 95% CI (2.41-3.90), the average incubation time to be 5.08 days with the 95% CI (4.77-5.39) (in day), the asymptomatic infection rate to be 46% with the 95% CI (18.48%-73.60%), and the case fatality rate to be 2.72% with 95% CI (1.29%-4.16%) where asymptomatic infections are accounted for.\n",
      "The coronavirus disease‐2019 (COVID‐19) has been found to be caused by the severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2). However, comprehensive knowledge of COVID‐19 remains incomplete and many important features are still unknown. This manuscript conducts a meta‐analysis and a sensitivity study to answer the questions: What is the basic reproduction number? How long is the incubation time of the disease on average? What portion of infections are asymptomatic? And ultimately, what is the case fatality rate? Our studies estimate the basic reproduction number to be 3.15 with the 95% CI (2.41‐3.90), the average incubation time to be 5.08 days with the 95% CI (4.77‐5.39) (in day), the asymptomatic infection rate to be 46% with the 95% CI (18.48%‐73.60%), and the case fatality rate to be 2.72% with 95% CI (1.29%‐4.16%) where asymptomatic infections are accounted for.\n",
      "Coronavirus disease 2019 (COVID-19) is a novel human respiratory disease caused by the SARS-CoV-2 virus. Asymptomatic carriers of the virus display no clinical symptoms but are known to be contagious. Recent evidence reveals that this sub-population, as well as persons with mild, represent a major contributor in the propagation of COVID-19. The asymptomatic sub-population frequently escapes detection by public health surveillance systems. Because of this, the currently accepted estimates of the basic reproduction number (Ro) of the virus are inaccurate. It is unlikely that a pathogen can blanket the planet in three months with an Ro in the vicinity of 3, as reported in the literature. In this manuscript, we present a mathematical model taking into account asymptomatic carriers. Our results indicate that an initial value of the effective reproduction number could range from 5.5 to 25.4, with a point estimate of 15.4, assuming mean parameters. The first three weeks of the model exhibit exponential growth, which is in agreement with average case data collected from thirteen countries with universal health care and robust communicable disease surveillance systems; the average rate of growth in the number of reported cases is 23.3% per day during this period.\n",
      "Our understanding of the Coronavirus disease 2019 (COVID-19) continues to evolve and there are many unknowns about its epidemiology. This study aims to synthesise case fatality rate (CFR) among confirmed COVID-19 patients, incubation period and time from onset of COVID-19 symptoms to first medical visit, intensive care unit (ICU) admission, recovery, and death. We searched MEDLINE, Embase, Google Scholar, and bibliographies of relevant articles from 01 December 2019 to 11 March 2020 without any language restrictions. Quantitative studies that recruited people with confirmed COVID-19 diagnosis were included. Two independent reviewers extracted the data. Out of 1675 non-duplicate studies, 43 were included in the meta-analysis. The pooled mean incubation period was 5.68 (99% confidence interval [CI]: 4.78, 6.59) days. The pooled mean number of days from the onset of COVID-19 symptoms to first clinical visit was 4.92 (95% CI: 3.95, 5.90), ICU admission was 9.84 (95% CI: 8.78, 10.90), recovery was 18.55 (95% CI: 13.69, 23.41), and death was 15.93 (95% CI: 13.07, 18.79). Pooled CFR among confirmed COVID-19 patients was 0.02 (95% CI: 0.02, 0.03). We found that the incubation period and lag between the onset of symptoms and first clinical visit for COVID-19 are longer than other respiratory viral infections including Middle East respiratory syndrome and severe acute respiratory syndrome; however, the current policy of 14 days of mandatory quarantine for everyone potentially exposed to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) might be too conservative. Longer quarantine periods might be more justified for extreme cases.\n",
      "Our understanding of SARS-CoV-2, the virus responsible for coronavirus disease 2019 (COVID-19), its clinical manifestations, and treatment options continues to evolve at an unparalleled pace. This review sought to summarize the key literature regarding transmission, case definitions, clinical management, and the burden of COVID-19. Our review of the literature showed that SARS-CoV-2 was mainly transmitted via inhalation of respiratory droplets containing the virus and had a mean incubation period of 4-6 days. The commonly reported symptoms were fever (75.3% ± 18.7%) and cough (62.6% ± 17.7%) across the spectrum of clinical disease-mild, moderate, severe, and critical, but with the disease phenotype varying with severity. Categorization of these cases for home care or hospital management needs to be defined, with risk stratification accounting for the age of the patient and the presence of underlying comorbidities. The case definitions varied among countries, which could have contributed to the differences in the case fatality rates among affected countries. The severity and risk of death due to COVID-19 was associated with age and underlying comorbidities. Asymptomatic cases, which constitute which constitute 40-80% of COVID-19 cases are a considerable threat to control efforts. The presence of fever and cough may be sufficient to warrant COVID-19 testing, but using these symptoms in isolation will miss a proportion of cases. A clear definition of a COVID-19 case is essential for the management, treatment, and tracking of clinical illness, and to inform the quarantine measures and social distancing that can help control the spread of SARS-CoV-2.\n",
      "Our understanding of SARS-CoV-2, the virus responsible for coronavirus disease 2019 (COVID-19), its clinical manifestations, and treatment options continues to evolve at an unparalleled pace. This review sought to summarize the key literature regarding transmission, case definitions, clinical management, and the burden of COVID-19. Our review of the literature showed that SARS-CoV-2 was mainly transmitted via inhalation of respiratory droplets containing the virus and had a mean incubation period of 4-6 days. The commonly reported symptoms were fever (75.3% ± 18.7%) and cough (62.6% ± 17.7%) across the spectrum of clinical disease-mild, moderate, severe, and critical, but with the disease phenotype varying with severity. Categorization of these cases for home care or hospital management needs to be defined, with risk stratification accounting for the age of the patient and the presence of underlying comorbidities. The case definitions varied among countries, which could have contributed to the differences in the case fatality rates among affected countries. The severity and risk of death due to COVID-19 was associated with age and underlying comorbidities. Asymptomatic cases, which constitute which constitute 40-80% of COVID-19 cases are a considerable threat to control efforts. The presence of fever and cough may be sufficient to warrant COVID-19 testing, but using these symptoms in isolation will miss a proportion of cases. A clear definition of a COVID-19 case is essential for the management, treatment, and tracking of clinical illness, and to inform the quarantine measures and social distancing that can help control the spread of SARS-CoV-2.\n",
      "Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a new ß­Coronavirus that was first detected in 2019 in Wuhan, China. In the ensuing months it has been transmitted worldwide. Here the authors present the current knowledge on the epidemiology of this virus. SARS-CoV­2 replicates mainly in the upper and lower respiratory tract and is primarily transmitted by droplets from asymptomatic and symptomatic infected subjects. The estimate for the basic reproduction number (R) is currently between 2 and 3, while the incubation period is 6 (median, range 2-14) days. Although most infections are uncomplicated, 5-10% of cases develop pneumonia, which can lead to hospitalization, respiratory failure and multiorgan failure. Risk factors for a complicated disease course include age, hypertension, chronic cardiovascular and pulmonary disease and immunodeficiency. The overall case fatality rate is 1.4%, with the rate rising in the sixth decade of life. Nosocomial and infections in medical personnel have been reported. Drastic reductions in social contact have been implemented in many countries with outbreaks of SARS-CoV­2, leading to rapid reductions in R. Which of the measures have been effective is still unknown.\n",
      "As the COVID-19 pandemic has strongly disrupted people’s daily work and life, a great amount of scientific research has been conducted to understand the key characteristics of this new epidemic. In this manuscript, we focus on four crucial epidemic metrics with regard to the COVID-19, namely the basic reproduction number, the incubation period, the serial interval and the epidemic doubling time. We collect relevant studies based on the COVID-19 data in China and conduct a meta-analysis to obtain pooled estimates on the four metrics. From the summary results, we conclude that the COVID-19 has stronger transmissibility than SARS, implying that stringent public health strategies are necessary.\n",
      "Objective: To estimate the serial interval of novel coronavirus (COVID-19) from information on 28 infector-infectee pairs. Methods: We collected dates of illness onset for primary cases (infectors) and secondary cases (infectees) from published research articles and case investigation reports. We subjectively ranked the credibility of the data and performed analyses on both the full dataset (n=28) and a subset of pairs with highest certainty in reporting (n=18). In addition, we adjusting for right truncation of the data as the epidemic is still in its growth phase. Results: Accounting for right truncation and analyzing all pairs, we estimated the median serial interval at 4.0 days (95% credible interval [CrI]: 3.1, 4.9). Limiting our data to only the most certain pairs, the median serial interval was estimated at 4.6 days (95% CrI: 3.5, 5.9). Conclusions: The serial interval of COVID-19 is shorter than its median incubation period. This suggests that a substantial proportion of secondary transmission may occur prior to illness onset. The COVID-19 serial interval is also shorter than the serial interval of severe acute respiratory syndrome (SARS), indicating that calculations made using the SARS serial interval may introduce bias.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: bjyzrmsb\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what are the initial symptoms of Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Since December 2019, the global pandemic caused by the highly infectious novel coronavirus 2019-nCoV (COVID-19) has been rapidly spreading. As of April 2020, the outbreak has spread to over 210 countries, with over 2,400,000 confirmed cases and over 170,000 deaths [1]. COVID-19 causes a severe pneumonia characterized by fever, cough, and shortness of breath. Similar coronavirus outbreaks have occurred in the past causing severe pneumonia like COVID-19, most recently, severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV). However, over time, SARS-CoV and MERS-CoV were shown to cause extra-pulmonary signs and symptoms including hepatitis, acute renal failure, encephalitis, myositis, and gastroenteritis. Similarly, sporadic reports of COVID-19 related extrapulmonary manifestations emerge. Unfortunately, there is no comprehensive summary of the multi-organ manifestations of COVID-19, making it difficult for clinicians to quickly educate themselves about this highly contagious and deadly pathogen. What's more, is that SARS-CoV and MERS-CoV are the closest humanity has come to combating something similar to COVID-19, however, there exists no comparison between the manifestations of any of these novel coronaviruses. In this review, we summarize the current knowledge of the manifestations of the novel coronaviruses SARS-CoV, MERS-CoV, and COVID-19, with a particular focus on the latter, and highlight their differences and similarities.\n",
      "Since December 2019, the global pandemic caused by the highly infectious novel coronavirus 2019-nCoV (COVID-19) has been rapidly spreading. As of April 2020, the outbreak has spread to over 210 countries, with over 2,400,000 confirmed cases and over 170,000 deaths.1 COVID-19 causes a severe pneumonia characterized by fever, cough and shortness of breath. Similar coronavirus outbreaks have occurred in the past causing severe pneumonia like COVID-19, most recently, severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV). However, over time, SARS-CoV and MERS-CoV were shown to cause extrapulmonary signs and symptoms including hepatitis, acute renal failure, encephalitis, myositis and gastroenteritis. Similarly, sporadic reports of COVID-19 related extrapulmonary manifestations emerge. Unfortunately, there is no comprehensive summary of the multiorgan manifestations of COVID-19, making it difficult for clinicians to quickly educate themselves about this highly contagious and deadly pathogen. What is more, is that SARS-CoV and MERS-CoV are the closest humanity has come to combating something similar to COVID-19, however, there exists no comparison between the manifestations of any of these novel coronaviruses. In this review, we summarize the current knowledge of the manifestations of the novel coronaviruses SARS-CoV, MERS-CoV and COVID-19, with a particular focus on the latter, and highlight their differences and similarities.\n",
      "Abstract Since 2003, coronavirus has caused multiple major public health events that resulted in global epidemics such as, severe acute respiratory syndrome （SARS）, Middle East respiratory syndrome （MERS） and corona virus disease 2019 （COVID-19）. Especially since COVID-19 has outbroken in Wuhan, Hubei, in December 2019, coronavirus has had a significant impact on people's health and lives. But so far, the pathological diagnosis of COVID-19 has been relatively deficient： it is still confined to the pathological findings of punctured organs, and the majority of medical workers have poor awareness of its pathological characteristics. The COVID-19, as same as SARS and MERS, is caused by coronaviruses and can cause viral pneumonia. They have certain similarities. This article comprehensively reviews the pathological features observed in the autopsies of the aforementioned three diseases, in order to provide reference to the analysis of pathological changes of COVID-19.\n",
      "Viral respiratory diseases such as severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) always pose a severe threat to people. First identified in late December 2019, a novel coronavirus (2019-nCoV; SARS-CoV-2) has affected many provinces in China and multiple countries worldwide. The viral outbreak has aroused panic and a public-health emergency around the world, and the number of infections continues to rise. However, the causes and consequences of the pneumonia remain unknown. To effectively implement epidemic prevention, early identification and diagnosis are critical to disease control. Here we scrutinise a series of available studies by global scientists on the clinical manifestations, detection methods and treatment options for the disease caused by SARS-CoV-2, named coronavirus disease 2019 (COVID-19), and also propose potential strategies for preventing the infection.\n",
      "The ongoing outbreak of the coronavirus disease 2019 (COVID-19) as named by the World Health Organization has millions of confirmed cases around the world and has claimed hundreds of thousands of lives. The virus was named SARS-CoV-2 in February by International Committee on Taxonomy of Viruses. COVID-19 presents as fever, dry cough, dyspnea, headache and pneumonia. In a small subset of severe cases, the disease quickly progresses to respiratory failure and even death. Since the 21st century, there have been three major outbreaks caused by human coronaviruses, including the severe acute respiratory syndrome (SARS) that broke out in 2003, the Middle East respiratory syndrome (MERS) in 2012, and the recent pandemic of COVID-19. Since 2003, significant progress has been made in the study of SARS-CoV and MERS-CoV concerning their natural origins, pathogenesis, antiviral development and vaccine design. Since SARS-CoV-2 and SARS-CoV are closely related, previous findings on SARS-CoV are highly relevant to a better understanding as well as diagnosis, treatment, prevention and control of SARS-CoV-2. In this review, we highlight recent progresses in the field; compare the biological characteristics of SARS-CoV and SARS-CoV-2; summarize the urgently-needed diagnostic, treatment, prevention and control options; and provide future perspectives for the outcome of the outbreak and research questions to be answered, including some of the difficulties in vaccine development. Hopefully, our comments and suggestions would prove useful for the control of the SARS-CoV-2 epidemic in China and the world.\n",
      "The ongoing outbreak of the coronavirus disease 2019 (COVID-19) as named by the World Health Organization has millions of confirmed cases around the world and has claimed hundreds of thousands of lives. The virus was named SARS-CoV-2 in February by International Committee on Taxonomy of Viruses. COVID-19 presents as fever, dry cough, dyspnea, headache and pneumonia. In a small subset of severe cases, the disease quickly progresses to respiratory failure and even death. Since the 21st century, there have been three major outbreaks caused by human coronaviruses, including the severe acute respiratory syndrome (SARS) that broke out in 2003, the Middle East respiratory syndrome (MERS) in 2012, and the recent pandemic of COVID-19. Since 2003, significant progress has been made in the study of SARS-CoV and MERS-CoV concerning their natural origins, pathogenesis, antiviral development and vaccine design. Since SARS-CoV-2 and SARS-CoV are closely related, previous findings on SARS-CoV are highly relevant to a better understanding as well as diagnosis, treatment, prevention and control of SARS-CoV-2. In this review, we highlight recent progresses in the field; compare the biological characteristics of SARS-CoV and SARS-CoV-2; summarize the urgently-needed diagnostic, treatment, prevention and control options; and provide future perspectives for the outcome of the outbreak and research questions to be answered, including some of the difficulties in vaccine development. Hopefully, our comments and suggestions would prove useful for the control of the SARS-CoV-2 epidemic in China and the world.\n",
      "ABSTRACT Viral respiratory diseases such as severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) always pose a severe threat to people. First identified in late December 2019, a novel coronavirus (2019-nCoV; SARS-CoV-2) has affected many provinces in China and multiple countries worldwide. The viral outbreak has aroused panic and a public-health emergency around the world, and the number of infections continues to rise. However, the causes and consequences of the pneumonia remain unknown. To effectively implement epidemic prevention, early identification and diagnosis are critical to disease control. Here we scrutinise a series of available studies by global scientists on the clinical manifestations, detection methods and treatment options for the disease caused by SARS-CoV-2, named coronavirus disease 2019 (COVID-19), and also propose potential strategies for preventing the infection.\n",
      "Abstract: Since 2003, coronavirus has caused multiple major public health events that resulted in global epidemics, such as severe acute respiratory syndrome ï¼SARSï¼, Middle East respiratory syndrome ï¼MERSï¼ and corona virus disease 2019 ï¼COVID-19ï¼. Especially since COVID-19 outbroke in Wuhan, Hubei, in December 2019, coronavirus has had a significant impact on people's health and lives. But so far, the pathological diagnosis of COVID-19 has been relatively deficientï¼ it is still confined to the pathological findings of punctured organs, and the majority of medical workers have poor awareness of its pathological characteristics. The COVID-19, as same as SARS and MERS, is caused by coronaviruses and can cause viral pneumonia. They have certain similarities. This article comprehensively reviews the pathological features observed in the autopsies of the aforementioned three diseases, in order to provide reference to the analysis of pathological changes of COVID-19.\n",
      "The novel Coronavirus disease of 2019 (nCOV-19) is a viral outbreak noted first in Wuhan, China. This disease is caused by Severe Acute Respiratory Syndrome (SARS) Coronavirus (CoV)-2. In the past, other members of the coronavirus family, such as SARS and Middle East Respiratory Syndrome (MERS), have made an impact in China and the Arabian peninsula respectively. Both SARS and COVID-19 share similar symptoms such as fever, cough, and difficulty in breathing that can become fatal in later stages. However, SARS and MERS infections were epidemic diseases constrained to limited regions. By March 2020 the SARS-CoV-2 had spread across the globe and on March 11th, 2020 the World Health Organization (WHO) declared COVID-19 as pandemic disease. In severe SARS-CoV-2 infection, many patients succumbed to pneumonia. Higher rates of deaths were seen in older patients who had co-morbidities such as diabetes mellitus, hypertension, cardiovascular disease (CVD), and dementia. In this review paper, we discuss the effect of SARS-CoV-2 on CNS diseases, such as Alzheimer's-like dementia, and diabetes mellitus. We also focus on the virus genome, pathophysiology, theranostics, and autophagy mechanisms. We will assess the multiorgan failure reported in advanced stages of SARS-CoV-2 infection. Our paper will provide mechanistic clues and therapeutic targets for physicians and investigators to combat COVID-19.\n",
      "OBJECTIVE Subsequent to a global outbreak of the Middle East Respiratory Syndrome (MERS) in 2012, a novel human coronavirus, known as Corona Virus Disease 2019 (COVID-19) has caused a major disease outbreak. The aim of this study was to perform a systematic review to compare epidemiological, clinical, and laboratory features of COVID-19 and MERS-COV populations. MATERIALS AND METHODS We searched PubMed, EMBASE, and Cochrane Central Register of Controlled Trials database to identify potential studies that have reported COVID-19 or MERS-COV disease. Epidemiology, clinical, and laboratory outcomes, intensive care unit (ICU) admission rates, discharge rates, and fatality rates were evaluated using Graph-Pad Prism software. RESULTS A total of forty-two studies were included in our research, involving in 4,720 patients (COVID-19 = 2,012, MERS-COV = 2,708). The present study revealed that main clinical manifestations of both COVID-19 and MERS-COV populations are fever, cough and generalized weakness or myalgia, and Acute Respiratory Distress Syndrome (ARDS) is the main complication. The COVID-19 population has a lower rate of ICU admissions, discharges, fatalities, and shorter incubation periods than those of MERS-COV population. CONCLUSIONS The main clinical features of both COVID-19 and MERS-COV populations are fever, cough and generalized weakness or myalgia. ARDS is the main complication of both populations. COVID-19 cases have a shorter incubation period and lower rate of ICU admissions, discharges and fatalities compared to MRES-COV population.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: o8df8szx\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what causes death from Covid-19?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Abstract Objective To describe the clinical features of coronavirus disease 2019 (COVID-19). Methods We recruited 73 patients with COVID-19 [49 men and 24 women; average age: 58.36 years (SD: 14.31)] admitted to the intensive care unit of Wuhan Jinyintan Hospital from December 30, 2019 to February 16, 2020. Demographics, underlying diseases, and laboratory test results on admission were collected and analyzed. Data were compared between survivors and non-survivors. Results The non-survivors were older (65.46 [SD 9.74] vs 46.23 [12.01]) and were more likely to have chronic medical illnesses. Non-survivors tend to develop more severe lymphopenia, with higher C-reactive protein, interleukin-6, D-dimer, and hs-Troponin I(hs-TnI) levels. Patients with elevated hs-TnI levels on admission had shorter duration from symptom onset to death. Increased hs-TnI level was related to dismal prognosis. Death risk increased by 20.8% when the hs-TnI level increased by one unit. After adjusting for inflammatory or coagulation index, the independent predictive relationship between hs-TnI and death disappeared. Conclusions Cardiac injury may occur at the early stage of COVID-19, which is associated with high mortality. Inflammatory factor cascade and coagulation abnormality may be the potential mechanisms of COVID-19 combined with cardiac injury.\n",
      "OBJECTIVE: To describe the clinical features of coronavirus disease 2019 (COVID-19). METHODS: We recruited 73 patients with COVID-19 [49 men and 24 women; average age: 58.36 years (SD: 14.31)] admitted to the intensive care unit of Wuhan Jinyintan Hospital from December 30, 2019 to February 16, 2020. Demographics, underlying diseases, and laboratory test results on admission were collected and analyzed. Data were compared between survivors and non-survivors. RESULTS: The non-survivors were older (65.46 [SD 9.74]vs 46.23 [12.01]) and were more likely to have chronic medical illnesses. Non-survivors tend to develop more severe lymphopenia, with higher C-reactive protein, interleukin-6, D-dimer, and hs-Troponin I(hs-TnI) levels. Patients with elevated hs-TnI levels on admission had shorter duration from symptom onset to death. Increased hs-TnI level was related to dismal prognosis. Death risk increased by 20.8% when the hs-TnI level increased by one unit. After adjusting for inflammatory or coagulation index, the independent predictive relationship between hs-TnI and death disappeared. CONCLUSIONS: Cardiac injury may occur at the early stage of COVID-19, which is associated with high mortality. Inflammatory factor cascade and coagulation abnormality may be the potential mechanisms of COVID-19 combined with cardiac injury.\n",
      "OBJECTIVE In December 2019, a new type of coronavirus-infected pneumonia broke out in Wuhan and spread rapidly to other parts of the country. The purpose of this study was to investigate the clinical features of coronavirus disease 2019 (COVID-19). MATERIALS AND METHODS A retrospective analysis was performed on the confirmed cases of COVID-19, who were admitted to the North Hospital of Changsha first Hospital (Changsha Public Health treatment Center) from January 17 to February 7, 2020. RESULTS The median age of COVID-19 patients was 45 years (range 33.5-57). The male patients accounted for 49.7%, 64.6% of the patients had a history of exposure in Wuhan, and 31.7% had family aggregation. The median days of onset were six, and the incidence of severe illness was 18.6%. Compared with the non-severe group, the severe group showed statistical significance in older age, hypertension, bilateral lung plaque shadow, decrease in lymphocyte count, increase in C-reactive protein (CRP), aspartate aminotransferase (AST), lactate dehydrogenase, and creatine kinase. CONCLUSIONS Age, combined hypertension, oxygenation index, double lung patch, decreased lymphocyte count, and elevated levels of C-reactive protein, aspartate aminotransferase, lactate dehydrogenase, and creatine kinase can be used as predictors of the disease severity.\n",
      "BACKGROUND In December, 2019, China, has experienced an outbreak of novel coronavirus disease 2019 (COVID-19). Coronavirus has now spread to all of the continents. We aimed to consider clinical characteristics, laboratory data of COVID-19 that provided more information for the research of this novel virus. METHODS We performed a retrospective cohort study on the clinical symptoms and laboratory findings of a series of the 100 confirmed patients with COVID-19. These patients were admitted to the hospitals affiliated to Babol University of Medical Sciences (Ayatollah Rohani, Shahid Beheshti and Yahyanejad hospitals) form 25 February 2020 to 12 March 2020. RESULTS Nineteen patients died during hospitalization and 81 were discharged. Non-survivor patients had a significantly higher C-reactive protein (CRP) (MD: 46.37, 95% CI: 20.84, 71.90; P= 0.001), white blood cells (WBCs) (MD: 3.10, 95% CI: 1.53, 4.67; P< 0.001) and lower lymphocyte (MD: -8.75, 95% CI: -12.62, -4.87; P< 0.001) compared to survivor patients Data analysis showed that comorbid conditions (aRR: 2.99, 95%CI: 1.09, 8.21, P= 0.034), higher CRP levels (aRR: 1.02, 95%CI: 1.01, 1.03, P= 0.044), and lower lymphocyte (aRR: 0.82, 95%CI: 0.73, 0.93, P= 0.003) were associated with increased risk of death. CONCLUSIONS Based on our findings, most non-survivors are elderly with comorbidities. Lymphopenia and increased levels of WBCs along with elevated CRP were associated with increased risk of death. Therefore, it is best to be regularly assessed these markers during treatment of COVID-19 patients.\n",
      "BACKGROUND: In December, 2019, China, has experienced an outbreak of novel coronavirus disease 2019 (COVID-19). Coronavirus has now spread to all of the continents. We aimed to consider clinical characteristics, laboratory data of COVID-19 that provided more information for the research of this novel virus. METHODS: We performed a retrospective cohort study on the clinical symptoms and laboratory findings of a series of the 100 confirmed patients with COVID-19. These patients were admitted to the hospitals affiliated to Babol University of Medical Sciences (Ayatollah Rohani, Shahid Beheshti and Yahyanejad hospitals) form 25 February 2020 to 12 March 2020. RESULTS: Nineteen patients died during hospitalization and 81 were discharged. Non-survivor patients had a significantly higher C-reactive protein (CRP) (MD: 46.37, 95% CI: 20.84, 71.90; P= 0.001), white blood cells (WBCs) (MD: 3.10, 95% CI: 1.53, 4.67; P< 0.001) and lower lymphocyte (MD: -8.75, 95% CI: -12.62, -4.87; P< 0.001) compared to survivor patients Data analysis showed that comorbid conditions (aRR: 2.99, 95%CI: 1.09, 8.21, P= 0.034), higher CRP levels (aRR: 1.02, 95%CI: 1.01, 1.03, P= 0.044), and lower lymphocyte (aRR: 0.82, 95%CI: 0.73, 0.93, P= 0.003) were associated with increased risk of death. CONCLUSIONS: Based on our findings, most non-survivors are elderly with comorbidities. Lymphopenia and increased levels of WBCs along with elevated CRP were associated with increased risk of death. Therefore, it is best to be regularly assessed these markers during treatment of COVID-19 patients.\n",
      "Background Although the outbreak of Coronavirus disease 2019 (COVID-19) has caused over 2200 deaths in China, there was no study about death yet. We aimed to describe the clinical characteristics of non-survivors with COVID-19. Methods For this retrospective, single-center study, we included 36 non-survivors with COVID-19 in the Fifth Hospital of Wuhan. Cases were confirmed by real-time RT-PCR between Jan 21 and Feb 10, 2020 according to the recommended protocol. The epidemiological, demographic, clinical, laboratory, radiological and treatment data were collected and analyzed. Outcomes were followed up until Feb 14, 2020. This study was approved by the ethics commissions of the Fifth Hospital of Wuhan, with a waiver of informed consent due to a public health outbreak investigation. Results We included 36 patients who died from COVID-19. The mean age of the patients was 69.22 years (SD 9.64, range 50-90). 25(69.44%) patients were males, and 11 (30.56%) female. 26 (72.22%) patients had chronic diseases, mainly including hypertension, cardiovascular disease and diabetes. Patients had common clinical symptoms of fever (34 [94.44%] patients), cough (28 [77.78%] patients), shortness of breath (21 [58.33%] patients), and fatigue (17 [47.22%] patient). Chest computed tomographic scans showed that 31 (96.88%) patients had bilateral pneumonia. Lymphopenia occurred in 24 patients (70.59%), decreased albumin (30.18, [SD, 4.76]) in 25 patients (80.65%), elevated D-dimer (8.64 [IQR, 2.39-20]) in 27 patients (100%), and elevated lactate dehydrogenase (502.5 U/L [IQR, 410-629]) in 26 patients (100%). Nearly all of the patients have elevated CRP (106.3 mg/L [IQR, 60.83-225.3]), PCT (0.61 ng/ml [IQR, 0.16-2.10]) and IL-6 (100.6 pg/ml [IQR, 51.51-919.5]). Most patients received antiviral therapy and antibiotic therapy, and more than half of patients received glucocorticoid therapy (25 [69.44%]). All the patients had acute respiratory distress syndrome (ARDS). The median time from onset to ARDS was 11 days. One (2.78%) patient presented with acute renal injury. The median time from onset to death was 17 days. Interpretation Lots of patients died from COVID-19 till now. The median survival time of these non-survivors from onset to death was about 2 weeks. Most patients were older males with comorbidities. They finally progressed to ARDS. The median time from onset to ARDS was 11 days. Gradually decreased lymphocytes and increased inflammation biomarkers were common, and need to be monitored in the routine treatment.\n",
      "Objective@#To explore the clinical characteristics and prognosis of the new coronavirus 2019-nCoV patients combined with cardiovascular disease (CVD).@*Methods@#A retrospective analysis was performed on 112 COVID-19 patients with CVD admitted to the western district of Union Hospital in Wuhan, from January 20, 2020 to February 15, 2020. They were divided into critical group (ICU, n=16) and general group (n=96) according to the severity of the disease and patients were followed up to the clinical endpoint. The observation indicators included total blood count, C-reactive protein (CRP), arterial blood gas analysis, myocardial injury markers, coagulation function, liver and kidney function, electrolyte, procalcitonin (PCT), B-type natriuretic peptide (BNP), blood lipid, pulmonary CT and pathogen detection.@*Results@#Compared with the general group, the lymphocyte count (0.74×109 (0.34×109, 0.94×109)/L vs. 0.99×109 (0.71×109, 1.29×109)/L, P=0.03) was extremely lower in the critical group, CRP (106.98 (81.57, 135.76) mg/L vs. 34.34 (9.55,76.54) mg/L, P&lt;0.001) and PCT (0.20 (0.15,0.48) &#x03BC;g/L vs. 0.11 (0.06,0.20)&#x03BC;g/L, P&lt;0.001) were significantly higher in the critical group. The BMI of the critical group was significantly higher than that of the general group (25.5 (23.0, 27.5) kg/m2 vs. 22.0 (20.0, 24.0) kg/m2, P=0.003). Patients were further divided into non-survivor group (17, 15.18%) group and survivor group (95, 84.82%). Among the non-survivors, there were 88.24% (15/17) patients with BMI&gt; 25 kg/m2, which was significantly higher than that of survivors (18.95% (18/95), P&lt;0.001). Compared with the survived patients, oxygenation index (130 (102, 415) vs. 434 (410, 444), P&lt;0.001) was significantly lower and lactic acid (1.70 (1.30, 3.00) mmol/L vs. 1.20 (1.10, 1.60) mmol/L, P&lt;0.001) was significantly higher in the non-survivors. There was no significant difference in the proportion of ACEI/ARB medication between the critical group and the general group or between non-survivors and survivors (all P&gt;0.05).@*Conclusion@#COVID-19 patients combined with CVD are associated with a higher risk of mortality. Critical patients are characterized with lower lymphocyte counts. Higher BMI are more often seen in critical patients and non-survivor. ACEI/ARB use does not affect the morbidity and mortality of COVID-19 combined with CVD. Aggravating causes of death include fulminant inflammation, lactic acid accumulation and thrombotic events.\n",
      "Background: The objectives of this study were to analyze the clinical features of coronavirus disease 2019 (COVID-19) and evaluate the diagnosis and treatment. Methods: A retrospective analysis of the clinical manifestation and auxiliary examination of 19 patients with COVID-19 from the Liyuan Hospital intensive care unit (ICU) between January 16, 2020 and February 20, 2020 was undertaken. Results: There were 11 male and 8 female cases among the patients. The median (range) age was 73 (38-91) years. Of these patients, 8 (42.1%) had died and the median duration from ICU admission to death was 2 (interquartile range (IQR): 1-10.75) days. Seven of these 8 patients had underlying diseases. The auxiliary examination showed fever (68.4%), dry cough (15.8%), dyspnea (10.5%), and diarrhea (5.3%). All 19 cases showed ground-glass changes on chest computed tomography. Serum hypersensitive C-reactive protein (hs-CRP) and serum amylase A (SAA) were clearly increased in all of the cases. Among the 19 cases, there were 16 (84.2%) cases in which the total number of lymphocytes decreased, 12 cases (63%) had reduced liver function, and 11 cases (58%) had deviant results for fibrinogen (FIB) and D-dimer, in particular, the D-dimer level was significantly higher in the non-survivors compared with the survivors. Conclusion: There were more men than women among critically ill patients. All of the cases showed ground-glass changes on chest computed tomography and the vast majority of patients displayed fever and dry cough. The clinical laboratory indices change significantly, especially the D-dimer level among non-survivors.\n",
      "AIMS: To investigate the characteristics and clinical significance of myocardial injury in patients with severe coronavirus disease 2019 (COVID-19). METHODS AND RESULTS: We enrolled 671 eligible hospitalized patients with severe COVID-19 from 1 January to 23 February 2020, with a median age of 63 years. Clinical, laboratory, and treatment data were collected and compared between patients who died and survivors. Risk factors of death and myocardial injury were analysed using multivariable regression models. A total of 62 patients (9.2%) died, who more often had myocardial injury (75.8% vs. 9.7%; P < 0.001) than survivors. The area under the receiver operating characteristic curve of initial cardiac troponin I (cTnI) for predicting in-hospital mortality was 0.92 [95% confidence interval (CI), 0.87-0.96; sensitivity, 0.86; specificity, 0.86; P < 0.001]. The single cut-off point and high level of cTnI predicted risk of in-hospital death, hazard ratio (HR) was 4.56 (95% CI, 1.28-16.28; P = 0.019) and 1.25 (95% CI, 1.07-1.46; P = 0.004), respectively. In multivariable logistic regression, senior age, comorbidities (e.g. hypertension, coronary heart disease, chronic renal failure, and chronic obstructive pulmonary disease), and high level of C-reactive protein were predictors of myocardial injury. CONCLUSION: The risk of in-hospital death among patients with severe COVID-19 can be predicted by markers of myocardial injury, and was significantly associated with senior age, inflammatory response, and cardiovascular comorbidities.\n",
      "Objective: To explore the clinical characteristics and prognosis of the new coronavirus 2019-nCoV patients combined with cardiovascular disease (CVD). Methods: A retrospective analysis was performed on 112 COVID-19 patients with CVD admitted to the western district of Union Hospital in Wuhan, from January 20, 2020 to February 15, 2020. They were divided into critical group (ICU, n=16) and general group (n=96) according to the severity of the disease and patients were followed up to the clinical endpoint. The observation indicators included total blood count, C-reactive protein (CRP), arterial blood gas analysis, myocardial injury markers, coagulation function, liver and kidney function, electrolyte, procalcitonin (PCT), B-type natriuretic peptide (BNP), blood lipid, pulmonary CT and pathogen detection. Results: Compared with the general group, the lymphocyte count (0.74×10(9) (0.34×10(9), 0.94×10(9))/L vs. 0.99×10(9) (0.71×10(9), 1.29×10(9))/L, P=0.03) was extremely lower in the critical group, CRP (106.98 (81.57, 135.76) mg/L vs. 34.34 (9.55,76.54) mg/L, P<0.001) and PCT (0.20 (0.15,0.48) μg/L vs. 0.11 (0.06,0.20)μg/L, P<0.001) were significantly higher in the critical group. The BMI of the critical group was significantly higher than that of the general group (25.5 (23.0, 27.5) kg/m(2) vs. 22.0 (20.0, 24.0) kg/m(2), P=0.003). Patients were further divided into non-survivor group (17, 15.18%) group and survivor group (95, 84.82%). Among the non-survivors, there were 88.24% (15/17) patients with BMI> 25 kg/m(2), which was significantly higher than that of survivors (18.95% (18/95), P<0.001). Compared with the survived patients, oxygenation index (130 (102, 415) vs. 434 (410, 444), P<0.001) was significantly lower and lactic acid (1.70 (1.30, 3.00) mmol/L vs. 1.20 (1.10, 1.60) mmol/L, P<0.001) was significantly higher in the non-survivors. There was no significant difference in the proportion of ACEI/ARB medication between the critical group and the general group or between non-survivors and survivors (all P>0.05). Conclusion: COVID-19 patients combined with CVD are associated with a higher risk of mortality. Critical patients are characterized with lower lymphocyte counts. Higher BMI are more often seen in critical patients and non-survivor. ACEI/ARB use does not affect the morbidity and mortality of COVID-19 combined with CVD. Aggravating causes of death include fulminant inflammation, lactic acid accumulation and thrombotic events.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 4fqsx5y5\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the mortality rates overall and in specific populations\n",
      "what causes death from Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "OBJECTIVE Diabetes is common in COVID-19 patients and associated with unfavorable outcomes. We aimed to describe the characteristics and outcomes and to analyze the risk factors for in-hospital mortality of COVID-19 patients with diabetes. RESEARCH DESIGN AND METHODS This two-center retrospective study was performed at two tertiary hospitals in Wuhan, China. Confirmed COVID-19 patients with diabetes (N = 153) who were discharged or died from 1 January 2020 to 8 March 2020 were identified. One sex- and age-matched COVID-19 patient without diabetes was randomly selected for each patient with diabetes. Demographic, clinical, and laboratory data were abstracted. Cox proportional hazards regression analyses were performed to identify the risk factors associated with the mortality in these patients. RESULTS Of 1,561 COVID-19 patients, 153 (9.8%) had diabetes, with a median age of 64.0 (interquartile range 56.0-72.0) years. A higher proportion of intensive care unit admission (17.6% vs. 7.8%, P = 0.01) and more fatal cases (20.3% vs. 10.5%, P = 0.017) were identified in COVID-19 patients with diabetes than in the matched patients. Multivariable Cox regression analyses of these 306 patients showed that hypertension (hazard ratio [HR] 2.50, 95% CI 1.30-4.78), cardiovascular disease (HR 2.24, 95% CI 1.19-4.23), and chronic pulmonary disease (HR 2.51, 95% CI 1.07-5.90) were independently associated with in-hospital death. Diabetes (HR 1.58, 95% CI 0.84-2.99) was not statistically significantly associated with in-hospital death after adjustment. Among patients with diabetes, nonsurvivors were older (76.0 vs. 63.0 years), most were male (71.0% vs. 29.0%), and were more likely to have underlying hypertension (83.9% vs. 50.0%) and cardiovascular disease (45.2% vs. 14.8%) (all P values <0.05). Age ≥70 years (HR 2.39, 95% CI 1.03-5.56) and hypertension (HR 3.10, 95% CI 1.14-8.44) were independent risk factors for in-hospital death of patients with diabetes. CONCLUSIONS COVID-19 patients with diabetes had worse outcomes compared with the sex- and age-matched patients without diabetes. Older age and comorbid hypertension independently contributed to in-hospital death of patients with diabetes.\n",
      "OBJECTIVE: Diabetes is common in COVID-19 patients and associated with unfavorable outcomes. We aimed to describe the characteristics and outcomes and to analyze the risk factors for in-hospital mortality of COVID-19 patients with diabetes. RESEARCH DESIGN AND METHODS: This two-center retrospective study was performed at two tertiary hospitals in Wuhan, China. Confirmed COVID-19 patients with diabetes (N = 153) who were discharged or died from 1 January 2020 to 8 March 2020 were identified. One sex- and age-matched COVID-19 patient without diabetes was randomly selected for each patient with diabetes. Demographic, clinical, and laboratory data were abstracted. Cox proportional hazards regression analyses were performed to identify the risk factors associated with the mortality in these patients. RESULTS: Of 1,561 COVID-19 patients, 153 (9.8%) had diabetes, with a median age of 64.0 (interquartile range 56.0-72.0) years. A higher proportion of intensive care unit admission (17.6% vs. 7.8%, P = 0.01) and more fatal cases (20.3% vs. 10.5%, P = 0.017) were identified in COVID-19 patients with diabetes than in the matched patients. Multivariable Cox regression analyses of these 306 patients showed that hypertension (hazard ratio [HR] 2.50, 95% CI 1.30-4.78), cardiovascular disease (HR 2.24, 95% CI 1.19-4.23), and chronic pulmonary disease (HR 2.51, 95% CI 1.07-5.90) were independently associated with in-hospital death. Diabetes (HR 1.58, 95% CI 0.84-2.99) was not statistically significantly associated with in-hospital death after adjustment. Among patients with diabetes, nonsurvivors were older (76.0 vs. 63.0 years), most were male (71.0% vs. 29.0%), and they were more likely to have underlying hypertension (83.9% vs. 50.0%) and cardiovascular disease (45.2% vs. 14.8%) (all P values <0.05). Age &#8805;70 years (HR 2.39, 95% CI 1.03-5.56) and hypertension (HR 3.10, 95% CI 1.14-8.44) were independent risk factors for in-hospital death of patients with diabetes. CONCLUSIONS: COVID-19 patients with diabetes had worse outcomes compared with the sex- and age-matched patients without diabetes. Older age and comorbid hypertension independently contributed to in-hospital death of patients with diabetes.\n",
      "Abstract The 2019 novel coronavirus disease (COVID-19) emerged in Wuhan, China, and was characterized as a pandemic by the World Health Organization. Diabetes is an established risk associated with poor clinical outcomes, but the association of diabetes with COVID-19 has not been reported yet. Methods In this cohort study, we retrospectively reviewed 258 consecutive hospitalized COVID-19 patients with or without diabetes at the West Court of Union Hospital in Wuhan, China, recruited from January 29 to February 12, 2020. The clinical features, treatment strategies and prognosis data were collected and analyzed. Prognosis was followed up until March 12, 2020. Results Of the 258 hospitalized patients (63 with diabetes) with COVID-19, the median age was 64 years (range 23-91), and 138 (53.5%) were male. Common symptoms included fever (82.2%), dry cough (67.1%), polypnea (48.1%), and fatigue (38%). Patients with diabetes had significantly higher leucocyte and neutrophil counts, and higher levels of fasting blood glucose, serum creatinine, urea nitrogen and creatine kinase isoenzyme MB at admission compared with those without diabetes. COVID-19 patients with diabetes were more likely to develop severe or critical disease conditions with more complications, and had higher incidence rates of antibiotic therapy, non-invasive and invasive mechanical ventilation, and death (11.1% vs. 4.1%). Cox proportional hazard model showed that diabetes (adjusted hazard ratio [aHR]=3.64; 95% confidence interval [CI]: 1.09, 12.21) and fasting blood glucose (aHR=1.19; 95% CI: 1.08, 1.31) were associated with the fatality due to COVID-19, adjusting for potential confounders. Conclusions Diabetes mellitus is associated with increased disease severity and a higher risk of mortality in patients with COVID-19.\n",
      "AIMS: The 2019 novel coronavirus disease (COVID-19) emerged in Wuhan, China, and was characterized as a pandemic by the World Health Organization. Diabetes is an established risk associated with poor clinical outcomes, but the association of diabetes with COVID-19 has not been reported yet. METHODS: In this cohort study, we retrospectively reviewed 258 consecutive hospitalized COVID-19 patients with or without diabetes at the West Court of Union Hospital in Wuhan, China, recruited from January 29 to February 12, 2020. The clinical features, treatment strategies and prognosis data were collected and analyzed. Prognosis was followed up until March 12, 2020. RESULTS: Of the 258 hospitalized patients (63 with diabetes) with COVID-19, the median age was 64 years (range 23-91), and 138 (53.5%) were male. Common symptoms included fever (82.2%), dry cough (67.1%), polypnea (48.1%), and fatigue (38%). Patients with diabetes had significantly higher leucocyte and neutrophil counts, and higher levels of fasting blood glucose, serum creatinine, urea nitrogen and creatine kinase isoenzyme MB at admission compared with those without diabetes. COVID-19 patients with diabetes were more likely to develop severe or critical disease conditions with more complications, and had higher incidence rates of antibiotic therapy, non-invasive and invasive mechanical ventilation, and death (11.1% vs. 4.1%). Cox proportional hazard model showed that diabetes (adjusted hazard ratio [aHR] = 3.64; 95% confidence interval [CI]: 1.09, 12.21) and fasting blood glucose (aHR = 1.19; 95% CI: 1.08, 1.31) were associated with the fatality due to COVID-19, adjusting for potential confounders. CONCLUSIONS: Diabetes mellitus is associated with increased disease severity and a higher risk of mortality in patients with COVID-19.\n",
      "Background The 2019 novel coronavirus disease (COVID-19) emerged in Wuhan, Hubei province, China, and was characterized as pandemic by the World Health Organization. Diabetes mellitus is an established risk factor for poor clinical outcomes, but the association of diabetes with the prognosis of COVID-19 have not been reported yet. Methods In this cohort study, we retrospectively reviewed 258 consecutive hospitalized COVID-19 patients with or without diabetes at the West Court of Union Hospital of Huazhong University of Science and Technology in Wuhan, China, recruited from January 29 to February 12, 2020. The cases were confirmed by real-time PCR and the demographic, clinical, laboratory, radiological, and treatment data were collected and analyzed. Prognosis was defined as hospitalization, discharged survivor and death, which was followed up until March 12, 2020. Results Of the 258 hospitalized patients (63 with diabetes) with COVID-19, the median age was 64 years (range 23-91), and 138 (53.5%) were male. No significant differences in age and sex were identified between patients with and without diabetes. Common symptoms included fever (82.2%), dry cough (67.1%), polypnea (48.1%), and fatigue (38%). Patients with diabetes had significantly higher leucocyte and neutrophil counts, and higher levels of fasting blood glucose, serum creatinine, urea nitrogen and creatine kinase isoenzyme MB at admission compared with those without diabetes. COVID-19 patients with diabetes were more likely to develop severe or critical disease condition with more complications at presentation, and had higher incidence rates of antibiotic therapy, non-invasive and invasive mechanical ventilation, and death (11.1% vs. 4.1%). Cox proportional hazard model showed that diabetes (adjusted hazard ratio [aHR]=3.64; 95% confidence interval [CI]: 1.09, 12.21) and fasting blood glucose (aHR=1.19; 95% CI: 1.08, 1.31) were associated with the fatality of COVID-19, adjusting for potential confounders. Conclusions Diabetes mellitus is associated with greater disease severity and a higher risk of mortality in patients with COVID-19. Primary and secondary prevention strategies are needed for COVID-19 patients with diabetes.\n",
      "BACKGROUND: Coronavirus disease 2019 (COVID-19) is an emerging infectious disease, first appeared in Wuhan, China, and quickly spread throughout the world. We aimed to understand the relationship between diabetes mellitus and the prognosis of COVID-19. METHODS: Demographic, clinical, laboratory, radiologic, treatments, complications, and clinical outcomes data were extracted from electronic medical records and compared between diabetes (n=84) and non-diabetes (n=500) groups. Kaplan-Meier method and multivariate Cox analysis were applied to determine the risk factors for the prognosis of COVID-19. RESULTS: Compared to non-diabetic patients, diabetic patients had higher levels of neutrophils (p = 0.014), c-reactive protein (p = 0.008), procalcitonin (p < 0.01), and D-dimer (p = 0.033), and lower levels of lymphocytes (p = 0.032) and albumin (p = 0.035). Furthermore, diabetic patients had a significant higher incidence of bilateral pneumonia (86.9%, p = 0.020). In terms of complications and clinical outcomes, the incidence of respiratory failure (36.9% vs. 24.2%, p = 0.022), acute cardiac injury (47.4% vs. 21.2%, p < 0.01) and death (20.2% vs. 8.0%, p = 0.001) in the diabetes group was significantly higher than that in non-diabetes group. Kaplan-Meier survival curve showed that COVID-19 patients with diabetes had a shorter overall survival time. Multivariate Cox analysis indicated that diabetes (HR 2.180, p = 0.031) was an independent risk factor for COVID-19 prognosis. In subgroup analysis, we divided diabetic patients into insulin required and non-insulin required groups according to whether they needed insulin, and found that diabetic patients requiring insulin may have a higher risk of disease progression and worse prognosis after the infection of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). CONCLUSIONS: Diabetes is an independent risk factor for the prognosis of COVID-19. More attention should be paid to the prevention and treatment for diabetic patients, especially those who require insulin therapy.\n",
      "AIM: The novel coronavirus infection (COVID-19), now a worldwide public health concern is associated with varied fatality. Patients with chronic underlying conditions like diabetes and hypertension have shown worst outcomes. The understanding of the association might be helpful in early vigilant monitoring and better management of COVID-19 patients at high risk. The aim of the meta-analysis was to assess the association of diabetes and hypertension with severity of disease. METHODS: A literature search was conducted using the databases PubMed and Cochrane until March 31, 2020. Seven studies were included in the meta- analysis, including 2018 CIVID-19 patients. RESULTS: Diabetes was lower in the survivors (OR: 0.56; 95%CI: 0.35-0.90; p=0.017; I2: 0.0%) and non-severe (OR: 1.66; 95%CI: 1.20-2.30; p=0.002; I2: 0.0%) patients. No association of diabetes was found with ICU care. Hypertension was positively associated with death (OR: 0.49; 95%CI: 0.34-0.73; p=0.000; I2: 0.0%), ICU care (OR: 0.42; 95%CI: 0.22-0.81; p=0.009; I2: 0.0%) and severity (OR: 2.69; 95%CI: 1.27-5.73; p=0.01; I2: 52.4%). CONCLUSIONS: Our findings suggest that diabetes and hypertension have a negative effect on health status of COVID-19 patients. However, large prevalence studies demonstrating the consequences of comorbid diabetes and hypertension are urgently needed to understand the magnitude of these vexatious comorbidities.\n",
      "Background: The coronavirus disease 2019 (Covid-19) spreads rapidly around the world. Objective: To evaluate the association between comorbidities and the risk of death in patients with COVID-19, and to further explore potential sex-specific differences. Methods: We analyzed the data from 18,465 laboratory-confirmed cases that completed an epidemiological investigation in Hubei Province as of February 27, 2020. Information on death was obtained from the Infectious Disease Information System. The Cox proportional hazards model was used to estimate the association between comorbidities and the risk of death in patients with COVID-19. Results: The median age for COVID-19 patients was 50.5 years. 8828(47.81%) patients were females. Severe cases accounted for 20.11% of the study population. As of March 7, 2020, a total of 919 cases deceased from COVID-19 for a fatality rate of 4.98%. Hypertension (13.87%), diabetes (5.53%), and cardiovascular and cerebrovascular diseases CBVDs (4.45%) were the most prevalent comorbidities, and 27.37% of patients with COVID-19 reported having at least one comorbidity. After adjustment for age, gender, address, and clinical severity, patients with hypertension (HR 1.55, 95%CI 1.35-1.78), diabetes (HR 1.35, 95%CI 1.13-1.62), CBVDs (HR 1.70, 95%CI 1.43-2.02), chronic kidney diseases (HR 2.09, 95%CI 1.47-2.98), and at least two comorbidities (HR 1.84, 95%CI 1.55-2.18) had significant increased risks of death. And the association between diabetes and the risk of death from COVID-19 was prominent in women (HR 1.69, 95%CI 1.27-2.25) than in men (HR 1.16, 95%CI 0.91-1.46) (P for interaction = 0.036). Conclusion: Among laboratory-confirmed cases of COVID-19 in Hubei province, China, patients with hypertension, diabetes, CBVDs, chronic kidney diseases were significantly associated with increased risk of death. The association between diabetes and the risk of death tended to be stronger in women than in men. Clinicians should increase their awareness of the increased risk of death in COVID-19 patients with comorbidities.\n",
      "CONTEXT: No studies evaluated the prospective association between hyperglycemia assessed by laboratory measurements and the risk of mortality among patients with coronavirus disease 2019 (COVID‐19). OBJECTIVE: We aimed to evaluate the association between different degrees of hyperglycemia and the risk of all‐cause mortality among hospitalized patients with COVID‐19. DESIGN: A retrospective study. SETTING: Union Hospital in Wuhan, China. PARTICIPANTS: 453 patients were admitted to the hospital with laboratory‐confirmed SARS‐Cov‐2 infection from January 22, 2020 to March 17, 2020. MAIN OUTCOMES AND MEASURES: Patients were classified into four categories: normal glucose, hyperglycemia (fasting glucose 5.6–6.9 mmol/L and/or HbA1c 5.7–6.4%), newly diagnosed diabetes (fasting glucose ≥7 mmol/L and/or HbA1c ≥6.5%), and known diabetes. The major outcomes included in‐hospital mortality, intensive care unit (ICU) admission, and invasive mechanical ventilation (IMV). RESULTS: Patients with newly diagnosed diabetes had the highest percentage to be admitted to the ICU (11.7%) and require IMV (11.7%), followed by patients with known diabetes (4.1%; 9.2%) and patients with hyperglycemia (6.2%; 4.7%), compared with patients with normal glucose (1.5%; 2.3%), respectively. The multivariable‐adjusted hazard ratios of mortality among COVID‐19 patients with normal glucose, hyperglycemia, newly diagnosed diabetes, and known diabetes were 1.00, 3.29 (95% confidence interval [CI] 0.65–16.6), 9.42 (95% CI 2.18–40.7), and 4.63 (95% CI 1.02–21.0), respectively. CONCLUSION: We firstly showed that COVID‐19 patients with newly diagnosed diabetes had the highest risk of all‐cause mortality compared with COVID‐19 patients with known diabetes, hyperglycemia and normal glucose. Patients with COVID‐19 need to be under surveillance for blood glucose screening. This article is protected by copyright. All rights reserved.\n",
      "BACKGROUND: Many studies on COVID-19 have reported diabetes to be associated with severe disease and mortality, however, the data is conflicting. The objectives of this meta-analysis were to explore the relationship between diabetes and COVID-19 mortality and severity, and to determine the prevalence of diabetes in patients with COVID-19. METHODS: We searched the PubMed for case-control studies in English, published between Jan 1 and Apr 22, 2020, that had data on diabetes in patients with COVID-19. The frequency of diabetes was compared between patients with and without the composite endpoint of mortality or severity. Random effects model was used with odds ratio as the effect size. We also determined the pooled prevalence of diabetes in patients with COVID-19. Heterogeneity and publication bias were taken care by meta-regression, sub-group analyses, and trim and fill methods. RESULTS: We included 33 studies (16,003 patients) and found diabetes to be significantly associated with mortality of COVID-19 with a pooled odds ratio of 1.90 (95% CI: 1.37-2.64; p < 0.01). Diabetes was also associated with severe COVID-19 with a pooled odds ratio of 2.75 (95% CI: 2.09-3.62; p < 0.01). The combined corrected pooled odds ratio of mortality or severity was 2.16 (95% CI: 1.74-2.68; p < 0.01). The pooled prevalence of diabetes in patients with COVID-19 was 9.8% (95% CI: 8.7%-10.9%) (after adjusting for heterogeneity). CONCLUSIONS: Diabetes in patients with COVID-19 is associated with a two-fold increase in mortality as well as severity of COVID-19, as compared to non-diabetics. Further studies on the pathogenic mechanisms and therapeutic implications need to be done.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: u6uqroi0\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "what are the mortality rates overall and in specific populations\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Abstract Background and aims Diabetes Mellitus (DM) is chronic conditions with devastating multi-systemic complication and may be associated with severe form of Coronavirus Disease 2019 (COVID-19). We conducted a systematic review and meta-analysis in order to investigate the association between DM and poor outcome in patients with COVID-19 pneumonia. Methods Systematic literature search was performed from several electronic databases on subjects that assess DM and outcome in COVID-19 pneumonia. The outcome of interest was composite poor outcome, including mortality, severe COVID-19, acute respiratory distress syndrome (ARDS), need for intensive care unit (ICU) care, and disease progression. Results There were a total of 6452 patients from 30 studies. Meta-analysis showed that DM was associated with composite poor outcome (RR 2.38 [1.88, 3.03], p < 0.001; I2: 62%) and its subgroup which comprised of mortality (RR 2.12 [1.44, 3.11], p < 0.001; I2: 72%), severe COVID-19 (RR 2.45 [1.79, 3.35], p < 0.001; I2: 45%), ARDS (RR 4.64 [1.86, 11.58], p = 0.001; I2: 9%), and disease progression (RR 3.31 [1.08, 10.14], p = 0.04; I2: 0%). Meta-regression showed that the association with composite poor outcome was influenced by age (p = 0.003) and hypertension (p < 0.001). Subgroup analysis showed that the association was weaker in studies with median age ≥55 years-old (RR 1.92) compared to <55 years-old (RR 3.48), and in prevalence of hypertension ≥25% (RR 1.93) compared to <25% (RR 3.06). Subgroup analysis on median age <55 years-old and prevalence of hypertension <25% showed strong association (RR 3.33) Conclusion DM was associated with mortality, severe COVID-19, ARDS, and disease progression in patients with COVID-19.\n",
      "BACKGROUND AND AIMS: Diabetes Mellitus (DM) is chronic conditions with devastating multi-systemic complication and may be associated with severe form of Coronavirus Disease 2019 (COVID-19). We conducted a systematic review and meta-analysis in order to investigate the association between DM and poor outcome in patients with COVID-19 pneumonia. METHODS: Systematic literature search was performed from several electronic databases on subjects that assess DM and outcome in COVID-19 pneumonia. The outcome of interest was composite poor outcome, including mortality, severe COVID-19, acute respiratory distress syndrome (ARDS), need for intensive care unit (ICU) care, and disease progression. RESULTS: There were a total of 6452 patients from 30 studies. Meta-analysis showed that DM was associated with composite poor outcome (RR 2.38 [1.88, 3.03], p < 0.001; I2: 62%) and its subgroup which comprised of mortality (RR 2.12 [1.44, 3.11], p < 0.001; I2: 72%), severe COVID-19 (RR 2.45 [1.79, 3.35], p < 0.001; I2: 45%), ARDS (RR 4.64 [1.86, 11.58], p = 0.001; I2: 9%), and disease progression (RR 3.31 [1.08, 10.14], p = 0.04; I2: 0%). Meta-regression showed that the association with composite poor outcome was influenced by age (p = 0.003) and hypertension (p < 0.001). Subgroup analysis showed that the association was weaker in studies with median age &#8805;55 years-old (RR 1.92) compared to <55 years-old (RR 3.48), and in prevalence of hypertension &#8805;25% (RR 1.93) compared to <25% (RR 3.06). Subgroup analysis on median age <55 years-old and prevalence of hypertension <25% showed strong association (RR 3.33) CONCLUSION: DM was associated with mortality, severe COVID-19, ARDS, and disease progression in patients with COVID-19.\n",
      "BACKGROUND: Many studies on COVID-19 have reported diabetes to be associated with severe disease and mortality, however, the data is conflicting. The objectives of this meta-analysis were to explore the relationship between diabetes and COVID-19 mortality and severity, and to determine the prevalence of diabetes in patients with COVID-19. METHODS: We searched the PubMed for case-control studies in English, published between Jan 1 and Apr 22, 2020, that had data on diabetes in patients with COVID-19. The frequency of diabetes was compared between patients with and without the composite endpoint of mortality or severity. Random effects model was used with odds ratio as the effect size. We also determined the pooled prevalence of diabetes in patients with COVID-19. Heterogeneity and publication bias were taken care by meta-regression, sub-group analyses, and trim and fill methods. RESULTS: We included 33 studies (16,003 patients) and found diabetes to be significantly associated with mortality of COVID-19 with a pooled odds ratio of 1.90 (95% CI: 1.37-2.64; p < 0.01). Diabetes was also associated with severe COVID-19 with a pooled odds ratio of 2.75 (95% CI: 2.09-3.62; p < 0.01). The combined corrected pooled odds ratio of mortality or severity was 2.16 (95% CI: 1.74-2.68; p < 0.01). The pooled prevalence of diabetes in patients with COVID-19 was 9.8% (95% CI: 8.7%-10.9%) (after adjusting for heterogeneity). CONCLUSIONS: Diabetes in patients with COVID-19 is associated with a two-fold increase in mortality as well as severity of COVID-19, as compared to non-diabetics. Further studies on the pathogenic mechanisms and therapeutic implications need to be done.\n",
      "Abstract Background Many studies on COVID-19 have reported diabetes to be associated with severe disease and mortality, however, the data is conflicting. The objectives of this meta-analysis were to explore the relationship between diabetes and COVID-19 mortality and severity, and to determine the prevalence of diabetes in patients with COVID-19. Methods We searched the PubMed for case-control studies in English, published between Jan 1 and Apr 22, 2020, that had data on diabetes in patients with COVID-19. The frequency of diabetes was compared between patients with and without the composite endpoint of mortality or severity. Random effects model was used with odds ratio as the effect size. We also determined the pooled prevalence of diabetes in patients with COVID-19. Heterogeneity and publication bias were taken care by meta-regression, sub-group analyses, and trim and fill methods. Results We included 33 studies (16,003 patients) and found diabetes to be significantly associated with mortality of COVID-19 with a pooled odds ratio of 1.90 (95% CI: 1.37–2.64; p < 0.01). Diabetes was also associated with severe COVID-19 with a pooled odds ratio of 2.75 (95% CI: 2.09–3.62; p < 0.01). The combined corrected pooled odds ratio of mortality or severity was 2.16 (95% CI: 1.74–2.68; p < 0.01). The pooled prevalence of diabetes in patients with COVID-19 was 9.8% (95% CI: 8.7%–10.9%) (after adjusting for heterogeneity). Conclusions Diabetes in patients with COVID-19 is associated with a two-fold increase in mortality as well as severity of COVID-19, as compared to non-diabetics. Further studies on the pathogenic mechanisms and therapeutic implications need to be done.\n",
      "BACKGROUND: To determine the role of diabetes mellitus (DM) in the coronavirus disease 2019 (COVID-19), we explored the clinical characteristics of patients with DM and compared risk factors such as age, glycemic control, and medications to those without DM. METHODS: This was a retrospective cohort study of 117 confirmed patients with COVID-19 which conducted at a tertiary hospital in Daegu, South Korea. The primary outcome was defined as the severe and critical outcome (SCO), of which the composite outcomes of acute respiratory distress syndrome, septic shock, intensive care unit care, and 28-day mortality. We analyzed what clinical features and glycemic control-related factors affect the prognosis of COVID-19 in the DM group. RESULTS: After exclusion, 110 participants were finally included. DM patients (n=29) was older, and showed higher blood pressure compared to non-DM patients. DM group showed higher levels of inflammation-related biomarkers and severity score, and highly progressed to SCO. After adjustment with other risk factors, DM increased the risk of SCO (odds ratio [OR], 10.771; P<0.001). Among the DM patients, SCO was more prevalent in elderly patients of &#8805;70 years old and age was an independent risk factor for SCO in patients with DM (OR, 1.175; P=0.014), while glycemic control was not. The use of medication did not affect the SCO, but the renin-angiotensin system inhibitors showed protective effects against acute cardiac injury (OR, 0.048; P=0.045). CONCLUSION: The COVID-19 patients with DM had higher severity and resulted in SCO. Intensive and aggressive monitoring of COVID-19 clinical outcomes in DM group, especially in elderly patients is warranted.\n",
      "BACKGROUND: To determine the role of diabetes mellitus (DM) in the coronavirus disease 2019 (COVID-19), we explored the clinical characteristics of patients with DM and compared risk factors such as age, glycemic control, and medications to those without DM. METHODS: This was a retrospective cohort study of 117 confirmed patients with COVID-19 which conducted at a tertiary hospital in Daegu, South Korea. The primary outcome was defined as the severe and critical outcome (SCO), of which the composite outcomes of acute respiratory distress syndrome, septic shock, intensive care unit care, and 28-day mortality. We analyzed what clinical features and glycemic control-related factors affect the prognosis of COVID-19 in the DM group. RESULTS: After exclusion, 110 participants were finally included. DM patients (n=29) was older, and showed higher blood pressure compared to non-DM patients. DM group showed higher levels of inflammation-related biomarkers and severity score, and highly progressed to SCO. After adjustment with other risk factors, DM increased the risk of SCO (odds ratio [OR], 10.771; P<0.001). Among the DM patients, SCO was more prevalent in elderly patients of ≥70 years old and age was an independent risk factor for SCO in patients with DM (OR, 1.175; P=0.014), while glycemic control was not. The use of medication did not affect the SCO, but the renin-angiotensin system inhibitors showed protective effects against acute cardiac injury (OR, 0.048; P=0.045). CONCLUSION: The COVID-19 patients with DM had higher severity and resulted in SCO. Intensive and aggressive monitoring of COVID-19 clinical outcomes in DM group, especially in elderly patients is warranted.\n",
      "AIMS: Diabetes mellitus is one of the most common comorbidities in Coronavirus disease 2019 (COVID‐19) patients. The objective of this study was to evaluate the influences of diabetes mellitus on the severity and fatality of SARS‐CoV‐2 infection. MATERIALS AND METHODS: Medical records of 66 hospitalized COVID‐19 patients were collected and classified into non‐severe (mild/moderate cases) and severe (severe/critical cases) groups, respectively. Logistic regression analysis was used to estimate the risk of severe COVID‐19 (severe/critical infection). In addition, a meta‐analysis including published studies reported the impacts of diabetes mellitus on severity and fatality of COVID‐19, and our current study was conducted using fixed‐effects models. RESULTS: There were 22 diabetic and 44 non‐diabetic cases among the 66 hospitalized COVID‐19 patients. As the results shown, seven cases (31.82%) were diagnosed as severe COVID‐19 in diabetic patients, which was significantly higher than that in non‐diabetic group (4/44, 9.09%, P = 0.033). After adjustment for age and gender, the results showed that diabetes mellitus was significantly associated with COVID‐19 severity (OR: 5.29, 95% CI: 1.07–26.02). A meta‐analysis further confirmed the positive association between diabetes mellitus and COVID‐19 severity (pooled OR = 2.58, 95% CI: 1.93–3.45). Moreover, the diabetic patients infected with SARS‐CoV‐2 showed to have 2.95‐fold higher risk of fatality compared to those patients without diabetes mellitus (95% CI: 1.93–4.53). CONCLUSIONS: Our findings provide new evidences that diabetes mellitus is associated with a higher risk of severity and fatality of COVID‐19. Therefore, intensive monitoring and antidiabetic therapy should be considered in diabetic patients with SARS‐CoV‐2 infection. This article is protected by copyright. All rights reserved.\n",
      "OBJECTIVE: To investigate the association between hypertension and outcome in patients with Coronavirus Disease 2019 (COVID-19) pneumonia. METHODS: We performed a systematic literature search from several databases on studies that assess hypertension and outcome in COVID-19. Composite of poor outcome, comprising of mortality, severe COVID-19, acute respiratory distress syndrome (ARDS), need for intensive care unit (ICU) care and disease progression were the outcomes of interest. RESULTS: A total of 6560 patients were pooled from 30 studies. Hypertension was associated with increased composite poor outcome (risk ratio (RR) 2.11 (95% confidence interval (CI) 1.85, 2.40), p < 0.001; I(2), 44%) and its sub-group, including mortality (RR 2.21 (1.74, 2.81), p < 0.001; I(2), 66%), severe COVID-19 (RR 2.04 (1.69, 2.47), p < 0.001; I(2) 31%), ARDS (RR 1.64 (1.11, 2.43), p = 0.01; I(2),0%, p = 0.35), ICU care (RR 2.11 (1.34, 3.33), p = 0.001; I(2) 18%, p = 0.30), and disease progression (RR 3.01 (1.51, 5.99), p = 0.002; I(2) 0%, p = 0.55). Meta-regression analysis showed that gender (p = 0.013) was a covariate that affects the association. The association was stronger in studies with a percentage of males < 55% compared to ⩾ 55% (RR 2.32 v. RR 1.79). CONCLUSION: Hypertension was associated with increased composite poor outcome, including mortality, severe COVID-19, ARDS, need for ICU care and disease progression in patients with COVID-19.\n",
      "Abstract Background The prognostic significance of diabetes mellitus (DM) in patients with coronavirus 2019 disease (COVID-19) remains unknown. Objectives To assess the risk of ICU admission and morality risk in diabetic COVID-19 patients. Study desing A database search was conducted to identify studies comparing diabetic COVID-19 patients hospitalized in intensive care unit (ICU) and those reporting the overall mortality of these patients published up to March 25, 2020 within MEDLINE, Scopus and Web of Science. Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) guidelines were followed in abstracting data and assessing validity. Quality assessment was performed using the Newcastle-Ottawa quality assessment scale. The main outcome was the risk of ICU admission in diabetic patients with COVID-19 infection while the second was the mortality risk in overall diabetic COVID-19 patients. Data were pooled using the Mantel-Haenszel random effects models with odds ratio (OR) as the effect measure with the related 95 % confidence interval (CI). Statistical heterogeneity between groups was measured using the Higgins I2 statistic. Results Among 1382 patients (mean age 51.5 years, 798 males), DM resulted to be the second more frequent comorbidities. Diabetic patients resulted to have a significant increased risk of ICU admission (OR: 2.79, 95 % CI 1.85–4.22, p < 0.0001, I2 = 46 %). In 471 patients (mean age 56.6 years, 294 males) analysed for the secondary outcome diabetic subjects resulted to be at higher mortality risk (OR 3.21, 95 % CI 1.82–5.64, p < 0.0001, I2 = 16 %). Conclusions Diabetic patients with COVID-19 patients are at higher risk of ICU admission and show an higher mortality risk.\n",
      "AIM: The novel coronavirus infection (COVID-19), now a worldwide public health concern is associated with varied fatality. Patients with chronic underlying conditions like diabetes and hypertension have shown worst outcomes. The understanding of the association might be helpful in early vigilant monitoring and better management of COVID-19 patients at high risk. The aim of the meta-analysis was to assess the association of diabetes and hypertension with severity of disease. METHODS: A literature search was conducted using the databases PubMed and Cochrane until March 31, 2020. Seven studies were included in the meta- analysis, including 2018 CIVID-19 patients. RESULTS: Diabetes was lower in the survivors (OR: 0.56; 95%CI: 0.35-0.90; p=0.017; I2: 0.0%) and non-severe (OR: 1.66; 95%CI: 1.20-2.30; p=0.002; I2: 0.0%) patients. No association of diabetes was found with ICU care. Hypertension was positively associated with death (OR: 0.49; 95%CI: 0.34-0.73; p=0.000; I2: 0.0%), ICU care (OR: 0.42; 95%CI: 0.22-0.81; p=0.009; I2: 0.0%) and severity (OR: 2.69; 95%CI: 1.27-5.73; p=0.01; I2: 52.4%). CONCLUSIONS: Our findings suggest that diabetes and hypertension have a negative effect on health status of COVID-19 patients. However, large prevalence studies demonstrating the consequences of comorbid diabetes and hypertension are urgently needed to understand the magnitude of these vexatious comorbidities.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 1vhxcbx7\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what causes death from Covid-19?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "The COVID-19 pandemic is the most significant public health issue in recent history. Its causal agent, SARS-CoV-2, has evolved rapidly since its first emergence in December 2019. Mutations in the viral genome have critical impacts on the adaptation of viral strains to the local environment, and may alter the characteristics of viral transmission, disease manifestation, and the efficacy of treatment and vaccination. Using the complete sequences of 1,932 SARS-CoV-2 genomes, we examined the genomic, geographic and temporal distributions of aged, new, and frequent mutations of SARS-CoV-2, and identified six phylogenetic clusters of the strains, which also exhibit a geographic preference in different continents. Mutations in the form of single nucleotide variations (SNVs) provide a direct interpretation for the six phylogenetic clusters. Linkage disequilibrium, haplotype structure, evolutionary process, global distribution of mutations unveiled a sketch of the mutational history. Additionally, we found a positive correlation between the average mutation count and case fatality, and this correlation had strengthened with time, suggesting an important role of SNVs on disease outcomes. This study suggests that SNVs may become an important consideration in virus detection, clinical treatment, drug design, and vaccine development to avoid target shifting, and that continued isolation and sequencing is a crucial component in the fight against this pandemic. Significance Statement Mutation is the driving force of evolution for viruses like SARS-CoV-2, the causal agent of COVID-19. In this study, we discovered that the genome of SARS-CoV-2 is changing rapidly from the originally isolated form. These mutations have been spreading around the world and caused more than 2.5 million of infected cases and 170 thousands of deaths. We found that fourteen frequent mutations identified in this study can characterize the six main clusters of SARS-CoV-2 strains. In addition, we found the mutation burden is positively correlated with the fatality of COVID-19 patients. Understanding mutations in the SARS-CoV-2 genome will provide useful insight for the design of treatment and vaccination.\n",
      "The COVID-19 pandemic is the most important public health threat in recent history. Here we study how its causal agent, SARS-CoV-2, has diversified genetically since its first emergence in December 2019. We have created a pipeline combining both phylogenetic and structural analysis to identify possible human-adaptation related mutations in a data set consisting of 4,894 SARS-CoV-2 complete genome sequences. Although the phylogenetic diversity of SARS-CoV-2 is low, the whole genome phylogenetic tree can be divided into five clusters/clades based on the tree topology and clustering of specific mutations, but its branches exhibit low genetic distance and bootstrap support values. We also identified 11 residues that are high-frequency substitutions, with four of them currently showing some signal for potential positive selection. These fast-evolving sites are in the non-structural proteins nsp2, nsp5 (3CL-protease), nsp6, nsp12 (polymerase) and nsp13 (helicase), in accessory proteins (ORF3a, ORF8) and in the structural proteins N and S. Temporal and spatial analysis of these potentially adaptive mutations revealed that the incidence of some of these sites was declining after having reached an (often local) peak, whereas the frequency of other sites is continually increasing and now exhibit a worldwide distribution. Structural analysis revealed that the mutations are located on the surface of the proteins that modulate biochemical properties. We speculate that this improves binding to cellular proteins and hence represents fine-tuning of adaptation to human cells. Our study has implications for the design of biochemical and clinical experiments to assess whether important properties of SARS-CoV-2 have changed during the epidemic.\n",
      "In late December 2019, an emerging viral infection COVID-19 was identified in Wuhan, China, and became a global pandemic. Characterization of the genetic variants of SARS-CoV-2 is crucial in following and evaluating it spread across countries. In this study, we collected and analyzed 3,067 SARS-CoV-2 genomes isolated from 55 countries during the first three months after the onset of this virus. Using comparative genomics analysis, we traced the profiles of the whole-genome mutations and compared the frequency of each mutation in the studied population. The accumulation of mutations during the epidemic period with their geographic locations was also monitored. The results showed 782 variant sites, of which 512 (65.47%) had a non-synonymous effect. Frequencies of mutated alleles revealed the presence of 38 recurrent non-synonymous mutations, including ten hotspot mutations with a prevalence higher than 0.10 in this population and distributed in six SARS-CoV-2 genes. The distribution of these recurrent mutations on the world map revealed certain genotypes specific to the geographic location. We also found co-occurring mutations resulting in the presence of several haplotypes. Moreover, evolution over time has shown a mechanism of mutation co-accumulation which might affect the severity and spread of the SARS-CoV-2. On the other hand, analysis of the selective pressure revealed the presence of negatively selected residues that could be taken into considerations as therapeutic targets We have also created an inclusive unified database (http://genoma.ma/covid-19/) that lists all of the genetic variants of the SARS-CoV-2 genomes found in this study with phylogeographic analysis around the world.\n",
      "The set of mutations observed at the outset of the SARS-CoV-2 pandemic may illuminate how the virus will adapt to humans as it continues to spread. Viruses are expected to quickly acquire beneficial mutations upon jumping to a new host species. Advantageous nucleotide substitutions can be identified by their parallel occurrence in multiple independent lineages and are likely to result in changes to protein sequences. Here we show that SARS-CoV-2 is acquiring mutations more slowly than expected for neutral evolution, suggesting purifying selection is the dominant mode of evolution during the initial phase of the pandemic. However, several parallel mutations arose in multiple independent lineages and may provide a fitness advantage over the ancestral genome. We propose plausible reasons for several of the most frequent mutations. The absence of mutations in other genome regions suggests essential components of SARS-CoV-2 that could be the target of drug development. Overall this study provides genomic insights into how SARS-CoV-2 has adapted and will continue to adapt to humans. SUMMARY In this study we sought signals of evolution to identify how the SARS-CoV-2 genome has adapted at the outset of the COVID-19 pandemic. We find that the genome is largely undergoing purifying selection that maintains its ancestral sequence. However, we identified multiple positions on the genome that appear to confer an adaptive advantage based on their repeated evolution in independent lineages. This information indicates how SARS-CoV-2 will evolve as it diversifies in an increasing number of hosts.\n",
      "The COVID-19 pandemic has spread across the globe at an alarming rate in the last four months. However, unlike any of the previous global outbreaks the availability of hundreds of SARS-CoV-2 sequences provides us with a unique opportunity to understand viral evolution in real time. We analysed 480 full-length (>29000 nt) sequences from the 1575 SARS-CoV-2 sequences available and identified 37 single-nucleotide substitutions occurring in >1% of the genomes. Majority of the substitutions were C to T or G to A. We identify C/Gs with an upstream TTT trinucleotide motif as hotspots for mutations in the SARS-CoV-2 genome. Interestingly, two of the 37 substitutions occur within highly conserved secondary structures in the 5’ and 3’ untranslated regions that are critical for the virus life cycle. Furthermore, clustering analysis revealed unique geographical distribution of SARS-CoV-2 variants defined by their mutation profile. Of note, we observed several co-occurring mutations that almost never occur individually. We define 3 mutually exclusive lineages (A1, B1 and C1) of SARS-CoV-2 which account for about three quarters of the genomes analysed. We identify lineage-defining leading mutations in the SARS-CoV-2 genome which precede the occurrence of sub-lineage defining trailing mutations. The identification of mutually exclusive lineage-defining mutations with geographically restricted patterns of distribution has potential implications for diagnosis, pathogenesis and vaccine design. Our work provides novel insights on the temporal evolution of SARS-CoV-2.\n",
      "The sudden outbreak of the severe acute respiratory syndrome-coronavirus (SARS-CoV-2) has spread globally with more than 1,300,000 patients diagnosed and a death toll of 70,000. Current genomic survey data suggest that single nucleotide variants (SNVs) are abundant. However, no mutation has been directly linked with functional changes in viral pathogenicity. Here we report functional characterizations of 11 patient-derived viral isolates, all of which have at least one mutation. Importantly, these viral isolates show significant variation in cytopathic effects and viral load, up to 270-fold differences, when infecting Vero-E6 cells. We observed intrapersonal variation and 6 different mutations in the spike glycoprotein (S protein), including 2 different SNVs that led to the same missense mutation. Therefore, we provide direct evidence that the SARS-CoV-2 has acquired mutations capable of substantially changing its pathogenicity.\n",
      "The COVID-19 pandemic has sparked an urgent need to uncover the underlying biology of this devastating disease. Though RNA viruses mutate more rapidly than DNA viruses, there are a relatively small number of single nucleotide polymorphisms (SNPs) that differentiate the main SARS-CoV-2 clades that have spread throughout the world. In this study, we investigated over 7,000 SARS-CoV-2 datasets to unveil both intrahost and interhost diversity. Our intrahost and interhost diversity analyses yielded three major observations. First, the mutational profile of SARS-CoV-2 highlights iSNV and SNP similarity, albeit with high variability in C>T changes. Second, iSNV and SNP patterns in SARS-CoV-2 are more similar to MERS-CoV than SARS-CoV-1. Third, a significant fraction of small indels fuel the genetic diversity of SARS-CoV-2. Altogether, our findings provide insight into SARS-CoV-2 genomic diversity, inform the design of detection tests, and highlight the potential of iSNVs for tracking the transmission of SARS-CoV-2.\n",
      "OBJECTIVES: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan, China, in December 2019 and has been rapidly spreading worldwide. Although the causal relationship among mutations and the features of SARS-CoV-2 such as rapid transmission, pathogenicity, and tropism, remains unclear, our results of genomic mutations in SARS-CoV-2 may help to interpret the interaction between genomic characterization in SARS-CoV-2 and infectivity with the host. METHODS: A total of 4,254 genomic sequences of SARS-CoV-2 were collected from the Global Initiative on Sharing all Influenza Data (GISAID). Multiple sequence alignment for phylogenetic analysis and comparative genomic approach for mutation analysis were conducted using Molecular Evolutionary Genetics Analysis (MEGA), and an in-house program based on Perl language, respectively. RESULTS: Phylogenetic analysis of SARS-CoV-2 strains indicated that there were 3 major clades including S, V, and G, and 2 subclades (G.1 and G.2). There were 767 types of synonymous and 1,352 types of non-synonymous mutation. ORF1a, ORF1b, S, and N genes were detected at high frequency, whereas ORF7b and E genes exhibited low frequency. In the receptor-binding domain (RBD) of the S gene, 11 non-synonymous mutations were observed in the region adjacent to the angiotensin converting enzyme 2 (ACE2) binding site. CONCLUSION: It has been reported that the rapid infectivity and transmission of SARS-CoV-2 associated with host receptor affinity are derived from several mutations in its genes. Without these genetic mutations to enhance evolutionary adaptation, species recognition, host receptor affinity, and pathogenicity, it would not survive. It is expected that our results could provide an important clue in understanding the genomic characteristics of SARS-CoV-2.\n",
      "We describe fifteen major mutation events from 2,058 high-quality SARS-CoV-2 genomes deposited up to March 31st, 2020. These events define five major clades (G, I, S, D and V) of globally-circulating viral populations, representing 85.7% of all sequenced cases, which we can identify using a 10 nucleotide genetic classifier or barcode. We applied this barcode to 4,000 additional genomes deposited between March 31st and April 15th and classified successfully 95.6% of the clades demonstrating the utility of this approach. An analysis of amino acid variation in SARS-CoV-2 ORFs provided evidence of substitution events in the viral proteins involved in both host-entry and genome replication. The systematic monitoring of dynamic changes in the SARS-CoV-2 genomes of circulating virus populations over time can guide therapeutic and prophylactic strategies to manage and contain the virus and, also, with available efficacious antivirals and vaccines, aid in the monitoring of circulating genetic diversity as we proceed towards elimination of the agent. The barcode will add the necessary genetic resolution to facilitate tracking and monitoring of infection clusters to distinguish imported and indigenous cases and thereby aid public health measures seeking to interrupt transmission chains without the requirement for real-time complete genomes sequencing.\n",
      "The recent pandemic of SARS-CoV-2 infection has affected more than 3.0 million people worldwide with more than 200 thousand reported deaths. The SARS-CoV-2 genome has the capability of gaining rapid mutations as the virus spreads. Whole-genome sequencing data offers a wide range of opportunities to study mutation dynamics. The advantage of an increasing amount of whole-genome sequence data of SARS-CoV-2 intrigued us to explore the mutation profile across the genome, to check the genome diversity, and to investigate the implications of those mutations in protein stability and viral transmission. We have identified frequently mutated residues by aligning ~660 SARS-CoV-2 genomes and validated in 10,000 datasets available in GISAID Nextstrain. We further evaluated the potential of these frequently mutated residues in protein structure stability of spike glycoprotein and their possible functional consequences in other proteins. Among the 11 genes, surface glycoprotein, nucleocapsid, ORF1ab, and ORF8 showed frequent mutations, while envelop, membrane, ORF6, ORF7a and ORF7b showed conservation in terms of amino acid substitutions. Combined analysis with the frequently mutated residues identified 20 viral variants, among which 12 specific combinations comprised more than 97% of the isolates considered for the analysis. Some of the mutations across different proteins showed co-occurrences, suggesting their structural and/or functional interaction among different SARS-COV-2 proteins, and their involvement in adaptability and viral transmission. Analysis of protein structure stability of surface glycoprotein mutants indicated the viability of specific variants and are more prone to be temporally and spatially distributed across the globe. A similar empirical analysis of other proteins indicated the existence of important functional implications of several variants. Identification of frequently mutated variants among COVID-19 patients might be useful for better clinical management, contact tracing, and containment of the disease.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 2qljx9cq\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "What vaccine candidates are being tested for Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Coronaviruses (CoVs) are a large family of viruses that cause illness ranging from the common cold to more severe diseases such as Severe Acute Respiratory Syndrome (SARS) and Middle East Respiratory Syndrome (MERS). Severe Acute Respiratory Syndrome coronavirus 2 (SARS-CoV-2) disease (COVID-19) has caused major public health crises. There have been more than 4,400,000 reported cases of COVID-2019 and more than 300,000 reported deaths to date (16/05/2020). SARS-CoV, MERS-CoV and SARS-CoV-2 have attracted widespread global attention due to their high infectivity and pathogenicity. To date, there is no specific treatment proven effective against these viral infectious diseases. Vaccination is considered one of the most effective strategies to prevent viral infections. Therefore, the development of effective vaccines against highly pathogenic coronaviruses is essential. In this review, we will briefly describe coronavirus vaccine design targets, summarize recent advances in the development of coronavirus vaccines, and highlight current adjuvants for improving the efficacy of coronavirus vaccines.\n",
      "Few human pathogens have been the focus of as much concentrated worldwide attention as Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV2), the cause of COVID-19. Its emergence into the human population and ensuing pandemic came on the heels of Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) and Middle East Respiratory Syndrome Coronavirus (MERS-CoV), two other highly pathogenic coronavirus spillovers, which collectively have reshaped our view of a virus family previously associated primarily with the common cold. It has placed intense pressure on the collective scientific community to develop therapeutics and vaccines, whose engineering relies on a detailed understanding of coronavirus biology. Here, we present the molecular virology of coronavirus infection, including its entry into cells, its remarkably sophisticated gene expression and replication mechanisms, its extensive remodeling of the intracellular environment and its multi-faceted immune evasion strategies. We highlight aspects of the viral lifecycle that may be amenable to antiviral targeting as well as key features of its biology that await discovery.\n",
      "Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the causative agent of coronavirus disease 2019 (COVID-19), has now become a serious global threat after inflicting more than 8 million infections and 425,000 deaths in less than 6 months. Currently, no definitive treatment or prevention therapy exists for COVID-19. The unprecedented rise of this pandemic has rapidly fueled research efforts to discover and develop new vaccines and treatment strategies against this novel coronavirus. While hundreds of vaccines/therapeutics are still in the preclinical or early stage of clinical development, a few of them have shown promising results in controlling the infection. Here, in this review, we discuss the promising vaccines and treatment options for COVID-19, their challenges, and potential alternative strategies.\n",
      "Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the causative agent of CoV disease 2019 (COVID-19) is a highly pathogenic and transmissible CoV that is presently plaguing the global human population and economy. No proven effective antiviral therapy or vaccine currently exists, and supportive care remains to be the cornerstone treatment. Through previous lessons learned from SARS-CoV-1 and MERS-CoV studies, scientific groups worldwide have rapidly expanded the knowledge pertaining to SARS-CoV-2 virology that includes in vitro and in vivo models for testing of antiviral therapies and randomized clinical trials. In the present narrative, we review SARS-CoV-2 virology, clinical features, pathophysiology, and animal models with a specific focus on the antiviral and adjunctive therapies currently being tested or that require testing in animal models and randomized clinical trials.\n",
      "Abstract The emergence of the strain of coronavirus SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) that causes corona virus disease 2019 (COVID-19) and its impact on in the world have made imperative progress to develop an effective and safe vaccine. Despite several measures undertaken, the spread of this virus is ongoing. So far, more than 1,560,000 cases and 1000,000 deaths occurred in the world. Efforts have been made to develop vaccines against human coronavirus (CoV) infections such as MERS and SARS. However, currently, no approved vaccine exists for these coronavirus strains. Such Previous research efforts to develop a coronavirus vaccine in the years following the 2003 pandemic have opened the door for the scientist to design a new vaccine for the COVID-19. Both SARS-CoV and SARS-CoV-2 has a high degree of genetic similarity and bind to the same host cell ACE2 receptor. By using different vaccine development platforms including whole virus vaccines, recombinant protein subunit vaccines, and nucleic acid vaccines several candidates displayed efficacy in vitro studies but few progressed to clinical trials. This review provides a brief introduction of the general features of SARS-CoV-2 and discusses the current progress of ongoing advances in designing vaccine development efforts to counter COVID-19.\n",
      "Coronavirus disease 2019 (COVID-19) is a kind of viral pneumonia which is caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). The emergence of SARS-CoV-2 has been marked as the third introduction of a highly pathogenic coronavirus into the human population after the severe acute respiratory syndrome coronavirus (SARS-CoV) and the Middle East respiratory syndrome coronavirus (MERS-CoV) in the twenty-first century. In this minireview, we provide a brief introduction of the general features of SARS-CoV-2 and discuss current knowledge of molecular immune pathogenesis, diagnosis and treatment of COVID-19 on the base of the present understanding of SARS-CoV and MERS-CoV infections, which may be helpful in offering novel insights and potential therapeutic targets for combating the SARS-CoV-2 infection.\n",
      "Coronavirus disease 2019 (COVID-19) caused by the novel coronavirus, also known as severe acute respiratory syndrome coronavirus 2 (SARS-Cov-2), has become a Public Health Emergency of International Concern. Due to the large infection population, broad transmissibility and high mortality, it is urgent to find out the efficient and specific methods to prevent and treat COVID-19. As biological products have broadly applied in the prevention and treatment of severe epidemic diseases, they are promising in blocking novel coronavirus infection. According to the research advances of severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS), we reviewed the potential application of biological products such as interferon, convalescent plasma, intestinal micro-ecological regulators, vaccines and therapeutic antibodies, etc. , on prevention and treatment of COVID-19. May this review be helpful for conquering COVID-19 in the near future.\n",
      "Coronavirus disease 2019 (COVID-19) caused by the novel coronavirus, also known as severe acute respiratory syndrome coronavirus 2 (SARS-Cov-2), has become a Public Health Emergency of International Concern. Due to the large infection population, broad transmissibility and high mortality, it is urgent to find out the efficient and specific methods to prevent and treat COVID-19. As biological products have broadly applied in the prevention and treatment of severe epidemic diseases, they are promising in blocking novel coronavirus infection. According to the research advances of severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS), we reviewed the potential application of biological products such as interferon, convalescent plasma, intestinal micro-ecological regulators, vaccines and therapeutic antibodies, etc. , on prevention and treatment of COVID-19. May this review be helpful for conquering COVID-19 in the near future.\n",
      "The emergence of the strain of coronavirus SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) and its impact on global health have made imperative the development of effective and safe vaccines for this lethal strain. SARS-CoV-2 now adds to the list of coronavirus diseases that have threatened global health, along with the SARS (severe acute respiratory syndrome) and MERS (Middle East respiratory syndrome) coronaviruses that emerged in 2002/2003 and 2012, respectively. As of April 2020, no vaccine is commercially available for these coronavirus strains. Nevertheless, the knowledge obtained from the vaccine development efforts for MERS and SARS can be of high value for COVID-19 (coronavirus disease 2019). Here, we review the past and ongoing vaccine development efforts for clinically relevant coronavirus strains with the intention that this information helps in the development of effective and safe vaccines for COVID-19. In addition, information from naturally exposed individuals and animal models to coronavirus strains is described for the same purpose of helping into the development of effective vaccines against COVID-19.\n",
      "The emergence of the strain of coronavirus SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) and its impact on global health have made imperative the development of effective and safe vaccines for this lethal strain. SARS-CoV-2 now adds to the list of coronavirus diseases that have threatened global health, along with the SARS (severe acute respiratory syndrome) and MERS (Middle East respiratory syndrome) coronaviruses that emerged in 2002/2003 and 2012, respectively. As of April 2020, no vaccine is commercially available for these coronavirus strains. Nevertheless, the knowledge obtained from the vaccine development efforts for MERS and SARS can be of high value for COVID-19 (coronavirus disease 2019). Here, we review the past and ongoing vaccine development efforts for clinically relevant coronavirus strains with the intention that this information helps in the development of effective and safe vaccines for COVID-19. In addition, information from naturally exposed individuals and animal models to coronavirus strains is described for the same purpose of helping into the development of effective vaccines against COVID-19.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: jsjxdmi2\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the initial symptoms of Covid-19?\n",
      "what is known about those infected with Covid-19 but are asymptomatic?\n",
      "what are the health outcomes for children who contract COVID-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Abstract Background: Most guidelines on COVID-19 published so far include recommendations for patients regardless of age. Clinicians need a more accurate understanding of the clinical characteristics of children with COVID-19. Methods: We searched studies reporting clinical characteristics in children with COVID-19 published until March 31, 2020. We screened the literature, extracted the data and evaluated the risk of bias and quality of evidence of the included studies. We combined some of the outcomes (symptoms) in a single-arm meta-analysis using a random-effects model. Results: Our search retrieved 49 studies, including 25 case reports, 23 case series and one cohort study, with a total of 1667 patients. Our meta-analysis showed that most children with COVID-19 have mild symptoms. Eighty-three percent of the children were within family clusters of cases, and 19% had no symptoms. At least 7% with digestive symptoms. The main symptoms of children were fever (48%, 95% confidence interval [CI]: 39%, 56%) and cough (39%, 95% CI: 30%, 48%). The lymphocyte count was below normal level in only 15% [95% CI: 8%, 22%] of children which is different from adult patients. 66% [95% CI: 55%, 77%] of children had abnormal findings in CT imaging. Conclusions: Most children with COVID-19 have only mild symptoms, and many children are asymptomatic. Fever and cough are the most common symptoms in children. Vomiting and diarrhea were not common in children. The lymphocyte count is usually within the normal range in children.\n",
      "Background Most guidelines on COVID-19 published so far include recommendations for patients regardless of age. Clinicians need a more accurate understanding of the clinical characteristics of children with COVID-19. Methods We searched studies reporting clinical characteristics in children with COVID-19 published until March 31, 2020. We screened the literature, extracted the data and evaluated the risk of bias and quality of evidence of the included studies. We combined some of the outcomes (symptoms) in a single-arm meta-analysis using a random-effects model. Results Our search retrieved 49 studies, including 25 case reports, 23 case series and one cohort study, with a total of 1,667 patients. Our meta-analysis showed that most children with COVID-19 have mild symptoms. Eighty-three percent of the children were within family clusters of cases, and 19% had no symptoms. At least 7% with digestive symptoms. The main symptoms of children were fever [48%, 95% confidence interval (CI): 39%, 56%] and cough (39%, 95% CI: 30%, 48%). The lymphocyte count was below normal level in only 15% (95% CI: 8%, 22%) of children which is different from adult patients. 66% (95% CI: 55%, 77%) of children had abnormal findings in CT imaging. Conclusions Most children with COVID-19 have only mild symptoms, and many children are asymptomatic. Fever and cough are the most common symptoms in children. Vomiting and diarrhea were not common in children. The lymphocyte count is usually within the normal range in children.\n",
      "BACKGROUND: The coronavirus disease 2019 (COVID-19) outbreak is an unprecedented global public health challenge, leading to thousands of deaths every day worldwide. Despite the epidemiological importance, clinical patterns of children with COVID-19 remain unclear. The aim of this study was to describe the clinical, laboratorial, and radiological characteristics of children with COVID-19. METHODS: The Medline database was searched between December 1st 2019 and April 6th 2020. No language restrictions were applied. Inclusion criteria were (a) studied patients younger than 18 years old; (b) presented original data from cases of COVID-19 confirmed by reverse-transcription polymerase chain reaction; and (c) contained descriptions of clinical manifestations, laboratory tests, or radiological examinations. RESULTS: A total of 38 studies (1124 cases) were included. From all the cases, 1117 had their severity classified: 14.2% were asymptomatic, 36.3% were mild, 46.0% were moderate, 2.1% were severe, and 1.2% were critical. The most prevalent symptom was fever (47.5%), followed by cough (41.5%), nasal symptoms (11.2%), diarrhea (8.1%), and nausea/vomiting (7.1%). One hundred forty-five (36.9%) children were diagnosed with pneumonia and 43 (10.9%) upper airway infections were reported. Reduced lymphocyte count was reported in 12.9% of cases. Abnormalities in computed tomography were reported in 63.0% of cases. The most prevalent abnormalities reported were ground-glass opacities, patchy shadows, and consolidations. Only one death was reported. CONCLUSIONS: Clinical manifestations of children with COVID-19 differ widely from adult cases. Fever and respiratory symptoms should not be considered a hallmark of COVID-19 in children.\n",
      "BACKGROUND AND OBJECTIVE Knowledge about COVID-19 in children is limited due to the paucity of reported data. The pediatric age group comprises only less than 5% of total COVID-19 worldwide, therefore, large studies in this population are unlikely in the immediate future. Hence, we planned to synthesize the current data that will help in a better understanding of COVID-19 in children. METHODS Four different electronic databases (MEDLINE, EMBASE, Web of Science, and CENTRAL) were searched for articles related to COVID-19 in the pediatric population. We included studies reporting disease characteristics and outcomes of COVID-19 in patients aged less than 19 years. We performed a random-effect meta-analysis to provide pooled estimates of various disease characteristics. RESULTS 27 studies (4857 patients) fulfilling the eligibility criteria were included in this systematic review, from a total of 883 records. About half of the patients had each of fever and cough, 11% (6-17%) had fast breathing, and 6-13% had gastrointestinal manifestations. Most of the patients had mild to moderate disease, and only 4% had a severe or critical illness. Leukopenia was the commonest reported laboratory abnormality. CONCLUSION Even among the symptomatic COVID-19 cases, severe manifestations are seen in very few children. Though fever and respiratory symptoms are most common, many children also have gastrointestinal manifestations.\n",
      "BACKGROUND: The coronavirus disease 2019 (COVID‐19) outbreak is an unprecedented global public health challenge, leading to thousands of deaths every day worldwide. Despite the epidemiological importance, clinical patterns of children with COVID‐19 remain unclear. The aim of this study was to describe the clinical, laboratorial, and radiological characteristics of children with COVID‐19. METHODS: The Medline database was searched between December 1st 2019 and April 6th 2020. No language restrictions were applied. Inclusion criteria were (a) studied patients younger than 18 years old; (b) presented original data from cases of COVID‐19 confirmed by reverse‐transcription polymerase chain reaction; and (c) contained descriptions of clinical manifestations, laboratory tests, or radiological examinations. RESULTS: A total of 38 studies (1124 cases) were included. From all the cases, 1117 had their severity classified: 14.2% were asymptomatic, 36.3% were mild, 46.0% were moderate, 2.1% were severe, and 1.2% were critical. The most prevalent symptom was fever (47.5%), followed by cough (41.5%), nasal symptoms (11.2%), diarrhea (8.1%), and nausea/vomiting (7.1%). One hundred forty‐five (36.9%) children were diagnosed with pneumonia and 43 (10.9%) upper airway infections were reported. Reduced lymphocyte count was reported in 12.9% of cases. Abnormalities in computed tomography were reported in 63.0% of cases. The most prevalent abnormalities reported were ground‐glass opacities, patchy shadows, and consolidations. Only one death was reported. CONCLUSIONS: Clinical manifestations of children with COVID‐19 differ widely from adult cases. Fever and respiratory symptoms should not be considered a hallmark of COVID‐19 in children.\n",
      "BACKGROUND AND OBJECTIVE: Knowledge about COVID-19 in children is limited due to the paucity of reported data. The pediatric age group comprises only less than 5% of total COVID-19 worldwide, therefore, large studies in this population are unlikely in the immediate future. Hence, we planned to synthesize the current data that will help in a better understanding of COVID-19 in children. METHODS: Four different electronic databases (MEDLINE, EMBASE, Web of Science, and CENTRAL) were searched for articles related to COVID-19 in the pediatric population. We included studies reporting disease characteristics and outcomes of COVID-19 in patients aged less than 19 years. We performed a random-effect meta-analysis to provide pooled estimates of various disease characteristics. RESULTS: 27 studies (4857 patients) fulfilling the eligibility criteria were included in this systematic review, from a total of 883 records. About half of the patients had each of fever and cough, 11% (6-17%) had fast breathing, and 6-13% had gastrointestinal manifestations. Most of the patients had mild to moderate disease, and only 4% had a severe or critical illness. Leukopenia was the commonest reported laboratory abnormality. CONCLUSION: Even among the symptomatic COVID-19 cases, severe manifestations are seen in very few children. Though fever and respiratory symptoms are most common, many children also have gastrointestinal manifestations.\n",
      "AIM: To summarize what we know so far about coronavirus disease (COVID‐19) in children. METHOD: We searched PubMed, Scientific Electronic Library Online, and Latin American and Caribbean Center on Health Sciences Information from 1 January 2020 to 4 May 2020. We selected randomized trials, observational studies, case series or case reports, and research letters of children ages birth to 18 years with laboratory‐confirmed COVID‐19. We conducted random‐effects meta‐analyses to calculate the weighted mean prevalence and 95% confidence interval (CI) or the weighted average means and 95% CI. RESULT: Forty‐six articles reporting 551 cases of COVID‐19 in children (aged 1 day‐17.5 years) were included. Eighty‐seven percent (95% CI: 77%‐95%) of patients had household exposure to COVID‐19. The most common symptoms and signs were fever (53%, 95% CI: 45%‐61%), cough (39%, 95% CI: 30%‐47%), and sore throat/pharyngeal erythema (14%, 95% CI: 4%‐28%); however, 18% (95% CI: 11%‐27%) of cases were asymptomatic. The most common radiographic and computed tomography (CT) findings were patchy consolidations (33%, 95% CI: 23%‐43%) and ground glass opacities (28%, 95% CI: 18%‐39%), but 36% (95% CI: 28%‐45%) of patients had normal CT images. Antiviral agents were given to 74% of patients (95% CI: 52%‐92%). Six patients, all with major underlying medical conditions, needed invasive mechanical ventilation, and one of them died. CONCLUSION: Previously healthy children with COVID‐19 have mild symptoms. The diagnosis is generally suspected from history of household exposure to COVID‐19 case. Children with COVID‐19 and major underlying condition are more likely to have severe/critical disease and poor prognosis, even death.\n",
      "AIM: To summarize what we know so far about coronavirus disease (COVID-19) in children. METHOD: We searched PubMed, Scientific Electronic Library Online, and Latin American and Caribbean Center on Health Sciences Information from 1 January 2020 to 4 May 2020. We selected randomized trials, observational studies, case series or case reports, and research letters of children ages birth to 18 years with laboratory-confirmed COVID-19. We conducted random-effects meta-analyses to calculate the weighted mean prevalence and 95% confidence interval (CI) or the weighted average means and 95% CI. RESULT: Forty-six articles reporting 551 cases of COVID-19 in children (aged 1 day-17.5 years) were included. Eighty-seven percent (95% CI: 77%-95%) of patients had household exposure to COVID-19. The most common symptoms and signs were fever (53%, 95% CI: 45%-61%), cough (39%, 95% CI: 30%-47%), and sore throat/pharyngeal erythema (14%, 95% CI: 4%-28%); however, 18% (95% CI: 11%-27%) of cases were asymptomatic. The most common radiographic and computed tomography (CT) findings were patchy consolidations (33%, 95% CI: 23%-43%) and ground glass opacities (28%, 95% CI: 18%-39%), but 36% (95% CI: 28%-45%) of patients had normal CT images. Antiviral agents were given to 74% of patients (95% CI: 52%-92%). Six patients, all with major underlying medical conditions, needed invasive mechanical ventilation, and one of them died. CONCLUSION: Previously healthy children with COVID-19 have mild symptoms. The diagnosis is generally suspected from history of household exposure to COVID-19 case. Children with COVID-19 and major underlying condition are more likely to have severe/critical disease and poor prognosis, even death.\n",
      "BACKGROUND To assess the overall prevalence of clinical signs, symptoms, and radiological findings in children and/or adolescents with COVID-19. METHODS We systematically researched in PubMed, Scopus and Web of Science databases observational studies describing COVID-19 in children and/or adolescents until April 11, 2020. Data regarding clinical and radiological features were extracted from eligible studies and meta-analysis was performed using random-effects modeling. RESULTS We examined 19 eligible studies for a total of 2855 children and/or adolescents with COVID-19. Approximately 47% of subjects had fever (95% confidence interval [CI] 22-72%; I2 = 98.6%), 37% cough (95%CI 15-63%; I2 = 98.6%), 4% diarrhea (95%CI 0-12%; I2 = 92.2%), 2% nasal congestion (95%CI 0-7%; I2 = 87.7%), 1% dyspnea (95%CI 0-7%; I2 = 91.5%) and 0% abdominal pain (95%CI 0-1%; I2 = 76.3%). Subjects presented mild symptoms in 79% (95%CI 65-91%; I2 = 93.5%) of cases, whereas only 4% (95%CI 1-9%; I2 = 76.4%) were critical. Among those with pneumonia on computed tomography, 26.4% (95%CI 13-41%; I2 = 80.8%) presented a unilateral involvement, 16% (95%CI 5-29%, I2 = 81.2%) had bilateral involvement and 9% (95%CI 0-24%; I2 = 88.7%) had interstitial pneumonia. CONCLUSIONS Children and/or adolescents tend to have a mild COVID-19 course with a good prognosis. IMPACT Compared to adults, children and/or adolescents tend to have a mild COVID-19 course with a good prognosis. This study provides new and consistence information on the clinical and radiological characteristics of COVID-19 in pediatrics. This study may help to fight COVID-19 in pediatric population.\n",
      "Context: The coronavirus disease 2019 (COVID-19) outbreak is an unprecedented global public health challenge, leading to thousands of deaths every day worldwide. Despite the epidemiological importance, clinical patterns of children with COVID-19 remain unclear. Objective: To describe the clinical, laboratorial and radiological characteristics of children with COVID-19. Data Sources: The Medline database was searched between December 1st 2019 and March 30th 2020. Study Selection: Inclusion criteria were: (1) studied patients younger than 18 years old; (2) presented original data from cases of COVID-19 confirmed by reverse-transcription polymerase chain reaction; and (3) contained descriptions of clinical manifestations, laboratory tests or radiological examinations. Data Extraction: Number of cases, gender, age, clinical manifestations, laboratory tests, radiological examinations and outcomes. Results: A total of 34 studies (1,118 cases) were included. From all the cases, 1,111 had their severity classified: 14.3% were asymptomatic, 36.4% were mild, 46.0% were moderate, 2.2% were severe and 1.2% were critical. The most prevalent symptom was fever (16.3%), followed by cough (14.4%), nasal symptoms (3.6%), diarrhea (2.7%) and nausea/vomiting (2.5%). One hundred forty-five (12.9%) children were diagnosed with pneumonia and 43 (3.8%) upper airway infections were reported. Reduced lymphocyte count were reported in 13.1% of cases. Abnormalities on computed tomography was reported in 62.7% of cases. The most prevalent abnormalities reported were ground glass opacities, patchy shadows and consolidations. Only one death was reported. Conclusions: Clinical manifestations of children with COVID-19 differ widely from adults cases. Fever and respiratory symptoms should not be considered a hallmark of COVID-19 in children.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 2705en59\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Abstract The novel coronavirus, SARS-CoV2, can cause a potentially fatal disease, COVID-19, in humans. Here, we will provide an overview of therapeutic options for COVID-19. Plasma from patients recovered from COVID-19 that contains antibodies against SARS-CoV2 has shown promising results in patients with severe COVID-19. Also, IVIG, combined with moderate-dose of corticosteroids, might improve patient outcomes. Evidence links COVID-19 to variable degrees of inflammation. Studies show that the use of corticosteroids might accelerate recovery from COVID-19. There are, however, no controlled clinical trials that show whether the use of corticosteroids can reduce COVID-19-related death. Also, the pro-inflammatory cytokine IL6 is the best-documented cytokine in COVID-19 correlated with severity, criticality, viral load, and prognosis of patients with COVID-19. Tocilizumab, a monoclonal antibody against IL6, could confer clinical benefit in patients with high IL6 levels. Essential elements that process SARS-CoV2 cell entry and specific characteristics that allow SARS-CoV2 to escape the immune system have the potential as targets for COVID-19 therapy.\n",
      "The novel coronavirus, SARS-CoV2, can cause a potentially fatal disease, COVID-19, in humans. Here, we will provide an overview of therapeutic options for COVID-19. Plasma from patients recovered from COVID-19 that contains antibodies against SARS-CoV2 has shown promising results in patients with severe COVID-19. Also, IVIG, combined with moderate-dose of corticosteroids, might improve patient outcomes. Evidence links COVID-19 to variable degrees of inflammation. Studies show that the use of corticosteroids might accelerate recovery from COVID-19. There are, however, no controlled clinical trials that show whether the use of corticosteroids can reduce COVID-19-related death. Also, the pro-inflammatory cytokine IL6 is the best-documented cytokine in COVID-19 correlated with severity, criticality, viral load, and prognosis of patients with COVID-19. Tocilizumab, a monoclonal antibody against IL6, could confer clinical benefit in patients with high IL6 levels. Essential elements that process SARS-CoV2 cell entry and specific characteristics that allow SARS-CoV2 to escape the immune system have the potential as targets for COVID-19 therapy.\n",
      "COVID‐19 leads to mortality of several patients and the cytokine storm is reportedly critical in the patients. In order to reduce the cytokine storm, we would like to propose the IL6 receptor antagonist therapy for the COVID‐19 patients. Two humanized monoclonal antibodies are in clinical trial following IL6R antagonist therapies namely tocilizumab and sarilumab. However, researchers and physicians should look for more IL6 receptor antagonists for the therapy of cytokine storm syndrome SARS‐CoV‐2 infected persons to enhance the therapeutic options for cytokine storm. This article is protected by copyright. All rights reserved.\n",
      "The epidemic due to severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) infection has been spreading globally, raising increasing concerns. This public health emergency has triggered a race to find medications to improve the prognosis of this disease. There is currently great interest in drug repositioning to manage SARS-CoV-2 infection, that is, the evaluation of the potential benefits of a drug that has already been proven safe and effective in humans for other approved indications. As interleukin-6 (IL-6) acts as a key driver of the inflammation associated with coronavirus disease 2019 (COVID-19), IL-6 and IL-6 receptor (IL-6R) inhibition appear to be promising targets for the treatment of COVID-19 patients. It is important to critically analyze the available evidence concerning the use of the available anti-IL-6 (siltuximab) and anti-IL-6R (tocilizumab and sarilumab) agents in COVID-19 patients, in terms of both benefit and risk. In this review, the pathogenesis of the cytokine storm induced by COVID-19, the role of IL-6 in this cytokine storm, the rationale for the use of anti-IL-6 agents, and key information on potential benefits and safety monitoring of these biologicals in COVID-19 patients is discussed. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (10.1007/s40259-020-00430-1) contains supplementary material, which is available to authorized users.\n",
      "Limited evidence is available to guide treatment of coronavirus disease 2019 (COVID-19), caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). A hyperinflammatory state mediated by interleukin-6 (IL-6) has been proposed as a driver of severe disease. Use of the IL-6 receptor inhibitor tocilizumab for severe COVID-19 was first reported in China, where a case series described marked improvements in inflammatory markers, fever, oxygen requirement, and outcomes following its administration. Here, we provide the first description of a tocilizumab-treated cohort of patients with COVID-19 in the United States. We describe 11 patients from a single academic medical center, nine (82%) of whom were critically ill requiring mechanical ventilation in an intensive care unit at the time of tocilizumab administration. C-reactive protein levels decreased in all patients following treatment (median 211.6 pre- vs. 19.7 mg/L 5 days post-tocilizumab [p=0.001]). When IL-6 levels were obtained before and after therapy, wide variation was seen in baseline levels; post-dose IL-6 concentrations were consistently increased. In contrast to prior reports, we did not observe significant clinical improvement in temperature or oxygen requirements in most patients. Two patients were discharged (18%), five remained in critical condition in the intensive care unit (46%), one was weaned off the ventilator to room air (9%), and three died (27%). Our findings suggest that tocilizumab should be used with caution in severe and critically ill patients and highlight the need for data from randomized controlled trials to determine its efficacy in the treatment of COVID-19.\n",
      "Introduction: The coronavirus disease 2019 (COVID-19) caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has developed since December 2019. It has caused a global pandemic with more than three hundred thousand case fatalities. However, apart from supportive care by respirators, no standard medical therapy is validated.Areas covered: This paper presents old drugs with potential in vitro efficacy against SARS-CoV-2. The in vitro database, adverse effects, and potential toxicities of these drugs are reviewed regarding their feasibility of clinical prescription for the treatment of patients with COVID-19. To obtain convincing recommendations, we referred to opinions from the US National Institute of Health regarding drugs repurposed for COVID-19 therapy.Expert opinion: Although strong evidence of well-designed randomized controlled studies regarding COVID-19 therapy is presently lacking, remdesivir, teicoplanin, hydroxychloroquine (not in combination with azithromycin), and ivermectin might be effective antiviral drugs and are deemed promising candidates for controlling SARS-CoV-2. In addition, tocilizumab might be considered as the supplementary treatment for COVID-19 patients with cytokine release syndrome. In future, clinical trials regarding a combination of potentially effective drugs against SARS-CoV-2 need to be conducted to establish the optimal regimen for the treatment of patients with moderate-to-severe COVID-19.\n",
      "Currently, the world is facing the pandemic of a novel strain of beta‐coronavirus known as severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2). Acute respiratory distress syndrome (ARDS) is the most devastating complication of SARS‐CoV‐2. It was indicated that cytokine release syndrome (CRS) and dominantly IL‐6 play a central role in the pathophysiology of ARDS related to the novel 2019 coronavirus disease (COVID‐19). Despite the global emergency of the disease, at this time, there are no proven therapies for the management of the disease. Tocilizumab is a potential recombinant monoclonal antibody against IL‐6 and currently is under investigation for the management of ARDS in patients with COVID‐19. Given these points, we reviewed the current evidence regarding the potential therapeutic role of tocilizumab and its important clinical issues in the treatment of ARDS related to COVID‐19. This article is protected by copyright. All rights reserved\n",
      "COVID-19 leads to mortality of several patients and the cytokine storm is reportedly critical in the patients. To reduce the cytokine storm, we would like to propose the interleukin (IL) 6 receptor (IL-6R) antagonist therapy for the COVID-19 patients. Two humanized monoclonal antibodies are in clinical trial following IL-6R antagonist therapies namely tocilizumab and sarilumab. However, researchers and physicians should look for more IL-6R antagonists for the therapy of cytokine storm syndrome severe acute respiratory syndrome coronavirus 2 infected persons to enhance the therapeutic options for cytokine storm.\n",
      "Currently, the world is facing the pandemic of a novel strain of beta-coronavirus known as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Acute respiratory distress syndrome (ARDS) is the most devastating complication of SARS-CoV-2. It was indicated that cytokine release syndrome (CRS) and dominantly IL-6 play a central role in the pathophysiology of ARDS related to the novel 2019 coronavirus disease (COVID-19). Despite the global emergency of the disease, at this time, there are no proven therapies for the management of the disease. Tocilizumab is a potential recombinant monoclonal antibody against IL-6 and currently is under investigation for the management of ARDS in patients with COVID-19. Given these points, we reviewed the current evidence regarding the potential therapeutic role of tocilizumab and its important clinical issues in the treatment of ARDS related to COVID-19. This article is protected by copyright. All rights reserved.\n",
      "Coronavirus disease 2019 (COVID‐19) caused by severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2) is a pandemic that is evolving rapidly at present with unprecedented global health and societal implications. Herein, we report a case of successful treatment of SARS‐CoV‐2 induced inflammatory syndrome by utilization of the interleukin‐6 receptor antagonist (IL‐6Ra) tocilizumab.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: hl956i02\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "How much impact do masks have on preventing the spread of the COVID-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "The main form of COVID-19 transmission is via “oral-respiratory droplet contamination” (droplet: very small drop of liquid) produced when individuals talk, sneeze, or cough. In hospitals, health-care workers wear facemasks as a minimum medical “droplet precaution” to protect themselves. Due to the shortage of masks during the pandemic, priority is given to hospitals for their distribution. As a result, the availability/use of medical masks is discouraged for the public. However, for asymptomatic individuals, not wearing masks in public could easily cause the spread of COVID-19. The prevention of “environmental droplet contamination” (EnvDC) from coughing/sneezing/speech is fundamental to reducing transmission. As an immediate solution to promote “public droplet safety,” we assessed household textiles to quantify their potential as effective environmental droplet barriers (EDBs). The synchronized implementation of a universal “community droplet reduction solution” is discussed as a model against COVID-19. Using a bacterial-suspension spray simulation model of droplet ejection (mimicking a sneeze), we quantified the extent by which widely available clothing fabrics reduce the dispersion of droplets onto surfaces within 1.8 m, the minimum distance recommended for COVID-19 “social distancing.” All textiles reduced the number of droplets reaching surfaces, restricting their dispersion to <30 cm, when used as single layers. When used as double-layers, textiles were as effective as medical mask/surgical-cloth materials, reducing droplet dispersion to <10 cm, and the area of circumferential contamination to ~0.3%. The synchronized implementation of EDBs as a “community droplet reduction solution” (i.e., face covers/scarfs/masks and surface covers) will reduce COVID-19 EnvDC and thus the risk of transmitting/acquiring COVID-19.\n",
      "The main form of COVID-19 transmission is via oral-respiratory droplet contamination (droplet; very small drop of liquid) produced when individuals talk, sneeze or cough. In hospitals, health-care workers wear facemasks as a minimum medical droplet precaution to protect themselves. Due to the shortage of masks during the pandemic, priority is given to hospitals for their distribution. As a result, the availability/use of medical masks is discouraged for the public. However, given that asymptomatic individuals, not wearing masks within the public, can be highly contagious for COVID-19, prevention of environmental droplet contamination (EnDC) from coughing/sneezing/speech is fundamental to reducing transmission. As an immediate solution to promote public droplet safety, we assessed household textiles to quantify their potential as effective environmental droplet barriers (EDBs). The synchronized implementation of a universal community droplet reduction solution is discussed as a model against COVID-19. Using a bacterial-suspension spray simulation model of droplet ejection (mimicking a sneeze), we quantified the extent by which widely available clothing fabrics reduce the dispersion of droplets onto surfaces within 1.8m, the minimum distance recommended for COVID-19 social distancing. All textiles reduced the number of droplets reaching surfaces, restricting their dispersion to <30cm, when used as single layers. When used as double-layers, textiles were as effective as medical mask/surgical-cloth materials, reducing droplet dispersion to <10cm, and the area of circumferential contamination to ~0.3%. The synchronized implementation of EDBs as a community droplet reduction solution (i.e., face covers/scarfs/masks & surface covers) could reduce EnDC and the risk of transmitting or acquiring infectious respiratory pathogens, including COVID-19.\n",
      "Respiratory infections may spread through droplets, airborne particles, and aerosols from infected individuals through coughing, sneezing, and speaking. In the case of Coronavirus Disease 2019 (COVID-19), droplet spread can occur from symptomatic as well as pre-symptomatic and asymptomatic persons. The U.S. Centers for Disease Control and Prevention (CDC) has therefore recently recommended home-made cloth face coverings for use by the general public in areas of significant community-based transmission. Because medical masks and N95 respirators are in short supply, these are to be reserved for healthcare workers. There is, however, little information on the effectiveness of home-made face coverings in reducing droplet dissemination. Here, we ascertained the performance of ten different fabrics, ranging from cotton to silk, in blocking high velocity droplets, using a 3-layered commercial medical mask as a benchmark material. We also assessed their breathability and ability to soak water. We reason that the materials should be as breathable as possible, without compromising blocking efficiency, to reduce air flow through the sides of the mask since such flow would defeat the purpose of the mask. We found that most home fabrics substantially block droplets, even as a single layer. With two layers, blocking performance can reach that of surgical mask without significantly compromising breathability. Furthermore, we observed that home fabrics are hydrophilic to varying degrees, and hence soak water. In contrast, medical masks are hydrophobic, and tend to repel water. Incoming droplets are thus soaked and 'held back' by home fabrics, which might offer an as of yet untapped and understudied advantage of home-made cloth masks. Overall, our study suggests that most double-layered cloth face coverings may help reduce droplet transmission of respiratory infections.\n",
      "Due to the shortage of masks during the pandemic, we recently demonstrated that household textiles are effective environmental droplet barriers (EDBs) with identical droplet retention potential as medical masks. To further promote the implementation of a universal community droplet reduction solution based on a synchronized encouragement/enforcement of mask utilization by the public based on widely available textiles (mask fabrication without the need for sewing machines), here we conducted a study using germ-free mice to determine to what extent textiles were effective in vivo. Using a bacterial-suspension spray simulation model of droplet ejection (mimicking a sneeze), we quantified the extent by which 100% cotton textile prevented the contamination of germ-free animals on the other side of the textile-barrier (simulating a properly worn mask). Of relevance, all mice protected with textiles remained germ-free after two sprays (inoculation dose: >600 bacterial droplet units per 56.75cm2) compared to the contamination of mice not protected by a textile (0/12 vs 6/6, Fisher’s exact, p<0.0001). In a second phase of the experiment with 12 germ-free mice exposed again to 10-fold more droplets remained germ-free, while 100% of mice at 180cm became colonized with a single spray (0/8 vs 4/4, Fisher exact, p=0.002). Collectively, barriers protected all mice (even with low-density textiles, heavy vs. light fabric, T-test, p=0.0028) when using textile-EDB to cover the cages (0/20 vs 10/10, Fisher exact, p<0.0001). This study demonstrated, in vivo, that widely available household textiles are 100% effective at preventing contamination of the environment and the exposed animals by microbe-carrying droplets.\n",
      "Many governments have instructed the population to wear simple mouth-and-nose covers or surgical face masks to protect themselves from droplet infection with the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in public. However, the basic protection mechanisms and benefits of these masks remain controversial. Therefore, the aim of this work is to show from a fluid physics point of view under which circumstances these masks can protect against droplet infection. First of all, we show that the masks protect people in the surrounding area quite well, since the flow resistance of the face masks effectively prevents the spread of exhaled air, e.g. when breathing, speaking, singing, coughing and sneezing. Secondly, we provide visual evidence that typical household materials used by the population to make masks do not provide highly efficient protection against respirable particles and droplets with a diameter of 0.3–2 μm as they pass through the materials largely unfiltered. According to our tests, only vacuum cleaner bags with fine dust filters show a comparable or even better filtering effect than commercial particle filtering FFP2/N95/KN95 half masks. Thirdly, we show that even simple mouth-and-nose covers made of good filter material cannot reliably protect against droplet infection in contaminated ambient air, since most of the air flows through gaps at the edge of the masks. Only a close-fitting, particle-filtering respirator without an outlet valve offers good self-protection and protection against droplet infection. Nevertheless, wearing simple homemade or surgical face masks in public is highly recommended if no particle filtrating respiratory mask is available. Firstly, because they protect against habitual contact of the face with the hands and thus serve as self-protection against contact infection. Secondly, because the flow resistance of the masks ensures that the air stays close to the head when breathing, speaking, singing, coughing and sneezing, thus protecting other people if they have sufficient distance from each other. However, if the distance rules cannot be observed and the risk of inhalation-based infection becomes high because many people in the vicinity are infectious and the air exchange rate is small, improved filtration efficiency masks are needed, to take full advantage of the three fundamental protective mechanisms these masks provide.\n",
      "Objectives While COVID-19 is known to be spread by respiratory droplets (which travel <2m horizontally), much less is known about its transmission via aerosols, which can become airborne and be widely distributed throughout room spaces. In order to quantify the risk posed by COVID-19 infectors exhaling respiratory aerosols in enclosed spaces, we undertook a computer modelling study to simulate transmission in an office building. Methods Respiratory droplet data from four published datasets were analysed to quantify the number and volume of droplets <100m diameter produced by a typical cough and speaking event (i.e. counting from 1 to 100). This was used in a stochastic model to simulate (10,000 simulations) the number of respiratory particles, originating from a COVID-19 infector, that would be inhaled in one hour by a susceptible individual practicing socially distancing in a 5 x 5 x 2.75m office space. Several scenarios were simulated that mimicked the presence of both symptomatic and asymptomatic COVID-19 infectors. Results On average, each cough and speaking event produced similar numbers of droplets <100 m diameter (median range = 955 - 1010). Computer simulations (at ventilation rate = 2AC/h) revealed that sharing the office space with a symptomatic COVID-19 infector (4 coughs per hour) for one hour resulted in the inhalation of 187.3 (median value) respiratory droplets, whereas sharing with an asymptomatic COVID-19 positive person (10 speaking events per hour) resulted in the inhalation of 482.9 droplets. Increasing the ventilation rate resulted in only modest reductions in particle numbers inhaled. Conclusions Given that live SARS-CoV-2 virions are known to be shed in high concentrations from the nasal cavity of both symptomatic and asymptomatic COVID-19 patients, the results suggest that individuals who share enclosed spaces with an infector may be at risk of contracting COVID-19 by the aerosol route, even when practicing social distancing.\n",
      "It has been confirmed that the coronavirus disease 2019 (COVID-19) can transmit through droplets created when an infected human coughs or sneezes. Accordingly, 1.83-m (6-feet) social distancing is advised to reduce the spread of the disease among humans. This is based on the assumption that no air circulation exists around people. However, it is not well investigated whether the ambient wind and relative humidity (RH) will cause SARS-CoV-2 laden droplets to transport farther in the air, making the current social distancing policy insufficient. To provide evidence and insight into the “social distancing” guidelines, a validated computational fluid-particle dynamics (CFPD) model was employed to simulate the transient transport, condensation/evaporation, and deposition of SARS-CoV-2 laden droplets emitted by coughs, with different environmental wind velocities and RHs. Initial droplet diameters range from 2 to 2000 μm, and the wind velocities range from 0 to 16 km/h, representing different wind forces from calm air to moderate breeze. The comparison between a steady-state wind and a gust with a constant frequency has also been performed. Ambient RHs are 40% and 99.5%. The distances between the two virtual humans are 1.83 m and 3.05 m (6 feet and 10 feet). The facial covering effect on reducing the airborne transmission of the cough droplets has also been evaluated. Numerical results indicate that the ambient wind will enhance the complexity of the secondary flows with recirculation between the two virtual humans. Microdroplets follow the airflow streamlines well and deposit on both human bodies and head regions, even with the 3.05-m (10-feet) separation distance. The rest of the microdroplets can transport in the air farther than 3.05 m (10 feet) due to wind convection, causing a potential health risk to nearby people. High RH will increase the droplet sizes due to the hygroscopic growth effect, which increases the deposition fractions on both humans and the ground. With the complex environmental wind and RH conditions, the 6-feet social distancing policy may not be sufficient to protect the inter-person aerosol transmission, since the suspending micro-droplets were influenced by convection effects and can be transported from the human coughs/sneezes to the other human in less than 5 s. Thus, due to the complex real-world environmental ventilation conditions, a social distance longer than 6 feet needs to be considered. Wearing masks should also be recommended for both infected and healthy humans to reduce the airborne cough droplet numbers.\n",
      "The spread of COVID19 through droplets ejected by infected individuals in the air during sneezing and coughing has been considered as a matter of key concern. Therefore, a quantitative understanding of the propagation of droplets containing the virus assumes immense importance in formulating the preventive strategies. In the present work we rigorously investigate the evolution of droplets in space and time under varying external conditions of temperature, humidity, air flow by using laws of statistical mechanics. The effects of drag, diffusion and the gravity on the droplets of different sizes and ejection velocities have been taken into consideration during its motion in the air. In still air it is found that bigger droplets traverse larger distance but the smaller droplets remain suspended in the air for longer time. Therefore, in still air the horizontal distance that a healthy individual should maintain from an infected one is determined by the bigger droplets but the time interval to be maintained between the two is determined by the smaller droplets. Here we show that in places with flowing air the lighter droplets travel larger distance and have the ability to remain suspended in the air for longer time. Therefore, we conclude that both temporal as well as the geometric distance that a healthy individual should maintain from an infected one is determined by the smaller droplets if there is flowing air. The small droplets travel larger distance and remain suspended in the air for long time under the influence of flow making the use of mask mandatory to prevent the virus. The the maintenance of only stationary separation between healthy and infected individual is not substantiated. The quantitative results obtained here will be useful to devise strategies for preventing also the spread of other types of droplets containing microorganisms.\n",
      "Coughs and sneezes disperse a large number of droplets of varying size into the environment, and they transmit respiratory viral infections by direct or indirect physical contact, by droplets or inhalation of fine particulate droplet nuclei. Larger droplets in the cloud produced by coughing and sneezing settle quickly, and the force with which they are expelled determines how far they are dispersed. The respiratory droplets evaporate to form smaller droplet nuclei that carry infectious agents, remain suspended in air, and susceptible individuals farther away from the source could inhale them. The particle size distribution within the multi-phase cloud produced by coughs/sneezes changes with time and distance from the source depending on several environmental factors. After inhalation, larger respiratory droplets are filtered by the nose or deposit in the oropharynx, whereas smaller droplet nuclei are carried by the airstream into the lungs where their site of deposition depends on their mass, size and shape and is governed by various mechanisms. Airborne particles could also be produced by various aerosol generating procedures, such as suctioning or tracheal intubation, and by aerosol generators, especially jet nebulizers. Prevention of respiratory viral infections depends upon whether they are carried in respiratory droplets or as fine droplet nuclei (airborne or aerosol transmission). The SARS-CoV-2 virus that causes COVID-19 is mainly transmitted by respiratory droplets or by contact. Airborne transmission of this virus has not been established, but is possible under special circumstances. Appropriate protective measures are necessary to prevent transmission of SARS-CoV-2 virus in various settings. This article is open access and distributed under the terms of the Creative Commons Attribution Non-Commercial No Derivatives License 4.0 (http://creativecommons.org/licenses/by-nc-nd/4.0/).\n",
      "Coughs and sneezes disperse a large number of droplets of varying size into the environment, and they transmit respiratory viral infections by direct or indirect physical contact, by droplets or inhalation of fine particulate droplet nuclei. Larger droplets in the cloud produced by coughing and sneezing settle quickly, and the force with which they are expelled determines how far they are dispersed. The respiratory droplets evaporate to form smaller droplet nuclei that carry infectious agents, remain suspended in air, and susceptible individuals farther away from the source could inhale them. The particle size distribution within the multi-phase cloud produced by coughs/sneezes changes with time and distance from the source depending on several environmental factors. After inhalation, larger respiratory droplets are filtered by the nose or deposit in the oropharynx, whereas smaller droplet nuclei are carried by the airstream into the lungs where their site of deposition depends on their mass, size and shape and is governed by various mechanisms. Airborne particles could also be produced by various aerosol generating procedures, such as suctioning or tracheal intubation, and by aerosol generators, especially jet nebulizers. Prevention of respiratory viral infections depends upon whether they are carried in respiratory droplets or as fine droplet nuclei (airborne or aerosol transmission). The SARS-CoV-2 virus that causes COVID-19 is mainly transmitted by respiratory droplets or by contact. Airborne transmission of this virus has not been established, but is possible under special circumstances. Appropriate protective measures are necessary to prevent transmission of SARS-CoV-2 virus in various settings. This article is open access and distributed under the terms of the Creative Commons Attribution Non-Commercial No Derivatives License 4.0 (http://creativecommons.org/licenses/by-nc-nd/4.0/).\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: ve34oyz0\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "How much impact do masks have on preventing the spread of the COVID-19?\n",
      "what are the transmission routes of coronavirus?\n",
      "what are the best masks for preventing infection by Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "We welcomed Professor Cook's article clarifying the use of personal protective equipment (PPE) in protecting staff during the current COVID-19 pandemic [1]. There remains considerable debate about the extent to which airborne spread of SARS-CoV-2 occurs. Small droplets (< 5µm) are thought to remain suspended in the air and could theoretically be inhaled into the lungs causing infection [2]. Loose fitting \"surgical\" masks will not prevent such inhalation and only a tight-fitting filtering mask is adequate. Conversely larger (> 5µm) particles do not remain suspended in the air [2] and can only cause infection if they are immediately inhaled, or after contact with a surface they land on.\n",
      "The coronavirus disease (COVID-19) pandemic has posed a challenge for healthcare systems, and healthcare workers (HCWs) are at high risk of exposure. Protecting HCWs is of paramount importance to maintain continuous patient care and keep healthcare systems functioning. Used alongside administrative and engineering control measures, personal protective equipment (PPE) is the last line of defense and the core component of protection. Current data suggest that severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is mainly transmitted through respiratory droplets and close contact. Airborne transmission may occur during aerosol-generating procedures. However, the modes of transmission still remain uncertain, especially regarding the possibility of airborne transmission when aerosol-generating procedures are not performed. Thus, there are some inconsistencies in the respiratory protective equipment recommended by international and national organizations. In Korea, there have been several modifications to PPE recommendations offering options in choosing PPE for respiratory and body protection, which confuses HCWs; they are often unsure what to wear and when to wear it. The choice of PPE is based on the risk of exposure and possible modes of transmission. The level of protection provided by PPE differs based on standards and test methods. Thus, understanding them is the key in selecting the proper PPE. This article reviews evidence on the mode of SARS-CoV-2 transmission, compares the current PPE recommendations of the World Health Organization with those in Korea, and discusses standard requirements and the proper selection of PPE.\n",
      "OBJECTIVES: To determine the risk of SARS-CoV-2 transmission by aerosols, to provide evidence on the rational use of masks, and to discuss additional measures important for the protection of healthcare workers from COVID-19. METHODS: Literature review and expert opinion. SHORT CONCLUSION: SARS-CoV-2, the pathogen causing COVID-19, is considered to be transmitted via droplets rather than aerosols, but droplets with strong directional airflow support may spread further than 2 m. High rates of COVID-19 infections in healthcare-workers (HCWs) have been reported from several countries. Respirators such as filtering face piece (FFP) 2 masks were designed to protect HCWs, while surgical masks were originally intended to protect patients (e.g., during surgery). Nevertheless, high quality standard surgical masks (type II/IIR according to European Norm EN 14683) appear to be as effective as FFP2 masks in preventing droplet-associated viral infections of HCWs as reported from influenza or SARS. So far, no head-to-head trials with these masks have been published for COVID-19. Neither mask type completely prevents transmission, which may be due to inappropriate handling and alternative transmission pathways. Therefore, compliance with a bundle of infection control measures including thorough hand hygiene is key. During high-risk procedures, both droplets and aerosols may be produced, reason why respirators are indicated for these interventions.\n",
      "Background and purpose - The COVID-19 pandemic caused by infection with SARS-CoV-2 has led to a global shortage of personal protective equipment (PPE). Various alternatives to ordinary PPE have been suggested to reduce transmission, which is primarily through droplets and aerosols. For many years orthopedic surgeons have been using surgical helmets as personal protection against blood-borne pathogens during arthroplasty surgery. We have investigated the possibility of using the Stryker Flyte surgical helmet as a respiratory protective device against airborne- and droplet-transmitted disease, since the helmet shares many features with powered air-purifying respirators.Materials and methods - Using an aerosol particle generator, we determined the filtration capacity of the Stryker Flyte helmet by placing particle counters measuring the concentrations of 0.3, 0.5, and 5 µm particles inside and outside of the helmet.Results - We found that the helmet has insufficient capacity for filtrating aerosol particles, and, for 0.3 µm sized particles, we even recorded an accumulation of particles inside the helmet.Interpretation - We conclude that the Stryker Flyte surgical helmet should not be used as a respiratory protective device when there is a risk for exposure to aerosol containing SARS-CoV-2, the virus causing COVID-19, in accordance with the recommendation from the manufacturer.\n",
      "Personal protective equipment has become an important and emotive subject during the current coronavirus (COVID-19) epidemic. COVID-19 is predominantly caused by contact or droplet transmission attributed to relatively large respiratory particles which are subject to gravitational forces and travel only approximately one metre from the patient. Airborne transmission may occur if patient respiratory activity or medical procedures generate respiratory aerosols. These aerosols contain particles that may travel much longer distances and remain airborne longer, but their infective potential is uncertain. Contact, droplet and airborne transmission are each relevant during airway manoeuvres in infected patients, particularly during tracheal intubation. Personal protective equipment is an important component, but only one part, of a system protecting staff and other patients from COVID-19 cross-infection. Appropriate use significantly reduces risk of viral transmission. Personal protective equipment should logically be matched to the potential mode of viral transmission occurring during patient care - contact, droplet, or airborne. Recommendations from international organisations are broadly consistent, but equipment use is not. Only airborne precautions include a fitted high-filtration mask, and this should be reserved for aerosol-generating procedures. Uncertainty remains around certain details of personal protective equipment including use of hoods, mask type and the potential for re-use of equipment.\n",
      "Personal protective equipment has become an important and emotive subject during the current coronavirus disease 2019 epidemic. Coronavirus disease 2019 is predominantly caused by contact or droplet transmission attributed to relatively large respiratory particles which are subject to gravitational forces and travel only approximately 1 metre from the patient. Airborne transmission may occur if patient respiratory activity or medical procedures generate respiratory aerosols. These aerosols contain particles that may travel much longer distances and remain airborne longer, but their infective potential is uncertain. Contact, droplet and airborne transmission are each relevant during airway manoeuvres in infected patients, particularly during tracheal intubation. Personal protective equipment is an important component, but only one part, of a system protecting staff and other patients from coronavirus disease 2019 cross-infection. Appropriate use significantly reduces risk of viral transmission. Personal protective equipment should logically be matched to the potential mode of viral transmission occurring during patient care - contact, droplet or airborne. Recommendations from international organisations are broadly consistent, but equipment use is not. Only airborne precautions include a fitted high-filtration mask, and this should be reserved for aerosol generating procedures. Uncertainty remains around certain details of personal protective equipment including use of hoods, mask type and the potential for re-use of equipment.\n",
      "INTRODUCTION SARS-CoV-2 is dispersed from patients by talking, coughing and sneezing. The generated micro-droplets aerosols can travel up to 8 meters, stay suspended for long periods and preserve viral infectivity for a median of 2.7 hours. An unprotected person exposed to this cloud, might inhale a considerable amount of infectious viral doses, which will attach to the ACE 2 receptors on alveoli epithelium, resulting in infection. N95 respirators and surgical masks block 95% and 50-60% respectively of inhalable particles and protect the wearer from infection. Surgical masks and N95 without exhalation valve, protect both the wearer and the environment from carriers and sick people.\n",
      "INTRODUCTION: SARS-CoV-2 is dispersed from patients by talking, coughing and sneezing. The generated micro-droplets aerosols can travel up to 8 meters, stay suspended for long periods and preserve viral infectivity for a median of 2.7 hours. An unprotected person exposed to this cloud, might inhale a considerable amount of infectious viral doses, which will attach to the ACE 2 receptors on alveoli epithelium, resulting in infection. N95 respirators and surgical masks block 95% and 50-60% respectively of inhalable particles and protect the wearer from infection. Surgical masks and N95 without exhalation valve, protect both the wearer and the environment from carriers and sick people.\n",
      "Background and purpose - COVID-19 is among the most impactful pandemics that the society has experienced. Orthopedic surgery involves procedures generating droplets and aerosols and there is concern amongst surgeons that otherwise rational precautionary principles are being set aside due to lack of scientific evidence and a shortage of personal protective equipment (PPE). This narrative review attempts to translate relevant knowledge into practical recommendations for healthcare workers involved in orthopedic surgery on patients with known or suspected COVID-19.Patients and methods - We unsystematically searched in PubMed, reference lists, and the WHO's web page for relevant publications concerning problems associated with the PPE used in perioperative practice when a patient is COVID-19 positive or suspected to be. A specific search for literature regarding COVID-19 was extended to include publications from the SARS epidemic in 2002/3.Results - Transmission of infectious viruses from patient to surgeon during surgery is possible, but does not appear to be a considerable problem in clinical practice. Seal-leakage is a problem with surgical masks. Due to the lack of studies and reports, the possibility of transmission of SARS-CoV-2 from patient to surgeon during droplet- and aerosol-generating procedures is unknown.Interpretation - Surgical masks should be used only in combination with a widely covering visor and when a respirator (N95, FFP2, P3) is not made available. Furthermore, basic measures to reduce shedding of droplets and aerosols during surgery and correct and consistent use of personal protective equipment is important.\n",
      "Background and purpose - COVID-19 is among the most impactful pandemics that the society has experienced. Orthopedic surgery involves procedures generating droplets and aerosols and there is concern amongst surgeons that otherwise rational precautionary principles are being set aside due to lack of scientific evidence and a shortage of personal protective equipment (PPE). This narrative review attempts to translate relevant knowledge into practical recommendations for healthcare workers involved in orthopedic surgery on patients with known or suspected COVID-19.Patients and methods - We unsystematically searched in PubMed, reference lists, and the WHO's web page for relevant publications concerning problems associated with the PPE used in perioperative practice when a patient is COVID-19 positive or suspected to be. A specific search for literature regarding COVID-19 was extended to include publications from the SARS epidemic in 2002/3.Results - Transmission of infectious viruses from patient to surgeon during surgery is possible, but does not appear to be a considerable problem in clinical practice. Seal-leakage is a problem with surgical masks. Due to the lack of studies and reports, the possibility of transmission of SARS-CoV-2 from patient to surgeon during droplet- and aerosol-generating procedures is unknown.Interpretation - Surgical masks should be used only in combination with a widely covering visor and when a respirator (N95, FFP2, P3) is not made available. Furthermore, basic measures to reduce shedding of droplets and aerosols during surgery and correct and consistent use of personal protective equipment is important.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 40xsypzt\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "SARS-CoV-2, the causative agent of COVID-19 pandemic, is an RNA virus prone to mutations. Interaction of SARS-CoV-2 Spike (S) protein with the host cell receptor, Angiotensin-I Converting Enzyme 2 (ACE2) is pivotal for attachment and entry of the virus. Yet, natural mutations acquired on S protein during the pandemic and their impact on viral infectivity, transmission dynamics and disease pathogenesis remains poorly understood. Here, we analysed 2952 SARS-CoV-2 genomes across the globe, and identified a total of 1815 non-synonymous mutations in the S-protein that fall into 54 different types. We observed that six of these distinct mutations were located in the Receptor Binding Domain (RBD) region that directly engages host ACE2. Molecular phylogenetic analysis revealed that these RBD mutations cluster into distinct phyletic clades among global subtypes of SARS-CoV-2 implying possible emergence of novel sublineages of the strain. Structure-guided homology modelling and docking analysis predicted key molecular rearrangements in the ACE2 binding interface of RBD mutants that could result in altered virus-host interactions. We propose that our findings could be significant in understanding disease dynamics and in developing vaccines, antibodies and therapeutics for COVID-19. Importance COVID-19 pandemic shows considerable variations in disease transmission and pathogenesis globally, yet reasons remain unknown. Our study identifies key S-protein mutations prevailing in SARS-CoV-2 strain that could alter viral attachment and infectivity. We propose that the interplay of these mutations could be one of the factors driving global variations in COVID-19 spread. In addition, the mutations identified in this study could be an important indicator in predicting efficacies of vaccines, antibodies and therapeutics that target SARS-CoV-2 RBD-ACE2 interface.\n",
      "The spike (S) glycoprotein of SARS-CoV-2 is responsible for the binding to the permissive cells. The receptor-binding domain (RBD) of SARS-CoV-2 S protein directly interacts with the human angiotensin-converting enzyme 2 (ACE2) on the host cell membrane. In this study, we used computational saturation mutagenesis approaches, including structure-based energy calculations and sequence-based pathogenicity predictions, to quantify the systemic effects of missense mutations on SARS-CoV-2 S protein structure and function. A total of 18,354 mutations in S protein were analyzed and we discovered that most of these mutations could destabilize the entire S protein and its RBD. Specifically, residues G431 and S514 in SARS-CoV-2 RBD are important for S protein stability. We analyzed 384 experimentally verified S missense variations and revealed that the dominant pandemic form, D614G, can stabilize the entire S protein. Moreover, many mutations in N-linked glycosylation sites can increase the stability of the S protein. In addition, we investigated 3,705 mutations in SARS-CoV-2 RBD and 11,324 mutations in human ACE2 and found that SARS-CoV-2 neighbor residues G496 and F497 and ACE2 residues D355 and Y41 are critical for the RBD-ACE2 interaction. The findings comprehensively provide potential target sites in the development of drugs and vaccines against COVID-19.\n",
      "The newly identified SARS-CoV-2 has now been reported from around 185 countries with more than a million confirmed human cases including more than 120,000 deaths. The genomes of SARS-COV-2 strains isolated from different parts of the world are now available and the unique features of constituent genes and proteins need to be explored to understand the biology of the virus. Spike glycoprotein is one of the major targets to be explored because of its role during the entry of coronaviruses into host cells. We analyzed 320 whole-genome sequences and 320 spike protein sequences of SARS-CoV-2 using multiple sequence alignment. In this study, 483 unique variations have been identified among the genomes of SARS-CoV-2 including 25 nonsynonymous mutations and one deletion in the spike (S) protein. Among the 26 variations detected in S, 12 variations were located at the N-terminal domain (NTD) and 6 variations at the receptor-binding domain (RBD) which might alter the interaction of S protein with the host receptor angiotensin-converting enzyme 2 (ACE2). Besides, 22 amino acid insertions were identified in the spike protein of SARS-CoV-2 in comparison with that of SARS-CoV. Phylogenetic analyses of spike protein revealed that Bat coronavirus have a close evolutionary relationship with circulating SARS-CoV-2. The genetic variation analysis data presented in this study can help a better understanding of SARS-CoV-2 pathogenesis. Based on results reported herein, potential inhibitors against S protein can be designed by considering these variations and their impact on protein structure.\n",
      "The spread of COVID-19 caused by the SARS-CoV-2 outbreak has been growing since its first identification in December 2019. The publishing of the first SARS-CoV-2 genome made a valuable source of data to study the details about its phylogeny, evolution, and interaction with the host. Protein-protein binding assays have confirmed that Angiotensin-converting enzyme 2 (ACE2) is more likely to be the cell receptor through which the virus invades the host cell. In the present work, we provide an insight into the interaction of the viral spike Receptor Binding Domain (RBD) from different coronavirus isolates with host ACE2 protein. By calculating the binding energy score between RBD and ACE2, we highlighted the putative jump in the affinity from a progenitor form of SARS-CoV-2 to the current virus responsible for COVID-19 outbreak. Our result was consistent with previously reported phylogenetic analysis and corroborates the opinion that the interface segment of the spike protein RBD might be acquired by SARS-CoV-2 via a complex evolutionary process rather than a progressive accumulation of mutations. We also highlighted the relevance of Q493 and P499 amino acid residues of SARS-CoV-2 RBD for binding to human ACE2 and maintaining the stability of the interface. Moreover, we show from the structural analysis that it is unlikely for the interface residues to be the result of genetic engineering. Finally, we studied the impact of eight different variants located at the interaction surface of ACE2, on the complex formation with SARS-CoV-2 RBD. We found that none of them is likely to disrupt the interaction with the viral RBD of SARS-CoV-2.\n",
      "The spread of COVID-19 caused by the SARS-CoV-2 outbreak has been growing since its first identification in December 2019. The publishing of the first SARS-CoV-2 genome made a valuable source of data to study the details about its phylogeny, evolution, and interaction with the host. Protein-protein binding assays have confirmed that Angiotensin-converting enzyme 2 (ACE2) is more likely to be the cell receptor through which the virus invades the host cell. In the present work, we provide an insight into the interaction of the viral spike Receptor Binding Domain (RBD) from different coronavirus isolates with host ACE2 protein. By calculating the binding energy score between RBD and ACE2, we highlighted the putative jump in the affinity from a progenitor form of SARS-CoV-2 to the current virus responsible for COVID-19 outbreak. Our result was consistent with previously reported phylogenetic analysis and corroborates the opinion that the interface segment of the spike protein RBD might be acquired by SARS-CoV-2 via a complex evolutionary process rather than a progressive accumulation of mutations. We also highlighted the relevance of Q493 and P499 amino acid residues of SARS-CoV-2 RBD for binding to human ACE2 and maintaining the stability of the interface. Moreover, we show from the structural analysis that it is unlikely for the interface residues to be the result of genetic engineering. Finally, we studied the impact of eight different variants located at the interaction surface of ACE2, on the complex formation with SARS-CoV-2 RBD. We found that none of them is likely to disrupt the interaction with the viral RBD of SARS-CoV-2.\n",
      "The pandemic caused by novel severe acute respiratory syndrome coronavirus (SARS-CoV-2) has resulted in over 452,822 deaths in the first twenty days of June 2020 due to the coronavirus virus disease 2019 (COVID-19). The SARS-CoV-2 uses the host angiotensin-converting enzyme 2 (ACE2) receptor to gain entry inside the human cells where it replicates by using the cell protein synthesis mechanisms. The knowledge of the tissue distribution of ACE2 in human organs is therefore important to predict the clinical course of the COVID-19. Also important is the understanding of the viral receptor-binding domain (RBD) a region within the spike (S) proteins that enable the entry of the virus into the host cells to synthesize vaccine and monoclonal antibodies (mAbs). We performed an exhaustive search of human protein databases to establish the tissues that express ACE2 and performed an in-depth analysis like sequence alignments and homology modeling of the spike protein (S) of the SARS-CoV-2 to identify antigenic regions in the RBD that can be exploited to synthesize vaccine and mAbs. Our results show that ACE2 is widely expressed in human organs that may explain the pulmonary, systemic, and neurological deficits seen in COVID-19 patients. We show that though the S protein of the SARS-CoV-2 is a homolog of S protein of SARS-CoV-1, it has regions of dissimilarities in the RBD and transmembrane segments. We show peptide sequences in the RBD of SARS-CoV-2 that can bind to the MHC alleles and serve as effective epitopes for vaccine and mAbs synthesis. This article is protected by copyright. All rights reserved.\n",
      "The newly identified SARS-CoV-2 has now been reported from around 185 countries with more than a million confirmed human cases including more than 120,000 deaths. The genomes of SARS-COV-2 strains isolated from different parts of the world are now available and the unique features of constituent genes and proteins need to be explored to understand the biology of the virus. Spike glycoprotein is one of the major targets to be explored because of its role during the entry of coronaviruses into host cells. We analyzed 320 whole-genome sequences and 320 spike protein sequences of SARS-CoV-2 using multiple sequence alignment. In this study, 483 unique variations have been identified among the genomes of SARS-CoV-2 including 25 nonsynonymous mutations and one deletion in the spike (S) protein. Among the 26 variations detected, 12 variations were located at the N-terminal domain and 6 variations at the receptor-binding domain (RBD) which might alter the interaction of S protein with the host receptor angiotensin converting enzyme-2 (ACE2). Besides, 22 amino acid insertions were identified in the spike protein of SARS-CoV-2 in comparison with that of SARS-CoV. Phylogenetic analyses of spike protein revealed that Bat coronavirus have a close evolutionary relationship with circulating SARS-CoV-2. The genetic variation analysis data presented in this study can help a better understanding of SARS-CoV-2 pathogenesis. Based on results reported herein, potential inhibitors against S protein can be designed by considering these variations and their impact on protein structure.\n",
      "SARS-CoV-2 invades host cells via an endocytic pathway that begins with the interaction of the SARS-CoV-2 Spike glycoprotein (S-protein) and human Angiotensin-converting enzyme 2 (ACE2). Genetic variability in ACE2 may be one factor that mediates the broad-spectrum severity of SARS-CoV-2 infection and COVID-19 outcomes. We investigated the capacity of ACE2 variation to influence SARS-CoV-2 infection with a focus on predicting the effect of missense variants on the ACE2 SARS-CoV-2 S-protein interaction. We validated the mCSM-PPI2 variant effect prediction algorithm with 26 published ACE2 mutant SARS-CoV S-protein binding assays and found it performed well in this closely related system (True Positive Rate = 0.7, True Negative Rate = 1). Application of mCSM-PPI2 to ACE2 missense variants from the Genome Aggregation Consortium Database (gnomAD) identified three that are predicted to strongly inhibit or abolish the S-protein ACE2 interaction altogether (p.Glu37Lys, p.Gly352Val and p.Asp355Asn) and one that is predicted to promote the interaction (p.Gly326Glu). The S-protein ACE2 inhibitory variants are expected to confer a high degree of resistance to SARS-CoV-2 infection whilst the S-protein ACE2 affinity enhancing variant may lead to additional susceptibility and severity. We also performed in silico saturation mutagenesis of the S-protein ACE2 interface and identified a further 38 potential missense mutations that could strongly inhibit binding and one more that is likely to enhance binding (Thr27Arg). A conservative estimate places the prevalence of the strongly protective variants between 12-70 per 100,000 population but there is the possibility of higher prevalence in local populations or those underrepresented in gnomAD. The probable interplay between these ACE2 affinity variants and ACE2 expression polymorphisms is highlighted as well as gender differences in penetrance arising from ACE2’s situation on the X-chromosome. It is also described how our data can help power future genetic association studies of COVID-19 phenotypes and how the saturation mutant predictions can help design a mutant ACE2 with tailored S-protein affinity, which may be an improvement over a current recombinant ACE2 that is undergoing clinical trial. Key results 1 ACE2 gnomAD missense variant (p.Gly326Glu) and one unobserved missense mutation (Thr27Arg) are predicted to enhance ACE2 binding with SARS-CoV-2 Spike protein, which could result in increased susceptibility and severity of COVID-19 3 ACE2 missense variants in gnomAD plus another 38 unobserved missense mutations are predicted to inhibit Spike binding, these are expected to confer a high degree of resistance to infection The prevalence of the strongly protective variants is estimated between 12-70 per 100,000 population but higher prevalence may exist in local populations or those underrepresented in gnomAD A strategy to design a recombinant ACE2 with tailored affinity towards Spike and its potential therapeutic value is presented The predictions were extensively validated against published ACE2 mutant binding assays for SARS-CoV Spike protein\n",
      "Motivation The recent emergence of the novel SARS-coronavirus 2 (SARS-CoV-2) and its international spread pose a global health emergency. The viral spike (S) glycoprotein binds the receptor (angiotensin-converting enzyme 2) ACE2 and promotes SARS-CoV-2 entry into host cells. The trimeric S protein binds the receptor using the distal receptor-binding domain (RBD) causing conformational changes in S protein that allow priming by host cell proteases. Unravelling the dynamic structural features used by SARS-CoV-2 for entry might provide insights into viral transmission and reveal novel therapeutic targets. Using structures determined by X-ray crystallography and cryo-EM, we performed structural analysis and atomic comparisons of the different conformational states adopted by the SARS-CoV-2-RBD. Results Here, we determined the key structural components induced by the receptor and characterized their intramolecular interactions. We show that κ-helix (also known as polyproline II) is a predominant structure in the binding interface and in facilitating the conversion to the active form of the S protein. We demonstrate a series of conversions between switch-like κ-helix and β-strand, and conformational variations in a set of short α-helices which affect the proximal hinge region. This conformational changes lead to an alternating pattern in conserved disulfide bond configurations positioned at the hinge, indicating a possible disulfide exchange, an important allosteric switch implicated in viral entry of various viruses, including HIV and murine coronavirus. The structural information presented herein enables us to inspect and understand the important dynamic features of SARS-CoV-2-RBD and propose a novel potential therapeutic strategy to block viral entry. Overall, this study provides guidance for the design and optimization of structure-based intervention strategies that target SARS-CoV-2.\n",
      "The recent pandemic of COVID-19, caused by SARS-CoV-2, is unarguably the most fearsome compared with the earlier outbreaks caused by other coronaviruses, SARS-CoV and MERS-CoV. Human ACE2 is now established as a receptor for the SARS-CoV-2 spike protein. Where variations in the viral spike protein, in turn, lead to the cross-species transmission of the virus, genetic variations in the host receptor ACE2 may also contribute to the susceptibility and/or resistance against the viral infection. This study aims to explore the binding of the proteins encoded by different human ACE2 allelic variants with SARS-CoV-2 spike protein. Briefly, coding variants of ACE2 corresponding to the reported binding sites for its attachment with coronavirus spike protein were selected and molecular models of these variants were constructed by homology modeling. The models were then superimposed over the native ACE2 and ACE2-spike protein complex, to observe structural changes in the ACE2 variants and their intermolecular interactions with SARS-CoV-2 spike protein, respectively. Despite strong overall structural similarities, the spatial orientation of the key interacting residues varies in the ACE2 variants compared with the wild-type molecule. Most ACE2 variants showed a similar binding affinity for SARS-CoV-2 spike protein as observed in the complex structure of wild-type ACE2 and SARS-CoV-2 spike protein. However, ACE2 alleles, rs73635825 (S19P) and rs143936283 (E329G) showed noticeable variations in their intermolecular interactions with the viral spike protein. In summary, our data provide a structural basis of potential resistance against SARS-CoV-2 infection driven by ACE2 allelic variants.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: yop76n7z\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "The mutation pattern of severe acute respiratory syndrome-coronavirus 2 (SARS-CoV-2) is constantly changing with the places of transmission, but the reason remains to be revealed. Here, we presented the study that comprehensively analyzed the potential selective pressure of immune system restriction, which can drive mutations in circulating SARS-CoV-2 isolates. The results showed that the most common mutation sites of SARS-CoV-2 proteins were located on the non-structural protein ORF1ab and the structural protein Spike. Further analysis revealed mutations in cross-reactive epitopes between SARS-CoV-2 and seasonal coronavirus may help SARS-CoV-2 to escape cellular immunity under the long-term and large-scale community transmission. Meanwhile, the mutations on Spike protein may enhance the ability of SARS-CoV-2 to enter the host cells and escape the recognition of B-cell immunity. This study will increase the understanding of the evolutionary direction and warn about the potential immune escape ability of SARS-CoV-2, which may provide important guidance for the potential vaccine design.\n",
      "Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which first occurred in Wuhan (China) in December 2019, is a novel virus that causes a severe acute respiratory disease. The virus spike glycoproteins and nucleocapsid proteins are the main targets for the development of vaccines and antiviral drugs, to control the disease spread. We herein study the structural order-disorder propensity and the rates of evolution of these two proteins to characterize their B cell and T cell epitopes, previously suggested to contribute to immune response caused by SARS-CoV-2 infections. We first analyzed the rates of evolution along the sequences of spike and nucleocapsid proteins in relation to the spatial locations of their epitopes. For this purpose, we compared orthologs from seven human coronaviruses: SARS-CoV-2, SARS-CoV, MERS-CoV, HCoV-229E, HCoV-OC43, HCoV-NL63, and HCoV-HKU1. We then focus on the local, structural order-disorder propensities of the protein regions where the SARS-CoV-2 epitopes are located. We show that the vast majority of nucleocapsid protein epitopes overlap the RNA-binding and dimerization domains and some of them are characterized by low rates of evolutions. Similarly, spike protein epitopes are preferentially located in regions that are predicted to be ordered and well-conserved, in correspondence of the heptad repeats 1 and 2. Interestingly, both the receptor-binding motif to ACE2 and the fusion peptide of spike protein are characterized by high rates of evolution, probably to overcome host immunity. In conclusion, our results provide evidence for conserved epitopes that may help to develop long-lasting, broad-spectrum SARS-CoV-2 vaccines.\n",
      "Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a highly pathogenic virus that has caused the global COVID-19 pandemic. Tracing the evolution and transmission of the virus is crucial to respond to and control the pandemic through appropriate intervention strategies. This paper reports and analyses genomic mutations in the coding regions of SARS-CoV-2 and their probable protein secondary structure and solvent accessibility changes, which are predicted using deep learning models. Prediction results suggest that mutation D614G in the virus spike protein, which has attracted much attention from researchers, is unlikely to make changes in protein secondary structure and relative solvent accessibility. Based on 6,324 viral genome sequences, we create a spreadsheet dataset of point mutations that can facilitate the investigation of SARS-CoV-2 in many perspectives, especially in tracing the evolution and worldwide spread of the virus. Our analysis results also show that coding genes E, M, ORF6, ORF7a, ORF7b and ORF10 are most stable, potentially suitable to be targeted for vaccine and drug development.\n",
      "The recent emergence of a novel coronavirus (SARS-CoV-2) is causing a severe global health threat characterized by severe acute respiratory syndrome (Covid-19). At the moment, there is no specific treatment for this disease, and vaccines are still under development. The structural protein Spike is essential for virus infection and has been used as the main target for vaccine and serological diagnosis test development. We analysed 2363 sequences of the Spike protein from SARS-CoV-2 isolates and identified variability in 44 amino acid residues and their worldwide distribution in all continents. We used the three-dimensional structure of the homo-trimer model to predict conformational epitopes of B-cell, and sequence of Spike protein Wuhan-Hu-1 to predict linear epitopes of T-Cytotoxic and T-Helper cells. We identified 45 epitopes with amino acid variations. Finally, we showed the distribution of mutations within the epitopes. Our findings can help researches to identify more efficient strategies for the development of vaccines, therapies, and serological diagnostic tests based on the Spike protein of Sars-Cov-2.\n",
      "A new severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) associated with human to human transmission and extreme human sickness has been as of late announced from the city of Wuhan in China. Our objectives were to mutation analysis between recently reported genomes at various times and locations and to characterize the genomic structure of SARS-CoV-2 using bioinformatics programs. Information on the variation of viruses is of considerable medical and biological impacts on the prevention, diagnosis, and therapy of infectious diseases. To understand the genomic structure and variations of the SARS-CoV-2. The study analyzed 95 SARS-CoV-2 complete genome sequences available in GenBank, National MicrobiologyData Center (NMDC) and NGDC Genome Warehouse from December-2019 until 05 of April-2020. The genomic signature analysis demonstrates that a strong association between the time of sample collection, location of sample and accumulation of genetic diversity. We found 116 mutations, the three most common mutations were 8782C>T in ORF1ab gene, 28144T>C in ORF8 gene and 29095C>T in the N gene. The mutations might affect the severity and spread of the SARS-CoV-2. The finding heavily supports an intense requirement for additional prompt, inclusive investigations that combine genomic detail, epidemiological information and graph records of the clinical features of patients with COVID-19.\n",
      "• Mutation studies hits cause for the superior infectious rate of SARS-CoV-2. • SARS-CoV-2 is evolving rapidly with number of mutations and SNPs. • Mutations were found in the ACE2 binding region of SARS-CoV-2 spike glycoprotein. • ORF1ab, ORF8 and spike glycoprotein regions of SARS-CoV-2 are highly mutated. • Overwhelming mutations or SNPs of SARS-CoV-2 alerts drug treatment options.\n",
      "The 2019 novel severe respiratory syndrome coronavirus-2 (SARS-CoV-2) outbreak has caused a large number of deaths with thousands of confirmed cases worldwide. The present study followed computational approaches to identify B- and T-cell epitopes for spike glycoprotein of SARS-CoV-2 by its interactions with the human leukocyte antigen alleles. We identified twenty-four peptide stretches on the SARS-CoV-2 spike protein that are well conserved among the reported strains. The S protein structure further validated the presence of predicted peptides on the surface. Out of which twenty are surface exposed and predicted to have reasonable epitope binding efficiency. The work could be useful for understanding the immunodominant regions in the surface protein of SARS-CoV-2 and could potentially help in designing some peptide-based diagnostics.\n",
      "The Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) pandemic offers a unique opportunity to study the introduction and evolution of a pathogen into a completely naïve human population. We identified and analysed the amino acid mutations that gained prominence worldwide in the early months of the pandemic. Eight mutations have been identified along the viral genome, mostly located in conserved segments of the structural proteins and showing low variability among coronavirus, which indicated that they might have a functional impact. At the moment of writing this paper, these mutations present a varied success in the SARS-CoV-2 virus population; ranging from a change in the spike protein that becomes absolutely prevalent, two mutations in the nucleocapsid protein showing frequencies around 25%, to a mutation in the matrix protein that nearly fades out after reaching a frequency of 20%.\n",
      "The 2019 novel severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) outbreak has caused a large number of deaths, with thousands of confirmed cases worldwide. The present study followed computational approaches to identify B- and T-cell epitopes for the spike (S) glycoprotein of SARS-CoV-2 by its interactions with the human leukocyte antigen alleles. We identified 24 peptide stretches on the SARS-CoV-2 S protein that are well conserved among the reported strains. The S protein structure further validated the presence of predicted peptides on the surface, of which 20 are surface exposed and predicted to have reasonable epitope binding efficiency. The work could be useful for understanding the immunodominant regions in the surface protein of SARS-CoV-2 and could potentially help in designing some peptide-based diagnostics. Also, identified T-cell epitopes might be considered for incorporation in vaccine designs.\n",
      "The causative agent of the coronavirus disease 2019 (COVID-19) pandemic, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), is steadily mutating during continuous transmission among humans. Such mutations can occur in the spike (S) protein that binds to the angiotensin-converting enzyme-2 (ACE2) receptor and is cleaved by transmembrane protease serine 2 (TMPRSS2). However, whether S mutations affect SARS-CoV-2 infectivity remains unknown. Here, we show that naturally occurring S mutations can reduce or enhance cell entry via ACE2 and TMPRSS2. A SARS-CoV-2 S-pseudotyped lentivirus exhibits substantially lower entry than SARS-CoV S. Among S variants, the D614G mutant shows the highest cell entry, as supported by structural observations. Nevertheless, the D614G mutant remains susceptible to neutralization by antisera against prototypic viruses. Taken together, these data indicate that the D614G mutation enhances viral infectivity while maintaining neutralization susceptibility.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 4hmecfi0\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "We have developed an analysis pipeline to facilitate real-time mutation tracking in SARS-CoV-2, focusing initially on the Spike (S) protein because it mediates infection of human cells and is the target of most vaccine strategies and antibody-based therapeutics. To date we have identified thirteen mutations in Spike that are accumulating. Mutations are considered in a broader phylogenetic context, geographically, and over time, to provide an early warning system to reveal mutations that may confer selective advantages in transmission or resistance to interventions. Each one is evaluated for evidence of positive selection, and the implications of the mutation are explored through structural modeling. The mutation Spike D614G is of urgent concern; it began spreading in Europe in early February, and when introduced to new regions it rapidly becomes the dominant form. Also, we present evidence of recombination between locally circulating strains, indicative of multiple strain infections. These finding have important implications for SARS-CoV-2 transmission, pathogenesis and immune interventions.\n",
      "The recent emergence of a novel coronavirus (SARS-CoV-2) is causing a severe global health threat characterized by severe acute respiratory syndrome (Covid-19). At the moment, there is no specific treatment for this disease, and vaccines are still under development. The structural protein Spike is essential for virus infection and has been used as the main target for vaccine and serological diagnosis test development. We analysed 2363 sequences of the Spike protein from SARS-CoV-2 isolates and identified variability in 44 amino acid residues and their worldwide distribution in all continents. We used the three-dimensional structure of the homo-trimer model to predict conformational epitopes of B-cell, and sequence of Spike protein Wuhan-Hu-1 to predict linear epitopes of T-Cytotoxic and T-Helper cells. We identified 45 epitopes with amino acid variations. Finally, we showed the distribution of mutations within the epitopes. Our findings can help researches to identify more efficient strategies for the development of vaccines, therapies, and serological diagnostic tests based on the Spike protein of Sars-Cov-2.\n",
      "The spike (S) protein of SARS-CoV-2 mediates receptor binding and cell entry and is the dominant target of the immune system. S exhibits substantial conformational flexibility. It transitions from closed to open conformations to expose its receptor binding site, and subsequently from prefusion to postfusion conformations to mediate fusion of viral and cellular membranes. S protein derivatives are components of vaccine candidates and diagnostic assays, as well as tools for research into the biology and immunology of SARS-CoV-2. Here we have designed mutations in S which allow production of thermostable, crosslinked, S protein trimers that are trapped in the closed, pre-fusion, state. We have determined the structures of crosslinked and non-crosslinked proteins, identifying two distinct closed conformations of the S trimer. We demonstrate that the designed, thermostable, closed S trimer can be used in serological assays. This protein has potential applications as a reagent for serology, virology and as an immunogen.\n",
      "The COVID-19 pandemic caused by the novel coronavirus SARS-CoV-2 has led to accelerated efforts to develop therapeutics, diagnostics, and vaccines to mitigate this public health emergency. A key target of these efforts is the spike (S) protein, a large trimeric class I fusion protein that is metastable and difficult to produce recombinantly in large quantities. Here, we designed and expressed over 100 structure-guided spike variants based upon a previously determined cryo-EM structure of the prefusion SARS-CoV-2 spike. Biochemical, biophysical and structural characterization of these variants identified numerous individual substitutions that increased protein yields and stability. The best variant, HexaPro, has six beneficial proline substitutions leading to ~10-fold higher expression than its parental construct and is able to withstand heat stress, storage at room temperature, and multiple freeze-thaws. A 3.2 Å-resolution cryo-EM structure of HexaPro confirmed that it retains the prefusion spike conformation. High-yield production of a stabilized prefusion spike protein will accelerate the development of vaccines and serological diagnostics for SARS-CoV-2.\n",
      "The newly identified SARS-CoV-2 has now been reported from around 185 countries with more than a million confirmed human cases including more than 120,000 deaths. The genomes of SARS-COV-2 strains isolated from different parts of the world are now available and the unique features of constituent genes and proteins need to be explored to understand the biology of the virus. Spike glycoprotein is one of the major targets to be explored because of its role during the entry of coronaviruses into host cells. We analyzed 320 whole-genome sequences and 320 spike protein sequences of SARS-CoV-2 using multiple sequence alignment. In this study, 483 unique variations have been identified among the genomes of SARS-CoV-2 including 25 nonsynonymous mutations and one deletion in the spike (S) protein. Among the 26 variations detected, 12 variations were located at the N-terminal domain and 6 variations at the receptor-binding domain (RBD) which might alter the interaction of S protein with the host receptor angiotensin converting enzyme-2 (ACE2). Besides, 22 amino acid insertions were identified in the spike protein of SARS-CoV-2 in comparison with that of SARS-CoV. Phylogenetic analyses of spike protein revealed that Bat coronavirus have a close evolutionary relationship with circulating SARS-CoV-2. The genetic variation analysis data presented in this study can help a better understanding of SARS-CoV-2 pathogenesis. Based on results reported herein, potential inhibitors against S protein can be designed by considering these variations and their impact on protein structure.\n",
      "The SARS-CoV-2 spike (S) protein, the viral mediator for binding and entry into the host cell, has sparked great interest as a target for vaccine development and treatments with neutralizing antibodies. Initial data suggest that the virus has low mutation rates, but its large genome could facilitate recombination, insertions, and deletions, as has been described in other coronaviruses. Here, we deep-sequenced the complete SARS-CoV-2 S gene from 18 patients (10 with mild and 8 with severe COVID-19), and found that the virus accumulates deletions upstream and very close to the S1/S2 cleavage site, generating a frameshift with appearance of a stop codon. These deletions were found in a small percentage of the viral quasispecies (2.2%) in samples from all the mild and only half the severe COVID-19 patients. Our results suggest that the virus may generate free S1 protein released to the circulation. We propose that natural selection has favored a “Don’t burn down the house” strategy, in which free S1 protein may compete with viral particles for the ACE2 receptor, thus reducing the severity of the infection and tissue damage without losing transmission capability.\n",
      "Considering the current status of the SARS-CoV-2 pandemic, sequence variations and possibly structural changes in the rapidly evolving SARS-CoV-2 is highly expected in the coming months. The SARS-CoV-2 spike (S) protein is responsible for mediating viral attachment and fusion with cell membranes. Mutations in the receptor-binding domain (RBD) of the S-protein occur at the most variable part of the SARS-CoV-2 genome, and specific sites of S-protein have undergone positive selection impacting the viral pathogenicity. In the present work, we used high-throughput computation to design 100,000 mutants in RBD interfacial residues and identify novel affinity-enhancing and affinity-weakening mutations. Our data suggest that SARS-CoV-2 can establish a higher rate of infectivity and pathogenesis when it acquires combinatorial mutations at the interfacial residues in RBD. Mapping of the mutational landscape of the interaction site suggests that a few of these residues are the hot-spot residues with a very high tendency to undergo positive selection. Knowledge of the affinity-enhancing mutations may guide the identification of potential cold-spots for this mutation as targets for developing a possible therapeutic strategy instead of hot-spots, and vice versa. Understanding of the molecular interactions between the virus and host protein presents a detailed systems view of viral infection mechanisms. The applications of the present research can be explored in multiple antiviral strategies, including monoclonal antibody therapy, vaccine design, and importantly in understanding the clinical pathogenesis of the virus itself. Our work presents research directions for the exploitation of non-conventional solutions for COVID-19.\n",
      "A novel isolate of the SARS-CoV-2 virus carrying a point mutation in the Spike protein (D614G) has recently emerged and rapidly surpassed others in prevalence. This mutation is in linkage disequilibrium with an ORF1b protein variant (P314L), making it difficult to discern the functional significance of the Spike D614G mutation from population genetics alone. Here, we perform site-directed mutagenesis to introduce the D614G variant and show that in multiple cell lines, including human lung epithelial cells, that the D614G mutation is up to 8-fold more effective at transducing cells than wild-type. We demonstrate increased infection using both Spike-pseudotyped lentivirus and intact SARS-CoV-2 virus. Although there is minimal difference in ACE2 receptor binding between the Spike variants, we show that the G614 variant is more resistant to proteolytic cleavage in vitro and in human cells, suggesting a possible mechanism for the increased transduction. This result has important implications for the efficacy of Spike-based vaccines currently under development in protecting against this recent and highly-prevalent SARS-CoV-2 isolate.\n",
      "The 2019 novel severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) outbreak has caused a large number of deaths, with thousands of confirmed cases worldwide. The present study followed computational approaches to identify B- and T-cell epitopes for the spike (S) glycoprotein of SARS-CoV-2 by its interactions with the human leukocyte antigen alleles. We identified 24 peptide stretches on the SARS-CoV-2 S protein that are well conserved among the reported strains. The S protein structure further validated the presence of predicted peptides on the surface, of which 20 are surface exposed and predicted to have reasonable epitope binding efficiency. The work could be useful for understanding the immunodominant regions in the surface protein of SARS-CoV-2 and could potentially help in designing some peptide-based diagnostics. Also, identified T-cell epitopes might be considered for incorporation in vaccine designs.\n",
      "The newly identified SARS-CoV-2 has now been reported from around 185 countries with more than a million confirmed human cases including more than 120,000 deaths. The genomes of SARS-COV-2 strains isolated from different parts of the world are now available and the unique features of constituent genes and proteins need to be explored to understand the biology of the virus. Spike glycoprotein is one of the major targets to be explored because of its role during the entry of coronaviruses into host cells. We analyzed 320 whole-genome sequences and 320 spike protein sequences of SARS-CoV-2 using multiple sequence alignment. In this study, 483 unique variations have been identified among the genomes of SARS-CoV-2 including 25 nonsynonymous mutations and one deletion in the spike (S) protein. Among the 26 variations detected in S, 12 variations were located at the N-terminal domain (NTD) and 6 variations at the receptor-binding domain (RBD) which might alter the interaction of S protein with the host receptor angiotensin-converting enzyme 2 (ACE2). Besides, 22 amino acid insertions were identified in the spike protein of SARS-CoV-2 in comparison with that of SARS-CoV. Phylogenetic analyses of spike protein revealed that Bat coronavirus have a close evolutionary relationship with circulating SARS-CoV-2. The genetic variation analysis data presented in this study can help a better understanding of SARS-CoV-2 pathogenesis. Based on results reported herein, potential inhibitors against S protein can be designed by considering these variations and their impact on protein structure.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: t996fbad\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "The outbreak of the 2019-nCoV infection began in December 2019 in Wuhan, Hubei province, and rapidly spread to many provinces in China as well as other countries. Here we report the epidemiological, clinical, laboratory, and radiological characteristics, as well as potential biomarkers for predicting disease severity in 2019-nCoV-infected patients in Shenzhen, China. All 12 cases of the 2019-nCoV-infected patients developed pneumonia and half of them developed acute respiratory distress syndrome (ARDS). The most common laboratory abnormalities were hypoalbuminemia, lymphopenia, decreased percentage of lymphocytes (LYM) and neutrophils (NEU), elevated C-reactive protein (CRP) and lactate dehydrogenase (LDH), and decreased CD8 count. The viral load of 2019-nCoV detected from patient respiratory tracts was positively linked to lung disease severity. ALB, LYM, LYM (%), LDH, NEU (%), and CRP were highly correlated to the acute lung injury. Age, viral load, lung injury score, and blood biochemistry indexes, albumin (ALB), CRP, LDH, LYM (%), LYM, and NEU (%), may be predictors of disease severity. Moreover, the Angiotensin II level in the plasma sample from 2019-nCoV infected patients was markedly elevated and linearly associated to viral load and lung injury. Our results suggest a number of potential diagnosis biomarkers and angiotensin receptor blocker (ARB) drugs for potential repurposing treatment of 2019-nCoV infection. ELECTRONIC SUPPLEMENTARY MATERIAL: Supplementary material is available for this article at 10.1007/s11427-020-1643-8 and is accessible for authorized users.\n",
      "The outbreak of the 2019-nCoV infection began in December 2019 in Wuhan, Hubei province, and rapidly spread to many provinces in China as well as other countries. Here we report the epidemiological, clinical, laboratory, and radiological characteristics, as well as potential biomarkers for predicting disease severity in 2019-nCoV-infected patients in Shenzhen, China. All 12 cases of the 2019-nCoV-infected patients developed pneumonia and half of them developed acute respiratory distress syndrome (ARDS). The most common laboratory abnormalities were hypoalbuminemia, lymphopenia, decreased percentage of lymphocytes (LYM) and neutrophils (NEU), elevated C-reactive protein (CRP) and lactate dehydrogenase (LDH), and decreased CD8 count. The viral load of 2019-nCoV detected from patient respiratory tracts was positively linked to lung disease severity. ALB, LYM, LYM (%), LDH, NEU (%), and CRP were highly correlated to the acute lung injury. Age, viral load, lung injury score, and blood biochemistry indexes, albumin (ALB), CRP, LDH, LYM (%), LYM, and NEU (%), may be predictors of disease severity. Moreover, the Angiotensin II level in the plasma sample from 2019-nCoV infected patients was markedly elevated and linearly associated to viral load and lung injury. Our results suggest a number of potential diagnosis biomarkers and angiotensin receptor blocker (ARB) drugs for potential repurposing treatment of 2019-nCoV infection.\n",
      "Background: Since December 2019, acute respiratory disease (ARD) due to 2019 novel coronavirus (2019-nCoV) emerged in Wuhan city and rapidly spread throughout China. We sought to delineate the clinical characteristics of these cases. Methods: We extracted the data on 1,099 patients with laboratory-confirmed 2019-nCoV ARD from 552 hospitals in 31 provinces/provincial municipalities through January 29th, 2020. Results: The median age was 47.0 years, and 41.90% were females. Only 1.18% of patients had a direct contact with wildlife, whereas 31.30% had been to Wuhan and 71.80% had contacted with people from Wuhan. Fever (87.9%) and cough (67.7%) were the most common symptoms. Diarrhea is uncommon. The median incubation period was 3.0 days (range, 0 to 24.0 days). On admission, ground-glass opacity was the typical radiological finding on chest computed tomography (50.00%). Significantly more severe cases were diagnosed by symptoms plus reverse-transcriptase polymerase-chain-reaction without abnormal radiological findings than non-severe cases (23.87% vs. 5.20%, P<0.001). Lymphopenia was observed in 82.1% of patients. 55 patients (5.00%) were admitted to intensive care unit and 15 (1.36%) succumbed. Severe pneumonia was independently associated with either the admission to intensive care unit, mechanical ventilation, or death in multivariate competing-risk model (sub-distribution hazards ratio, 9.80; 95% confidence interval, 4.06 to 23.67). Conclusions: The 2019-nCoV epidemic spreads rapidly by human-to-human transmission. Normal radiologic findings are present among some patients with 2019-nCoV infection. The disease severity (including oxygen saturation, respiratory rate, blood leukocyte/lymphocyte count and chest X-ray/CT manifestations) predict poor clinical outcomes.\n",
      "BACKGROUND: The 2019 novel coronavirus (2019-nCoV) causing an outbreak of pneumonia in Wuhan, Hubei province of China was isolated in January 2020. This study aims to investigate its epidemiologic history, and analyze the clinical characteristics, treatment regimens, and prognosis of patients infected with 2019-nCoV during this outbreak. METHODS: Clinical data from 137 2019-nCoV-infected patients admitted to the respiratory departments of nine tertiary hospitals in Hubei province from December 30, 2019 to January 24, 2020 were retrospectively collected, including general status, clinical manifestations, laboratory test results, imaging characteristics, and treatment regimens. RESULTS: None of the 137 patients (61 males, 76 females, aged 20-83 years, median age 57 years) had a definite history of exposure to Huanan Seafood Wholesale Market. Major initial symptoms included fever (112/137, 81.8%), coughing (66/137, 48.2%), and muscle pain or fatigue (44/137, 32.1%), with other, less typical initial symptoms observed at low frequency, including heart palpitations, diarrhea, and headache. Nearly 80% of the patients had normal or decreased white blood cell counts, and 72.3% (99/137) had lymphocytopenia. Lung involvement was present in all cases, with most chest computed tomography scans showing lesions in multiple lung lobes, some of which were dense; ground-glass opacity co-existed with consolidation shadows or cord-like shadows. Given the lack of effective drugs, treatment focused on symptomatic and respiratory support. Immunoglobulin G was delivered to some critically ill patients according to their conditions. Systemic corticosteroid treatment did not show significant benefits. Notably, early respiratory support facilitated disease recovery and improved prognosis. The risk of death was primarily associated with age, underlying chronic diseases, and median interval from the appearance of initial symptoms to dyspnea. CONCLUSIONS: The majority of patients with 2019-nCoV pneumonia present with fever as the first symptom, and most of them still showed typical manifestations of viral pneumonia on chest imaging. Middle-aged and elderly patients with underlying comorbidities are susceptible to respiratory failure and may have a poorer prognosis.\n",
      "BACKGROUND: Since December 2019, 2019 novel coronavirus pneumonia emerged in Wuhan city and rapidly spread throughout China and even the world. We sought to analyse the clinical characteristics and laboratory findings of some cases with 2019 novel coronavirus pneumonia . METHODS: In this retrospective study, we extracted the data on 95 patients with laboratory-confirmed 2019 novel coronavirus pneumonia in Wuhan Xinzhou District People’s Hospital from January 16th to February 25th, 2020. Cases were confirmed by real-time RT-PCR and abnormal radiologic findings. Outcomes were followed up until March 2th, 2020. RESULTS: Higher temperature, blood leukocyte count, neutrophil count, neutrophil percentage, C-reactive protein level, D-dimer level, alanine aminotransferase activity, aspartate aminotransferase activity, α - hydroxybutyrate dehydrogenase activity, lactate dehydrogenase activity and creatine kinase activity were related to severe 2019 novel coronavirus pneumonia and composite endpoint, and so were lower lymphocyte count, lymphocyte percentage and total protein level. Age below 40 or above 60 years old, male, higher Creatinine level, and lower platelet count also seemed related to severe 2019 novel coronavirus pneumonia and composite endpoint, however the P values were greater than 0.05, which mean under the same condition studies of larger samples are needed in the future. CONCLUSION: Multiple factors were related to severe 2019 novel coronavirus pneumonia and composite endpoint, and more related studies are needed in the future.\n",
      "BACKGROUND: A novel coronavirus (2019-nCoV) causing an outbreak of pneumonia in Wuhan, Hubei province of China was isolated in January 2020. This study aims to investigate its epidemiologic history, and analyze the clinical characteristics, treatment regimens, and prognosis of patients infected with 2019-nCoV during this outbreak. METHODS: Clinical data from 137 2019-nCoV-infected patients admitted to the respiratory departments of the respiratory departments of nine tertiary hospitals in Hubei province from December 30, 2019 to January 24, 2020 were retrospectively collected, including general status, clinical manifestations, laboratory test results, imaging characteristics, and treatment regimens. RESULTS: None of the 137 patients (61 males, 76 females, aged 20–83 years, median age 57 years) had a definite history of exposure to Huanan Seafood Wholesale Market. Major initial symptoms included fever (112/137, 81.8%), coughing (66/137, 48.2%), and muscle pain or fatigue (44/137, 32.1%), with other, less typical initial symptoms observed at low frequency, including heart palpitations, diarrhea, and headache. Nearly 80% of the patients had normal or decreased white blood cell counts, and 72.3% (99/137) had lymphocytopenia. Lung involvement was present in all cases, with most chest computed tomography scans showing lesions in multiple lung lobes, some of which were dense; ground-glass opacity co-existed with consolidation shadows or cord-like shadows. Given the lack of effective drugs, treatment focused on symptomatic and respiratory support. Immunoglobulin G was delivered to some critically ill patients according to their conditions. Systemic corticosteroid treatment did not show significant benefits. Notably, early respiratory support facilitated disease recovery and improved prognosis. The risk of death was primarily associated with age, underlying chronic diseases, and median interval from the appearance of initial symptoms to dyspnea. CONCLUSIONS: The majority of patients with 2019-nCoV pneumonia present with fever as the first symptom, and most of them still showed typical manifestations of viral pneumonia on chest imaging. Middle-aged and elderly patients with underlying comorbidities are susceptible to respiratory failure and may have a poorer prognosis.\n",
      "The outbreak of the novel coronavirus disease 2019 (COVID-19) infection began in December 2019 in Wuhan, and rapidly spread to many provinces in China. The number of cases has increased markedly in Anhui, but information on the clinical characteristics of patients is limited. We reported 75 patients with COVID-19 in the First Affiliated Hospital of USTC from Jan 21 to Feb 16, 2020, Hefei, Anhui Province, China. COVID-19 infection was confirmed by real-time RT-PCR of respiratory nasopharyngeal swab samples. Epidemiological, clinical and laboratory data were collected and analyzed. Of the 75 patients with COVID-19, 61 (81.33%) had a direct or indirect exposure history to Wuhan. Common symptoms at onset included fever (66 [88.0%] of 75 patients) and dry cough (62 [82.67%]). Of the patients without fever, cough could be the only or primary symptom. The most prominent laboratory abnormalities were lymphopenia, decreased percentage of lymphocytes (LYM%), decreased CD4+ and CD8+ T cell counts, elevated C-reactive protein (CRP) and lactate dehydrogenase (LDH). Patients with elevated interleukin 6 (IL-6) showed significant decreases in the LYM%, CD4+ and CD8+ T cell counts. Besides, the percentage of neutrophils, CRP, LDH and Procalcitonin levels increased significantly. We concluded that COVID-19 could cause different degrees of hematological abnormalities and damage of internal organs. Hematological profiles including LYM, LDH, CRP and IL-6 could be indicators of diseases severity and evaluation of treatment effectiveness. Antiviral treatment requires a comprehensive and supportive approach. Further targeted therapy should be determined based on individual clinical manifestations and laboratory indicators.\n",
      "BACKGROUND: Since December 2019, 2019 novel coronavirus pneumonia emerged in Wuhan city and rapidly spread throughout China and even the world. We sought to analyse the clinical characteristics and laboratory findings of some cases with 2019 novel coronavirus pneumonia . METHODS: In this retrospective study, we extracted the data on 95 patients with laboratory-confirmed 2019 novel coronavirus pneumonia in Wuhan Xinzhou District People's Hospital from January 16th to February 25th, 2020. Cases were confirmed by real-time RT-PCR and abnormal radiologic findings. Outcomes were followed up until March 2th, 2020. RESULTS: Higher temperature, blood leukocyte count, neutrophil count, neutrophil percentage, C-reactive protein level, D-dimer level, alanine aminotransferase activity, aspartate aminotransferase activity, &#945; - hydroxybutyrate dehydrogenase activity, lactate dehydrogenase activity and creatine kinase activity were related to severe 2019 novel coronavirus pneumonia and composite endpoint, and so were lower lymphocyte count, lymphocyte percentage and total protein level. Age below 40 or above 60 years old, male, higher Creatinine level, and lower platelet count also seemed related to severe 2019 novel coronavirus pneumonia and composite endpoint, however the P values were greater than 0.05, which mean under the same condition studies of larger samples are needed in the future. CONCLUSION: Multiple factors were related to severe 2019 novel coronavirus pneumonia and composite endpoint, and more related studies are needed in the future.\n",
      "OBJECTIVE In December 2019, a new type of coronavirus-infected pneumonia broke out in Wuhan and spread rapidly to other parts of the country. The purpose of this study was to investigate the clinical features of coronavirus disease 2019 (COVID-19). MATERIALS AND METHODS A retrospective analysis was performed on the confirmed cases of COVID-19, who were admitted to the North Hospital of Changsha first Hospital (Changsha Public Health treatment Center) from January 17 to February 7, 2020. RESULTS The median age of COVID-19 patients was 45 years (range 33.5-57). The male patients accounted for 49.7%, 64.6% of the patients had a history of exposure in Wuhan, and 31.7% had family aggregation. The median days of onset were six, and the incidence of severe illness was 18.6%. Compared with the non-severe group, the severe group showed statistical significance in older age, hypertension, bilateral lung plaque shadow, decrease in lymphocyte count, increase in C-reactive protein (CRP), aspartate aminotransferase (AST), lactate dehydrogenase, and creatine kinase. CONCLUSIONS Age, combined hypertension, oxygenation index, double lung patch, decreased lymphocyte count, and elevated levels of C-reactive protein, aspartate aminotransferase, lactate dehydrogenase, and creatine kinase can be used as predictors of the disease severity.\n",
      "OBJECTIVE: In December 2019, a new type of coronavirus-infected pneumonia broke out in Wuhan and spread rapidly to other parts of the country. The purpose of this study was to investigate the clinical features of coronavirus disease 2019 (COVID-19). MATERIALS AND METHODS: A retrospective analysis was performed on the confirmed cases of COVID-19, who were admitted to the North Hospital of Changsha first Hospital (Changsha Public Health treatment Center) from January 17 to February 7, 2020. RESULTS: The median age of COVID-19 patients was 45 years (range 33.5-57). The male patients accounted for 49.7%, 64.6% of the patients had a history of exposure in Wuhan, and 31.7% had family aggregation. The median days of onset were six, and the incidence of severe illness was 18.6%. Compared with the non-severe group, the severe group showed statistical significance in older age, hypertension, bilateral lung plaque shadow, decrease in lymphocyte count, increase in C-reactive protein (CRP), aspartate aminotransferase (AST), lactate dehydrogenase, and creatine kinase. CONCLUSIONS: Age, combined hypertension, oxygenation index, double lung patch, decreased lymphocyte count, and elevated levels of C-reactive protein, aspartate aminotransferase, lactate dehydrogenase, and creatine kinase can be used as predictors of the disease severity.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: f37cp6j3\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "During this global pandemic of COVID-19 infection, it became well known that morbidity and mortality is especially high at the extreme of life especially in certain racial or ethnic groups like Americans and Africans This is presumed due to low immunity associated with other comorbid conditions like diabetes, hypertension, cardiovascular disease, obesity and metabolic syndrome But the information available on the immune status of COVID-19 patients is limited Attempts must be made to enhance our understanding of the immune status of COVID-19 patients by revisiting our knowledge on the immune mechanisms of already known coronaviruses such as SARS-CoV and MERS-CoV Early elevation of the serum levels of pro-inflammatory cytokines observed in SARS and MERS infection suggests a possible same type of cytokine storm-mediated lung damage in COVID-19 patients too Dysregulation of interferon-1 response and downstream cascade in initial innate immune response at virus entry point has been related to lethal pneumonia in COVID-19 patients Adaptive response of increased CD8+ levels in COVID-19 patients seems to be useful in mild cases where it causes deteriorating effects in progressed severe disease patients resulting in destruction of type 2 pneumocytes hence inability to regenerate the alveolar epithelium A phenomenon called cytokine storm activates violent immunological reactions in the lung tissue resulting in ARDS followed by multiple organ system damages in COVID-19 patients Several immune evading mechanisms are thought to be employed by severe respiratory syndrome virus-2 (SARS-CoV-2) that might have resulted in its extremely increased contagiousness probably related with its frequent RNA mutations Failure to develop adequate virus limiting immune reactions by some cured patients warrant monitoring of all recovered patients This rapid mini review is aimed to enhance our knowledge of the immune status of COVID-19 infected patients with reference to SARS-CoV and MERS-CoV\n",
      "SARS-CoV-2 has been identified as the causative agent of a global outbreak of respiratory tract disease (COVID-19). In some patients the infection results in moderate to severe acute respiratory distress syndrome (ARDS), requiring invasive mechanical ventilation. High serum levels of IL-6, IL-10 and an immune hyperresponsiveness referred to as a 'cytokine storm' have been associated with poor clinical outcome. Despite the large numbers of COVID-19 cases and deaths, information on the phenotype and kinetics of SARS-CoV-2-specific T cells is limited. Here, we studied 10 COVID-19 patients who required admission to an intensive care unit and detected SARS-CoV-2-specific CD4+ and CD8+ T cells in 10 out of 10 and 8 out of 10 patients, respectively. We also detected low levels of SARS-CoV-2-reactive T cells in 2 out of 10 healthy controls not previously exposed to SARS-CoV-2, which is indicative of cross-reactivity due to past infection with 'common cold' coronaviruses. The strongest T-cell responses were directed to the spike (S) surface glycoprotein, and SARS-CoV-2-specific T cells predominantly produced effector and Th1 cytokines, although Th2 and Th17 cytokines were also detected. Furthermore, we studied T-cell kinetics and showed that SARS-CoV-2-specific T cells are present relatively early and increase over time. Collectively, these data shed light on the potential variations in T-cell responses as a function of disease severity, an issue that is key to understanding the potential role of immunopathology in the disease, and also inform vaccine design and evaluation.\n",
      "BACKGROUND AND AIMS: The outbreak of the new coronavirus, SARS-CoV-2, causes a respiratory disease and individuals with pre-existing cardiometabolic disorders display worse prognosis through the infection course. The aim of this minireview is to present epidemiological data related to metabolic comorbidities in association with the SARS-CoV-2. METHODS: This is a narrative mini-review with Pubmed search until April 23, 2020 using the keywords COVID-19, SARS-CoV-2, treatment of coronavirus and following terms: diabetes mellitus, obesity, arterial hypertension, ACE-inhibitors, cytokine storm, immune response and vitamin D. RESULTS: Studies indicate that obese individuals are more likely to develop infections, and that adipose tissue serves as a pathogen reservoir. In diabetic individuals higher rate of inflammatory processes is seen due to constant glucose recognition by C type lectin receptors. Hypertensive individuals, usually grouped with other conditions, are treated with drugs to reduce blood pressure mostly through ACEi and ARB, that leads to increased ACE2 expression, used by SARS-CoV-2 for human's cell entry. Until now, the studies have shown that individuals with those conditions and affected by COVID-19 present an uncontrolled release of pro-inflammatory cytokines and an unbalanced immune response, leading to the cytokine storm phenomenon. Vitamin D is highlighted as a potential therapeutic target, because in addition to acting on the immune system, it plays an important role in the control of cardiometabolic diseases. CONCLUSION: Currently, since there is no proven and effective antiviral therapy for SARS-CoV-2, the efforts should focus on controlling inflammatory response and reduce the risks of associated complications.\n",
      "A sudden outbreak of COVID-19 caused by a novel coronavirus, SARS-CoV-2, in Wuhan, China in December 2019 quickly grew into a global pandemic, putting at risk not only the global healthcare system, but also the world economy. As the disease continues to spread rapidly, the development of prophylactic and therapeutic approaches is urgently required. Although some progress has been made in understanding the viral structure and invasion mechanism of coronaviruses that may cause severe cases of the syndrome, due to the limited understanding of the immune effects caused by SARS-CoV-2, it is difficult for us to prevent patients from developing acute respiratory distress syndrome (ARDS) and pulmonary fibrosis (PF), the major complications of coronavirus infection. Therefore, any potential treatments should focus not only on direct killing of coronaviruses and prevention strategies by vaccine development, but also on keeping in check the acute immune/inflammatory responses, resulting in ARDS and PF. In addition, potential treatments currently under clinical trials focusing on killing coronaviruses or on developing vaccines preventing coronavirus infection largely ignore the host immune response. However, taking care of SARS-CoV-2 infected patients with ARDS and PF is considered to be the major difficulty. Therefore, further understanding of the host immune response to SARS-CoV-2 is extremely important for clinical resolution and saving medication cost. In addition to a breif overview of the structure, infection mechanism, and possible therapeutic approaches, we summarized and compared the hematopathologic effect and immune responses to SARS-CoV, MERS-CoV, and SARS-CoV-2. We also discussed the indirect immune response caused by SARS and direct infection, replication, and destroying of immune cells by MERS-CoV. The molecular mechanisms of SARS-CoV and MERS-CoV infection-induced lymphopenia or cytokine storm may provide some hint toward fight against SARS-CoV-2, the novel coronavirus. This may provide guidance over using immune therapy as a combined treatment to prevent patients developing severe respiratory syndrome and largely reduce complications.\n",
      "SARS-CoV-2 has been identified as the causative agent of a global outbreak of respiratory tract disease (COVID-19). In some patients the infection results in moderate to severe acute respiratory distress syndrome (ARDS), requiring invasive mechanical ventilation. High serum levels of IL-6, IL-10 and an immune hyperresponsiveness referred to as a ‘cytokine storm’ have been associated with poor clinical outcome. Despite the large numbers of COVID-19 cases and deaths, information on the phenotype and kinetics of SARS-CoV-2-specific T cells is limited. Here, we studied 10 COVID-19 patients who required admission to an intensive care unit and detected SARS-CoV-2-specific CD4(+) and CD8(+) T cells in 10 out of 10 and 8 out of 10 patients, respectively. We also detected low levels of SARS-CoV-2-reactive T cells in 2 out of 10 healthy controls not previously exposed to SARS-CoV-2, which is indicative of cross-reactivity due to past infection with ‘common cold’ coronaviruses. The strongest T-cell responses were directed to the spike (S) surface glycoprotein, and SARS-CoV-2-specific T cells predominantly produced effector and Th1 cytokines, although Th2 and Th17 cytokines were also detected. Furthermore, we studied T-cell kinetics and showed that SARS-CoV-2-specific T cells are present relatively early and increase over time. Collectively, these data shed light on the potential variations in T-cell responses as a function of disease severity, an issue that is key to understanding the potential role of immunopathology in the disease, and also inform vaccine design and evaluation.\n",
      "The world is currently embroiled in a pandemic of coronavirus disease 2019 (COVID-19), a respiratory illness caused by the novel betacoronavirus SARS-CoV-2. The severity of COVID-19 disease ranges from asymptomatic to fatal acute respiratory distress syndrome (ARDS). In few patients, the disease undergoes phenotypic differentiation between 7-14 days of acute illness, either resulting in full recovery or symptom escalation. However, the mechanism of such variation is not clear, but the facts suggest that patient's immune status, co-morbidities, and the systemic effects of the viral infection (potentially depending on the SARS-CoV-2 strain involved) play a key role. Subsequently, patients with the most severe symptoms tend to have poor outcomes, manifest severe hypoxia, and possess elevated levels of pro-inflammatory cytokines (including IL-1β, IL-6, IFN-γ, and TNF-α) along with elevated levels of the anti-inflammatory cytokine IL-10, marked lymphopenia, and elevated neutrophil-to-lymphocyte ratios. Based on the available evidence, we propose a mechanism wherein SARS-CoV-2 infection induces direct organ damage while also fueling an IL-6-mediated cytokine release syndrome (CRS) and hypoxia, resulting in escalating systemic inflammation, multi-organ damage, and end-organ failure. Elevated IL-6 and hypoxia together predisposes patients to pulmonary hypertension, and the presence of asymptomatic hypoxia in COVID-19 further compounds this problem. Due to the similar downstream mediators, we discuss the potential synergistic effects and systemic ramifications of SARS-CoV-2 and influenza virus during co-infection, a phenomenon we have termed \"COVI-Flu.\" Additionally, the differences between CRS and cytokine storm are highlighted. Finally, novel management approaches, clinical trials, and therapeutic strategies toward both SARS-CoV-2 and COVI-Flu infection are discussed, highlighting host response optimization and systemic inflammation reduction.\n",
      "Pathogenic human coronavirus infections, such as severe acute respiratory syndrome CoV (SARS-CoV) and Middle East respiratory syndrome CoV (MERS-CoV), cause high morbidity and mortality1, 2 Recently, a severe pneumonia-associated respiratory syndrome caused by a new coronavirus (SARS-CoV-2) was reported at December 2019 in the city Wuhan, Hubei province, China3, 4, 5, which was also named as pneumonia-associated respiratory syndrome (PARS)6 and can cause coronavirus disease 2019 (COVID-19) to seriously endanger human health Up to 24th of February 2020, at least 77779 cases have been reported with 2666 fatal cases according to the report from China CDC However, the immune mechanism that potential orchestrated acute mortality from COVID-19 patients is still unknown Here we show that after the SARS-CoV-2 infection, CD4+ T lymphocytes are rapidly activated to become pathogenic T helper (Th) 1 cells and generate GM-CSF etc The cytokines environment induces inflammatory CD14+CD16+ monocytes with high expression of IL-6 and accelerate the inflammation Given that large amount of inflammatory cells infiltrations have been observed in lungs from severe COVID-19 patients7, 8, these aberrant pathogenic Th1 cells and inflammatory monocytes may enter the pulmonary circulation in huge numbers and play an immune damaging role to causing lung functional disability and quick mortality Our results demonstrate that excessive non-effective host immune responses by pathogenic T cells and inflammatory monocytes may associate with severe lung pathology Thus, we suggest that monoclonal antibodies targeting GM-CSF or interleukin 6 may be effective in blocking inflammatory storms and, therefore, be a promising treatment of severe COVID-19 patients\n",
      "COVID-19 has become a major challenge to global health, and until now, no efficient antiviral agents have been developed. The SARS-CoV-2 infection is characterized by pulmonary and systemic inflammation in severe patients, and acute respiratory distress syndrome (ARDS) caused respiratory failure contributes to most mortalities. There is an urgent need for developing effective drugs and vaccines against SARS-CoV-2 and COVID-19 caused ARDS. However, most researchers cannot perform SARS-CoV-2 related researches due to lacking P3 or P4 facility. We developed a non-infectious, highly safety, time-saving SARS-CoV-2 components induced murine model to study the SARS-CoV-2 caused ARDS and cytokine storm syndrome (CSS). We also investigated mAbs and inhibitors which potentially neutralize the pro-inflammatory phenotype of COVID-19, and found that anti-IL-1α, anti-IL-6, anti-TNFα, anti-GM-CSF mAbs, p38 inhibitor, and JAK inhibitor partially relieved CSS. Besides, anti-IL-6, anti-TNFα, anti-GM-CSF mAbs and inhibitors of p38, ERK, and MPO somewhat reduced neutrophilic alveolitis in the lung. In all, we established the murine model mimic of COVID-19, opening a biosafety and less time-consuming avenue for clarifying the mechanism of ARDS and CSS in COVID-19 and developing the therapeutic drugs.\n",
      "PURPOSE: Novel Coronavirus disease 2019 (COVID-19), is an acute respiratory distress syndrome (ARDS), which is emerged in Wuhan, and recently become worldwide pandemic. Strangely, ample evidences have been shown that the severity of COVID-19 infections varies widely from children (asymptomatic), adults (mild infection), as well as elderly adults (deadly critical). It has proven that COVID-19 infection in some elderly critical adults leads to a cytokine storm, which is characterized by severe systemic elevation of several pro-inflammatory cytokines. Then, a cytokine storm can induce edematous, ARDS, pneumonia, as well as multiple organ failure in aged patients. It is far from clear till now why cytokine storm induces in only COVID-19 elderly patients, and not in young patients. However, it seems that aging is associated with mild elevated levels of local and systemic pro-inflammatory cytokines, which is characterized by “inflamm-aging”. It is highly likely that “inflamm-aging” is correlated to increased risk of a cytokine storm in some critical elderly patients with COVID-19 infection. METHODS: A systematic search in the literature was performed in PubMed, Scopus, Embase, Cochrane Library, Web of Science, as well as Google Scholar pre-print database using all available MeSH terms for COVID-19, Coronavirus, SARS-CoV-2, senescent cell, cytokine storm, inflame-aging, ACE2 receptor, autophagy, and Vitamin D. Electronic database searches combined and duplicates were removed. RESULTS: The aim of the present review was to summarize experimental data and clinical observations that linked the pathophysiology mechanisms of “inflamm-aging”, mild-grade inflammation, and cytokine storm in some elderly adults with severe COVID-19 infection.\n",
      "The seventh human coronavirus SARS-CoV2 belongs to the cluster of extremely pathogenic coronaviruses like SARS-CoV and MERS-CoV, which are established to cause lower respiratory tract infection. The viral infection can be fatal as the disease advances to pneumonia followed by acute respiratory distress syndrome (ARDS). Increasingly higher cytokine concentration on account of over-stimulated immune response against the virus, or the ‘cytokine storm’, is the reason behind the manifestation of lethal clinical symptoms. In this article, we discuss the immune pathogenesis of cytokine storm and its relation with SARS-CoV2/COVID-19 risk factors. People with underlying risk factors are more susceptible to fatal complications of COVID-19 infection leading to poor clinical outcome. The increased pro-inflammatory immune status in patients with risk factors (diabetes, hypertension, cardiovascular disease, COPD) exacerbates the Cytokine-storm of COVID-19 to a Cytokine Super Cyclone. We also overviewed antiviral immune response provided by BCG vaccine involving the IL-1β, IL-6 and TNF-α secretion via ‘trained monocytes’, which confers early protection against SARS-CoV2.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 3g06zxmx\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "ABSTRACT The outbreak of the novel SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) responsible for coronavirus disease 2019 (COVID-19) has developed into a major health hazard across the globe. The rapid escalation of the COVID-19 and the considerable associated illness leading to death has transformed it into a critical global public health issue. The current clinical investigation in patients with COVID-19 has showed strong upregulation of cytokine and interferon production after critical infection with SARS-CoV2- induced pneumonia. The current research suggests that few groups of patients with COVID-19 might have a cytokine storm syndrome. Thus, identification and treatment of hyperinflammation using existing, approved therapies with proven safety profiles to address the immediate need to reduce the rising mortality. To date, no specific therapeutic drug or vaccine has been discovered to treat this COVID-19. Intensive research is in progress to identify a promising drug therapy. Hence, several options were proposed to control this fast-emerging pandemic, including antiviral drugs, vaccines, small-molecules, monoclonal antibodies, oligonucleotides, peptides, and interferons (IFNs).\n",
      "The outbreak of the novel SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) responsible for coronavirus disease 2019 (COVID-19) has developed into an unprecedented global pandemic. Clinical investigations in patients with COVID-19 has shown a strong upregulation of cytokine and interferon production in SARS-CoV2- induced pneumonia, with an associated cytokine storm syndrome. Thus, the identification of existing approved therapies with proven safety profiles to treat hyperinflammation is a critical unmet need in order to reduce COVI-19 associated mortality. To date, no specific therapeutic drugs or vaccines are available to treat COVID-19 patients. This review evaluates several options that have been proposed to control SARS-CoV2 hyperinflammation and cytokine storm, eincluding antiviral drugs, vaccines, small-molecules, monoclonal antibodies, oligonucleotides, peptides, and interferons (IFNs).\n",
      "Coronavirus disease 2019 (COVID-19), caused by the virus designated as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has spread widely throughout the world. Despite the strict global outbreak management and quarantine measures that have been implemented, the incidence of COVID-19 continues to rise, resulting in more than 290,000 deaths and representing an extremely serious threat to human life and health. The clinical symptoms of the affected patients are heterogeneous, ranging from mild upper respiratory symptoms to severe pneumonitis and even acute respiratory distress syndrome (ARDS) or death. Systemic immune over activation due to SARS-CoV-2 infection causes the cytokine storm, which is especially noteworthy in severely ill patients with COVID-19. Pieces of evidence from current studies have shown that the cytokine storm may be an important factor in disease progression, even leading to multiple organ failure and death. This review provides an overview of the knowledge on the COVID-19 epidemiological profile, the molecular mechanisms of the SARS-CoV-2-induced cytokine storm and immune responses, the pathophysiological changes that occur during infection, the main antiviral compounds used in treatment strategies and the potential drugs for targeting cytokines, this information is presented to provide valuable guidance for further studies and for a therapeutic reduction of this excessive immune response.\n",
      "A novel coronavirus disease 2019 (COVID-19) triggered by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is currently spreading globally, causing severe pneumonia and acute lung injury in many patients. Even worse, severe respiratory may develop into acute respiratory distress syndrome and multiple organ dysfunction syndrome in COVID-19. The cytokine storm caused by immune over-activation due to virus infection may be an important cause of death in the late period of progress, but the pathogenesis of cytokine storm is still unclear. This article reviews the mechanisms of SARS-CoV-2-induced cytokine storm in detail based on the current discovered researches, and put forward some valuable medication ideas for the targeted cytokines drug researches and treatment. The goal of this work will be helpful for reducing excessive immune response.\n",
      "The recent outbreak of coronavirus disease 2019 (COVID‐19), triggered by the severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2) poses an enormous threat to global public health and economies. Human coronaviruses normally cause no or mild respiratory disease but in the past two decades, potentially fatal coronavirus infections have emerged, causing respiratory tract illnesses such as pneumonia and bronchitis. These include severe acute respiratory syndrome coronavirus (SARS‐CoV), followed by the Middle East respiratory syndrome coronavirus (MERS‐CoV), and recently the SARS‐CoV‐2 coronavirus outbreak which emerged in Wuhan, China, in December 2019. Currently, most COVID‐19 patients receive traditional supportive care including breathing assistance. To halt the ongoing spread of the pandemic SARS‐CoV‐2 coronavirus and rescue individual patients, established drugs and new therapies are under evaluation. Since it will be some time until a safe and effective vaccine will be available, the immediate priority is to harness innate immunity to accelerate early antiviral immune responses. Secondly since excessive inflammation is a major cause of pathology, targeted anti‐inflammatory responses are being evaluated to reduce inflammation‐induced damage to the respiratory tract and cytokine storms. Here, we highlight prominent immunotherapies at various stages of development which aim for augmented anti‐coronavirus immunity and reduction of pathological inflammation. This article is protected by copyright. All rights reserved\n",
      "The recent outbreak of coronavirus disease 2019 (COVID-19), triggered by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) poses an enormous threat to global public health and economies. Human coronaviruses normally cause no or mild respiratory disease but in the past two decades, potentially fatal coronavirus infections have emerged, causing respiratory tract illnesses such as pneumonia and bronchitis. These include severe acute respiratory syndrome coronavirus (SARS-CoV), followed by the Middle East respiratory syndrome coronavirus (MERS-CoV), and recently the SARS-CoV-2 coronavirus outbreak that emerged in Wuhan, China, in December 2019. Currently, most COVID-19 patients receive traditional supportive care including breathing assistance. To halt the ongoing spread of the pandemic SARS-CoV-2 coronavirus and rescue individual patients, established drugs and new therapies are under evaluation. Since it will be some time until a safe and effective vaccine will be available, the immediate priority is to harness innate immunity to accelerate early antiviral immune responses. Second, since excessive inflammation is a major cause of pathology, targeted anti-inflammatory responses are being evaluated to reduce inflammation-induced damage to the respiratory tract and cytokine storms. Here, we highlight prominent immunotherapies at various stages of development that aim for augmented anti-coronavirus immunity and reduction of pathological inflammation.\n",
      "Abstract Coronavirus disease 2019 (COVID-19) is caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). It has infected millions with more than 181,000 fatal cases as of 22nd April 2020. Currently, there are no specific COVID-19 therapies. Most patients depend on mechanical ventilation. Current COVID-19 data clearly highlight that cytokine storm and activated immune cell migration to the lungs characterize the early immune response to COVID-19 that causes severe lung damage and development of acute respiratory distress syndrome. In view of uncertainty associated with immunosuppressive treatments such as corticosteroids and their possible secondary effects, including risks of secondary infections, we suggest immunotherapies as an adjunct therapy in severe COVID-19 cases. Such immunotherapies based on inflammatory cytokine neutralization, immunomodulation and passive viral neutralization, not only reduce inflammation, inflammation-associated lung damage, or viral load, but could also prevent intensive care unit hospitalization and dependency on mechanical ventilation both of which are limited resources.\n",
      "The potentially lethal infection caused by the novel Severe Acute Respiratory Disease Coronavirus-2 (SARS-CoV-2) has evolved into a global crisis. Following the initial viral infection is the host inflammatory response that frequently results in excessive secretion of inflammatory cytokines (e.g., IL-6 and TNF&#945;), developing into a self-targeting, toxic \"cytokine storm\" causing critical pulmonary tissue damage. The need for a therapeutic that is available immediately is growing daily but the de novo development of a vaccine may take years. Therefore, repurposing of approved drugs offers a promising approach to address this urgent need. Inhaled furosemide, a small molecule capable of inhibiting IL-6 and TNF&#945;, may be an agent capable of treating the Coronavirus Disease 2019 cytokine storm in both resource-rich and developing countries. Furosemide is a \"repurpose-able\" small molecule therapeutics, that is safe, easily synthesized, handled, and stored, and is available in reasonable quantities worldwide.\n",
      "The potentially lethal infection caused by the novel Severe Acute Respiratory Disease Coronavirus-2 (SARS-CoV-2) has evolved into a global crisis. Following the initial viral infection is the host inflammatory response that frequently results in excessive secretion of inflammatory cytokines (e.g., IL-6 and TNFα), developing into a self-targeting, toxic \"cytokine storm\" causing critical pulmonary tissue damage. The need for a therapeutic that is available immediately is growing daily but the de novo development of a vaccine may take years. Therefore, repurposing of approved drugs offers a promising approach to address this urgent need. Inhaled furosemide, a small molecule capable of inhibiting IL-6 and TNFα, may be an agent capable of treating the Coronavirus Disease 2019 cytokine storm in both resource-rich and developing countries. Furosemide is a \"repurpose-able\" small molecule therapeutics, that is safe, easily synthesized, handled, and stored, and is available in reasonable quantities worldwide.\n",
      "Coronavirus disease 2019 (COVID-19) is a pandemic infection caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). COVID-19 significantly affects multiple systems including the cardiovascular system. Most importantly, in addition to the direct injury from the virus per se, the subsequent cytokine storm, an overproduction of immune cells and their activating compounds, causes devastating damage. To date, emerging anti-SARS-CoV-2 treatments are warranted to control epidemics. Several candidate drugs have been screened and are currently under investigation. These primarily include antiviral regimens and immunomodulatory regimens. However, beyond the anti-SARS-CoV-2 effects, these drugs may also have risks to the cardiovascular system, especially altering cardiac conduction. Herein, we review the cardiovascular risks of potential anti-COVID-19 drugs.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 5h7qyn1g\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Over the last two decades, the world experienced three outbreaks of coronaviruses with elevated morbidity rates. Currently, the global community is facing emerging virus SARS-CoV-2 belonging to Betacoronavirus, which appears to be more transmissible but less deadly than SARS-CoV. The current study aimed to track the evolutionary ancestors and different evolutionary strategies that were genetically adapted by SARS-CoV-2. Our whole-genome analysis revealed that SARS-CoV-2 was the descendant of Bat SARS/SARS-like CoVs and bats served as a natural reservoir. SARS-CoV-2 used mutations and recombination as crucial strategies in different genomic regions including the envelop, membrane, nucleocapsid, and spike glycoproteins to become a novel infectious agent. We confirmed that mutations in different genomic regions of SARS-CoV-2 have specific influence on virus reproductive adaptability, allowing for genotype adjustment and adaptations in rapidly changing environments. Moreover, for the first time we identified nine putative recombination patterns in SARS-CoV-2, which encompass spike glycoprotein, RdRp, helicase and ORF3a. Six recombination regions were spotted in the S gene and are undoubtedly important for evolutionary survival, meanwhile this permitted the virus to modify superficial antigenicity to find a way from immune reconnaissance in animals and adapt to a human host. With these combined natural selected strategies, SARS-CoV-2 emerged as a novel virus in human society.\n",
      "SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) is a novel coronavirus causing an outbreak of COVID-19 globally in the past six months. A relatively higher divergence on the spike protein of SASR-CoV-2 enables it to transmit across species efficiently. We particularly believe that the adaptive mutations of the receptor-binding domain (RBD) of spike protein in SARS-CoV-2 might be essential to its high transmissibility among humans. Thus here we collected 2,142 high-quality genome sequences of SARS-CoV-2 from 160 regions in over 50 countries and reconstructed their phylogeny, and also analyzed the interaction between the polymorphisms of spike protein and human ACE2 (hACE2). Phylogenetic analysis of SARS-CoV-2 and coronavirus in other hosts show SARS-CoV-2 is highly possible originated from Bat-CoV (RaTG13) found in horseshoe bat and a recombination event may occur on the spike protein of Pangolin-CoV to imbue it the ability to infect humans. Moreover, compared to the S gene of SARS-CoV-2, it is more conserved in the direct-binding sites of RBD and we noticed that spike protein of SARS-CoV-2 may under a consensus evolution to adapt to human hosts better. 3,860 amino acid mutations in spike protein RBD (T333-C525) of SARS-CoV-2 were simulated and their stability and affinity binding to hACE2 (S19-D615) were calculated. Our analysis indicates SARS-CoV-2 could infect humans from different populations with no preference, and a higher divergence in the spike protein of SARS-CoV-2 at the early stage of this pandemic may be a good indicator that could show the pathway of SARS-CoV-2 transmitting from the natural reservoir to human beings.\n",
      "The recently emerged SARS-CoV-2 (Coronaviridae; Betacoronavirus) is the underlying cause of COVID-19 disease. Here we assessed SARS-CoV2 from the Kingdom of Saudi Arabia alongside sequences of SARS-CoV, bat SARS-like CoVs and MERS-CoV, the latter currently detected in this region. Phylogenetic analysis, natural selection investigation and genome recombination analysis were performed. Our analysis showed that all Saudi SARS-CoV-2 sequences are of the same origin and closer proximity to bat SARS-like CoVs, followed by SARS-CoVs, however quite distant to MERS-CoV. Moreover, genome recombination analysis revealed two recombination events between SARS-CoV-2 and bat SARS-like CoVs. This was further assessed by S gene recombination analysis. These recombination events may be relevant to the emergence of this novel virus. Moreover, positive selection pressure was detected between SARS-CoV-2, bat SL-CoV isolates and human SARS-CoV isolates. However, the highest positive selection occurred between SARS-CoV-2 isolates and 2 bat-SL-CoV isolates (Bat-SL-RsSHC014 and Bat-SL-CoVZC45). This further indicates that SARS-CoV-2 isolates were adaptively evolved from bat SARS-like isolates, and that a virus with originating from bats triggered this pandemic. This study thuds sheds further light on the origin of this virus. AUTHOR SUMMARY The emergence and subsequent pandemic of SARS-CoV-2 is a unique challenge to countries all over the world, including Saudi Arabia where cases of the related MERS are still being reported. Saudi SARS-CoV-2 sequences were found to be likely of the same or similar origin. In our analysis, SARS-CoV-2 were more closely related to bat SARS-like CoVs rather than to MERS-CoV (which originated in Saudi Arabia) or SARS-CoV, confirming other phylogenetic efforts on this pathogen. Recombination and positive selection analysis further suggest that bat coronaviruses may be at the origin of SARS-CoV-2 sequences. The data shown here give hints on the origin of this virus and may inform efforts on transmissibility, host adaptation and other biological aspects of this virus.\n",
      "SARS-CoV-2, the causative agent of COVID-19, is widespread in several countries around the world following its late-2019 emergence in the human population. Rapid development of molecular diagnostic tests and subunit vaccines have been prioritized, and as such evaluating the SARS-CoV-2 genomic plasticity and evolutionary dynamics is an urgent need. We determined the SARS-CoV-2 selectome by calculating rates of pervasive and episodic diversifying selection for every amino acid coding position in the SARS-CoV-2 genome. To provide context for evolutionary dynamics of a highly pathogenic betacoronavirus following a zoonotic spillover into human hosts, we also determined the selectomes of SARS-CoV and MERS-CoV, and performed evolvability calculations for SARS-CoV-2 based on SARS-CoV. These analyses identify the amino acid sites within each coding sequence that have been subjected to pervasive diversifying selection or episodic diversifying selection, and report significantly evolvable sites in the ORF1a polyprotein, the spike protein, and the membrane protein of SARS-CoV-2. These findings provide a comprehensive view of zoonotic, highly pathogenic betacoronavirus evolutionary dynamics that can be directly applied to diagnostic assay and vaccine design for SARS-CoV-2.\n",
      "In depth evolutionary and structural analyses of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) isolated from bats, pangolins, and humans are necessary to assess the role of natural selection and recombination in the emergence of the current pandemic strain. The SARS-CoV-2 S glycoprotein unique features have been associated with efficient viral spread in the human population. Phylogeny-based and genetic algorithm methods clearly show that recombination events between viral progenitors infecting animal hosts led to a mosaic structure in the S gene. We identified recombination coldspots in the S glycoprotein and strong purifying selection. Moreover, although there is little evidence of diversifying positive selection during host-switching, structural analysis suggests that some of the residues emerged along the ancestral lineage of current pandemic strains may contribute to enhanced ability to infect human cells. Interestingly, recombination did not affect the long-range covariant movements of SARS-CoV-2 S glycoprotein monomer in pre-fusion conformation but, on the contrary, could contribute to the observed overall viral efficiency. Our dynamic simulations revealed that the movements between the host cell receptor binding domain (RBD) and the novel furin-like cleavage site are correlated. We identified threonine 333 (under purifying selection), at the beginning of the RBD, as the hinge of the opening/closing mechanism of the SARS-CoV-2 S glycoprotein monomer functional to hACE2 binding. Our findings support a scenario where ancestral recombination and fixation of amino acid residues in the RBD of the S glycoprotein generated a virus with unique features, capable of extremely efficient infection of the human host.\n",
      "SARS-CoV-2 poses an immediate, major threat to public health across the globe. Here we report an in-depth molecular analysis to reconstruct the evolutionary origins of the enhanced pathogenicity of SARS-CoV-2 and other coronaviruses that are severe human pathogens. Using integrated comparative genomics and machine learning techniques, we identify key genomic features that differentiate SARS-CoV-2 and the viruses behind the two previous deadly coronavirus outbreaks, SARS-CoV and MERS-CoV, from less pathogenic coronaviruses. These features include enhancement of the nuclear localization signals in the nucleocapsid protein and distinct inserts in the spike glycoprotein that appear to be associated with high case fatality rate of these coronaviruses as well as the host switch from animals to humans. The identified features could be crucial elements of coronavirus pathogenicity and possible targets for diagnostics, prognostication and interventions.\n",
      "The current coronavirus disease 2019 (COVID-19) pandemic is caused by the SARS-CoV-2 virus and is still spreading rapidly worldwide. Full-genome-sequence computational analysis of the SARS-CoV-2 genome will allow us to understand the recent evolutionary events and adaptability mechanisms more accurately, as there is still neither effective therapeutic nor prophylactic strategy. In this study, we used population genetics analysis to infer the mutation rate and plausible recombination events that may have contributed to the evolution of the SARS-CoV-2 virus. Furthermore, we localized targets of recent and strong positive selection. The genomic regions that appear to be under positive selection are largely co-localized with regions in which recombination from non-human hosts appeared to have taken place in the past. Our results suggest that the pangolin coronavirus genome may have contributed to the SARS-CoV-2 genome by recombination with the bat coronavirus genome. However, we find evidence for additional recombination events that involve coronavirus genomes from other hosts, i.e., Hedgehog and Sparrow. Even though recombination events within human hosts cannot be directly assessed, due to the high similarity of SARS-CoV-2 genomes, we infer that recombinations may have recently occurred within human hosts using a linkage disequilibrium analysis. In addition, we employed an Approximate Bayesian Computation approach to estimate the parameters of a demographic scenario involving an exponential growth of the size of the SARS-CoV-2 populations that have infected European, Asian and Northern American cohorts, and we demonstrated that a rapid exponential growth in population size can support the observed polymorphism patterns in SARS-CoV-2 genomes.\n",
      "Monitoring the mutation dynamics of SARS-CoV-2 is critical for the development of effective approaches to contain the pathogen. By analyzing 106 SARS-CoV-2 and 39 SARS genome sequences, we provided direct genetic evidence that SARS-CoV-2 has a much lower mutation rate than SARS. Minimum Evolution phylogeny analysis revealed the putative original status of SARS-CoV-2 and the early-stage spread history. The discrepant phylogenies for the spike protein and its receptor binding domain proved a previously reported structural rearrangement prior to the emergence of SARS-CoV-2. Despite that we found the spike glycoprotein of SARS-CoV-2 is particularly more conserved, we identified a mutation that leads to weaker receptor binding capability, which concerns a SARS-CoV-2 sample collected on 27th January 2020 from India. This represents the first report of a significant SARS-CoV-2 mutant, and raises the alarm that the ongoing vaccine development may become futile in future epidemic if more mutations were identified. Highlights Based on the currently available genome sequence data, we proved that SARS-COV-2 genome has a much lower mutation rate and genetic diversity than SARS during the 2002-2003 outbreak. The spike (S) protein encoding gene of SARS-COV-2 is found relatively more conserved than other protein-encoding genes, which is a good indication for the ongoing antiviral drug and vaccine development. Minimum Evolution phylogeny analysis revealed the putative original status of SARS-CoV-2 and the early-stage spread history. We confirmed a previously reported rearrangement in the S protein arrangement of SARS-COV-2, and propose that this rearrangement should have occurred between human SARS-CoV and a bat SARS-CoV, at a time point much earlier before SARS-COV-2 transmission to human. We provided first evidence that a mutated SARS-COV-2 with reduced human ACE2 receptor binding affinity have emerged in India based on a sample collected on 27th January 2020.\n",
      "Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) poses an immediate, major threat to public health across the globe. Here we report an in-depth molecular analysis to reconstruct the evolutionary origins of the enhanced pathogenicity of SARS-CoV-2 and other coronaviruses that are severe human pathogens. Using integrated comparative genomics and machine learning techniques, we identify key genomic features that differentiate SARS-CoV-2 and the viruses behind the two previous deadly coronavirus outbreaks, SARS-CoV and Middle East respiratory syndrome coronavirus (MERS-CoV), from less pathogenic coronaviruses. These features include enhancement of the nuclear localization signals in the nucleocapsid protein and distinct inserts in the spike glycoprotein that appear to be associated with high case fatality rate of these coronaviruses as well as the host switch from animals to humans. The identified features could be crucial contributors to coronavirus pathogenicity and possible targets for diagnostics, prognostication, and interventions.\n",
      "There are outstanding evolutionary questions on the recent emergence of coronavirus SARS-CoV-2/hCoV-19 in Hubei province that caused the COVID-19 pandemic, including (1) the relationship of the new virus to the SARS-related coronaviruses, (2) the role of bats as a reservoir species, (3) the potential role of other mammals in the emergence event, and (4) the role of recombination in viral emergence. Here, we address these questions and find that the sarbecoviruses – the viral subgenus responsible for the emergence of SARS-CoV and SARS-CoV-2 – exhibit frequent recombination, but the SARS-CoV-2 lineage itself is not a recombinant of any viruses detected to date. In order to employ phylogenetic methods to date the divergence events between SARS-CoV-2 and the bat sarbecovirus reservoir, recombinant regions of a 68-genome sarbecovirus alignment were removed with three independent methods. Bayesian evolutionary rate and divergence date estimates were consistent for all three recombination-free alignments and robust to two different prior specifications based on HCoV-OC43 and MERS-CoV evolutionary rates. Divergence dates between SARS-CoV-2 and the bat sarbecovirus reservoir were estimated as 1948 (95% HPD: 1879-1999), 1969 (95% HPD: 1930-2000), and 1982 (95% HPD: 1948-2009). Despite intensified characterization of sarbecoviruses since SARS, the lineage giving rise to SARS-CoV-2 has been circulating unnoticed for decades in bats and been transmitted to other hosts such as pangolins. The occurrence of a third significant coronavirus emergence in 17 years together with the high prevalence and virus diversity in bats implies that these viruses are likely to cross species boundaries again. In Brief The Betacoronavirus SARS-CoV-2 is a member of the sarbecovirus subgenus which shows frequent recombination in its evolutionary history. We characterize the extent of this genetic exchange and identify non-recombining regions of the sarbecovirus genome using three independent methods to remove the effects of recombination. Using these non-recombining genome regions and prior information on coronavirus evolutionary rates, we obtain estimates from three approaches that the most likely divergence date of SARS-CoV-2 from its most closely related available bat sequences ranges from 1948 to 1982. Key Points RaTG13 is the closest available bat virus to SARS-CoV-2; a sub-lineage of these bat viruses is able to infect humans. Two sister lineages of the RaTG13/SARS-CoV-2 lineage infect Malayan pangolins. The sarbecoviruses show a pattern of deep recombination events, indicating that there are high levels of co-infection in horseshoe bats and that the viral pool can generate novel allele combinations and substantial genetic diversity; the sarbecoviruses are efficient ‘explorers’ of phenotype space. The SARS-CoV-2 lineage is not a recent recombinant, at least not involving any of the bat or pangolin viruses sampled to date. Non-recombinant regions of the sarbecoviruses can be identified, allowing for phylogenetic inference and dating to be performed. We constructed three such regions using different methods. We estimate that RaTG13 and SARS-CoV-2 diverged 40 to 70 years ago. There is a diverse unsampled reservoir of generalist viruses established in horseshoe bats. While an intermediate host responsible for the zoonotic event cannot be ruled out, the relevant evolution for spillover to humans very likely occurred in horseshoe bats.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: ltar1aa5\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "what is the origin of COVID-19\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "The coronavirus disease (COVID-19) belongs to the family Severe Acute Respiratory Syndrome (SARS-CoV). It can be more severe for some persons and can lead to pneumonia or breathing difficulties resulting in the death of immune-compromised patients. We performed a phylogenomic and phylogeographic tree from the collected datasets. Phylogenomic analysis or sequence-based phylogeny showed an evolutionary relationship between the geographical strains. The phylogenomic tree grouped into two major clades consists of various isolates of SARS-CoV-2 and Bat SARS-like coronavirus, Bat coronavirus, and Pangolin coronavirus. The phylogenetic neighbor of newly sequenced Indian strains (Accession: MT012098.1, MT050493.1) was revealed to identify the variations between the nCoV-19 strains. The results showed keen evidence that SARS-CoV-2 has evolved from Bat SARS-like coronavirus. The evolutionary history and comparative proteomic analysis provide a new avenue for the current scientific research related to the coronavirus.\n",
      "The recent global outbreak of viral pneumonia designated as Coronavirus Disease 2019 (COVID-19) by coronavirus (SARS-CoV-2) has threatened global public health and urged to investigate its source. Whole genome analysis of SARS-CoV-2 revealed ~96% genomic similarity with bat CoV (RaTG13) and clustered together in phylogenetic tree. Furthermore, RaTGl3 also showed 97.43% spike protein similarity with SARS-CoV-2 suggesting that RaTGl3 is the closest strain. However, RBD and key amino acid residues supposed to be crucial for human-to-human and cross-species transmission are homologues between SARS-CoV-2 and pangolin CoVs. These results from our analysis suggest that SARS-CoV-2 is a recombinant virus of bat and pangolin CoVs. Moreover, this study also reports mutations in coding regions of 125 SARS-CoV-2 genomes signifying its aptitude for evolution. In short, our findings propose that homologous recombination has been occurred between bat and pangolin CoVs that triggered cross-species transmission and emergence of SARS-CoV-2, and, during the ongoing outbreak, SARS-CoV-2 is still evolving for its adaptability.\n",
      "The Coronavirus Disease 2019 (COVID-19) is a new viral infection caused by the severe acute respiratory coronavirus 2 (SARS-CoV-2). Genomic analyses have revealed that SARS-CoV-2 is related to Pangolin and Bat coronaviruses. In this report, a structural comparison between the Sars-CoV-2 Envelope and Membrane proteins from different human isolates with homologous proteins from closely related viruses is described. The analyses here reported show the high structural similarity of Envelope and Membrane proteins to the counterparts from Pangolin and Bat coronavirus isolates. However, the comparisons have also highlighted structural differences specific of Sars-CoV-2 proteins which may be correlated to the cross-species transmission and/or to the properties of the virus. Structural modelling has been applied to map the variant sites onto the predicted three-dimensional structure of the Envelope and Membrane proteins.\n",
      "The severe respiratory disease COVID-19 was initially reported in Wuhan, China, in December 2019, and spread into many provinces from Wuhan. The corresponding pathogen was soon identified as a novel coronavirus named SARS-CoV-2 (formerly, 2019-nCoV). As of 2 May, 2020, over 3 million COVID-19 cases had been confirmed, and 235,290 deaths had been reported globally, and the numbers are still increasing. It is important to understand the phylogenetic relationship between SARS-CoV-2 and known coronaviruses, and to identify its hosts for preventing the next round of emergency outbreak. In this study, we employ an effective alignment-free approach, the Natural Vector method, to analyze the phylogeny and classify the coronaviruses based on genomic and protein data. Our results show that SARS-CoV-2 is closely related to, but distinct from the SARS-CoV branch. By analyzing the genetic distances from the SARS-CoV-2 strain to the coronaviruses residing in animal hosts, we establish that the most possible transmission path originates from bats to pangolins to humans.\n",
      "The outbreak of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in Wuhan city, China in December 2019 and thereafter its spillover across the world has created a global pandemic and public health crisis. Today, it appears as a threat to human civilization. Scientists and medical practitioners across the world are involved to trace out the origin and evolution of SARS-CoV-2 (also called 2019 novel coronavirus and referred as 2019-nCoV), its transmission route, cause of pathogenicity, and possible remedial action. In this work, we aim to find out the origin, evolutionary patternthat led to its pathogenicity and possible transmission pathway of 2019-nCoV. To achieve the aims we conducted a large-scale deep phylogenetic analysis on the 162 complete Orthocoronavirinae genomes consisting of four genera namely Alphacoronavirus, Betacoronavirus, Deltacoronavirus and Gammacoronavirus, their gene trees analysis and subsequently genome and gene recombination analyses. Our analyses revealed that i) bat, pangolin and anteater are the natural hosts of 2019-nCoV, ii) outbreak of 2019-nCoV took place via inter-intra species transmission mode, iii) host-specific adaptive mutation made 2019-nCoV more virulent, and the presence of widespread recombination events led to the evolution of new 2019-nCoV strain and/or could be determinant of its pathogenicity. Highlights Orthocoronavirinae genome phylogeny revealed that bat, pangolin and anteater are natural reservoir hosts of novel coronavirus (2019-nCoV/SARS-CoV-2). Host-specific adaptive mutation occurred among the coronaviruses. Transmission of 2019-nCoV to human took place by inter-intra species mode of transmission. Presence of widespread recombination events led to the evolution of new 2019-nCoV strain and/or could be determinant of its pathogenicity.\n",
      "To investigate the evolutionary history of the recent outbreak of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in China, a total of 70 genomes of virus strains from China and elsewhere with sampling dates between 24 December 2019 and 3 February 2020 were analyzed. To explore the potential intermediate animal host of the SARS-CoV-2 virus, we reanalyzed virome data sets from pangolins and representative SARS-related coronaviruses isolates from bats, with particular attention paid to the spike glycoprotein gene. We performed phylogenetic, split network, transmission network, likelihood-mapping, and comparative analyses of the genomes. Based on Bayesian time-scaled phylogenetic analysis using the tip-dating method, we estimated the time to the most recent common ancestor and evolutionary rate of SARS-CoV-2, which ranged from 22 to 24 November 2019 and 1.19 to 1.31 × 10-3 substitutions per site per year, respectively. Our results also revealed that the BetaCoV/bat/Yunnan/RaTG13/2013 virus was more similar to the SARS-CoV-2 virus than the coronavirus obtained from the two pangolin samples (SRR10168377 and SRR10168378). We also identified a unique peptide (PRRA) insertion in the human SARS-CoV-2 virus, which may be involved in the proteolytic cleavage of the spike protein by cellular proteases, and thus could impact host range and transmissibility. Interestingly, the coronavirus carried by pangolins did not have the RRAR motif. Therefore, we concluded that the human SARS-CoV-2 virus, which is responsible for the recent outbreak of COVID-19, did not come directly from pangolins.\n",
      "Coronavirus disease 2019 (COVID-19) is a pandemic infectious disease caused by novel severe acute respiratory syndrome coronavirus-2 (SARS CoV-2). The SARS CoV-2 is transmitted more rapidly and readily than SARS CoV. Both, SARS CoV and SARS CoV-2 via their glycosylated spike proteins recognize the human angiotensin converting enzyme-2 (ACE-2) receptor. We generated multiple sequence alignments and phylogenetic trees for representative spike proteins of SARS CoV and SARS CoV-2 from various host sources in order to analyze the specificity in SARS CoV-2 spike proteins required for causing infection in humans. Our results show that among the genomes analyzed, two sequence regions in the N-terminal domain \"MESEFR\" and \"SYLTPG\" are specific to human SARS CoV-2. In the receptor-binding domain, two sequence regions \"VGGNY\" and \"EIYQAGSTPCNGV\" and a disulfide bridge connecting 480C and 488C in the extended loop are structural determinants for the recognition of human ACE-2 receptor. The complete genome analysis of representative SARS CoVs from bat, civet, human host sources, and human SARS CoV-2 identified the bat genome (GenBank code: MN996532.1) as closest to the recent novel human SARS CoV-2 genomes. The bat SARS CoV genomes (GenBank codes: MG772933 and MG772934) are evolutionary intermediates in the mutagenesis progression toward becoming human SARS CoV-2.\n",
      "Although primary genomic analysis has revealed that severe acute respiratory syndrome coronavirus (SARS CoV) is a new type of coronavirus, the different protein trees published in previous reports have provided no conclusive evidence indicating the phylogenetic position of SARS CoV. To clarify the phylogenetic relationship between SARS CoV and other coronaviruses, we compiled a large data set composed of 7 concatenated protein sequences and performed comprehensive analyses, using the maximum-likelihood, Bayesian-inference, and maximum-parsimony methods. All resulting phylogenetic trees displayed an identical topology and supported the hypothesis that the relationship between SARS CoV and group 2 CoVs is monophyletic. Relationships among all major groups were well resolved and were supported by all statistical analyses.\n",
      "There is a worldwide concern about the new coronavirus 2019-nCoV as a global public health threat. In this article, we provide a preliminary evolutionary and molecular epidemiological analysis of this new virus. A phylogenetic tree has been built using the 15 available whole genome sequences of 2019-nCoV, 12 whole genome sequences of 2019-nCoV, and 12 highly similar whole genome sequences available in gene bank (five from the severe acute respiratory syndrome, two from Middle East respiratory syndrome, and five from bat SARS-like coronavirus). Fast unconstrained Bayesian approximation analysis shows that the nucleocapsid and the spike glycoprotein have some sites under positive pressure, whereas homology modeling revealed some molecular and structural differences between the viruses. The phylogenetic tree showed that 2019-nCoV significantly clustered with bat SARS-like coronavirus sequence isolated in 2015, whereas structural analysis revealed mutation in Spike Glycoprotein and nucleocapsid protein. From these results, the new 2019-nCoV is distinct from SARS virus, probably trasmitted from bats after mutation conferring ability to infect humans.\n",
      "The outbreak of 2019-nCoV pneumonia (COVID-19) in the city of Wuhan, China has resulted in more than 70,000 laboratory confirmed cases, and recent studies showed that 2019-nCoV (SARS-CoV-2) could be of bat origin but involve other potential intermediate hosts. In this study, we assembled the genomes of coronaviruses identified in sick pangolins. The molecular and phylogenetic analyses showed that pangolin Coronaviruses (pangolin-CoV) are genetically related to both the 2019-nCoV and bat Coronaviruses but do not support the 2019-nCoV arose directly from the pangolin-CoV. Our study also suggested that pangolin be natural host of Betacoronavirus, with a potential to infect humans. Large surveillance of coronaviruses in pangolins could improve our understanding of the spectrum of coronaviruses in pangolins. Conservation of wildlife and limits of the exposures of humans to wildlife will be important to minimize the spillover risks of coronaviruses from wild animals to humans.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: fofy6whl\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "what is the origin of COVID-19\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "OBJECTIVE: To analyze the evolution and variation of SARS-CoV-2 during the epidemic starting at the end of 2019. METHODS: We downloaded the full-length genome sequence of SARS-CoV-2 from the databases of GISAID and NCBI. Using the software for bioinformatics including MEGA-X, BEAST, and TempEst, we constructed the genomic evolution tree, inferred the time evolution signal of the virus, calculated the tMRCA time of the virus and analyzed the selection pressure of the virus during evolution. RESULTS: The phylogenetic tree showed that SARS-CoV-2 belonged to the Sarbecovirus subgenus of ß Coronavirus genus together with bat coronavirus BetaCoV/bat/Yunnan/RaTG13/2013, bat-SL-CoVZC45, bat-SL-CoVZXC21 and SARS-CoV. The genomic sequences of SARS-CoV-2 isolated from the ongoing epidemic showed a weak time evolution signal with an average tMRCA time of 73 days (95% CI: 38.9-119.3 days). No positive time evolution signal was found between SARS-CoV-2 and BetaCoV/bat/Yunnan/RaTG13/2013, but the former virus had a strong positive temporal evolution relationship with bat-SL-CoVZC45 and SARS-CoV. The major cause for mutations of SARS-CoV-2 was the pressure of purification selection during the epidemic. CONCLUSIONS: SARS-CoV-2 may have emerged as early as November, 2019, originating most likely from bat-associated coronavirus. This finding may provide evidence for tracing the sources and evolution of the virus.\n",
      "OBJECTIVE To analyze the evolution and variation of SARS-CoV-2 during the epidemic starting at the end of 2019. METHODS We downloaded the full-length genome sequence of SARS-CoV-2 from the databases of GISAID and NCBI. Using the software for bioinformatics including MEGA-X, BEAST, and TempEst, we constructed the genomic evolution tree, inferred the time evolution signal of the virus, calculated the tMRCA time of the virus and analyzed the selection pressure of the virus during evolution. RESULTS The phylogenetic tree showed that SARS-CoV-2 belonged to the Sarbecovirus subgenus of β Coronavirus genus together with bat coronavirus BetaCoV/bat/Yunnan/RaTG13/2013, bat-SL-CoVZC45, bat-SL-CoVZXC21 and SARS-CoV. The genomic sequences of SARS-CoV-2 isolated from the ongoing epidemic showed a weak time evolution signal with an average tMRCA time of 73 days (95% CI: 38.9-119.3 days). No positive time evolution signal was found between SARS-CoV-2 and BetaCoV/bat/Yunnan/RaTG13/2013, but the former virus had a strong positive temporal evolution relationship with bat-SL-CoVZC45 and SARS-CoV. The major cause for mutations of SARS-CoV-2 was the pressure of purification selection during the epidemic. CONCLUSIONS SARS-CoV-2 may have emerged as early as November, 2019, originating most likely from bat-associated coronavirus. This finding may provide evidence for tracing the sources and evolution of the virus.\n",
      "To investigate the evolutionary history of the recent outbreak of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in China, a total of 70 genomes of virus strains from China and elsewhere with sampling dates between 24 December 2019 and 3 February 2020 were analyzed. To explore the potential intermediate animal host of the SARS-CoV-2 virus, we reanalyzed virome data sets from pangolins and representative SARS-related coronaviruses isolates from bats, with particular attention paid to the spike glycoprotein gene. We performed phylogenetic, split network, transmission network, likelihood-mapping, and comparative analyses of the genomes. Based on Bayesian time-scaled phylogenetic analysis using the tip-dating method, we estimated the time to the most recent common ancestor and evolutionary rate of SARS-CoV-2, which ranged from 22 to 24 November 2019 and 1.19 to 1.31 × 10-3 substitutions per site per year, respectively. Our results also revealed that the BetaCoV/bat/Yunnan/RaTG13/2013 virus was more similar to the SARS-CoV-2 virus than the coronavirus obtained from the two pangolin samples (SRR10168377 and SRR10168378). We also identified a unique peptide (PRRA) insertion in the human SARS-CoV-2 virus, which may be involved in the proteolytic cleavage of the spike protein by cellular proteases, and thus could impact host range and transmissibility. Interestingly, the coronavirus carried by pangolins did not have the RRAR motif. Therefore, we concluded that the human SARS-CoV-2 virus, which is responsible for the recent outbreak of COVID-19, did not come directly from pangolins.\n",
      "OBJECTIVES: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan, China, in December 2019 and has been rapidly spreading worldwide. Although the causal relationship among mutations and the features of SARS-CoV-2 such as rapid transmission, pathogenicity, and tropism, remains unclear, our results of genomic mutations in SARS-CoV-2 may help to interpret the interaction between genomic characterization in SARS-CoV-2 and infectivity with the host. METHODS: A total of 4,254 genomic sequences of SARS-CoV-2 were collected from the Global Initiative on Sharing all Influenza Data (GISAID). Multiple sequence alignment for phylogenetic analysis and comparative genomic approach for mutation analysis were conducted using Molecular Evolutionary Genetics Analysis (MEGA), and an in-house program based on Perl language, respectively. RESULTS: Phylogenetic analysis of SARS-CoV-2 strains indicated that there were 3 major clades including S, V, and G, and 2 subclades (G.1 and G.2). There were 767 types of synonymous and 1,352 types of non-synonymous mutation. ORF1a, ORF1b, S, and N genes were detected at high frequency, whereas ORF7b and E genes exhibited low frequency. In the receptor-binding domain (RBD) of the S gene, 11 non-synonymous mutations were observed in the region adjacent to the angiotensin converting enzyme 2 (ACE2) binding site. CONCLUSION: It has been reported that the rapid infectivity and transmission of SARS-CoV-2 associated with host receptor affinity are derived from several mutations in its genes. Without these genetic mutations to enhance evolutionary adaptation, species recognition, host receptor affinity, and pathogenicity, it would not survive. It is expected that our results could provide an important clue in understanding the genomic characteristics of SARS-CoV-2.\n",
      "To investigate the evolutionary history of the recent outbreak of severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2) in China, a total of 70 genomes of virus strains from China and elsewhere with sampling dates between 24 December 2019 and 3 February 2020 were analyzed. To explore the potential intermediate animal host of the SARS‐CoV‐2 virus, we reanalyzed virome data sets from pangolins and representative SARS‐related coronaviruses isolates from bats, with particular attention paid to the spike glycoprotein gene. We performed phylogenetic, split network, transmission network, likelihood‐mapping, and comparative analyses of the genomes. Based on Bayesian time‐scaled phylogenetic analysis using the tip‐dating method, we estimated the time to the most recent common ancestor and evolutionary rate of SARS‐CoV‐2, which ranged from 22 to 24 November 2019 and 1.19 to 1.31 × 10(−3) substitutions per site per year, respectively. Our results also revealed that the BetaCoV/bat/Yunnan/RaTG13/2013 virus was more similar to the SARS‐CoV‐2 virus than the coronavirus obtained from the two pangolin samples (SRR10168377 and SRR10168378). We also identified a unique peptide (PRRA) insertion in the human SARS‐CoV‐2 virus, which may be involved in the proteolytic cleavage of the spike protein by cellular proteases, and thus could impact host range and transmissibility. Interestingly, the coronavirus carried by pangolins did not have the RRAR motif. Therefore, we concluded that the human SARS‐CoV‐2 virus, which is responsible for the recent outbreak of COVID‐19, did not come directly from pangolins.\n",
      "There are outstanding evolutionary questions on the recent emergence of coronavirus SARS-CoV-2/hCoV-19 in Hubei province that caused the COVID-19 pandemic, including (1) the relationship of the new virus to the SARS-related coronaviruses, (2) the role of bats as a reservoir species, (3) the potential role of other mammals in the emergence event, and (4) the role of recombination in viral emergence. Here, we address these questions and find that the sarbecoviruses – the viral subgenus responsible for the emergence of SARS-CoV and SARS-CoV-2 – exhibit frequent recombination, but the SARS-CoV-2 lineage itself is not a recombinant of any viruses detected to date. In order to employ phylogenetic methods to date the divergence events between SARS-CoV-2 and the bat sarbecovirus reservoir, recombinant regions of a 68-genome sarbecovirus alignment were removed with three independent methods. Bayesian evolutionary rate and divergence date estimates were consistent for all three recombination-free alignments and robust to two different prior specifications based on HCoV-OC43 and MERS-CoV evolutionary rates. Divergence dates between SARS-CoV-2 and the bat sarbecovirus reservoir were estimated as 1948 (95% HPD: 1879-1999), 1969 (95% HPD: 1930-2000), and 1982 (95% HPD: 1948-2009). Despite intensified characterization of sarbecoviruses since SARS, the lineage giving rise to SARS-CoV-2 has been circulating unnoticed for decades in bats and been transmitted to other hosts such as pangolins. The occurrence of a third significant coronavirus emergence in 17 years together with the high prevalence and virus diversity in bats implies that these viruses are likely to cross species boundaries again. In Brief The Betacoronavirus SARS-CoV-2 is a member of the sarbecovirus subgenus which shows frequent recombination in its evolutionary history. We characterize the extent of this genetic exchange and identify non-recombining regions of the sarbecovirus genome using three independent methods to remove the effects of recombination. Using these non-recombining genome regions and prior information on coronavirus evolutionary rates, we obtain estimates from three approaches that the most likely divergence date of SARS-CoV-2 from its most closely related available bat sequences ranges from 1948 to 1982. Key Points RaTG13 is the closest available bat virus to SARS-CoV-2; a sub-lineage of these bat viruses is able to infect humans. Two sister lineages of the RaTG13/SARS-CoV-2 lineage infect Malayan pangolins. The sarbecoviruses show a pattern of deep recombination events, indicating that there are high levels of co-infection in horseshoe bats and that the viral pool can generate novel allele combinations and substantial genetic diversity; the sarbecoviruses are efficient ‘explorers’ of phenotype space. The SARS-CoV-2 lineage is not a recent recombinant, at least not involving any of the bat or pangolin viruses sampled to date. Non-recombinant regions of the sarbecoviruses can be identified, allowing for phylogenetic inference and dating to be performed. We constructed three such regions using different methods. We estimate that RaTG13 and SARS-CoV-2 diverged 40 to 70 years ago. There is a diverse unsampled reservoir of generalist viruses established in horseshoe bats. While an intermediate host responsible for the zoonotic event cannot be ruled out, the relevant evolution for spillover to humans very likely occurred in horseshoe bats.\n",
      "The novel coronavirus SARS-CoV-2 (2019-nCoV) is a member of the family coronaviridae and contains a single-stranded RNA genome with positive-polarity. To reveal the evolution mechanism of SARS-CoV-2 genome, we performed comprehensive genomic analysis with newly sequenced SARS-CoV-2 strains and 20 closely related coronavirus strains. Among 98 nucleotide mutations at 93 sites of the genome among different SARS-CoV-2 strains, 58 of them caused amino acid change, indicating a result of neutral evolution. However, the ratio of nucleotide substitutions to amino acid substitutions of spike gene (9.07) between SARS-CoV-2 WIV04 and Bat-SARSr-CoV RaTG13 was extensively higher than those from comparisons between other coronaviruses (range 1.29 - 4.81). The elevated synonymous mutations between SARS-CoV-2 and RaTG13, suggesting they underwent stronger purifying selection. Moreover, their nucleotide substitutions are enriched with T:C transition, which is consistent with the mutation signature caused by deactivity of RNA 3’-to-5’ exoribonuclease (ExoN). The codon usage was similar between SARS-CoV-2 and other strains in beta-coronavirus lineage B, suggesting it had small impact on the mutation pattern. In comparison of SARS-CoV-2 WIV04 with Bat-SARSr-CoV RaTG13, the ratios of non-synonymous to synonymous substitution rates (dN/dS) was the lowest among all performed comparisons, reconfirming the evolution of SARS-CoV-2 under stringent selective pressure. Moreover, some sites of spike protein might be subjected to positive selection. Therefore, our results will help understanding the evolutionary mechanisms contribute to viral pathogenicity and its adaptation with hosts.\n",
      "The spread of SARS-CoV-2 since December 2019 has become a pandemic and impacted many aspects of human society. Here, we analyzed genetic variation of SARS-CoV-2 and its related coronavirus and found the evidence of intergenomic recombination. After correction for mutational bias, analysis of 137 SARS-CoV-2 genomes as of 2/23/2020 revealed the excess of low frequency mutations on both synonymous and nonsynonymous sites which is consistent with recent origin of the virus. In contrast to adaptive evolution previously reported for SARS-CoV in its brief epidemic in 2003, our analysis of SARS-CoV-2 genomes shows signs of relaxation of selection. The sequence similarity of the spike receptor binding domain between SARS-CoV-2 and a sequence from pangolin is probably due to an ancient intergenomic introgression. Therefore, SARS-CoV-2 might have cryptically circulated within humans for years before being recently noticed. Data from the early outbreak and hospital archives are needed to trace its evolutionary path and reveal critical steps required for effective spreading. Two mutations, 84S in orf8 protein and 251V in orf3 protein, occurred coincidentally with human intervention. The 84S first appeared on 1/5/2020 and reached a plateau around 1/23/2020, the lockdown of Wuhan. 251V emerged on 1/21/2020 and rapidly increased its frequency. Thus, the roles of these mutations on infectivity need to be elucidated. Genetic diversity of SARS-CoV-2 collected from China was two time higher than those derived from the rest of the world. In addition, in network analysis, haplotypes collected from Wuhan city were at interior and have more mutational connections, both of which are consistent with the observation that the outbreak of cov-19 was originated from China. SUMMARY In contrast to adaptive evolution previously reported for SARS-CoV in its brief epidemic, our analysis of SARS-CoV-2 genomes shows signs of relaxation of selection. The sequence similarity of the spike receptor binding domain between SARS-CoV-2 and a sequence from pangolin is probably due to an ancient intergenomic introgression. Therefore, SARS-CoV-2 might have cryptically circulated within humans for years before being recently noticed. Data from the early outbreak and hospital archives are needed to trace its evolutionary path and reveal critical steps required for effective spreading. Two mutations, 84S in orf8 protein and 251V in orf3 protein, occurred coincidentally with human intervention. The 84S first appeared on 1/5/2020 and reached a plateau around 1/23/2020, the lockdown of Wuhan. 251V emerged on 1/21/2020 and rapidly increased its frequency. Thus, the roles of these mutations on infectivity need to be elucidated.\n",
      "A novel RNA virus, the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), is responsible for the ongoing outbreak of coronavirus disease 2019 (COVID-19). Population genetic analysis could be useful for investigating the origin and evolutionary dynamics of COVID-19. However, due to extensive sampling bias and existence of infection clusters during the epidemic spread, direct applications of existing approaches can lead to biased parameter estimations and data misinterpretation. In this study, we first present robust estimator for the time to the most recent common ancestor (TMRCA) and the mutation rate, and then apply the approach to analyze 12,909 genomic sequences of SARS-CoV-2. The mutation rate is inferred to be 8.69 × 10(-4) per site per year with a 95% confidence interval (CI) of [8.61 × 10(-4), 8.77 × 10(-4)], and the TMRCA of the samples inferred to be Nov 28, 2019 with a 95% CI of [Oct 20, 2019, Dec 9, 2019]. The results indicate that COVID-19 might originate earlier than and outside of Wuhan Seafood Market. We further demonstrate that genetic polymorphism patterns, including the enrichment of specific haplotypes and the temporal allele frequency trajectories generated from infection clusters, are similar to those caused by evolutionary forces such as natural selection. Our results show that population genetic methods need to be developed to efficiently detangle the effects of sampling bias and infection clusters to gain insights into the evolutionary mechanism of SARS-CoV-2. Software for implementing VirusMuT can be downloaded at https://bigd.big.ac.cn/biocode/tools/BT007081.\n",
      "BACKGROUND: In late December, 2019, patients presenting with viral pneumonia due to an unidentified microbial agent were reported in Wuhan, China. A novel coronavirus was subsequently identified as the causative pathogen, provisionally named 2019 novel coronavirus (2019-nCoV). As of Jan 26, 2020, more than 2000 cases of 2019-nCoV infection have been confirmed, most of which involved people living in or visiting Wuhan, and human-to-human transmission has been confirmed. METHODS: We did next-generation sequencing of samples from bronchoalveolar lavage fluid and cultured isolates from nine inpatients, eight of whom had visited the Huanan seafood market in Wuhan. Complete and partial 2019-nCoV genome sequences were obtained from these individuals. Viral contigs were connected using Sanger sequencing to obtain the full-length genomes, with the terminal regions determined by rapid amplification of cDNA ends. Phylogenetic analysis of these 2019-nCoV genomes and those of other coronaviruses was used to determine the evolutionary history of the virus and help infer its likely origin. Homology modelling was done to explore the likely receptor-binding properties of the virus. FINDINGS: The ten genome sequences of 2019-nCoV obtained from the nine patients were extremely similar, exhibiting more than 99·98% sequence identity. Notably, 2019-nCoV was closely related (with 88% identity) to two bat-derived severe acute respiratory syndrome (SARS)-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21, collected in 2018 in Zhoushan, eastern China, but were more distant from SARS-CoV (about 79%) and MERS-CoV (about 50%). Phylogenetic analysis revealed that 2019-nCoV fell within the subgenus Sarbecovirus of the genus Betacoronavirus, with a relatively long branch length to its closest relatives bat-SL-CoVZC45 and bat-SL-CoVZXC21, and was genetically distinct from SARS-CoV. Notably, homology modelling revealed that 2019-nCoV had a similar receptor-binding domain structure to that of SARS-CoV, despite amino acid variation at some key residues. INTERPRETATION: 2019-nCoV is sufficiently divergent from SARS-CoV to be considered a new human-infecting betacoronavirus. Although our phylogenetic analysis suggests that bats might be the original host of this virus, an animal sold at the seafood market in Wuhan might represent an intermediate host facilitating the emergence of the virus in humans. Importantly, structural analysis suggests that 2019-nCoV might be able to bind to the angiotensin-converting enzyme 2 receptor in humans. The future evolution, adaptation, and spread of this virus warrant urgent investigation. FUNDING: National Key Research and Development Program of China, National Major Project for Control and Prevention of Infectious Disease in China, Chinese Academy of Sciences, Shandong First Medical University.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: ayhsbfd1\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "what is the origin of COVID-19\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "To investigate the evolutionary history of the current pandemic outbreak of severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2), a total of 137 genomes of coronavirus strains with release dates between January 2019 and 25 March 2020, were analyzed. To investigate the potential intermediate host of the SARS‐CoV‐2, we analyzed spike glycoprotein sequences from different animals, with particular emphasis on bats. We performed phylogenetic analysis and structural reconstruction of the spike glycoproteins with subsequent alignment and comparison. Our phylogenetic results revealed that SARS‐CoV‐2 was more similar to the bats' betacoronavirus isolates: HKU5‐related from Pipistrellus abramus and HKU4‐related from Tylonycteris pachypus. We also identified a yak betacoronavirus strain, YAK/HY24/CH/2017, as the closest match in the comparison of the structural models of spike glycoproteins. Interestingly, a set of unique features has been described for this particular strain of the yak betacoronavirus. Therefore, our results suggest that the human SARS‐CoV‐2, responsible for the current outbreak of COVID‐19, could also come from yak as an intermediate host.\n",
      "To investigate the evolutionary history of the current pandemic outbreak of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), a total of 137 genomes of coronavirus strains with release dates between January 2019 and 25 March 2020, were analyzed. To investigate the potential intermediate host of the SARS-CoV-2, we analyzed spike glycoprotein sequences from different animals, with particular emphasis on bats. We performed phylogenetic analysis and structural reconstruction of the spike glycoproteins with subsequent alignment and comparison. Our phylogenetic results revealed that SARS-CoV-2 was more similar to the bats' betacoronavirus isolates: HKU5-related from Pipistrellus abramus and HKU4-related from Tylonycteris pachypus. We also identified a yak betacoronavirus strain, YAK/HY24/CH/2017, as the closest match in the comparison of the structural models of spike glycoproteins. Interestingly, a set of unique features has been described for this particular strain of the yak betacoronavirus. Therefore, our results suggest that the human SARS-CoV-2, responsible for the current outbreak of COVID-19, could also come from yak as an intermediate host.\n",
      "To investigate the evolutionary history of the recent outbreak of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in China, a total of 70 genomes of virus strains from China and elsewhere with sampling dates between 24 December 2019 and 3 February 2020 were analyzed. To explore the potential intermediate animal host of the SARS-CoV-2 virus, we reanalyzed virome data sets from pangolins and representative SARS-related coronaviruses isolates from bats, with particular attention paid to the spike glycoprotein gene. We performed phylogenetic, split network, transmission network, likelihood-mapping, and comparative analyses of the genomes. Based on Bayesian time-scaled phylogenetic analysis using the tip-dating method, we estimated the time to the most recent common ancestor and evolutionary rate of SARS-CoV-2, which ranged from 22 to 24 November 2019 and 1.19 to 1.31 × 10-3 substitutions per site per year, respectively. Our results also revealed that the BetaCoV/bat/Yunnan/RaTG13/2013 virus was more similar to the SARS-CoV-2 virus than the coronavirus obtained from the two pangolin samples (SRR10168377 and SRR10168378). We also identified a unique peptide (PRRA) insertion in the human SARS-CoV-2 virus, which may be involved in the proteolytic cleavage of the spike protein by cellular proteases, and thus could impact host range and transmissibility. Interestingly, the coronavirus carried by pangolins did not have the RRAR motif. Therefore, we concluded that the human SARS-CoV-2 virus, which is responsible for the recent outbreak of COVID-19, did not come directly from pangolins.\n",
      "To investigate the evolutionary history of the recent outbreak of severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2) in China, a total of 70 genomes of virus strains from China and elsewhere with sampling dates between 24 December 2019 and 3 February 2020 were analyzed. To explore the potential intermediate animal host of the SARS‐CoV‐2 virus, we reanalyzed virome data sets from pangolins and representative SARS‐related coronaviruses isolates from bats, with particular attention paid to the spike glycoprotein gene. We performed phylogenetic, split network, transmission network, likelihood‐mapping, and comparative analyses of the genomes. Based on Bayesian time‐scaled phylogenetic analysis using the tip‐dating method, we estimated the time to the most recent common ancestor and evolutionary rate of SARS‐CoV‐2, which ranged from 22 to 24 November 2019 and 1.19 to 1.31 × 10(−3) substitutions per site per year, respectively. Our results also revealed that the BetaCoV/bat/Yunnan/RaTG13/2013 virus was more similar to the SARS‐CoV‐2 virus than the coronavirus obtained from the two pangolin samples (SRR10168377 and SRR10168378). We also identified a unique peptide (PRRA) insertion in the human SARS‐CoV‐2 virus, which may be involved in the proteolytic cleavage of the spike protein by cellular proteases, and thus could impact host range and transmissibility. Interestingly, the coronavirus carried by pangolins did not have the RRAR motif. Therefore, we concluded that the human SARS‐CoV‐2 virus, which is responsible for the recent outbreak of COVID‐19, did not come directly from pangolins.\n",
      "OBJECTIVE To analyze the evolution and variation of SARS-CoV-2 during the epidemic starting at the end of 2019. METHODS We downloaded the full-length genome sequence of SARS-CoV-2 from the databases of GISAID and NCBI. Using the software for bioinformatics including MEGA-X, BEAST, and TempEst, we constructed the genomic evolution tree, inferred the time evolution signal of the virus, calculated the tMRCA time of the virus and analyzed the selection pressure of the virus during evolution. RESULTS The phylogenetic tree showed that SARS-CoV-2 belonged to the Sarbecovirus subgenus of β Coronavirus genus together with bat coronavirus BetaCoV/bat/Yunnan/RaTG13/2013, bat-SL-CoVZC45, bat-SL-CoVZXC21 and SARS-CoV. The genomic sequences of SARS-CoV-2 isolated from the ongoing epidemic showed a weak time evolution signal with an average tMRCA time of 73 days (95% CI: 38.9-119.3 days). No positive time evolution signal was found between SARS-CoV-2 and BetaCoV/bat/Yunnan/RaTG13/2013, but the former virus had a strong positive temporal evolution relationship with bat-SL-CoVZC45 and SARS-CoV. The major cause for mutations of SARS-CoV-2 was the pressure of purification selection during the epidemic. CONCLUSIONS SARS-CoV-2 may have emerged as early as November, 2019, originating most likely from bat-associated coronavirus. This finding may provide evidence for tracing the sources and evolution of the virus.\n",
      "The outbreak of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in Wuhan city, China in December 2019 and thereafter its spillover across the world has created a global pandemic and public health crisis. Today, it appears as a threat to human civilization. Scientists and medical practitioners across the world are involved to trace out the origin and evolution of SARS-CoV-2 (also called 2019 novel coronavirus and referred as 2019-nCoV), its transmission route, cause of pathogenicity, and possible remedial action. In this work, we aim to find out the origin, evolutionary patternthat led to its pathogenicity and possible transmission pathway of 2019-nCoV. To achieve the aims we conducted a large-scale deep phylogenetic analysis on the 162 complete Orthocoronavirinae genomes consisting of four genera namely Alphacoronavirus, Betacoronavirus, Deltacoronavirus and Gammacoronavirus, their gene trees analysis and subsequently genome and gene recombination analyses. Our analyses revealed that i) bat, pangolin and anteater are the natural hosts of 2019-nCoV, ii) outbreak of 2019-nCoV took place via inter-intra species transmission mode, iii) host-specific adaptive mutation made 2019-nCoV more virulent, and the presence of widespread recombination events led to the evolution of new 2019-nCoV strain and/or could be determinant of its pathogenicity. Highlights Orthocoronavirinae genome phylogeny revealed that bat, pangolin and anteater are natural reservoir hosts of novel coronavirus (2019-nCoV/SARS-CoV-2). Host-specific adaptive mutation occurred among the coronaviruses. Transmission of 2019-nCoV to human took place by inter-intra species mode of transmission. Presence of widespread recombination events led to the evolution of new 2019-nCoV strain and/or could be determinant of its pathogenicity.\n",
      "OBJECTIVE: To analyze the evolution and variation of SARS-CoV-2 during the epidemic starting at the end of 2019. METHODS: We downloaded the full-length genome sequence of SARS-CoV-2 from the databases of GISAID and NCBI. Using the software for bioinformatics including MEGA-X, BEAST, and TempEst, we constructed the genomic evolution tree, inferred the time evolution signal of the virus, calculated the tMRCA time of the virus and analyzed the selection pressure of the virus during evolution. RESULTS: The phylogenetic tree showed that SARS-CoV-2 belonged to the Sarbecovirus subgenus of ß Coronavirus genus together with bat coronavirus BetaCoV/bat/Yunnan/RaTG13/2013, bat-SL-CoVZC45, bat-SL-CoVZXC21 and SARS-CoV. The genomic sequences of SARS-CoV-2 isolated from the ongoing epidemic showed a weak time evolution signal with an average tMRCA time of 73 days (95% CI: 38.9-119.3 days). No positive time evolution signal was found between SARS-CoV-2 and BetaCoV/bat/Yunnan/RaTG13/2013, but the former virus had a strong positive temporal evolution relationship with bat-SL-CoVZC45 and SARS-CoV. The major cause for mutations of SARS-CoV-2 was the pressure of purification selection during the epidemic. CONCLUSIONS: SARS-CoV-2 may have emerged as early as November, 2019, originating most likely from bat-associated coronavirus. This finding may provide evidence for tracing the sources and evolution of the virus.\n",
      "SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) is a novel coronavirus causing an outbreak of COVID-19 globally in the past six months. A relatively higher divergence on the spike protein of SASR-CoV-2 enables it to transmit across species efficiently. We particularly believe that the adaptive mutations of the receptor-binding domain (RBD) of spike protein in SARS-CoV-2 might be essential to its high transmissibility among humans. Thus here we collected 2,142 high-quality genome sequences of SARS-CoV-2 from 160 regions in over 50 countries and reconstructed their phylogeny, and also analyzed the interaction between the polymorphisms of spike protein and human ACE2 (hACE2). Phylogenetic analysis of SARS-CoV-2 and coronavirus in other hosts show SARS-CoV-2 is highly possible originated from Bat-CoV (RaTG13) found in horseshoe bat and a recombination event may occur on the spike protein of Pangolin-CoV to imbue it the ability to infect humans. Moreover, compared to the S gene of SARS-CoV-2, it is more conserved in the direct-binding sites of RBD and we noticed that spike protein of SARS-CoV-2 may under a consensus evolution to adapt to human hosts better. 3,860 amino acid mutations in spike protein RBD (T333-C525) of SARS-CoV-2 were simulated and their stability and affinity binding to hACE2 (S19-D615) were calculated. Our analysis indicates SARS-CoV-2 could infect humans from different populations with no preference, and a higher divergence in the spike protein of SARS-CoV-2 at the early stage of this pandemic may be a good indicator that could show the pathway of SARS-CoV-2 transmitting from the natural reservoir to human beings.\n",
      "The 2019 novel coronavirus (2019-nCoV/SARS-CoV-2) originally arose as part of a major outbreak of respiratory disease centered on Hubei province, China. It is now a global pandemic and is a major public health concern. Taxonomically, SARS-CoV-2 was shown to be a Betacoronavirus (lineage B) closely related to SARS-CoV and SARS-related bat coronaviruses, and it has been reported to share a common receptor with SARS-CoV (ACE-2). Subsequently, betacoronaviruses from pangolins were identified as close relatives to SARS-CoV-2. Here, we perform structural modeling of the SARS-CoV-2 spike glycoprotein. Our data provide support for the similar receptor utilization between SARS-CoV-2 and SARS-CoV, despite a relatively low amino acid similarity in the receptor binding module. Compared to SARS-CoV and all other coronaviruses in Betacoronavirus lineage B, we identify an extended structural loop containing basic amino acids at the interface of the receptor binding (S1) and fusion (S2) domains. We suggest this loop confers fusion activation and entry properties more in line with betacoronaviruses in lineages A and C, and be a key component in the evolution of SARS-CoV-2 with this structural loop affecting virus stability and transmission.\n",
      "The novel coronavirus (CoV), severe acute respiratory syndrome (SARS)-CoV-2 is an international public health emergency. Until now, the intermediate host and mechanisms of the interspecies jump of this virus are unknown. Phylogenetic analysis of all available bat CoV complete genomes was performed to analyze the relationships between bat CoV and SARS-CoV-2. To suggest a possible intermediate host, another phylogenetic reconstruction of CoV genomes obtained from animals that were hypothetically commercialized in the Chinese markets was also carried out. Moreover, mutation analysis was executed to suggest genomic regions that may have permitted the adaptation of SARS-CoV-2 to the human host. The phylogenetic analysis demonstrated that SARS-CoV-2 formed a cluster with the bat CoV isolate RaTG13. Possible CoV interspecies jumps among bat isolates were also observed. The phylogenetic tree reconstructed from CoV strains belonging to different animals demonstrated that SARS-CoV-2, bat RaTG13, and pangolin CoV genomes formed a monophyletic cluster, demonstrating that pangolins may be suggested as SARS-CoV-2 intermediate hosts. Three AA substitutions localized in the S1 portion of the S gene were observed, some of which have been correlated to structural modifications of the S protein which may facilitate SARS-CoV-2 tropism to human cells. Our analysis shows the tight relationship between SARS-CoV-2 and bat SARS-like strains. It also hypothesizes that pangolins might have been possible intermediate hosts of the infection. Some of the observed AA substitutions in the S-binding protein may serve as possible adaptation mutations in humans but more studies are needed to elucidate their function.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: qw05apnf\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what is the origin of COVID-19\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "what evidence is there related to COVID-19 super spreaders\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "The human pathogen severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is responsible for the major pandemic of the 21st century. We analyzed >4,700 SARS-CoV-2 genomes and associated meta-data retrieved from public repositories. SARS-CoV-2 sequences have a high sequence identity (>99.9%), which drops to >96% when compared to bat coronavirus. We built a mutation-annotated reference SARS-CoV-2 phylogeny with two main macro-haplogroups, A and B, both of Asian origin, and >160 sub-branches representing virus strains of variable geographical origins worldwide, revealing a uniform mutation occurrence along branches that could complicate the design of future vaccines. The root of SARS-CoV-2 genomes locates at the Chinese haplogroup B1, with a TMRCA dating to 12 November 2019 - thus matching epidemiological records. Sub-haplogroup A2a originates in China and represents the major non-Asian outbreak. Multiple founder effect episodes, most likely associated with super-spreader hosts, explain COVID-19 pandemic to a large extent.\n",
      "Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is an emergent RNA virus that spread around the planet in about 4 months. The consequences of this rapid spread on the virus evolution are under investigation. In this work, we analyzed ca. 9,000 SARS-CoV-2 genomes from Africa, America, Asia, Europe, and Oceania. We show that the virus is a complex of slightly different genetic variants that are unevenly distributed on Earth. Furthermore, we demonstrate that SARS-CoV-2 phylogeny is spatially structured. We hypothesize this could be the result of founder effects occurring as a consequence of, and local evolution occurring after, long-distance dispersal. In light of our results, we discuss how dispersal may constitute an opportunity for the virus to fix otherwise rare, and/or develop new, mutations. Based on previous studies, the possibility that this could significantly affect the virus phenotype is not remote. Relevance A cluster of pneumonia cases of unknown etiology was reported in December 2019 in Wuhan, Hubei province, China. Since then, hundreds of thousands of people have died all around the planet. Quickly after the pandemic onset, metagenomic studies showed the causative agent, now named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), belonged to the Betacoronavirus genus, responsible for spillover events in 2002 and 2012 (severe acute respiratory syndrome and Middle East respiratory syndrome, respectively). The zoonotic origins of these viruses (possibly bats, camelids, pangolins and/or palm civets) have received much attention. However, other evolutionary aspects, such as spatial variation, have received comparatively little attention. This study shows that SARS-CoV-2 variants, which we call virotypes, are heterogeneously distributed on Earth and demonstrates that the virus phylogeny is geographically structured. We explain how this may be due to founder effects combined with high mutation rates.\n",
      "OBJECTIVES: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan, China, in December 2019 and has been rapidly spreading worldwide. Although the causal relationship among mutations and the features of SARS-CoV-2 such as rapid transmission, pathogenicity, and tropism, remains unclear, our results of genomic mutations in SARS-CoV-2 may help to interpret the interaction between genomic characterization in SARS-CoV-2 and infectivity with the host. METHODS: A total of 4,254 genomic sequences of SARS-CoV-2 were collected from the Global Initiative on Sharing all Influenza Data (GISAID). Multiple sequence alignment for phylogenetic analysis and comparative genomic approach for mutation analysis were conducted using Molecular Evolutionary Genetics Analysis (MEGA), and an in-house program based on Perl language, respectively. RESULTS: Phylogenetic analysis of SARS-CoV-2 strains indicated that there were 3 major clades including S, V, and G, and 2 subclades (G.1 and G.2). There were 767 types of synonymous and 1,352 types of non-synonymous mutation. ORF1a, ORF1b, S, and N genes were detected at high frequency, whereas ORF7b and E genes exhibited low frequency. In the receptor-binding domain (RBD) of the S gene, 11 non-synonymous mutations were observed in the region adjacent to the angiotensin converting enzyme 2 (ACE2) binding site. CONCLUSION: It has been reported that the rapid infectivity and transmission of SARS-CoV-2 associated with host receptor affinity are derived from several mutations in its genes. Without these genetic mutations to enhance evolutionary adaptation, species recognition, host receptor affinity, and pathogenicity, it would not survive. It is expected that our results could provide an important clue in understanding the genomic characteristics of SARS-CoV-2.\n",
      "A novel coronavirus known as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the cause of the ongoing COVID-19 pandemic. In this study, we performed a comprehensive epidemiological and genomic analysis of SARS-CoV-2 genomes from ten patients in Shaoxing, a mid-sized city outside of the epicenter Hubei province, China, during the early stage of the outbreak (late January to early February, 2020). We obtained viral genomes with > 99% coverage and a mean depth of 296X demonstrating that viral genomic analysis is feasible via metagenomics sequencing directly on nasopharyngeal samples with SARS-CoV-2 Real-time PCR Ct values less than 28. We found that a cluster of 4 patients with travel history to Hubei shared the exact same virus with patients from Wuhan, Taiwan, Belgium and Australia, highlighting how quickly this virus spread to the globe. The virus from another cluster of husband and wife without travel history but with a sick contact of a confirmed case from another city outside of Hubei accumulated significantly more mutations (9 SNPs vs average 4 SNPs), suggesting a complex and dynamic nature of this outbreak. We also found 70% patients in this study had the S genotype, consistent with an early study showing a higher prevalence of S genotype out of Hubei than that inside Hubei. We calculated an average mutation rate of 1.37x10-3 nucleotide substitution per site per year, which is similar to that of other coronaviruses. Our findings add to the growing knowledge of the epidemiological and genomic characteristics of SARS-CoV-2 that are important for guiding outbreak containment and vaccine development. The moderate mutation rate of this virus also lends hope that development of an effective, long-lasting vaccine may be possible.\n",
      "SARS-CoV-2 is a betacoronavirus responsible for the COVID-19 pandemic. Although the SARS-CoV-2 genome was reported recently, its transcriptomic architecture is unknown. Utilizing two complementary sequencing techniques, we present a high-resolution map of the SARS-CoV-2 transcriptome and epitranscriptome. DNA nanoball sequencing shows that the transcriptome is highly complex owing to numerous discontinuous transcription events. In addition to the canonical genomic and 9 subgenomic RNAs, SARS-CoV-2 produces transcripts encoding unknown ORFs with fusion, deletion, and/or frameshift. Using nanopore direct RNA sequencing, we further find at least 41 RNA modification sites on viral transcripts, with the most frequent motif, AAGAA. Modified RNAs have shorter poly(A) tails than unmodified RNAs, suggesting a link between the modification and the 3' tail. Functional investigation of the unknown transcripts and RNA modifications discovered in this study will open new directions to our understanding of the life cycle and pathogenicity of SARS-CoV-2.\n",
      "SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) is a novel coronavirus causing an outbreak of COVID-19 globally in the past six months. A relatively higher divergence on the spike protein of SASR-CoV-2 enables it to transmit across species efficiently. We particularly believe that the adaptive mutations of the receptor-binding domain (RBD) of spike protein in SARS-CoV-2 might be essential to its high transmissibility among humans. Thus here we collected 2,142 high-quality genome sequences of SARS-CoV-2 from 160 regions in over 50 countries and reconstructed their phylogeny, and also analyzed the interaction between the polymorphisms of spike protein and human ACE2 (hACE2). Phylogenetic analysis of SARS-CoV-2 and coronavirus in other hosts show SARS-CoV-2 is highly possible originated from Bat-CoV (RaTG13) found in horseshoe bat and a recombination event may occur on the spike protein of Pangolin-CoV to imbue it the ability to infect humans. Moreover, compared to the S gene of SARS-CoV-2, it is more conserved in the direct-binding sites of RBD and we noticed that spike protein of SARS-CoV-2 may under a consensus evolution to adapt to human hosts better. 3,860 amino acid mutations in spike protein RBD (T333-C525) of SARS-CoV-2 were simulated and their stability and affinity binding to hACE2 (S19-D615) were calculated. Our analysis indicates SARS-CoV-2 could infect humans from different populations with no preference, and a higher divergence in the spike protein of SARS-CoV-2 at the early stage of this pandemic may be a good indicator that could show the pathway of SARS-CoV-2 transmitting from the natural reservoir to human beings.\n",
      "Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2), which was first reported in Wuhan, China in November 2019 has developed into a pandemic since March 2020, causing substantial human casualties and economic losses. Studies on SARS-CoV-2 are being carried out at an unprecedented rate to tackle this threat. Genomics studies, in particular, are indispensable to elucidate the dynamic nature of the RNA genome of SARS-CoV-2. RNA viruses are marked by their unique ability to undergo high rates of mutation in their genome, much more frequently than their hosts, which diversifies their strengths qualifying them to elude host immune response and amplify drug resistance. In this study, we sequenced and analyzed the genomic information of the SARS-CoV-2 isolates from two infected Indian patients and explored the possible implications of point mutations in its biology. In addition to multiple point mutations, we found a remarkable similarity between relatively common mutations of 36-nucleotide deletion in ORF8 of SARS-CoV-2. Our results corroborate with the earlier reported 29-nucleotide deletion in SARS, which was frequent during the early stage of human-to-human transmission. The results will be useful to understand the biology of SARS-CoV-2 and itsattenuation for vaccine development.\n",
      "The ongoing pandemic of the coronavirus disease 2019 (COVID-19) is an infectious disease caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV2). We have performed an integrated sequence-based analysis of SARS-CoV2 genomes from different geographical locations in order to identify its unique features absent in SARS-CoV and other related coronavirus family genomes, conferring unique infection, facilitation of transmission, virulence and immunogenic features to the virus. The phylogeny of the genomes yields some interesting results. Systematic gene level mutational analysis of the genomes has enabled us to identify several unique features of the SARS-CoV2 genome, which includes a unique mutation in the spike surface glycoprotein (A930V (24351C>T)) in the Indian SARS-CoV2, absent in other strains studied here. We have also predicted the impact of the mutations in the spike glycoprotein function and stability, using computational approach. To gain further insights into host responses to viral infection, we predict that antiviral host-miRNAs may be controlling the viral pathogenesis. Our analysis reveals nine host miRNAs which can potentially target SARS-CoV2 genes. Interestingly, the nine miRNAs do not have targets in SARS and MERS genomes. Also, hsa-miR-27b is the only unique miRNA which has a target gene in the Indian SARS-CoV2 genome. We also predicted immune epitopes in the genomes\n",
      "The COVID-19 pandemic is caused by the single-stranded RNA virus severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), a virus of zoonotic origin that was first detected in Wuhan, China in December 2019. There is evidence that homologous recombination contributed to this cross-species transmission. Since that time the virus has demonstrated a high propensity for human-to-human transmission. Here we report two newly identified adjacent amino acid polymorphisms in the nucleocapsid at positions 203 and 204 (R203K/G204R) due to three adjacent nucleotide changes across the two codons (i.e. AGG GGA to AAA CGA). This new strain within the LGG clade may have arisen by a form of homologous recombination from the core sequence (CS-B) of the transcription-regulating sequences of SAS-CoV-2 itself and has rapidly increased to approximately one third of reported sequences from Europe during the month of March 2020. We note that these polymorphisms are predicted to reduce the binding of an overlying putative HLA-C*07-restricted epitope and that HLA-C*07 is prevalent in Caucasians being carried by >40% of the population. The findings suggest that homologous recombination may have occurred since its introduction into humans and be a mechanism for increased viral fitness and adaptation of SARS-CoV-2 to human populations.\n",
      "SARS-CoV-2 is a SARS-like coronavirus of likely zoonotic origin first identified in December 2019 in Wuhan, the capital of China's Hubei province. The virus has since spread globally, resulting in the currently ongoing COVID-19 pandemic. The first whole genome sequence was published on January 5 2020, and thousands of genomes have been sequenced since this date. This resource allows unprecedented insights into the past demography of SARS-CoV-2 but also monitoring of how the virus is adapting to its novel human host, providing information to direct drug and vaccine design. We curated a dataset of 7666 public genome assemblies and analysed the emergence of genomic diversity over time. Our results are in line with previous estimates and point to all sequences sharing a common ancestor towards the end of 2019, supporting this as the period when SARS-CoV-2 jumped into its human host. Due to extensive transmission, the genetic diversity of the virus in several countries recapitulates a large fraction of its worldwide genetic diversity. We identify regions of the SARS-CoV-2 genome that have remained largely invariant to date, and others that have already accumulated diversity. By focusing on mutations which have emerged independently multiple times (homoplasies), we identify 198 filtered recurrent mutations in the SARS-CoV-2 genome. Nearly 80% of the recurrent mutations produced non-synonymous changes at the protein level, suggesting possible ongoing adaptation of SARS-CoV-2. Three sites in Orf1ab in the regions encoding Nsp6, Nsp11, Nsp13, and one in the Spike protein are characterised by a particularly large number of recurrent mutations (>15 events) which may signpost convergent evolution and are of particular interest in the context of adaptation of SARS-CoV-2 to the human host. We additionally provide an interactive user-friendly web-application to query the alignment of the 7666 SARS-CoV-2 genomes.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: bu1bc2bi\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "This study explores the divergence pattern of SARS-CoV-2 using whole genome sequences of the isolates from various COVID-19 affected countries. The phylogenomic analysis indicates the presence of at least four distinct groups of the SARS-CoV-2 genomes. The emergent groups have been found to be associated with signature structural changes in specific proteins. Also, this study reveals the differential levels of divergence patterns for the protein coding regions. Moreover, we have predicted the impact of structural changes on a couple of important viral proteins via structural modelling techniques. This study further advocates for more viral genetic studies with associated clinical outcomes and hosts’ response for better understanding of SARS-CoV-2 pathogenesis enabling better mitigation of this pandemic situation.\n",
      "In this study, we analyzed full-length SARS-CoV-2 genomes from multiple countries to determine early trends in the evolutionary dynamics of the novel COVID-19 pandemic. Results indicated SARS-CoV-2 evolved early into at least three phylogenetic groups, characterized by positive selection at specific residues of the accessory proteins OFR3a and ORF8a. We also report evidence of epistatic interactions among sites in the genome that may be important in the generation of variants adapted to humans. These observations might impact not only public health, but also suggest more studies are needed to understand the genetic mechanisms that may affect the development of therapeutic and preventive tools, like antivirals and vaccines.\n",
      "The COVID-19 pandemic is the most important public health threat in recent history. Here we study how its causal agent, SARS-CoV-2, has diversified genetically since its first emergence in December 2019. We have created a pipeline combining both phylogenetic and structural analysis to identify possible human-adaptation related mutations in a data set consisting of 4,894 SARS-CoV-2 complete genome sequences. Although the phylogenetic diversity of SARS-CoV-2 is low, the whole genome phylogenetic tree can be divided into five clusters/clades based on the tree topology and clustering of specific mutations, but its branches exhibit low genetic distance and bootstrap support values. We also identified 11 residues that are high-frequency substitutions, with four of them currently showing some signal for potential positive selection. These fast-evolving sites are in the non-structural proteins nsp2, nsp5 (3CL-protease), nsp6, nsp12 (polymerase) and nsp13 (helicase), in accessory proteins (ORF3a, ORF8) and in the structural proteins N and S. Temporal and spatial analysis of these potentially adaptive mutations revealed that the incidence of some of these sites was declining after having reached an (often local) peak, whereas the frequency of other sites is continually increasing and now exhibit a worldwide distribution. Structural analysis revealed that the mutations are located on the surface of the proteins that modulate biochemical properties. We speculate that this improves binding to cellular proteins and hence represents fine-tuning of adaptation to human cells. Our study has implications for the design of biochemical and clinical experiments to assess whether important properties of SARS-CoV-2 have changed during the epidemic.\n",
      "Considering the current pandemic of COVID-19, it is imperative to gauge the role of molecular divergence in SARS-CoV-2 with time, due to clinical and epidemiological concerns. Our analyses involving molecular phylogenetics is a step toward understanding the transmission clusters that can be correlated to pathophysiology of the disease to gain insight into virulence mechanism. As the infections are increasing rapidly, more divergence is expected followed possibly by viral adaptation. We could identify mutational hotspots which appear to be major drivers of diversity among strains, with RBD of spike protein emerging as the key region involved in interaction with ACE2 and consequently a major determinant of infection outcome. We believe that such molecular analyses correlated with clinical characteristics and host predisposition need to be evaluated at the earliest to understand viral adaptability, disease prognosis, and transmission dynamics.\n",
      "This study presents a comprehensive phylogenetic analysis of SARS-CoV2 isolates to understand discrete mutations that are occurring between patient samples. The analysis unravel various amino acid mutations in the viral proteins which may provide an explanation for varying treatment efficacies of different inhibitory drugs and a future direction towards a combinatorial treatment therapies based on the kind of mutation in the viral genome.\n",
      "Abstract Considering the current pandemic of COVID-19, it is imperative to gauge the role of molecular divergence in SARS-CoV-2 with time, due to clinical and epidemiological concerns. Our analyses involving molecular phylogenetics is a step toward understanding the transmission clusters that can be correlated to pathophysiology of the disease to gain insight into virulence mechanism. As the infections are increasing rapidly, more divergence is expected followed possibly by viral adaptation. We could identify mutational hotspots which appear to be major drivers of diversity among strains, with RBD of spike protein emerging as the key region involved in interaction with ACE2 and consequently a major determinant of infection outcome. We believe that such molecular analyses correlated with clinical characteristics and host predisposition need to be evaluated at the earliest to understand viral adaptability, disease prognosis, and transmission dynamics.\n",
      "The newly identified SARS-CoV-2 has now been reported from around 183 countries with more than a million confirmed human cases including more than 68000 deaths. The genomes of SARS-COV-2 strains isolated from different parts of the world are now available and the unique features of constituent genes and proteins have gotten substantial attention recently. Spike glycoprotein is widely considered as a possible target to be explored because of its role during the entry of coronaviruses into host cells. We analyzed 320 whole-genome sequences and 320 spike protein sequences of SARS-CoV-2 using multiple sequence alignment tools. In this study, 483 unique variations have been identified among the genomes including 25 non-synonymous mutations and one deletion in the spike protein of SARS-CoV-2. Among the 26 variations detected, 12 variations were located at the N-terminal domain and 6 variations at the receptor-binding domain (RBD) which might alter the interaction with receptor molecules. In addition, 22 amino acid insertions were identified in the spike protein of SARS-CoV-2 in comparison with that of SARS-CoV. Phylogenetic analyses of spike protein revealed that Bat coronavirus have a close evolutionary relationship with circulating SARS-CoV-2. The genetic variation analysis data presented in this study can help a better understanding of SARS-CoV-2 pathogenesis. Based on our findings, potential inhibitors can be designed and tested targeting these proposed sites of variation.\n",
      "In late December 2019, an emerging viral infection COVID-19 was identified in Wuhan, China, and became a global pandemic. Characterization of the genetic variants of SARS-CoV-2 is crucial in following and evaluating it spread across countries. In this study, we collected and analyzed 3,067 SARS-CoV-2 genomes isolated from 55 countries during the first three months after the onset of this virus. Using comparative genomics analysis, we traced the profiles of the whole-genome mutations and compared the frequency of each mutation in the studied population. The accumulation of mutations during the epidemic period with their geographic locations was also monitored. The results showed 782 variant sites, of which 512 (65.47%) had a non-synonymous effect. Frequencies of mutated alleles revealed the presence of 38 recurrent non-synonymous mutations, including ten hotspot mutations with a prevalence higher than 0.10 in this population and distributed in six SARS-CoV-2 genes. The distribution of these recurrent mutations on the world map revealed certain genotypes specific to the geographic location. We also found co-occurring mutations resulting in the presence of several haplotypes. Moreover, evolution over time has shown a mechanism of mutation co-accumulation which might affect the severity and spread of the SARS-CoV-2. On the other hand, analysis of the selective pressure revealed the presence of negatively selected residues that could be taken into considerations as therapeutic targets We have also created an inclusive unified database (http://genoma.ma/covid-19/) that lists all of the genetic variants of the SARS-CoV-2 genomes found in this study with phylogeographic analysis around the world.\n",
      "Phylogenomic analysis of SARS-CoV-2 as available from publicly available repositories suggests the presence of 3 prevalent groups of viral episomes (super-clades), which are mostly associated with outbreaks in distinct geographic locations (China, USA and Europe). While levels of genomic variability between SARS-CoV-2 isolates are limited, to our knowledge, it is not clear whether the observed patterns of variability in viral super-clades reflect ongoing adaptation of SARS-CoV-2, or merely genetic drift and founder effects. Here, we analyze more than 1100 complete, high quality SARS-CoV-2 genome sequences, and provide evidence for the absence of distinct evolutionary patterns/signatures in the genomes of the currently known major clades of SARS-CoV-2. Our analyses suggest that the presence of distinct viral episomes at different geographic locations are consistent with founder effects, coupled with the rapid spread of this novel virus. We observe that while cross species adaptation of the virus is associated with hypervariability of specific protein coding regions (including the RDB domain of the spike protein), the more variable genomic regions between extant SARS-CoV-2 episomes correspond with the 3’ and 5’ UTRs, suggesting that at present viral protein coding genes should not be subjected to different adaptive evolutionary pressures in different viral strains. Although this study can not be conclusive, we believe that the evidence presented here is strongly consistent with the notion that the biased geographic distribution of SARS-CoV-2 isolates should not be associated with adaptive evolution of this novel pathogen.\n",
      "The COVID-19 pandemic is the most significant public health issue in recent history. Its causal agent, SARS-CoV-2, has evolved rapidly since its first emergence in December 2019. Mutations in the viral genome have critical impacts on the adaptation of viral strains to the local environment, and may alter the characteristics of viral transmission, disease manifestation, and the efficacy of treatment and vaccination. Using the complete sequences of 1,932 SARS-CoV-2 genomes, we examined the genomic, geographic and temporal distributions of aged, new, and frequent mutations of SARS-CoV-2, and identified six phylogenetic clusters of the strains, which also exhibit a geographic preference in different continents. Mutations in the form of single nucleotide variations (SNVs) provide a direct interpretation for the six phylogenetic clusters. Linkage disequilibrium, haplotype structure, evolutionary process, global distribution of mutations unveiled a sketch of the mutational history. Additionally, we found a positive correlation between the average mutation count and case fatality, and this correlation had strengthened with time, suggesting an important role of SNVs on disease outcomes. This study suggests that SNVs may become an important consideration in virus detection, clinical treatment, drug design, and vaccine development to avoid target shifting, and that continued isolation and sequencing is a crucial component in the fight against this pandemic. Significance Statement Mutation is the driving force of evolution for viruses like SARS-CoV-2, the causal agent of COVID-19. In this study, we discovered that the genome of SARS-CoV-2 is changing rapidly from the originally isolated form. These mutations have been spreading around the world and caused more than 2.5 million of infected cases and 170 thousands of deaths. We found that fourteen frequent mutations identified in this study can characterize the six main clusters of SARS-CoV-2 strains. In addition, we found the mutation burden is positively correlated with the fatality of COVID-19 patients. Understanding mutations in the SARS-CoV-2 genome will provide useful insight for the design of treatment and vaccination.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: m2cu5iof\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "what is the origin of COVID-19\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "The outbreak of a novel coronavirus, which was later formally named the severe acute respiratory coronavirus 2 (SARS-CoV-2), has caused a worldwide public health crisis. Previous studies showed that SARS-CoV-2 is highly homologous to SARS-CoV and infects humans through the binding of the spike protein to ACE2. Here, we have systematically studied the molecular mechanisms of human infection with SARS-CoV-2 and SARS-CoV by protein-protein docking and MD simulations. It was found that SARS-CoV-2 binds ACE2 with a higher affinity than SARS-CoV, which may partly explain that SARS-CoV-2 is much more infectious than SARS-CoV. In addition, the spike protein of SARS-CoV-2 has a significantly lower free energy than that of SARS-CoV, suggesting that SARS-CoV-2 is more stable and may survive a higher temperature than SARS-CoV. This provides insights into the evolution of SARS-CoV-2 because SARS-like coronaviruses have originated in bats. Our computation also suggested that the RBD-ACE2 binding for SARS-CoV-2 is much more temperature-sensitive than that for SARS-CoV. Thus, it is expected that SARS-CoV-2 would decrease its infection ability much faster than SARS-CoV when the temperature rises. These findings would be beneficial for the disease prevention and drug/vaccine development of SARS-CoV-2.\n",
      "Since December, 2019, an outbreak of pneumonia caused by the new coronavirus (2019-nCoV) has hit the city of Wuhan in the Hubei Province. With the continuous development of the epidemic, it has become a national public health crisis and calls for urgent antiviral treatments or vaccines. The spike protein on the coronavirus envelope is critical for host cell infection and virus vitality. Previous studies showed that 2019-nCoV is highly homologous to human SARS-CoV and attaches host cells though the binding of the spike receptor binding domain (RBD) domain to the angiotensin-converting enzyme II (ACE2). However, the molecular mechanisms of 2019-nCoV binding to human ACE2 and evolution of 2019-nCoV remain unclear. In this study, we have extensively studied the RBD-ACE2 complex, spike protein, and free RBD systems of 2019-nCoV and SARS-CoV using protein-protein docking and molecular dynamics (MD) simulations. It was shown that the RBD-ACE2 binding free energy for 2019-nCoV is significantly lower than that for SARS-CoV, which is consistent with the fact that 2019-nCoV is much more infectious than SARS-CoV. In addition, the spike protein of 2019-nCoV shows a significantly lower free energy than that of SARS-CoV, suggesting that 2019-nCoV is more stable and able to survive a higher temperature than SARS-CoV. This may also provide insights into the evolution of 2019-nCoV because SARS-like coronaviruses are thought to have originated in bats that are known to have a higher body-temperature than humans. It was also revealed that the RBD of 2019-nCoV is much more flexible especially near the binding site and thus will have a higher entropy penalty upon binding ACE2, compared to the RBD of SARS-CoV. That means that 2019-nCoV will be much more temperature-sensitive in terms of human infection than SARS-CoV. With the rising temperature, 2019-nCoV is expected to decrease its infection ability much faster than SARS-CoV, and get controlled more easily. The present findings are expected to be helpful for the disease prevention and control as well as drug and vaccine development of 2019-nCoV.\n",
      "SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) is a novel coronavirus causing an outbreak of COVID-19 globally in the past six months. A relatively higher divergence on the spike protein of SASR-CoV-2 enables it to transmit across species efficiently. We particularly believe that the adaptive mutations of the receptor-binding domain (RBD) of spike protein in SARS-CoV-2 might be essential to its high transmissibility among humans. Thus here we collected 2,142 high-quality genome sequences of SARS-CoV-2 from 160 regions in over 50 countries and reconstructed their phylogeny, and also analyzed the interaction between the polymorphisms of spike protein and human ACE2 (hACE2). Phylogenetic analysis of SARS-CoV-2 and coronavirus in other hosts show SARS-CoV-2 is highly possible originated from Bat-CoV (RaTG13) found in horseshoe bat and a recombination event may occur on the spike protein of Pangolin-CoV to imbue it the ability to infect humans. Moreover, compared to the S gene of SARS-CoV-2, it is more conserved in the direct-binding sites of RBD and we noticed that spike protein of SARS-CoV-2 may under a consensus evolution to adapt to human hosts better. 3,860 amino acid mutations in spike protein RBD (T333-C525) of SARS-CoV-2 were simulated and their stability and affinity binding to hACE2 (S19-D615) were calculated. Our analysis indicates SARS-CoV-2 could infect humans from different populations with no preference, and a higher divergence in the spike protein of SARS-CoV-2 at the early stage of this pandemic may be a good indicator that could show the pathway of SARS-CoV-2 transmitting from the natural reservoir to human beings.\n",
      "COVID-19, since emerged in Wuhan, China, has been a major concern due to its high infection rate, leaving more than one million infected people around the world. Huge number of studies tried to reveal the structure of the SARS-CoV-2 compared to the SARS-CoV-1, in order to suppress this high infection rate. Some of these studies showed that the mutations in the SARS-CoV-1 Spike protein might be responsible for its higher affinity to the ACE2 human cell receptor. In this work, we used molecular dynamics simulations and Monte Carlo sampling to compare the binding affinities of the spike proteins of SARS-CoV and SARS-CoV-2 to the ACE2. We found that the SARS-CoV-2 binds to ACE2 stronger than SARS-CoV by 7 kcal/mol, due to enhanced electrostatic interactions. The major contributions to the electrostatic binding energies are resulting from the salt-bridges formed between R426 and ACE2-E329 in case of SARS-CoV and K417 and ACE2-D30 for SARS-CoV2. In addition, there is no significant contribution from a single mutant to the binding energies. However, these mutations induce sophisticated structural changes that enhance the binding energies. Our results also indicate that the SARS-CoV-2 is unlikely a lab engineered virus.\n",
      "A novel severe acute respiratory syndrome (SARS)-like coronavirus (SARS-CoV-2) recently emerged and is rapidly spreading in humans, causing COVID-191,2. A key to tackling this pandemic is to understand the receptor recognition mechanism of the virus, which regulates its infectivity, pathogenesis and host range. SARS-CoV-2 and SARS-CoV recognize the same receptor-angiotensin-converting enzyme 2 (ACE2)-in humans3,4. Here we determined the crystal structure of the receptor-binding domain (RBD) of the spike protein of SARS-CoV-2 (engineered to facilitate crystallization) in complex with ACE2. In comparison with the SARS-CoV RBD, an ACE2-binding ridge in SARS-CoV-2 RBD has a more compact conformation; moreover, several residue changes in the SARS-CoV-2 RBD stabilize two virus-binding hotspots at the RBD-ACE2 interface. These structural features of SARS-CoV-2 RBD increase its ACE2-binding affinity. Additionally, we show that RaTG13, a bat coronavirus that is closely related to SARS-CoV-2, also uses human ACE2 as its receptor. The differences among SARS-CoV-2, SARS-CoV and RaTG13 in ACE2 recognition shed light on the potential animal-to-human transmission of SARS-CoV-2. This study provides guidance for intervention strategies that target receptor recognition by SARS-CoV-2.\n",
      "The ongoing outbreak of a new coronavirus (2019-nCoV, or severe acute respiratory syndrome coronavirus 2 [SARS-CoV-2]) has caused an epidemic of the acute respiratory syndrome known as coronavirus disease (COVID-19) in humans. SARS-CoV-2 rapidly spread to multiple regions of China and multiple other countries, posing a serious threat to public health. The spike (S) proteins of SARS-CoV-1 and SARS-CoV-2 may use the same host cellular receptor, angiotensin-converting enzyme 2 (ACE2), for entering host cells. The affinity between ACE2 and the SARS-CoV-2 S protein is much higher than that of ACE2 binding to the SARS-CoV S protein, explaining why SARS-CoV-2 seems to be more readily transmitted from human to human. Here, we report that ACE2 can be significantly upregulated after infection of various viruses, including SARS-CoV-1 and SARS-CoV-2, or by the stimulation with inflammatory cytokines such as interferons. We propose that SARS-CoV-2 may positively induce its cellular entry receptor, ACE2, to accelerate its replication and spread; high inflammatory cytokine levels increase ACE2 expression and act as high-risk factors for developing COVID-19, and the infection of other viruses may increase the risk of SARS-CoV-2 infection. Therefore, drugs targeting ACE2 may be developed for the future emerging infectious diseases caused by this cluster of coronaviruses.\n",
      "The ongoing outbreak of a new coronavirus (2019‐nCoV, or SARS‐CoV‐2) has caused an epidemic of the acute respiratory syndrome known as COVID‐19 in humans. SARS‐CoV‐2 rapidly spread to multiple regions of China and multiple other countries, posing a serious threat to public health. The spike (S) proteins of SARS‐CoV‐1 and SARS‐CoV‐2 may use the same host cellular receptor, angiotensin‐converting enzyme 2 (ACE2), for entering into host cells. The affinity between ACE2 and the SARS‐CoV‐2 S protein is much higher than that of ACE2 binding to the SARS‐CoV S protein, explaining why SARS‐CoV‐2 seems to be more readily transmitted from the human to human. Here, we report that ACE2 can be significantly upregulated after infection of various viruses, including SARS‐CoV‐1 and SARS‐CoV‐2, or by the stimulation with inflammatory cytokines such as interferons. We propose that SARS‐CoV‐2 may positively induce its cellular entry receptor, ACE2, to accelerate its replication and spread; high inflammatory cytokine levels increase ACE2 expression and act as high‐risk factors for developing COVID‐19, and the infection of other viruses may increase the risk of SARS‐CoV‐2 infection. Therefore, drugs targeting ACE2 may be developed for the future emerging infectious diseases caused by this cluster of coronaviruses. This article is protected by copyright. All rights reserved.\n",
      "[Image: see text] SARS-CoV-2, since emerging in Wuhan, China, has been a major concern because of its high infection rate and has left more than six million infected people around the world. Many studies endeavored to reveal the structure of the SARS-CoV-2 compared to the SARS-CoV, in order to find solutions to suppress this high infection rate. Some of these studies showed that the mutations in the SARS-CoV spike (S) protein might be responsible for its higher affinity to the ACE2 human cell receptor. In this work, we used molecular dynamics simulations and Monte Carlo sampling to compare the binding affinities of the S proteins of SARS-CoV and SARS-CoV-2 to the ACE2. Our results show that the protein surface of the ACE2 at the receptor binding domain (RBD) exhibits negative electrostatic potential, while a positive potential is observed for the S proteins of SARS-CoV/SARS-CoV-2. In addition, the binding energies at the interface are slightly higher for SARS-CoV-2 because of enhanced electrostatic interactions. The major contributions to the electrostatic binding energies result from the salt bridges forming between R426 and ACE-2-E329 in the case of SARS-CoV and K417 and ACE2-D30 in the SARS-CoV-2. In addition, our results indicate that the enhancement in the binding energy is not due to a single mutant but rather because of the sophisticated structural changes induced by all these mutations together. This finding suggests that it is implausible for the SARS-CoV-2 to be a lab-engineered virus.\n",
      "SARS-CoV-2, since emerging in Wuhan, China, has been a major concern because of its high infection rate and has left more than six million infected people around the world. Many studies endeavored to reveal the structure of the SARS-CoV-2 compared to the SARS-CoV, in order to find solutions to suppress this high infection rate. Some of these studies showed that the mutations in the SARS-CoV spike (S) protein might be responsible for its higher affinity to the ACE2 human cell receptor. In this work, we used molecular dynamics simulations and Monte Carlo sampling to compare the binding affinities of the S proteins of SARS-CoV and SARS-CoV-2 to the ACE2. Our results show that the protein surface of the ACE2 at the receptor binding domain (RBD) exhibits negative electrostatic potential, while a positive potential is observed for the S proteins of SARS-CoV/SARS-CoV-2. In addition, the binding energies at the interface are slightly higher for SARS-CoV-2 because of enhanced electrostatic interactions. The major contributions to the electrostatic binding energies result from the salt bridges forming between R426 and ACE-2-E329 in the case of SARS-CoV and K417 and ACE2-D30 in the SARS-CoV-2. In addition, our results indicate that the enhancement in the binding energy is not due to a single mutant but rather because of the sophisticated structural changes induced by all these mutations together. This finding suggests that it is implausible for the SARS-CoV-2 to be a lab-engineered virus.\n",
      "Coronavirus disease 2019 (COVID-19) is a pandemic infectious disease caused by novel severe acute respiratory syndrome coronavirus-2 (SARS CoV-2). The SARS CoV-2 is transmitted more rapidly and readily than SARS CoV. Both, SARS CoV and SARS CoV-2 via their glycosylated spike proteins recognize the human angiotensin converting enzyme-2 (ACE-2) receptor. We generated multiple sequence alignments and phylogenetic trees for representative spike proteins of SARS CoV and SARS CoV-2 from various host sources in order to analyze the specificity in SARS CoV-2 spike proteins required for causing infection in humans. Our results show that among the genomes analyzed, two sequence regions in the N-terminal domain \"MESEFR\" and \"SYLTPG\" are specific to human SARS CoV-2. In the receptor-binding domain, two sequence regions \"VGGNY\" and \"EIYQAGSTPCNGV\" and a disulfide bridge connecting 480C and 488C in the extended loop are structural determinants for the recognition of human ACE-2 receptor. The complete genome analysis of representative SARS CoVs from bat, civet, human host sources, and human SARS CoV-2 identified the bat genome (GenBank code: MN996532.1) as closest to the recent novel human SARS CoV-2 genomes. The bat SARS CoV genomes (GenBank codes: MG772933 and MG772934) are evolutionary intermediates in the mutagenesis progression toward becoming human SARS CoV-2.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: m95bmi9t\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what types of rapid testing for Covid-19 have been developed?\n",
      "are there serological tests that detect antibodies to coronavirus?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Rapid diagnostics, vaccines and therapeutics are important interventions for the management of the 2019 novel coronavirus (2019-nCoV) outbreak. It is timely to systematically review the potential of these interventions, including those for Middle East respiratory syndrome-Coronavirus (MERS-CoV) and severe acute respiratory syndrome (SARS)-CoV, to guide policymakers globally on their prioritization of resources for research and development. A systematic search was carried out in three major electronic databases (PubMed, Embase and Cochrane Library) to identify published studies in accordance with the Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) guidelines. Supplementary strategies through Google Search and personal communications were used. A total of 27 studies fulfilled the criteria for review. Several laboratory protocols for confirmation of suspected 2019-nCoV cases using real-time reverse transcription polymerase chain reaction (RT-PCR) have been published. A commercial RT-PCR kit developed by the Beijing Genomic Institute is currently widely used in China and likely in Asia. However, serological assays as well as point-of-care testing kits have not been developed but are likely in the near future. Several vaccine candidates are in the pipeline. The likely earliest Phase 1 vaccine trial is a synthetic DNA-based candidate. A number of novel compounds as well as therapeutics licensed for other conditions appear to have in vitro efficacy against the 2019-nCoV. Some are being tested in clinical trials against MERS-CoV and SARS-CoV, while others have been listed for clinical trials against 2019-nCoV. However, there are currently no effective specific antivirals or drug combinations supported by high-level evidence.\n",
      "Background The purpose of the current systematic review is to evaluate the efficacy of antiviral therapies in treatment of COVID-19. In addition, clinical trials on the efficacy of antiviral therapies in the management of Severe Acute Respiratory Syndrome coronavirus (SARS-Cov) or Middle East Respiratory Syndrome coronavirus (MERS-CoV) have also been reviewed, in order to identify potential treatment options for COVID-19. Method An extensive search was performed in Medline, Embase, Scopus, Web of Science and CENTRAL databases until the end of March 15, 2020. Two independent researchers performed the screening, and finally the related studies were included. Results Only one clinical trial on the efficacy of antiviral therapy in management of COVID-19 was found. The results depicted that adding Lopinavir-Ritonavir to the standard treatment regimen of patients with severe COVID-19 has no benefits. Moreover, 21 case-series and case-report studies reported the prescription of antiviral agents in COVID-19, none of which can be used to determine the efficacy of antiviral therapies in confronting COVID-19. In addition, no clinical trials were found to be performed on the efficacy of antiviral agents in the management of SARS-CoV and MERS-CoV. Conclusion The current evidence impede researchers from proposing an appropriate antiviral therapy against COVID-19, making the current situation a serious concern for international organizations such as World Health Organization (WHO). In the time of the current pandemic and future epidemics, organizations such as WHO should pursue more proactive actions and plan well-designed clinical trials so that their results can be used in managing future epidemics.\n",
      "Background: The purpose of the current systematic review is to evaluate the efficacy of antiviral therapies in treatment of COVID-19 In addition, clinical trials on the efficacy of antiviral therapies in the management of Severe Acute Respiratory Syndrome coronavirus (SARS-Cov) or Middle East Respiratory Syndrome coronavirus (MERS-CoV) have also been reviewed, in order to identify potential treatment options for COVID-19 Method: An extensive search was performed in Medline, Embase, Scopus, Web of Science and CENTRAL databases until the end of March 15, 2020 Two independent researchers performed the screening, and finally the related studies were included Results: Only one clinical trial on the efficacy of antiviral therapy in management of COVID-19 was found The results depicted that adding Lopinavir-Ritonavir to the standard treatment regimen of patients with severe COVID-19 has no benefits Moreover, 21 case-series and case-report studies reported the prescription of antiviral agents in COVID-19, none of which can be used to determine the efficacy of antiviral therapies in confronting COVID-19 In addition, no clinical trials were found to be performed on the efficacy of antiviral agents in the management of SARS-CoV and MERS-CoV Conclusion: The current evidence impede researchers from proposing an appropriate antiviral therapy against COVID-19, making the current situation a serious concern for international organizations such as World Health Organization (WHO) In the time of the current pandemic and future epidemics, organizations such as WHO should pursue more proactive actions and plan well-designed clinical trials so that their results can be used in managing future epidemics\n",
      "BACKGROUND: The purpose of the current systematic review is to evaluate the efficacy of antiviral therapies in treatment of COVID-19. In addition, clinical trials on the efficacy of antiviral therapies in the management of Severe Acute Respiratory Syndrome coronavirus (SARS-Cov) or Middle East Respiratory Syndrome coronavirus (MERS-CoV) have also been reviewed, in order to identify potential treatment options for COVID-19. METHOD: An extensive search was performed in Medline, Embase, Scopus, Web of Science and CENTRAL databases until the end of March 15, 2020. Two independent researchers performed the screening, and finally the related studies were included. RESULTS: Only one clinical trial on the efficacy of antiviral therapy in management of COVID-19 was found. The results depicted that adding Lopinavir-Ritonavir to the standard treatment regimen of patients with severe COVID-19 has no benefits. Moreover, 21 case-series and case-report studies reported the prescription of antiviral agents in COVID-19, none of which can be used to determine the efficacy of antiviral therapies in confronting COVID-19. In addition, no clinical trials were found to be performed on the efficacy of antiviral agents in the management of SARS-CoV and MERS-CoV. CONCLUSION: The current evidence impede researchers from proposing an appropriate antiviral therapy against COVID-19, making the current situation a serious concern for international organizations such as World Health Organization (WHO). In the time of the current pandemic and future epidemics, organizations such as WHO should pursue more proactive actions and plan well-designed clinical trials so that their results can be used in managing future epidemics.\n",
      "Severe acute respiratory syndrome coronavirus 2 (SARS-Cov-2), initially termed 2019-new CoV (2019-nCoV), is a novel coronavirus responsible for the severe respiratory illness currently ongoing worldwide from the beginning of December 2019. This beta gene virus, very close to bat coronaviruses (bat-CoV-RaTG13) and bat-SL-CoVZC45, causes a severe disease, similar to those caused by Middle East respiratory syndrome (MERS)-CoV and SARS-CoV viruses, featured by low to moderate mortality rate. Unfortunately, the antiviral drugs commonly used in clinical practice to treat viral infections, are not applicable to SARS-Cov-2 and no vaccine is available. Thus, it is extremely necessary to identify new drugs suitable for the treatment of the 2019-nCoV outbreak. Different preclinical studies conducted on other coronaviruses suggested that promising clinical outcomes for 2019-nCoV should be obtained by using alpha-interferon, chloroquine phosphate, arabinol, remdesivir, lopinavir/ritonavir, and anti-inflammatory drugs. Moreover, clinical trials with these suitable drugs should be performed on patients affected by SARS-Cov-2 to prove their efficacy and safety. Finally, a very promising therapeutic drug, tocilizumab, is discussed; it is currently used to treat patients presenting COVID-19 pneumonia. Herein, we recapitulate these experimental studies to highlight the use of antiviral drugs for the treatment of SARS-Cov-2 disease.\n",
      "Severe acute respiratory syndrome coronavirus 2 (SARS-Cov-2), initially termed 2019-new CoV (2019-nCoV), is a novel coronavirus responsible for the severe respiratory illness currently ongoing worldwide from the beginning of December 2019. This beta gene virus, very close to bat coronaviruses (bat-CoV-RaTG13) and bat-SL-CoVZC45, causes a severe disease, similar to those caused by Middle East respiratory syndrome (MERS)-CoV and SARS-CoV viruses, featured by low to moderate mortality rate. Unfortunately, the antiviral drugs commonly used in clinical practice to treat viral infections, are not applicable to SARS-Cov-2 and no vaccine is available. Thus, it is extremely necessary to identify new drugs suitable for the treatment of the 2019-nCoV outbreak. Different preclinical studies conducted on other coronaviruses suggested that promising clinical outcomes for 2019-nCoV should be obtained by using alpha-interferon, chloroquine phosphate, arabinol, remdesivir, lopinavir/ritonavir, and anti-inflammatory drugs. Moreover, clinical trials with these suitable drugs should be performed on patients affected by SARS-Cov-2 to prove their efficacy and safety. Finally, a very promising therapeutic drug, tocilizumab, is discussed; it is currently used to treat patients presenting COVID-19 pneumonia. Herein, we recapitulate these experimental studies to highlight the use of antiviral drugs for the treatment of SARS-Cov-2 disease.\n",
      "Recently, severe acute respiratory syndrome coronavirus 2 (SARS-COV-2), commonly known as coronavirus disease-2019 (COVID-19) has rapidly spread across China and around the world. By the declaration of WHO, COVID-19 outbreak considered as a public health problem of international concern. The aim of this study is to provide a comprehensive view on COVID-19 and the future expectations to control virus progression. Patients with liver disease, diabetes, high blood pressure, and obesity are more susceptible to the incidence of COVID-19 infection. So, there is a rapid need for disease diagnosis, vaccine development, and drug discovery to detect, prevent, and treat this sudden and lethal virus. Real-time polymerase chain reaction (RT-PCR) is considered as a rapid, accurate, and specific tool for disease diagnosis. Under this emergency situation that the world facing against COVID-19, there are about 15 potential vaccine candidates tested globally based on messenger RNA, DNA-based, nanoparticle, synthetic, and modified virus-like particle. Certain drugs that are clinically approved for other diseases were tested against COVID-19 as chloroquine, hydroxychloroquine, ivermectin, favipiravir, ribavirin, and remdesivir. Convalescent plasma transfusion and traditional herbal medicine were also taken into consideration. Due to the absence of effective treatment or vaccines against COVID-19 so far, the precautionary measures according to WHO’s strategic objectives are the only way to confront this crisis. Governments should adopt national medical care programs to reduce the risk of exposure to any future viral outbreaks especially to patients with pre-existing medical conditions.\n",
      "Despite to outbreaks of highly pathogenic beta and alpha coronaviruses including severe acute respiratory syndrome coronavirus (SARS-CoV), Middle East respiratory syndrome coronavirus (MERS-CoV), and human coronavirus, the newly emerged 2019 coronavirus (COVID-19) is considered as a lethal zoonotic virus due to its deadly respiratory syndrome and high mortality rate among the human. Globally, more than 3,517,345 cases have been confirmed with 243,401 deaths due to Acute Respiratory Distress Syndrome (ARDS) caused by COVID-19. The antiviral drug discovery activity is required to control the persistence of COVID-19 circulation and the potential of the future emergence of coronavirus. However, the present review aims to highlight the important antiviral approaches, including interferons, ribavirin, mycophenolic acids, ritonavir, lopinavir, inhibitors, and monoclonal antibodies (mAbs) to provoke the nonstructural proteins and deactivate the structural and essential host elements of the virus to control and treat the infection of COVID-19 by inhibiting the viral entry, viral RNA replication and suppressing the viral protein expression. Moreover, the present review investigates the epidemiology, diagnosis, structure, and replication of COVID-19 for better understanding. It is recommended that these proteases, inhibitors, and antibodies could be a good therapeutic option in drug discovery to control the newly emerged coronavirus. HIGHLIGHTS: COVID-19 has more than 79.5% identical sequence to SARS-CoV and a 96% identical sequence of the whole genome of bat coronaviruses. Acute respiratory distress syndrome (ARDS), renal failure, and septic shock are the possible clinical symptoms associated with COVID-19. Different antivirals, including interferons, ribavirin, lopinavir, and monoclonal antibodies (mAbs) could be the potent therapeutic agents against COVID-19. The initial clinical trials on hydroquinone in combination with azithromycin showed an admirable result in the reduction of COVID-19. The overexpression of inflammation response, cytokine dysregulation, and induction of apoptosis could be an well-organized factors to reduce the pathogenicity of COVID-19.\n",
      "In the 21st century, human civilization has witnessed three major epidemics caused by Coronaviruses namely severe acute respiratory syndrome coronavirus (SARS CoV) in 2003, Middle East respiratory syndrome coronavirus (MERS CoV) in 2012 and 2019 novel coronavirus (2019 nCoV) or coronavirus disease (COVID 19) in 2019. Among these, COVID-19 has greater transmission and mortality rate. 2019 nCoV belongs to a large family of positive sense single-stranded RNA viruses (+ssRNA) that can be isolated in different animal species. The most communal symptoms of COVID-19 include fever, cough, and shortness of breath during the incubation period (2-14 days) of infection. COVID-19 transmission is occurring from infected humans to close contact with one another through respiratory droplets, coughs, and sneezes of infected person. Moreover, the virus containing surfaces may also transmit the infection. Diagnosis is being carried out by collecting a nasopharyngeal swab or sputum specimen for detection of SARS-CoV-2 RNA by reverse-transcription polymerase chain reaction (RT-PCR). Rapid diagnosing methods are also under development which can diagnose COVID 19 in few minutes to hours. Currently, there is no specific cure or preventive therapeutics available. Hence, based upon limited in-vitro and anecdotal data, Chloroquine, or Hydroxychloroquine, Remdesivir, Lopinavir and Ritonavir are being employed in the management. Search for new specific anti-viral drugs from natural/synthetic origins is under full swing and many of them are currently used as chemotherapeutic drugs under clinical investigation. Yet, there is a strong need for development of vaccine, which may take several months to few years for the development.\n",
      "In the 21st century, human civilization has witnessed three major epidemics caused by Coronaviruses namely severe acute respiratory syndrome coronavirus (SARS CoV) in 2003, Middle East respiratory syndrome coronavirus (MERS CoV) in 2012 and 2019 novel coronavirus (2019 nCoV) or coronavirus disease (COVID 19) in 2019. Among these, COVID-19 has greater transmission and mortality rate. 2019 nCoV belongs to a large family of positive sense single-stranded RNA viruses (+ssRNA) that can be isolated in different animal species. The most communal symptoms of COVID-19 include fever, cough, and shortness of breath during the incubation period (2-14 days) of infection. COVID-19 transmission is occurring from infected humans to close contact with one another through respiratory droplets, coughs, and sneezes of infected person. Moreover, the virus containing surfaces may also transmit the infection. Diagnosis is being carried out by collecting a nasopharyngeal swab or sputum specimen for detection of SARS-CoV-2 RNA by reverse-transcription polymerase chain reaction (RT-PCR). Rapid diagnosing methods are also under development which can diagnose COVID 19 in few minutes to hours. Currently, there is no specific cure or preventive therapeutics available. Hence, based upon limited in-vitro and anecdotal data, Chloroquine, or Hydroxychloroquine, Remdesivir, Lopinavir and Ritonavir are being employed in the management. Search for new specific anti-viral drugs from natural/synthetic origins is under full swing and many of them are currently used as chemotherapeutic drugs under clinical investigation. Yet, there is a strong need for development of vaccine, which may take several months to few years for the development.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: a47onmje\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the transmission routes of coronavirus?\n",
      "how long can the coronavirus live outside the body\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "It is a widely accepted view that COVID 19 is either transmitted via surface contamination or via close contact of an un-infected person with an infected person. Surface contamination usually happens when infected water droplets from exhalation/sneeze/cough of COVID sick person settle on nearby surfaces. To curb this, social distancing and good hand hygiene advise is advocated by World health Organization (WHO). We argue that COVID 19 coronovirus can also be airborne in a puff cloud loaded with infected droplets generated by COVID sick person. An elementary calculation shows that a $5~\\mu m$ respiratory infected droplet can remain suspended for about 9.0 minutes and a $2~\\mu m$ droplet can remain suspended for about an hour! And social distancing advise of 3 feet by WHO and 6 feet by CDC (Centers for Disease Control and Prevention) may not be sufficient in some circumstances as discussed in the text.\n",
      "Hand washing and maintaining social distance are the main measures recommended by the World Health Organization (WHO) to avoid contracting COVID-19. Unfortunately, these measured do not prevent infection by inhalation of small droplets exhaled by an infected person that can travel distance of meters or tens of meters in the air and carry their viral content. Science explains the mechanisms of such transport and there is evidence that this is a significant route of infection in indoor environments. Despite this, no countries or authorities consider airborne spread of COVID-19 in their regulations to prevent infections transmission indoors. It is therefore extremely important, that the national authorities acknowledge the reality that the virus spreads through air, and recommend that adequate control measures be implemented to prevent further spread of the SARS-CoV-2 virus, in particularly removal of the virus-laden droplets from indoor air by ventilation.\n",
      "Today, the whole world is fighting a public health emergency called ‘COVID-19’ caused by a new infectious virus called SARS-CoV2. Any person can catch COVID-19 from an infected person via aerosol droplets when the person coughs, sneezes, or speaks. To limit such a transmission, World Health Organization (WHO) has recommended people to wear masks and physically distance themselves by staying at least 1 m (3 feet) away from others. As aerosol droplets (by cough or sneeze) land on objects and surfaces around the person such as tables, doorknobs and handrails, and remain active on these surfaces for hours to days, people are advised to use soaps for at least 20 s. and alcohol-based sanitizers as well. As the public made efforts, clinicians and researchers investigated and found that drugs which were initially used to treat other diseases may work as a treatment option for COVID-19. One of those drugs was Chloroquine and its related derivative called hydroxychloroquine. In this review article, we have systematically searched for details of COVID-19 pandemic till May 2020 and assembled few data pertaining to (i) Corona viruses; (ii) SARS-CoV2, the virus that causes COVID-19’ and (iii) How chloroquine and hydroxychloroquine mediates anti-viral effect in both prophylactic and therapeutic setting. These data have been acquired mostly from PubMed and websites of WHO and Indian Council for Medical Research (ICMR). We did a systematic search and found that the properties of chloroquine are very much essential for the COVID-19 scenario. We also bring to you some evidence that the anti-lysosomal activity of chloroquine may be increased by botanicals like betulinic acid.\n",
      "Abstract Hand washing and maintaining social distance are the main measures recommended by the World Health Organization (WHO) to avoid contracting COVID-19. Unfortunately, these measured do not prevent infection by inhalation of small droplets exhaled by an infected person that can travel distance of meters or tens of meters in the air and carry their viral content. Science explains the mechanisms of such transport and there is evidence that this is a significant route of infection in indoor environments. Despite this, no countries or authorities consider airborne spread of COVID-19 in their regulations to prevent infections transmission indoors. It is therefore extremely important, that the national authorities acknowledge the reality that the virus spreads through air, and recommend that adequate control measures be implemented to prevent further spread of the SARS-CoV-2 virus, in particularly removal of the virus-laden droplets from indoor air by ventilation.\n",
      "Cases of COVID-19 have been reported in over 200 countries. Thousands of health workers have been infected and outbreaks have occurred in hospitals, aged care facilities and prisons. World Health Organization (WHO) has issued guidelines for contact and droplet precautions for Healthcare Workers (HCWs) caring for suspected COVID-19 patients, whilst the US Centre for Disease Control (CDC) has recommended airborne precautions. The 1 - 2 m (&#8776;3 - 6 ft) rule of spatial separation is central to droplet precautions and assumes large droplets do not travel further than 2 m (&#8776;6 ft). We aimed to review the evidence for horizontal distance travelled by droplets and the guidelines issued by the World Health Organization (WHO), US Center for Diseases Control (CDC) and European Centre for Disease Prevention and Control (ECDC) on respiratory protection for COVID-19. We found that the evidence base for current guidelines is sparse, and the available data do not support the 1 - 2 m (&#8776;3 - 6 ft) rule of spatial separation. Of ten studies on horizontal droplet distance, eight showed droplets travel more than 2 m (&#8776;6 ft), in some cases more than 8 meters (&#8776;26 ft). Several studies of SARS-CoV-2 support aerosol transmission and one study documented virus at a distance of 4 meters (&#8776;13 ft) from the patient. Moreover, evidence suggests infections cannot neatly be separated into the dichotomy of droplet versus airborne transmission routes. Available studies also show that SARS-CoV-2 can be detected in the air, 3 hours after aeroslisation. The weight of combined evidence supports airborne precautions for the occupational health and safety of health workers treating patients with COVID-19.\n",
      "• COVID-19 virus is primarily transmitted between humans through respiratory droplets and contact routes. • Transportation of COVID-19 patient carries very high transmission risk to personals involved. • Worldwide, clinicians are using many measures that can be executed easily to prevent aerosol exposure.\n",
      "Cases of COVID-19 have been reported in over 200 countries. Thousands of health workers have been infected and outbreaks have occurred in hospitals, aged care facilities and prisons. World Health Organization (WHO) has issued guidelines for contact and droplet precautions for Healthcare Workers (HCWs) caring for suspected COVID-19 patients, whilst the US Centre for Disease Control (CDC) has recommended airborne precautions. The 1 – 2 m (≈3 – 6 ft) rule of spatial separation is central to droplet precautions and assumes large droplets do not travel further than 2 m (≈6 ft). We aimed to review the evidence for horizontal distance travelled by droplets and the guidelines issued by the World Health Organization (WHO), US Center for Diseases Control (CDC) and European Centre for Disease Prevention and Control (ECDC) on respiratory protection for COVID-19. We found that the evidence base for current guidelines is sparse, and the available data do not support the 1 – 2 m (≈3 – 6 ft) rule of spatial separation. Of ten studies on horizontal droplet distance, eight showed droplets travel more than 2 m (≈6 ft), in some cases more than 8 meters (≈26 ft). Several studies of SARS-CoV-2 support aerosol transmission and one study documented virus at a distance of 4 meters (≈13 ft) from the patient. Moreover, evidence suggests infections cannot neatly be separated into the dichotomy of droplet versus airborne transmission routes. Available studies also show that SARS-CoV-2 can be detected in the air, 3 hours after aeroslisation. The weight of combined evidence supports airborne precautions for the occupational health and safety of health workers treating patients with COVID-19.\n",
      "World Health Organization has suggested that severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is transmitted through person-to-person transmission and contact with contaminated surfaces. However, rapid spread of the coronavirus disease 2019 (COVID-19) suggests other routes such as airborne transmission may be involved. A few research studies have been conducted to evaluate the potential transmission of this virus through air. Although some studies have found no evidence of airborne transmission, other more recent work is proving the presence of SARS-CoV-2 even in public places. Also, the past experiences and knowledge about the mechanisms of similar viruses such as SARS-CoV support this hypothesis. It seems that the best decision at the moment is to follow a conservative approach, and accept the hypothesis that SARS-CoV-2 is able to be transmitted through air. By this, control measures could be employed to prevent further COVID-19 infection.\n",
      "Health threat from SARS-CoV-2 airborne infection has become a public emergency of international concern. During the ongoing coronavirus pandemic, people have been advised by the Centers for Disease Control and Prevention to maintain social distancing of at least 2 m to limit the risk of exposure to the coronavirus. We carry out a physics modeling study for SARS-CoV-2 transport in air. We show that if aerosols and droplets follow semi-ballistic emission trajectories, then their horizontal range is proportional to the particle's diameter. For standard ambient temperature and pressure conditions, the horizontal range of these aerosols remains safely below 2 m. We also show that aerosols and droplets can remain suspended for hours in the air, providing a health threat of airborne infection. The latter argues in favor of implementing additional precautions to the recommended 2~m social distancing, e.g. wearing a face mask when we are out in public.\n",
      "The spread of COVID19 through droplets ejected by infected individuals in the air during sneezing and coughing has been considered as a matter of key concern. Therefore, a quantitative understanding of the propagation of droplets containing the virus assumes immense importance in formulating the preventive strategies. In the present work we rigorously investigate the evolution of droplets in space and time under varying external conditions of temperature, humidity, air flow by using laws of statistical mechanics. The effects of drag, diffusion and the gravity on the droplets of different sizes and ejection velocities have been taken into consideration during its motion in the air. In still air it is found that bigger droplets traverse larger distance but the smaller droplets remain suspended in the air for longer time. Therefore, in still air the horizontal distance that a healthy individual should maintain from an infected one is determined by the bigger droplets but the time interval to be maintained between the two is determined by the smaller droplets. Here we show that in places with flowing air the lighter droplets travel larger distance and have the ability to remain suspended in the air for longer time. Therefore, we conclude that both temporal as well as the geometric distance that a healthy individual should maintain from an infected one is determined by the smaller droplets if there is flowing air. The small droplets travel larger distance and remain suspended in the air for long time under the influence of flow making the use of mask mandatory to prevent the virus. The the maintenance of only stationary separation between healthy and infected individual is not substantiated. The quantitative results obtained here will be useful to devise strategies for preventing also the spread of other types of droplets containing microorganisms.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: f7gr98eb\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the transmission routes of coronavirus?\n",
      "how long does coronavirus remain stable  on surfaces?\n",
      "how long can the coronavirus live outside the body\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Abstract COVID-19 is the name of the disease supposedly manifested in December 2019 from Wuhan, because of virus named as SARS-CoV-2. Now this disease has spread to almost all other parts of the world. COVID-19 pandemic has various reasons for its dramatic worldwide increase. Here, we have studied Coronavirus sustainability on various surfaces. Various disinfectants and their roles are discussed from the available literature. The infection capabilities of SARS-CoV-1 and SARS-CoV-2 for different materials are discussed and finally studies infection decay for SARS-CoV-1 and SARS-CoV-2.\n",
      "COVID-19 is the name of the disease supposedly manifested in December 2019 from Wuhan, because of virus named as SARS-CoV-2. Now this disease has spread to almost all other parts of the world. COVID-19 pandemic has various reasons for its dramatic worldwide increase. Here, we have studied Coronavirus sustainability on various surfaces. Various disinfectants and their roles are discussed from the available literature. The infection capabilities of SARS-CoV-1 and SARS-CoV-2 for different materials are discussed and finally studies infection decay for SARS-CoV-1 and SARS-CoV-2.\n",
      "BACKGROUND: In late 2019, a novel human coronavirus, SARS-CoV-2, emerged in Wuhan, China. This virus has caused a global pandemic involving more than 200 countries. SARS-CoV-2 is highly adapted to humans and readily transmits from person-to-person. AIM: The aim of this study was to investigate the infectivity of SARS-CoV-2 under various environmental factors, disinfectants and different pH conditions. The efficacy of a variety of laboratory virus inactivation methods and home disinfectants against SARS-CoV-2 were investigated. METHODS: The residual virus in dried form or in solution was titrated on Vero E6 cell line at day 0, 1, 3, 5, and 7 after incubation at different temperatures. The viability of virus was determined after treatment with different disinfectants and pH solutions at room temperature (20â¼25oC). FINDINGS: SARS-CoV-2 was able to retain viability for 3-5 days in dried form or 7 days in solution at room temperature. SARS-CoV-2 could be detected under a wide range of pH conditions from pH4 to pH11 for several days and 1 to 2 days in stool at room temperature but lost 5 logs of infectivity. A variety of commonly used disinfectants and laboratory inactivation procedures were found to reduce viral viability effectively. CONCLUSION: This study demonstrates the stability of SARS-CoV-2 on environmental surfaces and raises the possibility of faecal-oral transmission. Commonly used fixatives, nucleic acid extraction methods and heat inactivation were found to significantly reduce viral infectivity that could ensure hospital and laboratory safety during the COVID-19 pandemic.\n",
      "BACKGROUND: In late 2019, a novel human coronavirus, SARS-CoV-2, emerged in Wuhan, China. This virus has caused a global pandemic involving more than 200 countries. SARS-CoV-2 is highly adapted to humans and readily transmits from person-to-person. AIM: The aim of this study was to investigate the infectivity of SARS-CoV-2 under various environmental factors, disinfectants and different pH conditions. The efficacy of a variety of laboratory virus inactivation methods and home disinfectants against SARS-CoV-2 were investigated. METHODS: The residual virus in dried form or in solution was titrated on Vero E6 cell line at day 0, 1, 3, 5, and 7 after incubation at different temperatures. The viability of virus was determined after treatment with different disinfectants and pH solutions at room temperature (20∼25(o)C). FINDINGS: SARS-CoV-2 was able to retain viability for 3-5 days in dried form or 7 days in solution at room temperature. SARS-CoV-2 could be detected under a wide range of pH conditions from pH4 to pH11 for several days and 1 to 2 days in stool at room temperature but lost 5 logs of infectivity. A variety of commonly used disinfectants and laboratory inactivation procedures were found to reduce viral viability effectively. CONCLUSION: This study demonstrates the stability of SARS-CoV-2 on environmental surfaces and raises the possibility of faecal-oral transmission. Commonly used fixatives, nucleic acid extraction methods and heat inactivation were found to significantly reduce viral infectivity that could ensure hospital and laboratory safety during the COVID-19 pandemic.\n",
      "A novel human coronavirus, now named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2, referred to as HCoV-19 here) that emerged in Wuhan, China in late 2019 is now causing a pandemic. Here, we analyze the aerosol and surface stability of HCoV-19 and compare it with SARS-CoV-1, the most closely related human coronavirus.2 We evaluated the stability of HCoV-19 and SARS-CoV-1 in aerosols and on different surfaces and estimated their decay rates using a Bayesian regression model\n",
      "Summary The novel human coronavirus SARS-CoV-2 has become a global health concern causing severe respiratory tract infections in humans. Human-to-human transmissions have been described, probably via droplets but possibly also via contaminated hands or surfaces. In a recent review on the persistence of human and veterinary coronaviruses on inanimate surfaces it was shown that human coronaviruses such as Severe Acute Respiratory Syndrome (SARS) coronavirus, Middle East Respiratory Syndrome (MERS) coronavirus or endemic human coronaviruses (HCoV) can persist on inanimate surfaces like metal, glass or plastic for up to 9 days. Some disinfectant agents effectively reduce coronavirus infectivity within 1 minute such 62%–71% ethanol, 0.5% hydrogen peroxide or 0.1% sodium hypochlorite. Other compounds such as 0.05%–0.2% benzalkonium chloride or 0.02% chlorhexidine digluconate are less effective. An effective surface disinfection may help to ensure an early containment and prevention of further viral spread.\n",
      "Coronavirus disease 2019 (COVID-19) was first identified in China in late 2019 and is caused by newly identified severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Previous studies had reported the stability of SARS-CoV-2 in cell culture media and deposited onto surfaces under a limited set of environmental conditions. Here, we broadly investigated the effects of relative humidity, temperature, and droplet size on the stability of SARS-CoV-2 in a simulated clinically relevant matrix dried on nonporous surfaces. The results show that SARS-CoV-2 decayed more rapidly when either humidity or temperature was increased but that droplet volume (1 to 50 µl) and surface type (stainless steel, plastic, or nitrile glove) did not significantly impact decay rate. At room temperature (24°C), virus half-life ranged from 6.3 to 18.6 h depending on the relative humidity but was reduced to 1.0 to 8.9 h when the temperature was increased to 35°C. These findings suggest that a potential for fomite transmission may persist for hours to days in indoor environments and have implications for assessment of the risk posed by surface contamination in indoor environments.IMPORTANCE Mitigating the transmission of SARS-CoV-2 in clinical settings and public spaces is critically important to reduce the number of COVID-19 cases while effective vaccines and therapeutics are under development. SARS-CoV-2 transmission is thought to primarily occur through direct person-to-person transfer of infectious respiratory droplets or through aerosol-generating medical procedures. However, contact with contaminated surfaces may also play a significant role. In this context, understanding the factors contributing to SARS-CoV-2 persistence on surfaces will enable a more accurate estimation of the risk of contact transmission and inform mitigation strategies. To this end, we have developed a simple mathematical model that can be used to estimate virus decay on nonporous surfaces under a range of conditions and which may be utilized operationally to identify indoor environments in which the virus is most persistent.\n",
      "Coronavirus disease 2019 (COVID-19) was first identified in China in late 2019 and is caused by newly identified severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Previous studies had reported the stability of SARS-CoV-2 in cell culture media and deposited onto surfaces under a limited set of environmental conditions. Here, we broadly investigated the effects of relative humidity, temperature, and droplet size on the stability of SARS-CoV-2 in a simulated clinically relevant matrix dried on nonporous surfaces. The results show that SARS-CoV-2 decayed more rapidly when either humidity or temperature was increased but that droplet volume (1 to 50 μl) and surface type (stainless steel, plastic, or nitrile glove) did not significantly impact decay rate. At room temperature (24°C), virus half-life ranged from 6.3 to 18.6 h depending on the relative humidity but was reduced to 1.0 to 8.9 h when the temperature was increased to 35°C. These findings suggest that a potential for fomite transmission may persist for hours to days in indoor environments and have implications for assessment of the risk posed by surface contamination in indoor environments. IMPORTANCE Mitigating the transmission of SARS-CoV-2 in clinical settings and public spaces is critically important to reduce the number of COVID-19 cases while effective vaccines and therapeutics are under development. SARS-CoV-2 transmission is thought to primarily occur through direct person-to-person transfer of infectious respiratory droplets or through aerosol-generating medical procedures. However, contact with contaminated surfaces may also play a significant role. In this context, understanding the factors contributing to SARS-CoV-2 persistence on surfaces will enable a more accurate estimation of the risk of contact transmission and inform mitigation strategies. To this end, we have developed a simple mathematical model that can be used to estimate virus decay on nonporous surfaces under a range of conditions and which may be utilized operationally to identify indoor environments in which the virus is most persistent.\n",
      "Summary Currently, the emergence of a novel human coronavirus, SARS-CoV-2, has become a global health concern causing severe respiratory tract infections in humans. Human-to-human transmissions have been described with incubation times between 2-10 days, facilitating its spread via droplets, contaminated hands or surfaces. We therefore reviewed the literature on all available information about the persistence of human and veterinary coronaviruses on inanimate surfaces as well as inactivation strategies with biocidal agents used for chemical disinfection, e.g. in healthcare facilities. The analysis of 22 studies reveals that human coronaviruses such as Severe Acute Respiratory Syndrome (SARS) coronavirus, Middle East Respiratory Syndrome (MERS) coronavirus or endemic human coronaviruses (HCoV) can persist on inanimate surfaces like metal, glass or plastic for up to 9 days, but can be efficiently inactivated by surface disinfection procedures with 62–71% ethanol, 0.5% hydrogen peroxide or 0.1% sodium hypochlorite within 1 minute. Other biocidal agents such as 0.05–0.2% benzalkonium chloride or 0.02% chlorhexidine digluconate are less effective. As no specific therapies are available for SARS-CoV-2, early containment and prevention of further spread will be crucial to stop the ongoing outbreak and to control this novel infectious thread.\n",
      "Abstract With the outbreak of unknown pneumonia in Wuhan, China, in December 2019, a new coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), aroused the attention of the entire world. The current outbreak of infections with SARS-CoV-2 is termed Coronavirus Disease 2019 (COVID-19). The World Health Organization declared COVID-19 in China as a Public Health Emergency of International Concern. Two other coronavirus infections—SARS in 2002-2003 and Middle East Respiratory Syndrome (MERS) in 2012—both caused severe respiratory syndrome in humans. All 3 of these emerging infectious diseases leading to a global spread are caused by β-coronaviruses. Although coronaviruses usually infect the upper or lower respiratory tract, viral shedding in plasma or serum is common. Therefore, there is still a theoretical risk of transmission of coronaviruses through the transfusion of labile blood products. Because more and more asymptomatic infections are being found among COVID-19 cases, considerations of blood safety and coronaviruses have arisen especially in endemic areas. In this review, we detail current evidence and understanding of the transmission of SARS-CoV, MERS–CoV, and SARS-CoV-2 through blood products as of February 10, 2020, and also discuss pathogen inactivation methods on coronaviruses.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 7kw9lws0\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Abstract Background and aims COVID-19 is already a pandemic. Emerging data suggest an increased association and a heightened mortality in patients of COVID-19 with comorbidities. We aimed to evaluate the outcome in hypertensive patients with COVID-19 and its relation to the use of renin-angiotensin system blockers (RASB). Methods We have systematically searched the medical database up to March 27, 2020 and retrieved all the published articles in English language related to our topic using MeSH key words. Results From the pooled data of all ten available Chinese studies (n = 2209) that have reported the characteristics of comorbidities in patients with COVID-19, hypertension was present in nearly 21%, followed by diabetes in nearly 11%, and established cardiovascular disease (CVD) in approximately 7% of patients. Although the emerging data hints to an increase in mortality in COVID-19 patients with known hypertension, diabetes and CVD, it should be noted that it was not adjusted for multiple confounding factors. Harm or benefit in COVID-19 patients receiving RASB has not been typically assessed in these studies yet, although mechanistically and plausibly both, benefit and harm is possible with these agents, given that COVID-19 expresses to tissues through the receptor of angiotensin converting enzyme-2. Conclusion Special attention is definitely required in patients with COVID-19 with associated comorbidities including hypertension, diabetes and established CVD. Although the role of RASB has a mechanistic equipoise, patients with COVID-19 should not stop these drugs at this point of time, as recommended by various world organizations and without the advice of health care provider.\n",
      "BACKGROUND AND AIMS: COVID-19 is already a pandemic. Emerging data suggest an increased association and a heightened mortality in patients of COVID-19 with comorbidities. We aimed to evaluate the outcome in hypertensive patients with COVID-19 and its relation to the use of renin-angiotensin system blockers (RASB). METHODS: We have systematically searched the medical database up to March 27, 2020 and retrieved all the published articles in English language related to our topic using MeSH key words. RESULTS: From the pooled data of all ten available Chinese studies (n = 2209) that have reported the characteristics of comorbidities in patients with COVID-19, hypertension was present in nearly 21%, followed by diabetes in nearly 11%, and established cardiovascular disease (CVD) in approximately 7% of patients. Although the emerging data hints to an increase in mortality in COVID-19 patients with known hypertension, diabetes and CVD, it should be noted that it was not adjusted for multiple confounding factors. Harm or benefit in COVID-19 patients receiving RASB has not been typically assessed in these studies yet, although mechanistically and plausibly both, benefit and harm is possible with these agents, given that COVID-19 expresses to tissues through the receptor of angiotensin converting enzyme-2. CONCLUSION: Special attention is definitely required in patients with COVID-19 with associated comorbidities including hypertension, diabetes and established CVD. Although the role of RASB has a mechanistic equipoise, patients with COVID-19 should not stop these drugs at this point of time, as recommended by various world organizations and without the advice of health care provider.\n",
      "PURPOSE OF REVIEW: There is increasing evidence indicating an association between several risk factors and worse prognosis in patients with coronavirus disease 2019 (COVID-19), including older age, hypertension, heart failure, diabetes, and pulmonary disease. Hypertension is of particular interest because it is common in adults and there are concerns related to the use of renin-angiotensin system (RAS) inhibitors in patients with hypertension infected with COVID-19. Levels of angiotensin-converting enzyme 2 (ACE2), a protein that facilitates entry of coronavirus into cells, may increase in patients using RAS inhibitors. Thus, chronic use of RAS inhibition could potentially lead to a more severe and fatal form of COVID-19. In this review, we provide a critical review to the following questions: (1) Does hypertension influence immunity or ACE2 expression favoring viral infections? (2) Are the risks of complications in hypertension mediated by its treatment? (3) Is aging a major factor associated with worse prognosis in patients with COVID-19 and hypertension? RECENT FINDINGS: Despite the potential involvement of immune responses in the pathogenesis of hypertension, there is no evidence supporting that hypothesis that hypertension or RAS inhibitors contributes to unfavorable outcomes in viral infections. Future investigations adopting a strict protocol for confirming hypertension status as well as assessing associated comorbidities that may influence outcomes are necessary. From the therapeutic perspective, recombinant ACE2 may serve as a potential therapy, but relevant studies in humans are lacking. Definitive evidence regarding the use of RAS inhibitors in patients with COVID-19 is needed; 5 randomized trials examining this issue are currently underway. There is no current scientific support for claiming that hypertension or its treatment with RAS inhibitors contribute to unfavorable outcomes in COVID-19.\n",
      "PURPOSE OF REVIEW: There is increasing evidence indicating an association between several risk factors and worse prognosis in patients with coronavirus disease 2019 (COVID-19), including older age, hypertension, heart failure, diabetes, and pulmonary disease. Hypertension is of particular interest because it is common in adults and there are concerns related to the use of renin-angiotensin system (RAS) inhibitors in patients with hypertension infected with COVID-19. Levels of angiotensin-converting enzyme 2 (ACE2), a protein that facilitates entry of coronavirus into cells, may increase in patients using RAS inhibitors. Thus, chronic use of RAS inhibition could potentially lead to a more severe and fatal form of COVID-19. In this review, we provide a critical review to the following questions: (1) Does hypertension influence immunity or ACE2 expression favoring viral infections? (2) Are the risks of complications in hypertension mediated by its treatment? (3) Is aging a major factor associated with worse prognosis in patients with COVID-19 and hypertension? RECENT FINDINGS: Despite the potential involvement of immune responses in the pathogenesis of hypertension, there is no evidence supporting that hypothesis that hypertension or RAS inhibitors contributes to unfavorable outcomes in viral infections. Future investigations adopting a strict protocol for confirming hypertension status as well as assessing associated comorbidities that may influence outcomes are necessary. From the therapeutic perspective, recombinant ACE2 may serve as a potential therapy, but relevant studies in humans are lacking. Definitive evidence regarding the use of RAS inhibitors in patients with COVID-19 is needed; 5 randomized trials examining this issue are currently underway. SUMMARY: There is no current scientific support for claiming that hypertension or its treatment with RAS inhibitors contribute to unfavorable outcomes in COVID-19.\n",
      "Introduction: COVID-19 disproportionately affects those with comorbidities and the elderly. Hypertension is the most common pre-existing condition amongst COVID-19 patients. Upregulation of the renin-angiotensin-aldosterone system (RAAS) is common in hypertensive patients and may promote inflammation and ensuing cytokine storm in COVID-19. It is unknown whether RAAS inhibition with ACE1 inhibitors or angiotensin-receptor blockers (ARB) can be harmful or beneficial. Methods: Within Hackensack Meridian Health network, the largest healthcare provider in New Jersey, we performed a retrospective, multicenter, convenience sampling study of hospitalized COVID-19 patients. Demographics, clinical characteristics, treatments, and outcomes were manually abstracted. Fishers exact tests, and logistic regression were performed. Results: Among 3017 hospitalized COVID-19 patients, 1584 (52.5%) carried a diagnosis of hypertension. In the discharged or deceased cohort, the overall mortality was significantly increased at 35% vs 13% among COVID-19 patients with hypertension. However, when adjusted for age, the effect of hypertension on mortality was greatly diminished, with a reduction in odds-ratio by over half; and completely disappeared when adjusted for other major covariates. The mortality rates were lower for hypertensive patients prescribed ACE1 (27%, p=0.001) or ARBs (33%, p=0.12) compared to other anti-hypertensive agents (39%) in the unadjusted analyses. RAAS inhibitor therapy appeared protective compared to other anti-hypertensive agents (p=0.001). Conclusions: While our results are limited by the retrospective nature of our study and by potential confounders, our data argue against a harmful effect of RAAS inhibition and support the HFSA/AHA/ACC joint statement recommending continuing ACE1 and ARB therapy in hypertensive COVID-19 patients.\n",
      "Investigations reported that hypertension, diabetes, and cardiovascular diseases were the most prevalent comorbidities among the patients with coronavirus disease 2019 (COVID-19). Hypertension appeared consistently as the most prevalent risk factors in COVID-19 patients. Some investigations speculated about the association between renin-angiotensin-aldosterone system (RAAS) and susceptibility to COVID-19, as well as the relationship between RAAS inhibitors and increased mortality in these patients. This raised concern about the potential association between hypertension (and its treatment) and propensity for COVID-19. There are only a few follow-up studies that investigated the impact of comorbidities on outcome in these patients with conflicting findings. Hypertension has been proven to be more prevalent in patients with an adverse outcome (admission in intensive care unit, use of mechanical ventilation, or death). So far, there is no study that demonstrated independent predictive value of hypertension on mortality in COVID-19 patients. There are many speculations about this coronavirus and its relation with different risk factors and underlying diseases. The aim of this review was to summarize the current knowledge about the relationship between hypertension and COVID-19 and the role of hypertension on outcome in these patients.\n",
      "The coronavirus disease (COVID-19) has spread all around the world in a very short period of time. Recent data are showing significant prevalence of arterial hypertension and cardiovascular diseases (CVD) among patients with COVID-19, which raised many questions about higher susceptibility of patients with these comorbidities to the novel coronavirus, as well as the role of hypertension and CVD in progression and the prognosis of COVID-19 patients. There is a very limited amount of data, usually obtained from a small population, regarding the effect of the underlying disease on the outcome in patients with COVID-19. The evaluation of the treatment of these comorbidities at baseline and during COVID-19 is scarce and the results are conflicting. Hypertension and CVD, after the adjustment for other clinical and demographic parameters, primarily age, did not remain independent predictors of the lethal outcome in COVID-19 patients. Some investigations speculated about the association between the renin-angiotensin-aldosterone system (RAAS) and susceptibility to COVID-19, as well as the relationship between RAAS inhibitors and the adverse outcome in these patients. Withdrawing or switching RAAS inhibitors would have uncertain benefits, but it would definitely have many disadvantages such as uncontrolled hypertension, cardiac function deterioration and renal function impairment, which could potentially induce more complications in patients with COVID-19 than the infection of coronavirus itself. The aim of this review article was to summarize the prevalence of hypertension and CVD in patients with COVID-19, their influence on the outcome and the effect of treatment of hypertension and CVD in COVID-19 patients.\n",
      "Summary Background The novel coronavirus (CoV) severe acute respiratory syndrome (SARS)-CoV-2 outbreak started at the end of 2019 in Wuhan, China, and spread over 100 countries. SARS-CoV-2 uses the membrane protein Angiotensin I converting enzyme 2(ACE2) as a cell entry receptor. Indeed, it was reported that the balance of Renin-Angiotensin System (RAS), regulated by both ACE and ACE2, was altered in COVID-19 patients. It is controversial, however, whether commonly used anti-hypertensive drugs Angiotensin I converting enzyme inhibitor (ACEI) and Angiotensin II receptor blocker (ARB) shall be continued in the confirmed COVID-19 patients. This study was designed to investigate any difference in disease severity between COVID-19 patients with hypertension comorbidity. The included COVID-19 patients used ACEI, ARB, calcium channel blockers (CCB), beta blockers (BB), or thiazide to treat preexisting hypertension prior to the hospital were compared to patients who did not take any of those drugs. Methods In this multicentre retrospective study, clinical data of 511 COVID-19 patients were analyzed. Patients were categorized into six sub-groups of hypertension comorbidity based on treatment using one of anti-hypertension drugs (ACEI, ARB, CCB, BB, thiazide), or none. A meta-analysis was performed to evaluate the use of ACEI and ARB associated with pneumonia using published studies. Findings Among the elderly (age>65) COVID-19 patients with hypertension comorbidity, the risk of COVID-19-S (severe disease) was significantly decreased in patients who took ARB drugs prior to hospitalization compared to patients who took no drugs (OR=0.343, 95% CI 0.128-0.916, p=0.025). The meta-analysis showed that ARB use has positive effects associated with morbidity and mortality of pneumonia. Interpretation Elderly (age>65) COVID-19 patients with hypertension comorbidity who are taking ARB anti-hypertension drugs may be less likely to develop severe lung disease compared to patients who take no anti-hypertension drugs. Funding National Natural Science Foundation of China, Chinese Academy of Medical Sciences\n",
      "BACKGROUND: The effect of chronic use of renin-angiotensin-aldosterone system (RAAS) inhibitors on the severity of COVID-19 infection is still unclear in patients with hypertension. We aimed to investigate the association between chronic use of angiotensin-converting enzyme inhibitors (ACEIs) or angiotensin II receptor blockers (ARBs) and COVID-19 related outcomes in hypertensive patients. METHODS: A single center study was conducted on 133 consecutive hypertensive subjects presenting to the Emergency Department with acute respiratory symptoms and/or fever who were diagnosed with COVID-19 infection between 9th and 31st March 2020. RESULTS: All patients were grouped according to their chronic antihypertensive medications (ACEIs, N=40; ARBs, N=42; not on RAAS inhibitors, N=51). There was no statistical difference between ACEIs and ARBs groups in terms of hospital admission rate, oxygen therapy and need for non-invasive ventilation. Patients chronically treated with RAAS inhibitors showed a significantly lower rate of admission to semi-intensive/intensive care units, when compared to the non-RAAS population (odds ratio [OR] 0.25, CI95% 0.09-0.66 p=0.006). Similarly, the risk of mortality was lower in the former group, although not reaching statistical significance (OR 0.56, CI95% 0.17-1.83, p=0.341). CONCLUSIONS: Our data suggest that chronic use of RAAS inhibitors does not negatively affect clinical course of COVID-19 in hypertensive patients. Further studies are needed to confirm this finding and determine whether RAAS inhibitors may have a protective effect on COVID 19-related morbidity and mortality.\n",
      "Investigations reported that hypertension, diabetes, and cardiovascular diseases were the most prevalent comorbidities among the patients with coronavirus disease 2019 (COVID‐19). Hypertension appeared consistently as the most prevalent risk factors in COVID‐19 patients. Some investigations speculated about the association between renin‐angiotensin‐aldosterone system (RAAS) and susceptibility to COVID‐19, as well as the relationship between RAAS inhibitors and increased mortality in these patients. This raised concern about the potential association between hypertension (and its treatment) and propensity for COVID‐19. There are only a few follow‐up studies that investigated the impact of comorbidities on outcome in these patients with conflicting findings. Hypertension has been proven to be more prevalent in patients with an adverse outcome (admission in intensive care unit, use of mechanical ventilation, or death). So far, there is no study that demonstrated independent predictive value of hypertension on mortality in COVID‐19 patients. There are many speculations about this coronavirus and its relation with different risk factors and underlying diseases. The aim of this review was to summarize the current knowledge about the relationship between hypertension and COVID‐19 and the role of hypertension on outcome in these patients.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: nadzy6lm\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what is known about those infected with Covid-19 but are asymptomatic?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Abstract Asymptomatic but infectious people have been reported in many infectious diseases. Asymptomatic and pre-symptomatic carriers would be a hidden reservoir of COVID-19. Aim This review identifies primary empirical evidence about the ability of asymptomatic carriers to infect others with COVID-19 pandemic and reflects on the implications for control measures. Methods A systematic review is followed by a narrative report and commentary inclusion criteria were: studies reporting primary data on asymptomatic or pre-symptomatic patients, who were considered to have passed on COVID-19 infection; and published in indexed journals or in peer review between January 1 and March 31, 2020. Results Nine articles reported on 83 asymptomatic or pre-symptomatic persons. Conclusions The evidence confirms COVID-19 transmission from people who were asymptomatic at the time. A series of implications for health service response are laid out. Keywords: Covid-19, Asymptomatic, Pre-symptomatic, Public Health\n",
      "BACKGROUND: the recent outbreak of the coronavirus disease 2019 (COVID‐19) has quickly spread globally since its discovery in Wuhan, China, in December 2019. A comprehensive strategy, including surveillance, diagnostics, research, and clinical treatment is urgently needed to win the battle against COVID-19. Recently, numerous studies reported the incidence of SARS-CoV-2 in asymptomatic patients. Yet, the incidence and viral transmission from the asymptomatic cases are not apparent yet. AIM: this study aims to systematically review the published literature on SARS-CoV-2 in the asymptomatic patients to estimate the incidence of COVID-19 among asymptomatic cases, as well as describe its epidemiological and clinical significance. METHOD: the literature was searched through four scientific databases: PubMed, Web of Science, Scopus, and Science Direct. RESULTS: a total of 63 studies satisfied the inclusion criteria where the majority of the reported studies were from China. However, there was a lack of SARS-CoV-2 epidemiological studies from several countries worldwide, tracing the actual incidence of COVID-19, especially in asymptomatic patients. Studies with a large sample size (n>1000) estimated that percentage of people contracting SARS-CoV-2 and are likely to be asymptomatic ranges from 1.2-12.9%. However, the other studies with a smaller sample size reported a much higher incidence and indicated that up to 87.9% of COVID-19 infected individuals could be asymptomatic. Most of these studies indicated that asymptopatics are a potential source of infection to the community. CONCLUSION: this review highlighted the need for more robust and well-designed studies to better estimate COVID-19 incidence among asymptomatic patients worldwide. The early identification of the asymptomatic cases, as well as monitoring and tracing close contact, could help in mitigating the spread of COVID-19.\n",
      "BACKGROUND: the recent outbreak of the coronavirus disease 2019 (COVID-19) has quickly spread globally since its discovery in Wuhan, China, in December 2019. A comprehensive strategy, including surveillance, diagnostics, research, and clinical treatment is urgently needed to win the battle against COVID-19. Recently, numerous studies reported the incidence of SARS-CoV-2 in asymptomatic patients. Yet, the incidence and viral transmission from the asymptomatic cases are not apparent yet. AIM: this study aims to systematically review the published literature on SARS-CoV-2 in the asymptomatic patients to estimate the incidence of COVID-19 among asymptomatic cases, as well as describe its epidemiological and clinical significance. METHOD: the literature was searched through four scientific databases: PubMed, Web of Science, Scopus, and Science Direct. RESULTS: a total of 63 studies satisfied the inclusion criteria where the majority of the reported studies were from China. However, there was a lack of SARS-CoV-2 epidemiological studies from several countries worldwide, tracing the actual incidence of COVID-19, especially in asymptomatic patients. Studies with a large sample size (n>1000) estimated that percentage of people contracting SARS-CoV-2 and are likely to be asymptomatic ranges from 1.2-12.9%. However, the other studies with a smaller sample size reported a much higher incidence and indicated that up to 87.9% of COVID-19 infected individuals could be asymptomatic. Most of these studies indicated that asymptopatics are a potential source of infection to the community. CONCLUSION: this review highlighted the need for more robust and well-designed studies to better estimate COVID-19 incidence among asymptomatic patients worldwide. The early identification of the asymptomatic cases, as well as monitoring and tracing close contact, could help in mitigating the spread of COVID-19.\n",
      "The 2019 novel coronavirus disease, SARS-CoV-2, is now spreading globally and is characterized by person-to-person transmission. However, it has recently been found that individuals infected with SARS-CoV-2 can be asymptomatic, and simultaneously a source of infection in others. The viral load detected in nasopharyngeal swabs of asymptomatic carriers is relatively high, with a great potential for transmission. More attention should be paid to the insidious spread of disease and harm contributed by asymptomatic SARS-CoV-2 carriers. To provide a theoretical basis for the accurate and early clinical identification of asymptomatic patients, this review objectively summarizes the epidemic status, transmission characteristics and clinical features of asymptomatic patients with SARS-CoV-2 infection.\n",
      "Background Asymptomatic carriers contribute to the spread of Coronavirus Disease 2019 (COVID-19), but their clinical characteristics, viral kinetics, and antibody responses remain unclear. Methods A total of 56 COVID-19 patients without symptoms at admission and 19 age-matched symptomatic patients were enrolled. RNA of SARS-CoV-2 was tested using transcriptase quantitative PCR, and the total antibodies (Ab), IgG, IgA and IgM against the SARS-CoV-2 were tested using Chemiluminescence Microparticle Immuno Assay. Results Among 56 patients without symptoms at admission, 33 cases displayed symptoms and 23 remained asymptomatic throughout the follow-up period. 43.8% of the asymptomatic carriers were children and none of the asymptomatic cases had recognizable changes in C-reactive protein or interleukin-6, except one 64-year-old patient. The initial threshold cycle value of nasopharyngeal SARS-CoV-2 in asymptomatic carriers was similar to that in pre-symptomatic and symptomatic patients, but the communicable period of asymptomatic carriers (9.63 days) was shorter than pre-symptomatic patients (13.6 days). There was no obvious differences of the seropositive conversion rate of total Ab, IgG, and IgA among the three groups, though the rates of IgM varied largely. The average peak IgG and IgM COI of asymptomatic cases was 3.5 and 0.8, respectively, which is also lower than those in symptomatic patients with peaked IgG and IgM COI of 4.5 and 2.4 (p <0.05). Conclusion Young COVID-19 patients seem to be asymptomatic cases with early clearance of SARS-CoV-2 and low levels of IgM generation but high total Ab, IgG and IgA. Our findings provide empirical information for viral clearance and antibody kinetics of asymptomatic COVID-19 patients.\n",
      "Background: The prevalence of true asymptomatic COVID-19 cases is critical to policy makers considering the effectiveness of mitigation measures against the SARS-CoV-2 pandemic. We aimed to synthesize all available research on the asymptomatic rates and transmission rates where possible. Methods: We searched PubMed, Embase, Cochrane COVID-19 trials, and European PMC for pre-print platforms such as MedRxiv. We included primary studies reporting on asymptomatic prevalence where: (a) the sample frame includes at-risk population, and (b) there was sufficiently long follow up to identify pre-symptomatic cases. Meta-analysis used fixed effect and random effects models. Results: We screened 571 articles and included five low risk-of-bias studies from three countries (China (2), USA (2), Italy (1)) that tested 9,242 at-risk people, of which 413 were positive and 65 were asymptomatic. Diagnosis in all studies was confirmed using a RT-qPCR test. The proportion of asymptomatic cases ranged from 6% to 41%. Meta-analysis (fixed effect) found that the proportion of asymptomatic cases was 16% (95% CI: 12% - 20%) overall; higher in non-aged care 19% (15% - 24%), and lower in long-term aged care 8% (4% - 14%). Two studies provided direct evidence of forward transmission of the infection by asymptomatic cases but suggested lower rates than symptomatic cases. Conclusion: Our estimates of the prevalence of asymptomatic COVID-19 cases are lower than many highly publicized studies, but still substantial. Further robust epidemiological evidence is urgently needed, including in sub-populations such as children, to better understand the importance of asymptomatic cases for driving spread of the pandemic.\n",
      "The asymptomatic carrier state of COVID-19 has become a topic of concern for preventing a possible epidemic rebound. This review describes and defines the COVID-19 asymptomatic carrier state and outlines methods for identifying counting and reporting these cases. The author elaborates that the asymptomatic carrier state can be further divided into asymptomatic infection and pre-symptomatic infection after extended follow-up based on the nature of disease progression. The author presents the limited available data about infectiousness of asymptomatic and pre-symptomatic cases and their possible contributions to the overall epidemic of COVID-19 observed so far in China. Challenges of a possible second epidemic wave of COVID-19 caused by asymptomatic and pre-symptomatic cases are discussed and suggestions for control strategies and scientific research are provided.\n",
      "Abstract From 71 laboratory-confirmed cases, we found three presymptomatic patients and 10 entirely asymptomatic infections. In two of the three incubation period patients, the viral titer in the presymptomatic period was very high (Ct value <20). The median days to first negative RT-PCR in the entirely asymptomatic carriers was 4.5 (range 2.5–9) days and all asymptomatic carriers reached a first Ct>35 RT-PCR within 14 days after diagnosis. Patients who have COVID-19 may already be infectious before there are symptoms, and 14 days of isolation after diagnosis may be sufficient in entirely asymptomatic cases.\n",
      "BACKGROUND AND PURPOSE: Many of the statutes comprising the shelter-in-place and phased-reopening orders are centered around minimizing asymptomatic and presymptomatic transmission. Assumptions about the presence and relative importance of asymptomatic and presymptomatic transmission are based on case reports, the failing of quarantine measures aimed at sequestering ill patients, viral dynamic studies suggesting SARS-CoV-2 production peaks before symptoms appear, and modeling evidence that calculates serial interval between successive generations of infection. In aggregate, these data offer compelling evidence of asymptomatic and presymptomatic transmission, but individually these studies have notable shortcomings that undermine their conclusions. The purpose of this review is to discuss the literature of asymptomatic and presymptomatic transmission, highlight limitations of recent studies, and propose experiments that, if conducted, would provide a more definitive analysis of the relative role of asymptomatic and presymptomatic transmission in the ongoing SARS-CoV-2 pandemic. METHODS: We conducted a systematic review of literature on PubMed using search filters that relate to asymptomatic and presymptomatic transmission as well as serial interval and viral dynamics. We focused on studies that provided primary clinical data. RESULTS: 34 studies were eligible for inclusion in this systematic review: 11 case reports pertaining to asymptomatic transmission, 9 viral kinetic studies, 13 serial interval studies, and 1 study with viral kinetics and serial interval. CONCLUSION: Different approaches to determining the presence and prevalence of asymptomatic and presymptomatic SARS-CoV-2 transmission have notable shortcomings, which were highlighted in this review and limit our ability to draw definitive conclusions. Conducting high quality studies with the aim of understanding the relative role of asymptomatic and presymptomatic transmission is instrumental to developing the most informed policies on reopening our cities, states, and countries.\n",
      "BACKGROUND An ongoing outbreak of coronavirus disease 2019 (COVID-19) has spread around the world. It is debatable whether asymptomatic COVID-19 virus carriers are contagious. We report here a case of the asymptomatic patient and present clinical characteristics of 455 contacts, which aims to study the infectivity of asymptomatic carriers. MATERIAL AND METHODS 455 contacts who were exposed to the asymptomatic COVID-19 virus carrier became the subjects of our research. They were divided into three groups: 35 patients, 196 family members and 224 hospital staffs. We extracted their epidemiological information, clinical records, auxiliary examination results and therapeutic schedules. RESULTS The median contact time for patients was four days and that for family members was five days. Cardiovascular disease accounted for 25% among original diseases of patients. Apart from hospital staffs, both patients and family members were isolated medically. During the quarantine, seven patients plus one family member appeared new respiratory symptoms, where fever was the most common one. The blood counts in most contacts were within a normal range. All CT images showed no sign of COVID-19 infection. No severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infections was detected in 455 contacts by nucleic acid test. CONCLUSION In summary, all the 455 contacts were excluded from SARS-CoV-2 infection and we conclude that the infectivity of some asymptomatic SARS-CoV-2 carriers might be weak.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: kugr54if\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are best practices in hospitals and at home in maintaining quarantine?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Abstract Since its emergence in December 2019, the virus known as severe acute respiratory syndrome coronavirus 2 has quickly caused a pandemic. This virus causes a disease now known as coronavirus disease 2019, or COVID-19. As an increasing proportion of the at-risk population becomes infected, and patients with severe illness are hospitalized, it is essential for hospitalists to remain current on how to best care for people with suspected or confirmed disease. Establishing a system for logistical planning, and accurate information sharing is strongly recommended. Infection control remains the ultimate goal. As such, healthcare workers should be educated on universal and isolation precautions, and the appropriate use of personal protective equipment. Social distancing should be encouraged to prevent the spread of infection, and creative and innovative ways to reduce contact may need to be considered. Moreover, it is imperative to prepare for contingencies as medical staff will inevitably get sick or become unavailable. Hospitalists have the difficult task of caring for patients, while also adapting to the many logistical and social elements of a pandemic.\n",
      "Since its emergence in December 2019, the virus known as severe acute respiratory syndrome coronavirus 2 has quickly caused a pandemic. This virus causes a disease now known as coronavirus disease 2019, or COVID-19. As an increasing proportion of the at-risk population becomes infected, and patients with severe illness are hospitalized, it is essential for hospitalists to remain current on how to best care for people with suspected or confirmed disease. Establishing a system for logistical planning, and accurate information sharing is strongly recommended. Infection control remains the ultimate goal. As such, health care workers should be educated on universal and isolation precautions, and the appropriate use of personal protective equipment. Social distancing should be encouraged to prevent the spread of infection, and creative and innovative ways to reduce contact may need to be considered. Moreover, it is imperative to prepare for contingencies as medical staff will inevitably get sick or become unavailable. Hospitalists have the difficult task of caring for patients while also adapting to the many logistical and social elements of a pandemic.\n",
      "The coronavirus disease (COVID-19) pandemic caused by the severe acute respiratory syndrome (SARS-CoV-2) virus is challenging healthcare providers across the world. Current best practices for personal protective equipment (PPE) during this time are rapidly evolving and fluid due to the novel and acute nature of the pandemic and the dearth of high-level evidence. Routine infection control practices augmented by airborne precautions are paramount when treating the COVID-19-positive patient. Best practices for PPE use in patients who have unknown COVID-19 status are a highly charged and emotional issue. The variables to be considered include protection of patients and healthcare providers, accuracy and availability of testing, and responsible use of PPE resources. This article also explores the concerns of surgeons regarding possible transmission to their own family members as a result of caring for COVID-19 patients.\n",
      "INTRODUCTION: Rapid worldwide spread of Coronavirus Disease 2019 (COVID-19) has resulted in a global pandemic. OBJECTIVE: This review article provides emergency physicians with an overview of the most current understanding of COVID-19 and recommendations on the evaluation and management of patients with suspected COVID-19. DISCUSSION: Severe Acute Respiratory Syndrome coronavirus 2 (SARS-CoV-2), the virus responsible for causing COVID-19, is primarily transmitted from person-to-person through close contact (approximately 6 ft) by respiratory droplets. Symptoms of COVID-19 are similar to other viral upper respiratory illnesses. Three major trajectories include mild disease with upper respiratory symptoms, non-severe pneumonia, and severe pneumonia complicated by acute respiratory distress syndrome (ARDS). Emergency physicians should focus on identifying patients at risk, isolating suspected patients, and informing hospital infection prevention and public health authorities. Patients with suspected COVID-19 should be asked to wear a facemask. Respiratory etiquette, hand washing, and personal protective equipment are recommended for all healthcare personnel caring for suspected cases. Disposition depends on patient symptoms, hemodynamic status, and patient ability to self-quarantine. CONCLUSION: This narrative review provides clinicians with an updated approach to the evaluation and management of patients presenting to the emergency department with suspected COVID-19.\n",
      "Coronavirus disease 2019 (COVID-19) stands out as the major pandemic that we have experienced in the last century. As it affects every social structure, it brought the importance of intensive care support once again to the agenda of healthcare system after causing severe acute respiratory syndrome. The precautions to be taken against this virus, where our knowledge is extremely small, intensive care units take an indispensable place in pandemic planning. In this review, we aimed to emphasize the crucial points regarding intensive care management of COVID-19 patients, which we have written not only for intensivists but also for all healthcare professionals.\n",
      "Coronavirus disease 2019 (COVID-19) stands out as the major pandemic that we have experienced in the last century. As it affects every social structure, it brought the importance of intensive care support once again to the agenda of healthcare system after causing severe acute respiratory syndrome. The precautions to be taken against this virus, where our knowledge is extremely small, intensive care units take an indispensable place in pandemic planning. In this review, we aimed to emphasize the crucial points regarding intensive care management of COVID-19 patients, which we have written not only for intensivists but also for all healthcare professionals.\n",
      "Abstract COVID-19 disease is a pandemic disease caused by a single-stranded RNA virus that belongs to the coronavirus family known as 2019-nCoV (SARS-CoV-2). The disease is highly contagious and transmitted mainly by droplets or close contact. As the global incidence of COVID-19 disease is increasing dramatically, health systems worldwide started to suffer in terms of its capability to manage affected people and the ability to provide standard treatment for critically ill patients in a safe environment. As Medical staff has extensive contact with patients, families as well as other health care providers, they are very likely to cause cross-infection. This paper aims to provide a comprehensive overview of COVID-19 disease as well as to highlight essential measures that healthcare providers and surgeons need to take into consideration during their management of the patient during the COVID-19 pandemic.\n",
      "Since the first recognition of a cluster of novel respiratory viral infections in China in late December 2019, intensivists in the United States have watched with growing concern as infections with the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) virus-now named coronavirus disease of 2019 (COVID-19)-have spread to hospitals in the United States. Because COVID-19 is extremely transmissible and can progress to a severe form of respiratory failure, the potential to overwhelm available critical care resources is high and critical care management of COVID-19 patients has been thrust into the spotlight. COVID-19 arrived in the United States in January and, as anticipated, has dramatically increased the usage of critical care resources. Three of the hardest-hit cities have been Seattle, New York City, and Chicago with a combined total of over 14,000 cases as of March 23, 2020.In this special article, we describe initial clinical impressions of critical care of COVID-19 in these areas, with attention to clinical presentation, laboratory values, organ system effects, treatment strategies, and resource management. We highlight clinical observations that align with or differ from already published reports. These impressions represent only the early empiric experience of the authors and are not intended to serve as recommendations or guidelines for practice, but rather as a starting point for intensivists preparing to address COVID-19 when it arrives in their community.\n",
      "Since the first recognition of a cluster of novel respiratory viral infections in China in late December 2019, intensivists in the United States have watched with growing concern as infections with the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) virus―now named Coronavirus Disease of 2019 (COVID-19)―have spread to hospitals in the United States. Because COVID-19 is extremely transmissible and can progress to a severe form of respiratory failure, the potential to overwhelm available critical care resources is high and critical care management of COVID-19 patients has been thrust into the spotlight. COVID-19 arrived in the United States in January and, as anticipated, has dramatically increased the usage of critical care resources. Three of the hardest-hit cities have been Seattle, New York City, and Chicago with a combined total of over 14,000 cases as of March 23, 2020. In this special article, we describe initial clinical impressions of critical care of COVID-19 in these areas, with attention to clinical presentation, laboratory values, organ system effects, treatment strategies, and resource management. We highlight clinical observations that align with or differ from already published reports. These impressions represent only the early empiric experience of the authors and are not intended to serve as recommendations or guidelines for practice, but rather as a starting point for intensivists preparing to address COVID-19 when it arrives in their community.\n",
      "Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) is a newly identified agent foisted upon humanity and responsible for the contagious affliction, coronavirus disease 2019 (COVID-19) [1] that has rapidly evolved into a pandemic testing to their limits, and sometimes beyond, the capacity to respond of healthcare systems across the world [2]. Management is purely supportive and social isolation crucial to containment [3]. Enforced reallocation of hospital resources and personnel to cope with the increasing numbers requiring hospital admission and intensive care [4] in the most trying of conditions has been at the expense of many hospital departments, among them those offering diagnostic and support services for lung cancer [5–7].\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: ax9btc74\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what types of rapid testing for Covid-19 have been developed?\n",
      "are there serological tests that detect antibodies to coronavirus?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "The current practice for diagnosis of COVID-19, based on SARS-CoV-2 PCR testing of pharyngeal or respiratory specimens in a symptomatic patient at high epidemiologic risk, likely underestimates the true prevalence of infection. Serologic methods can more accurately estimate the disease burden by detecting infections missed by the limited testing performed to date. Here, we describe the validation of a coronavirus antigen microarray containing immunologically significant antigens from SARS-CoV-2, in addition to SARS-CoV, MERS-CoV, common human coronavirus strains, and other common respiratory viruses. A comparison of antibody profiles detected on the array from control sera collected prior to the SARS-CoV-2 pandemic versus convalescent blood specimens from virologically confirmed COVID-19 cases demonstrates complete discrimination of these two groups. This array can be used as a diagnostic tool, as an epidemiologic tool to more accurately estimate the disease burden of COVID-19, and as a research tool to correlate antibody responses with clinical outcomes.\n",
      "The current practice for diagnosis of SARS-CoV-2 infection relies on PCR testing of nasopharyngeal or respiratory specimens in a symptomatic patient at high epidemiologic risk. This testing strategy likely underestimates the true prevalence of infection, creating the need for serologic methods to detect infections missed by the limited testing to date. Here, we describe the development of a coronavirus antigen microarray containing immunologically significant antigens from SARS-CoV-2, in addition to SARS-CoV, MERS-CoV, common human coronavirus strains, and other common respiratory viruses. A preliminary study of human sera collected prior to the SARS-CoV-2 pandemic demonstrates overall high IgG reactivity to common human coronaviruses and low IgG reactivity to epidemic coronaviruses including SARS-CoV-2, with some cross-reactivity of conserved antigenic domains including S2 domain of spike protein and nucleocapsid protein. This array can be used to answer outstanding questions regarding SARS-CoV-2 infection, including whether baseline serology for other coronaviruses impacts disease course, how the antibody response to infection develops over time, and what antigens would be optimal for vaccine development.\n",
      "Detection of antibodies to upper respiratory pathogens is critical to surveillance, assessment of the immune status of individuals, vaccine development, and basic biology. The urgent need for antibody detection tools has proven particularly acute in the COVID-19 era. We report a multiplex label-free antigen microarray on the Arrayed Imaging Reflectometry (AIR) platform for detection of antibodies to SARS-CoV-2, SARS-CoV-1, MERS, three circulating coronavirus strains (HKU1, 229E, OC43) and three strains of influenza. We find that the array is readily able to distinguish uninfected from convalescent COVID-19 subjects, and provides quantitative information about total Ig, as well as IgG- and IgM-specific responses.\n",
      "Serologic assays are needed to determine SARS-CoV-2 seroprevalence, but poor specificity can overestimate exposures. Here, we built a pan-human coronavirus proteome-wide programmable phage display assay (VirScan) to profile coronavirus antigens specifically enriched by 20 COVID-19 patient serum IgG. With ReScan, a new diagnostic development workflow which combines the isolation of phage expressing the most immunogenic peptides with paper-based microarrays manufactured via acoustic liquid handling, we identified 9 candidate antigens from a library of 534 SARS-CoV-2 peptides. These arrays could form the basis of a multiplexed COVID-19 serologic assay with enhanced specificity. ReScan has broad applicability for serologic assay development.\n",
      "A new coronavirus, SARS-CoV-2, has recently emerged to cause a human pandemic. Whereas molecular diagnostic tests were rapidly developed, serologic assays are still lacking, yet urgently needed. Validated serologic assays are important for contact tracing, identifying the viral reservoir and epidemiological studies. Here, we developed serological assays for the detection of SARS-CoV-2 neutralizing, spike- and nucleocapsid-specific antibodies. Using serum samples from patients with PCR-confirmed infections of SARS-CoV-2, other coronaviruses, or other respiratory pathogenic infections, we validated and tested various antigens in different in-house and commercial ELISAs. We demonstrate that most PCR-confirmed SARS-CoV-2 infected individuals seroconverted, as revealed by sensitive and specific in-house ELISAs. We found that commercial S1 IgG or IgA ELISAs were of lower specificity while sensitivity varied between the two, with IgA showing higher sensitivity. Overall, the validated assays described here can be instrumental for the detection of SARS-CoV-2-specific antibodies for diagnostic, seroepidemiological and vaccine evaluation studies.\n",
      "A new coronavirus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has recently emerged to cause a human pandemic. Although molecular diagnostic tests were rapidly developed, serologic assays are still lacking, yet urgently needed. Validated serologic assays are needed for contact tracing, identifying the viral reservoir, and epidemiologic studies. We developed serologic assays for detection of SARS-CoV-2 neutralizing, spike protein-specific, and nucleocapsid-specific antibodies. Using serum samples from patients with PCR-confirmed SARS-CoV-2 infections, other coronaviruses, or other respiratory pathogenic infections, we validated and tested various antigens in different in-house and commercial ELISAs. We demonstrated that most PCR-confirmed SARS-CoV-2-infected persons seroconverted by 2 weeks after disease onset. We found that commercial S1 IgG or IgA ELISAs were of lower specificity, and sensitivity varied between the 2 assays; the IgA ELISA showed higher sensitivity. Overall, the validated assays described can be instrumental for detection of SARS-CoV-2-specific antibodies for diagnostic, seroepidemiologic, and vaccine evaluation studies.\n",
      "Accurate serologic tests to detect host antibodies to severe acute respiratory syndrome-related coronavirus-2 (SARS-CoV-2) will be critical for the public health response to the coronavirus disease 2019 pandemic. Many use cases are envisaged, including complementing molecular methods for diagnosis of active disease and estimating immunity for individuals. At the population level, carefully designed seroepidemiologic studies will aid in the characterization of transmission dynamics and refinement of disease burden estimates and will provide insight into the kinetics of humoral immunity. Yet, despite an explosion in the number and availability of serologic assays to test for antibodies against SARS-CoV-2, most have undergone minimal external validation to date. This hinders assay selection and implementation, as well as interpretation of study results. In addition, critical knowledge gaps remain regarding serologic correlates of protection from infection or disease, and the degree to which these assays cross-react with antibodies against related coronaviruses. This article discusses key use cases for SARS-CoV-2 antibody detection tests and their application to serologic studies, reviews currently available assays, highlights key areas of ongoing research, and proposes potential strategies for test implementation.\n",
      "Accurate serologic tests to detect host antibodies to severe acute respiratory syndrome–related coronavirus-2 (SARS-CoV-2) will be critical for the public health response to the coronavirus disease 2019 pandemic. Many use cases are envisaged, including complementing molecular methods for diagnosis of active disease and estimating immunity for individuals. At the population level, carefully designed seroepidemiologic studies will aid in the characterization of transmission dynamics and refinement of disease burden estimates and will provide insight into the kinetics of humoral immunity. Yet, despite an explosion in the number and availability of serologic assays to test for antibodies against SARS-CoV-2, most have undergone minimal external validation to date. This hinders assay selection and implementation, as well as interpretation of study results. In addition, critical knowledge gaps remain regarding serologic correlates of protection from infection or disease, and the degree to which these assays cross-react with antibodies against related coronaviruses. This article discusses key use cases for SARS-CoV-2 antibody detection tests and their application to serologic studies, reviews currently available assays, highlights key areas of ongoing research, and proposes potential strategies for test implementation.\n",
      "A new coronavirus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has recently emerged to cause a human pandemic. Although molecular diagnostic tests were rapidly developed, serologic assays are still lacking, yet urgently needed. Validated serologic assays are needed for contact tracing, identifying the viral reservoir, and epidemiologic studies. We developed serologic assays for detection of SARS-CoV-2 neutralizing, spike protein–specific, and nucleocapsid-specific antibodies. Using serum samples from patients with PCR-confirmed SARS-CoV-2 infections, other coronaviruses, or other respiratory pathogenic infections, we validated and tested various antigens in different in-house and commercial ELISAs. We demonstrated that most PCR-confirmed SARS-CoV-2–infected persons seroconverted by 2 weeks after disease onset. We found that commercial S1 IgG or IgA ELISAs were of lower specificity, and sensitivity varied between the 2 assays; the IgA ELISA showed higher sensitivity. Overall, the validated assays described can be instrumental for detection of SARS-CoV-2–specific antibodies for diagnostic, seroepidemiologic, and vaccine evaluation studies.\n",
      "Rapid and accurate tests that detect IgM and IgG antibodies to SARS-CoV-2 proteins are essential in slowing the spread of COVID-19 by identifying patients who are infected with COVID-19. Using a SARS-CoV-2 proteome microarray developed in our lab, we comprehensively profiled both IgM and IgG antibodies in forty patients with early-stage COVID-19, influenza, or non-influenza who had similar symptoms. The results revealed that the SARS-CoV-2 N protein is not an ideal biomarker for COVID-19 diagnosis because of its low immunogenicity, thus tests that rely on this marker alone will have a high false negative rate. Our data further suggest that the S protein subunit 1 receptor binding domain (S1-RBD) might be the optimal antigen for IgM antibody detection, while the S protein extracellular domain (S1+S2ECD) would be the optimal antigen for both IgM and IgG antibody detection. Notably, the combination of all IgM and IgG biomarkers can identify 87% and 73.3% COVID-19 patients, respectively. Finally, the COVID-19-specific antibodies are significantly correlated with the clinical indices of viral infection and acute myocardial injury (p≤0.05). Our data may help understand the function of anti-SARS-CoV-2 antibodies and improve serology tests for rapid COVID-19 screening.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: oihyz5v7\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "are there serological tests that detect antibodies to coronavirus?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "BACKGROUND: A novel coronavirus, SARS-CoV-2, has recently emerged and caused the rapid spread of COVID-19 worldwide. METHODS: We did a retrospective study and included COVID-19 patients admitted to Renmin Hospital of Wuhan University between February 1 and February 29, 2020. Antibody assay was conducted to detect COVID-19 envelope protein E and nucleocapsid protein N antigen. RESULTS: 112 patients were recruited with symptoms of fever, cough, fatigue, myalgia, and diarrhoea. All patients underwent antibody tests. Fifty-eight (51.79%) were positive for both IgM and IgG, 7 (6.25%) were negative for both antibodies, 1 (0.89%) was positive for only IgM, and 46 (41.07%) were positive for only IgG. IgM antibody appeared within a week post disease onset, and lasted for one month and gradually decreased, while IgG antibody was produced 10 days after infection, and lasted for a longer time. However, no significant difference in level of IgM and IgG antibody between positive and negative patients of nucleic acid test after treatment was found. CONCLUSIONS: Our results indicate that serological tests could be powerful approach for the early diagnosis of COVID-19.\n",
      "BACKGROUND: A novel coronavirus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has recently emerged and caused the rapid spread of coronavirus disease 2019 (COVID-19) worldwide. METHODS: We did a retrospective study and included COVID-19 patients admitted to Renmin Hospital of Wuhan University between 1 February and 29 February 2020. Antibody assay was conducted to detect COVID-19 envelope protein E and nucleocapsid protein N antigen. RESULTS: One hundred twelve patients were recruited with symptoms of fever, cough, fatigue, myalgia, and diarrhea. All patients underwent antibody tests. Fifty-eight (51.79%) were positive for both immunoglobulin M (IgM) and immunoglobulin G (IgG), 7 (6.25%) were negative for both antibodies, 1 (0.89%) was positive for only IgM, and 46 (41.07%) were positive for only IgG. IgM antibody appeared within a week post-disease onset, lasted for 1 month, and gradually decreased, whereas IgG antibody was produced 10 days after infection and lasted for a longer time. However, no significant difference in levels of IgM and IgG antibodies between positive and negative patients of nucleic acid test after treatment was found. CONCLUSIONS: Our results indicate that serological tests could be a powerful approach for the early diagnosis of COVID-19.\n",
      "Background: As the increasing number of Corona Virus Disease 2019 (COVID-19) patients caused by the severe acute respiratory coronavirus 2 (SARS-CoV-2), which caused an outbreak initiated from Wuhan, China in December, 2019, the clinical features and treatment of COVID-19 patients have been understood. However, it is urgent to need the rapid and accurate detection for SARS-CoV-2 infection diagnosis. We aimed to evaluate the antibodies-based and nucleic acid-based tests (NAT) for SARS-CoV-2-infected patients. Method: We retrospectively and observationally studied 133 patients diagnosed with SARS-CoV-2 and admitted in Renmin Hospital of Wuhan University, China, from Feb 17 to Mar 1, 2020. Demographic data, symptoms, clinical examination, laboratory tests, and clinical outcomes were collected. Data were compared between IgM-IgG antibody test and real-time RT-PCR detection for COVID-19 patients. Results: Of 133 patients with SARS-CoV-2 infection, there were 44 moderate cases, 52 severe cases, and 37 critical cases with no significant difference of gender and age among three subgroups. Overall, the positive ratio in IgM antibody test was higher than in RT-PCR detection. In RT-PCR detection, the positive ratio was 65.91%, 71.15%, and 67.57% in moderate, severe, and critical cases, respectively. Whereas, the positive ratio of IgM/IgG antibody detection in patients was 79.55%/93.18%, 82.69%/100%, and 72.97%/97.30% in moderate, severe, and critical cases, respectively. Moreover, the concentrations of antibodies were also measured in three subgroups. Conclusion: The IgM-IgG antibodies-based test exhibited a comparative superiority to the NAT for COVID-19 diagnosis, which provides an effective complement to the false negative results from NAT for SARS-CoV-2 infection diagnosis.\n",
      "BACKGROUND: The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) caused coronavirus disease 2019 (COVID-19) epidemic in China, December 2019. The clinical features and treatment of COVID-19 patients remain largely elusive. However, accurate detection is required for SARS-CoV-2 infection diagnosis. We aimed to evaluate the antibodies-based test and nucleic acid-based test for SARS-CoV-2-infected patients. METHODS: We retrospectively studied 133 patients diagnosed with SARS-CoV-2 and admitted to Renmin Hospital of Wuhan University, China, from January 23 to March 1, 2020. Demographic data, clinical records, laboratory tests, and outcomes were collected. Data were accessed by SARS-CoV-2 IgM-IgG antibody test and real-time reverse transcriptase PCR (RT-PCR) detection for SARS-CoV-2 nucleic acid in COVID-19 patients. RESULTS: Of 133 COVID-19 patients, there were 44 moderate cases, 52 severe cases, and 37 critical cases with no differences in gender and age among three subgroups. In RT-PCR detection, the positive rate was 65.9%, 71.2%, and 67.6% in moderate, severe, and critical cases, respectively. Whereas the positive rate of IgM/IgG antibody detection in patients was 79.5%/93.2%, 82.7%/100%, and 73.0%/97.3% in moderate, severe, and critical cases, respectively. Moreover, the IgM and IgG antibodies concentrations were also examined with no differences among three subgroups. CONCLUSION: The IgM-IgG antibody test exhibited a useful adjunct to RT-PCR detection, and improved the accuracy in COVID-19 diagnosis regardless of the severity of illness, which provides an effective complement to the false-negative results from a nucleic acid test for SARS-CoV-2 infection diagnosis after onsets.\n",
      "Background: This study was aimed to investigate the application of SARS- COV-2 IgM and IgG antibodies in diagnosis of COVID-19 infection. Method: This study enrolled a total of 178 patients at Huangshi Central Hospital from January to February, 2020. Among them, 68 patients were SARS-COV-2 infected confirmed with nucleic acid test (NAT) and CT imaging. 9 patients were in the suspected group (NAT negative) with fever and other respiratory symptoms. 101 patients were in the control group with other diseases and negative to SARS-COV-2 infection. After serum samples were collected, SARS-COV-2 IgG and IgM antibodies were tested by chemiluminescence immunoassay (CLIA) for all patients. Results: The specificity of serum IgM and IgG antibodies to SARS-COV-2 were 99.01% (100/101) and 96.04% (97/101) respectively, and the sensitivity were 88.24% (60/68) and 97.06% (66/68) respectively. The combined detection rate of SARS-COV-2 IgM and IgG antibodies were 98.53% (67/68). Conclusion: Combined detection of serum SARS-COV-2 IgM and IgG antibodies had better sensitivity compared with single IgM or IgG test, which can be used as an important diagnostic tool for SARS-COV-2 infection and a screening tool of potential SARS-COV-2 carriers in clinics, hospitals and accredited scientific laboratory.\n",
      "BACKGROUND: The corona virus disease 2019 (COVID-19) caused by the corona virus 2 (SARS-CoV-2) has been rapidly spreading nationwide and abroad. A serologic test to identify antibody dynamics and response to SARS-CoV-2 was developed. METHODS: The antibodies against SARS-CoV-2 were detected by an enzyme-linked immunosorbent assay (ELISA) based on the recombinant nucleocapsid protein of SARS-CoV-2 in patients with confirmed or suspected COVID-19 at 3-40 days after symptom onset. The gold standard for COVID-19 diagnosis was nucleic acid testing for SARS-CoV-2 by RT-PCR. The serodiagnostic power of the specific IgM and IgG antibodies against SARS-CoV-2 was investigated in terms of sensitivity, specificity, positive predictive value (PPV), negative predictive value (NPV) and consistency rate. RESULTS: The seroconversion of specific IgM and IgG antibodies were observed as early as the 4(th) day after symptom onset. In the confirmed patients with COVID-19, sensitivity, specificity, PPV, NPV, and consistency rate of IgM were 77.3% (51/66), 100%, 100%, 80.0%, and 88.1%, and those of IgG were 83.3.3% (55/66), 95.0%, 94.8%, 83.8%, and 88.9 %. In patients with suspected COVID-19, sensitivity, specificity, PPV, NPV, and consistency rate of IgM were 87.5% (21/24), 100%, 100%, 95.2%, and 96.4%, and those of IgG were 70.8% (17/24), 96.6%, 85.0%, 89.1%, and 88.1%. Both antibodies performed well in serodiagnosis for COVID-19 rely on great specificity. CONCLUSIONS: The antibodies against SARS-CoV-2 can be detected in the middle and later stage of the illness. Antibody detection may play an important role in the diagnosis of COVID-19 as complement approach for viral nucleid acid assays.\n",
      "Background: Several immunochromatographic serological test kits have been developed to detect severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) specific antibodies, but their relative performance and potential clinical utility is unclear. Methods: Three commercially available serological test kits were evaluated using 99 serum samples collected from 29 patients diagnosed with coronavirus disease 2019 (COVID-19). Results: The IgM antibody-positive rates of the three serological test kits for samples taken at the early stage of the disease (0-6 days after onset) were 19.0%, 23.8%, and 19.0%, respectively. The IgM antibody-positive rates over the entire period were 21.2%, 60.6%, and 15.2%, respectively. The IgG antibody-positive rates for samples taken after 13 days of onset were 100.0%, 97.6%, and 97.6%, respectively. Conclusion: There were large differences among the results of the three test kits. Only few cases showed positive results for IgM in the early stage of disease and the IgM antibody-positive rates over the entire period were low, suggesting that the kits used in this study were unsuitable for diagnosis of COVID-19. The IgG antibody was positive in almost all samples after 13 days of onset, suggesting that it may be useful for determining infections in the recent past.\n",
      "BACKGROUND: The corona virus disease 2019 (COVID-19) caused by the corona virus 2 (SARS-CoV-2) has been rapidly spreading nationwide and abroad. A serologic test to identify antibody dynamics and response to SARS-CoV-2 was developed. METHODS: The antibodies against SARS-CoV-2 were detected by an enzyme-linked immunosorbent assay (ELISA) based on the recombinant nucleocapsid protein of SARS-CoV-2 in patients with confirmed or suspected COVID-19 at 3-40 days after symptom onset. The gold standard for COVID-19 diagnosis was nucleic acid testing for SARS-CoV-2 by RT-PCR. The serodiagnostic power of the specific IgM and IgG antibodies against SARS-CoV-2 was investigated in terms of sensitivity, specificity, positive predictive value (PPV), negative predictive value (NPV) and consistency rate. RESULTS: The seroconversion of specific IgM and IgG antibodies were observed as early as the 4th day after symptom onset. In the confirmed patients with COVID-19, sensitivity, specificity, PPV, NPV, and consistency rate of IgM were 77.3% (51/66), 100%, 100%, 80.0%, and 88.1%, and those of IgG were 83.3.3% (55/66), 95.0%, 94.8%, 83.8%, and 88.9 %. In patients with suspected COVID-19, sensitivity, specificity, PPV, NPV, and consistency rate of IgM were 87.5% (21/24), 100%, 100%, 95.2%, and 96.4%, and those of IgG were 70.8% (17/24), 96.6%, 85.0%, 89.1%, and 88.1%. Both antibodies performed well in serodiagnosis for COVID-19 rely on great specificity. CONCLUSIONS: The antibodies against SARS-CoV-2 can be detected in the middle and later stage of the illness. Antibody detection may play an important role in the diagnosis of COVID-19 as complement approach for viral nucleid acid assays.\n",
      "Background The outbreak of the recently emerged novel corona virus disease 2019 (COVID-19) poses a challenge for public health laboratories. We aimed to evaluate the diagnostic value of serological assay for SARS-CoV-2. Methods A newly-developed ELISA assay for IgM and IgG antibodies against N protein of SARS-CoV-2 were used to screen the serums of 238 admitted hospital patients with confirmed or suspected SARS-CoV-2 infection from February 6 to February 14, 2020. SARS-CoV-2 RNA was detected by real time RT-PCR on pharyngeal swab specimens. Findings Of the 238 patients, 194 (81.5%) were detected to be antibody (IgM and/or IgG) positive, which was significantly higher than the positive rate of viral RNA (64.3%). There was no difference in the positive rate of antibody between the confirmed patients (83.0%, 127/153) and the suspected patients (78.8%, 67/85) whose nucleic acid tests were negative. After the patients were defined to the different stages of disease based on the day when the test samples were collected, the analysis results showed that the antibody positive rates were very low in the first five days after initial onset of symptoms, and then rapidly increased as the disease progressed. After 10 days, the antibody positive rates jumped to above 80% from less than 50%. On the contrary, the positive rates of viral RNA kept above 60% in the first 11 days after initial onset of symptoms, and then rapidly decreased. In addition, half of the suspected patients with symptoms for 6-10 days were detected to be antibody positive. Interpretation The suspected patients were most likely infected by SARS-CoV-2. Before the 11th day after initial onset of symptoms, nucleic acid test is important for confirmation of viral infection. The combination of serological assay can greatly improve the diagnostic efficacy. After that, the diagnosis for viral infection should be majorly dependent on serological assay. Keywords. SARS-CoV-2; diagnosis; serological assay; nucleic acid test\n",
      "Background: Severe acute respiratory syndrome coronavirus 2(SARS-CoV-2), a novel betacoronavirus, has caused an outburst of pneumonia cases in Wuhan, China. We report the production of specific IgM and IgG antibodies after the infection of SARS-CoV-2 and its implication for the diagnosis, pathology and the course of the disease as well as the recurrence of positive nucleic acid tests after discharge. Methods: Test results for SARS-CoV-2 IgM and IgG antibodies of 221 confirmed COVID-19 patients were retrospectively examined, and their clinical data were collected and analyzed based on various subgroups. SARS-CoV-2 IgM and IgG antibodies were determined with the chemiluminescence method. Findings: The concentration (S/CO) of SARS-CoV-2 IgM and IgG antibodies peaked on day 19-21 after symptom onset, with a median of 17.38 (IQR 4.39-36.4) for IgM and 5.59 (IQR 0.73-13.65) for IgG. Detection rates reached highest on day 16-18 and day 19-21 for IgM and IgG, which were 73.6% and 98.6%, respectively, with significantly higher concentration of IgG in critically ill patients than in those with mild to moderate disease (P=0.027). The concentration of the antibodies on day 16-21 is not correlated with the course or outcome of the disease (Spearman r < 0.20, P > 0.05). Nasopharyngeal swabs revealed positive SARS-CoV-2 RNA in up to 52.7% of recovered patients after discharge, whose IgG proved to be significantly lower than that of those with negative RNA results (P = 0.009). IgG and IgM were tested twice within 14 days after discharge with a 7-day interval, and the second testing of these antibodies displayed a decrease in concentration of 21.2% (IQR, 11.2%34.48%) for IgG and 23.05% (IQR, -27.96%46.13%) for IgM, without statistical significance between the patients with re-detectable positive RNA results and those with negative RNA results after discharge. However, those with positive results experienced a count decrease in lymphocyte subsets. Interpretation: The concentration of SARS-CoV-2 IgM and IgG antibodies peaked on day 19-21 after symptom onset, and antibody testing on day 16-21 is associated with increased detection rates, but the antibody concentration does not affect the course and outcome of the infection. Recovering patients with re-detectable positive SARS-CoV-2 RNA displayed lower concentration of IgG, but the downward trend of IgG during recovery indicated its limited duration of protection, and the protective effect of IgG remains to be investigated.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 9595vm0k\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "are there serological tests that detect antibodies to coronavirus?\n",
      "what types of rapid testing for Covid-19 have been developed?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "A new coronavirus, SARS-CoV-2, has recently emerged to cause a human pandemic. Whereas molecular diagnostic tests were rapidly developed, serologic assays are still lacking, yet urgently needed. Validated serologic assays are important for contact tracing, identifying the viral reservoir and epidemiological studies. Here, we developed serological assays for the detection of SARS-CoV-2 neutralizing, spike- and nucleocapsid-specific antibodies. Using serum samples from patients with PCR-confirmed infections of SARS-CoV-2, other coronaviruses, or other respiratory pathogenic infections, we validated and tested various antigens in different in-house and commercial ELISAs. We demonstrate that most PCR-confirmed SARS-CoV-2 infected individuals seroconverted, as revealed by sensitive and specific in-house ELISAs. We found that commercial S1 IgG or IgA ELISAs were of lower specificity while sensitivity varied between the two, with IgA showing higher sensitivity. Overall, the validated assays described here can be instrumental for the detection of SARS-CoV-2-specific antibodies for diagnostic, seroepidemiological and vaccine evaluation studies.\n",
      "A new coronavirus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has recently emerged to cause a human pandemic. Although molecular diagnostic tests were rapidly developed, serologic assays are still lacking, yet urgently needed. Validated serologic assays are needed for contact tracing, identifying the viral reservoir, and epidemiologic studies. We developed serologic assays for detection of SARS-CoV-2 neutralizing, spike protein-specific, and nucleocapsid-specific antibodies. Using serum samples from patients with PCR-confirmed SARS-CoV-2 infections, other coronaviruses, or other respiratory pathogenic infections, we validated and tested various antigens in different in-house and commercial ELISAs. We demonstrated that most PCR-confirmed SARS-CoV-2-infected persons seroconverted by 2 weeks after disease onset. We found that commercial S1 IgG or IgA ELISAs were of lower specificity, and sensitivity varied between the 2 assays; the IgA ELISA showed higher sensitivity. Overall, the validated assays described can be instrumental for detection of SARS-CoV-2-specific antibodies for diagnostic, seroepidemiologic, and vaccine evaluation studies.\n",
      "A new coronavirus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has recently emerged to cause a human pandemic. Although molecular diagnostic tests were rapidly developed, serologic assays are still lacking, yet urgently needed. Validated serologic assays are needed for contact tracing, identifying the viral reservoir, and epidemiologic studies. We developed serologic assays for detection of SARS-CoV-2 neutralizing, spike protein–specific, and nucleocapsid-specific antibodies. Using serum samples from patients with PCR-confirmed SARS-CoV-2 infections, other coronaviruses, or other respiratory pathogenic infections, we validated and tested various antigens in different in-house and commercial ELISAs. We demonstrated that most PCR-confirmed SARS-CoV-2–infected persons seroconverted by 2 weeks after disease onset. We found that commercial S1 IgG or IgA ELISAs were of lower specificity, and sensitivity varied between the 2 assays; the IgA ELISA showed higher sensitivity. Overall, the validated assays described can be instrumental for detection of SARS-CoV-2–specific antibodies for diagnostic, seroepidemiologic, and vaccine evaluation studies.\n",
      "Serological assays for anti-SARS-CoV-2 antibodies are now of critical importance to support diagnosis, guide epidemiological intervention, and understand immune response to natural infection and vaccine administration. We developed and validated new anti-SARS-CoV-2 IgG, IgM and IgA ELISA tests (ENZY-WELL SARS-CoV-2 ELISA, DIESSE Diagnostica Senese S.p.a.) based on whole-virus antigens. We used a total of 553 serum samples including samples from COVID-19 suspected and confirmed cases, healthy donors, and patients positive for other infections or autoimmune conditions. Overall, the assays showed good concordance with the indirect immunofluorescence reference test in terms of sensitivity and specificity. Especially for IgG and IgA, we observed high sensitivity (92.5 and 93.6%, respectively); specificity was high (>96%) for all antibody types ELISAs. In addition, sensitivity was linked to the days from symptoms onset (DSO) due to the seroconversion window, and for ENZY-WELL SARS-CoV-2 IgG and IgA ELISAs resulted 100% in those samples collected after 10 and 12 DSO, respectively. The results showed that ENZY-WELL SARS-CoV-2 ELISAs may represent a valid option for both diagnostic and epidemiological purposes, covering all different antibody types developed in SARS-CoV-2 immune response.\n",
      "Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) serological assays are urgently needed for rapid diagnosis, contact tracing, and for epidemiological studies. So far, there is limited data on how commercially available tests perform with real patient samples, and if positive tested samples show neutralizing abilities. Focusing on IgG antibodies, we demonstrate the performance of two enzyme-linked immunosorbent assay (ELISA) assays (Euroimmun SARS-CoV-2 IgG and Vircell COVID-19 ELISA IgG) in comparison to one lateral flow assay (FaStep COVID-19 IgG/IgM Rapid Test Device) and two in-house developed assays (immunofluorescence assay [IFA] and plaque reduction neutralization test [PRNT]). We tested follow up serum/plasma samples of individuals polymerase chain reaction-diagnosed with COVID-19. Most of the SARS-CoV-2 samples were from individuals with moderate to the severe clinical course, who required an in-patient hospital stay. For all examined assays, the sensitivity ranged from 58.8 to 76.5% for the early phase of infection (days 5-9) and from 93.8% to 100% for the later period (days 10-18).\n",
      "BACKGROUND: Knowledge of the COVID-19 epidemic extent and the level of herd immunity is urgently needed to help manage this pandemic. METHODS: We used a panel of 167 samples (77 pre-epidemic and 90 COVID-19 seroconverters) and SARS-CoV1, SARS-CoV2 and MERS-CoV Spike and/or Nucleopcapsid (NC) proteins to develop a high throughput multiplex screening assay to detect IgG antibodies in human plasma. Assay performances were determined by ROC curves analysis. A subset of the COVID-19+ samples (n = 36) were also tested by a commercial NC-based ELISA test and the results compared with those of the novel assay. RESULTS: On samples collected ≥14 days after symptoms onset, the accuracy of the assay is 100% (95% CI: 100-100) for the Spike antigen and 99.9% (95% CI:99.7-100) for NC. By logistic regression, we estimated that 50% of the patients have seroconverted at 5.7 ± 1.6; 5.7 ± 1.8 and 7.9 ± 1.0 days after symptoms onset against Spike, NC or both antigens, respectively and all have seroconverted two weeks after symptoms onset. IgG titration in a subset of samples showed that early phase samples present lower IgG titers than those from later phase. IgG to SARS-CoV2 NC cross-reacted at 100% with SARS-CoV1 NC. Twenty-nine of the 36 (80.5%) samples tested were positive by the commercial ELISA while 31/36 (86.1%) were positive by the novel assay. CONCLUSIONS: Our assay is highly sensitive and specific for the detection of IgG antibodies to SARS-CoV2 proteins, suitable for high throughput epidemiological surveys. The novel assay is more sensitive than a commercial ELISA.\n",
      "BACKGROUND: Knowledge of the COVID-19 epidemic extent and the level of herd immunity is urgently needed to help manage this pandemic. METHODS: We used a panel of 167 samples (77 pre-epidemic and 90 COVID-19 seroconverters) and SARS-CoV1, SARS-CoV2 and MERS-CoV Spike and/or Nucleopcapsid (NC) proteins to develop a high throughput multiplex screening assay to detect IgG antibodies in human plasma. Assay performances were determined by ROC curves analysis. A subset of the COVID-19+ samples (n = 36) were also tested by a commercial NC-based ELISA test and the results compared with those of the novel assay. RESULTS: On samples collected &#8805;14 days after symptoms onset, the accuracy of the assay is 100 % (95 % CI: 100-100) for the Spike antigen and 99.9 % (95 % CI:99.7-100) for NC. By logistic regression, we estimated that 50 % of the patients have seroconverted at 5.7 ± 1.6; 5.7 ± 1.8 and 7.9 ± 1.0 days after symptoms onset against Spike, NC or both antigens, respectively and all have seroconverted two weeks after symptoms onset. IgG titration in a subset of samples showed that early phase samples present lower IgG titers than those from later phase. IgG to SARS-CoV2 NC cross-reacted at 100 % with SARS-CoV1 NC. Twenty-nine of the 36 (80.5 %) samples tested were positive by the commercial ELISA while 31/36 (86.1 %) were positive by the novel assay. CONCLUSIONS: Our assay is highly sensitive and specific for the detection of IgG antibodies to SARS-CoV2 proteins, suitable for high throughput epidemiological surveys. The novel assay is more sensitive than a commercial ELISA.\n",
      "As the current SARS-CoV-2 pandemic continues, serological assays are urgently needed for rapid diagnosis, contact tracing and for epidemiological studies. So far, there is little data on how commercially available tests perform with real patient samples and if detected IgG antibodies provide protective immunity. Focusing on IgG antibodies, we demonstrate the performance of two ELISA assays (Euroimmun SARS-CoV-2 IgG & Vircell COVID-19 ELISA IgG) in comparison to one lateral flow assay ((LFA) FaStep COVID-19 IgG/IgM Rapid Test Device) and two in-house developed assays (immunofluorescence assay (IFA) and plaque reduction neutralization test (PRNT)). We tested follow up serum/plasma samples of individuals PCR-diagnosed with COVID-19. Most of the SARS-CoV-2 samples were from individuals with moderate to severe clinical course, who required an in-patient hospital stay. For all examined assays, the sensitivity ranged from 58.8 to 76.5% for the early phase of infection (days 5-9) and from 93.8 to 100% for the later period (days 10-18) after PCR-diagnosed with COVID-19. With exception of one sample, all positive tested samples in the analysed cohort, using the commercially available assays examined (including the in-house developed IFA), demonstrated neutralizing (protective) properties in the PRNT, indicating a potential protective immunity to SARS-CoV-2. Regarding specificity, there was evidence that samples of endemic coronavirus (HCoV-OC43, HCoV-229E) and Epstein Barr virus (EBV) infected individuals cross-reacted in the ELISA assays and IFA, in one case generating a false positive result (may giving a false sense of security). This need to be further investigated.\n",
      "Since emergence of SARS-CoV-2 in late 2019, there has been a critical need to understand prevalence, transmission patterns, to calculate the burden of disease and case fatality rates. Molecular diagnostics, the gold standard for identifying viremic cases, are not ideal for determining true case counts and rates of asymptomatic infection. Serological detection of SARS-CoV-2 specific antibodies can contribute to filling these knowledge gaps. In this study, we describe optimization and validation of a SARS-CoV-2-specific-enzyme linked immunosorbent assay (ELISA) using the prefusion-stabilized form of the spike protein [1]. We performed receiver operator characteristic (ROC) analyses to define the specificities and sensitivities of the optimized assay and examined cross reactivity with immune sera from persons confirmed to have had infections with other coronaviruses. These assays will be used to perform contact investigations and to conduct large-scale, cross sectional surveillance to define disease burden in the population.\n",
      "SARS‐CoV‐2 serological assays are urgently needed for rapid diagnosis, contact tracing and for epidemiological studies. So far, there is limited data on how commercially available tests perform with real patient samples and if positive tested samples show neutralizing abilities. Focusing on IgG antibodies, we demonstrate the performance of two ELISA assays (Euroimmun SARS‐CoV‐2 IgG and Vircell COVID‐19 ELISA IgG) in comparison to one lateral flow assay ((LFA) FaStep COVID‐19 IgG/IgM Rapid Test Device) and two in‐house developed assays (immunofluorescence assay (IFA) and plaque reduction neutralization test (PRNT)). We tested follow up serum/plasma samples of individuals PCR‐diagnosed with COVID‐19. Most of the SARS‐CoV‐2 samples were from individuals with moderate to severe clinical course, who required an in‐patient hospital stay. For all examined assays, the sensitivity ranged from 58.8 to 76.5% for the early phase of infection (days 5‐9) and from 93.8 to 100% for the later period (days 10‐18). With exception of one sample, all positive tested COVID‐19 follow up‐samples, using the commercially available assays examined (including the in‐house developed IFA), demonstrated neutralizing properties in the PRNT. Regarding specificity, some samples of endemic coronavirus (HCoV‐OC43, HCoV‐229E) and Epstein Barr virus (EBV) infected individuals cross‐reacted in the ELISA assays and IFA, in one case generating a false positive result. This article is protected by copyright. All rights reserved.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 8ruux4sw\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "are there any clinical trials available for the coronavirus\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "The SARS-CoV-2 virus emerged in December 2019 and then spread rapidly worldwide, particularly to China, Japan, and South Korea. Scientists are endeavoring to find antivirals specific to the virus. Several drugs such as chloroquine, arbidol, remdesivir, and favipiravir are currently undergoing clinical studies to test their efficacy and safety in the treatment of coronavirus disease 2019 (COVID-19) in China; some promising results have been achieved thus far. This article summarizes agents with potential efficacy against SARS-CoV-2.\n",
      "The SARS-CoV-2 virus emerged in December 2019 and then spread rapidly worldwide, particularly to China, Japan, and South Korea. Scientists are endeavoring to find antivirals specific to the virus. Several drugs such as chloroquine, arbidol, remdesivir, and favipiravir are currently undergoing clinical studies to test their efficacy and safety in the treatment of coronavirus disease 2019 (COVID-19) in China; some promising results have been achieved thus far. This article summarizes agents with potential efficacy against SARS-CoV-2.\n",
      "Since a novel coronavirus pneumonia outbreak in late December 2019, coronavirus disease -19 (COVID-19) epidemic has gradually spread worldwide, becoming a major public health event. No specific antivirals are currently available for COVID-19 caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). The treatments for COVID-19 are mainly based on the experiences of similar virus such SARS-CoV, MERS-CoV, HIV and influenza viruses. Scientists have taken great efforts to investigate the effective methods for the treatment of COVID-19. Up to now, there are over 1000 clinical studies for COVID-19 all over the world. In this article, we reviewed the current options for COVID-19 therapy including small molecules such as Remdesivir, Favipiravir, Lopinavir/Ritonavir etc, peptide inhibitors of ACE2, Traditional Chinese Medicines and Biologics such as SARS-CoV-2-specific neutralizing antibodies, mesenchymal stem cells and vaccines etc. Meanwhile, we systematically reviewed their clinical safety, clinical applications and progress of antiviral researches. The therapeutic effect of these antiviral drugs is summarized and compared, hoping to provide some ideas for clinical options of COVID-19 treatment and also provide experiences for the life-threatening virus diseases in the future.\n",
      "PURPOSE: SARS-CoV-2 first emerged in China in December 2019 and rapidly spread worldwide. No vaccine or approved drug is available to eradicate the virus, however, some drugs that are indicated for other afflictions seems to be potentially beneficial to treat the infection albeit without unequivocal evidence. The aim of this article is to review the published background on the effectiveness of these drugs against COVID-19 Methods: A thorough literature search was conducted on recently published studies which have published between January 1 to March 25, 2020. PubMed, Google Scholar and Science Direct databases were searched Results: A total 22 articles were found eligible. 8 discuss about treatment outcomes from their applied drugs during treatment of COVID-19 patients, 4 report laboratory tests, one report animal trial and other 9 articles discuss recommendations and suggestions based on the treatment process and clinical outcomes of other diseases such as malaria, ebola, severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS). The data and/or recommendations are categorized in 4 classes: (a) anti-viral and anti-inflammatory drugs, (b) anti-malaria drugs, (c) traditional Chinese drugs and (d) other treatments/drugs. CONCLUSION: All examined treatments, although potentiality effective against COVID-19, need either appropriate drug development or clinical trial to be suitable for clinical use.\n",
      "Since the outbreak of novel coronavirus pneumonia (COVID-19) in December 2019, more than 2,500,000 people worldwide have been diagnosed with SARS-CoV-2 as of April 22. In response to this epidemic, China has issued seven trial versions of diagnosis and treatment protocol for COVID-19. According to the information that we have collected so far, this article provides an overview of potential therapeutic drugs and compounds with much attention, including favipiravir and hydroxychloroquine, as well as traditional Chinese medicine, which have been reported with good clinical treatment effects. Moreover, with further understanding of SARS-CoV-2 virus, new drugs targeting specific SARS-CoV-2 viral components arise and investigations on these novel anti-SARS-CoV-2 agents are also reviewed.\n",
      "PURPOSE SARS-CoV-2 first emerged in China in December 2019 and rapidly spread worldwide. No vaccine or approved drug is available to eradicate the virus, however, some drugs that are indicated for other afflictions seems to be potentially beneficial to treat the infection albeit without unequivocal evidence. The aim of this article is to review the published background on the effectiveness of these drugs against COVID-19 Methods: A thorough literature search was conducted on recently published studies which have published between January 1 to March 25, 2020. PubMed, Google Scholar and Science Direct databases were searched Results: A total 22 articles were found eligible. 8 discuss about treatment outcomes from their applied drugs during treatment of COVID-19 patients, 4 report laboratory tests, one report animal trial and other 9 articles discuss recommendations and suggestions based on the treatment process and clinical outcomes of other diseases such as malaria, ebola, severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS). The data and/or recommendations are categorized in 4 classes: (a) anti-viral and anti-inflammatory drugs, (b) anti-malaria drugs, (c) traditional Chinese drugs and (d) other treatments/drugs. CONCLUSION All examined treatments, although potentiality effective against COVID-19, need either appropriate drug development or clinical trial to be suitable for clinical use.\n",
      "The emerging outbreak of coronavirus disease 2019 (COVID-19) caused by the severe acute respiratory syndrome coronavirus 2 continues to spread all over the world. Agents or vaccines of proven efficacy to treat or prevent human coronavirus infection are in urgent need and are being investigated vigorously worldwide. This review summarizes the current evidence of potential therapeutic agents, such as lopinavir/ritonavir, remdesivir, favipiravir, chloroquine, hydroxychloroquine, interferon, ribavirin, tocilizumab, and sarilumab. More clinical trials are being conducted for further confirmation of the efficacy and safety of these agents in treating COVID-19.\n",
      "BACKGROUND: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), termed coronavirus disease 2019 (COVID-19) by the World Health Organization, is a newly emerging zoonotic agent that emerged in China in December 2019. No specific treatment for COVID-19 is currently available. Usual palliative treatment includes maintaining hydration and nutrition and controlling fever and cough. The clinical severity and extent of transmission need to be determined, and therapeutic options need to be developed and optimized. METHODS: The present review discusses the recent repurposing of drugs for COVID-19 treatment. RESULTS: Several compounds, including remdesivir, lopinavir, ritonavir, interferon-ß, ribavirin, chloroquine/hydroxychloroquine, azithromycin, tocilizumab, and ivermectin, have emerged as promising alternatives. They block the virus from entering host cells, prevent viral replication, and attenuate exacerbation of the host's immune response. CONCLUSION: Although some evidence indicates the positive actions of different classes of compounds for the treatment of COVID-19, few clinical assays have been established to definitively demonstrate their therapeutic value in humans. Multicenter clinical studies are urgently needed to validate and standardize therapeutic regimens that involve these agents. Although science has not yet presented us with a specific drug against COVID-19, the repurposing of drugs appears to be promising in our fight against this devastating disease.\n",
      "Since the infectious disease caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) was reported in China during December 2019, the coronavirus disease 2019 (COVID-19) has spread on a global scale, causing the World Health Organization (WHO) to issue a warning. While novel vaccines and drugs that target SARS-CoV-2 are under development, this review provides information on therapeutics which are under clinical trials or are proposed to antagonize SARS-CoV-2. Based on the information gained from the responses to other RNA coronaviruses, including the strains that cause severe acute respiratory syndrome (SARS)-coronaviruses and Middle East respiratory syndrome (MERS), drug repurposing might be a viable strategy. Since several antiviral therapies can inhibit viral replication cycles or relieve symptoms, mechanisms unique to RNA viruses will be important for the clinical development of antivirals against SARS-CoV-2. Given that several currently marketed drugs may be efficient therapeutic agents for severe COVID-19 cases, they may be beneficial for future viral pandemics and other infections caused by RNA viruses when standard treatments are unavailable.\n",
      "The novel coronavirus, later identified as SARS-CoV-2, originating from Wuhan in China in November 2019, quickly spread around the world becoming a pandemic. Despite the knowledge of previous coronaviruses, such as those responsible for the SARS and MERS-CoV epidemic, there is no drug or prophylaxis treatment to this day. The rapid succession of scientific findings on SARS-CoV-2 provides a significant number of potential drug targets. Nevertheless, at the same time, the high quantity of clinical data, generated by a large number of rapidly infected people, require accurate tests regarding effective medical treatments. Several in vitro and in vivo studies were rapidly initiated after the outbreak of the pandemic COVID-19. Initial clinical studies revealed the promising potential of remdesivir that demonstrated a powerful and specific in vitro antiviral activity for COVID-19. Promising effects appear to be attributable to hydroxychloroquine. Remdesivir and hydroxychloroquine are being tested in ongoing randomized trials. In contrast, oseltamivir was not effective and corticosteroids are not currently recommended. However, few data from ongoing clinical trials are identifying low molecular weight heparins, innate immune system stimulating agents, and inflammatory modulating agents as potential effective agents. The authors assume that the current pandemic will determine the need for a systematic approach based on big data analysis for identifying effective drugs to defeat SARS-Cov-2. This work is aimed to be a general reference point and to provide an overview as comprehensive as possible regarding the main clinical trials in progress at the moment.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: bgm4dlwz\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) lies behind the ongoing outbreak of coronavirus disease 2019 (COVID-19). There is a growing understanding of SARS-CoV-2 in virology, epidemiology, and clinical management strategies. However, no anti-SARS-CoV-2 drug or vaccine has been officially approved due to the absence of adequate evidence. Scientists are racing to develop a treatment for COVID-19. Recent studies have revealed many attractive therapeutic options, even if some of them remain to be further confirmed in rigorous preclinical models and clinical trials. In this minireview, we aim to summarize the updated potential approaches against SARS-CoV-2. We emphasize that further efforts are warranted to develop the safest and most effective approach.\n",
      "Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the causative agent of coronavirus disease 2019 (COVID-19), has now become a serious global threat after inflicting more than 8 million infections and 425,000 deaths in less than 6 months. Currently, no definitive treatment or prevention therapy exists for COVID-19. The unprecedented rise of this pandemic has rapidly fueled research efforts to discover and develop new vaccines and treatment strategies against this novel coronavirus. While hundreds of vaccines/therapeutics are still in the preclinical or early stage of clinical development, a few of them have shown promising results in controlling the infection. Here, in this review, we discuss the promising vaccines and treatment options for COVID-19, their challenges, and potential alternative strategies.\n",
      "The Coronavirus disease 19 (COVID-19) outbreak has been recognized as a global threat to public health. It is caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and no effective therapies currently exist against this novel viral agent. Along with extensive public health measures, an unprecedented global effort in identifying effective drugs for the treatment is being implemented. Potential drug targets are emerging as the result of a fast-evolving understanding of SARS-CoV-2 virology, host response to the infection, and clinical course of the disease. This brief review focuses on the latest and most promising pharmacological treatments against COVID-19 currently under investigation and discuss their potential use based on either documented efficacy in similar viral infections, or their activity against inflammatory syndromes. Ongoing clinical trials are also emphasized.\n",
      "The Coronavirus disease 19 (COVID-19) outbreak has been recognized as a global threat to public health. It is caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and no effective therapies currently exist against this novel viral agent. Along with extensive public health measures, an unprecedented global effort in identifying effective drugs for the treatment is being implemented. Potential drug targets are emerging as the result of a fast-evolving understanding of SARS-CoV-2 virology, host response to the infection, and clinical course of the disease. This brief review focuses on the latest and most promising pharmacological treatments against COVID-19 currently under investigation and discuss their potential use based on either documented efficacy in similar viral infections, or their activity against inflammatory syndromes. Ongoing clinical trials are also emphasized.\n",
      "Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a new viral disease that has gained global attention owing to its ability to provoke community and health-care-associated outbreaks of severe infections in human populations. The virus poses serious challenges to clinical management because there are still no approved anti- SARS-CoV-2 drugs available. In this mini-review, we summarize the much updated published reports that demonstrate the mechanism of SARS-CoV-2 entry into host cells, and discuss the availability and development of attractive host-based therapeutic options for SARS-CoV-2 infections.\n",
      "Coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) represents the most significant global public health crisis of this generation. From the beginning of the pandemic, several publications and on-line resources about different treatment lines have been done, and development effort in response to the COVID-19 pandemic to investigate potential therapies is unprecedented. Unfortunately, until now, there is not enough evidence to recommend any specific anti-COVID19 treatment. Randomized clinical trials and high-quality evidence, even in the middle of a pandemic, are needed. We provide a review of the latest published literature on the therapeutic strategies and current investigational lines for SARS-CoV-2.\n",
      "Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the causative agent of CoV disease 2019 (COVID-19) is a highly pathogenic and transmissible CoV that is presently plaguing the global human population and economy. No proven effective antiviral therapy or vaccine currently exists, and supportive care remains to be the cornerstone treatment. Through previous lessons learned from SARS-CoV-1 and MERS-CoV studies, scientific groups worldwide have rapidly expanded the knowledge pertaining to SARS-CoV-2 virology that includes in vitro and in vivo models for testing of antiviral therapies and randomized clinical trials. In the present narrative, we review SARS-CoV-2 virology, clinical features, pathophysiology, and animal models with a specific focus on the antiviral and adjunctive therapies currently being tested or that require testing in animal models and randomized clinical trials.\n",
      "The current pandemic of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has presented unprecedented challenges to the healthcare systems in almost every country around the world. Currently, there are no proven effective vaccines or therapeutic agents against the virus. Current clinical management includes infection prevention and control measures and supportive care including supplemental oxygen and mechanical ventilatory support. Evolving research and clinical data regarding the virologic SARS-CoV-2 suggest a potential list of repurposed drugs with appropriate pharmacological effects and therapeutic efficacies in treating COVID-19 patients. In this review, we will update and summarize the most common and plausible drugs for the treatment of COVID-19 patients. These drugs and therapeutic agents include antiviral agents (remdesivir, hydroxychloroquine, chloroquine, lopinavir, umifenovir, favipiravir, and oseltamivir), and supporting agents (Ascorbic acid, Azithromycin, Corticosteroids, Nitric oxide, IL-6 antagonists), among others. We hope that this review will provide useful and most updated therapeutic drugs to prevent, control, and treat COVID-19 patients until the approval of vaccines and specific drugs targeting SARS-CoV-2.\n",
      "The virus severe acute respiratory syndrome-coronavirus-2 (SARS-CoV-2) is currently affecting more than 200 countries and territories worldwide. It has been declared as pandemic by World Health Organization (WHO) and the whole world is suffering from corona virus disease 2019 (COVID-19). Currently, no treatment for SARS-CoV-2 are approved because of lack of evidence, but a number of clinical trials are in process and we are expecting fruitful results very soon. This review focuses on various approaches of treatment and few of the most recent clinical trials carried out in this field.\n",
      "BACKGROUND Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), termed coronavirus disease 2019 (COVID-19) by the World Health Organization, is a newly emerging zoonotic agent that emerged in China in December 2019. No specific treatment for COVID-19 is currently available. Usual palliative treatment includes maintaining hydration and nutrition and controlling fever and cough. The clinical severity and extent of transmission need to be determined, and therapeutic options need to be developed and optimized. METHODS The present review discusses the recent repurposing of drugs for COVID-19 treatment. RESULTS Several compounds, including remdesivir, lopinavir, ritonavir, interferon-β, ribavirin, chloroquine/hydroxychloroquine, azithromycin, tocilizumab, and ivermectin, have emerged as promising alternatives. They block the virus from entering host cells, prevent viral replication, and attenuate exacerbation of the host's immune response. CONCLUSION Although some evidence indicates the positive actions of different classes of compounds for the treatment of COVID-19, few clinical assays have been established to definitively demonstrate their therapeutic value in humans. Multicenter clinical studies are urgently needed to validate and standardize therapeutic regimens that involve these agents. Although science has not yet presented us with a specific drug against COVID-19, the repurposing of drugs appears to be promising in our fight against this devastating disease.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: agc89fqv\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Abstract Given the extreme importance of the current pandemic caused by COVID-19, and as scientists agree there is no identified pharmacological treatment, where possible, therapeutic alternatives are raised through drug repositioning. This paper presents a selection of studies involving drugs from different pharmaceutical classes with activity against SARS-CoV-2 and SARS-CoV, with the potential for use in the treatment of COVID-19 disease.\n",
      "Given the extreme importance of the current pandemic caused by COVID-19, and as scientists agree there is no identified pharmacological treatment, where possible, therapeutic alternatives are raised through drug repositioning. This paper presents a selection of studies involving drugs from different pharmaceutical classes with activity against SARS-CoV-2 and SARS-CoV, with the potential for use in the treatment of COVID-19 disease.\n",
      "Summary The rapid and global spread of a new human coronavirus (SARS-CoV-2) has produced an immediate urgency to discover promising targets for treatment of COVID-19. Drug repositioning is an attractive approach that can facilitate the drug discovery process by repurposing existing pharmaceuticals to treat illnesses other than their primary indications. Here, we review current information concerning the global health issue of COVID-19 including promising approved drugs and ongoing clinical trials for prospective treatment options. In addition, we describe computational approaches to be used in drug repurposing and highlight examples of in-silico studies of drug development efforts against SARS-CoV-2.\n",
      "Since December 2019 SARS-Cov-2 was found responsible for the disease COVID-19, which has spread worldwide. No specific therapies/vaccines are yet available for the treatment of COVID-19. Drug repositioning may offer a strategy and a number of drugs have been repurposed, including lopinavir/ritonavir, remdesivir, favipiravir and tocilizumab. This paper describes the main pharmacological properties of such drugs administered to patients with COVID-19, focusing on their antiviral, immune-modulatory and/or anti-inflammatory actions. Where available, data from clinical trials involving patients with COVID-19 are reported. Preliminary clinical trials seem to support their benefit. However, such drugs in COVID-19 patients have peculiar safety profiles. Thus, adequate clinical trials are necessary for these compounds. Nevertheless, while waiting for effective preventive measures i.e. vaccines, many clinical trials on drugs belonging to different therapeutic classes are currently underway. Their results will help us in defining the best way to treat COVID-19 and reducing its symptoms and complications.\n",
      "The rapid and global spread of a new human coronavirus (SARS-CoV-2) has produced an immediate urgency to discover promising targets for the treatment of COVID-19. Drug repositioning is an attractive approach that can facilitate the drug discovery process by repurposing existing pharmaceuticals to treat illnesses other than their primary indications. Here, we review current information concerning the global health issue of COVID-19 including promising approved drugs and ongoing clinical trials for prospective treatment options. In addition, we describe computational approaches to be used in drug repurposing and highlight examples of in silico studies of drug development efforts against SARS-CoV-2.\n",
      "The coronavirus disease 2019 (COVID-19) pandemic caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has infected near 5 million people and led to over 0.3 million deaths. Currently, there is no specific anti-SARS-CoV-2 medication. New drug discovery typically takes more than ten years. Drug repositioning becomes one of the most feasible approaches for combating COVID-19. This work curates the largest available experimental dataset for SARS-CoV-2 or SARS-CoV main protease inhibitors. Based on this dataset, we develop validated machine learning models with relatively low root mean square error to screen 1553 FDA-approved drugs as well as other 7012 investigational or off-market drugs in DrugBank. We found that many existing drugs might be potentially potent to SARS-CoV-2. The druggability of many potent SARS-CoV-2 main protease inhibitors is analyzed. This work offers a foundation for further experimental studies of COVID-19 drug repositioning.\n",
      "The novel human coronavirus (SARS-CoV-2), the causative agent of COVID-19, has quickly become a threat to the public health and economy worldwide. Despite the severity of some cases, there are no current pathogen-specific antivirals available to treat the disease. Therefore, many studies have focused on the evaluation of the anti-SARS-CoV-2 activity of clinically available drugs. Here, we conducted a systematic review to describe the drug repositioning strategy against SARS-CoV-2 and to discuss the clinical impact of this approach in the current pandemic context. The systematic review was performed on March 23, 2020, using PubMed/MEDLINE, Scopus, Cochrane Library, and Biblioteca Virtual de Saúde (BVS). The data were summarized in tables and critically analyzed. After the database search, 12 relevant studies were identified as eligible for the review. Among the drugs reported in these studies, 57 showed some evidence of antiviral activity. Antivirals, especially antiretrovirals, are the main class of therapeutic agents evaluated against COVID-19. Moreover, studies have reported the anti-SARS-CoV-2 activity of antitumor (16%; 9/57), antimalarial (7%, 4/57), and antibacterial (5%; 3/57) agents. Additionally, seven pharmacological agents (chloroquine, tetrandrine, umifenovir (arbidol), carrimycin, damageprevir, lopinavir/ritonavir) are in phase IV of clinical trials. Due to the evidence of the anti-SARS-CoV-2 activity of various clinically available agents, drug repositioning stands out as a promising strategy for a short-term response in the fight against the novel coronavirus. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (10.1007/s00705-020-04693-5) contains supplementary material, which is available to authorized users.\n",
      "The novel human coronavirus (SARS-CoV-2), the causative agent of COVID-19, has quickly become a threat to the public health and economy worldwide. Despite the severity of some cases, there are no current pathogen-specific antivirals available to treat the disease. Therefore, many studies have focused on the evaluation of the anti-SARS-CoV-2 activity of clinically available drugs. Here, we conducted a systematic review to describe the drug repositioning strategy against SARS-CoV-2 and to discuss the clinical impact of this approach in the current pandemic context. The systematic review was performed on March 23, 2020, using PubMed/MEDLINE, Scopus, Cochrane Library, and Biblioteca Virtual de Saúde (BVS). The data were summarized in tables and critically analyzed. After the database search, 12 relevant studies were identified as eligible for the review. Among the drugs reported in these studies, 57 showed some evidence of antiviral activity. Antivirals, especially antiretrovirals, are the main class of therapeutic agents evaluated against COVID-19. Moreover, studies have reported the anti-SARS-CoV-2 activity of antitumor (16%; 9/57), antimalarial (7%, 4/57), and antibacterial (5%; 3/57) agents. Additionally, seven pharmacological agents (chloroquine, tetrandrine, umifenovir (arbidol), carrimycin, damageprevir, lopinavir/ritonavir) are in phase IV of clinical trials. Due to the evidence of the anti-SARS-CoV-2 activity of various clinically available agents, drug repositioning stands out as a promising strategy for a short-term response in the fight against the novel coronavirus.\n",
      "Since December 2019 SARS‐Cov‐2 was found responsible for the disease COVID‐19, which has spread worldwide. No specific therapies/vaccines are yet available for the treatment of COVID‐19. Drug repositioning may offer a strategy and a number of drugs have been repurposed, including lopinavir/ritonavir, remdesivir, favipiravir and tocilizumab. This paper describes the main pharmacological properties of such drugs administered to patients with COVID‐19, focusing on their antiviral, immune‐modulatory and/or anti‐inflammatory actions. Where available, data from clinical trials involving patients with COVID‐19 are reported. Preliminary clinical trials seem to support their benefit. However, such drugs in COVID‐19 patients have peculiar safety profiles. Thus, adequate clinical trials are necessary for these compounds. Nevertheless, while waiting for effective preventive measures i.e. vaccines, many clinical trials on drugs belonging to different therapeutic classes are currently underway. Their results will help us in defining the best way to treat COVID‐19 and reducing its symptoms and complications.\n",
      "The coronavirus disease 2019 (COVID-19) pandemic caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has infected over 7.1 million people and led to over 0.4 million deaths. Currently, there is no specific anti-SARS-CoV-2 medication. New drug discovery typically takes more than 10 years. Drug repositioning becomes one of the most feasible approaches for combating COVID-19. This work curates the largest available experimental data set for SARS-CoV-2 or SARS-CoV 3CL (main) protease inhibitors. On the basis of this data set, we develop validated machine learning models with relatively low root-mean-square error to screen 1553 FDA-approved drugs as well as another 7012 investigational or off-market drugs in DrugBank. We found that many existing drugs might be potentially potent to SARS-CoV-2. The druggability of many potent SARS-CoV-2 3CL protease inhibitors is analyzed. This work offers a foundation for further experimental studies of COVID-19 drug repositioning.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: fcmzdcuh\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the initial symptoms of Covid-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what causes death from Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Background A recently developing pneumonia caused by SARS-CoV-2 was originated in Wuhan, China, and has quickly spread across the world. We reported the clinical characteristics of 82 death cases with COVID-19 in a single center. Methods Clinical data on 82 death cases laboratory-confirmed as SARS-CoV-2 infection were obtained from a Wuhan local hospital′s electronic medical records according to previously designed standardized data collection forms. Findings All patients were local residents of Wuhan, and the great proportion of them were diagnosed as severe illness when admitted. Most of the death cases were male (65.9%). More than half of dead patients were older than 60 years (80.5%) and the median age was 72.5 years. The bulk of death cases had comorbidity (76.8%), including hypertension (56.1%), heart disease (20.7%), diabetes (18.3%), cerebrovascular disease (12.2%), and cancer (7.3%). Respiratory failure remained the leading cause of death (69.5%), following by sepsis syndrome/MOF (28.0%), cardiac failure (14.6%), hemorrhage (6.1%), and renal failure (3.7%). Furthermore, respiratory, cardiac, hemorrhage, hepatic, and renal damage were found in 100%, 89%, 80.5%, 78.0%, and 31.7% of patients, respectively. On the admission, lymphopenia (89.2%), neutrophilia (74.3%), and thrombocytopenia (24.3%) were usually observed. Most patients had a high neutrophil-to-lymphocyte ratio of >5 (94.5%), high systemic immune-inflammation index of >500 (89.2%), increased C-reactive protein level (100%), lactate dehydrogenase (93.2%), and D-dimer (97.1%). A high level of IL-6 (>10 pg/ml) was observed in all detected patients. Median time from initial symptom to death was 15 days (IQR 11-20), and a significant association between aspartate aminotransferase (p=0.002), alanine aminotransferase (p=0.037) and time from initial symptom to death were interestingly observed. Conclusion Older males with comorbidities are more likely to develop severe disease, even die from SARS-CoV-2 infection. Respiratory failure is the main cause of COVID-19, but either virus itself or cytokine release storm mediated damage to other organ including cardiac, renal, hepatic, and hemorrhage should be taken seriously as well.\n",
      "A recently developed pneumonia caused by SARS-CoV-2 bursting in Wuhan, China, has quickly spread across the world. We report the clinical characteristics of 82 cases of death from COVID-19 in a single center. Clinical data on 82 death cases laboratory-confirmed as SARS-CoV-2 infection were obtained from a Wuhan local hospital's electronic medical records according to previously designed standardized data collection forms. All patients were local residents of Wuhan, and a large proportion of them were diagnosed with severe illness when admitted. Due to the overwhelming of our system, a total of 14 patients (17.1%) were treated in the ICU, 83% of deaths never received Critical Care Support, only 40% had mechanical ventilation support despite 100% needing oxygen and the leading cause of death being pulmonary. Most of the patients who died were male (65.9%). More than half of the patients who died were older than 60 years (80.5%), and the median age was 72.5 years. The bulk of the patients who died had comorbidities (76.8%), including hypertension (56.1%), heart disease (20.7%), diabetes (18.3%), cerebrovascular disease (12.2%), and cancer (7.3%). Respiratory failure remained the leading cause of death (69.5%), followed by sepsis/MOF (28.0%), cardiac failure (14.6%), hemorrhage (6.1%), and renal failure (3.7%). Furthermore, respiratory, cardiac, hemorrhagic, hepatic, and renal damage were found in 100%, 89%, 80.5%, 78.0%, and 31.7% of patients, respectively. On admission, lymphopenia (89.2%), neutrophilia (74.3%), and thrombocytopenia (24.3%) were usually observed. Most patients had a high neutrophil-to-lymphocyte ratio of >5 (94.5%), high systemic immune-inflammation index of >500 (89.2%), and increased C-reactive protein (100%), lactate dehydrogenase (93.2%), and D-dimer (97.1%) levels. A high level of IL-6 (>10 pg/ml) was observed in all detected patients. The median time from initial symptoms to death was 15 days (IQR 11-20), and a significant association between aspartate aminotransferase (p = 0.002), alanine aminotransferase (p = 0.037) and time from initial symptoms to death was remarkably observed. Older males with comorbidities are more likely to develop severe disease and even die from SARS-CoV-2 infection. Respiratory failure is the main cause of COVID-19, but the virus itself and cytokine release syndrome-mediated damage to other organs, including cardiac, renal, hepatic, and hemorrhagic damage, should be taken seriously as well.\n",
      "A recently developed pneumonia caused by SARS-CoV-2 bursting in Wuhan, China, has quickly spread across the world. We report the clinical characteristics of 82 cases of death from COVID-19 in a single center. Clinical data on 82 death cases laboratory-confirmed as SARS-CoV-2 infection were obtained from a Wuhan local hospital's electronic medical records according to previously designed standardized data collection forms. All patients were local residents of Wuhan, and a large proportion of them were diagnosed with severe illness when admitted. Due to the overwhelming of our system, a total of 14 patients (17.1%) were treated in the ICU, 83% of deaths never received Critical Care Support, only 40% had mechanical ventilation support despite 100% needing oxygen and the leading cause of death being pulmonary. Most of the patients who died were male (65.9%). More than half of the patients who died were older than 60 years (80.5%), and the median age was 72.5 years. The bulk of the patients who died had comorbidities (76.8%), including hypertension (56.1%), heart disease (20.7%), diabetes (18.3%), cerebrovascular disease (12.2%), and cancer (7.3%). Respiratory failure remained the leading cause of death (69.5%), followed by sepsis/MOF (28.0%), cardiac failure (14.6%), hemorrhage (6.1%), and renal failure (3.7%). Furthermore, respiratory, cardiac, hemorrhagic, hepatic, and renal damage were found in 100%, 89%, 80.5%, 78.0%, and 31.7% of patients, respectively. On admission, lymphopenia (89.2%), neutrophilia (74.3%), and thrombocytopenia (24.3%) were usually observed. Most patients had a high neutrophil-to-lymphocyte ratio of >5 (94.5%), high systemic immune-inflammation index of >500 (89.2%), and increased C-reactive protein (100%), lactate dehydrogenase (93.2%), and D-dimer (97.1%) levels. A high level of IL-6 (>10 pg/ml) was observed in all detected patients. The median time from initial symptoms to death was 15 days (IQR 11-20), and a significant association between aspartate aminotransferase (p = 0.002), alanine aminotransferase (p = 0.037) and time from initial symptoms to death was remarkably observed. Older males with comorbidities are more likely to develop severe disease and even die from SARS-CoV-2 infection. Respiratory failure is the main cause of COVID-19, but the virus itself and cytokine release syndrome-mediated damage to other organs, including cardiac, renal, hepatic, and hemorrhagic damage, should be taken seriously as well.\n",
      "Summary Background The pneumonia caused by the 2019 novel coronavirus (SARS-CoV-2) is a highly infectious disease, which was occurred in Wuhan, Hubei Province, China in December 2019. As of February 13, 2020, a total of 59883 cases of COVID-19 in China have been confirmed and 1368 patients have died from the disease. However, the clinical characteristics of the dyed patients were still not clearly clarified. This study aims to summarize the clinical characteristics of death cases with COVID-19 and to identify critically ill patients of COVID-19 early and reduce their mortality. Methods The clinical records, laboratory findings and radiologic assessments included chest X-ray or computed tomography were extracted from electronic medical records of 25 died patients with COVID-19 in Renmin Hospital of Wuhan University from Jan 14 to Feb 13, 2020. Two experienced clinicians reviewed and abstracted the data. Findings The mean age of the dead was 71.48 years, the average course of the disease was 10.56 days, all patients eventually died of respiratory failure. All of those who died had underlying diseases, the most common of which was hypertension (16/25, 64%), followed by diabetes (10/25, 40%), heart diseases (8/25, 32%), kidney diseases (5/25, 20%), cerebral infarction (4/25, 16%), chronic obstructive pulmonary disease (COPD, 2/25, 8%), malignant tumors (2/25, 8%) and acute pancreatitis (1/25, 4%). The most common organ damage outside the lungs was the heart, followed by kidney and liver. In the patients' last examination before death, white blood cell and neutrophil counts were elevated in 17 patients (17/25, 68%) and 18 patients (18/25, 72%), lymphocyte counts were decreased in 22 patients (22/25, 88%). Most patients' PCT, CRP and SAA levels were elevated, the percentages were 90.5% (19/21), 85% (19/20) and 100% (21/21) respectively. The levels of the last test of neutrophils (15/16, 93.8%), PCT (11/11, 100%), CRP (11/13, 84.6%), cTnI (8/9, 88.9%), D-Dimer (11/12, 91.6%) and LDH (9/9, 100%) were increased as compared to the first test, while the levels of lymphocytes were decreased (14/16, 87.5%). Interpretation The age and underlying diseases (hypertension, diabetes, etc.) were the most important risk factors for death of COVID-19 pneumonia. Bacterial infections may play an important role in promoting the death of patients. Malnutrition was common to severe patients. Multiple organ dysfunction can be observed, the most common organ damage was lung, followed by heart, kidney and liver. The rising of neutrophils, SAA, PCT, CRP, cTnI, D-Dimer and LDH levels can be used as indicators of disease progression, as well as the decline of lymphocytes counts.\n",
      "Background In December 2019, novel coronavirus (SARS-CoV-2) infected pneumonia occurred in Wuhan, China. The number of cases has increased rapidly but information on the clinical characteristics of SARS-CoV-2 pneumonia without comorbidities compared to normal controls in Chinese Han population is limited. Our objective is to describe the epidemiological and clinical characteristics of SARS-CoV-2 pneumonia without comorbidities compared to normal controls in the Chinese Han population. Methods Retrospective, multi-center case series of the 69 consecutive hospitalized patients with confirmed SARS-CoV-2 pneumonia, from February 7 to February 28, 2020; final date of follow-up was February 29, 2020. Results The study population included 69 hospitalized patients with confirmed SARS-CoV-2 pneumonia without comorbidities and 14,117 normal controls. 50.7% patients were male and 49.3% were female; 1.5% patients were asymptomatic cases, 63.8% patients were mild cases, and 36.2% patients were severe or critical cases. Compared with mild patients (n = 44), severe or critical patients (n = 25) were significantly older (median age, 67 years [IQR, 58-79] vs. 49 years [IQR, 36-60]; P < 0.01). Fever was present in 98.6% of the patients. The second most common symptom was cough (62.3%), fatigue (58.0%), sputum (39.1%), and headache (33.3%). The median incubation period was 4 days (IQR, 2 to 7). Leukocyte count was 74.1% of normal controls and lymphocyte count was 45.9% of normal controls. The phenomenon of lymphocyte depletion (PLD) observed in severe or critical cases in 100%. Levels of lactate dehydrogenase, D-dimer, procalcitonin, and interleukin-6 were showed significant differences between mild and severe or critical cases. Chest computed tomographic scans showed bilateral patchy patterns (49.3%), local patchy shadowing (29.0%), and ground glass opacity (21.7%). 7.3% patients were diagnosed ARDS, 7.3% patients were diagnosed acute cardiac injury (troponin I >28 pg/mL) and 4.4% patients were diagnosed fungal infections or shock. 4.3% patients have been discharged; 1.5% patient had died; 1.5% patient had recovery. Conclusions In this multicenter case series of 69 patients without comorbidities, the full spectrum of asymptomatic, mild, severe, and critical cases is described. 50.7% patients were male and 49.3% were female; 1.5% patients were asymptomatic cases, 63.8% patients were mild cases, and 36.2% patients were severe or critical cases. 4.3% patients have been discharged; 1.5% patient had died; 1.5% patient had recovery. Among the 25 patients with severe or critical disease, 12.0% patients were underwent non-invasive mechanical ventilation, 8.0% patients underwent invasive mechanical ventilation, and 4.0% patients died.\n",
      "BACKGROUND: In late December 2019, an outbreak of acute respiratory illness, coronavirus disease 2019 (COVID-19), emerged in Wuhan, China. We aimed to study the epidemiology, clinical features and short-term outcomes of patients with COVID-19 in Wuhan, China. METHODS: We performed a single center, retrospective case series study in 221 patients with laboratory confirmed SARS-CoV-2 pneumonia at a university hospital, including 55 severe patients and 166 non-severe patients, from January 2, 2020 to February 10, 2020. RESULTS: Of the 221 patients with COVID-19, the median age was 55.0 years and 48.9% were male and only 8 (3.6%) patients had a history of exposure to the Huanan Seafood Market. Compared to the non-severe pneumonia patients, the median age of the severe patients was significantly older, and they were more likely to have chronic comorbidities. Most common symptoms in severe patients were high fever, anorexia and dyspnea. On admission, 33.0% patients showed leukopenia and 73.8% showed lymphopenia. In addition, the severe patients suffered a higher rate of co-infections with bacteria or fungus and they were more likely to developing complications. As of February 15, 2020, 19.0% patients had been discharged and 5.4% patients died. 80% of severe cases received ICU (intensive care unit) care, and 52.3% of them transferred to the general wards due to relieved symptoms, and the mortality rate of severe patients in ICU was 20.5%. CONCLUSIONS: Patients with elder age, chronic comorbidities, blood leukocyte/lymphocyte count, procalcitonin level, co-infection and severe complications might increase the risk of poor clinical outcomes.\n",
      "Rationale: The current outbreak of coronavirus disease (COVID-19) pneumonia caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in Wuhan, China, spreads across national and international borders. The overall death rate of COVID-19 pneumonia in the Chinese population was 4%. Objectives: To describe the process of hospitalization and critical care of patients who died of COVID-19 pneumonia. Methods: This was a multicenter observational study of 109 decedents with COVID-19 pneumonia from three hospitals in Wuhan. Demographic, clinical, laboratory, and treatment data were collected and analyzed, and the final date of follow-up was February 24, 2020. Results: The mean age of 109 decedents with COVID-19 pneumonia was 70.7 years, 35 patients (32.1%) were female, and 85 patients (78.0%) suffered from one or more underlying comorbidities. Multiple organ failure, especially respiratory failure and heart failure, appeared in all patients even at the early stage of disease. Overall, the mean time from onset of symptoms to death was 22.3 days. All 109 hospitalized patients needed admission to an intensive care unit (ICU); however, because of limited availability, only 51 (46.8%) could be admitted. The period from hospitalization to death in the ICU group and non-ICU group was 15.9 days (standard deviation = 8.8 d) and 12.5 days (8.6 d, P = 0.044), respectively. Conclusions: Mortality due to COVID-19 pneumonia was concentrated in patients above the age of 65 years, especially those with major comorbidities. Patients who were admitted to the ICU lived longer than those who were not. Our findings should aid in the recognition and clinical management of such infections, especially with regard to ICU resource allocation.\n",
      "BACKGROUND: Since December, 2019, Wuhan, China, has experienced an outbreak of coronavirus disease 2019 (COVID-19), caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Epidemiological and clinical characteristics of patients with COVID-19 have been reported but risk factors for mortality and a detailed clinical course of illness, including viral shedding, have not been well described. METHODS: In this retrospective, multicentre cohort study, we included all adult inpatients (≥18 years old) with laboratory-confirmed COVID-19 from Jinyintan Hospital and Wuhan Pulmonary Hospital (Wuhan, China) who had been discharged or had died by Jan 31, 2020. Demographic, clinical, treatment, and laboratory data, including serial samples for viral RNA detection, were extracted from electronic medical records and compared between survivors and non-survivors. We used univariable and multivariable logistic regression methods to explore the risk factors associated with in-hospital death. FINDINGS: 191 patients (135 from Jinyintan Hospital and 56 from Wuhan Pulmonary Hospital) were included in this study, of whom 137 were discharged and 54 died in hospital. 91 (48%) patients had a comorbidity, with hypertension being the most common (58 [30%] patients), followed by diabetes (36 [19%] patients) and coronary heart disease (15 [8%] patients). Multivariable regression showed increasing odds of in-hospital death associated with older age (odds ratio 1·10, 95% CI 1·03–1·17, per year increase; p=0·0043), higher Sequential Organ Failure Assessment (SOFA) score (5·65, 2·61–12·23; p<0·0001), and d-dimer greater than 1 μg/mL (18·42, 2·64–128·55; p=0·0033) on admission. Median duration of viral shedding was 20·0 days (IQR 17·0–24·0) in survivors, but SARS-CoV-2 was detectable until death in non-survivors. The longest observed duration of viral shedding in survivors was 37 days. INTERPRETATION: The potential risk factors of older age, high SOFA score, and d-dimer greater than 1 μg/mL could help clinicians to identify patients with poor prognosis at an early stage. Prolonged viral shedding provides the rationale for a strategy of isolation of infected patients and optimal antiviral interventions in the future. FUNDING: Chinese Academy of Medical Sciences Innovation Fund for Medical Sciences; National Science Grant for Distinguished Young Scholars; National Key Research and Development Program of China; The Beijing Science and Technology Project; and Major Projects of National Science and Technology on New Drug Creation and Development.\n",
      "Background Coronavirus disease 2019 (COVID-19), caused by severe acute respiratory syndrome-coronavirus-2 (SARS-CoV2), is an ongoing pandemic that has already affected millions of patients worldwide, and is associated with significant morbidity and mortality burden. Although the clinical and laboratory characteristics of this illness have been reported in patients from China and Europe, data are scant in the United States. Methods We extracted data regarding all patients hospitalized at our hospital with COVID-19 infection between March 1 and April 4, 2020. Presenting signs and symptoms, laboratory and imaging findings, treatment, and complications were recorded from electronic medical records (EMRs). The primary composite endpoint was admission to intensive care unit (ICU), shock, or death. Results We had a total of 43 patients tested for COVID-19 at the emergency room (ER) or during hospitalization, 16 (37%) of whom were admitted with COVID-19 infection. The mean age was 65.5 years and 75% were males. The most common presenting symptoms were fever (94%), cough (88%), and dyspnea (81%). A loss of smell and taste sensations were reported by three (19%) patients. Low oxygen saturation was present in 38% of patients, whilst 31% were hypotensive on admission. Hyponatremia (50%), elevated C-reactive protein (CRP; 100%), and lactate dehydrogenase (LDH; 80%) were common. Acute renal failure, myocardial injury, and elevation in aminotransferases occurred in 69%, 19%, and 38% patients, respectively. The primary composite endpoint occurred in 50% of patients. A total of three patients died; all were aged 70 years or older. Conclusions Laboratory abnormalities and acute renal failure were common in hospitalized patients with SARS-CoV2 infection in our center. Admission to ICU and mechanical ventilation were common.\n",
      "Background Coronavirus disease 2019 (COVID-19), caused by severe acute respiratory syndrome-coronavirus-2 (SARS-CoV2), is an ongoing pandemic that has already affected millions of patients worldwide, and is associated with significant morbidity and mortality burden. Although the clinical and laboratory characteristics of this illness have been reported in patients from China and Europe, data are scant in the United States. Methods We extracted data regarding all patients hospitalized at our hospital with COVID-19 infection between March 1 and April 4, 2020. Presenting signs and symptoms, laboratory and imaging findings, treatment, and complications were recorded from electronic medical records (EMRs). The primary composite endpoint was admission to intensive care unit (ICU), shock, or death. Results We had a total of 43 patients tested for COVID-19 at the emergency room (ER) or during hospitalization, 16 (37%) of whom were admitted with COVID-19 infection. The mean age was 65.5 years and 75% were males. The most common presenting symptoms were fever (94%), cough (88%), and dyspnea (81%). A loss of smell and taste sensations were reported by three (19%) patients. Low oxygen saturation was present in 38% of patients, whilst 31% were hypotensive on admission. Hyponatremia (50%), elevated C-reactive protein (CRP; 100%), and lactate dehydrogenase (LDH; 80%) were common. Acute renal failure, myocardial injury, and elevation in aminotransferases occurred in 69%, 19%, and 38% patients, respectively. The primary composite endpoint occurred in 50% of patients. A total of three patients died; all were aged 70 years or older. Conclusions Laboratory abnormalities and acute renal failure were common in hospitalized patients with SARS-CoV2 infection in our center. Admission to ICU and mechanical ventilation were common.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: qvhwypjt\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what causes death from Covid-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Abstract Objectives This study aims to summarize the clinical characteristics of death cases with COVID-19 and to identify critically ill patients of COVID-19 early and reduce their mortality. Methods The clinical records, laboratory findings and radiological assessments included chest X-ray or computed tomography were extracted from electronic medical records of 25 died patients with COVID-19 in Renmin Hospital of Wuhan University from Jan 14 to Feb 13, 2020. Two experienced clinicians reviewed and abstracted the data. Results The age and underlying diseases (hypertension, diabetes, etc.) were the most important risk factors for death of COVID-19 pneumonia. Bacterial infections may play an important role in promoting the death of patients. Malnutrition was common to severe patients. Multiple organ dysfunction can be observed, the most common organ damage was lung, followed by heart, kidney and liver. The rising of neutrophils, SAA, PCT, CRP, cTnI, D-dimer, LDH and lactate levels can be used as indicators of disease progression, as well as the decline of lymphocytes counts. Conclusions The clinical characteristics of 25 death cases with COVID-19 we summarized, which would be helpful to identify critically ill patients of COVID-19 early and reduce their mortality.\n",
      "OBJECTIVES: This study aims to summarize the clinical characteristics of death cases with COVID-19 and to identify critically ill patients of COVID-19 early and reduce their mortality. METHODS: The clinical records, laboratory findings and radiological assessments included chest X-ray or computed tomography were extracted from electronic medical records of 25 died patients with COVID-19 in Renmin Hospital of Wuhan University from Jan 14 to Feb 13, 2020. Two experienced clinicians reviewed and abstracted the data. RESULTS: The age and underlying diseases (hypertension, diabetes, etc.) were the most important risk factors for death of COVID-19 pneumonia. Bacterial infections may play an important role in promoting the death of patients. Malnutrition was common to severe patients. Multiple organ dysfunction can be observed, the most common organ damage was lung, followed by heart, kidney and liver. The rising of neutrophils, SAA, PCT, CRP, cTnI, D-dimer, LDH and lactate levels can be used as indicators of disease progression, as well as the decline of lymphocytes counts. CONCLUSIONS: The clinical characteristics of 25 death cases with COVID-19 we summarized, which would be helpful to identify critically ill patients of COVID-19 early and reduce their mortality.\n",
      "Summary Background The pneumonia caused by the 2019 novel coronavirus (SARS-CoV-2) is a highly infectious disease, which was occurred in Wuhan, Hubei Province, China in December 2019. As of February 13, 2020, a total of 59883 cases of COVID-19 in China have been confirmed and 1368 patients have died from the disease. However, the clinical characteristics of the dyed patients were still not clearly clarified. This study aims to summarize the clinical characteristics of death cases with COVID-19 and to identify critically ill patients of COVID-19 early and reduce their mortality. Methods The clinical records, laboratory findings and radiologic assessments included chest X-ray or computed tomography were extracted from electronic medical records of 25 died patients with COVID-19 in Renmin Hospital of Wuhan University from Jan 14 to Feb 13, 2020. Two experienced clinicians reviewed and abstracted the data. Findings The mean age of the dead was 71.48 years, the average course of the disease was 10.56 days, all patients eventually died of respiratory failure. All of those who died had underlying diseases, the most common of which was hypertension (16/25, 64%), followed by diabetes (10/25, 40%), heart diseases (8/25, 32%), kidney diseases (5/25, 20%), cerebral infarction (4/25, 16%), chronic obstructive pulmonary disease (COPD, 2/25, 8%), malignant tumors (2/25, 8%) and acute pancreatitis (1/25, 4%). The most common organ damage outside the lungs was the heart, followed by kidney and liver. In the patients' last examination before death, white blood cell and neutrophil counts were elevated in 17 patients (17/25, 68%) and 18 patients (18/25, 72%), lymphocyte counts were decreased in 22 patients (22/25, 88%). Most patients' PCT, CRP and SAA levels were elevated, the percentages were 90.5% (19/21), 85% (19/20) and 100% (21/21) respectively. The levels of the last test of neutrophils (15/16, 93.8%), PCT (11/11, 100%), CRP (11/13, 84.6%), cTnI (8/9, 88.9%), D-Dimer (11/12, 91.6%) and LDH (9/9, 100%) were increased as compared to the first test, while the levels of lymphocytes were decreased (14/16, 87.5%). Interpretation The age and underlying diseases (hypertension, diabetes, etc.) were the most important risk factors for death of COVID-19 pneumonia. Bacterial infections may play an important role in promoting the death of patients. Malnutrition was common to severe patients. Multiple organ dysfunction can be observed, the most common organ damage was lung, followed by heart, kidney and liver. The rising of neutrophils, SAA, PCT, CRP, cTnI, D-Dimer and LDH levels can be used as indicators of disease progression, as well as the decline of lymphocytes counts.\n",
      "Background: Since mid-December 2019, a cluster of pneumonia-like diseases caused by a novel coronavirus, now designated COVID-19 by the WHO, emerged in Wuhan city and rapidly spread throughout China. Here we identify the clinical characteristics of COVID-19 in a cohort of patients in Shanghai. Methods: Cases were confirmed by real-time RT-PCR and were analysed for demographic, clinical, laboratory and radiological features. Results: Of 198 patients, the median duration from disease onset to hospital admission was 4 days. The mean age of the patients was 50.1 years, and 51.0% patients were male. The most common symptom was fever. Less than half of the patients presented with respiratory systems including cough, sputum production, itchy or sore throat, shortness of breath, and chest congestion. 5.6% patients had diarrhoea. On admission, T lymphocytes were decreased in 45.8% patients. Ground glass opacity was the most common radiological finding on chest computed tomography. 9.6% were admitted to the ICU because of the development of organ dysfunction. Compared with patients not treated in ICU, patients treated in the ICU were older, had longer waiting time to admission, fever over 38.5o C, dyspnoea, reduced T lymphocytes, elevated neutrophils and organ failure. Conclusions: In this single centre cohort of COVID-19 patients, the most common symptom was fever, and the most common laboratory abnormality was decreased blood T cell counts. Older age, male, fever over 38.5oC, symptoms of dyspnoea, and underlying comorbidity, were the risk factors most associated with severity of disease. Key words: 2019 novel coronavirus; acute respiratory infection; risk factors for disease severity\n",
      "OBJECTIVE: This retrospective study aimed to analysis the clinical characteristics and complications in death cases with novel coronavirus disease-19 (COVID-19). METHOD: We collected the medical records of 92 patients with COVID-19 in Renmin Hospital of Wuhan University who died during January 6th to February 25th, 2020, summarized the clinical characteristics of complications. RESULTS: There were 91 death cases who developed different complications including acute respiratory distress syndrome (ARDS) (73/91), myocardial injury (31/91), liver injury (15/91), renal insufficiency (14/91), multiple organ dysfunction syndrome (MODS) (14/91) and pneumothorax (1/91). Among these patients, 83 patients had at least one complication. While 1 patient who died of recurrent gastrointestinal bleeding was not directly linked to COVID-19. CONCLUSION: The main complications of deceased patients with COVID-19 were ARDS, myocardial injury, liver injury, renal insufficiency and MODS. This article is protected by copyright. All rights reserved.\n",
      "OBJECTIVE: This retrospective study aimed to analysis the clinical characteristics and complications in death cases with novel coronavirus disease‐19 (COVID‐19). METHOD: We collected the medical records of 92 patients with COVID‐19 in Renmin Hospital of Wuhan University who died during January 6th to February 25th, 2020, summarized the clinical characteristics of complications. RESULTS: There were 91 death cases who developed different complications including acute respiratory distress syndrome (ARDS) (73/91), myocardial injury (31/91), liver injury (15/91), renal insufficiency (14/91), multiple organ dysfunction syndrome (MODS) (14/91) and pneumothorax (1/91). Among these patients, 83 patients had at least one complication. While 1 patient who died of recurrent gastrointestinal bleeding was not directly linked to COVID‐19. CONCLUSION: The main complications of deceased patients with COVID‐19 were ARDS, myocardial injury, liver injury, renal insufficiency and MODS. This article is protected by copyright. All rights reserved.\n",
      "Summary Objective To investigate the characteristics and prognostic factors in the elderly patients with COVID-19. Methods Consecutive cases over 60 years old with COVID-19 in Renmin Hospital of Wuhan University from Jan 1 to Feb 6, 2020 were included. The primary outcomes were death and survival till March 5. Data of demographics, clinical features, comorbidities, laboratory tests and complications were collected and compared for different outcomes. Cox regression was performed for prognostic factors. Results 339 patients with COVID-19 (aged 71±8 years,173 females (51%)) were enrolled, including 80 (23.6%) critical, 159 severe (46.9%) and 100 moderate (29.5%) cases. Common comorbidities were hypertension (40.8%), diabetes (16.0%) and cardiovascular disease (15.7%). Common symptoms included fever (92.0%), cough (53.0%), dyspnea (40.8%) and fatigue (39.9%). Lymphocytopenia was a common laboratory finding (63.2%). Common complications included bacterial infection (42.8%), liver enzyme abnormalities (28.7%) and acute respiratory distress syndrome (21.0%). Till Mar 5, 2020, 91 cases were discharged (26.8%), 183 cases stayed in hospital (54.0%) and 65 cases (19.2%) were dead. Shorter length of stay was found for the dead compared with the survivors (5 (3–8) vs. 28 (26–29), P < 0.001). Symptoms of dyspnea (HR 2.35, P = 0.001), comorbidities including cardiovascular disease (HR 1.86, P = 0.031) and chronic obstructive pulmonary disease (HR 2.24, P = 0.023), and acute respiratory distress syndrome (HR 29.33, P < 0.001) were strong predictors of death. And a high level of lymphocytes was predictive of better outcome (HR 0.10, P < 0.001). Conclusions High proportion of severe to critical cases and high fatality rate were observed in the elderly COVID-19 patients. Rapid disease progress was noted in the dead with a median survival time of 5 days after admission. Dyspnea, lymphocytopenia, comorbidities including cardiovascular disease and chronic obstructive pulmonary disease, and acute respiratory distress syndrome were predictive of poor outcome. Close monitoring and timely treatment should be performed for the elderly patients at high risk.\n",
      "Background A recently developing pneumonia caused by SARS-CoV-2 was originated in Wuhan, China, and has quickly spread across the world. We reported the clinical characteristics of 82 death cases with COVID-19 in a single center. Methods Clinical data on 82 death cases laboratory-confirmed as SARS-CoV-2 infection were obtained from a Wuhan local hospital′s electronic medical records according to previously designed standardized data collection forms. Findings All patients were local residents of Wuhan, and the great proportion of them were diagnosed as severe illness when admitted. Most of the death cases were male (65.9%). More than half of dead patients were older than 60 years (80.5%) and the median age was 72.5 years. The bulk of death cases had comorbidity (76.8%), including hypertension (56.1%), heart disease (20.7%), diabetes (18.3%), cerebrovascular disease (12.2%), and cancer (7.3%). Respiratory failure remained the leading cause of death (69.5%), following by sepsis syndrome/MOF (28.0%), cardiac failure (14.6%), hemorrhage (6.1%), and renal failure (3.7%). Furthermore, respiratory, cardiac, hemorrhage, hepatic, and renal damage were found in 100%, 89%, 80.5%, 78.0%, and 31.7% of patients, respectively. On the admission, lymphopenia (89.2%), neutrophilia (74.3%), and thrombocytopenia (24.3%) were usually observed. Most patients had a high neutrophil-to-lymphocyte ratio of >5 (94.5%), high systemic immune-inflammation index of >500 (89.2%), increased C-reactive protein level (100%), lactate dehydrogenase (93.2%), and D-dimer (97.1%). A high level of IL-6 (>10 pg/ml) was observed in all detected patients. Median time from initial symptom to death was 15 days (IQR 11-20), and a significant association between aspartate aminotransferase (p=0.002), alanine aminotransferase (p=0.037) and time from initial symptom to death were interestingly observed. Conclusion Older males with comorbidities are more likely to develop severe disease, even die from SARS-CoV-2 infection. Respiratory failure is the main cause of COVID-19, but either virus itself or cytokine release storm mediated damage to other organ including cardiac, renal, hepatic, and hemorrhage should be taken seriously as well.\n",
      "BACKGROUND: Since December 2019, the 2019 coronavirus disease (COVID-19) has expanded to cause a worldwide outbreak that more than 600,000 people infected and tens of thousands died. To date, the clinical characteristics of COVID-19 patients in the non-Wuhan areas of Hubei Province in China have not been described. METHODS: We retrospectively analyzed the clinical characteristics and treatment progress of 91 patients diagnosed with COVID-19 in Jingzhou Central Hospital. RESULTS: Of the 91 patients diagnosed with COVID-19, 30 cases (33.0%) were severe and two patients (2.2%) died. The severe disease group tended to be older (50.5 vs. 42.0 years; p = 0.049) and have more chronic disease (40% vs. 14.8%; p = 0.009) relative to mild disease group. Only 73.6% of the patients were quantitative polymerase chain reaction (qPCR)-positive on their first tests, while typical chest computed tomography images were obtained for each patient. The most common complaints were cough (n = 75; 82.4%), fever (n = 59; 64.8%), fatigue (n = 35; 38.5%), and diarrhea (n = 14; 15.4%). Non-respiratory injury was identified by elevated levels of aspartate aminotransferase (n = 18; 19.8%), creatinine (n = 5; 5.5%), and creatine kinase (n = 14; 15.4%) in laboratory tests. Twenty-eight cases (30.8%) suffered non-respiratory injury, including 50% of the critically ill patients and 21.3% of the mild patients. CONCLUSIONS: Overall, the mortality rate of patients in Jingzhou was lower than that of Wuhan. Importantly, we found liver, kidney, digestive tract, and heart injuries in COVID-19 cases besides respiratory problems. Combining chest computed tomography images with the qPCR analysis of throat swab samples can improve the accuracy of COVID-19 diagnosis.\n",
      "OBJECTIVES: To compare clinical, laboratory, and chest computed tomography (CT) findings in critically ill patients diagnosed with coronavirus disease 2019 (COVID-19) who survived and who died. METHODS: This retrospective study reviewed 60 critically ill patients (43 males and 17 females, mean age 64.4 ± 11.0 years) with COVID-19 pneumonia who were admitted to two different clinical centers. Their clinical and medical records were analyzed, and the chest CT images were assessed to determine the involvement of lobes and the distribution of lesions in the lungs between the patients who recovered from the illness and those who died. RESULTS: Compared with recovered patients (50/60, 83%), deceased patients (10/60, 17%) were older (mean age, 70.6 vs. 62.6 years, p = 0.044). C-reactive protein (CRP) (110.8 ± 26.3 mg/L vs 63.0 ± 50.4 mg/L, p < 0.001) and neutrophil-to-lymphocyte ratio (NLR) (18.7 ± 16.6 vs 8.4 ± 7.5, p = 0.030) were significantly elevated in the deceased as opposed to the recovered. Medial or parahilar area involvement was observed in all the deceased patients (10/10, 100%), when compared to only 54% (27/50) in the recovered. Ground-glass opacities (97%), crazy-paving pattern (92%), and air bronchogram (93%) were the most common radiological findings. There was significant difference in diabetes (p = 0.025) and emphysema (p = 0.013), and the odds ratio on a deceased patient having diabetes and emphysema was 6 times and 21 times the odds ratio on a recovered patient having diabetes and emphysema, respectively. CONCLUSIONS: Older patients with comorbidities such as diabetes and emphysema, and higher CRP and NLRs with diffuse lung involvement were more likely to die of COVID-19. KEY POINTS: • Almost all patients critically ill with COVID-19 pneumonia had five lung lobes involved. • Medial or parahilar area involvement and degree of lung involvement were more serious in the deceased patients when compared with those who recovered from treatment. • Chronic lung disease, e.g., emphysema, diabetes, and higher serum CRP and NLR characterized patients who died of COVID-19.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 2y452utz\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "A novel coronavirus strain 2019-nCoV has caused a rapid global pandemic-COVID-19. Scientists have taken onto the task of characterizing this new virus and understanding how this virus has transmitted to humans. All preliminary studies have found some striking similarities between this new virus and the SARS-CoV that caused a similar kind of epidemic in 2002–2003. Through bioinformatics tools, a great deal of information has been gathered about the origin, evolution and zoonosis of this virus. We, in this review, report the symptoms, mode of transmission and available and putative treatments to tackle 2019-nCoV infections. We also comprehensively summarize all the information so far made available regarding the genome, evolution and zoonosis of this virus.\n",
      "BACKGROUND: The rapid spread of the coronavirus disease 2019 (COVID-19), caused by a zoonotic beta-coronavirus entitled 2019 novel coronavirus (2019-nCoV), has become a global threat. Awareness of the biological features of 2019-nCoV should be updated in time and needs to be comprehensively summarized to help optimize control measures and make therapeutic decisions. METHODS: Based on recently published literature, official documents and selected up-to-date preprint studies, we reviewed the virology and origin, epidemiology, clinical manifestations, pathology and treatment of 2019-nCoV infection, in comparison with severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV) infection. RESULTS: The genome of 2019-nCoV partially resembled SARS-CoV and MERS-CoV, and indicated a bat origin. The COVID-19 generally had a high reproductive number, a long incubation period, a short serial interval and a low case fatality rate (much higher in patients with comorbidities) than SARS and MERS. Clinical presentation and pathology of COVID-19 greatly resembled SARS and MERS, with less upper respiratory and gastrointestinal symptoms, and more exudative lesions in post-mortems. Potential treatments included remdesivir, chloroquine, tocilizumab, convalescent plasma and vaccine immunization (when possible). CONCLUSION: The initial experience from the current pandemic and lessons from the previous two pandemics can help improve future preparedness plans and combat disease progression.\n",
      "Abstract Background The rapid spread of the coronavirus disease 2019 (COVID-19), caused by a zoonotic beta-coronavirus entitled 2019 novel coronavirus (2019-nCoV), has become a global threat. Awareness of the biological features of 2019-nCoV should be updated in time and needs to be comprehensively summarized to help optimize control measures and make therapeutic decisions. Methods Based on recently published literatures, official documents and selected up-to-date preprint studies, we reviewed the virology and origin, epidemiology, clinical manifestations, pathology and treatment of 2019-nCoV infection, in comparison with severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV) infection. Results The genome of 2019-nCoV partially resembled SARS-CoV and MERS-CoV, and indicating a bat origin. The COVID-19 generally had a high reproductive number, a long incubation period, a short serial interval and a low case fatality rate (much higher in patients with comorbidities) than SARS and MERS. Clinical presentation and pathology of COVID-19 greatly resembled SARS and MERS, with less upper respiratory and gastrointestinal symptoms, and more exudative lesions in post-mortems. Potential treatments included remdesivir, chloroquine, tocilizumab, convalescent plasma and vaccine immunization (when possible). Conclusion The initial experience from the current pandemic and lessons from the previous two pandemics can help improve future preparedness plans and combat disease progression.\n",
      "The novel coronavirus (2019-nCoV) outbreak, which initially began in China, has spread to many countries around the globe, with the number of confirmed cases increasing every day. With a death toll exceeding that of the SARS-CoV outbreak back in 2002 and 2003 in China, 2019-nCoV has led to a public health emergency of international concern, putting all health organizations on high alert. Herein, we present on an overview of the currently available information on the pathogenesis, epidemiology, clinical presentation, diagnosis, and treatment of this virus.\n",
      "The emerging coronavirus disease (COVID-19) swept across the world, affecting more than 200 countries and territories. Genomic analysis suggests that the COVID-19 virus originated in bats and transmitted to humans through unknown intermediate hosts in the Wuhan seafood market, China, in December of 2019. This virus belongs to the Betacoronavirus group, the same group of the 2003 severe acute respiratory syndrome coronavirus (SARS-CoV), and for the similarity, it was named SARS-CoV-2. Given the lack of registered clinical therapies or vaccines, many physicians and scientists are investigating previously used clinical drugs for COVID-19 treatment. In this review, we aim to provide an overview of the CoVs origin, pathogenicity, and genomic structure, with a focus on SARS-CoV-2. Besides, we summarize the recently investigated drugs that constitute an option for COVID-19 treatment.\n",
      "There is rising global concern for the recently emerged novel Coronavirus (2019-nCov). Full genomic sequences have been released by the worldwide scientific community in the last few weeks in order to understand the evolutionary origin and molecular characteristics of this virus. Taking advantage of all the genomic information currently available, we constructed a phylogenetic tree including also representatives of other coronaviridae, such as Bat coronavirus (BCoV) and SARS. We confirm high sequence similarity (>99%) between all sequenced 2019-nCoVs genomes available, with the closest BCoV sequence sharing 96.2% sequence identity, confirming the notion of a zoonotic origin of 2019-nCoV. Despite the low heterogeneity of the 2019-nCoV genomes, we could identify at least two hyper-variable genomic hotspots, one of which is responsible for a Serine/Leucine variation in the viral ORF8-encoded protein. Finally, we perform a full proteomic comparison with other coronaviridae, identifying key aminoacidic differences to be considered for antiviral strategies deriving from previous anti-coronavirus approaches.\n",
      "There is a rising global concern for the recently emerged novel coronavirus (2019-nCoV). Full genomic sequences have been released by the worldwide scientific community in the last few weeks to understand the evolutionary origin and molecular characteristics of this virus. Taking advantage of all the genomic information currently available, we constructed a phylogenetic tree including also representatives of other coronaviridae, such as Bat coronavirus (BCoV) and severe acute respiratory syndrome. We confirm high sequence similarity (>99%) between all sequenced 2019-nCoVs genomes available, with the closest BCoV sequence sharing 96.2% sequence identity, confirming the notion of a zoonotic origin of 2019-nCoV. Despite the low heterogeneity of the 2019-nCoV genomes, we could identify at least two hypervariable genomic hotspots, one of which is responsible for a Serine/Leucine variation in the viral ORF8-encoded protein. Finally, we perform a full proteomic comparison with other coronaviridae, identifying key aminoacidic differences to be considered for antiviral strategies deriving from previous anti-coronavirus approaches.\n",
      "There is a rising global concern for the recently emerged novel coronavirus (2019‐nCoV). Full genomic sequences have been released by the worldwide scientific community in the last few weeks to understand the evolutionary origin and molecular characteristics of this virus. Taking advantage of all the genomic information currently available, we constructed a phylogenetic tree including also representatives of other coronaviridae, such as Bat coronavirus (BCoV) and severe acute respiratory syndrome. We confirm high sequence similarity (>99%) between all sequenced 2019‐nCoVs genomes available, with the closest BCoV sequence sharing 96.2% sequence identity, confirming the notion of a zoonotic origin of 2019‐nCoV. Despite the low heterogeneity of the 2019‐nCoV genomes, we could identify at least two hypervariable genomic hotspots, one of which is responsible for a Serine/Leucine variation in the viral ORF8‐encoded protein. Finally, we perform a full proteomic comparison with other coronaviridae, identifying key aminoacidic differences to be considered for antiviral strategies deriving from previous anti‐coronavirus approaches.\n",
      "The COVID-19 pandemic is due to infection caused by the novel SARS-CoV-2 virus that impacts the lower respiratory tract. The spectrum of symptoms ranges from asymptomatic infections to mild respiratory symptoms to the lethal form of COVID-19 which is associated with severe pneumonia, acute respiratory distress, and fatality. To address this global crisis, up-to-date information on viral genomics and transcriptomics is crucial for understanding the origins and global dispersion of the virus, providing insights into viral pathogenicity, transmission, and epidemiology, and enabling strategies for therapeutic interventions, drug discovery, and vaccine development. Therefore, this review provides a comprehensive overview of COVID-19 epidemiology, genomic etiology, findings from recent transcriptomic map analysis, viral-human protein interactions, molecular diagnostics, and the current status of vaccine and novel therapeutic intervention development. Moreover, we provide an extensive list of resources that will help the scientific community access numerous types of databases related to SARS-CoV-2 OMICs and approaches to therapeutics related to COVID-19 treatment.\n",
      "The emergence of the SARS‐CoV‐2 strain of the human coronavirus has thrown the world into the midst of a new pandemic. In the human body, the virus causes COVID‐19, a disease characterized by shortness of breath, fever, and pneumonia, which can be fatal in vulnerable individuals. SARS‐CoV‐2 has characteristics of past human coronaviruses, with close genomic similarities to SARS‐CoV, the virus that causes the disease SARS. Like these related coronaviruses, SARS‐CoV‐2 is transmitted through the inhalation of droplets and interaction with contaminated surfaces. Across the world, laboratories are developing candidate vaccines for the virus – with vaccine trials underway in the US and the United Kingdom ‐ and considering various drugs for possible treatments and prophylaxis. Here, we provide an overview of SARS‐CoV‐2 by analyzing its virology, epidemiology, and modes of transmission while examining the current progress of testing procedures and possible treatments through drugs and vaccines.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 6k5ac3f2\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "In this review, we focused on the origins of the novel coronavirus (SARS-CoV-2), origin, pathogenesis, immune responses, genes and genetic variations, phylogenetic analyses, and potential therapeutic strategies to summarize approaches for developing broadly effective preventions and vaccines to cope COVID-19. Towards the end of 2019, SARS-CoV-2 has emerged in association with the SARS, later was named COVID-19 caused an environment of chaos worldwide and infected a massive number of lives. Since these epidemics or pandemics had spread to 210 countries and territories around the world and 2 international conveyances with 6,467,229 confirmed cases, including, 382,766 deaths, as of June 03, 2020 (https://www.worldometers.info/coronavirus/), hence the World Health Organization declared it as a global Public Health Emergency. There are no clinically approved vaccines or antiviral drugs available for either of new or old corona infections; thus, the development of effective therapeutic and preventive strategies that can be readily available to cope with these strains.\n",
      "INTRODUCTION: The Coronavirus disease-19 (COVID-19) caused by the novel beta coronavirus named severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) started in late December 2019 in Wuhan, China. Within a short span, COVID-19 was declared a global public health emergency affecting 214 countries with 5,939,234 confirmed cases and 3,67,255 deaths as of 30 May 2020. With limited knowledge about SARS-CoV-2, no approved treatment or vaccine is available till date. AREAS COVERED: We performed a review of literature on PubMed on the SARS-CoV-2 virus and COVID-19 illness including trials of preventive and therapeutic measures. This review presents the basic biology of coronaviruses, epidemiology of COVID-19, clinical presentations, investigational therapies and vaccines, infection prevention and control measures and the lessons from the present pandemic. EXPERT OPINION: The scale of the outbreak has brought the governments, health-care professionals, and scientists around the world under tremendous pressure to devise control strategies and develop novel prevention measures. While availability of vaccine for COVID-19 may take time, the disease may be contained through hand hygiene, physical distancing, travel restriction, and aggressive steps such as 'lockdown.' Clinical trials at different phases are ongoing across different countries to expedite the development of effective drugs and vaccine to overcome the pandemic.\n",
      "INTRODUCTION: The Coronavirus disease-19 (COVID-19) caused by the novel beta coronavirus named Severe Acute Respiratory Syndromecoronavirus-2 (SARS-CoV-2) started in late December 2019 in Wuhan, China. Within a short span, COVID-19 was declared a global public health emergency affecting 214 countries with 5,939,234 confirmed cases and 3,67,255 deaths as of 30 May, 2020. With limited knowledge about SARS-CoV-2, no approved treatment or vaccine is available till date. AREAS COVERED: We performed a review of literature on PubMed on the SARS-CoV-2 virus and COVID-19 illness including trials of preventive and therapeutic measures. This review presents the basic biology of coronaviruses, epidemiology of COVID-19, clinical presentations, investigational therapies and vaccines, infection prevention and control measures and the lessons from the present pandemic. EXPERT OPINION: The scale of the outbreak has brought the governments, healthcare professionals and scientists around the world under tremendous pressure to devise control strategies and develop novel prevention measures. While availability of vaccine for COVID-19 may take time, the disease may be contained through hand hygiene, physical distancing, travel restriction and aggressive steps such as 'lockdown'. Clinical trials at different phases are ongoing across different countries to expedite development of effective drugs and vaccine to overcome the pandemic.\n",
      "Coronavirus Disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has been reported as a worldwide emergency. Due to the extensiveness of spread and death, it has been declared as a pandemic. This review focused on the current pandemic situation and understanding the prevention and control strategies of COVID-19. Data presented here was by April 3, 2020. A total of 1,016,399 cases of COVID-19 with 53,238 deaths was reported from 204 countries and territories including two international conveyances over the world. After China, most of the new cases were from Europe, particularly Italy acting as the source of importation to many of the other countries around the world. China has obtained success by ascribing control strategies against COVID-19. The implementation of China’s strategy, as well as the development of a vaccine, may control the pandemic of COVID-19. Further robust studies are required for a clear understanding of transmission parameters, prevention, and control strategies of SARS-CoV-2. This review paper describes the nature of COVID-19 and the possible ways for the effective controlling of the COVID-19 or similar viral diseases that may come in the future.\n",
      "INTRODUCTION The Coronavirus disease-19 (COVID-19) caused by the novel beta coronavirus named Severe Acute Respiratory Syndromecoronavirus-2 (SARS-CoV-2) started in late December 2019 in Wuhan, China. Within a short span, COVID-19 was declared a global public health emergency affecting 214 countries with 5,939,234 confirmed cases and 3,67,255 deaths as of 30 May, 2020. With limited knowledge about SARS-CoV-2, no approved treatment or vaccine is available till date. AREAS COVERED We performed a review of literature on PubMed on the SARS-CoV-2 virus and COVID-19 illness including trials of preventive and therapeutic measures. This review presents the basic biology of coronaviruses, epidemiology of COVID-19, clinical presentations, investigational therapies and vaccines, infection prevention and control measures and the lessons from the present pandemic. EXPERT OPINION The scale of the outbreak has brought the governments, healthcare professionals and scientists around the world under tremendous pressure to devise control strategies and develop novel prevention measures. While availability of vaccine for COVID-19 may take time, the disease may be contained through hand hygiene, physical distancing, travel restriction and aggressive steps such as 'lockdown'. Clinical trials at different phases are ongoing across different countries to expedite development of effective drugs and vaccine to overcome the pandemic.\n",
      "Abstract In late December 2019, the world woke to a reality of a pandemic of Coronavirus Disease (COVID-19), elicited by the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), which belongs to a group of β-coronavirus. The potential to cause life-threatening respiratory failure and rapid transmission puts COVID-19 in the list of Public Health Emergency of International Concern (PHEIC). In the last two decades, this is the 3rd deadliest Coronavirus pandemic, following SARS which lasted between 2002 and 2003 and Middle East Respiratory Syndrome (MERS) from 2012 till date. Globally and as of April 1st 2020, COVID-19 has affected 924,688 individuals in over 200 countries, causing 46,368 fatalities. While the SARS-CoV-2 originated in China with over 82,724 confirmed cases and more than 3000 deaths as at the time of writing this review, the rapid transmission of SARS-CoV-2 has resulted in exponential increase in the number of cases outside of China to about 10 times the report case and death in mainland China. SARS-CoV-2 is suspected to be zoonotic in nature as genetic studies have shown sequence similarity to viruses originating from bats. Extreme precautionary measures, such as curfew, shutting of borders and quarantining of individuals suspected to be infected have been instituted with immediate effect; however, due to individuals that are asymptomatic, uncontrolled human-to-human transmission has resulted in exponential infection rate and numerous loss of lives even with this lockdown measures. This review article summarizes the developing situation surrounding the SARS-CoV-2 pandemic with respect to its epidemiology, unique genomic structure, possible origins, transmission, pathogenesis, comparison with other deadly species of Coronaviruses (CoV) and emerging treatment strategies built on informed literature.\n",
      "In late December 2019, an unprecedented outbreak of coronavirus disease 2019 (COVID-19) caused by SARS coronavirus 2 (SARS-CoV-2) (previously named 2019-nCoV) in Wuhan became the most challenging health emergency. Since its rapid spread in China and many other countries, the World Health Organization (WHO) declared COVID-19 a public health emergency of international concern (PHEIC) on 30th January 2020 and a pandemic on 11th March 2020. Thousands of people have died, and there are currently no vaccines or specific antiviral drugs for COVID-19. Therefore, it is critical to have a comprehensive understanding of the virus. In this review, we highlight the etiology, epidemiology, pathogenesis and pathology, clinical characteristics, diagnosis, clinical management, prognosis, infection control and prevention of COVID-19 based on recent studies.\n",
      "BACKGROUND Emerging infectious diseases pose a great hazard to public health and the global economy. A novel coronavirus, SARS-CoV-2, causing coronavirus disease 2019 (COVID-19), emerged from Wuhan, China, in December 2019. Since then, it has spread globally causing a global health emergency of inconceivable magnitude, with significant morbidity and mortality. There is the need for clinicians and health care providers to understand and gain knowledge of the different aspects of the disease as it evolves. OBJECTIVE We reviewed existing literature on COVID-19 in order to present a synopsis of current understanding of the disease. METHODS Using PubMed, Embase and Medline, articles published between January and April 2020 on Coronavirus disease 2019 (COVID-19), and resources from World Health Organisation were analyzed in order to have an understanding of the clinical characteristics, transmission dynamics, virology, diagnostic possibilities, prevention, management approach, controversies and impact of COVID-19 on the healthcare workforce. RESULTS Our review revealed that COVID-19 patients present with a myriad of symptoms, ranging from mild to severe respiratory disease, which can often be fatal. The mode of transmission via droplet infections makes it unduly contagious and difficult to control. Currently, antiviral drugs or vaccines are being developed to manage this condition. CONCLUSION There is urgent need for potent antivirals and vaccine to manage those infected with COVID 19 and prevent infection. Global efforts need to be unified in combating this public health emergency and flattening the curve of spread.\n",
      "SARS-Coronavirus-2 (SARS-CoV-2) causes Coronavirus disease 2019 (COVID-19), an infectious respiratory disease causing thousands of deaths and overwhelming public health systems. The international spread of SARS-CoV-2 is associated with the ease of global travel, and societal dynamics, immunologic naiveté of the host population, and muted innate immune responses. Based on these factors and the expanding geographic scale of the disease, the World Health Organization (WHO) declared the COVID-19 outbreak a pandemic-the first caused by a coronavirus. In this review, we summarize the current epidemiological status of COVID-19 and consider the virological and immunological lessons, animal models, and tools developed in response to prior SARS-CoV and MERS-CoV outbreaks that can serve as resources for development of SARS-CoV-2 therapeutics and vaccines. In particular, we discuss structural insights into the SARS-CoV-2 spike protein, a major determinant of transmissibility, and discuss key molecular aspects that will aid in understanding and fighting this new global threat.\n",
      "Coronavirus disease 2019 (COVID-19) originated in the city of Wuhan, Hubei Province, Central China, and has spread quickly to 72 countries to date. COVID-19 is caused by a novel coronavirus, named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) [previously provisionally known as 2019 novel coronavirus (2019-nCoV)]. At present, the newly identified SARS-CoV-2 has caused a large number of deaths with tens of thousands of confirmed cases worldwide, posing a serious threat to public health. However, there are no clinically approved vaccines or specific therapeutic drugs available for COVID-19. Intensive research on the newly emerged SARS-CoV-2 is urgently needed to elucidate the pathogenic mechanisms and epidemiological characteristics and to identify potential drug targets, which will contribute to the development of effective prevention and treatment strategies. Hence, this review will focus on recent progress regarding the structure of SARS-CoV-2 and the characteristics of COVID-19, such as the aetiology, pathogenesis and epidemiological characteristics.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: qp4efhwq\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what is the origin of COVID-19\n",
      "how long can the coronavirus live outside the body\n",
      "What vaccine candidates are being tested for Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Coronavirus disease (COVID‐19) is now dominating the lives of everyone, and its history is constantly being rewritten. This article gives a brief account of the story so far: where SARS‐CoV‐2 might have originated, how it compares with other viruses that cause major respiratory disease, and some of the treatments and vaccines currently being investigated to combat it.\n",
      "The article presents some introductory remarks on viruses and in general how they lead to diseases. With reference to same, it discusses the emergence of a fairly novel coronavirus, 2019-CoV-2 and its possible medication.\n",
      "In December 2019, an outbreak of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) was reported in Wuhan, China, and it subsequently spread in many countries around the world. Many efforts have been applied to control and prevent the spread of COVID-19, and many scientific studies have been conducted in a short period of time. Here we present an overview of the viral structure, pathogenesis, diagnosis, and clinical features of COVID-19 based on the current state of knowledge, and we compare its clinical characteristics with SARS and Middle East Respiratory Syndrome (MERS). Current researches on potentially effective treatment alternatives are discussed. We hope this review can help medical workers and researchers around the world contain the current COVID-19 pandemic.\n",
      "In December 2019, an outbreak of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) was reported in Wuhan, China, and it subsequently spread in many countries around the world. Many efforts have been applied to control and prevent the spread of COVID-19, and many scientific studies have been conducted in a short period of time. Here we present an overview of the viral structure, pathogenesis, diagnosis, and clinical features of COVID-19 based on the current state of knowledge, and we compare its clinical characteristics with SARS and Middle East Respiratory Syndrome (MERS). Current researches on potentially effective treatment alternatives are discussed. We hope this review can help medical workers and researchers around the world contain the current COVID-19 pandemic.\n",
      "Summary The coronavirus disease-2019 (COVID-19), a contagious disease caused by severe acute respiratory syndrome coronavirus-2 (SARS-CoV2), has reached pandemic status. As it spreads across the world, it has overwhelmed healthcare systems, strangled the global economy and led to a devastating loss of life. Widespread efforts from regulators, clinicians and scientists are driving a rapid expansion of knowledge of the SARS-CoV2 virus and the COVID-19 disease. We review the most current data with a focus on our basic understanding of the mechanism(s) of disease and translation to the clinical syndrome and potential therapeutics. We discuss the basic virology, epidemiology, clinical manifestation, multi-organ consequences, and outcomes. With a focus on cardiovascular complications, we propose several mechanisms of injury. The virology and potential mechanism of injury form the basis for a discussion of potential disease-modifying therapies.\n",
      "The coronavirus disease-2019 (COVID-19), a contagious disease caused by severe acute respiratory syndrome coronavirus-2 (SARS-CoV2), has reached pandemic status. As it spreads across the world, it has overwhelmed healthcare systems, strangled the global economy and led to a devastating loss of life. Widespread efforts from regulators, clinicians and scientists are driving a rapid expansion of knowledge of the SARS-CoV2 virus and the COVID-19 disease. We review the most current data with a focus on our basic understanding of the mechanism(s) of disease and translation to the clinical syndrome and potential therapeutics. We discuss the basic virology, epidemiology, clinical manifestation, multi-organ consequences, and outcomes. With a focus on cardiovascular complications, we propose several mechanisms of injury. The virology and potential mechanism of injury form the basis for a discussion of potential disease-modifying therapies.\n",
      "BACKGROUND Emerging infectious diseases pose a great hazard to public health and the global economy. A novel coronavirus, SARS-CoV-2, causing coronavirus disease 2019 (COVID-19), emerged from Wuhan, China, in December 2019. Since then, it has spread globally causing a global health emergency of inconceivable magnitude, with significant morbidity and mortality. There is the need for clinicians and health care providers to understand and gain knowledge of the different aspects of the disease as it evolves. OBJECTIVE We reviewed existing literature on COVID-19 in order to present a synopsis of current understanding of the disease. METHODS Using PubMed, Embase and Medline, articles published between January and April 2020 on Coronavirus disease 2019 (COVID-19), and resources from World Health Organisation were analyzed in order to have an understanding of the clinical characteristics, transmission dynamics, virology, diagnostic possibilities, prevention, management approach, controversies and impact of COVID-19 on the healthcare workforce. RESULTS Our review revealed that COVID-19 patients present with a myriad of symptoms, ranging from mild to severe respiratory disease, which can often be fatal. The mode of transmission via droplet infections makes it unduly contagious and difficult to control. Currently, antiviral drugs or vaccines are being developed to manage this condition. CONCLUSION There is urgent need for potent antivirals and vaccine to manage those infected with COVID 19 and prevent infection. Global efforts need to be unified in combating this public health emergency and flattening the curve of spread.\n",
      "Few human pathogens have been the focus of as much concentrated worldwide attention as Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV2), the cause of COVID-19. Its emergence into the human population and ensuing pandemic came on the heels of Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) and Middle East Respiratory Syndrome Coronavirus (MERS-CoV), two other highly pathogenic coronavirus spillovers, which collectively have reshaped our view of a virus family previously associated primarily with the common cold. It has placed intense pressure on the collective scientific community to develop therapeutics and vaccines, whose engineering relies on a detailed understanding of coronavirus biology. Here, we present the molecular virology of coronavirus infection, including its entry into cells, its remarkably sophisticated gene expression and replication mechanisms, its extensive remodeling of the intracellular environment and its multi-faceted immune evasion strategies. We highlight aspects of the viral lifecycle that may be amenable to antiviral targeting as well as key features of its biology that await discovery.\n",
      "The current COVID-19 pandemic is caused by the novel coronavirus SARS-CoV-2. The virus causes severe respiratory symptoms which manifest disproportionately in the elderly. Currently, there are over 6.5 million cases and 380,000 deaths reported. Given the current severity of the outbreak, there is a great need for antiviral therapies and vaccines to treat and prevent COVID-19. In this review, we provide an overview of SARS-CoV-2 and discuss the emerging therapies and vaccines that show promise in combating COVID-19. We also highlight potential viral targets that could be exploited by researchers and drug manufacturers.\n",
      "The current COVID-19 pandemic is caused by the novel coronavirus SARS-CoV-2. The virus causes severe respiratory symptoms which manifest disproportionately in the elderly. Currently, there are over 6.5 million cases and 380,000 deaths reported. Given the current severity of the outbreak, there is a great need for antiviral therapies and vaccines to treat and prevent COVID-19. In this review, we provide an overview of SARS-CoV-2 and discuss the emerging therapies and vaccines that show promise in combating COVID-19. We also highlight potential viral targets that could be exploited by researchers and drug manufacturers.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 0kss5r7u\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "what causes death from Covid-19?\n",
      "What are the longer-term complications of those who recover from COVID-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Abstract Background A novel coronavirus disease (COVID-19) in Wuhan has caused an outbreak and become a major public health issue in China and great concern from international community. Myocarditis and myocardial injury were suspected and may even be considered as one of the leading causes for death of COVID-19 patients. Therefore, we focused on the condition of the heart, and sought to provide firsthand evidence for whether myocarditis and myocardial injury were caused by COVID-19. Methods We enrolled patients with confirmed diagnosis of COVID-19 retrospectively and collected heart-related clinical data, mainly including cardiac imaging findings, laboratory results and clinical outcomes. Serial tests of cardiac markers were traced for the analysis of potential myocardial injury/myocarditis. Results 112 COVID-19 patients were enrolled in our study. There was evidence of myocardial injury in COVID-19 patients and 14 (12.5%) patients had presented abnormalities similar to myocarditis. Most of patients had normal levels of troponin at admission, that in 42 (37.5%) patients increased during hospitalization, especially in those that died. Troponin levels were significantly increased in the week preceding the death. 15 (13.4%) patients have presented signs of pulmonary hypertension. Typical signs of myocarditis were absent on echocardiography and electrocardiogram. Conclusions The clinical evidence in our study suggested that myocardial injury is more likely related to systemic consequences rather than direct damage by the 2019 novel coronavirus. The elevation in cardiac markers was probably due to secondary and systemic consequences and can be considered as the warning sign for recent adverse clinical outcomes of the patients.\n",
      "BACKGROUND: A novel coronavirus disease (COVID-19) in Wuhan has caused an outbreak and become a major public health issue in China and great concern from international community. Myocarditis and myocardial injury were suspected and may even be considered as one of the leading causes for death of COVID-19 patients. Therefore, we focused on the condition of the heart, and sought to provide firsthand evidence for whether myocarditis and myocardial injury were caused by COVID-19. METHODS: We enrolled patients with confirmed diagnosis of COVID-19 retrospectively and collected heart-related clinical data, mainly including cardiac imaging findings, laboratory results and clinical outcomes. Serial tests of cardiac markers were traced for the analysis of potential myocardial injury/myocarditis. RESULTS: 112 COVID-19 patients were enrolled in our study. There was evidence of myocardial injury in COVID-19 patients and 14 (12.5%) patients had presented abnormalities similar to myocarditis. Most of patients had normal levels of troponin at admission, that in 42 (37.5%) patients increased during hospitalization, especially in those that died. Troponin levels were significantly increased in the week preceding the death. 15 (13.4%) patients have presented signs of pulmonary hypertension. Typical signs of myocarditis were absent on echocardiography and electrocardiogram. CONCLUSIONS: The clinical evidence in our study suggested that myocardial injury is more likely related to systemic consequences rather than direct damage by the 2019 novel coronavirus. The elevation in cardiac markers was probably due to secondary and systemic consequences and can be considered as the warning sign for recent adverse clinical outcomes of the patients.\n",
      "ABSTRACT Importance: It is postulated that myocardial injury may be common in Coronavirus Disease (COVID-19) reflected by troponin elevation. The nature, degree and mechanism of myocardial injury in affected patients in the United States are unknown. Objective: To describe the severity and trends of myocardial injury in a large cohort of hospitalized patients with confirmed COVID-19. Design: Retrospective cohort study of data captured between February 27th and April 12th, 2020. Setting: Patients were admitted to one of five hospitals in the Mount Sinai Health System (MSHS) in New York City. Data was collected from the electronic health record (EHR) databases within the hospital system. Participants: We included all patients with confirmed COVID-19 admitted to the aforementioned hospitals during the study period who had at least one troponin-I measured within 24 hours of admission. Exposure(s) (for observational studies): SARS-CoV-2 infection as confirmed by real-time reverse transcriptase polymerase chain reaction (RT-PCR) assay of nasal and pharyngeal swabs. Main Outcome(s) and Measure(s): During the study period, 2736 patients met the inclusion criteria. Demographics, medical history, admission labs, and all troponin-I measurements during hospitalization were recorded for each patient. Admission troponin levels were characterized as mildly elevated (between one and three times the upper limit of normal) and elevated (above three times the upper limit of normal). Results: The median age was 66.4 years, with 59.6% men and 40.7% over age 70. In those patients with elevated troponins, cardiovascular disease (CVD) including coronary artery disease, atrial fibrillation, and heart failure, were more prevalent as were conditions of chronic kidney disease, hypertension and diabetes. Inflammatory markers were higher among patients with more substantial troponin elevations as well. Patients who had lower hemoglobin, hypo- or hypertension, or tachycardia generally presented with higher troponins. Conclusions and Relevance: Myocardial injury, manifesting as troponin elevation appears to be common among patients hospitalized with COVID-19 but present at generally low levels. Patients with a history of CVD are more likely to have myocardial injury than patients without CVD or risk factors for CVD. Troponin elevation among patients hospitalized with COVID-19 likely represents non-ischemic or secondary myocardial injury.\n",
      "OBJECTIVE We sought to explore the prevalence and immediate clinical implications of acute myocardial injury in a cohort of patients with covid-19 in a region of China where medical resources are less stressed than in Wuhan (the epicentre of the pandemic). METHODS We prospectively assessed the medical records, laboratory results, chest CT images and use of medication in a cohort of patients presenting to two designated covid-19 treatment centres in Sichuan, China. Outcomes of interest included death, admission to an intensive care unit (ICU), need for mechanical ventilation, treatment with vasoactive agents and classification of disease severity. Acute myocardial injury was defined by a value of high-sensitivity troponin T (hs-TnT) greater than the normal upper limit. RESULTS A total of 101 cases were enrolled from January to 10 March 2020 (average age 49 years, IQR 34-62 years). Acute myocardial injury was present in 15.8% of patients, nearly half of whom had a hs-TnT value fivefold greater than the normal upper limit. Patients with acute myocardial injury were older, with a higher prevalence of pre-existing cardiovascular disease and more likely to require ICU admission (62.5% vs 24.7%, p=0.003), mechanical ventilation (43.5% vs 4.7%, p<0.001) and treatment with vasoactive agents (31.2% vs 0%, p<0.001). Log hs-TnT was associated with disease severity (OR 6.63, 95% CI 2.24 to 19.65), and all of the three deaths occurred in patients with acute myocardial injury. CONCLUSION Acute myocardial injury is common in patients with covid-19 and is associated with adverse prognosis.\n",
      "Abstract Background The degree of myocardial injury, as reflected by troponin elevation, and associated outcomes among US hospitalized patients with Coronavirus Disease 2019 (COVID-19) are unknown. Objectives To describe the degree of myocardial injury and associated outcomes in a large hospitalized cohort with laboratory-confirmed COVID-19. Methods Patients with COVID-19 admitted to one of five Mount Sinai Health System hospitals in New York City between February 27th and April 12th, 2020 with troponin-I (normal value <0.03ng/mL) measured within 24 hours of admission were included (n=2,736). Demographics, medical history, admission labs, and outcomes were captured from the hospitals’ EHR. Results The median age was 66.4 years, with 59.6% men. Cardiovascular disease (CVD) including coronary artery disease, atrial fibrillation, and heart failure, was more prevalent in patients with higher troponin concentrations, as were hypertension and diabetes. A total of 506 (18.5%) patients died during hospitalization. In all, 985 (36%) patients had elevated troponin concentrations. After adjusting for disease severity and relevant clinical factors, even small amounts of myocardial injury (e.g. troponin I 0.03-0.09ng/mL, n=455, 16.6%) were significantly associated with death (adjusted HR: 1.75, 95% CI 1.37-2.24; P<0.001) while greater amounts (e.g. troponin I>0.09 ng/dL, n=530, 19.4%) were significantly associated with higher risk (adjusted HR 3.03, 95% CI 2.42-3.80; P<0.001). Conclusions Myocardial injury is prevalent among patients hospitalized with COVID-19 however troponin concentrations were generally present at low levels. Patients with CVD are more likely to have myocardial injury than patients without CVD. Troponin elevation among patients hospitalized with COVID-19 is associated with higher risk of mortality.\n",
      "OBJECTIVE: We sought to explore the prevalence and immediate clinical implications of acute myocardial injury in a cohort of patients with covid-19 in a region of China where medical resources are less stressed than in Wuhan (the epicentre of the pandemic). METHODS: We prospectively assessed the medical records, laboratory results, chest CT images and use of medication in a cohort of patients presenting to two designated covid-19 treatment centres in Sichuan, China. Outcomes of interest included death, admission to an intensive care unit (ICU), need for mechanical ventilation, treatment with vasoactive agents and classification of disease severity. Acute myocardial injury was defined by a value of high-sensitivity troponin T (hs-TnT) greater than the normal upper limit. RESULTS: A total of 101 cases were enrolled from January to 10 March 2020 (average age 49 years, IQR 34-62 years). Acute myocardial injury was present in 15.8% of patients, nearly half of whom had a hs-TnT value fivefold greater than the normal upper limit. Patients with acute myocardial injury were older, with a higher prevalence of pre-existing cardiovascular disease and more likely to require ICU admission (62.5% vs 24.7%, p=0.003), mechanical ventilation (43.5% vs 4.7%, p<0.001) and treatment with vasoactive agents (31.2% vs 0%, p<0.001). Log hs-TnT was associated with disease severity (OR 6.63, 95% CI 2.24 to 19.65), and all of the three deaths occurred in patients with acute myocardial injury. CONCLUSION: Acute myocardial injury is common in patients with covid-19 and is associated with adverse prognosis.\n",
      "Abstract Coronavirus disease 2019 (COVID-19) is a pandemic that has affected more than 1.8 million people worldwide, overwhelmed health care systems due to the high proportion of critical presentations, and resulted in over 100,000 deaths. Since the first data analyses in China, elevated cardiac troponin has been noted in a substantial proportion of patients, implicating myocardial injury as a possible pathogenic mechanism contributing to severe illness and mortality. Accordingly, high troponin levels are associated with increased mortality in patients with COVID-19. This brief review explores the available evidence regarding the association between COVID-19 and myocardial injury.\n",
      "BACKGROUND: The degree of myocardial injury, as reflected by troponin elevation, and associated outcomes among US hospitalized patients with Coronavirus Disease 2019 (COVID-19) are unknown. OBJECTIVES: To describe the degree of myocardial injury and associated outcomes in a large hospitalized cohort with laboratory-confirmed COVID-19. METHODS: Patients with COVID-19 admitted to one of five Mount Sinai Health System hospitals in New York City between February 27th and April 12th, 2020 with troponin-I (normal value <0.03ng/mL) measured within 24 hours of admission were included (n=2,736). Demographics, medical history, admission labs, and outcomes were captured from the hospitals' EHR. RESULTS: The median age was 66.4 years, with 59.6% men. Cardiovascular disease (CVD) including coronary artery disease, atrial fibrillation, and heart failure, was more prevalent in patients with higher troponin concentrations, as were hypertension and diabetes. A total of 506 (18.5%) patients died during hospitalization. In all, 985 (36%) patients had elevated troponin concentrations. After adjusting for disease severity and relevant clinical factors, even small amounts of myocardial injury (e.g. troponin I 0.03-0.09ng/mL, n=455, 16.6%) were significantly associated with death (adjusted HR: 1.75, 95% CI 1.37-2.24; P<0.001) while greater amounts (e.g. troponin I>0.09 ng/dL, n=530, 19.4%) were significantly associated with higher risk (adjusted HR 3.03, 95% CI 2.42-3.80; P<0.001). CONCLUSIONS: Myocardial injury is prevalent among patients hospitalized with COVID-19 however troponin concentrations were generally present at low levels. Patients with CVD are more likely to have myocardial injury than patients without CVD. Troponin elevation among patients hospitalized with COVID-19 is associated with higher risk of mortality.\n",
      "BACKGROUND The global coronavirus disease 2019 pandemic has highlighted the importance of understanding the cardiovascular implications of coronavirus infections, with more severe disease in those with cardiovascular co-morbidities, and resulting cardiac manifestations such as myocardial injury, arrhythmias, and heart failure. DESIGN A systematic review of the current knowledge on the effects of coronavirus infection on the cardiovascular system in humans was performed and results were summarized. METHODS Databases such as MEDLINE, EMBASE, CENTRAL, Scopus, Web of Science, ClinicalTrials.gov, Chinese Knowledge Resource Integrated Database and Chinese Clinical Trial Registry were searched on 20 March 2020. RESULTS In total, 135 studies were included, involving severe acute respiratory syndrome, Middle East respiratory syndrome, coronavirus disease 2019 and other coronaviruses. Most were case reports, case series and cohort studies of poor to fair quality. In post-mortem examinations of subjects who died from infection, around half had virus identified in heart tissues in severe acute respiratory syndrome, but none in Middle East respiratory syndrome and coronavirus disease 2019. Cardiac manifestations reported include tachycardia, bradycardia, arrhythmias, and myocardial injury, secondary to both systemic infection and treatment. Cardiac injury and arrhythmias are more prevalent in coronavirus disease 2019, and elevated cardiac markers are associated with intensive care unit admission and death. In severe acute respiratory syndrome, Middle East respiratory syndrome, and coronavirus disease 2019, comorbidities such as hypertension, diabetes mellitus, and heart disease are associated with intensive care unit admission, mechanical ventilation, and mortality. There were cases of misdiagnosis due to overlapping presentations of cardiovascular diseases and coronavirus infections, leading to hospital spread and delayed management of life-threatening conditions. CONCLUSION This review highlighted the ways in which coronaviruses affect cardiovascular function and interacts with pre-existing cardiovascular diseases.\n",
      "BACKGROUND: Coronavirus disease 2019 (COVID-19) has been demonstrated to be the cause of pneumonia. Nevertheless, it has not been reported as the cause of acute myocarditis or fulminant myocarditis. CASE PRESENTATION: A 63-year-old male was admitted with pneumonia and cardiac symptoms. He was genetically confirmed as having COVID-19 according to sputum testing on the day of admission. He also had elevated troponin I (Trop I) level (up to 11.37 g/L) and diffuse myocardial dyskinesia along with a decreased left ventricular ejection fraction (LVEF) on echocardiography. The highest level of interleukin-6 was 272.40 pg/ml. Bedside chest radiographs showed typical ground-glass changes indicative of viral pneumonia. Laboratory test results for viruses that cause myocarditis were all negative. The patient conformed to the diagnostic criteria of the Chinese expert consensus statement for fulminant myocarditis. After receiving antiviral therapy and mechanical life support, Trop I was reduced to 0.10 g/L, and interleukin-6 was reduced to 7.63 pg/mL. Moreover, the LVEF of the patient gradually recovered to 68%. The patient died of aggravation of secondary infection on the 33rd day of hospitalization. CONCLUSION: COVID-19 patients may develop severe cardiac complications such as myocarditis and heart failure. This is the first report of COVID-19 complicated with fulminant myocarditis. The mechanism of cardiac pathology caused by COVID-19 needs further study.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: jg6v644y\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Background: Since the outbreak of the Coronavirus Disease 2019 (COVID-19) in China, respiratory manifestations of the disease have been observed. However, as a fatal comorbidity, acute myocardial injury (AMI) in COVID-19 patients has not been previously investigated in detail. We investigated the clinical characteristics of COVID-19 patients with AMI and determined the risk factors for AMI in them. Methods: We analyzed data from 53 consecutive laboratory-confirmed and hospitalized COVID-19 patients (28 men, 25 women; age, 19-81 years). We collected information on epidemiological and demographic characteristics, clinical features, routine laboratory tests (including cardiac injury biomarkers), echocardiography, electrocardiography, imaging findings, management methods, and clinical outcomes. Results: Cardiac complications were found in 42 of the 53 (79.25%) patients: tachycardia (n=15), electrocardiography abnormities (n=11), diastolic dysfunction (n=20), elevated myocardial enzymes (n=30), and AMI (n=6). All the six AMI patients were aged >60 years; five of them had two or more underlying comorbidities (hypertension, diabetes, cardiovascular diseases, and chronic obstructive pulmonary disease). Novel coronavirus pneumonia (NCP) severity was higher in the AMI patients than in patients with non-definite AMI (p<0.001). All the AMI patients required care in intensive care unit; of them, three died, two remain hospitalized. Multivariate analyses showed that C-reactive protein (CRP) levels, NCP severity, and underlying comorbidities were the risk factors for cardiac abnormalities in COVID-19 patients. Conclusions: Cardiac complications are common in COVID-19 patients. Elevated CRP levels, underlying comorbidities, and NCP severity are the main risk factors for cardiac complications in COVID-19 patients.\n",
      "AIMS: To investigate the characteristics and clinical significance of myocardial injury in patients with severe coronavirus disease 2019 (COVID-19). METHODS AND RESULTS: We enrolled 671 eligible hospitalized patients with severe COVID-19 from 1 January to 23 February 2020, with a median age of 63 years. Clinical, laboratory, and treatment data were collected and compared between patients who died and survivors. Risk factors of death and myocardial injury were analysed using multivariable regression models. A total of 62 patients (9.2%) died, who more often had myocardial injury (75.8% vs. 9.7%; P < 0.001) than survivors. The area under the receiver operating characteristic curve of initial cardiac troponin I (cTnI) for predicting in-hospital mortality was 0.92 [95% confidence interval (CI), 0.87-0.96; sensitivity, 0.86; specificity, 0.86; P < 0.001]. The single cut-off point and high level of cTnI predicted risk of in-hospital death, hazard ratio (HR) was 4.56 (95% CI, 1.28-16.28; P = 0.019) and 1.25 (95% CI, 1.07-1.46; P = 0.004), respectively. In multivariable logistic regression, senior age, comorbidities (e.g. hypertension, coronary heart disease, chronic renal failure, and chronic obstructive pulmonary disease), and high level of C-reactive protein were predictors of myocardial injury. CONCLUSION: The risk of in-hospital death among patients with severe COVID-19 can be predicted by markers of myocardial injury, and was significantly associated with senior age, inflammatory response, and cardiovascular comorbidities.\n",
      "AIMS: To investigate the characteristics and clinical significance of myocardial injury in patients with severe coronavirus disease 2019 (COVID-19). METHODS AND RESULTS: We enrolled 671 eligible hospitalized patients with severe COVID-19 from 1 January to 23 February 2020, with a median age of 63 years. Clinical, laboratory, and treatment data were collected and compared between patients who died and survivors. Risk factors of death and myocardial injury were analysed using multivariable regression models. A total of 62 patients (9.2%) died, who more often had myocardial injury (75.8% vs. 9.7%; P < 0.001) than survivors. The area under the receiver operating characteristic curve of initial cardiac troponin I (cTnI) for predicting in-hospital mortality was 0.92 [95% confidence interval (CI), 0.87–0.96; sensitivity, 0.86; specificity, 0.86; P < 0.001]. The single cut-off point and high level of cTnI predicted risk of in-hospital death, hazard ratio (HR) was 4.56 (95% CI, 1.28–16.28; P = 0.019) and 1.25 (95% CI, 1.07–1.46; P = 0.004), respectively. In multivariable logistic regression, senior age, comorbidities (e.g. hypertension, coronary heart disease, chronic renal failure, and chronic obstructive pulmonary disease), and high level of C-reactive protein were predictors of myocardial injury. CONCLUSION: The risk of in-hospital death among patients with severe COVID-19 can be predicted by markers of myocardial injury, and was significantly associated with senior age, inflammatory response, and cardiovascular comorbidities.\n",
      "Objective: To investigate the correlation between clinical characteristics and cardiac injury of COVID-2019 pneumonia. Methods: In this retrospective, single-center study, 41 consecutive corona virus disease 2019 (COVID-2019) patients (including 2 deaths) of COVID-2019 in Beijing Youan Hospital, China Jan 21 to Feb 03, 2020, were involved in this study. The high risk factors of cardiac injury in different COVID-2019 patients were analyzed. Computed tomographic (CT) imaging of epicardial adipose tissue (EAT) has been used to demonstrate the cardiac inflammation of COVID-2019. ResultsOf the 41 COVID-2019 patients, 2 (4.88%), 32 (78.05%), 4 (9.75%) and 3 (7.32%) patients were clinically diagnosed as light, mild, severe and critical cases, according to the 6th guidance issued by the National Health Commission of China. 10 (24.4%) patients had underlying complications, such as hypertension, CAD, type 2 diabetes mellites and tumor. The peak value of TnI in critical patients is 40-fold more than normal value. 2 patients in the critical group had the onset of atrial fibrillation, and the peak heart rates reached up to 160 bpm. CT scan showed low EAT density in severe and critical patients. Conclusion: Our results indicated that cardiac injury of COVID-2019 was rare in light and mild patients, while common in severe and critical patients. Therefore, the monitoring of the heart functions of COVID-2019 patients and applying potential interventions for those with abnormal cardiac injury related characteristics, is vital to prevent the fatality.\n",
      "BACKGROUND: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) could cause virulent infection leading to Corona Virus Disease 2019 (COVID-19)-related pneumonia as well as multiple organ injuries. HYPOTHESIS: COVID-19 infection may result in cardiovascular manifestations leading to worse clinical outcome. METHODS: Fifty four severe and critical patients with confirmed COVID-19 were enrolled. Risk factors predicting the severity of COVID-19 were analyzed. RESULTS: Of the 54 patients (56.1 ± 13.5 years old, 66.7% male) with COVID-19, 39 were diagnosed as severe and 15 as critical cases. The occurrence of diabetes, the level of D-dimer, inflammatory and cardiac markers in critical cases were significantly higher. Troponin I (TnI) elevation occurred in 42.6% of all the severe and critical patients. Three patients experienced hypotension at admission and were all diagnosed as critical cases consequently. Hypotension was found in one severe case and seven critical cases during hospitalization. Sinus tachycardia is the most common type of arrythmia and was observed in 23 severe patients and all the critical patients. Atrioventricular block and ventricular tachycardia were observed in critical patients at end stage while bradycardia and atrial fibrillation were less common. Mild pericardial effusion was observed in one severe case and five critical cases. Three critical cases suffered new onset of heart failure. Hypotension during treatment, severe myocardial injury and pericardial effusion were independent risk factors predicting the critical status of COVID-19 infection. CONCLUSION: This study has systemically observed the impact of COVID-19 on cardiovascular system, including myocardial injury, blood pressure, arrythmia and cardiac function in severe and critical cases. Monitoring of vital signs and cardiac function of COVID-19 patients and applying potential interventions especially for those with hypotension during treatment, severe myocardial injury or pericardial effusion, is of vital importance.\n",
      "Objective: To analyze the clinical characteristics of the severe or critically ill patients with novel coronavirus pneumonia (COVID-19), and evaluate the impact of complicated myocardial injury on the prognosis of these patients. Methods: A retrospective study was conducted in 54 patients who admitted to Tongji hospital from February 3, 2020 to February 24, 2020 and met the criteria of severe or critical conditions of COVID-19. The clinical characteristics and hospital mortality rate were analyzed and compared between the patients with or without myocardial injury, which was defined with 3 times higher serum cardiac troponin value. Results: The median age of the 54 patients was 68 (59.8, 74.3) years. Among all the patients, 24 (44.4%) patients were complicated with hypertension, 13 (24.1%) with diabetes, 8 (14.8%) with coronary heart disease, and 3 (5.6%) with previous cerebral infarction. During hospitalization, 24 (44.4%) of the patients were complicated with myocardial injury and 26 (48.1%) patients died in hospital. In-hospital mortality was significantly higher in patients with myocardial injury than in patients without myocardial injury (14 (60.9%) vs. 8 (25.8%), P=0.013). Moreover, the levels of C-reactive protein (153.6 (80.3, 240.7) ng/L vs. 49.8 (15.9, 101.9) ng/L) and N-terminal pro-B-type natriuretic peptide (852.0 (400.0, 2 315.3) ng/L vs. 197.0 (115.3, 631.0) ng/L) were significantly higher than patients without myocardial injury (all P<0.01). Conclusions: Prevalence of myocardial injury is high among severe or critically ill COVID-19 patients. Severe or critically ill COVID-19 patients with myocardial injury face a significantly higher risk of in-hospital mortality. The study suggests that it is important to monitor and manage the myocardial injury during hospitalization for severe or critically ill COVID-19 patients.\n",
      "Objective: To analyze the clinical characteristics of the severe or critically ill patients with novel coronavirus pneumonia (COVID-19), and evaluate the impact of complicated myocardial injury on the prognosis of these patients. Methods: A retrospective study was conducted in 54 patients who admitted to Tongji hospital from February 3, 2020 to February 24, 2020 and met the criteria of severe or critical conditions of COVID-19. The clinical characteristics and hospital mortality rate were analyzed and compared between the patients with or without myocardial injury, which was defined with 3 times higher serum cardiac troponin value. Results: The median age of the 54 patients was 68 (59.8, 74.3) years. Among all the patients, 24 (44.4%) patients were complicated with hypertension, 13 (24.1%) with diabetes, 8 (14.8%) with coronary heart disease, and 3 (5.6%) with previous cerebral infarction. During hospitalization, 24 (44.4%) of the patients were complicated with myocardial injury and 26 (48.1%) patients died in hospital. In-hospital mortality was significantly higher in patients with myocardial injury than in patients without myocardial injury (14 (60.9%) vs. 8 (25.8%), P=0.013). Moreover, the levels of C-reactive protein (153.6 (80.3, 240.7) ng/L vs. 49.8 (15.9, 101.9) ng/L) and N-terminal pro-B-type natriuretic peptide (852.0 (400.0, 2 315.3) ng/L vs. 197.0 (115.3, 631.0) ng/L) were significantly higher than patients without myocardial injury (all P<0.01). Conclusions: Prevalence of myocardial injury is high among severe or critically ill COVID-19 patients. Severe or critically ill COVID-19 patients with myocardial injury face a significantly higher risk of in-hospital mortality. The study suggests that it is important to monitor and manage the myocardial injury during hospitalization for severe or critically ill COVID-19 patients.\n",
      "BACKGROUND In December, 2019, China, has experienced an outbreak of novel coronavirus disease 2019 (COVID-19). Coronavirus has now spread to all of the continents. We aimed to consider clinical characteristics, laboratory data of COVID-19 that provided more information for the research of this novel virus. METHODS We performed a retrospective cohort study on the clinical symptoms and laboratory findings of a series of the 100 confirmed patients with COVID-19. These patients were admitted to the hospitals affiliated to Babol University of Medical Sciences (Ayatollah Rohani, Shahid Beheshti and Yahyanejad hospitals) form 25 February 2020 to 12 March 2020. RESULTS Nineteen patients died during hospitalization and 81 were discharged. Non-survivor patients had a significantly higher C-reactive protein (CRP) (MD: 46.37, 95% CI: 20.84, 71.90; P= 0.001), white blood cells (WBCs) (MD: 3.10, 95% CI: 1.53, 4.67; P< 0.001) and lower lymphocyte (MD: -8.75, 95% CI: -12.62, -4.87; P< 0.001) compared to survivor patients Data analysis showed that comorbid conditions (aRR: 2.99, 95%CI: 1.09, 8.21, P= 0.034), higher CRP levels (aRR: 1.02, 95%CI: 1.01, 1.03, P= 0.044), and lower lymphocyte (aRR: 0.82, 95%CI: 0.73, 0.93, P= 0.003) were associated with increased risk of death. CONCLUSIONS Based on our findings, most non-survivors are elderly with comorbidities. Lymphopenia and increased levels of WBCs along with elevated CRP were associated with increased risk of death. Therefore, it is best to be regularly assessed these markers during treatment of COVID-19 patients.\n",
      "BACKGROUND: In December, 2019, China, has experienced an outbreak of novel coronavirus disease 2019 (COVID-19). Coronavirus has now spread to all of the continents. We aimed to consider clinical characteristics, laboratory data of COVID-19 that provided more information for the research of this novel virus. METHODS: We performed a retrospective cohort study on the clinical symptoms and laboratory findings of a series of the 100 confirmed patients with COVID-19. These patients were admitted to the hospitals affiliated to Babol University of Medical Sciences (Ayatollah Rohani, Shahid Beheshti and Yahyanejad hospitals) form 25 February 2020 to 12 March 2020. RESULTS: Nineteen patients died during hospitalization and 81 were discharged. Non-survivor patients had a significantly higher C-reactive protein (CRP) (MD: 46.37, 95% CI: 20.84, 71.90; P= 0.001), white blood cells (WBCs) (MD: 3.10, 95% CI: 1.53, 4.67; P< 0.001) and lower lymphocyte (MD: -8.75, 95% CI: -12.62, -4.87; P< 0.001) compared to survivor patients Data analysis showed that comorbid conditions (aRR: 2.99, 95%CI: 1.09, 8.21, P= 0.034), higher CRP levels (aRR: 1.02, 95%CI: 1.01, 1.03, P= 0.044), and lower lymphocyte (aRR: 0.82, 95%CI: 0.73, 0.93, P= 0.003) were associated with increased risk of death. CONCLUSIONS: Based on our findings, most non-survivors are elderly with comorbidities. Lymphopenia and increased levels of WBCs along with elevated CRP were associated with increased risk of death. Therefore, it is best to be regularly assessed these markers during treatment of COVID-19 patients.\n",
      "BACKGROUND: Coronavirus disease 2019 (COVID‐19) has become a serious public health problem worldwide. Here, we stratified COVID‐19 patients based on their comorbidities to assess their risk of serious adverse outcomes. METHODS: We collected 856 hospitalized cases diagnosed with COVID‐19 from January 17 to February 7, 2020, in Zhejiang Province, and analyzed their comorbidities and composite endpoint (including admission to intensive care unit owing to disease progression, shock, invasive ventilation, and death) to determine the relationship between comorbidities and adverse outcomes. RESULTS: The median age of patients was 46 (36‐56) years; 439 (51.3%) were men, 242 (28.3%) had comorbidities, and 152 (17.8%) had two or more comorbidities. The most common comorbidity was hypertension (142 [16.6%]), followed by diabetes (64 [7.5%]). Of the 856 patients, there are 154 (18.0%) severe cases.32(3.7%) reached composite endpoints, of which 22 (9.1%) were from the comorbidity group and 10 (1.6%) from the non‐comorbidity group (p < 0.001). After adjusting for age and gender status, the risk of reaching the composite endpoint was higher in the group with comorbidity than in that without comorbidity (hazard ratio [HR] 3.04, 95% confidence interval [CI]: 1.40‐6.60). HR values for patients with one, two, and three or more comorbidities were 1.61 (95% CI: 0.44‐5.91), 3.44 (95% CI: 1.31‐9.08), and 6.90 (95% CI: 2.69‐17.69), respectively. CONCLUSIONS: COVID‐19 patients with comorbidities had worse clinical outcomes as compared to those without any comorbidity. The higher the number of comorbidities, the greater was the risk of serious adverse outcomes. This article is protected by copyright. All rights reserved.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: fzmrvjtl\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what are the guidelines for triaging patients infected with coronavirus?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "The novel coronavirus disease (COVID-19) pandemic has already caused more than 300,000 deaths worldwide. Several studies have elucidated the central role of cardiovascular complications in the disease course. Herein, we provide a concise review of current knowledge regarding the involvement of cardiovascular system in the pathogenesis and prognosis of COVID-19. We summarize data from 21 studies involving in total more than 21,000 patients from Asia, Europe and the USA indicating that severe disease is associated with the presence of myocardial injury, heart failure and arrhythmias. Additionally, we present the clinical and laboratory differences between recovered and deceased patients highlighting the importance of cardiac manifestations. For the infected patients, underlying cardiovascular comorbidities and especially existing cardiovascular disease seem to predispose to the development of cardiovascular complications, which are in turn associated with higher mortality rates. We provide mechanistic insights into the underlying mechanisms including direct myocardial damage by the virus and the consequences of the hyperinflammatory syndrome developed later in the disease course. Finally, we summarize current knowledge on therapeutic modalities and recommendations by scientific societies and experts regarding the cardiovascular management of COVID-19 patients.\n",
      "BACKGROUND The global coronavirus disease 2019 pandemic has highlighted the importance of understanding the cardiovascular implications of coronavirus infections, with more severe disease in those with cardiovascular co-morbidities, and resulting cardiac manifestations such as myocardial injury, arrhythmias, and heart failure. DESIGN A systematic review of the current knowledge on the effects of coronavirus infection on the cardiovascular system in humans was performed and results were summarized. METHODS Databases such as MEDLINE, EMBASE, CENTRAL, Scopus, Web of Science, ClinicalTrials.gov, Chinese Knowledge Resource Integrated Database and Chinese Clinical Trial Registry were searched on 20 March 2020. RESULTS In total, 135 studies were included, involving severe acute respiratory syndrome, Middle East respiratory syndrome, coronavirus disease 2019 and other coronaviruses. Most were case reports, case series and cohort studies of poor to fair quality. In post-mortem examinations of subjects who died from infection, around half had virus identified in heart tissues in severe acute respiratory syndrome, but none in Middle East respiratory syndrome and coronavirus disease 2019. Cardiac manifestations reported include tachycardia, bradycardia, arrhythmias, and myocardial injury, secondary to both systemic infection and treatment. Cardiac injury and arrhythmias are more prevalent in coronavirus disease 2019, and elevated cardiac markers are associated with intensive care unit admission and death. In severe acute respiratory syndrome, Middle East respiratory syndrome, and coronavirus disease 2019, comorbidities such as hypertension, diabetes mellitus, and heart disease are associated with intensive care unit admission, mechanical ventilation, and mortality. There were cases of misdiagnosis due to overlapping presentations of cardiovascular diseases and coronavirus infections, leading to hospital spread and delayed management of life-threatening conditions. CONCLUSION This review highlighted the ways in which coronaviruses affect cardiovascular function and interacts with pre-existing cardiovascular diseases.\n",
      "BACKGROUND: Respiratory complications have been well remarked in the novel coronavirus disease (SARS-CoV-2/COVID-19), yet an emerging body of research indicates that cardiac involvement may be implicated in poor outcomes for these patients. AIMS: This review seeks to gather and distill the existing body of literature that describes the cardiac implications of COVID-19. MATERIALS AND METHODS: The English literature was reviewed for papers dealing with the cardiac effects of COVID-19. RESULTS: Notably, COVID-19 patients with pre-existing cardiovascular disease are counted in greater frequency in intensive care unit settings, and ultimately suffer greater rates of mortality. Other studies have noted cardiac presentations for COVID-19, rather than respiratory, such as acute pericarditis and left ventricular dysfunction. In some patients there has been evidence of acute myocardial injury, with correspondingly increased serum troponin I levels. With regard to surgical interventions, there is a dearth of data describing myocardial protection during cardiac surgery for COVID-19 patients. Although some insights have been garnered in the study of cardiovascular diseases for these patients, these insights remain fragmented and have yet to cement clear guidelines for actionable clinical practice. CONCLUSION: While some information is available, further studies are imperative for a more cohesive understanding of the cardiac pathophysiology in COVID-19 patients to promote more informed treatment and, ultimately, better clinical outcomes.\n",
      "BACKGROUND: The global coronavirus disease 2019 pandemic has highlighted the importance of understanding the cardiovascular implications of coronavirus infections, with more severe disease in those with cardiovascular co-morbidities, and resulting cardiac manifestations such as myocardial injury, arrhythmias, and heart failure. DESIGN: A systematic review of the current knowledge on the effects of coronavirus infection on the cardiovascular system in humans was performed and results were summarized. METHODS: Databases such as MEDLINE, EMBASE, CENTRAL, Scopus, Web of Science, ClinicalTrials.gov, Chinese Knowledge Resource Integrated Database and Chinese Clinical Trial Registry were searched on 20 March 2020. RESULTS: In total, 135 studies were included, involving severe acute respiratory syndrome, Middle East respiratory syndrome, coronavirus disease 2019 and other coronaviruses. Most were case reports, case series and cohort studies of poor to fair quality. In post-mortem examinations of subjects who died from infection, around half had virus identified in heart tissues in severe acute respiratory syndrome, but none in Middle East respiratory syndrome and coronavirus disease 2019. Cardiac manifestations reported include tachycardia, bradycardia, arrhythmias, and myocardial injury, secondary to both systemic infection and treatment. Cardiac injury and arrhythmias are more prevalent in coronavirus disease 2019, and elevated cardiac markers are associated with intensive care unit admission and death. In severe acute respiratory syndrome, Middle East respiratory syndrome, and coronavirus disease 2019, comorbidities such as hypertension, diabetes mellitus, and heart disease are associated with intensive care unit admission, mechanical ventilation, and mortality. There were cases of misdiagnosis due to overlapping presentations of cardiovascular diseases and coronavirus infections, leading to hospital spread and delayed management of life-threatening conditions. CONCLUSION: This review highlighted the ways in which coronaviruses affect cardiovascular function and interacts with pre-existing cardiovascular diseases.\n",
      "BACKGROUND: Respiratory complications have been well remarked in the novel coronavirus disease (SARS‐CoV‐2/COVID‐19), yet an emerging body of research indicates that cardiac involvement may be implicated in poor outcomes for these patients. AIMS: This review seeks to gather and distill the existing body of literature that describes the cardiac implications of COVID‐19. MATERIALS AND METHODS: The English literature was reviewed for papers dealing with the cardiac effects of COVID‐19. RESULTS: Notably, COVID‐19 patients with pre‐existing cardiovascular disease are counted in greater frequency in intensive care unit settings, and ultimately suffer greater rates of mortality. Other studies have noted cardiac presentations for COVID‐19, rather than respiratory, such as acute pericarditis and left ventricular dysfunction. In some patients there has been evidence of acute myocardial injury, with correspondingly increased serum troponin I levels. With regard to surgical interventions, there is a dearth of data describing myocardial protection during cardiac surgery for COVID‐19 patients. Although some insights have been garnered in the study of cardiovascular diseases for these patients, these insights remain fragmented and have yet to cement clear guidelines for actionable clinical practice. CONCLUSION: While some information is available, further studies are imperative for a more cohesive understanding of the cardiac pathophysiology in COVID‐19 patients to promote more informed treatment and, ultimately, better clinical outcomes.\n",
      "Abstract Coronavirus disease (COVID-19) pandemic has so far involved 184 countries and more than 2.79 million patients worldwide. Over the past three months, it has attributed to more than 196, 000 deaths, with more than 50, 000 deaths in the United States alone. Pulmonary manifestations are predominant and have been well identified. Cardiac involvement is also common. Acute cardiac injury, the most common cardiac manifestation of this disease can be seen in patients even without prior cardiac comorbidities. Established cardiovascular risk factors such as diabetes mellitus, hypertension, and coronary artery disease predispose to cardiac injury, the severity of illness and mortality. Non-ischemic myocardial injury secondary to cytokine storm is thought to be the predominant mechanism of acute cardiac injury associated with COVID-19. Multiple mechanisms and processes contribute to cardiac injury resulting in a poor outcome. Some of these are not clearly understood. Clinical and diagnostic details of cardiovascular involvement in these patients are mostly limited to biochemical markers. Multiple therapeutic agents have been tried with questionable efficacy and without clinical evidence. Interactions of comorbidities, cardiovascular drugs, the cardiac effect of therapeutic agents on the illness continue to be under investigation. With an increasing number of patients, newer promising therapies, and ongoing clinical trials, the exact mechanisms and extent to which these risk factors contribute to outcomes will be clearer in the future.\n",
      "Abstract The coronavirus disease-2019 (COVID-19) is an infectious disease caused by severe acute respiratory syndrome coronavirus 2 that has significant implications for the cardiovascular care of patients. First, those with COVID-19 and preexisting cardiovascular disease (CVD) have an increased risk of severe disease and death. Second, infection has been associated with multiple direct and indirect cardiovascular complications including acute myocardial injury, myocarditis, arrhythmias and venous thromboembolism. Third, therapies under investigation for COVID-19 may have cardiovascular side effects. Fourth, the response to COVID-19 can compromise the rapid triage of non-COVID-19 patients with cardiovascular conditions. Finally, the provision of cardiovascular care may place health care workers in a position of vulnerability as they become host or vectors of virus transmission. We hereby review the peer-reviewed and preprint literature pertaining to cardiovascular considerations related to COVID-19 and highlight gaps in knowledge that require further study pertinent to patients, health care workers, and health systems.\n",
      "The coronavirus disease 2019 (COVID-19) is an infectious disease caused by severe acute respiratory syndrome coronavirus 2 that has significant implications for the cardiovascular care of patients. First, those with COVID-19 and pre-existing cardiovascular disease have an increased risk of severe disease and death. Second, infection has been associated with multiple direct and indirect cardiovascular complications including acute myocardial injury, myocarditis, arrhythmias, and venous thromboembolism. Third, therapies under investigation for COVID-19 may have cardiovascular side effects. Fourth, the response to COVID-19 can compromise the rapid triage of non-COVID-19 patients with cardiovascular conditions. Finally, the provision of cardiovascular care may place health care workers in a position of vulnerability as they become hosts or vectors of virus transmission. We hereby review the peer-reviewed and pre-print reports pertaining to cardiovascular considerations related to COVID-19 and highlight gaps in knowledge that require further study pertinent to patients, health care workers, and health systems.\n",
      "As the severe acute respiratory syndrome coronavirus 2 virus pandemic continues to grow globally, an association is apparent between patients with underlying cardiovascular disease comorbidities and the risk of developing severe COVID-19. Furthermore, there are potential cardiac manifestations of severe acute respiratory syndrome coronavirus 2 including myocyte injury, ventricular dysfunction, coagulopathy, and electrophysiologic abnormalities. Balancing management of the infection and treatment of underlying cardiovascular disease requires further study. Addressing the increasing reports of health care worker exposure and deaths remains paramount. This review summarizes the most contemporary literature on the relationship of the cardiovascular system and COVID-19 and society statements with relevance to protection of health care workers, and provides illustrative case reports in this context.\n",
      "As the severe acute respiratory syndrome coronavirus 2 virus pandemic continues to grow globally, an association is apparent between patients with underlying cardiovascular disease comorbidities and the risk of developing severe COVID-19. Furthermore, there are potential cardiac manifestations of severe acute respiratory syndrome coronavirus 2 including myocyte injury, ventricular dysfunction, coagulopathy, and electrophysiologic abnormalities. Balancing management of the infection and treatment of underlying cardiovascular disease requires further study. Addressing the increasing reports of health care worker exposure and deaths remains paramount. This review summarizes the most contemporary literature on the relationship of the cardiovascular system and COVID-19 and society statements with relevance to protection of health care workers, and provides illustrative case reports in this context.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: yr1dq258\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the transmission routes of coronavirus?\n",
      "what type of hand sanitizer is needed to destroy Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "BACKGROUND: In the 2003 severe acute respiratory syndrome outbreak, finding viral nucleic acids on hospital surfaces suggested surfaces could play a role in spread in health care environments. Surface disinfection may interrupt transmission, but few data exist on the effectiveness of health care germicides against coronaviruses on surfaces. METHODS: The efficacy of health care germicides against 2 surrogate coronaviruses, mouse hepatitis virus (MHV) and transmissible gastroenteritis virus (TGEV), was tested using the quantitative carrier method on stainless steel surfaces. Germicides were o-phenylphenol/p-tertiary amylphenol) (a phenolic), 70% ethanol, 1:100 sodium hypochlorite, ortho-phthalaldehyde (OPA), instant hand sanitizer (62% ethanol), and hand sanitizing spray (71% ethanol). RESULTS: After 1-minute contact time, for TGEV, there was a log(10) reduction factor of 3.2 for 70% ethanol, 2.0 for phenolic, 2.3 for OPA, 0.35 for 1:100 hypochlorite, 4.0 for 62% ethanol, and 3.5 for 71% ethanol. For MHV, log(10) reduction factors were 3.9 for 70% ethanol, 1.3 for phenolic, 1.7 for OPA, 0.62 for 1:100 hypochlorite, 2.7 for 62% ethanol, and 2.0 for 71% ethanol. CONCLUSION: Only ethanol reduced infectivity of the 2 coronaviruses by >3-log(10) after 1 minute. Germicides must be chosen carefully to ensure they are effective against viruses such as severe acute respiratory syndrome coronavirus.\n",
      "BACKGROUND: The 2003 outbreak of severe acute respiratory syndrome (SARS) infected over 8000 people and killed 774. Transmission of SARS occurred through direct and indirect contact and large droplet nuclei. The World Health Organization recommended the use of household disinfectants, which have not been previously tested against SARS coronavirus (SARS-CoV), to disinfect potentially contaminated environmental surfaces. There is a need for a surrogate test system given the limited availability of the SARS-CoV for testing and biosafety requirements necessary to safely handle it. In this study, the antiviral activity of standard household products was assayed against murine hepatitis virus (MHV), as a potential surrogate for SARS-CoV. METHODS: A surface test method, which involves drying an amount of virus on a surface and then applying the product for a specific contact time, was used to determine the virucidal activity. The virus titers and log reductions were determined by the Reed and Muench tissue culture infective dose (TCID)(50) end point method. RESULTS: When tested as directed, common household disinfectants or antiseptics, containing either 0.050% of triclosan, 0.12% of PCMX, 0.21% of sodium hypochlorite, 0.23% of pine oil, or 0.10% of a quaternary compound with 79% of ethanol, demonstrated a 3-log reduction or better against MHV without any virus recovered in a 30-second contact time. CONCLUSION: Common household disinfectants and antiseptics were effective at inactivating MHV, a possible surrogate for SARS-CoV, from surfaces when used as directed. In an outbreak caused by novel agents, it is important to know the effectiveness of disinfectants and antiseptics to prevent or reduce the possibility of human-to-human transmission via surfaces.\n",
      "Summary The recent severe acute respiratory syndrome (SARS) epidemic in Asia and Northern America led to broad use of various types of disinfectant in order to control the public spread of the highly contagious virus. However, only limited data were available to demonstrate their efficacy against SARS coronavirus (SARS-CoV). We therefore investigated eight disinfectants for their activity against SARS-CoV according to prEN 14476. Four hand rubs were tested at 30s (Sterillium, based on 45% iso-propanol, 30% n-propanol and 0.2% mecetronium etilsulphate; Sterillium Rub, based on 80% ethanol; Sterillium Gel, based on 85% ethanol; Sterillium Virugard, based on 95% ethanol). Three surface disinfectants were investigated at 0.5% for 30min and 60min (Mikrobac forte, based on benzalkonium chloride and laurylamine; Kohrsolin FF, based on benzalkonium chloride, glutaraldehyde and didecyldimonium chloride; Dismozon pur, based on magnesium monoperphthalate), and one instrument disinfectant was investigated at 4% for 15min, 3% for 30min and 2% for 60min [Korsolex basic, based on glutaraldehyde and (ethylenedioxy)dimethanol]. Three types of organic load were used: 0.3% albumin, 10% fetal calf serum, and 0.3% albumin with 0.3% sheep erythrocytes. Virus titres were determined by a quantitative test (endpoint titration) in 96-well microtitre plates. With all tested preparations, SARS-CoV was inactivated to below the limit of detection (reduction factor mostly ≥4), regardless of the type of organic load. In summary, SARS-CoV can be inactivated quite easily with many commonly used disinfectants.\n",
      "Summary Background Surface contamination has been implicated in the transmission of certain viruses, and surface disinfection can be an effective measure to interrupt the spread of these agents. Aim To evaluate the in-vitro efficacy of hydrogen peroxide vapour (HPV), a vapour-phase disinfection method, for the inactivation of a number of structurally distinct viruses of importance in the healthcare, veterinary and public sectors. The viruses studied were: feline calicivirus (FCV, a norovirus surrogate); human adenovirus type 1; transmissible gastroenteritis coronavirus of pigs (TGEV, a severe acute respiratory syndrome coronavirus [SARS-CoV] surrogate); avian influenza virus (AIV); and swine influenza virus (SwIV). Methods The viruses were dried on stainless steel discs in 20- or 40-μL aliquots and exposed to HPV produced by a Clarus L generator (Bioquell, Horsham, PA, USA) in a 0.2-m3 environmental chamber. Three vaporized volumes of hydrogen peroxide were tested in triplicate for each virus: 25, 27 and 33mL. Findings No viable viruses were identified after HPV exposure at any of the vaporized volumes tested. HPV was virucidal (>4-log reduction) against FCV, adenovirus, TGEV and AIV at the lowest vaporized volume tested (25mL). For SwIV, due to low virus titre on the control discs, >3.8-log reduction was shown for the 25-mL vaporized volume and >4-log reduction was shown for the 27-mL and 33-mL vaporized volumes. Conclusion HPV was virucidal for structurally distinct viruses dried on surfaces, suggesting that HPV can be considered for the disinfection of virus-contaminated surfaces.\n",
      "The chemical disinfection of virus-contaminated non-porous inanimate surfaces was investigated using coxsackievirus B3, adenovirus type 5, parainfluenza virus type 3 and coronavirus 229E as representatives of important nosocomial viral pathogens. A 10 microliter amount of the test virus, suspended in either faeces or mucin, was placed onto each stainless steel disk (about 1 cm in diameter) and the inoculum allowed to dry for 1 h under ambient conditions. Sixteen disinfectant formulations were selected for this study based on the findings of an earlier investigation with a human rotavirus. After 1 min exposure to 20 microliters of the disinfectant, the virus from the disks was immediately eluted into tryptose phosphate broth and plaque assayed. Using an efficacy criterion of a 3 log10 or greater reduction in virus infectivity titre and irrespective of the virus suspending medium, only the following five disinfectants proved to be effective against all the four viruses tested: (1) 2% glutaraldehyde normally used as an instrument soak, (2) a strongly alkaline mixture of 0.5% sodium o-benzyl-p-chlorophenate and 0.6% sodium lauryl sulphate, generally used as a domestic disinfectant cleaner for hard surfaces, (3) a 0.04% solution of a quaternary ammonium compound containing 7% hydrochloric acid, which is the basis of many toilet bowl cleaners, (4) chloramine T at a minimum free chlorine level of 3000 p.p.m. and (5) sodium hypochlorite at a minimum free chlorine concentration of 5000 p.p.m. Of those chemicals suitable for use as topical antiseptics, 70% ethanol alone or products containing at least 70% ethanol were ineffective only against coxsackievirus B3. These results emphasize the care needed in selecting chemical disinfectants for routine use in infection control.\n",
      "BACKGROUND: The emergence of the novel virus, SARS-CoV-2, has posed unprecedented challenges to public health around the world. Currently, strategies to deal with COVID-19 are purely supportive and preventative, aimed at reducing transmission. An effective and simple method for reducing transmission of infections in the public or healthcare settings is hand hygiene. Unfortunately, little is known regarding the efficacy of hand sanitizers against SARS-CoV-2. METHODS: In this review, an extensive literature search was performed to succinctly summarize the primary active ingredients and mechanisms of action of hand sanitizers, compare the effectiveness and compliance of gel and foam sanitizers, and predict whether alcohol and non-alcohol hand sanitizers would be effective against SARS-CoV-2. RESULTS: Most alcohol based hand sanitizers are effective at inactivating enveloped viruses, including coronaviruses. With what is currently known in the literature, one may not confidently suggest one mode of hand sanitizing delivery over the other. When hand washing with soap and water is unavailable, a sufficient volume of sanitizer is necessary to ensure complete hand coverage, and compliance is critical for appropriate hand hygiene. CONCLUSIONS: By extrapolating effectiveness of hand sanitizers on viruses of similar structure to SARS-CoV-2, this virus should be effectively inactivated with current hand hygiene products, though future research should attempt to determine this directly.\n",
      "BACKGROUND The efficacies of disinfection by sodium hypochlorite, accelerated hydrogen peroxide (AHP), and quaternary ammonium compound (QUAT) commonly used in health care facilities were determined using the surrogate viruses murine norovirus (MNV-1) and feline calicivirus (FCV). METHODS A virus suspension of known concentration (with or without a soil load) was deposited onto stainless steel discs under wet or dry load conditions and exposed to defined concentrations of the disinfectant/cleaning agent for 1-, 5-, or 10-minute contact time using the quantitative carrier test (QCT-2) method. Virus inactivation was determined by plaque assay. RESULTS At an exposure time of 1 minute, sodium hypochlorite at 2,700 ppm was able to inactivate MNV-1 and FCV with a >5 log10 reduction. After 10 minutes, MNV-1 was inactivated by AHP at 35,000 ppm, whereas FCV was inactivated at 3,500 ppm. MNV-1 was not inactivated by QUAT at 2,800 ppm. A QUAT-alcohol formulation containing 2,000 ppm QUAT and 70% ethanol was effective in inactivating MNV-1 after 5 minutes, but resulted in only a <3 log10 reduction of FCV after 10 minutes. CONCLUSIONS AHP and QUAT products were less effective than sodium hypochlorite for the inactivation of MNV-1 and FCV.\n",
      "Assessment of the risks posed by severe acute respiratory syndrome (SARS) coronavirus (SARS-CoV) on surfaces requires data on survival of this virus on environmental surfaces and on how survival is affected by environmental variables, such as air temperature (AT) and relative humidity (RH). The use of surrogate viruses has the potential to overcome the challenges of working with SARS-CoV and to increase the available data on coronavirus survival on surfaces. Two potential surrogates were evaluated in this study; transmissible gastroenteritis virus (TGEV) and mouse hepatitis virus (MHV) were used to determine effects of AT and RH on the survival of coronaviruses on stainless steel. At 4 degrees C, infectious virus persisted for as long as 28 days, and the lowest level of inactivation occurred at 20% RH. Inactivation was more rapid at 20 degrees C than at 4 degrees C at all humidity levels; the viruses persisted for 5 to 28 days, and the slowest inactivation occurred at low RH. Both viruses were inactivated more rapidly at 40 degrees C than at 20 degrees C. The relationship between inactivation and RH was not monotonic, and there was greater survival or a greater protective effect at low RH (20%) and high RH (80%) than at moderate RH (50%). There was also evidence of an interaction between AT and RH. The results show that when high numbers of viruses are deposited, TGEV and MHV may survive for days on surfaces at ATs and RHs typical of indoor environments. TGEV and MHV could serve as conservative surrogates for modeling exposure, the risk of transmission, and control measures for pathogenic enveloped viruses, such as SARS-CoV and influenza virus, on health care surfaces.\n",
      "The World Health Organization (WHO) published 2 alcohol-based formulations to be used in healthcare settings and for outbreak-associated infections, but inactivation efficacies of these products have not been determined against (re-)emerging viruses. In this study, we evaluated the virucidal activity of these WHO products in a comparative analysis. Zika virus (ZIKV), Ebola virus (EBOV), severe acute respiratory syndrome coronavirus (SARS-CoV), and Middle East respiratory syndrome coronavirus (MERS-CoV) as (re-)emerging viral pathogens and other enveloped viruses could be efficiently inactivated by both WHO formulations, implicating their use in healthcare systems and viral outbreak situations.\n",
      "Summary Ethanol is used worldwide in healthcare facilities for hand rubbing. It has been reported to have a stronger and broader virucidal activity compared with propanols. The aim of this review was to describe the spectrum of virucidal activity of ethanol in solution or as commercially available products. A systematic search was conducted. Studies were selected when they contained original data on reduction of viral infectivity from suspension tests (49 studies) and contaminated hands (17 studies). Ethanol at 80% was highly effective against all 21 tested, enveloped viruses within 30 s. Murine norovirus and adenovirus type 5 are usually inactivated by ethanol between 70% and 90% in 30 s whereas poliovirus type 1 was often found to be too resistant except for ethanol at 95% (all test viruses of EN 14476). Ethanol at 80% is unlikely to be sufficiently effective against poliovirus, calicivirus (FCV), polyomavirus, hepatitis A virus (HAV) and foot-and-mouth disease virus (FMDV). The spectrum of virucidal activity of ethanol at 95%, however, covers the majority of clinically relevant viruses. Additional acids can substantially improve the virucidal activity of ethanol at lower concentrations against, e.g. poliovirus, FCV, polyomavirus and FMDV although selected viruses such as HAV may still be too resistant. The selection of a suitable virucidal hand rub should be based on the viruses most prevalent in a unit and on the user acceptability of the product under frequent-use conditions.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: qwngx01h\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the best masks for preventing infection by Covid-19?\n",
      "How much impact do masks have on preventing the spread of the COVID-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "In this perspective, we recommend universal masking of the US public during coronavirus disease 2019 due to the high contagiousness of severe acute respiratory syndrome-coronavirus 2 (SARS-CoV-2), viral shedding of viable SARS-CoV-2 from asymptomatic individuals, and the likely contribution of masking to core distancing public health strategies for curbing transmission.\n",
      "It has been reported that asymptomatic people can transmit the new coronavirus disease 2019 (COVID-19) and become important sources of COVID-19. To reduce the role of asymptomatic or poorly symptomatic people in COVID-19, universal use of face masks in addition to hand hygiene and safety distance seems extremely useful. Consequently, preparing the healthy child to use face masks is strongly needed. To obtain maximal compliance, reasons for mask wearing without attempts of removing must be clearly explained. Moreover, child’s will must not be forced. Conclusion: On the basis of clinical findings, we think that the universal use of facial masks seems necessary when people have to go out in their everyday lives. In addition to the availability of masks of different sizes capable of adapting perfectly to the face, it is necessary that the use of masks in children is preceded by a strong parental work and school lessons on this issue and other hygiene topics with the main aim to obtain child cooperation.\n",
      "BACKGROUND: Severe acute respiratory syndrome-coronavirus-2 (SARS-CoV-2; COVID-19), which causes coronavirus disease 2019, is highly contagious and a particularly popular problem in all around the world and also in all departments of every hospital. AIMS: Protecting the well-being of the aesthetic dermatologists while providing a sufficient workforce is vital for pandemic planning. RESULTS AND CONCLUSIONS: In this article, we will discuss this problem from an aesthetic dermatology aspect and we will review whether these procedures are safe or not.\n",
      "It has been reported that asymptomatic people can transmit the new coronavirus disease 2019 (COVID-19) and become important sources of COVID-19. To reduce the role of asymptomatic or poorly symptomatic people in COVID-19, universal use of face masks in addition to hand hygiene and safety distance seems extremely useful. Consequently, preparing the healthy child to use face masks is strongly needed. To obtain maximal compliance, reasons for mask wearing without attempts of removing must be clearly explained. Moreover, child's will must not be forced.Conclusion: On the basis of clinical findings, we think that the universal use of facial masks seems necessary when people have to go out in their everyday lives. In addition to the availability of masks of different sizes capable of adapting perfectly to the face, it is necessary that the use of masks in children is preceded by a strong parental work and school lessons on this issue and other hygiene topics with the main aim to obtain child cooperation. What is Known: â¢ Asymptomatic people can transmit and become important sources of COVID-19. â¢ Asymptomatic cases are common also in pediatrics. What is New: â¢ Universal use of face masks for success against COVID-19 seems necessary also in pediatric age when people have to go out in their everyday lives. â¢ In addition to the availability of masks of different sizes capable of adapting perfectly to the face, it is necessary that the use of masks in children is preceded by a strong parental work and school lessons with the main aim to obtain child cooperation.\n",
      "It has been reported that asymptomatic people can transmit the new coronavirus disease 2019 (COVID-19) and become important sources of COVID-19. To reduce the role of asymptomatic or poorly symptomatic people in COVID-19, universal use of face masks in addition to hand hygiene and safety distance seems extremely useful. Consequently, preparing the healthy child to use face masks is strongly needed. To obtain maximal compliance, reasons for mask wearing without attempts of removing must be clearly explained. Moreover, child's will must not be forced.Conclusion: On the basis of clinical findings, we think that the universal use of facial masks seems necessary when people have to go out in their everyday lives. In addition to the availability of masks of different sizes capable of adapting perfectly to the face, it is necessary that the use of masks in children is preceded by a strong parental work and school lessons on this issue and other hygiene topics with the main aim to obtain child cooperation.What is Known:â¢ Asymptomatic people can transmit and become important sources of COVID-19.â¢ Asymptomatic cases are common also in pediatrics.What is New:â¢ Universal use of face masks for success against COVID-19 seems necessary also in pediatric age when people have to go out in their everyday lives.â¢ In addition to the availability of masks of different sizes capable of adapting perfectly to the face, it is necessary that the use of masks in children is preceded by a strong parental work and school lessons with the main aim to obtain child cooperation.\n",
      "Since December 2019, the world has been experiencing a catastrophic pandemic of coronavirus disease (COVID-19) caused by SARS-CoV2. This virus primarily targets the human respiratory system. Available information suggests that people with epilepsy (PWE) are not at higher risk of being infected by the virus, nor of more severe COVID-19 manifestations, as a result of the epilepsy alone. However, COVID-19 is a serious disease that currently has no effective treatment or vaccine. A face mask is probably effective in preventing the spread of a respiratory pathogen, at least to some extent. So, should we recommend wearing a face mask to all during a pandemic of respiratory infectious disease (e.g., COVID-19) without any precautions or exemptions? While concrete evidence is lacking, if we consider that wearing a face mask may simulate hyperventilation, at least to some extent, we would probably avoid recommending this practice indiscriminately to all PWE. On the other hand, in the absence of any proven treatment or vaccine to combat COVID-19, prevention is the best available strategy and it is probably not reasonable to suggest avoid wearing face masks in PWE under any circumstances. Logically, PWE do not need to wear a face mask most of the time, as long as there is no close contact with others, especially during intense physical activities such as exercise. To the contrary, it is probably more advantageous to wear a face mask in crowded locations, with intermittent breaks in safe locations, away from others.\n",
      "In the context of Coronavirus Disease (2019) (COVID-19) cases globally, there is a lack of consensus across cultures on whether wearing face masks is an effective physical intervention against disease transmission. This study 1) illustrates transmission routes of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2); 2) addresses controversies surrounding the mask from perspectives of attitude, effectiveness, and necessity of wearing the mask with evidence that the use of mask would effectively interrupt the transmission of infectious diseases in both hospital settings and community settings; and 3) provides suggestion that the public should wear the mask during COVID-19 pandemic according to local context. To achieve this goal, government should establish a risk adjusted strategy of mask use to scientifically publicize the use of masks, guarantee sufficient supply of masks, and cooperate for reducing health resources inequities.\n",
      "In the context of Coronavirus Disease (2019) (COVID-19) cases globally, there is a lack of consensus across cultures on whether wearing face masks is an effective physical intervention against disease transmission. This study 1) illustrates transmission routes of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2); 2) addresses controversies surrounding the mask from perspectives of attitude, effectiveness, and necessity of wearing the mask with evidence that the use of mask would effectively interrupt the transmission of infectious diseases in both hospital settings and community settings; and 3) provides suggestion that the public should wear the mask during COVID-19 pandemic according to local context. To achieve this goal, government should establish a risk adjusted strategy of mask use to scientifically publicize the use of masks, guarantee sufficient supply of masks, and cooperate for reducing health resources inequities.\n",
      "Abstract Coronavirus disease 2019 (COVID-19) is a highly contagious disease caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Infection with SARS-CoV may cause coronary plaque instability and lead to acute coronary syndrome (ACS). Management of ACS in patients with COVID-19 needs more consideration of the balance between clinical benefit and transmission risk of virus. This review provides recommendations of management strategies for ACS in patients with suspected or confirmed COVID-19 in Taiwan.\n",
      "Severe acute respiratory syndrome‐corona virus‐2 (SARS‐CoV‐2; COVID‐19), which causes coronavirus disease 2019, is highly contagious and a particularly popular problem in all around the World and also in all departments of every hospital. Protecting the well‐being of the aesthetic dermatologists while providing a sufficient workforce is vital for pandemic planning. In this article, we will discuss this problem from an aesthetic dermatology aspect and we will review whether these procedures are safe or not.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: jc9ugexn\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "are there any clinical trials available for the coronavirus\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "The novel corona virus disease -2019 (COVID-19) pandemic has caused a massive global public health havoc. Recent clinical trials carried out in China has found a promising therapeutic application of chloroquine and hydroxychloroquine for COVID-19. This study meticulously evaluated the various dosages of chloroquine and hydroxychloroquine utilized in clinical trials registered in Chinese and US clinical trial registries for the treatment of pneumonia caused by SARS-CoV-2.\n",
      "Background: Coronavirus pandemic is currently a global public health emergency. At present, no pharmacological treatment is known to treat this condition, and there is a need to review the available treatments. Objective: While there have been studies to describe the role of chloroquine and hydroxychloroquine in various viral conditions, there is limited information about the use of them in COVID-19. This systematic review aims to summarize the available evidence regarding the role of chloroquine in treating coronavirus infection. Methods: The preferred reporting items for systematic reviews and meta-analyses (PRISMA) guidelines were used for this review. A literature search was performed using PUBMED & Google Scholar to find articles about the role of CQ in COVID-19 patients. Results: We included 19 publications (Five published articles, three letters/correspondence, one commentary, five pre-proofs of accepted articles, one abstract of yet to be published article, and four were pre-prints (not yet peer-reviewed) articles) in this systematic review. All the articles mentioned about the role of chloroquine and /or hydroxychloroquine in limiting the infection with SARS-CoV-2 (the virus causing COVID-19). Conclusions: There is theoretical, experimental, preclinical and clinical evidence of the effectiveness of chloroquine in patients affected with COVID-19. There is adequate evidence of drug safety from the long-time clinical use of chloroquine and hydroxychloroquine in other indications. More data from ongoing and future trials will add more insight into the role of chloroquine and hydroxychloroquine in COVID-19 infection.\n",
      "OBJECTIVE: The novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), also called COVID-19, has caused a pandemic which has swiftly involved the entire world and raised great public health concerns. The scientific community is actively exploring treatments that would potentially be effective in combating COVID-19. Hydroxychloroquine has been demonstrated to limit the replication of SARS-CoV-2 virus in vitro. In malarial pandemic countries, chloroquine is widely used to treat malaria. In malarial non-pandemic nations, chloroquine is not widely used. Chloroquine and hydroxychloroquine share similar chemical structures and mechanisms of action. The aim of this study was to indirectly investigate the efficacy of chloroquine and hydroxychloroquine for the treatment of COVID-19 by determining the prevalence of COVID-19 in malaria pandemic and non-pandemic nations. We sought evidence to support or refute the hypothesis that these drugs could show efficacy in the treatment of COVID-19. MATERIALS AND METHODS: We reviewed in vitro studies, in vivo studies, original studies, clinical trials, and consensus reports, that were conducted to evaluate the antiviral activities of chloroquine and hydroxychloroquine. The studies on \"COVID-19 and its allied treatment were found from World Health Organization (WHO), ISI-Web of Science, PubMed, EMBASE, Scopus, Google Scholar, and clinical trial registries. The search was based on keywords: antiviral drugs, chloroquine, hydroxychloroquine, COVID-19, COVID-19 treatment modalities, and coronavirus. In addition, we analyzed the prevalence of COVID-19 in malaria pandemic and non-pandemic countries. The review and analyses were performed on March 28, 2020. RESULTS: For this study, we identified a total of 09 published articles: 03 clinical trials with sample size 150; 03 in vitro studies and 03 expert consensus reports. These studies were all suggestive that chloroquine and hydroxychloroquine can successfully treat COVID-19 infections. We found that COVID-19 infections are highly pandemic in countries where malaria is least pandemic and are least pandemic in nations where malaria is highly pandemic. CONCLUSIONS: Chloroquine and hydroxychloroquine have antiviral characteristics in vitro. The findings support the hypothesis that these drugs have efficacy in the treatment of COVID-19. People are currently using these drugs for malaria. It is reasonable, given the hypothetical benefit of these two drugs, that they are now being tested in clinical trials to assess their effectiveness to combat this global health crisis.\n",
      "OBJECTIVE The novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), also called COVID-19, has caused a pandemic which has swiftly involved the entire world and raised great public health concerns. The scientific community is actively exploring treatments that would potentially be effective in combating COVID-19. Hydroxychloroquine has been demonstrated to limit the replication of SARS-CoV-2 virus in vitro. In malarial pandemic countries, chloroquine is widely used to treat malaria. In malarial non-pandemic nations, chloroquine is not widely used. Chloroquine and hydroxychloroquine share similar chemical structures and mechanisms of action. The aim of this study was to indirectly investigate the efficacy of chloroquine and hydroxychloroquine for the treatment of COVID-19 by determining the prevalence of COVID-19 in malaria pandemic and non-pandemic nations. We sought evidence to support or refute the hypothesis that these drugs could show efficacy in the treatment of COVID-19. MATERIALS AND METHODS We reviewed in vitro studies, in vivo studies, original studies, clinical trials, and consensus reports, that were conducted to evaluate the antiviral activities of chloroquine and hydroxychloroquine. The studies on \"COVID-19 and its allied treatment were found from World Health Organization (WHO), ISI-Web of Science, PubMed, EMBASE, Scopus, Google Scholar, and clinical trial registries. The search was based on keywords: antiviral drugs, chloroquine, hydroxychloroquine, COVID-19, COVID-19 treatment modalities, and coronavirus. In addition, we analyzed the prevalence of COVID-19 in malaria pandemic and non-pandemic countries. The review and analyses were performed on March 28, 2020. RESULTS For this study, we identified a total of 09 published articles: 03 clinical trials with sample size 150; 03 in vitro studies and 03 expert consensus reports. These studies were all suggestive that chloroquine and hydroxychloroquine can successfully treat COVID-19 infections. We found that COVID-19 infections are highly pandemic in countries where malaria is least pandemic and are least pandemic in nations where malaria is highly pandemic. CONCLUSIONS Chloroquine and hydroxychloroquine have antiviral characteristics in vitro. The findings support the hypothesis that these drugs have efficacy in the treatment of COVID-19. People are currently using these drugs for malaria. It is reasonable, given the hypothetical benefit of these two drugs, that they are now being tested in clinical trials to assess their effectiveness to combat this global health crisis.\n",
      "BACKGROUND AND OBJECTIVE: The world is currently experiencing the Coronavirus Disease-19 (COVID-19) pandemic. There is no approved drug for the definitive treatment of the disease. Various drugs are being tried for the treatment of COVID-19, including hydroxychloroquine (HCQ). This study was performed to systematically review the therapeutic role of HCQ in COVID-19 from the available literature. METHODS: PubMed, Embase, ClinicalTrials.gov, ICTRP (WHO), Cochrane Library databases, and two pre-print servers (medRxiv.org and Research Square) were searched for clinical studies that evaluated the therapeutic role of HCQ on COVID-19 until 10 May 2020. The available studies were critically analyzed and the data were extracted. RESULTS: A total of 663 articles were screened and 12 clinical studies (seven peer-reviewed and published studies and five non-peer-reviewed studies from pre-print servers) with a total sample size of 3543 patients were included. Some of the clinical studies demonstrated good virological and clinical outcomes with HCQ alone or in combination with azithromycin in COVID-19 patients, although the studies had major methodological limitations. Some of the other studies showed negative results with HCQ therapy along with the risk of adverse reactions. CONCLUSION: The results of efficacy and safety of HCQ in COVID-19, as obtained from the clinical studies, are not satisfactory, although many of these studies had major methodological limitations. Stronger evidence from well-designed robust randomized clinical trials is required before conclusively determining the role of HCQ in the treatment of COVID-19. Clinical prudence is required in advocating HCQ as a therapeutic armamentarium in COVID-19.\n",
      "Coronavirus disease 2019 (COVID-19) is now spreading as a pandemic ravaging the whole world In the absence of a vaccine and an effective antiviral chemotherapy, there is currently an intense global interest in repositioning chloroquine (CQ) and its derivative hydroxychloroquine (HCQ) to combat the pandemic CQ has been used for decades for the treatment and prophylaxis against malaria in endemic countries It is readily available and has also been manufactured in these countries CQ is cheap, stable under field conditions and has been well tolerated as an antimalarial This experience could be adapted to deploy CQ or HCQ for prophylaxis or treatment of COVID19 if strong evidence could be generated for these uses We believe that well-designed drug trials should be initiated in malaria-endemic countries, taking into account the local context of the epidemic and the capacity of the health system in combating it In this paper, we are presenting the current status of evidence for using CQ and HCQ against COVID19\n",
      "Chloroquine and closely related structural analogs, employed initially for the treatment of malaria, are now gaining worldwide attention due to the rapidly spreading pandemic caused by severe acute respiratory syndrome-coronavirus-2, named coronavirus disease (COVID) of 2019 (COVID-19). Although much of this attention has a mechanistic basis, the hard efficacy data for chloroquine/hydroxychloroquine in the management of the clinical syndrome of COVID-19 have been limited thus far. This review aims to present the available in vitro and clinical data for the role of chloroquine/hydroxychloroquine in COVID-19 and attempts to put them into perspective, especially in relation to the different risks/benefits particular to each patient who may require treatment.\n",
      "PURPOSE: COVID-19 (coronavirus disease 2019) is a public health emergency of international concern. As of this time, there is no known effective pharmaceutical treatment, although it is much needed for patient contracting the severe form of the disease. The aim of this systematic review was to summarize the evidence regarding chloroquine for the treatment of COVID-19. METHODS: PubMed, EMBASE, and three trial Registries were searched for studies on the use of chloroquine in patients with COVID-19. RESULTS: We included six articles (one narrative letter, one in-vitro study, one editorial, expert consensus paper, two national guideline documents) and 23 ongoing clinical trials in China. Chloroquine seems to be effective in limiting the replication of SARS-CoV-2 (virus causing COVID-19) in vitro. CONCLUSIONS: There is rationale, pre-clinical evidence of effectiveness and evidence of safety from long-time clinical use for other indications to justify clinical research on chloroquine in patients with COVID-19. However, clinical use should either adhere to the Monitored Emergency Use of Unregistered Interventions (MEURI) framework or be ethically approved as a trial as stated by the World Health Organization. Safety data and data from high-quality clinical trials are urgently needed.\n",
      "Since the outbreak of novel coronavirus pneumonia (COVID-19) in December 2019, more than 2,500,000 people worldwide have been diagnosed with SARS-CoV-2 as of April 22. In response to this epidemic, China has issued seven trial versions of diagnosis and treatment protocol for COVID-19. According to the information that we have collected so far, this article provides an overview of potential therapeutic drugs and compounds with much attention, including favipiravir and hydroxychloroquine, as well as traditional Chinese medicine, which have been reported with good clinical treatment effects. Moreover, with further understanding of SARS-CoV-2 virus, new drugs targeting specific SARS-CoV-2 viral components arise and investigations on these novel anti-SARS-CoV-2 agents are also reviewed.\n",
      "BACKGROUND AND AIMS: No drugs are currently approved for Coronavirus Disease-2019 (COVID-19), although some have been tried. In view of recent studies and discussion on chloroquine and hydroxychloroquine (HCQ), we aimed to review existing literature and relevant websites regarding these drugs and COVID-19, adverse effects related to drugs, and related guidelines. AIMS AND METHODS: We systematically searched the PubMed database up till March 21, 2020 and retrieved all the articles published on chloroquine and HCQ and COVID-19. RESULTS: Two small human studies have been conducted with both these drugs in COVID-19, and have shown significant improvement in some parameters in patients with COVID-19. CONCLUSION: Considering minimal risk upon use, a long experience of use in other diseases, cost-effectiveness and easy availability across India, we propose that both these drugs are worthy of fast track clinical trial for treatment, and may be carefully considered for clinical use as experimental drugs. Since HCQ has been approved for treatment of diabetes in India, it should be further researched in diabetes and COVID-19, a subgroup where significant mortality has been shown.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: vxqdfiel\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "are there any clinical trials available for the coronavirus\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "The World Health Organization (WHO) was informed on December 2019 about a coronavirus pneumonia outbreak in Wuhan, Hubei province (China). Subsequently, on March 12, 2020, 125,048 cases and 4,614 deaths were reported. Coronavirus is an enveloped RNA virus, from the genus Betacoronavirus, that is distributed in birds, humans, and other mammals. WHO has named the novel coronavirus disease as COVID-19. More than 80 clinical trials have been launched to test coronavirus treatment, including some drug repurposing or repositioning for COVID-19. Hence, we performed a search in March 2020 of the clinicaltrials.gov database. The eligibility criteria for the retrieved studies were: contain a clinicaltrials.gov base identifier number; describe the number of participants and the period for the study; describe the participants’ clinical conditions; and utilize interventions with medicines already studied or approved for any other disease in patients infected with the novel coronavirus SARS-CoV-2 (2019-nCoV). It is essential to emphasize that this article only captured trials listed in the clinicaltrials.gov database. We identified 24 clinical trials, involving more than 20 medicines, such as human immunoglobulin, interferons, chloroquine, hydroxychloroquine, arbidol, remdesivir, favipiravir, lopinavir, ritonavir, oseltamivir, methylprednisolone, bevacizumab, and traditional Chinese medicines (TCM). Although drug repurposing has some limitations, repositioning clinical trials may represent an attractive strategy because they facilitate the discovery of new classes of medicines; they have lower costs and take less time to reach the market; and there are existing pharmaceutical supply chains for formulation and distribution.\n",
      "ABSTRACT Background. The spread of COVID-19 from Wuhan in China throughout the world has been alamingly rapid. Epidemiologic techniques succeeded in containing the disease in China, but efforts were not as successful in the rest of the world, particularly the United States where there have been 1,592, 599 cases with 95,863 deaths as of May 25, 2020. Projections are for continued new infections and deaths if no effective treatments can be activated over the next six months.. We performed a systematic review to determine the potential time course of development of treatments and vaccines focusing on availability in the last half of 2020. Methods. Our search was performed during the week of May 17, 2020. Publications: We reviewed up to date information from several sources to identify potential treatments for COVID-19: We used the Reagan-Udall Expanded Access Navigator COVID-19 Treatment Hub to track the efforts of companies to develop treatments. We then used the results to search for publications on pubmed.gov and on MedRxiv, the preprint server. We further used a targeted Google search to find announcements of trial results. Clinical Trials: We searched for all investigational trials involving potential treatments using several different sources: (A) covid-trials.org, then validated results on (B) clinicaltrials.gov and the (C) World Health Organization's International Clinical Trials Registry Platform (WHO ICTRP). We focused on trials which were completed or currently recruiting for patients, reasoning that the timeline to arrive at treatments by the end of the year would require completion within the next 6 months. We excluded studies which were clearly observational, with no randomization, control or comparison group. We furtherset a cutoff of 100 for numbers of subjects since smaller trial size could lack statistical power to establish superiority of the intervention over the control condition. Results. Published Data: We found 19 publications reporting findings on 6 classes of agents. There were 9 publications related to hydroxychloroquine (HCQ), three on lopanovir/ritonavir (LPV/R) (one of which was combined with interferon beta-1),three on remdesivir, two on favipiravir, one on mepaluzamab, and one on tocilizumab. Ten publications reported randomized controlled trials, the rest were retrospective observational. Only one publication touched on outpatient management, the rest on hospitalized patients. Clinical Trials: We found 409 trials meeting our minimum requirement of 100 subjects which were recruiting or completed. The WHO has launched the Solidarity megatrial performed in over 100 countries actively comparing HCQ, lopanovir/ritonavir (LPV/R) alone and in combination with interferon beta-1, and remdesivir. That trial is scheduled to complete enrollment in the first quarter of 2021. In addition, we found 46 trials of HCQ, 11 trials of LPV/R and 8 trials of interferons. There were 18 ongoing trials of antiviral agents, 24 immune modulator trials, 9 vaccine trials, and 62 trials of other agents.We excluded a large number of trials of Chinese traditional medications, reasoning that there was insufficient clinical experience with these agents outside China to offer these treatments to the rest of the world. Forty four trials were hoping to complete enrollment by the end of the second quarter of 2020. Of these, only 9 were conducted on outpatients. A few vaccine trials are hoping to complete Phase 3 enrollment by the end of the third quarter, but a prolonged follow-up of patients will likely be required. Conclusion. Several treatments for severe disease in hospitalized patients have now been granted emergency authorization. However, the disease is propagated primarily by infected ambulatory individuals. There are only a few randomized controlled studies in outpatients which can be expected to yield results in time to impact on the continuing spread of the epidemic in 2020. It will be necessary for public health authorities to make hard decisions with limited data. The choices will be hardest in dealing with potential early release of vaccines. Keywords: Coronavirus, COVID-19, SARS-Cov-2, remdesivir, hydroxychloroquine\n",
      "The novel human coronavirus (SARS-CoV-2), the causative agent of COVID-19, has quickly become a threat to the public health and economy worldwide. Despite the severity of some cases, there are no current pathogen-specific antivirals available to treat the disease. Therefore, many studies have focused on the evaluation of the anti-SARS-CoV-2 activity of clinically available drugs. Here, we conducted a systematic review to describe the drug repositioning strategy against SARS-CoV-2 and to discuss the clinical impact of this approach in the current pandemic context. The systematic review was performed on March 23, 2020, using PubMed/MEDLINE, Scopus, Cochrane Library, and Biblioteca Virtual de Saúde (BVS). The data were summarized in tables and critically analyzed. After the database search, 12 relevant studies were identified as eligible for the review. Among the drugs reported in these studies, 57 showed some evidence of antiviral activity. Antivirals, especially antiretrovirals, are the main class of therapeutic agents evaluated against COVID-19. Moreover, studies have reported the anti-SARS-CoV-2 activity of antitumor (16%; 9/57), antimalarial (7%, 4/57), and antibacterial (5%; 3/57) agents. Additionally, seven pharmacological agents (chloroquine, tetrandrine, umifenovir (arbidol), carrimycin, damageprevir, lopinavir/ritonavir) are in phase IV of clinical trials. Due to the evidence of the anti-SARS-CoV-2 activity of various clinically available agents, drug repositioning stands out as a promising strategy for a short-term response in the fight against the novel coronavirus.\n",
      "The outbreak of coronavirus disease 2019 (COVID-19) caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) poses a serious threat to global public health and economies Currently, hundreds of clinical trials on a wide variety of treatments against COVID-19 are being conducted around the world Here, we conducted a search for ongoing clinical trials for the treatment of COVID-19 at the clinicaltrials gov database on April 2, 2020 In total, 48 clinical trials were identified, and of these, 41 trials adopted drug intervention and the other 7 trials utilized biological intervention The number of trials stratified by a chief country conducting the investigation were 18 in China, 5 in the United States, 4 in Canada, 3 in Italy, 2 in France and Brazil, and 4 trials are being performed multinationally The drugs utilized in more than one trials were remdesivir (6 trials), lopinavir/ritonavir (6 trials), hydroxychloroquine (6 trials), interferon (5 trials), methylprednisolone (3 trials), nitric oxide gas (3 trials), oseltamivir (2 trials), arbidol (2 trials), and vitamin C (2 trials) We also described the Japanese trials which are now being conducted or scheduled, utilizing lopinavir/ritonavir, remdesivir, favipiravir, ciclesonide and nafamostat\n",
      "BACKGROUND In December 2019, a new coronavirus, named Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), has emerged from China causing pneumonia outbreaks first in the Wuhan region and currently worldwide. In the light of the lack of efficient and specific treatments and the need to contain the epidemic, drug repurposing appears to be the most efficient tool to find therapeutic solution. OBJECTIVES The aim of this study was to summarize in vitro data of current agents used for the management of SARSCoV-2 all over the world. METHODS A literature search of articles from January 2000 until April 2020 was performed using MEDLINE, EMBASE and the Cochrane Library to assess in vitro data of current or putative therapies for SARS-CoV-2. RESULTS Although in vitro studies are scarce, data regarding chloroquine and hydroxychloroquine, remdesivir, nitazoxanide, teicoplanin, ivermectin, lopinavir, homoharringtonine and emetine seem promising. CONCLUSION Scientist all over the world should work together and increase their efforts in order to find feasible and efficient solutions against this new global viral threat.\n",
      "The novel human coronavirus (SARS-CoV-2), the causative agent of COVID-19, has quickly become a threat to the public health and economy worldwide. Despite the severity of some cases, there are no current pathogen-specific antivirals available to treat the disease. Therefore, many studies have focused on the evaluation of the anti-SARS-CoV-2 activity of clinically available drugs. Here, we conducted a systematic review to describe the drug repositioning strategy against SARS-CoV-2 and to discuss the clinical impact of this approach in the current pandemic context. The systematic review was performed on March 23, 2020, using PubMed/MEDLINE, Scopus, Cochrane Library, and Biblioteca Virtual de Saúde (BVS). The data were summarized in tables and critically analyzed. After the database search, 12 relevant studies were identified as eligible for the review. Among the drugs reported in these studies, 57 showed some evidence of antiviral activity. Antivirals, especially antiretrovirals, are the main class of therapeutic agents evaluated against COVID-19. Moreover, studies have reported the anti-SARS-CoV-2 activity of antitumor (16%; 9/57), antimalarial (7%, 4/57), and antibacterial (5%; 3/57) agents. Additionally, seven pharmacological agents (chloroquine, tetrandrine, umifenovir (arbidol), carrimycin, damageprevir, lopinavir/ritonavir) are in phase IV of clinical trials. Due to the evidence of the anti-SARS-CoV-2 activity of various clinically available agents, drug repositioning stands out as a promising strategy for a short-term response in the fight against the novel coronavirus. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (10.1007/s00705-020-04693-5) contains supplementary material, which is available to authorized users.\n",
      "BACKGROUND: In December 2019, a new coronavirus, named Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), has emerged from China causing pneumonia outbreaks first in the Wuhan region and currently worldwide. In the light of the lack of efficient and specific treatments and the need to contain the epidemic, drug repurposing appears to be the most efficient tool to find therapeutic solution. OBJECTIVES: The aim of this study was to summarize in vitro data of current agents used for the management of SARSCoV-2 all over the world. METHODS: A literature search of articles from January 2000 until April 2020 was performed using MEDLINE, EMBASE and the Cochrane Library to assess in vitro data of current or putative therapies for SARS-CoV-2. RESULTS: Although in vitro studies are scarce, data regarding chloroquine and hydroxychloroquine, remdesivir, nitazoxanide, teicoplanin, ivermectin, lopinavir, homoharringtonine and emetine seem promising. CONCLUSION: Scientist all over the world should work together and increase their efforts in order to find feasible and efficient solutions against this new global viral threat.\n",
      "Background: As of May 2 2020, 3,267,184 confirmed cases of COVID-19 and 229,971 COVID-19-caused deaths have been reported worldwide. Currently, there is limited clarity on the pharmacological treatments available for the novel coronavirus. We systematically identified the current evidence and ongoing research on the pharmacological treatments for COVID-19. Methods: We conducted a scoping review using PRISMA-ScR. Observational studies, including cohort studies and case series, as well as experimental studies, including randomized controlled trials (RCTs) and non-RCTs were searched electronically on April 7, 2020 and by hand on May 1, 2020. PubMed, EMBASE, and Cochrane library databases were searched along with seven trial registries. The inclusion criteria were patients with confirmed COVID-19 who received pharmacological therapies, including hydroxychloroquine and chloroquine, lopinavir/ritonavir, remdesivir, tocilizumab, and favipiravir. Results: We identified 222 studies on pharmacological treatment of the novel coronavirus. We included 11 of these studies in this review, including the ones on hydroxychloroquine and chloroquine (one cohort), lopinavir/ritonavir (one RCT, three cohorts, and two case series), remdesivir (one RCT and one case series), tocilizumab (one case series), and favipiravir (one RCT). In the three RCTs carried out in China, both lopinavir/ritonavir and remdesivir did not show any significant earlier clinical improvement in case of severe infection [Hazard ratio (HR): 1.31, p=0.09 and HR: 1.24, p=0.24, respectively], The clinical recovery rate on day seven was not significantly different between the favipiravir and arbidol groups (p=0.14) for moderate patients, although the duration of pyrexia and cough in the favipiravir group was significantly shorter as compared to the arbidol group (p<0.01). There are 135 ongoing RCTs, including 72 for hydroxychloroquine and chloroquine, 29 for lopinavir/ritonavir, 14 for remdesivir, 16 for tocilizumab, and 4 for favipiravir. Conclusion: The clinical effectiveness and safety of these drugs for the treatment of COVID-19 remains unclear owing to the lack of large, high-quality RCTs. However, in the event of emerging infectious diseases, we need to repeatedly and systematically update the best available evidence to avoid misleading information.\n",
      "BACKGROUND Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), termed coronavirus disease 2019 (COVID-19) by the World Health Organization, is a newly emerging zoonotic agent that emerged in China in December 2019. No specific treatment for COVID-19 is currently available. Usual palliative treatment includes maintaining hydration and nutrition and controlling fever and cough. The clinical severity and extent of transmission need to be determined, and therapeutic options need to be developed and optimized. METHODS The present review discusses the recent repurposing of drugs for COVID-19 treatment. RESULTS Several compounds, including remdesivir, lopinavir, ritonavir, interferon-β, ribavirin, chloroquine/hydroxychloroquine, azithromycin, tocilizumab, and ivermectin, have emerged as promising alternatives. They block the virus from entering host cells, prevent viral replication, and attenuate exacerbation of the host's immune response. CONCLUSION Although some evidence indicates the positive actions of different classes of compounds for the treatment of COVID-19, few clinical assays have been established to definitively demonstrate their therapeutic value in humans. Multicenter clinical studies are urgently needed to validate and standardize therapeutic regimens that involve these agents. Although science has not yet presented us with a specific drug against COVID-19, the repurposing of drugs appears to be promising in our fight against this devastating disease.\n",
      "PURPOSE: Recently, the world has been dealing with a new type of coronavirus called COVID-19 that in terms of symptoms is similar to the SARS coronavirus. Unfortunately, researchers could not find a registered therapy to treat the infection related to the virus yet. Regarding the fact that drug repurposing is a good strategy for epidemic viral infection, we applied the drug repurposing strategy using virtual screening to identify therapeutic options for COVID-19. For this purpose, five proteins of COVID-19 (3-chymotrypsin-like protease (3CLpro), Papain-Like protease (PLpro), cleavage site, HR1 and RBD in Spike protein) were selected as target proteins for drug repositioning. METHODS: First, five proteins of COVID-19 were built by homology modeling. Then FDA-approved drugs (2471 drugs) were screened against cleavage site and RBD in Spike protein via virtual screening. One hundred and twenty-eight FDA-approved drugs with the most favorable free-binding energy were attached to the cleavage site and RBD in Spike protein. Of these 128 drugs, 18 drugs have either been used currently as antiviral or have been reported to possess antiviral effects. Virtual screening was then performed for the 18 selected drugs with ACE2, 3CLpro and PLpro and HR1 and TMPRSS2. RESULTS: According to the results, glecaprevir, paritaprevir, simeprevir, ledipasvir, glycyrrhizic acid, TMC-310911, and hesperidin showed highly favorably free binding energies with all tested target proteins. CONCLUSION: The above-mentioned drugs can be regarded as candidates to treat COVID-19 infections, but further study on the efficiency of these drugs is also necessary.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 7y87ktmi\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "are there any clinical trials available for the coronavirus\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Therapeutic options for coronavirus disease 2019 are desperately needed to respond to the ongoing severe acute respiratory syndrome coronavirus 2 pandemic. Both antiviral drugs and immunomodulators might have their place in the management of coronavirus disease 2019. Unfortunately, no drugs have been approved yet to treat infections with human coronaviruses. As it will take years to develop new therapies for severe acute respiratory syndrome coronavirus 2, the current focus is on the repurposing of drugs that have been approved or are in development for other conditions. Several clinical trials have already been conducted or are currently ongoing to evaluate the efficacy of such drugs. Here, we discuss the potential of these therapies for the treatment of coronavirus disease 2019.\n",
      "Abstract The novel severe acute respiratory syndrome coronavirus 2 is causing a worldwide pandemic that may lead to a highly morbid and potentially fatal coronavirus disease-19 (COVID-19). There is currently no drug that has been proven as an effective therapy for COVID-19. Several candidate drugs are being considered and evaluated for treatment. This includes clinically-available drugs, such as chloroquine, hydroxychloroquine, and lopinavir/ritonavir, which are being repurposed for the treatment of COVID-19. Novel experimental therapies, such as remdesivir and favipiravir, are also actively being investigated for antiviral efficacy. Clinically-available and investigational immunomodulators, such as the IL-6 inhibitors tocilizumab and sarilumab and the anti-GMCSF lenzilumab, are being tested for their anticipated effect in counteracting the pro-inflammatory cytokine environment that characterizes severe and critical COVID-19. This review article examines the evidence behind the potential use of these leading drug candidates for the treatment of COVID-19. The authors conclude, based on this review, that there is still no high-quality evidence to support any of these proposed drug therapies. The authors, therefore, encourage the enrollment of eligible patients to multiple ongoing clinical trials that assess the efficacy and safety of these candidate therapies. Until the results of controlled trials are available, none of the suggested therapeutics is clinically proven as an effective therapy for COVID-19.\n",
      "BACKGROUND Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), termed coronavirus disease 2019 (COVID-19) by the World Health Organization, is a newly emerging zoonotic agent that emerged in China in December 2019. No specific treatment for COVID-19 is currently available. Usual palliative treatment includes maintaining hydration and nutrition and controlling fever and cough. The clinical severity and extent of transmission need to be determined, and therapeutic options need to be developed and optimized. METHODS The present review discusses the recent repurposing of drugs for COVID-19 treatment. RESULTS Several compounds, including remdesivir, lopinavir, ritonavir, interferon-β, ribavirin, chloroquine/hydroxychloroquine, azithromycin, tocilizumab, and ivermectin, have emerged as promising alternatives. They block the virus from entering host cells, prevent viral replication, and attenuate exacerbation of the host's immune response. CONCLUSION Although some evidence indicates the positive actions of different classes of compounds for the treatment of COVID-19, few clinical assays have been established to definitively demonstrate their therapeutic value in humans. Multicenter clinical studies are urgently needed to validate and standardize therapeutic regimens that involve these agents. Although science has not yet presented us with a specific drug against COVID-19, the repurposing of drugs appears to be promising in our fight against this devastating disease.\n",
      "The novel severe acute respiratory syndrome coronavirus 2 is the cause of Coronavirus Disease 2019, a new illness with no effective treatment or vaccine that has reached pandemic proportions. In this document, we analyze how health authorities and agencies around the world position themselves regarding the off-label use of repurposed drugs or new investigational drugs to treat Coronavirus Disease 2019. We review the most promising candidate medicines, including available evidence, clinical recommendations and current options for access. Our concluding remarks stress the importance of administering off-label and investigational drugs in the setting of clinical trials, or at least in standardized scenarios, to generate as much scientific knowledge as achievable while engaging in the best efforts to treat patients and save lives.\n",
      "The novel severe acute respiratory syndrome coronavirus 2 is the cause of Coronavirus Disease 2019, a new illness with no effective treatment or vaccine that has reached pandemic proportions. In this document, we analyze how health authorities and agencies around the world position themselves regarding the off-label use of repurposed drugs or new investigational drugs to treat Coronavirus Disease 2019. We review the most promising candidate medicines, including available evidence, clinical recommendations and current options for access. Our concluding remarks stress the importance of administering off-label and investigational drugs in the setting of clinical trials, or at least in standardized scenarios, to generate as much scientific knowledge as achievable while engaging in the best efforts to treat patients and save lives.\n",
      "BACKGROUND: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), termed coronavirus disease 2019 (COVID-19) by the World Health Organization, is a newly emerging zoonotic agent that emerged in China in December 2019. No specific treatment for COVID-19 is currently available. Usual palliative treatment includes maintaining hydration and nutrition and controlling fever and cough. The clinical severity and extent of transmission need to be determined, and therapeutic options need to be developed and optimized. METHODS: The present review discusses the recent repurposing of drugs for COVID-19 treatment. RESULTS: Several compounds, including remdesivir, lopinavir, ritonavir, interferon-ß, ribavirin, chloroquine/hydroxychloroquine, azithromycin, tocilizumab, and ivermectin, have emerged as promising alternatives. They block the virus from entering host cells, prevent viral replication, and attenuate exacerbation of the host's immune response. CONCLUSION: Although some evidence indicates the positive actions of different classes of compounds for the treatment of COVID-19, few clinical assays have been established to definitively demonstrate their therapeutic value in humans. Multicenter clinical studies are urgently needed to validate and standardize therapeutic regimens that involve these agents. Although science has not yet presented us with a specific drug against COVID-19, the repurposing of drugs appears to be promising in our fight against this devastating disease.\n",
      "The major impact produced by the severe acute respiratory syndrome coronavirus 2 (SARS­CoV­2) focused many researchers attention to find treatments that can suppress transmission or ameliorate the disease. Despite the very fast and large flow of scientific data on possible treatment solutions, none have yet demonstrated unequivocal clinical utility against coronavirus disease 2019 (COVID­19). This work represents an exhaustive and critical review of all available data on potential treatments for COVID­19, highlighting their mechanistic characteristics and the strategy development rationale. Drug repurposing, also known as drug repositioning, and target based methods are the most used strategies to advance therapeutic solutions into clinical practice. Current in silico, in vitro and in vivo evidence regarding proposed treatments are summarized providing strong support for future research efforts.\n",
      "The current pandemic of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has presented unprecedented challenges to the healthcare systems in almost every country around the world. Currently, there are no proven effective vaccines or therapeutic agents against the virus. Current clinical management includes infection prevention and control measures and supportive care including supplemental oxygen and mechanical ventilatory support. Evolving research and clinical data regarding the virologic SARS-CoV-2 suggest a potential list of repurposed drugs with appropriate pharmacological effects and therapeutic efficacies in treating COVID-19 patients. In this review, we will update and summarize the most common and plausible drugs for the treatment of COVID-19 patients. These drugs and therapeutic agents include antiviral agents (remdesivir, hydroxychloroquine, chloroquine, lopinavir, umifenovir, favipiravir, and oseltamivir), and supporting agents (Ascorbic acid, Azithromycin, Corticosteroids, Nitric oxide, IL-6 antagonists), among others. We hope that this review will provide useful and most updated therapeutic drugs to prevent, control, and treat COVID-19 patients until the approval of vaccines and specific drugs targeting SARS-CoV-2.\n",
      "Novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) lies behind the ongoing outbreak of coronavirus disease 2019 (COVID-19). There is a growing understanding of SARS-CoV-2 in virology, epidemiology, and clinical management strategies. However, no anti-SARS-CoV-2 drug or vaccine has been officially approved due to the absence of adequate evidence. Scientists are racing to develop a treatment for COVID-19. Recent studies have revealed many attractive therapeutic options, even if some of them remain to be further confirmed in rigorous preclinical models and clinical trials. In this minireview, we aim to summarize the updated potential approaches against SARS-CoV-2. We emphasize that further efforts are warranted to develop the safest and most effective approach.\n",
      "Coronavirus disease‐19 (COVID‐19), caused by severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2), is one of the most contagious diseases in human history that has already affected millions of lives worldwide. To date, no vaccines or effective therapeutics have been discovered yet that may successfully treat COVID‐19 patients or contain the transmission of the virus. Scientific communities across the globe responded rapidly and have been working relentlessly to develop drugs and vaccines, which may require considerable time. In this uncertainty, repurposing the existing antiviral drugs could be the best strategy to speed up the discovery of effective therapeutics against SARS‐CoV‐2. Moreover, drug repurposing may leave some vital information on druggable targets that could be capitalized in target‐based drug discovery. Information on possible drug targets and the progress on therapeutic and vaccine development also needs to be updated. In this review, we revisited the druggable targets that may hold promise in the development of the anti‐SARS‐CoV‐2 agent. Progresses on the development of potential therapeutics and vaccines that are under the preclinical studies and clinical trials have been highlighted. We anticipate that this review will provide valuable information that would help to accelerate the development of therapeutics and vaccines against SARS‐CoV‐2 infection.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: eitnkdi9\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "The emerging outbreak of coronavirus disease 2019 (COVID-19) caused by the severe acute respiratory syndrome coronavirus 2 continues to spread all over the world. Agents or vaccines of proven efficacy to treat or prevent human coronavirus infection are in urgent need and are being investigated vigorously worldwide. This review summarizes the current evidence of potential therapeutic agents, such as lopinavir/ritonavir, remdesivir, favipiravir, chloroquine, hydroxychloroquine, interferon, ribavirin, tocilizumab, and sarilumab. More clinical trials are being conducted for further confirmation of the efficacy and safety of these agents in treating COVID-19.\n",
      "The Coronavirus disease 19 (COVID-19) outbreak has been recognized as a global threat to public health. It is caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and no effective therapies currently exist against this novel viral agent. Along with extensive public health measures, an unprecedented global effort in identifying effective drugs for the treatment is being implemented. Potential drug targets are emerging as the result of a fast-evolving understanding of SARS-CoV-2 virology, host response to the infection, and clinical course of the disease. This brief review focuses on the latest and most promising pharmacological treatments against COVID-19 currently under investigation and discuss their potential use based on either documented efficacy in similar viral infections, or their activity against inflammatory syndromes. Ongoing clinical trials are also emphasized.\n",
      "The Coronavirus disease 19 (COVID-19) outbreak has been recognized as a global threat to public health. It is caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and no effective therapies currently exist against this novel viral agent. Along with extensive public health measures, an unprecedented global effort in identifying effective drugs for the treatment is being implemented. Potential drug targets are emerging as the result of a fast-evolving understanding of SARS-CoV-2 virology, host response to the infection, and clinical course of the disease. This brief review focuses on the latest and most promising pharmacological treatments against COVID-19 currently under investigation and discuss their potential use based on either documented efficacy in similar viral infections, or their activity against inflammatory syndromes. Ongoing clinical trials are also emphasized.\n",
      "Abstract The novel severe acute respiratory syndrome coronavirus 2 is causing a worldwide pandemic that may lead to a highly morbid and potentially fatal coronavirus disease-19 (COVID-19). There is currently no drug that has been proven as an effective therapy for COVID-19. Several candidate drugs are being considered and evaluated for treatment. This includes clinically-available drugs, such as chloroquine, hydroxychloroquine, and lopinavir/ritonavir, which are being repurposed for the treatment of COVID-19. Novel experimental therapies, such as remdesivir and favipiravir, are also actively being investigated for antiviral efficacy. Clinically-available and investigational immunomodulators, such as the IL-6 inhibitors tocilizumab and sarilumab and the anti-GMCSF lenzilumab, are being tested for their anticipated effect in counteracting the pro-inflammatory cytokine environment that characterizes severe and critical COVID-19. This review article examines the evidence behind the potential use of these leading drug candidates for the treatment of COVID-19. The authors conclude, based on this review, that there is still no high-quality evidence to support any of these proposed drug therapies. The authors, therefore, encourage the enrollment of eligible patients to multiple ongoing clinical trials that assess the efficacy and safety of these candidate therapies. Until the results of controlled trials are available, none of the suggested therapeutics is clinically proven as an effective therapy for COVID-19.\n",
      "The current pandemic of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has presented unprecedented challenges to the healthcare systems in almost every country around the world. Currently, there are no proven effective vaccines or therapeutic agents against the virus. Current clinical management includes infection prevention and control measures and supportive care including supplemental oxygen and mechanical ventilatory support. Evolving research and clinical data regarding the virologic SARS-CoV-2 suggest a potential list of repurposed drugs with appropriate pharmacological effects and therapeutic efficacies in treating COVID-19 patients. In this review, we will update and summarize the most common and plausible drugs for the treatment of COVID-19 patients. These drugs and therapeutic agents include antiviral agents (remdesivir, hydroxychloroquine, chloroquine, lopinavir, umifenovir, favipiravir, and oseltamivir), and supporting agents (Ascorbic acid, Azithromycin, Corticosteroids, Nitric oxide, IL-6 antagonists), among others. We hope that this review will provide useful and most updated therapeutic drugs to prevent, control, and treat COVID-19 patients until the approval of vaccines and specific drugs targeting SARS-CoV-2.\n",
      "Despite intense research there is currently no effective vaccine available against the new severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) emerged in the later 2019 and responsible for the COVID-19 pandemic. This infectious and communicable disease has become one of the major public health challenges in the world. The clinical management of COVID-19 has been limited to infection prevention and control measures associated with supportive care such as supplemental oxygen and mechanical ventilation. Meanwhile efforts to find an effective treatment to inhibit virus replication, mitigate the symptoms, increase survival and decrease mortality rate are ongoing. Several classes of drugs, many of them already in use for other diseases, are being evaluated based on the body of clinical knowledge obtained from infected patients regarding to the natural history and evolution of the infection. Herein we will provide an updated overview of the natural history and current knowledge on drugs and therapeutic agents being tested for the prevention and treatment of COVID-19. These include different classes of drugs such as antiviral agents (chloroquine, ivermectin, nitazoxanide, hydroxychloroquine, lopinavir, remdesivir, tocilizumab), supporting agents (Vitamin C, Vitamin D, azithromycin, corticosteroids) and promising investigational vaccines. Considering the controversies and excessive number of compounds being tested and reported in the literature we hope that this review can provide useful and updated consolidated information on potential drugs used to prevent, control and treat COVID-19 patients worldwide.\n",
      "Importance The pandemic of coronavirus disease 2019 (COVID-19) caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) presents an unprecedented challenge to identify effective drugs for prevention and treatment. Given the rapid pace of scientific discovery and clinical data generated by the large number of people rapidly infected by SARS-CoV-2, clinicians need accurate evidence regarding effective medical treatments for this infection. Observations No proven effective therapies for this virus currently exist. The rapidly expanding knowledge regarding SARS-CoV-2 virology provides a significant number of potential drug targets. The most promising therapy is remdesivir. Remdesivir has potent in vitro activity against SARS-CoV-2, but it is not US Food and Drug Administration approved and currently is being tested in ongoing randomized trials. Oseltamivir has not been shown to have efficacy, and corticosteroids are currently not recommended. Current clinical evidence does not support stopping angiotensin-converting enzyme inhibitors or angiotensin receptor blockers in patients with COVID-19. Conclusions and Relevance The COVID-19 pandemic represents the greatest global public health crisis of this generation and, potentially, since the pandemic influenza outbreak of 1918. The speed and volume of clinical trials launched to investigate potential therapies for COVID-19 highlight both the need and capability to produce high-quality evidence even in the middle of a pandemic. No therapies have been shown effective to date.\n",
      "Importance: The pandemic of coronavirus disease 2019 (COVID-19) caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) presents an unprecedented challenge to identify effective drugs for prevention and treatment. Given the rapid pace of scientific discovery and clinical data generated by the large number of people rapidly infected by SARS-CoV-2, clinicians need accurate evidence regarding effective medical treatments for this infection. Observations: No proven effective therapies for this virus currently exist. The rapidly expanding knowledge regarding SARS-CoV-2 virology provides a significant number of potential drug targets. The most promising therapy is remdesivir. Remdesivir has potent in vitro activity against SARS-CoV-2, but it is not US Food and Drug Administration approved and currently is being tested in ongoing randomized trials. Oseltamivir has not been shown to have efficacy, and corticosteroids are currently not recommended. Current clinical evidence does not support stopping angiotensin-converting enzyme inhibitors or angiotensin receptor blockers in patients with COVID-19. Conclusions and Relevance: The COVID-19 pandemic represents the greatest global public health crisis of this generation and, potentially, since the pandemic influenza outbreak of 1918. The speed and volume of clinical trials launched to investigate potential therapies for COVID-19 highlight both the need and capability to produce high-quality evidence even in the middle of a pandemic. No therapies have been shown effective to date.\n",
      "BACKGROUND Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), termed coronavirus disease 2019 (COVID-19) by the World Health Organization, is a newly emerging zoonotic agent that emerged in China in December 2019. No specific treatment for COVID-19 is currently available. Usual palliative treatment includes maintaining hydration and nutrition and controlling fever and cough. The clinical severity and extent of transmission need to be determined, and therapeutic options need to be developed and optimized. METHODS The present review discusses the recent repurposing of drugs for COVID-19 treatment. RESULTS Several compounds, including remdesivir, lopinavir, ritonavir, interferon-β, ribavirin, chloroquine/hydroxychloroquine, azithromycin, tocilizumab, and ivermectin, have emerged as promising alternatives. They block the virus from entering host cells, prevent viral replication, and attenuate exacerbation of the host's immune response. CONCLUSION Although some evidence indicates the positive actions of different classes of compounds for the treatment of COVID-19, few clinical assays have been established to definitively demonstrate their therapeutic value in humans. Multicenter clinical studies are urgently needed to validate and standardize therapeutic regimens that involve these agents. Although science has not yet presented us with a specific drug against COVID-19, the repurposing of drugs appears to be promising in our fight against this devastating disease.\n",
      "BACKGROUND: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), termed coronavirus disease 2019 (COVID-19) by the World Health Organization, is a newly emerging zoonotic agent that emerged in China in December 2019. No specific treatment for COVID-19 is currently available. Usual palliative treatment includes maintaining hydration and nutrition and controlling fever and cough. The clinical severity and extent of transmission need to be determined, and therapeutic options need to be developed and optimized. METHODS: The present review discusses the recent repurposing of drugs for COVID-19 treatment. RESULTS: Several compounds, including remdesivir, lopinavir, ritonavir, interferon-ß, ribavirin, chloroquine/hydroxychloroquine, azithromycin, tocilizumab, and ivermectin, have emerged as promising alternatives. They block the virus from entering host cells, prevent viral replication, and attenuate exacerbation of the host's immune response. CONCLUSION: Although some evidence indicates the positive actions of different classes of compounds for the treatment of COVID-19, few clinical assays have been established to definitively demonstrate their therapeutic value in humans. Multicenter clinical studies are urgently needed to validate and standardize therapeutic regimens that involve these agents. Although science has not yet presented us with a specific drug against COVID-19, the repurposing of drugs appears to be promising in our fight against this devastating disease.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: iymmf4k6\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Abstract The novel severe acute respiratory syndrome coronavirus 2 is causing a worldwide pandemic that may lead to a highly morbid and potentially fatal coronavirus disease-19 (COVID-19). There is currently no drug that has been proven as an effective therapy for COVID-19. Several candidate drugs are being considered and evaluated for treatment. This includes clinically-available drugs, such as chloroquine, hydroxychloroquine, and lopinavir/ritonavir, which are being repurposed for the treatment of COVID-19. Novel experimental therapies, such as remdesivir and favipiravir, are also actively being investigated for antiviral efficacy. Clinically-available and investigational immunomodulators, such as the IL-6 inhibitors tocilizumab and sarilumab and the anti-GMCSF lenzilumab, are being tested for their anticipated effect in counteracting the pro-inflammatory cytokine environment that characterizes severe and critical COVID-19. This review article examines the evidence behind the potential use of these leading drug candidates for the treatment of COVID-19. The authors conclude, based on this review, that there is still no high-quality evidence to support any of these proposed drug therapies. The authors, therefore, encourage the enrollment of eligible patients to multiple ongoing clinical trials that assess the efficacy and safety of these candidate therapies. Until the results of controlled trials are available, none of the suggested therapeutics is clinically proven as an effective therapy for COVID-19.\n",
      "The emerging outbreak of coronavirus disease 2019 (COVID-19) caused by the severe acute respiratory syndrome coronavirus 2 continues to spread all over the world. Agents or vaccines of proven efficacy to treat or prevent human coronavirus infection are in urgent need and are being investigated vigorously worldwide. This review summarizes the current evidence of potential therapeutic agents, such as lopinavir/ritonavir, remdesivir, favipiravir, chloroquine, hydroxychloroquine, interferon, ribavirin, tocilizumab, and sarilumab. More clinical trials are being conducted for further confirmation of the efficacy and safety of these agents in treating COVID-19.\n",
      "Introduction: The coronavirus disease 2019 (COVID-19) caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has developed since December 2019. It has caused a global pandemic with more than three hundred thousand case fatalities. However, apart from supportive care by respirators, no standard medical therapy is validated.Areas covered: This paper presents old drugs with potential in vitro efficacy against SARS-CoV-2. The in vitro database, adverse effects, and potential toxicities of these drugs are reviewed regarding their feasibility of clinical prescription for the treatment of patients with COVID-19. To obtain convincing recommendations, we referred to opinions from the US National Institute of Health regarding drugs repurposed for COVID-19 therapy.Expert opinion: Although strong evidence of well-designed randomized controlled studies regarding COVID-19 therapy is presently lacking, remdesivir, teicoplanin, hydroxychloroquine (not in combination with azithromycin), and ivermectin might be effective antiviral drugs and are deemed promising candidates for controlling SARS-CoV-2. In addition, tocilizumab might be considered as the supplementary treatment for COVID-19 patients with cytokine release syndrome. In future, clinical trials regarding a combination of potentially effective drugs against SARS-CoV-2 need to be conducted to establish the optimal regimen for the treatment of patients with moderate-to-severe COVID-19.\n",
      "INTRODUCTION: The coronavirus disease 2019 (COVID-19) caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has developed since December 2019. It has caused a global pandemic with more than three hundred thousand case fatalities. However, apart from supportive care by respirators, no standard medical therapy is validated. AREAS COVERED: This paper presents old drugs with potential in vitro efficacy against SARS-CoV-2. The in vitro database, adverse effects, and potential toxicities of these drugs are reviewed regarding their feasibility of clinical prescription for the treatment of patients with COVID-19. To obtain convincing recommendations, we referred to opinions from the US National Institute of Health regarding drugs repurposed for COVID-19 therapy. EXPERT OPINION: Although strong evidence of well-designed randomized controlled studies regarding COVID-19 therapy is presently lacking, remdesivir, teicoplanin, hydroxychloroquine (not in combination with azithromycin), and ivermectin might be effective antiviral drugs and are deemed promising candidates for controlling SARS-CoV-2. In addition, tocilizumab might be considered as the supplementary treatment for COVID-19 patients with cytokine release syndrome. In future, clinical trials regarding a combination of potentially effective drugs against SARS-CoV-2 need to be conducted to establish the optimal regimen for the treatment of patients with moderate-to-severe COVID-19.\n",
      "BACKGROUND Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), termed coronavirus disease 2019 (COVID-19) by the World Health Organization, is a newly emerging zoonotic agent that emerged in China in December 2019. No specific treatment for COVID-19 is currently available. Usual palliative treatment includes maintaining hydration and nutrition and controlling fever and cough. The clinical severity and extent of transmission need to be determined, and therapeutic options need to be developed and optimized. METHODS The present review discusses the recent repurposing of drugs for COVID-19 treatment. RESULTS Several compounds, including remdesivir, lopinavir, ritonavir, interferon-β, ribavirin, chloroquine/hydroxychloroquine, azithromycin, tocilizumab, and ivermectin, have emerged as promising alternatives. They block the virus from entering host cells, prevent viral replication, and attenuate exacerbation of the host's immune response. CONCLUSION Although some evidence indicates the positive actions of different classes of compounds for the treatment of COVID-19, few clinical assays have been established to definitively demonstrate their therapeutic value in humans. Multicenter clinical studies are urgently needed to validate and standardize therapeutic regimens that involve these agents. Although science has not yet presented us with a specific drug against COVID-19, the repurposing of drugs appears to be promising in our fight against this devastating disease.\n",
      "Importance The pandemic of coronavirus disease 2019 (COVID-19) caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) presents an unprecedented challenge to identify effective drugs for prevention and treatment. Given the rapid pace of scientific discovery and clinical data generated by the large number of people rapidly infected by SARS-CoV-2, clinicians need accurate evidence regarding effective medical treatments for this infection. Observations No proven effective therapies for this virus currently exist. The rapidly expanding knowledge regarding SARS-CoV-2 virology provides a significant number of potential drug targets. The most promising therapy is remdesivir. Remdesivir has potent in vitro activity against SARS-CoV-2, but it is not US Food and Drug Administration approved and currently is being tested in ongoing randomized trials. Oseltamivir has not been shown to have efficacy, and corticosteroids are currently not recommended. Current clinical evidence does not support stopping angiotensin-converting enzyme inhibitors or angiotensin receptor blockers in patients with COVID-19. Conclusions and Relevance The COVID-19 pandemic represents the greatest global public health crisis of this generation and, potentially, since the pandemic influenza outbreak of 1918. The speed and volume of clinical trials launched to investigate potential therapies for COVID-19 highlight both the need and capability to produce high-quality evidence even in the middle of a pandemic. No therapies have been shown effective to date.\n",
      "BACKGROUND: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), termed coronavirus disease 2019 (COVID-19) by the World Health Organization, is a newly emerging zoonotic agent that emerged in China in December 2019. No specific treatment for COVID-19 is currently available. Usual palliative treatment includes maintaining hydration and nutrition and controlling fever and cough. The clinical severity and extent of transmission need to be determined, and therapeutic options need to be developed and optimized. METHODS: The present review discusses the recent repurposing of drugs for COVID-19 treatment. RESULTS: Several compounds, including remdesivir, lopinavir, ritonavir, interferon-ß, ribavirin, chloroquine/hydroxychloroquine, azithromycin, tocilizumab, and ivermectin, have emerged as promising alternatives. They block the virus from entering host cells, prevent viral replication, and attenuate exacerbation of the host's immune response. CONCLUSION: Although some evidence indicates the positive actions of different classes of compounds for the treatment of COVID-19, few clinical assays have been established to definitively demonstrate their therapeutic value in humans. Multicenter clinical studies are urgently needed to validate and standardize therapeutic regimens that involve these agents. Although science has not yet presented us with a specific drug against COVID-19, the repurposing of drugs appears to be promising in our fight against this devastating disease.\n",
      "Abstract The outbreak of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection has posed the world at a pandemic risk. Coronavirus-19 disease (COVID-19) is an infectious disease caused by SARS-CoV-2, which causes pneumonia, requires intensive care unit hospitalization in about 10% of cases and can lead to a fatal outcome. Several efforts are currently made to find a treatment for COVID-19 patients. So far, several anti-viral and immunosuppressive or immunomodulating drugs have demonstrated some efficacy on COVID-19 both in vitro and in animal models as well as in cases series. In COVID-19 patients a pro-inflammatory status with high levels of interleukin (IL)-1B, IL-1 receptor (R)A and tumor necrosis factor (TNF)-α has been demonstrated. Moreover, high levels of IL-6 and TNF-α have been observed in patients requiring intensive-care-unit hospitalization. This provided rationale for the use of anti-rheumatic drugs as potential treatments for this severe viral infection. Other agents, such as hydroxychloroquine and chloroquine might have a direct anti-viral effect. The anti-viral aspect of immunosuppressants towards a variety of viruses has been known since long time and it is herein discussed in the view of searching for a potential treatment for SARS-CoV-2 infection.\n",
      "COVID-19 has grown into a global pandemic that has strained healthcare throughout the world. There is a sense of urgency in finding a cure for this deadly virus. In this study, we reviewed the empiric options used in common practice for COVID-19, based on the literature available online, with an emphasis on human experiences with these treatments on severe acute respiratory syndrome-associated coronavirus (SARS-COV-1) and other viruses. Convalescent blood products are the most promising potential treatment for use in COVID-19. The use of chloroquine or hydroxychloroquine (HCQ), remdesivir, and tocilizumab are some of the other promising potential therapies; however, they are yet to be tested in randomized clinical trials (RCTs). The use of lopinavir-ritonavir did not prove beneficial in a large RCT. The use of corticosteroids should be avoided in COVID-19 pneumonia unless used for other indications, based on the suggestion of harm in patients with SARS-COV-1 and Middle Eastern Respiratory Syndrome (MERS) infection. The reviews of this paper are available via the supplemental material section.\n",
      "Importance: The pandemic of coronavirus disease 2019 (COVID-19) caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) presents an unprecedented challenge to identify effective drugs for prevention and treatment. Given the rapid pace of scientific discovery and clinical data generated by the large number of people rapidly infected by SARS-CoV-2, clinicians need accurate evidence regarding effective medical treatments for this infection. Observations: No proven effective therapies for this virus currently exist. The rapidly expanding knowledge regarding SARS-CoV-2 virology provides a significant number of potential drug targets. The most promising therapy is remdesivir. Remdesivir has potent in vitro activity against SARS-CoV-2, but it is not US Food and Drug Administration approved and currently is being tested in ongoing randomized trials. Oseltamivir has not been shown to have efficacy, and corticosteroids are currently not recommended. Current clinical evidence does not support stopping angiotensin-converting enzyme inhibitors or angiotensin receptor blockers in patients with COVID-19. Conclusions and Relevance: The COVID-19 pandemic represents the greatest global public health crisis of this generation and, potentially, since the pandemic influenza outbreak of 1918. The speed and volume of clinical trials launched to investigate potential therapies for COVID-19 highlight both the need and capability to produce high-quality evidence even in the middle of a pandemic. No therapies have been shown effective to date.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: yth9hh5q\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Importance: The pandemic of coronavirus disease 2019 (COVID-19) caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) presents an unprecedented challenge to identify effective drugs for prevention and treatment. Given the rapid pace of scientific discovery and clinical data generated by the large number of people rapidly infected by SARS-CoV-2, clinicians need accurate evidence regarding effective medical treatments for this infection. Observations: No proven effective therapies for this virus currently exist. The rapidly expanding knowledge regarding SARS-CoV-2 virology provides a significant number of potential drug targets. The most promising therapy is remdesivir. Remdesivir has potent in vitro activity against SARS-CoV-2, but it is not US Food and Drug Administration approved and currently is being tested in ongoing randomized trials. Oseltamivir has not been shown to have efficacy, and corticosteroids are currently not recommended. Current clinical evidence does not support stopping angiotensin-converting enzyme inhibitors or angiotensin receptor blockers in patients with COVID-19. Conclusions and Relevance: The COVID-19 pandemic represents the greatest global public health crisis of this generation and, potentially, since the pandemic influenza outbreak of 1918. The speed and volume of clinical trials launched to investigate potential therapies for COVID-19 highlight both the need and capability to produce high-quality evidence even in the middle of a pandemic. No therapies have been shown effective to date.\n",
      "Importance The pandemic of coronavirus disease 2019 (COVID-19) caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) presents an unprecedented challenge to identify effective drugs for prevention and treatment. Given the rapid pace of scientific discovery and clinical data generated by the large number of people rapidly infected by SARS-CoV-2, clinicians need accurate evidence regarding effective medical treatments for this infection. Observations No proven effective therapies for this virus currently exist. The rapidly expanding knowledge regarding SARS-CoV-2 virology provides a significant number of potential drug targets. The most promising therapy is remdesivir. Remdesivir has potent in vitro activity against SARS-CoV-2, but it is not US Food and Drug Administration approved and currently is being tested in ongoing randomized trials. Oseltamivir has not been shown to have efficacy, and corticosteroids are currently not recommended. Current clinical evidence does not support stopping angiotensin-converting enzyme inhibitors or angiotensin receptor blockers in patients with COVID-19. Conclusions and Relevance The COVID-19 pandemic represents the greatest global public health crisis of this generation and, potentially, since the pandemic influenza outbreak of 1918. The speed and volume of clinical trials launched to investigate potential therapies for COVID-19 highlight both the need and capability to produce high-quality evidence even in the middle of a pandemic. No therapies have been shown effective to date.\n",
      "BACKGROUND: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), termed coronavirus disease 2019 (COVID-19) by the World Health Organization, is a newly emerging zoonotic agent that emerged in China in December 2019. No specific treatment for COVID-19 is currently available. Usual palliative treatment includes maintaining hydration and nutrition and controlling fever and cough. The clinical severity and extent of transmission need to be determined, and therapeutic options need to be developed and optimized. METHODS: The present review discusses the recent repurposing of drugs for COVID-19 treatment. RESULTS: Several compounds, including remdesivir, lopinavir, ritonavir, interferon-ß, ribavirin, chloroquine/hydroxychloroquine, azithromycin, tocilizumab, and ivermectin, have emerged as promising alternatives. They block the virus from entering host cells, prevent viral replication, and attenuate exacerbation of the host's immune response. CONCLUSION: Although some evidence indicates the positive actions of different classes of compounds for the treatment of COVID-19, few clinical assays have been established to definitively demonstrate their therapeutic value in humans. Multicenter clinical studies are urgently needed to validate and standardize therapeutic regimens that involve these agents. Although science has not yet presented us with a specific drug against COVID-19, the repurposing of drugs appears to be promising in our fight against this devastating disease.\n",
      "Abstract An outbreak related to the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) was first reported in Wuhan, China in December 2019. An extremely high potential for dissemination resulted in the global coronavirus disease 2019 (COVID-19) pandemic in 2020. Despite the worsening trends of COVID-19, no drugs are validated to have significant efficacy in clinical treatment of COVID-19 patients in large-scale studies. Remdesivir is considered the most promising antiviral agent; it works by inhibiting the activity of RNA-dependent RNA polymerase (RdRp). A large-scale study investigating the clinical efficacy of remdesivir (200 mg on day 1, followed by 100 mg once daily) is on-going. The other excellent anti-influenza RdRp inhibitor favipiravir is also being clinically evaluated for its efficacy in COVID-19 patients. The protease inhibitor lopinavir/ritonavir (LPV/RTV) alone is not shown to provide better antiviral efficacy than standard care. However, the regimen of LPV/RTV plus ribavirin was shown to be effective against SARS-CoV in vitro. Another promising alternative is hydroxychloroquine (200 mg thrice daily) plus azithromycin (500 mg on day 1, followed by 250 mg once daily on day 2-5), which showed excellent clinical efficacy on Chinese COVID-19 patients and anti-SARS-CoV-2 potency in vitro. The roles of teicoplanin (which inhibits the viral genome exposure in cytoplasm) and monoclonal and polyclonal antibodies in the treatment of SARS-CoV-2 are under investigation. Avoiding the prescription of non-steroidal anti-inflammatory drugs, angiotensin converting enzyme inhibitors, or angiotensin II type I receptor blockers is advised for COVID-19 patients.\n",
      "The Coronavirus disease 19 (COVID-19) outbreak has been recognized as a global threat to public health. It is caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and no effective therapies currently exist against this novel viral agent. Along with extensive public health measures, an unprecedented global effort in identifying effective drugs for the treatment is being implemented. Potential drug targets are emerging as the result of a fast-evolving understanding of SARS-CoV-2 virology, host response to the infection, and clinical course of the disease. This brief review focuses on the latest and most promising pharmacological treatments against COVID-19 currently under investigation and discuss their potential use based on either documented efficacy in similar viral infections, or their activity against inflammatory syndromes. Ongoing clinical trials are also emphasized.\n",
      "The Coronavirus disease 19 (COVID-19) outbreak has been recognized as a global threat to public health. It is caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and no effective therapies currently exist against this novel viral agent. Along with extensive public health measures, an unprecedented global effort in identifying effective drugs for the treatment is being implemented. Potential drug targets are emerging as the result of a fast-evolving understanding of SARS-CoV-2 virology, host response to the infection, and clinical course of the disease. This brief review focuses on the latest and most promising pharmacological treatments against COVID-19 currently under investigation and discuss their potential use based on either documented efficacy in similar viral infections, or their activity against inflammatory syndromes. Ongoing clinical trials are also emphasized.\n",
      "The novel coronavirus, later identified as SARS-CoV-2, originating from Wuhan in China in November 2019, quickly spread around the world becoming a pandemic. Despite the knowledge of previous coronaviruses, such as those responsible for the SARS and MERS-CoV epidemic, there is no drug or prophylaxis treatment to this day. The rapid succession of scientific findings on SARS-CoV-2 provides a significant number of potential drug targets. Nevertheless, at the same time, the high quantity of clinical data, generated by a large number of rapidly infected people, require accurate tests regarding effective medical treatments. Several in vitro and in vivo studies were rapidly initiated after the outbreak of the pandemic COVID-19. Initial clinical studies revealed the promising potential of remdesivir that demonstrated a powerful and specific in vitro antiviral activity for COVID-19. Promising effects appear to be attributable to hydroxychloroquine. Remdesivir and hydroxychloroquine are being tested in ongoing randomized trials. In contrast, oseltamivir was not effective and corticosteroids are not currently recommended. However, few data from ongoing clinical trials are identifying low molecular weight heparins, innate immune system stimulating agents, and inflammatory modulating agents as potential effective agents. The authors assume that the current pandemic will determine the need for a systematic approach based on big data analysis for identifying effective drugs to defeat SARS-Cov-2. This work is aimed to be a general reference point and to provide an overview as comprehensive as possible regarding the main clinical trials in progress at the moment.\n",
      "The global pandemic of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2 or 2019-nCoV) has prompted multiple clinical trials to jumpstart search for anti-SARS-CoV-2 therapies from existing drugs, including those with reported in vitro efficacies as well as those ones that are not known to inhibit SARS-CoV-2, such as ritonavir/lopinavir and favilavir. Here we report that after screening 19 antiviral drugs that are either in clinical trials or with proposed activity against SARS-CoV-2, remdesivir was the most effective. Chloroquine only effectively protected virus-induced cytopathic effect at around 30 µM with a therapeutic index of 1.5. Our findings also suggest that velpatasvir, ledipasvir, ritonavir, litonavir, lopinavir, favilavir, sofosbuvir, danoprevir, and pocapavir do not have direct antiviral effect.\n",
      "BACKGROUND Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), termed coronavirus disease 2019 (COVID-19) by the World Health Organization, is a newly emerging zoonotic agent that emerged in China in December 2019. No specific treatment for COVID-19 is currently available. Usual palliative treatment includes maintaining hydration and nutrition and controlling fever and cough. The clinical severity and extent of transmission need to be determined, and therapeutic options need to be developed and optimized. METHODS The present review discusses the recent repurposing of drugs for COVID-19 treatment. RESULTS Several compounds, including remdesivir, lopinavir, ritonavir, interferon-β, ribavirin, chloroquine/hydroxychloroquine, azithromycin, tocilizumab, and ivermectin, have emerged as promising alternatives. They block the virus from entering host cells, prevent viral replication, and attenuate exacerbation of the host's immune response. CONCLUSION Although some evidence indicates the positive actions of different classes of compounds for the treatment of COVID-19, few clinical assays have been established to definitively demonstrate their therapeutic value in humans. Multicenter clinical studies are urgently needed to validate and standardize therapeutic regimens that involve these agents. Although science has not yet presented us with a specific drug against COVID-19, the repurposing of drugs appears to be promising in our fight against this devastating disease.\n",
      "Countries around the world are currently fighting the coronavirus disease 2019 (COVID-19) pandemic, which is caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). SARS-CoV-2 is a betacoronavirus, belonging to the same genus as severe acute respiratory syndrome (SARS)-CoV and Middle East respiratory syndrome (MERS)-CoV. Currently, there are no proven antiviral therapies for COVID-19. Numerous clinical trials have been initiated to identify an effective treatment. One leading candidate is remdesivir (GS-5734), a broad-spectrum antiviral that was initially developed for the treatment of Ebola virus (EBOV). Although remdesivir performed well in preclinical studies, it did not meet efficacy endpoints in a randomized trial conducted during an Ebola outbreak. Remdesivir holds promise for treating COVID-19 based on in vitro activity against SARS-CoV-2, uncontrolled clinical reports, and limited data from randomized trials. Overall, current data are insufficient to judge the efficacy of remdesivir for COVID-19, and the results of additional randomized studies are eagerly anticipated. In this narrative review, we provide an overview of Ebola and coronavirus outbreaks. We then summarize preclinical and clinical studies of remdesivir for Ebola and COVID-19.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: juz9jnfk\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what is the origin of COVID-19\n",
      "what are the initial symptoms of Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Abstract Background The rapid spread of the coronavirus disease 2019 (COVID-19), caused by a zoonotic beta-coronavirus entitled 2019 novel coronavirus (2019-nCoV), has become a global threat. Awareness of the biological features of 2019-nCoV should be updated in time and needs to be comprehensively summarized to help optimize control measures and make therapeutic decisions. Methods Based on recently published literatures, official documents and selected up-to-date preprint studies, we reviewed the virology and origin, epidemiology, clinical manifestations, pathology and treatment of 2019-nCoV infection, in comparison with severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV) infection. Results The genome of 2019-nCoV partially resembled SARS-CoV and MERS-CoV, and indicating a bat origin. The COVID-19 generally had a high reproductive number, a long incubation period, a short serial interval and a low case fatality rate (much higher in patients with comorbidities) than SARS and MERS. Clinical presentation and pathology of COVID-19 greatly resembled SARS and MERS, with less upper respiratory and gastrointestinal symptoms, and more exudative lesions in post-mortems. Potential treatments included remdesivir, chloroquine, tocilizumab, convalescent plasma and vaccine immunization (when possible). Conclusion The initial experience from the current pandemic and lessons from the previous two pandemics can help improve future preparedness plans and combat disease progression.\n",
      "BACKGROUND: The rapid spread of the coronavirus disease 2019 (COVID-19), caused by a zoonotic beta-coronavirus entitled 2019 novel coronavirus (2019-nCoV), has become a global threat. Awareness of the biological features of 2019-nCoV should be updated in time and needs to be comprehensively summarized to help optimize control measures and make therapeutic decisions. METHODS: Based on recently published literature, official documents and selected up-to-date preprint studies, we reviewed the virology and origin, epidemiology, clinical manifestations, pathology and treatment of 2019-nCoV infection, in comparison with severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV) infection. RESULTS: The genome of 2019-nCoV partially resembled SARS-CoV and MERS-CoV, and indicated a bat origin. The COVID-19 generally had a high reproductive number, a long incubation period, a short serial interval and a low case fatality rate (much higher in patients with comorbidities) than SARS and MERS. Clinical presentation and pathology of COVID-19 greatly resembled SARS and MERS, with less upper respiratory and gastrointestinal symptoms, and more exudative lesions in post-mortems. Potential treatments included remdesivir, chloroquine, tocilizumab, convalescent plasma and vaccine immunization (when possible). CONCLUSION: The initial experience from the current pandemic and lessons from the previous two pandemics can help improve future preparedness plans and combat disease progression.\n",
      "Abstract Background The 2019 novel coronavirus (SARS-CoV-2) is a new human coronavirus which is spreading with epidemic features in China and other Asian countries; cases have also been reported worldwide. This novel coronavirus disease (COVID-19) is associated with a respiratory illness that may lead to severe pneumonia and acute respiratory distress syndrome (ARDS). Although related to the severe acute respiratory syndrome (SARS) and the Middle East respiratory syndrome (MERS), COVID-19 shows some peculiar pathogenetic, epidemiological and clinical features which to date are not completely understood. Aims To provide a review of the differences in pathogenesis, epidemiology and clinical features of COVID-19, SARS and MERS. Sources The most recent literature in the English language regarding COVID-19 has been reviewed, and extracted data have been compared with the current scientific evidence about SARS and MERS epidemics. Content COVID-19 seems not to be very different from SARS regarding its clinical features. However, it has a fatality rate of 2.3%, lower than that of SARS (9.5%) and much lower than that of MERS (34.4%). The possibility cannot be excluded that because of the less severe clinical picture of COVID-19 it can spread in the community more easily than MERS and SARS. The actual basic reproductive number (R0) of COVID-19 (2.0–2.5) is still controversial. It is probably slightly higher than the R0 of SARS (1.7–1.9) and higher than that of MERS (<1). A gastrointestinal route of transmission for SARS-CoV-2, which has been assumed for SARS-CoV and MERS-CoV, cannot be ruled out and needs further investigation. Implications There is still much more to know about COVID-19, especially as concerns mortality and its capacity to spread on a pandemic level. Nonetheless, all of the lessons we learned in the past from the SARS and MERS epidemics are the best cultural weapons with which to face this new global threat.\n",
      "Abstract At the end of the year 2019, the novel coronavirus (2019-nCoV) was spreading in Wuhan, China, and the outbreak process has a high speed. It was recognized as a pandemic by the World Health Organization (WHO) on 11 March 2020. Coronaviruses are enveloped and single-stranded RNA that have several families including Severe Acute Respiratory Syndrome (SARS) and Middle East Respiratory Syndrome (MERS). The pathogenesis mechanism and disease outcomes of SARS and MERS are now clear to some extent, but little information is available for 2019-nCoV. This newly identified corona virus infection represents flu-like symptoms, but usually the first symptoms are fever and dry cough. There has been no specific treatment against 2019-nCoV up to now, and physicians only apply supportive therapy. In the present article, we made an attempt to review the behavior of the virus around the world, epidemiology, a pathway for influx into the host cells, clinical presentation, as well as the treatments currently in use and future approaches; nitazoxanide may be our dream drug. We hope that this review has a positive impact on public knowledge for helping to deal with the 2019-nCoV and move one step forward toward its treatment in the near future.\n",
      "A novel coronavirus strain 2019-nCoV has caused a rapid global pandemic-COVID-19. Scientists have taken onto the task of characterizing this new virus and understanding how this virus has transmitted to humans. All preliminary studies have found some striking similarities between this new virus and the SARS-CoV that caused a similar kind of epidemic in 2002–2003. Through bioinformatics tools, a great deal of information has been gathered about the origin, evolution and zoonosis of this virus. We, in this review, report the symptoms, mode of transmission and available and putative treatments to tackle 2019-nCoV infections. We also comprehensively summarize all the information so far made available regarding the genome, evolution and zoonosis of this virus.\n",
      "OBJECTIVE Human infections with zoonotic coronavirus contain emerging and reemerging pathogenic characteristics which have raised great public health concern. This study aimed at investigating the global prevalence, biological and clinical characteristics of novel coronavirus, Wuhan China (2019-nCoV), Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV), and Middle East Respiratory Syndrome Coronavirus (MERS-CoV) infection outbreaks. MATERIALS AND METHODS The data on the global outbreak of \"2019-nCoV, SARS-CoV, and MERS-CoV\" were obtained from World Health Organization (WHO), Centers for Disease Control and Prevention (CDC), concerned ministries and research institutes. We also recorded the information from research documents published in global scientific journals indexed in ISI Web of Science and research centers on the prevalence, biological and clinical characteristics of 2019-nCoV, SARS-CoV, and MERS-CoV. RESULTS Worldwide, SARS-CoV involved 32 countries, with 8422 confirmed cases and 916 (10.87%) casualties from November 2002 to August 2003. MERS-CoV spread over 27 states, causing 2496 cases and 868 (34.77%) fatalities during the period April 2012 to December 2019. However, the novel coronavirus 2019-nCoV spread swiftly the global borders of 27 countries. It infected 34799 people and resulted in 724 (2.08%) casualties during the period December 29, 2019 to February 7, 2020. The fatality rate of coronavirus MERS-CoV was (34.77%) higher than SARS-CoV (10.87%) and 2019-nCoV (2.08%); however, the 2019-nCoV transmitted rapidly in comparison to SARS-CoV and MERS-CoV. CONCLUSIONS The novel coronavirus 2019-nCoV has diverse epidemiological and biological characteristics, making it more contagious than SARS-CoV and MERS-CoV. It has affected more people in a short time period compared to SARS-CoV and MERS-CoV, although the fatality rate of MERS-CoV was higher than SARS-CoV and 2019-nCoV. The major clinical manifestations in coronavirus infections 2019-nCoV, MERS-CoV, and SARS CoV are fever, chills, cough, shortness of breath, generalized myalgia, malaise, drowsy, diarrhea, confusion, dyspnea, and pneumonia. Global health authorities should take immediate measures to prevent the outbreaks of such emerging and reemerging pathogens across the globe to minimize the disease burden locally and globally.\n",
      "Context An outbreak of a novel, zoonotic coronavirus occurred in December 2019 in the city of Wuhan, China and has now affected almost the entire world, with the maximum confirmed cases being 1 521 252 as of April 10, 2020. The WHO named this coronavirus 2019-nCoV, with COVID-19 being the name for diseases allied with it. Objective The study intended to examine the features and characteristics of existing human coronaviruses and identify their resemblance to the newly identified 2019-nCoV. Design The research team performed a literature review, searching relevant literature databases. We searched four databases, PubMed, EMBASE, Web of Science and CNKI (Chinese Database), to identify studies reporting COVID-19. Articles published on or before April 10, 2020 were eligible for inclusion. We used the following search terms: \"Coronavirus\" or \"2019-nCoV\" or \"COVID-19\" or \"SARS-CoV\" or \"MERS-CoV\" or \"Bat SARS-CoV\" or \"ACE2 receptor\". Setting This study was take place in School of Pharmacy, Suresh Gyan Vihar University, Jaipur, India. Results The undistinguishable similarity of the genomic sequences of Severe Respiratory Syndrome (SARS)-CoV, Middle East Respiratory Syndrome (MERS)-CoV, and Bat SARS-CoV-bat-SL-CoVZC45 and bat-SL-CoVZXC21-to nCoV-2019 has facilitated the process of identifying primary treatment measures. Researchers are presuming the existence of angiotensin-converting enzyme 2 (ACE2) receptor binding in nCoV-2019, as in SARS-CoV. Researchers have been examining human-to-human transmission, the possibility of an intermediate host between bats and humans, and the existence of asymptomatic cases. An incubation period of 0 to 14 days has been reported, with acute to chronic symptoms being cough, nasal congestion, high fever, dyspnea, pneumonia, invasive lesions in both lungs, respiratory failure, and even death, including in pediatric cases. Mechanical ventilation, extracorporeal membrane oxygenation, repurposing of antivirals, and plasma infusion have proven to be somewhat effective. Several countries have started clinical trials to evaluate the safety and effectiveness of some drugs, but the ability to vaccinate people with existing or new molecules will require time. Previously learned lessons from SARS and MERS have led some areas to be well equipped in terms of the ability to take speedy action. Conclusions First-level treatments include repurposing antivirals and antimalarials, and plasma infusion should help, but development of existing or new molecules into vaccines will take time. The unpredictable trajectory of this outbreak demands careful surveillance to monitor the situation, draw strategies, implement control measures, and create proper ethical laws and medical guidelines.\n",
      "BACKGROUND: The 2019 novel coronavirus (SARS-CoV-2) is a new human coronavirus which is spreading with epidemic features in China and other Asian countries; cases have also been reported worldwide. This novel coronavirus disease (COVID-19) is associated with a respiratory illness that may lead to severe pneumonia and acute respiratory distress syndrome (ARDS). Although related to the severe acute respiratory syndrome (SARS) and the Middle East respiratory syndrome (MERS), COVID-19 shows some peculiar pathogenetic, epidemiological and clinical features which to date are not completely understood. AIMS: To provide a review of the differences in pathogenesis, epidemiology and clinical features of COVID-19, SARS and MERS. SOURCES: The most recent literature in the English language regarding COVID-19 has been reviewed, and extracted data have been compared with the current scientific evidence about SARS and MERS epidemics. CONTENT: COVID-19 seems not to be very different from SARS regarding its clinical features. However, it has a fatality rate of 2.3%, lower than that of SARS (9.5%) and much lower than that of MERS (34.4%). The possibility cannot be excluded that because of the less severe clinical picture of COVID-19 it can spread in the community more easily than MERS and SARS. The actual basic reproductive number (R0) of COVID-19 (2.0-2.5) is still controversial. It is probably slightly higher than the R0 of SARS (1.7-1.9) and higher than that of MERS (<1). A gastrointestinal route of transmission for SARS-CoV-2, which has been assumed for SARS-CoV and MERS-CoV, cannot be ruled out and needs further investigation. IMPLICATIONS: There is still much more to know about COVID-19, especially as concerns mortality and its capacity to spread on a pandemic level. Nonetheless, all of the lessons we learned in the past from the SARS and MERS epidemics are the best cultural weapons with which to face this new global threat.\n",
      "BACKGROUND Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), termed coronavirus disease 2019 (COVID-19) by the World Health Organization, is a newly emerging zoonotic agent that emerged in China in December 2019. No specific treatment for COVID-19 is currently available. Usual palliative treatment includes maintaining hydration and nutrition and controlling fever and cough. The clinical severity and extent of transmission need to be determined, and therapeutic options need to be developed and optimized. METHODS The present review discusses the recent repurposing of drugs for COVID-19 treatment. RESULTS Several compounds, including remdesivir, lopinavir, ritonavir, interferon-β, ribavirin, chloroquine/hydroxychloroquine, azithromycin, tocilizumab, and ivermectin, have emerged as promising alternatives. They block the virus from entering host cells, prevent viral replication, and attenuate exacerbation of the host's immune response. CONCLUSION Although some evidence indicates the positive actions of different classes of compounds for the treatment of COVID-19, few clinical assays have been established to definitively demonstrate their therapeutic value in humans. Multicenter clinical studies are urgently needed to validate and standardize therapeutic regimens that involve these agents. Although science has not yet presented us with a specific drug against COVID-19, the repurposing of drugs appears to be promising in our fight against this devastating disease.\n",
      "Abstract The ongoing wreaking global outbreak of the novel human beta coronavirus (CoV) pathogen was presumed to be from a seafood wholesale market in Wuhan, China, belongs to the Coronaviridae family in the Nidovirales order. The virus is highly contagious with potential human-human transmission which was named as the severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2), has spread across six continents and emerged as a global pandemic in short span with alarming levels of spread and severity. This virus associated symptoms and infectious respiratory illness is designated as coronavirus disease 19 (COVID-19). The SARS-CoV-2 possesses enveloped club-like spike protein projections with positive-sense large RNA genome and has a unique replication strategy. This virus was believed to have zoonotic origin with genetical identity to bat and pangolin CoV. In the current review, we introduce a general overview about the human CoVs and the associated diseases, the origin, structure, replication and key clinical events that occur in the COVID-19 pathogenicity. Furthermore, we focused on possible therapeutic options such as repurposing drugs including antimalarials, antivirals, antiparasitic drugs, and anti-HIV drugs, as well as monoclonal antibodies, vaccines as potential treatment options. Also we have summarized the latest research progress on the usage of stem cell therapy, human convalescent serum, interferon's, in the treatment of COVID-19.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 7ivw72l0\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "In the Wuhan Province of China, in December 2019, the novel coronavirus 2019 (COVID-19) has caused a severe involvement of the lower respiratory tract leading to an acute respiratory syndrome. Subsequently, coronavirus 2 (SARS-CoV-2) provoked a pandemic which is considered a life-threatening disease. The SARS-CoV-2, a family member of betacoronaviruses, possesses single-stranded positive-sense RNA with typical structural proteins, involving the envelope, membrane, nucleocapsid and spike proteins that are responsible for the viral infectivity, and nonstructural proteins. The effectual host immune response including innate and adaptive immunity against SARS-Cov-2 seems crucial to control and resolve the viral infection. However, the severity and outcome of the COVID-19 might be associated with the excessive production of proinflammatory cytokines “cytokine storm” leading to an acute respiratory distress syndrome. Regretfully, the exact pathophysiology and treatment, especially for the severe COVID-19, is still uncertain. The results of preliminary studies have shown that immune-modulatory or immune-suppressive treatments such as hydroxychloroquine, interleukin (IL)-6 and IL-1 antagonists, commonly used in rheumatology, might be considered as treatment choices for COVID-19, particularly in severe disease. In this review, to gain better information about appropriate anti-inflammatory treatments, mostly used in rheumatology for COVID-19, we have focused the attention on the structural features of SARS-CoV-2, the host immune response against SARS-CoV-2 and its association with the cytokine storm.\n",
      "In the Wuhan Province of China, in December 2019, the novel coronavirus 2019 (COVID-19) has caused a severe involvement of the lower respiratory tract leading to an acute respiratory syndrome. Subsequently, coronavirus 2 (SARS-CoV-2) provoked a pandemic which is considered a life-threatening disease. The SARS-CoV-2, a family member of betacoronaviruses, possesses single-stranded positive-sense RNA with typical structural proteins, involving the envelope, membrane, nucleocapsid and spike proteins that are responsible for the viral infectivity, and nonstructural proteins. The effectual host immune response including innate and adaptive immunity against SARS-Cov-2 seems crucial to control and resolve the viral infection. However, the severity and outcome of the COVID-19 might be associated with the excessive production of proinflammatory cytokines \"cytokine storm\" leading to an acute respiratory distress syndrome. Regretfully, the exact pathophysiology and treatment, especially for the severe COVID-19, is still uncertain. The results of preliminary studies have shown that immune-modulatory or immune-suppressive treatments such as hydroxychloroquine, interleukin (IL)-6 and IL-1 antagonists, commonly used in rheumatology, might be considered as treatment choices for COVID-19, particularly in severe disease. In this review, to gain better information about appropriate anti-inflammatory treatments, mostly used in rheumatology for COVID-19, we have focused the attention on the structural features of SARS-CoV-2, the host immune response against SARS-CoV-2 and its association with the cytokine storm.\n",
      "In December 2019, coronavirus disease 2019 (COVID-19), which is caused by the new coronavirus severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), was identified in Wuhan (Hubei province, China)1; it soon spread across the world. In this ongoing pandemic, public health concerns and the urgent need for effective therapeutic measures require a deep understanding of the epidemiology, transmissibility and pathogenesis of COVID-19. Here we analysed clinical, molecular and immunological data from 326 patients with confirmed SARS-CoV-2 infection in Shanghai. The genomic sequences of SARS-CoV-2, assembled from 112 high-quality samples together with sequences in the Global Initiative on Sharing All Influenza Data (GISAID) dataset, showed a stable evolution and suggested that there were two major lineages with differential exposure history during the early phase of the outbreak in Wuhan. Nevertheless, they exhibited similar virulence and clinical outcomes. Lymphocytopenia, especially reduced CD4+ and CD8+ T cell counts upon hospital admission, was predictive of disease progression. High levels of interleukin (IL)-6 and IL-8 during treatment were observed in patients with severe or critical disease and correlated with decreased lymphocyte count. The determinants of disease severity seemed to stem mostly from host factors such as age and lymphocytopenia (and its associated cytokine storm), whereas viral genetic variation did not significantly affect outcomes.\n",
      "The outbreak of the new coronavirus infections COVID-19 in December 2019 in China has quickly become a global health emergency. Given the lack of specific anti-viral therapies, the current management of severe acute respiratory syndrome coronaviruses (SARS-CoV-2) is mainly supportive, even though several compounds are now under investigation for the treatment of this life-threatening disease. COVID-19 pandemic is certainly conditioning the treatment strategy of a complex disorder as rheumatoid arthritis (RA), whose infectious risk is increased compared to the general population because of an overall impairment of immune system typical of autoimmune diseases combined with the iatrogenic effect generated by corticosteroids and immunosuppressive drugs. However, the increasing knowledge about the pathophysiology of SARS-CoV-2 infection is leading to consider some anti-rheumatic drugs as potential treatment options for the management of COVID-19. In this review we will critically analyse the evidences on either positive or negative effect of drugs commonly used to treat RA in this particular scenario, in order to optimize the current approach to RA patients.\n",
      "One of the most serious viral outbreaks of the decade, infecting humans, originated from the city of Wuhan, China, by the end of December 2019, has left the world shaken up. It is the successor infection of severe acute respiratory syndrome coronavirus (SARS-CoV) named as SARS-CoV-2 causing a disease called as COVID-19 (Coronavirus disease-19). Being one of the most severe diseases in terms of transmission, this disease agitates the immune system of an individual quite disturbingly which at times leads to death, which is why it has become the need of the hour to step forward to extensively involve in understanding the genetics, pathogenesis, and immunopathology of SARS-CoV-2 in order to design drugs to treat or to design a vaccine to prevent. In this chapter, we have tried to review and summarize the studies done so far to understand the host–pathogen relationship and the host immune response during COVID-19 infection. One of the recent developments regarding the understanding of SARS-CoV-2 infection is the mechanism of immune evasion involved during the pathogenesis and cytokine storm syndrome during infection in the patient against which a drug called as Hydroxychloroquine has been designed. Comprehensively, we have tried to give an immunological insight into the SARS-CoV-2 infection in order to understand the possible outcome for any therapeutic advancement.\n",
      "Coronavirus disease 2019 (COVID-19), which began in Wuhan, China in December 2019 has caused a large global pandemic and poses a serious threat to public health. More than four million cases of COVID-19, which is caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), have been confirmed as of May 11, 2020. SARS-CoV-2 is a highly pathogenic and transmissible coronavirus that primarily spreads through respiratory droplets and close contact. A growing body of clinical data suggests that a cytokine storm is associated with COVID-19 severity and is also a crucial cause of death from COVID-19. In the absence of antivirals and vaccines for COVID-19, there is an urgent need to understand the cytokine storm in COVID-19. Here, we have reviewed the current understanding of the features of SARS-CoV-2 and the pathological features, pathophysiological mechanisms, and treatments of the cytokine storm induced by COVID-19. Additionally, we suggest that the identification and treatment of the cytokine storm are important components for rescuing patients with severe COVID-19. This article is protected by copyright. All rights reserved.\n",
      "The COVID-19 pandemic resulting from the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) which emerged in December 2019 in the Chinese city of Wuhan in the province Hubei has placed immense burden on national economies and global health. At present neither vaccination nor therapies are available although several antiviral agents such as remdesivir, originally an Ebola drug, nelfinavir, an HIV-1 protease inhibitor and other drugs such as lopinavir have been evaluated. Here, we performed a meta-analysis of RNA-sequencing data from three studies employing human lung epithelial cells. Of these one focused on lung epithelial cells infected with SARS-CoV-2. We aimed at identifying genes co-expressed with angiotensin I converting enzyme 2 (ACE2) the human cell entry receptor of SARS-CoV-2, and unveiled several genes correlated or inversely correlated with high significance, among the most significant of these was the transmembrane serine protease 4 (TMPRSS4). Serine proteases are known to be involved in the infection process by priming the virus spike protein. Pathway analysis revealed papilloma virus infection amongst the most significantly correlated pathways. Gene Ontologies revealed regulation of viral life cycle, immune responses, pro-inflammatory responses-several interleukins such as IL6, IL1, IL20 and IL33, IFI16 regulating the interferon response to a virus, chemo-attraction of macrophages, last and not least cellular stress resulting from activated Reactive Oxygen Species. We believe that this dataset will aid in a better understanding of the molecular mechanism(s) underlying COVID-19.\n",
      "The recent outbreak of coronavirus disease 2019 (COVID-19), triggered by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) poses an enormous threat to global public health and economies. Human coronaviruses normally cause no or mild respiratory disease but in the past two decades, potentially fatal coronavirus infections have emerged, causing respiratory tract illnesses such as pneumonia and bronchitis. These include severe acute respiratory syndrome coronavirus (SARS-CoV), followed by the Middle East respiratory syndrome coronavirus (MERS-CoV), and recently the SARS-CoV-2 coronavirus outbreak that emerged in Wuhan, China, in December 2019. Currently, most COVID-19 patients receive traditional supportive care including breathing assistance. To halt the ongoing spread of the pandemic SARS-CoV-2 coronavirus and rescue individual patients, established drugs and new therapies are under evaluation. Since it will be some time until a safe and effective vaccine will be available, the immediate priority is to harness innate immunity to accelerate early antiviral immune responses. Second, since excessive inflammation is a major cause of pathology, targeted anti-inflammatory responses are being evaluated to reduce inflammation-induced damage to the respiratory tract and cytokine storms. Here, we highlight prominent immunotherapies at various stages of development that aim for augmented anti-coronavirus immunity and reduction of pathological inflammation.\n",
      "Abstract The new coronavirus (SARS-CoV2) that causes a severe acute respiratory syndrome emerged in Wuhan, China, in December 2019. It produces the aforementioned disease due to coronavirus 2019 (COVID-19), and has led to a declaration of a world public health emergency by the World Health Organisation. This new SARS-CoV2 virus could share characteristics and an immune response similar to those described for other coronavirus. Given its activity on the interferon pathway, and the manner in which it dysregulates innate immunity, the use of treatments directed at modulating or containing this could be of interest. A narrative review was made of the current evidence about immunity against coronavirus and it applicability to SARS-CoV2. The physiopathogenesis is also described, along with the underlying leucocyte activity, with the intention of clarifying the possible usefulness of inflammatory biomarkers and the development of personalised treatments.\n",
      "Coronavirus disease 2019 (COVID‐19), which began in Wuhan, China in December 2019 has caused a large global pandemic and poses a serious threat to public health. More than four million cases of COVID‐19, which is caused by the severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2), have been confirmed as of May 11, 2020. SARS‐CoV‐2 is a highly pathogenic and transmissible coronavirus that primarily spreads through respiratory droplets and close contact. A growing body of clinical data suggests that a cytokine storm is associated with COVID‐19 severity and is also a crucial cause of death from COVID‐19. In the absence of antivirals and vaccines for COVID‐19, there is an urgent need to understand the cytokine storm in COVID‐19. Here, we have reviewed the current understanding of the features of SARS‐CoV‐2 and the pathological features, pathophysiological mechanisms, and treatments of the cytokine storm induced by COVID‐19. Additionally, we suggest that the identification and treatment of the cytokine storm are important components for rescuing patients with severe COVID‐19. This article is protected by copyright. All rights reserved.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: t8q99tlq\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Monitoring the mutation dynamics of SARS-CoV-2 is critical for the development of effective approaches to contain the pathogen. By analyzing 106 SARS-CoV-2 and 39 SARS genome sequences, we provided direct genetic evidence that SARS-CoV-2 has a much lower mutation rate than SARS. Minimum Evolution phylogeny analysis revealed the putative original status of SARS-CoV-2 and the early-stage spread history. The discrepant phylogenies for the spike protein and its receptor binding domain proved a previously reported structural rearrangement prior to the emergence of SARS-CoV-2. Despite that we found the spike glycoprotein of SARS-CoV-2 is particularly more conserved, we identified a mutation that leads to weaker receptor binding capability, which concerns a SARS-CoV-2 sample collected on 27th January 2020 from India. This represents the first report of a significant SARS-CoV-2 mutant, and raises the alarm that the ongoing vaccine development may become futile in future epidemic if more mutations were identified. Highlights Based on the currently available genome sequence data, we proved that SARS-COV-2 genome has a much lower mutation rate and genetic diversity than SARS during the 2002-2003 outbreak. The spike (S) protein encoding gene of SARS-COV-2 is found relatively more conserved than other protein-encoding genes, which is a good indication for the ongoing antiviral drug and vaccine development. Minimum Evolution phylogeny analysis revealed the putative original status of SARS-CoV-2 and the early-stage spread history. We confirmed a previously reported rearrangement in the S protein arrangement of SARS-COV-2, and propose that this rearrangement should have occurred between human SARS-CoV and a bat SARS-CoV, at a time point much earlier before SARS-COV-2 transmission to human. We provided first evidence that a mutated SARS-COV-2 with reduced human ACE2 receptor binding affinity have emerged in India based on a sample collected on 27th January 2020.\n",
      "SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) is a novel coronavirus causing an outbreak of COVID-19 globally in the past six months. A relatively higher divergence on the spike protein of SASR-CoV-2 enables it to transmit across species efficiently. We particularly believe that the adaptive mutations of the receptor-binding domain (RBD) of spike protein in SARS-CoV-2 might be essential to its high transmissibility among humans. Thus here we collected 2,142 high-quality genome sequences of SARS-CoV-2 from 160 regions in over 50 countries and reconstructed their phylogeny, and also analyzed the interaction between the polymorphisms of spike protein and human ACE2 (hACE2). Phylogenetic analysis of SARS-CoV-2 and coronavirus in other hosts show SARS-CoV-2 is highly possible originated from Bat-CoV (RaTG13) found in horseshoe bat and a recombination event may occur on the spike protein of Pangolin-CoV to imbue it the ability to infect humans. Moreover, compared to the S gene of SARS-CoV-2, it is more conserved in the direct-binding sites of RBD and we noticed that spike protein of SARS-CoV-2 may under a consensus evolution to adapt to human hosts better. 3,860 amino acid mutations in spike protein RBD (T333-C525) of SARS-CoV-2 were simulated and their stability and affinity binding to hACE2 (S19-D615) were calculated. Our analysis indicates SARS-CoV-2 could infect humans from different populations with no preference, and a higher divergence in the spike protein of SARS-CoV-2 at the early stage of this pandemic may be a good indicator that could show the pathway of SARS-CoV-2 transmitting from the natural reservoir to human beings.\n",
      "The spread of SARS-CoV-2 since December 2019 has become a pandemic and impacted many aspects of human society. Here, we analyzed genetic variation of SARS-CoV-2 and its related coronavirus and found the evidence of intergenomic recombination. After correction for mutational bias, analysis of 137 SARS-CoV-2 genomes as of 2/23/2020 revealed the excess of low frequency mutations on both synonymous and nonsynonymous sites which is consistent with recent origin of the virus. In contrast to adaptive evolution previously reported for SARS-CoV in its brief epidemic in 2003, our analysis of SARS-CoV-2 genomes shows signs of relaxation of selection. The sequence similarity of the spike receptor binding domain between SARS-CoV-2 and a sequence from pangolin is probably due to an ancient intergenomic introgression. Therefore, SARS-CoV-2 might have cryptically circulated within humans for years before being recently noticed. Data from the early outbreak and hospital archives are needed to trace its evolutionary path and reveal critical steps required for effective spreading. Two mutations, 84S in orf8 protein and 251V in orf3 protein, occurred coincidentally with human intervention. The 84S first appeared on 1/5/2020 and reached a plateau around 1/23/2020, the lockdown of Wuhan. 251V emerged on 1/21/2020 and rapidly increased its frequency. Thus, the roles of these mutations on infectivity need to be elucidated. Genetic diversity of SARS-CoV-2 collected from China was two time higher than those derived from the rest of the world. In addition, in network analysis, haplotypes collected from Wuhan city were at interior and have more mutational connections, both of which are consistent with the observation that the outbreak of cov-19 was originated from China. SUMMARY In contrast to adaptive evolution previously reported for SARS-CoV in its brief epidemic, our analysis of SARS-CoV-2 genomes shows signs of relaxation of selection. The sequence similarity of the spike receptor binding domain between SARS-CoV-2 and a sequence from pangolin is probably due to an ancient intergenomic introgression. Therefore, SARS-CoV-2 might have cryptically circulated within humans for years before being recently noticed. Data from the early outbreak and hospital archives are needed to trace its evolutionary path and reveal critical steps required for effective spreading. Two mutations, 84S in orf8 protein and 251V in orf3 protein, occurred coincidentally with human intervention. The 84S first appeared on 1/5/2020 and reached a plateau around 1/23/2020, the lockdown of Wuhan. 251V emerged on 1/21/2020 and rapidly increased its frequency. Thus, the roles of these mutations on infectivity need to be elucidated.\n",
      "Background SARS-CoV-2 most likely evolved from a bat beta-coronavirus and started infecting humans in December 2019. Since then it has rapidly infected people around the world, with more than 4.5 million confirmed cases by the middle of May 2020. Early genome sequencing of the virus has enabled the development of molecular diagnostics and the commencement of therapy and vaccine development. The analysis of the early sequences showed relatively few evolutionary selection pressures. However, with the rapid worldwide expansion into diverse human populations, significant genetic variations are becoming increasingly likely. The current limitations on social movement between countries also offers the opportunity for these viral variants to become distinct strains with potential implications for diagnostics, therapies and vaccines. Methods We used the current sequencing archives (NCBI and GISAID) to investigate 15,487 whole genomes, looking for evidence of strain diversification and selective pressure. Results We used 6,294 SNPs to build a phylogenetic tree of SARS-CoV-2 diversity and noted strong evidence for the existence of two major clades and six sub-clades, unevenly distributed across the world. We also noted that convergent evolution has potentially occurred across several locations in the genome, showing selection pressures, including on the spike glycoprotein where we noted a potentially critical mutation that could affect its binding to the ACE2 receptor. We also report on mutations that could prevent current molecular diagnostics from detecting some of the sub-clades. Conclusion The worldwide whole genome sequencing effort is revealing the challenge of developing SARS-CoV-2 containment tools suitable for everyone and the need for data to be continually evaluated to ensure accuracy in outbreak estimations.\n",
      "The newly identified SARS-CoV-2 has now been reported from around 185 countries with more than a million confirmed human cases including more than 120,000 deaths. The genomes of SARS-COV-2 strains isolated from different parts of the world are now available and the unique features of constituent genes and proteins need to be explored to understand the biology of the virus. Spike glycoprotein is one of the major targets to be explored because of its role during the entry of coronaviruses into host cells. We analyzed 320 whole-genome sequences and 320 spike protein sequences of SARS-CoV-2 using multiple sequence alignment. In this study, 483 unique variations have been identified among the genomes of SARS-CoV-2 including 25 nonsynonymous mutations and one deletion in the spike (S) protein. Among the 26 variations detected in S, 12 variations were located at the N-terminal domain (NTD) and 6 variations at the receptor-binding domain (RBD) which might alter the interaction of S protein with the host receptor angiotensin-converting enzyme 2 (ACE2). Besides, 22 amino acid insertions were identified in the spike protein of SARS-CoV-2 in comparison with that of SARS-CoV. Phylogenetic analyses of spike protein revealed that Bat coronavirus have a close evolutionary relationship with circulating SARS-CoV-2. The genetic variation analysis data presented in this study can help a better understanding of SARS-CoV-2 pathogenesis. Based on results reported herein, potential inhibitors against S protein can be designed by considering these variations and their impact on protein structure.\n",
      "SARS-CoV-2, since emerging in Wuhan, China, has been a major concern because of its high infection rate and has left more than six million infected people around the world. Many studies endeavored to reveal the structure of the SARS-CoV-2 compared to the SARS-CoV, in order to find solutions to suppress this high infection rate. Some of these studies showed that the mutations in the SARS-CoV spike (S) protein might be responsible for its higher affinity to the ACE2 human cell receptor. In this work, we used molecular dynamics simulations and Monte Carlo sampling to compare the binding affinities of the S proteins of SARS-CoV and SARS-CoV-2 to the ACE2. Our results show that the protein surface of the ACE2 at the receptor binding domain (RBD) exhibits negative electrostatic potential, while a positive potential is observed for the S proteins of SARS-CoV/SARS-CoV-2. In addition, the binding energies at the interface are slightly higher for SARS-CoV-2 because of enhanced electrostatic interactions. The major contributions to the electrostatic binding energies result from the salt bridges forming between R426 and ACE-2-E329 in the case of SARS-CoV and K417 and ACE2-D30 in the SARS-CoV-2. In addition, our results indicate that the enhancement in the binding energy is not due to a single mutant but rather because of the sophisticated structural changes induced by all these mutations together. This finding suggests that it is implausible for the SARS-CoV-2 to be a lab-engineered virus.\n",
      "[Image: see text] SARS-CoV-2, since emerging in Wuhan, China, has been a major concern because of its high infection rate and has left more than six million infected people around the world. Many studies endeavored to reveal the structure of the SARS-CoV-2 compared to the SARS-CoV, in order to find solutions to suppress this high infection rate. Some of these studies showed that the mutations in the SARS-CoV spike (S) protein might be responsible for its higher affinity to the ACE2 human cell receptor. In this work, we used molecular dynamics simulations and Monte Carlo sampling to compare the binding affinities of the S proteins of SARS-CoV and SARS-CoV-2 to the ACE2. Our results show that the protein surface of the ACE2 at the receptor binding domain (RBD) exhibits negative electrostatic potential, while a positive potential is observed for the S proteins of SARS-CoV/SARS-CoV-2. In addition, the binding energies at the interface are slightly higher for SARS-CoV-2 because of enhanced electrostatic interactions. The major contributions to the electrostatic binding energies result from the salt bridges forming between R426 and ACE-2-E329 in the case of SARS-CoV and K417 and ACE2-D30 in the SARS-CoV-2. In addition, our results indicate that the enhancement in the binding energy is not due to a single mutant but rather because of the sophisticated structural changes induced by all these mutations together. This finding suggests that it is implausible for the SARS-CoV-2 to be a lab-engineered virus.\n",
      "The recent pandemic of COVID-19, caused by SARS-CoV-2, is unarguably the most fearsome compared with the earlier outbreaks caused by other coronaviruses, SARS-CoV and MERS-CoV. Human ACE2 is now established as a receptor for the SARS-CoV-2 spike protein. Where variations in the viral spike protein, in turn, lead to the cross-species transmission of the virus, genetic variations in the host receptor ACE2 may also contribute to the susceptibility and/or resistance against the viral infection. This study aims to explore the binding of the proteins encoded by different human ACE2 allelic variants with SARS-CoV-2 spike protein. Briefly, coding variants of ACE2 corresponding to the reported binding sites for its attachment with coronavirus spike protein were selected and molecular models of these variants were constructed by homology modeling. The models were then superimposed over the native ACE2 and ACE2-spike protein complex, to observe structural changes in the ACE2 variants and their intermolecular interactions with SARS-CoV-2 spike protein, respectively. Despite strong overall structural similarities, the spatial orientation of the key interacting residues varies in the ACE2 variants compared with the wild-type molecule. Most ACE2 variants showed a similar binding affinity for SARS-CoV-2 spike protein as observed in the complex structure of wild-type ACE2 and SARS-CoV-2 spike protein. However, ACE2 alleles, rs73635825 (S19P) and rs143936283 (E329G) showed noticeable variations in their intermolecular interactions with the viral spike protein. In summary, our data provide a structural basis of potential resistance against SARS-CoV-2 infection driven by ACE2 allelic variants.\n",
      "The newly identified SARS-CoV-2 has now been reported from around 185 countries with more than a million confirmed human cases including more than 120,000 deaths. The genomes of SARS-COV-2 strains isolated from different parts of the world are now available and the unique features of constituent genes and proteins need to be explored to understand the biology of the virus. Spike glycoprotein is one of the major targets to be explored because of its role during the entry of coronaviruses into host cells. We analyzed 320 whole-genome sequences and 320 spike protein sequences of SARS-CoV-2 using multiple sequence alignment. In this study, 483 unique variations have been identified among the genomes of SARS-CoV-2 including 25 nonsynonymous mutations and one deletion in the spike (S) protein. Among the 26 variations detected, 12 variations were located at the N-terminal domain and 6 variations at the receptor-binding domain (RBD) which might alter the interaction of S protein with the host receptor angiotensin converting enzyme-2 (ACE2). Besides, 22 amino acid insertions were identified in the spike protein of SARS-CoV-2 in comparison with that of SARS-CoV. Phylogenetic analyses of spike protein revealed that Bat coronavirus have a close evolutionary relationship with circulating SARS-CoV-2. The genetic variation analysis data presented in this study can help a better understanding of SARS-CoV-2 pathogenesis. Based on results reported herein, potential inhibitors against S protein can be designed by considering these variations and their impact on protein structure.\n",
      "The SARS-CoV-2 epidemic started in late December 2019 in Wuhan, China, and has since impacted a large portion of China and raised major global concern. Herein, we investigated the extent of molecular divergence between SARS-CoV-2 and other related coronaviruses. Although we found only 4% variability in genomic nucleotides between SARS-CoV-2 and a bat SARS-related coronavirus (SARSr-CoV; RaTG13), the difference at neutral sites was 17%, suggesting the divergence between the two viruses is much larger than previously estimated. Our results suggest that the development of new variations in functional sites in the receptor-binding domain (RBD) of the spike seen in SARS-CoV-2 and viruses from pangolin SARSr-CoVs are likely caused by mutations and natural selection besides recombination. Population genetic analyses of 103 SARS-CoV-2 genomes indicated that these viruses evolved into two major types (designated L and S), that are well defined by two different SNPs that show nearly complete linkage across the viral strains sequenced to date. Although the L type (∼70%) is more prevalent than the S type (∼30%), the S type was found to be the ancestral version. Whereas the L type was more prevalent in the early stages of the outbreak in Wuhan, the frequency of the L type decreased after early January 2020. Human intervention may have placed more severe selective pressure on the L type, which might be more aggressive and spread more quickly. On the other hand, the S type, which is evolutionarily older and less aggressive, might have increased in relative frequency due to relatively weaker selective pressure. These findings strongly support an urgent need for further immediate, comprehensive studies that combine genomic data, epidemiological data, and chart records of the clinical symptoms of patients with coronavirus disease 2019 (COVID-19).\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: w5ytp1q7\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "The newly identified SARS-CoV-2 has now been reported from around 185 countries with more than a million confirmed human cases including more than 120,000 deaths. The genomes of SARS-COV-2 strains isolated from different parts of the world are now available and the unique features of constituent genes and proteins need to be explored to understand the biology of the virus. Spike glycoprotein is one of the major targets to be explored because of its role during the entry of coronaviruses into host cells. We analyzed 320 whole-genome sequences and 320 spike protein sequences of SARS-CoV-2 using multiple sequence alignment. In this study, 483 unique variations have been identified among the genomes of SARS-CoV-2 including 25 nonsynonymous mutations and one deletion in the spike (S) protein. Among the 26 variations detected, 12 variations were located at the N-terminal domain and 6 variations at the receptor-binding domain (RBD) which might alter the interaction of S protein with the host receptor angiotensin converting enzyme-2 (ACE2). Besides, 22 amino acid insertions were identified in the spike protein of SARS-CoV-2 in comparison with that of SARS-CoV. Phylogenetic analyses of spike protein revealed that Bat coronavirus have a close evolutionary relationship with circulating SARS-CoV-2. The genetic variation analysis data presented in this study can help a better understanding of SARS-CoV-2 pathogenesis. Based on results reported herein, potential inhibitors against S protein can be designed by considering these variations and their impact on protein structure.\n",
      "The newly identified SARS-CoV-2 has now been reported from around 185 countries with more than a million confirmed human cases including more than 120,000 deaths. The genomes of SARS-COV-2 strains isolated from different parts of the world are now available and the unique features of constituent genes and proteins need to be explored to understand the biology of the virus. Spike glycoprotein is one of the major targets to be explored because of its role during the entry of coronaviruses into host cells. We analyzed 320 whole-genome sequences and 320 spike protein sequences of SARS-CoV-2 using multiple sequence alignment. In this study, 483 unique variations have been identified among the genomes of SARS-CoV-2 including 25 nonsynonymous mutations and one deletion in the spike (S) protein. Among the 26 variations detected in S, 12 variations were located at the N-terminal domain (NTD) and 6 variations at the receptor-binding domain (RBD) which might alter the interaction of S protein with the host receptor angiotensin-converting enzyme 2 (ACE2). Besides, 22 amino acid insertions were identified in the spike protein of SARS-CoV-2 in comparison with that of SARS-CoV. Phylogenetic analyses of spike protein revealed that Bat coronavirus have a close evolutionary relationship with circulating SARS-CoV-2. The genetic variation analysis data presented in this study can help a better understanding of SARS-CoV-2 pathogenesis. Based on results reported herein, potential inhibitors against S protein can be designed by considering these variations and their impact on protein structure.\n",
      "The newly identified SARS-CoV-2 has now been reported from around 183 countries with more than a million confirmed human cases including more than 68000 deaths. The genomes of SARS-COV-2 strains isolated from different parts of the world are now available and the unique features of constituent genes and proteins have gotten substantial attention recently. Spike glycoprotein is widely considered as a possible target to be explored because of its role during the entry of coronaviruses into host cells. We analyzed 320 whole-genome sequences and 320 spike protein sequences of SARS-CoV-2 using multiple sequence alignment tools. In this study, 483 unique variations have been identified among the genomes including 25 non-synonymous mutations and one deletion in the spike protein of SARS-CoV-2. Among the 26 variations detected, 12 variations were located at the N-terminal domain and 6 variations at the receptor-binding domain (RBD) which might alter the interaction with receptor molecules. In addition, 22 amino acid insertions were identified in the spike protein of SARS-CoV-2 in comparison with that of SARS-CoV. Phylogenetic analyses of spike protein revealed that Bat coronavirus have a close evolutionary relationship with circulating SARS-CoV-2. The genetic variation analysis data presented in this study can help a better understanding of SARS-CoV-2 pathogenesis. Based on our findings, potential inhibitors can be designed and tested targeting these proposed sites of variation.\n",
      "SARS-CoV-2, the causative agent of COVID-19 pandemic, is an RNA virus prone to mutations. Interaction of SARS-CoV-2 Spike (S) protein with the host cell receptor, Angiotensin-I Converting Enzyme 2 (ACE2) is pivotal for attachment and entry of the virus. Yet, natural mutations acquired on S protein during the pandemic and their impact on viral infectivity, transmission dynamics and disease pathogenesis remains poorly understood. Here, we analysed 2952 SARS-CoV-2 genomes across the globe, and identified a total of 1815 non-synonymous mutations in the S-protein that fall into 54 different types. We observed that six of these distinct mutations were located in the Receptor Binding Domain (RBD) region that directly engages host ACE2. Molecular phylogenetic analysis revealed that these RBD mutations cluster into distinct phyletic clades among global subtypes of SARS-CoV-2 implying possible emergence of novel sublineages of the strain. Structure-guided homology modelling and docking analysis predicted key molecular rearrangements in the ACE2 binding interface of RBD mutants that could result in altered virus-host interactions. We propose that our findings could be significant in understanding disease dynamics and in developing vaccines, antibodies and therapeutics for COVID-19. Importance COVID-19 pandemic shows considerable variations in disease transmission and pathogenesis globally, yet reasons remain unknown. Our study identifies key S-protein mutations prevailing in SARS-CoV-2 strain that could alter viral attachment and infectivity. We propose that the interplay of these mutations could be one of the factors driving global variations in COVID-19 spread. In addition, the mutations identified in this study could be an important indicator in predicting efficacies of vaccines, antibodies and therapeutics that target SARS-CoV-2 RBD-ACE2 interface.\n",
      "SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) is a novel coronavirus causing an outbreak of COVID-19 globally in the past six months. A relatively higher divergence on the spike protein of SASR-CoV-2 enables it to transmit across species efficiently. We particularly believe that the adaptive mutations of the receptor-binding domain (RBD) of spike protein in SARS-CoV-2 might be essential to its high transmissibility among humans. Thus here we collected 2,142 high-quality genome sequences of SARS-CoV-2 from 160 regions in over 50 countries and reconstructed their phylogeny, and also analyzed the interaction between the polymorphisms of spike protein and human ACE2 (hACE2). Phylogenetic analysis of SARS-CoV-2 and coronavirus in other hosts show SARS-CoV-2 is highly possible originated from Bat-CoV (RaTG13) found in horseshoe bat and a recombination event may occur on the spike protein of Pangolin-CoV to imbue it the ability to infect humans. Moreover, compared to the S gene of SARS-CoV-2, it is more conserved in the direct-binding sites of RBD and we noticed that spike protein of SARS-CoV-2 may under a consensus evolution to adapt to human hosts better. 3,860 amino acid mutations in spike protein RBD (T333-C525) of SARS-CoV-2 were simulated and their stability and affinity binding to hACE2 (S19-D615) were calculated. Our analysis indicates SARS-CoV-2 could infect humans from different populations with no preference, and a higher divergence in the spike protein of SARS-CoV-2 at the early stage of this pandemic may be a good indicator that could show the pathway of SARS-CoV-2 transmitting from the natural reservoir to human beings.\n",
      "The emergence of SARS-CoV-2 has resulted in >90,000 infections and >3,000 deaths. Coronavirus spike (S) glycoproteins promote entry into cells and are the main target of antibodies. We show that SARS-CoV-2 S uses ACE2 to enter cells and that the receptor-binding domains of SARS-CoV-2 S and SARS-CoV S bind with similar affinities to human ACE2, correlating with the efficient spread of SARS-CoV-2 among humans. We found that the SARS-CoV-2 S glycoprotein harbors a furin cleavage site at the boundary between the S(1)/S(2) subunits, which is processed during biogenesis and sets this virus apart from SARS-CoV and SARS-related CoVs. We determined cryo-EM structures of the SARS-CoV-2 S ectodomain trimer, providing a blueprint for the design of vaccines and inhibitors of viral entry. Finally, we demonstrate that SARS-CoV S murine polyclonal antibodies potently inhibited SARS-CoV-2 S mediated entry into cells, indicating that cross-neutralizing antibodies targeting conserved S epitopes can be elicited upon vaccination.\n",
      "The emergence of SARS-CoV-2 has resulted in >90,000 infections and >3,000 deaths. Coronavirus spike (S) glycoproteins promote entry into cells and are the main target of antibodies. We show that SARS-CoV-2 S uses ACE2 to enter cells and that the receptor-binding domains of SARS-CoV-2 S and SARS-CoV S bind with similar affinities to human ACE2, correlating with the efficient spread of SARS-CoV-2 among humans. We found that the SARS-CoV-2 S glycoprotein harbors a furin cleavage site at the boundary between the S1/S2 subunits, which is processed during biogenesis and sets this virus apart from SARS-CoV and SARS-related CoVs. We determined cryo-EM structures of the SARS-CoV-2 S ectodomain trimer, providing a blueprint for the design of vaccines and inhibitors of viral entry. Finally, we demonstrate that SARS-CoV S murine polyclonal antibodies potently inhibited SARS-CoV-2 S mediated entry into cells, indicating that cross-neutralizing antibodies targeting conserved S epitopes can be elicited upon vaccination.\n",
      "The spike (S) glycoprotein of SARS-CoV-2 is responsible for the binding to the permissive cells. The receptor-binding domain (RBD) of SARS-CoV-2 S protein directly interacts with the human angiotensin-converting enzyme 2 (ACE2) on the host cell membrane. In this study, we used computational saturation mutagenesis approaches, including structure-based energy calculations and sequence-based pathogenicity predictions, to quantify the systemic effects of missense mutations on SARS-CoV-2 S protein structure and function. A total of 18,354 mutations in S protein were analyzed and we discovered that most of these mutations could destabilize the entire S protein and its RBD. Specifically, residues G431 and S514 in SARS-CoV-2 RBD are important for S protein stability. We analyzed 384 experimentally verified S missense variations and revealed that the dominant pandemic form, D614G, can stabilize the entire S protein. Moreover, many mutations in N-linked glycosylation sites can increase the stability of the S protein. In addition, we investigated 3,705 mutations in SARS-CoV-2 RBD and 11,324 mutations in human ACE2 and found that SARS-CoV-2 neighbor residues G496 and F497 and ACE2 residues D355 and Y41 are critical for the RBD-ACE2 interaction. The findings comprehensively provide potential target sites in the development of drugs and vaccines against COVID-19.\n",
      "Abstract As severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) continues to disperse globally with worrisome speed, identifying amino acid variations in the virus could help to understand the characteristics of it. Here, we studied 489 SARS-CoV-2 genomes obtained from 32 countries from the Nextstrain database and performed phylogenetic tree analysis by clade, country, and genotype of the surface spike glycoprotein (S protein) at site 614. We found that virus strains from mainland China were mostly distributed in Clade B and Clade undefined in the phylogenetic tree, with very few found in Clade A. In contrast, Clades A2 (one case) and A2a (112 cases) predominantly contained strains from European regions. Moreover, Clades A2 and A2a differed significantly from those of mainland China in age of infected population (P = 0.0071, mean age 40.24 to 46.66), although such differences did not exist between the US and mainland China. Further analysis demonstrated that the variation of the S protein at site 614 (QHD43416.1: p.614D>G) was a characteristic of stains in Clades A2 and A2a. Importantly, this variation was predicted to have neutral or benign effects on the function of the S protein. In addition, global quality estimates and 3D protein structures tended to be different between the two S proteins. In summary, we identified different genomic epidemiology among SARS-CoV-2 strains in different clades, especially in an amino acid variation of the S protein at 614, revealing potential viral genome divergence in SARS-CoV-2 strains.\n",
      "Monitoring the mutation dynamics of SARS-CoV-2 is critical for the development of effective approaches to contain the pathogen. By analyzing 106 SARS-CoV-2 and 39 SARS genome sequences, we provided direct genetic evidence that SARS-CoV-2 has a much lower mutation rate than SARS. Minimum Evolution phylogeny analysis revealed the putative original status of SARS-CoV-2 and the early-stage spread history. The discrepant phylogenies for the spike protein and its receptor binding domain proved a previously reported structural rearrangement prior to the emergence of SARS-CoV-2. Despite that we found the spike glycoprotein of SARS-CoV-2 is particularly more conserved, we identified a mutation that leads to weaker receptor binding capability, which concerns a SARS-CoV-2 sample collected on 27th January 2020 from India. This represents the first report of a significant SARS-CoV-2 mutant, and raises the alarm that the ongoing vaccine development may become futile in future epidemic if more mutations were identified. Highlights Based on the currently available genome sequence data, we proved that SARS-COV-2 genome has a much lower mutation rate and genetic diversity than SARS during the 2002-2003 outbreak. The spike (S) protein encoding gene of SARS-COV-2 is found relatively more conserved than other protein-encoding genes, which is a good indication for the ongoing antiviral drug and vaccine development. Minimum Evolution phylogeny analysis revealed the putative original status of SARS-CoV-2 and the early-stage spread history. We confirmed a previously reported rearrangement in the S protein arrangement of SARS-COV-2, and propose that this rearrangement should have occurred between human SARS-CoV and a bat SARS-CoV, at a time point much earlier before SARS-COV-2 transmission to human. We provided first evidence that a mutated SARS-COV-2 with reduced human ACE2 receptor binding affinity have emerged in India based on a sample collected on 27th January 2020.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 8zzi6gu9\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What vaccine candidates are being tested for Covid-19?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "The global pandemic of COVID-19 caused by SARS-CoV-2 (also known as 2019-nCoV and HCoV-19) has posed serious threats to public health and economic stability worldwide, thus calling for development of vaccines against SARS-CoV-2 and other emerging and reemerging coronaviruses. Since SARS-CoV-2 and SARS-CoV have high similarity of their genomic sequences and share the same cellular receptor (ACE2), it is essential to learn the lessons and experiences from the development of SARS-CoV vaccines for the development of SARS-CoV-2 vaccines. In this review, we summarized the current knowledge on the advantages and disadvantages of the SARS-CoV vaccine candidates and prospected the strategies for the development of safe, effective and broad-spectrum coronavirus vaccines for prevention of infection by currently circulating SARS-CoV-2 and other emerging and reemerging coronaviruses that may cause future epidemics or pandemics.\n",
      "Purpose of Review: The goal of this review is to provide a timely overview on efforts to develop a vaccine for the 2019 novel coronavirus SARS-CoV-2, the causative agent of coronavirus disease (COVID-19). Recent Findings: Previous research efforts to develop a severe acute respiratory syndrome coronavirus (SARS-CoV) vaccine in the years following the 2003 pandemic have opened the door for investigators to design vaccine concepts and approaches for the COVID-19 epidemic in China. Both SARS-CoV and SARS-CoV-2 exhibit a high degree of genetic similarity and bind to the same host cell ACE2 receptor. Based on previous experience with SARS-CoV vaccines, it is expected that all COVID-19 vaccines will require careful safety evaluations for immunopotentiation that could lead to increased infectivity or eosinophilic infiltration. Besides this, a COVID-19 vaccine target product profile must address vaccinating at-risk human populations including frontline healthcare workers, individuals over the age of 60, and those with underlying and debilitating chronic conditions. Among the vaccine technologies under evaluation are whole virus vaccines, recombinant protein subunit vaccines, and nucleic acid vaccines. Summary: Each current vaccine strategy has distinct advantages and disadvantages. Therefore, it is paramount that multiple strategies be advanced quickly and then evaluated for safety and efficacy. Ultimately, the safety studies to minimize undesired immunopotentiation will become the most significant bottleneck in terms of time.\n",
      "PURPOSE OF REVIEW: The goal of this review is to provide a timely overview on efforts to develop a vaccine for the 2019 novel coronavirus SARS-CoV-2, the causative agent of coronavirus disease (COVID-19). RECENT FINDINGS: Previous research efforts to develop a severe acute respiratory syndrome coronavirus (SARS-CoV) vaccine in the years following the 2003 pandemic have opened the door for investigators to design vaccine concepts and approaches for the COVID-19 epidemic in China. Both SARS-CoV and SARS-CoV-2 exhibit a high degree of genetic similarity and bind to the same host cell ACE2 receptor. Based on previous experience with SARS-CoV vaccines, it is expected that all COVID-19 vaccines will require careful safety evaluations for immunopotentiation that could lead to increased infectivity or eosinophilic infiltration. Besides this, a COVID-19 vaccine target product profile must address vaccinating at-risk human populations including frontline healthcare workers, individuals over the age of 60, and those with underlying and debilitating chronic conditions. Among the vaccine technologies under evaluation are whole virus vaccines, recombinant protein subunit vaccines, and nucleic acid vaccines. SUMMARY: Each current vaccine strategy has distinct advantages and disadvantages. Therefore, it is paramount that multiple strategies be advanced quickly and then evaluated for safety and efficacy. Ultimately, the safety studies to minimize undesired immunopotentiation will become the most significant bottleneck in terms of time.\n",
      "As the world is witnessing the epidemic of COVID-19, a disease caused by a novel coronavirus, SARS-CoV-2, emerging genetics and clinical evidences suggest a similar path to those of SARS and MERS. The rapid genomic sequencing and open access data, together with advanced vaccine technology, are expected to give us more knowledge on the pathogen itself, including the host immune response as well as the plan for therapeutic vaccines in the near future. This review aims to provide a comparative view among SARS-CoV, MERS-CoV and the newly epidemic SARS-CoV-2, in the hope to gain a better understanding of the host-pathogen interaction, host immune responses, and the pathogen immune evasion strategies. This predictive view may help in designing an immune intervention or preventive vaccine for COVID-19 in the near future.\n",
      "As the world is witnessing the epidemic of COVID-19, a disease caused by a novel coronavirus, SARS-CoV-2, emerging genetics and clinical evidences suggest a similar path to those of SARS and MERS. The rapid genomic sequencing and open access data, together with advanced vaccine technology, are expected to give us more knowledge on the pathogen itself, including the host immune response as well as the plan for therapeutic vaccines in the near future. This review aims to provide a comparative view among SARS-CoV, MERS-CoV and the newly epidemic SARS-CoV-2, in the hope to gain a better understanding of the host-pathogen interaction, host immune responses, and the pathogen immune evasion strategies. This predictive view may help in designing an immune intervention or preventive vaccine for COVID-19 in the near future.\n",
      "The 2019-novel coronavirus disease (COVID-19) is caused by SARS-CoV-2 is transmitted from human to human has recently reported in China. Now COVID-19 has been spread all over the world and declared epidemics by WHO. It has caused a Public Health Emergency of International Concern. The elderly and people with underlying diseases are susceptible to infection and prone to serious outcomes, which may be associated with acute respiratory distress syndrome (ARDS) and cytokine storm. Due to the rapid increase of SARS-CoV-2 infections and unavailability of antiviral therapeutic agents, developing an effective SAR-CoV-2 vaccine is urgently required. SARS-CoV-2 which is genetically similar to SARS-CoV and Middle East respiratory syndrome coronavirus (MERS-CoV) is an enveloped, single and positive-stranded RNA virus with a genome comprising 29,891 nucleotides, which encode the 12 putative open reading frames responsible for the synthesis of viral structural and nonstructural proteins which are very similar to SARS-CoV and MERS-CoV proteins. In this review we have summarized various vaccine candidates i.e., nucleotide, subunit and vector based as well as attenuated and inactivated forms, which have already been demonstrated their prophylactic efficacy against MERS-CoV and SARS-CoV, so these candidates could be used as a potential tool for the development of a safe and effective vaccine against SARS-CoV-2.\n",
      "Abstract The emergence of the strain of coronavirus SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) that causes corona virus disease 2019 (COVID-19) and its impact on in the world have made imperative progress to develop an effective and safe vaccine. Despite several measures undertaken, the spread of this virus is ongoing. So far, more than 1,560,000 cases and 1000,000 deaths occurred in the world. Efforts have been made to develop vaccines against human coronavirus (CoV) infections such as MERS and SARS. However, currently, no approved vaccine exists for these coronavirus strains. Such Previous research efforts to develop a coronavirus vaccine in the years following the 2003 pandemic have opened the door for the scientist to design a new vaccine for the COVID-19. Both SARS-CoV and SARS-CoV-2 has a high degree of genetic similarity and bind to the same host cell ACE2 receptor. By using different vaccine development platforms including whole virus vaccines, recombinant protein subunit vaccines, and nucleic acid vaccines several candidates displayed efficacy in vitro studies but few progressed to clinical trials. This review provides a brief introduction of the general features of SARS-CoV-2 and discusses the current progress of ongoing advances in designing vaccine development efforts to counter COVID-19.\n",
      "A novel coronavirus SARS-CoV-2 causing Coronavirus disease 2019 (COVID-19) has entered the human population and has spread rapidly around the world in the first half of 2020 causing a global pandemic. The virus uses its spike glycoprotein receptor-binding domain to interact with host cell angiotensin-converting enzyme 2 (ACE2) sites to initiate a cascade of events that culminate in severe acute respiratory syndrome in some individuals. In efforts to curtail viral spread, authorities initiated far-reaching lockdowns that have disrupted global economies. The scientific and medical communities are mounting serious efforts to limit this pandemic and subsequent waves of viral spread by developing preventative vaccines and repurposing existing drugs as potential therapies. In this review, we focus on the latest developments in COVID-19 vaccine development, including results of the first Phase I clinical trials and describe a number of the early candidates that are emerging in the field. We seek to provide a balanced coverage of the seven main platforms used in vaccine development that will lead to a desired target product profile for the “ideal” vaccine. Using tales of past vaccine discovery efforts that have taken many years or that have failed, we temper over exuberant enthusiasm with cautious optimism that the global medical community will reach the elusive target to treat COVID-19 and end the pandemic.\n",
      "BACKGROUND AND OBJECTIVE: Coronavirus disease (COVID-19) is an ongoing pandemic caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Due to the incessant spread of the disease with substantial morbidity and mortality rates, there is an urgent demand for effective therapeutics and vaccines to control and diminish this pandemic. A critical step in the crosstalk between the virus and the host cell is the binding of SARS-CoV-2 spike protein to the angiotensin-converting enzyme 2 (ACE2) receptor present on the surface of the host cells. Hence, inhibition of this interaction could be a promising strategy to combat the SARS-CoV-2 infection. METHODS: Docking and Molecular Dynamics (MD) simulation studies revealed that designed peptide maintains their secondary structure and provide a highly specific and stable binding (blocking) to SARS-CoV-2. RESULTS: We have designed a novel peptide that could inhibit SARS-CoV-2 spike protein interaction with ACE2, thereby blocking the cellular entry of the virus. CONCLUSION: Our findings suggest that computationally developed inhibitory peptide may be developed as an anti-SARS-CoV-2 agent for the treatment of SARS-CoV-2 infection. We further plan to pursue the peptide in cell-based assays and eventually for clinical trials.\n",
      "The emergence of the strain of coronavirus SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) and its impact on global health have made imperative the development of effective and safe vaccines for this lethal strain. SARS-CoV-2 now adds to the list of coronavirus diseases that have threatened global health, along with the SARS (severe acute respiratory syndrome) and MERS (Middle East respiratory syndrome) coronaviruses that emerged in 2002/2003 and 2012, respectively. As of April 2020, no vaccine is commercially available for these coronavirus strains. Nevertheless, the knowledge obtained from the vaccine development efforts for MERS and SARS can be of high value for COVID-19 (coronavirus disease 2019). Here, we review the past and ongoing vaccine development efforts for clinically relevant coronavirus strains with the intention that this information helps in the development of effective and safe vaccines for COVID-19. In addition, information from naturally exposed individuals and animal models to coronavirus strains is described for the same purpose of helping into the development of effective vaccines against COVID-19.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: vf32wxkx\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "The newly identified SARS-CoV-2 has now been reported from around 183 countries with more than a million confirmed human cases including more than 68000 deaths. The genomes of SARS-COV-2 strains isolated from different parts of the world are now available and the unique features of constituent genes and proteins have gotten substantial attention recently. Spike glycoprotein is widely considered as a possible target to be explored because of its role during the entry of coronaviruses into host cells. We analyzed 320 whole-genome sequences and 320 spike protein sequences of SARS-CoV-2 using multiple sequence alignment tools. In this study, 483 unique variations have been identified among the genomes including 25 non-synonymous mutations and one deletion in the spike protein of SARS-CoV-2. Among the 26 variations detected, 12 variations were located at the N-terminal domain and 6 variations at the receptor-binding domain (RBD) which might alter the interaction with receptor molecules. In addition, 22 amino acid insertions were identified in the spike protein of SARS-CoV-2 in comparison with that of SARS-CoV. Phylogenetic analyses of spike protein revealed that Bat coronavirus have a close evolutionary relationship with circulating SARS-CoV-2. The genetic variation analysis data presented in this study can help a better understanding of SARS-CoV-2 pathogenesis. Based on our findings, potential inhibitors can be designed and tested targeting these proposed sites of variation.\n",
      "The newly identified SARS-CoV-2 has now been reported from around 185 countries with more than a million confirmed human cases including more than 120,000 deaths. The genomes of SARS-COV-2 strains isolated from different parts of the world are now available and the unique features of constituent genes and proteins need to be explored to understand the biology of the virus. Spike glycoprotein is one of the major targets to be explored because of its role during the entry of coronaviruses into host cells. We analyzed 320 whole-genome sequences and 320 spike protein sequences of SARS-CoV-2 using multiple sequence alignment. In this study, 483 unique variations have been identified among the genomes of SARS-CoV-2 including 25 nonsynonymous mutations and one deletion in the spike (S) protein. Among the 26 variations detected, 12 variations were located at the N-terminal domain and 6 variations at the receptor-binding domain (RBD) which might alter the interaction of S protein with the host receptor angiotensin converting enzyme-2 (ACE2). Besides, 22 amino acid insertions were identified in the spike protein of SARS-CoV-2 in comparison with that of SARS-CoV. Phylogenetic analyses of spike protein revealed that Bat coronavirus have a close evolutionary relationship with circulating SARS-CoV-2. The genetic variation analysis data presented in this study can help a better understanding of SARS-CoV-2 pathogenesis. Based on results reported herein, potential inhibitors against S protein can be designed by considering these variations and their impact on protein structure.\n",
      "The newly identified SARS-CoV-2 has now been reported from around 185 countries with more than a million confirmed human cases including more than 120,000 deaths. The genomes of SARS-COV-2 strains isolated from different parts of the world are now available and the unique features of constituent genes and proteins need to be explored to understand the biology of the virus. Spike glycoprotein is one of the major targets to be explored because of its role during the entry of coronaviruses into host cells. We analyzed 320 whole-genome sequences and 320 spike protein sequences of SARS-CoV-2 using multiple sequence alignment. In this study, 483 unique variations have been identified among the genomes of SARS-CoV-2 including 25 nonsynonymous mutations and one deletion in the spike (S) protein. Among the 26 variations detected in S, 12 variations were located at the N-terminal domain (NTD) and 6 variations at the receptor-binding domain (RBD) which might alter the interaction of S protein with the host receptor angiotensin-converting enzyme 2 (ACE2). Besides, 22 amino acid insertions were identified in the spike protein of SARS-CoV-2 in comparison with that of SARS-CoV. Phylogenetic analyses of spike protein revealed that Bat coronavirus have a close evolutionary relationship with circulating SARS-CoV-2. The genetic variation analysis data presented in this study can help a better understanding of SARS-CoV-2 pathogenesis. Based on results reported herein, potential inhibitors against S protein can be designed by considering these variations and their impact on protein structure.\n",
      "SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) is a novel coronavirus causing an outbreak of COVID-19 globally in the past six months. A relatively higher divergence on the spike protein of SASR-CoV-2 enables it to transmit across species efficiently. We particularly believe that the adaptive mutations of the receptor-binding domain (RBD) of spike protein in SARS-CoV-2 might be essential to its high transmissibility among humans. Thus here we collected 2,142 high-quality genome sequences of SARS-CoV-2 from 160 regions in over 50 countries and reconstructed their phylogeny, and also analyzed the interaction between the polymorphisms of spike protein and human ACE2 (hACE2). Phylogenetic analysis of SARS-CoV-2 and coronavirus in other hosts show SARS-CoV-2 is highly possible originated from Bat-CoV (RaTG13) found in horseshoe bat and a recombination event may occur on the spike protein of Pangolin-CoV to imbue it the ability to infect humans. Moreover, compared to the S gene of SARS-CoV-2, it is more conserved in the direct-binding sites of RBD and we noticed that spike protein of SARS-CoV-2 may under a consensus evolution to adapt to human hosts better. 3,860 amino acid mutations in spike protein RBD (T333-C525) of SARS-CoV-2 were simulated and their stability and affinity binding to hACE2 (S19-D615) were calculated. Our analysis indicates SARS-CoV-2 could infect humans from different populations with no preference, and a higher divergence in the spike protein of SARS-CoV-2 at the early stage of this pandemic may be a good indicator that could show the pathway of SARS-CoV-2 transmitting from the natural reservoir to human beings.\n",
      "Background SARS-CoV-2 most likely evolved from a bat beta-coronavirus and started infecting humans in December 2019. Since then it has rapidly infected people around the world, with more than 4.5 million confirmed cases by the middle of May 2020. Early genome sequencing of the virus has enabled the development of molecular diagnostics and the commencement of therapy and vaccine development. The analysis of the early sequences showed relatively few evolutionary selection pressures. However, with the rapid worldwide expansion into diverse human populations, significant genetic variations are becoming increasingly likely. The current limitations on social movement between countries also offers the opportunity for these viral variants to become distinct strains with potential implications for diagnostics, therapies and vaccines. Methods We used the current sequencing archives (NCBI and GISAID) to investigate 15,487 whole genomes, looking for evidence of strain diversification and selective pressure. Results We used 6,294 SNPs to build a phylogenetic tree of SARS-CoV-2 diversity and noted strong evidence for the existence of two major clades and six sub-clades, unevenly distributed across the world. We also noted that convergent evolution has potentially occurred across several locations in the genome, showing selection pressures, including on the spike glycoprotein where we noted a potentially critical mutation that could affect its binding to the ACE2 receptor. We also report on mutations that could prevent current molecular diagnostics from detecting some of the sub-clades. Conclusion The worldwide whole genome sequencing effort is revealing the challenge of developing SARS-CoV-2 containment tools suitable for everyone and the need for data to be continually evaluated to ensure accuracy in outbreak estimations.\n",
      "The recent emergence of a novel coronavirus (SARS-CoV-2) is causing a severe global health threat characterized by severe acute respiratory syndrome (Covid-19). At the moment, there is no specific treatment for this disease, and vaccines are still under development. The structural protein Spike is essential for virus infection and has been used as the main target for vaccine and serological diagnosis test development. We analysed 2363 sequences of the Spike protein from SARS-CoV-2 isolates and identified variability in 44 amino acid residues and their worldwide distribution in all continents. We used the three-dimensional structure of the homo-trimer model to predict conformational epitopes of B-cell, and sequence of Spike protein Wuhan-Hu-1 to predict linear epitopes of T-Cytotoxic and T-Helper cells. We identified 45 epitopes with amino acid variations. Finally, we showed the distribution of mutations within the epitopes. Our findings can help researches to identify more efficient strategies for the development of vaccines, therapies, and serological diagnostic tests based on the Spike protein of Sars-Cov-2.\n",
      "The recent pandemic of SARS-CoV-2 infection has affected more than 3.0 million people worldwide with more than 200 thousand reported deaths. The SARS-CoV-2 genome has the capability of gaining rapid mutations as the virus spreads. Whole-genome sequencing data offers a wide range of opportunities to study mutation dynamics. The advantage of an increasing amount of whole-genome sequence data of SARS-CoV-2 intrigued us to explore the mutation profile across the genome, to check the genome diversity, and to investigate the implications of those mutations in protein stability and viral transmission. We have identified frequently mutated residues by aligning ~660 SARS-CoV-2 genomes and validated in 10,000 datasets available in GISAID Nextstrain. We further evaluated the potential of these frequently mutated residues in protein structure stability of spike glycoprotein and their possible functional consequences in other proteins. Among the 11 genes, surface glycoprotein, nucleocapsid, ORF1ab, and ORF8 showed frequent mutations, while envelop, membrane, ORF6, ORF7a and ORF7b showed conservation in terms of amino acid substitutions. Combined analysis with the frequently mutated residues identified 20 viral variants, among which 12 specific combinations comprised more than 97% of the isolates considered for the analysis. Some of the mutations across different proteins showed co-occurrences, suggesting their structural and/or functional interaction among different SARS-COV-2 proteins, and their involvement in adaptability and viral transmission. Analysis of protein structure stability of surface glycoprotein mutants indicated the viability of specific variants and are more prone to be temporally and spatially distributed across the globe. A similar empirical analysis of other proteins indicated the existence of important functional implications of several variants. Identification of frequently mutated variants among COVID-19 patients might be useful for better clinical management, contact tracing, and containment of the disease.\n",
      "Abstract As severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) continues to disperse globally with worrisome speed, identifying amino acid variations in the virus could help to understand the characteristics of it. Here, we studied 489 SARS-CoV-2 genomes obtained from 32 countries from the Nextstrain database and performed phylogenetic tree analysis by clade, country, and genotype of the surface spike glycoprotein (S protein) at site 614. We found that virus strains from mainland China were mostly distributed in Clade B and Clade undefined in the phylogenetic tree, with very few found in Clade A. In contrast, Clades A2 (one case) and A2a (112 cases) predominantly contained strains from European regions. Moreover, Clades A2 and A2a differed significantly from those of mainland China in age of infected population (P = 0.0071, mean age 40.24 to 46.66), although such differences did not exist between the US and mainland China. Further analysis demonstrated that the variation of the S protein at site 614 (QHD43416.1: p.614D>G) was a characteristic of stains in Clades A2 and A2a. Importantly, this variation was predicted to have neutral or benign effects on the function of the S protein. In addition, global quality estimates and 3D protein structures tended to be different between the two S proteins. In summary, we identified different genomic epidemiology among SARS-CoV-2 strains in different clades, especially in an amino acid variation of the S protein at 614, revealing potential viral genome divergence in SARS-CoV-2 strains.\n",
      "Monitoring the mutation dynamics of SARS-CoV-2 is critical for the development of effective approaches to contain the pathogen. By analyzing 106 SARS-CoV-2 and 39 SARS genome sequences, we provided direct genetic evidence that SARS-CoV-2 has a much lower mutation rate than SARS. Minimum Evolution phylogeny analysis revealed the putative original status of SARS-CoV-2 and the early-stage spread history. The discrepant phylogenies for the spike protein and its receptor binding domain proved a previously reported structural rearrangement prior to the emergence of SARS-CoV-2. Despite that we found the spike glycoprotein of SARS-CoV-2 is particularly more conserved, we identified a mutation that leads to weaker receptor binding capability, which concerns a SARS-CoV-2 sample collected on 27th January 2020 from India. This represents the first report of a significant SARS-CoV-2 mutant, and raises the alarm that the ongoing vaccine development may become futile in future epidemic if more mutations were identified. Highlights Based on the currently available genome sequence data, we proved that SARS-COV-2 genome has a much lower mutation rate and genetic diversity than SARS during the 2002-2003 outbreak. The spike (S) protein encoding gene of SARS-COV-2 is found relatively more conserved than other protein-encoding genes, which is a good indication for the ongoing antiviral drug and vaccine development. Minimum Evolution phylogeny analysis revealed the putative original status of SARS-CoV-2 and the early-stage spread history. We confirmed a previously reported rearrangement in the S protein arrangement of SARS-COV-2, and propose that this rearrangement should have occurred between human SARS-CoV and a bat SARS-CoV, at a time point much earlier before SARS-COV-2 transmission to human. We provided first evidence that a mutated SARS-COV-2 with reduced human ACE2 receptor binding affinity have emerged in India based on a sample collected on 27th January 2020.\n",
      "Abstract SARS-CoV-2 is a SARS-like coronavirus of likely zoonotic origin first identified in December 2019 in Wuhan, the capital of China's Hubei province. The virus has since spread globally, resulting in the currently ongoing COVID-19 pandemic. The first whole genome sequence was published on January 52,020, and thousands of genomes have been sequenced since this date. This resource allows unprecedented insights into the past demography of SARS-CoV-2 but also monitoring of how the virus is adapting to its novel human host, providing information to direct drug and vaccine design. We curated a dataset of 7666 public genome assemblies and analysed the emergence of genomic diversity over time. Our results are in line with previous estimates and point to all sequences sharing a common ancestor towards the end of 2019, supporting this as the period when SARS-CoV-2 jumped into its human host. Due to extensive transmission, the genetic diversity of the virus in several countries recapitulates a large fraction of its worldwide genetic diversity. We identify regions of the SARS-CoV-2 genome that have remained largely invariant to date, and others that have already accumulated diversity. By focusing on mutations which have emerged independently multiple times (homoplasies), we identify 198 filtered recurrent mutations in the SARS-CoV-2 genome. Nearly 80% of the recurrent mutations produced non-synonymous changes at the protein level, suggesting possible ongoing adaptation of SARS-CoV-2. Three sites in Orf1ab in the regions encoding Nsp6, Nsp11, Nsp13, and one in the Spike protein are characterised by a particularly large number of recurrent mutations (>15 events) which may signpost convergent evolution and are of particular interest in the context of adaptation of SARS-CoV-2 to the human host. We additionally provide an interactive user-friendly web-application to query the alignment of the 7666 SARS-CoV-2 genomes.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 8cvjsisw\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Importance The pandemic of coronavirus disease 2019 (COVID-19) caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) presents an unprecedented challenge to identify effective drugs for prevention and treatment. Given the rapid pace of scientific discovery and clinical data generated by the large number of people rapidly infected by SARS-CoV-2, clinicians need accurate evidence regarding effective medical treatments for this infection. Observations No proven effective therapies for this virus currently exist. The rapidly expanding knowledge regarding SARS-CoV-2 virology provides a significant number of potential drug targets. The most promising therapy is remdesivir. Remdesivir has potent in vitro activity against SARS-CoV-2, but it is not US Food and Drug Administration approved and currently is being tested in ongoing randomized trials. Oseltamivir has not been shown to have efficacy, and corticosteroids are currently not recommended. Current clinical evidence does not support stopping angiotensin-converting enzyme inhibitors or angiotensin receptor blockers in patients with COVID-19. Conclusions and Relevance The COVID-19 pandemic represents the greatest global public health crisis of this generation and, potentially, since the pandemic influenza outbreak of 1918. The speed and volume of clinical trials launched to investigate potential therapies for COVID-19 highlight both the need and capability to produce high-quality evidence even in the middle of a pandemic. No therapies have been shown effective to date.\n",
      "Importance: The pandemic of coronavirus disease 2019 (COVID-19) caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) presents an unprecedented challenge to identify effective drugs for prevention and treatment. Given the rapid pace of scientific discovery and clinical data generated by the large number of people rapidly infected by SARS-CoV-2, clinicians need accurate evidence regarding effective medical treatments for this infection. Observations: No proven effective therapies for this virus currently exist. The rapidly expanding knowledge regarding SARS-CoV-2 virology provides a significant number of potential drug targets. The most promising therapy is remdesivir. Remdesivir has potent in vitro activity against SARS-CoV-2, but it is not US Food and Drug Administration approved and currently is being tested in ongoing randomized trials. Oseltamivir has not been shown to have efficacy, and corticosteroids are currently not recommended. Current clinical evidence does not support stopping angiotensin-converting enzyme inhibitors or angiotensin receptor blockers in patients with COVID-19. Conclusions and Relevance: The COVID-19 pandemic represents the greatest global public health crisis of this generation and, potentially, since the pandemic influenza outbreak of 1918. The speed and volume of clinical trials launched to investigate potential therapies for COVID-19 highlight both the need and capability to produce high-quality evidence even in the middle of a pandemic. No therapies have been shown effective to date.\n",
      "Abstract An outbreak related to the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) was first reported in Wuhan, China in December 2019. An extremely high potential for dissemination resulted in the global coronavirus disease 2019 (COVID-19) pandemic in 2020. Despite the worsening trends of COVID-19, no drugs are validated to have significant efficacy in clinical treatment of COVID-19 patients in large-scale studies. Remdesivir is considered the most promising antiviral agent; it works by inhibiting the activity of RNA-dependent RNA polymerase (RdRp). A large-scale study investigating the clinical efficacy of remdesivir (200 mg on day 1, followed by 100 mg once daily) is on-going. The other excellent anti-influenza RdRp inhibitor favipiravir is also being clinically evaluated for its efficacy in COVID-19 patients. The protease inhibitor lopinavir/ritonavir (LPV/RTV) alone is not shown to provide better antiviral efficacy than standard care. However, the regimen of LPV/RTV plus ribavirin was shown to be effective against SARS-CoV in vitro. Another promising alternative is hydroxychloroquine (200 mg thrice daily) plus azithromycin (500 mg on day 1, followed by 250 mg once daily on day 2-5), which showed excellent clinical efficacy on Chinese COVID-19 patients and anti-SARS-CoV-2 potency in vitro. The roles of teicoplanin (which inhibits the viral genome exposure in cytoplasm) and monoclonal and polyclonal antibodies in the treatment of SARS-CoV-2 are under investigation. Avoiding the prescription of non-steroidal anti-inflammatory drugs, angiotensin converting enzyme inhibitors, or angiotensin II type I receptor blockers is advised for COVID-19 patients.\n",
      "BACKGROUND: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), termed coronavirus disease 2019 (COVID-19) by the World Health Organization, is a newly emerging zoonotic agent that emerged in China in December 2019. No specific treatment for COVID-19 is currently available. Usual palliative treatment includes maintaining hydration and nutrition and controlling fever and cough. The clinical severity and extent of transmission need to be determined, and therapeutic options need to be developed and optimized. METHODS: The present review discusses the recent repurposing of drugs for COVID-19 treatment. RESULTS: Several compounds, including remdesivir, lopinavir, ritonavir, interferon-ß, ribavirin, chloroquine/hydroxychloroquine, azithromycin, tocilizumab, and ivermectin, have emerged as promising alternatives. They block the virus from entering host cells, prevent viral replication, and attenuate exacerbation of the host's immune response. CONCLUSION: Although some evidence indicates the positive actions of different classes of compounds for the treatment of COVID-19, few clinical assays have been established to definitively demonstrate their therapeutic value in humans. Multicenter clinical studies are urgently needed to validate and standardize therapeutic regimens that involve these agents. Although science has not yet presented us with a specific drug against COVID-19, the repurposing of drugs appears to be promising in our fight against this devastating disease.\n",
      "The Coronavirus disease 19 (COVID-19) outbreak has been recognized as a global threat to public health. It is caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and no effective therapies currently exist against this novel viral agent. Along with extensive public health measures, an unprecedented global effort in identifying effective drugs for the treatment is being implemented. Potential drug targets are emerging as the result of a fast-evolving understanding of SARS-CoV-2 virology, host response to the infection, and clinical course of the disease. This brief review focuses on the latest and most promising pharmacological treatments against COVID-19 currently under investigation and discuss their potential use based on either documented efficacy in similar viral infections, or their activity against inflammatory syndromes. Ongoing clinical trials are also emphasized.\n",
      "The Coronavirus disease 19 (COVID-19) outbreak has been recognized as a global threat to public health. It is caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and no effective therapies currently exist against this novel viral agent. Along with extensive public health measures, an unprecedented global effort in identifying effective drugs for the treatment is being implemented. Potential drug targets are emerging as the result of a fast-evolving understanding of SARS-CoV-2 virology, host response to the infection, and clinical course of the disease. This brief review focuses on the latest and most promising pharmacological treatments against COVID-19 currently under investigation and discuss their potential use based on either documented efficacy in similar viral infections, or their activity against inflammatory syndromes. Ongoing clinical trials are also emphasized.\n",
      "BACKGROUND Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), termed coronavirus disease 2019 (COVID-19) by the World Health Organization, is a newly emerging zoonotic agent that emerged in China in December 2019. No specific treatment for COVID-19 is currently available. Usual palliative treatment includes maintaining hydration and nutrition and controlling fever and cough. The clinical severity and extent of transmission need to be determined, and therapeutic options need to be developed and optimized. METHODS The present review discusses the recent repurposing of drugs for COVID-19 treatment. RESULTS Several compounds, including remdesivir, lopinavir, ritonavir, interferon-β, ribavirin, chloroquine/hydroxychloroquine, azithromycin, tocilizumab, and ivermectin, have emerged as promising alternatives. They block the virus from entering host cells, prevent viral replication, and attenuate exacerbation of the host's immune response. CONCLUSION Although some evidence indicates the positive actions of different classes of compounds for the treatment of COVID-19, few clinical assays have been established to definitively demonstrate their therapeutic value in humans. Multicenter clinical studies are urgently needed to validate and standardize therapeutic regimens that involve these agents. Although science has not yet presented us with a specific drug against COVID-19, the repurposing of drugs appears to be promising in our fight against this devastating disease.\n",
      "The current pandemic of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has presented unprecedented challenges to the healthcare systems in almost every country around the world. Currently, there are no proven effective vaccines or therapeutic agents against the virus. Current clinical management includes infection prevention and control measures and supportive care including supplemental oxygen and mechanical ventilatory support. Evolving research and clinical data regarding the virologic SARS-CoV-2 suggest a potential list of repurposed drugs with appropriate pharmacological effects and therapeutic efficacies in treating COVID-19 patients. In this review, we will update and summarize the most common and plausible drugs for the treatment of COVID-19 patients. These drugs and therapeutic agents include antiviral agents (remdesivir, hydroxychloroquine, chloroquine, lopinavir, umifenovir, favipiravir, and oseltamivir), and supporting agents (Ascorbic acid, Azithromycin, Corticosteroids, Nitric oxide, IL-6 antagonists), among others. We hope that this review will provide useful and most updated therapeutic drugs to prevent, control, and treat COVID-19 patients until the approval of vaccines and specific drugs targeting SARS-CoV-2.\n",
      "The global pandemic of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2 or 2019-nCoV) has prompted multiple clinical trials to jumpstart search for anti-SARS-CoV-2 therapies from existing drugs, including those with reported in vitro efficacies as well as those ones that are not known to inhibit SARS-CoV-2, such as ritonavir/lopinavir and favilavir. Here we report that after screening 19 antiviral drugs that are either in clinical trials or with proposed activity against SARS-CoV-2, remdesivir was the most effective. Chloroquine only effectively protected virus-induced cytopathic effect at around 30 µM with a therapeutic index of 1.5. Our findings also suggest that velpatasvir, ledipasvir, ritonavir, litonavir, lopinavir, favilavir, sofosbuvir, danoprevir, and pocapavir do not have direct antiviral effect.\n",
      "INTRODUCTION: The coronavirus disease 2019 (COVID-19) caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has developed since December 2019. It has caused a global pandemic with more than three hundred thousand case fatalities. However, apart from supportive care by respirators, no standard medical therapy is validated. AREAS COVERED: This paper presents old drugs with potential in vitro efficacy against SARS-CoV-2. The in vitro database, adverse effects, and potential toxicities of these drugs are reviewed regarding their feasibility of clinical prescription for the treatment of patients with COVID-19. To obtain convincing recommendations, we referred to opinions from the US National Institute of Health regarding drugs repurposed for COVID-19 therapy. EXPERT OPINION: Although strong evidence of well-designed randomized controlled studies regarding COVID-19 therapy is presently lacking, remdesivir, teicoplanin, hydroxychloroquine (not in combination with azithromycin), and ivermectin might be effective antiviral drugs and are deemed promising candidates for controlling SARS-CoV-2. In addition, tocilizumab might be considered as the supplementary treatment for COVID-19 patients with cytokine release syndrome. In future, clinical trials regarding a combination of potentially effective drugs against SARS-CoV-2 need to be conducted to establish the optimal regimen for the treatment of patients with moderate-to-severe COVID-19.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: nwu4tdgh\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "On March 11, 2020, the World Health Organization declared the worldwide spread of the infectious disease COVID-19, caused by a new strain of coronavirus, SARS-CoV-2, as a pandemic. Like in all other infectious diseases, the host immune system plays a key role in our defense against SARS-CoV-2 infection. However, viruses are able to evade the immune attack and proliferate and, in susceptible individuals, cause severe inflammatory response known as cytokine storm, particularly in the lungs. The advancement in our understanding of the mechanisms underlying the host immune responses promises to facilitate the development of approaches for prevention or treatment of diseases. Components of immune system, such as antibodies, can also be used to develop sensitive and specific diagnostic methods as well as novel therapeutic agents. In this review, we summarize our knowledge about how the host mounts immune responses to infection by SARS-CoV-2. We also describe the diagnostic methods being used for COVID-19 identification and summarize the current status of various therapeutic strategies, including vaccination, being considered for treatment of the disease.\n",
      "With the worldwide spread of the novel Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2) resulting in declaration of a pandemic by the World Health Organization (WHO) on March 11, 2020, the SARS-CoV-2-induced Coronavirus disease-19 (COVID-19) has become one of the main challenges of our times. The high infection rate and the severe disease course led to major safety and social restriction measures worldwide. There is an urgent need of unbiased expert knowledge guiding the development of efficient treatment and prevention strategies. This report summarizes current immunological data on mechanisms associated with the SARS-CoV-2 infection and COVID-19 development and progression to the most severe forms. We characterize the differences between adequate innate and adaptive immune response in mild disease and the deep immune dysfunction in the severe multi-organ disease. The similarities of the human immune response to SARS-CoV-2 and the SARS-CoV and MERS-CoV are underlined. We also summarize known and potential SARS-CoV-2 receptors on epithelial barriers, immune cells, endothelium and clinically involved organs such as lung, gut, kidney, cardiovascular and neuronal system. Finally, we discuss the known and potential mechanisms underlying the involvement of comorbidities, gender and age in development of COVID-19. Consequently, we highlight the knowledge gaps and urgent research requirements to provide a quick roadmap for ongoing and needed COVID-19 studies.\n",
      "The outbreak of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is spreading globally, and the WHO has declared this outbreak a pandemic. Vaccines are an effective way to prevent the rapid spread of COVID-19. Furthermore, the immune response against SARS-CoV-2 infection needs to be understood for the development of an efficient and safe vaccine. Here, we review the current understanding of vaccine targets and the status of vaccine development for COVID-19. We also describe host immune responses to highly pathogenic human coronaviruses in terms of innate and adaptive immunities.\n",
      "The outbreak of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is spreading globally, and the WHO has declared this outbreak a pandemic. Vaccines are an effective way to prevent the rapid spread of COVID-19. Furthermore, the immune response against SARS-CoV-2 infection needs to be understood for the development of an efficient and safe vaccine. Here, we review the current understanding of vaccine targets and the status of vaccine development for COVID-19. We also describe host immune responses to highly pathogenic human coronaviruses in terms of innate and adaptive immunities.\n",
      "With the worldwide spread of the novel Severe Acute Respiratory Syndrome Coronavirus‐2 (SARS‐CoV‐2) resulting in declaration of a pandemic by the World Health Organization (WHO) on March 11, 2020, the SARS‐CoV‐2‐induced Coronavirus disease‐19 (COVID‐19) has become one of the main challenges of our times. The high infection rate and the severe disease course led to major safety and social restriction measures worldwide. There is an urgent need of unbiased expert knowledge guiding the development of efficient treatment and prevention strategies. This report summarizes current immunological data on mechanisms associated with the SARS‐CoV‐2 infection and COVID‐19 development and progression to the most severe forms. We characterize the differences between adequate innate and adaptive immune response in mild disease and the deep immune dysfunction in the severe multi‐organ disease. The similarities of the human immune response to SARS‐CoV‐2 and the SARS‐CoV and MERS‐CoV are underlined. We also summarize known and potential SARS‐CoV‐2 receptors on epithelial barriers, immune cells, endothelium and clinically involved organs such as lung, gut, kidney, cardiovascular and neuronal system. Finally, we discuss the known and potential mechanisms underlying the involvement of comorbidities, gender and age in development of COVID‐19. Consequently, we highlight the knowledge gaps and urgent research requirements to provide a quick roadmap for ongoing and needed COVID‐19 studies.\n",
      "The pandemic caused by SARS-COV2 Virus/COVID-19, which was initially reported in China in December 2019, has become a major global health concern. COVID-19 can manifest with cytokine storm - an exaggerated systemic inflammatory phenomenon due to over-production of proinflammatory cytokines by immune cells that results in diffuse inflammatory lung disease and acute respiratory distress syndrome. It may be complicated by septic shock and subsequent multi-organ failure. Based on the most recently published evidence, this article will review and discuss comprehensive information on its clinical manifestations, laboratory tests, potential therapeutics, and prevention guidelines.\n",
      "The pandemic caused by SARS-COV2 Virus/COVID-19, which was initially reported in China in December 2019, has become a major global health concern COVID-19 can manifest with cytokine storm - an exaggerated systemic inflammatory phenomenon due to over-production of proinflammatory cytokines by immune cells that results in diffuse inflammatory lung disease and acute respiratory distress syndrome It may be complicated by septic shock and subsequent multi-organ failure Based on the most recently published evidence, this article will review and discuss comprehensive information on its clinical manifestations, laboratory tests, potential therapeutics, and prevention guidelines\n",
      "IMPACT STATEMENT In late 2019, a novel virus called SARS-CoV-2, expanded globally from Wuhan, China and was declared a pandemic on 11 March 2020 by the WHO. The mechanism of virus entry inside the host cell depends upon the cellular proteases including cathepsins, HAT, and TMPRSS2, which splits up the spike protein and causes further penetration. MERS coronavirus uses DPP4, while coronavirus HCoV-NL63 and SARS-CoV and SARS-CoV-2 employ ACE-2 as the key receptor. Cytokine storm syndrome was analyzed in critically ill nCOVID-19 patients and it is presented with high inflammatory mediators, systemic inflammation, and multiple organ failure. Among various inflammatory mediators, the level of interleukins (IL-2, IL-7, IL-10), G-CSF, MIP1A, MCP1, and TNF-α was reported to be higher in critically ill patients. Understanding this molecular mechanism of ILs, T cells, and dendritic cells will be helpful to design immunotherapy and novel drugs for the treatment of nCOVID-19 infection.\n",
      "Coronavirus disease-2019 (COVID-19) is caused by the newly emerged virus severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and was recently declared as a pandemic by the World Health Organization. In its severe form, the disease is characterized by acute respiratory distress syndrome, and there are no targeted intervention strategies to treat or prevent it. The immune response is thought to both contribute to the pathogenesis of disease and provide protection during its resolution. Thus, understanding the immune response to SARS-CoV-2 is of the utmost importance for developing and testing vaccines and therapeutics. In this review, we discuss the earliest knowledge and hypotheses of the mechanisms of immune pathology in the lung during acute infection as well at the later stages of disease resolution, recovery, and immune memory formation.\n",
      "Coronavirus disease-2019 (COVID-19) is caused by the newly emerged virus severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and was recently declared as a pandemic by the World Health Organization. In its severe form, the disease is characterized by acute respiratory distress syndrome, and there are no targeted intervention strategies to treat or prevent it. The immune response is thought to both contribute to the pathogenesis of disease and provide protection during its resolution. Thus, understanding the immune response to SARS-CoV-2 is of the utmost importance for developing and testing vaccines and therapeutics. In this review, we discuss the earliest knowledge and hypotheses of the mechanisms of immune pathology in the lung during acute infection as well at the later stages of disease resolution, recovery, and immune memory formation.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: khpc9f98\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "how does the coronavirus respond to changes in the weather\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Frequently touched surfaces of a university classroom that is cleaned daily contained viable human coronavirus 229E (CoV-229E). Tests of a CoV-229E laboratory strain under conditions that simulated the ambient light, temperature, and relative humidity conditions of the classroom revealed that some of the virus remained viable on various surfaces for 7 days, suggesting CoV-229E is relatively stable in the environment. Our findings reinforce the notion that contact transmission may be possible for this virus.\n",
      "At room temperature, SARS-CoV-2 was stable on environmental surfaces and remained viable up to 7 days on smooth surfaces. This virus could survive for several hours in feces and 3-4 days in urine.\n",
      "SARS-CoV-2, the causative agent of the COVID-19 pandemic, may be transmitted via airborne droplets or contact with surfaces onto which droplets have deposited. In this study, the ability of SARS-CoV-2 to survive in the dark, at two different relative humidity values and within artificial saliva, a clinically relevant matrix, was investigated. SARS-CoV-2 was found to be stable, in the dark, in a dynamic small particle aerosol under the four experimental conditions we tested and viable virus could still be detected after 90 minutes. The decay rate and half-life was determined and decay rates ranged from 0.4 to 2.27 % per minute and the half lives ranged from 30 to 177 minutes for the different conditions. This information can be used for advice and modelling and potential mitigation strategies.\n",
      "BACKGROUND: In late 2019, a novel human coronavirus, SARS-CoV-2, emerged in Wuhan, China. This virus has caused a global pandemic involving more than 200 countries. SARS-CoV-2 is highly adapted to humans and readily transmits from person-to-person. AIM: The aim of this study was to investigate the infectivity of SARS-CoV-2 under various environmental factors, disinfectants and different pH conditions. The efficacy of a variety of laboratory virus inactivation methods and home disinfectants against SARS-CoV-2 were investigated. METHODS: The residual virus in dried form or in solution was titrated on Vero E6 cell line at day 0, 1, 3, 5, and 7 after incubation at different temperatures. The viability of virus was determined after treatment with different disinfectants and pH solutions at room temperature (20â¼25oC). FINDINGS: SARS-CoV-2 was able to retain viability for 3-5 days in dried form or 7 days in solution at room temperature. SARS-CoV-2 could be detected under a wide range of pH conditions from pH4 to pH11 for several days and 1 to 2 days in stool at room temperature but lost 5 logs of infectivity. A variety of commonly used disinfectants and laboratory inactivation procedures were found to reduce viral viability effectively. CONCLUSION: This study demonstrates the stability of SARS-CoV-2 on environmental surfaces and raises the possibility of faecal-oral transmission. Commonly used fixatives, nucleic acid extraction methods and heat inactivation were found to significantly reduce viral infectivity that could ensure hospital and laboratory safety during the COVID-19 pandemic.\n",
      "BACKGROUND: In late 2019, a novel human coronavirus, SARS-CoV-2, emerged in Wuhan, China. This virus has caused a global pandemic involving more than 200 countries. SARS-CoV-2 is highly adapted to humans and readily transmits from person-to-person. AIM: The aim of this study was to investigate the infectivity of SARS-CoV-2 under various environmental factors, disinfectants and different pH conditions. The efficacy of a variety of laboratory virus inactivation methods and home disinfectants against SARS-CoV-2 were investigated. METHODS: The residual virus in dried form or in solution was titrated on Vero E6 cell line at day 0, 1, 3, 5, and 7 after incubation at different temperatures. The viability of virus was determined after treatment with different disinfectants and pH solutions at room temperature (20∼25(o)C). FINDINGS: SARS-CoV-2 was able to retain viability for 3-5 days in dried form or 7 days in solution at room temperature. SARS-CoV-2 could be detected under a wide range of pH conditions from pH4 to pH11 for several days and 1 to 2 days in stool at room temperature but lost 5 logs of infectivity. A variety of commonly used disinfectants and laboratory inactivation procedures were found to reduce viral viability effectively. CONCLUSION: This study demonstrates the stability of SARS-CoV-2 on environmental surfaces and raises the possibility of faecal-oral transmission. Commonly used fixatives, nucleic acid extraction methods and heat inactivation were found to significantly reduce viral infectivity that could ensure hospital and laboratory safety during the COVID-19 pandemic.\n",
      "The disinfection of high-contact surfaces is seen as an infection control practice to prevent the spread of pathogens by fomites. Unfortunately, recontamination of these surfaces can occur any time after the use of common disinfectants. We recently reported on a novel continuously active antimicrobial coating which was shown to reduce the spread of healthcare acquired infections in hospitals. We evaluated a modified coating that demonstrated a residual efficacy against viruses. The coated surfaces were found to be effective against human coronavirus (HCoV) 229E, reducing the concentration of these viruses by greater than 90% in 10 minutes and by greater than 99.9% after two hours of contact. The coating formulation when tested in suspension yielded a greater than 99.99% reduction of HCoV 229E within ten minutes of contact. This outcome presents an opportunity for controlling the transmission of COVID-19 from contaminated fomites.\n",
      "COVID-19 has become a pandemic and is spreading fast worldwide. The COVID-19 virus is transmitted mainly through respiratory droplets and close contact. However, the fecal-oral transmission of the virus has not been ruled out and it is important to ascertain how acidic condition in the stomach affects the infectivity of the virus. Besides, it is unclear how stable the COVID-19 virus is under dry and wet conditions. In the present study, we have shown that the COVID-19 virus is extremely infectious as manifested by the infection of Vero-E6 cells by one PFU (Plaque Forming Unit) of the virus. We then investigated the stability of the COVID-19 virus in wet, dry and acidic (pH2.2) environments at room temperature. Results showed that the COVID-19 virus could survive for three days in wet and dry environments, but the dry condition is less favorable for the survival of the virus. Our study also demonstrated that the COVID-19 virus at a relative high titer (1.2 x 103 PFU) exhibits a certain degree of tolerance to acidic environment at least for 60 minutes. When the virus titer was ≤1.0 x 103 PFU, acid treatment (pH2.2) for 30 or 60 minute resulted in virus inactivation. It suggests that the virus at a high concentration may survive in the acidic environment of the stomach. The finding of the present study will contribute to the control of the spread of the COVID-19 virus.\n",
      "Abstract Strains OC43 and 229E of human coronaviruses (HCoV) cause one-third of common colds and hospital-acquired upper respiratory tract HCoV infections have been reported in premature newborns. To evaluate possible sources of infection, virus survival was studied in aqueous suspensions and on absorptive and non-absorptive surfaces representative of a hospital environment. Virus susceptibility to chemical disinfection with standard products was also characterized. Virus survived in saline solution for as long as six days but less in culture medium, with or without added cells. After drying, HCoV-229E infectivity was still detectable after 3h on various surfaces (aluminum, sterile latex surgical gloves, sterile sponges) but HCoV-OC43 survived 1h or less. Of the various chemical disinfectants tested, Proviodine® reduced the virus infectious titre by at least 50%. This study suggests that surfaces and suspensions can be considered as possible sources of contamination that may lead to hospital-acquired infections with HCoV and should be appropriately disinfected.\n",
      "OBJECTIVE The causal agent for SARS is considered as a novel coronavirus that has never been described both in human and animals previously. The stability of SARS coronavirus in human specimens and in environments was studied. METHODS Using a SARS coronavirus strain CoV-P9, which was isolated from pharyngeal swab of a probable SARS case in Beijing, its stability in mimic human specimens and in mimic environment including surfaces of commonly used materials or in household conditions, as well as its resistance to temperature and UV irradiation were analyzed. A total of 10(6) TCID50 viruses were placed in each tested condition, and changes of the viral infectivity in samples after treatments were measured by evaluating cytopathic effect (CPE) in cell line Vero-E6 at 48 h after infection. RESULTS The results showed that SARS coronavirus in the testing condition could survive in serum, 1:20 diluted sputum and feces for at least 96 h, whereas it could remain alive in urine for at least 72 h with a low level of infectivity. The survival abilities on the surfaces of eight different materials and in water were quite comparable, revealing reduction of infectivity after 72 to 96 h exposure. Viruses stayed stable at 4 degrees C, at room temperature (20 degrees C) and at 37 degrees C for at least 2 h without remarkable change in the infectious ability in cells, but were converted to be non-infectious after 90-, 60- and 30-min exposure at 56 degrees C, at 67 degrees C and at 75 degrees C, respectively. Irradiation of UV for 60 min on the virus in culture medium resulted in the destruction of viral infectivity at an undetectable level. CONCLUSION The survival ability of SARS coronavirus in human specimens and in environments seems to be relatively strong. Heating and UV irradiation can efficiently eliminate the viral infectivity.\n",
      "A new coronavirus (SARS-CoV-2) emerged in the winter of 2019 in Wuhan, China, and rapidly spread around the world. The extent and efficiency of SARS-CoV-2 pandemic is far greater than previous coronaviruses that emerged in the 21st Century. Here, we modeled stability of SARS-CoV-2 on skin, paper currency, and clothing to determine if these surfaces may factor in the fomite transmission dynamics of SARS-CoV-2. Skin, currency, and clothing samples were exposed to SARS-CoV-2 under laboratory conditions and incubated at three different temperatures (4C, 22C, and 37C). Stability was evaluated at 0 hours (h), 4 h, 8 h, 24 h, 72 h, 96 h, 7 days, and 14 days post-exposure. SARS-CoV-2 was shown to be stable on skin through the duration of the experiment at 4C (14 days). Virus remained stable on skin for at least 96 h at 22C and for at least 8h at 37C. There were minimal differences between the tested currency samples. The virus remained stable on the $1 U.S.A. Bank Note for at least 96 h at 4C while viable virus was not detected on the $20 U.S.A. Bank Note samples beyond 72 h. The virus remained stable on both Bank Notes for at least 8 h at 22C and 4 h at 37C. Clothing samples were similar in stability to the currency with the virus being detected for at least 96 h at 4C and at least 4 h at 22C. No viable virus was detected on clothing samples at 37C after initial exposure. This study confirms the inverse relationship between virus stability and temperature. Furthermore, virus stability on skin demonstrates the need for continued hand hygiene practices to minimize fomite transmission both in the general population as well as workplaces where close contact is common.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: vjg2auh7\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "How much impact do masks have on preventing the spread of the COVID-19?\n",
      "what are the transmission routes of coronavirus?\n",
      "what are the best masks for preventing infection by Covid-19?\n",
      "are there any clinical trials available for the coronavirus\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "There are few published data on the protection of masks or respirators against coronavirus infections. This is an important research question to inform the response to the COVID-19 epidemic. The transmission modes of human coronaviruses are similar, thought to be by droplet, contact and sometimes airborne routes. There are several randomised clinical trials of masks and respirators, but most used clinical endpoints or tested only for influenza. In four trials which we conducted, we tested for human coronaviruses, but only composite viral endpoints were reported in the trials. We reviewed and analysed the coronavirus data from four of our trials. Laboratory-confirmed coronavirus infections were identified in our community household trial (1 case), health worker trials (8 cases) and trial of mask use by sick patients (19 cases). No coronavirus infections were transmitted in households to parents who wore P2 or surgical masks, but one child with coronavirus infection transmitted infection to a parent in the control arm. No transmissions to close contacts occurred when worn by sick patients with coronavirus infections. There was a higher risk of coronavirus infection in HCWs who wore a mask compared to a respirator, but the difference was not statistically significant. These are the only available data on coronavirus infections associated with mask or respirator use. More clinical trials are needed to assess the efficacy of respiratory protection against coronavirus infections.\n",
      "There are few published data on the protection of masks or respirators against coronavirus infections. This is an important research question to inform the response to the COVID-19 epidemic. The transmission modes of human coronaviruses are similar, thought to be by droplet, contact and sometimes airborne routes. There are several randomised clinical trials of masks and respirators, but most used clinical endpoints or tested only for influenza. In four trials which we conducted, we tested for human coronaviruses, but only composite viral endpoints were reported in the trials. We reviewed and analysed the coronavirus data from four of our trials. Laboratory-confirmed coronavirus infections were identified in our community household trial (1 case), health worker trials (8 cases) and trial of mask use by sick patients (19 cases). No coronavirus infections were transmitted in households to parents who wore P2 or surgical masks, but one child with coronavirus infection transmitted infection to a parent in the control arm. No transmissions to close contacts occurred when worn by sick patients with coronavirus infections. There was a higher risk of coronavirus infection in HCWs who wore a mask compared to a respirator, but the difference was not statistically significant. These are the only available data on coronavirus infections associated with mask or respirator use. More clinical trials are needed to assess the efficacy of respiratory protection against coronavirus infections.\n",
      "BACKGROUND: The pandemic of COVID-19 is growing, and a shortage of masks and respirators has been reported globally. Policies of health organizations for healthcare workers are inconsistent, with a change in policy in the US for universal face mask use. The aim of this study was to review the evidence around the efficacy of masks and respirators for healthcare workers, sick patients and the general public. METHODS: A systematic review of randomized controlled clinical trials on use of respiratory protection by healthcare workers, sick patients and community members was conducted. Articles were searched on Medline and Embase using key search terms. RESULTS: A total of 19 randomised controlled trials were included in this study - 8 in community settings, 6 in healthcare settings and 5 as source control. Most of these randomised controlled trials used different interventions and outcome measures. In the community, masks appeared to be effective with and without hand hygiene, and both together are more protective. Randomised controlled trials in health care workers showed that respirators, if worn continually during a shift, were effective but not if worn intermittently. Medical masks were not effective, and cloth masks even less effective. When used by sick patients randomised controlled trials suggested protection of well contacts. CONCLUSION: The study suggests that community mask use by well people could be beneficial, particularly for COVID-19, where transmission may be pre-symptomatic. The studies of masks as source control also suggest a benefit, and may be important during the COVID-19 pandemic in universal community face mask use as well as in health care settings. Trials in healthcare workers support the use of respirators continuously during a shift. This may prevent health worker infections and deaths from COVID-19, as aerosolisation in the hospital setting has been documented.\n",
      "ABSTRACT BACKGROUND Objectives: \"In patient-facing healthcare workers delivering secondary care, what is the evidence behind UK Government PPE Guidance on surgical masks versus respirators for SARS-CoV-2 protection?\" METHODS Two independent reviewers searched MEDLINE, Google Scholar and grey literature 11th-30th April 2020. Studies published on any date containing primary data comparing surgical facemasks and respirators specific to SARS-CoV-2, and studies underpinning government PPE guidance, were included. Appraisal was performed using CASP checklists. Results were synthesised by comparison of findings and appraisals. RESULTS In all three laboratory studies of 14 different respirators and 12 surgical facemasks, respirators were significantly more effective than facemasks in protection factors, reduction factors, filter penetrations, and total inspiratory leakages at differing particle sizes, mean inspiratory flows, and breathing rates. Tests included live viruses and inert particles on dummies and humans. In six clinical studies, 6,502 participants, there was no consistent definition of \"exposure\" to determine the efficacy of RPE. It is difficult to define \"safe\". The only statistically significant result found continuous use of respirators more effective in clinical respiratory illness compared to targeted use or surgical facemask. CONCLUSIONS There is a paucity of evidence on the comparison of FRSMs and respirators specific to SARS-CoV-2, and poor-quality evidence in other contexts. Indirectness results in extrapolation of non-SARS-CoV-2 specific data to guide UK Government PPE guidance. The appropriateness of this is unknown given the uncertainty over the transmission of SARS-CoV-2. 1. The evidence base for UK Government PPE guidelines is not based on SARS-CoV-2 and requires generalisation from low-quality evidence of other pathogens/particles. 2. There is a paucity of high-quality evidence regarding the efficacy of RPE specific to SARS-CoV-2. 3. HMG's PPE guidelines are underpinned by the assumption of droplet transmission of SARS-CoV-2. Triaging the use of FFP3 respirators might increase the risk of COVID-19 faced by some. FUNDING This review was unfunded and unsponsored.\n",
      "OBJECTIVES: To determine the risk of SARS-CoV-2 transmission by aerosols, to provide evidence on the rational use of masks, and to discuss additional measures important for the protection of healthcare workers from COVID-19. METHODS: Literature review and expert opinion. SHORT CONCLUSION: SARS-CoV-2, the pathogen causing COVID-19, is considered to be transmitted via droplets rather than aerosols, but droplets with strong directional airflow support may spread further than 2 m. High rates of COVID-19 infections in healthcare-workers (HCWs) have been reported from several countries. Respirators such as filtering face piece (FFP) 2 masks were designed to protect HCWs, while surgical masks were originally intended to protect patients (e.g., during surgery). Nevertheless, high quality standard surgical masks (type II/IIR according to European Norm EN 14683) appear to be as effective as FFP2 masks in preventing droplet-associated viral infections of HCWs as reported from influenza or SARS. So far, no head-to-head trials with these masks have been published for COVID-19. Neither mask type completely prevents transmission, which may be due to inappropriate handling and alternative transmission pathways. Therefore, compliance with a bundle of infection control measures including thorough hand hygiene is key. During high-risk procedures, both droplets and aerosols may be produced, reason why respirators are indicated for these interventions.\n",
      "BACKGROUND: Respiratory protective devices are critical in protecting against infection in healthcare workers at high risk of novel 2019 coronavirus disease (COVID-19); however, recommendations are conflicting and epidemiological data on their relative effectiveness against COVID-19 are limited. PURPOSE: To compare medical masks to N95 respirators in preventing laboratory-confirmed viral infection and respiratory illness including coronavirus specifically in healthcare workers. DATA SOURCES: MEDLINE, Embase, and CENTRAL from January 1, 2014, to March 9, 2020. Update of published search conducted from January 1, 1990, to December 9, 2014. STUDY SELECTION: Randomized controlled trials (RCTs) comparing the protective effect of medical masks to N95 respirators in healthcare workers. DATA EXTRACTION: Reviewer pair independently screened, extracted data, and assessed risk of bias and the certainty of the evidence. DATA SYNTHESIS: Four RCTs were meta-analyzed adjusting for clustering. Compared with N95 respirators; the use of medical masks did not increase laboratory-confirmed viral (including coronaviruses) respiratory infection (OR 1.06; 95% CI 0.90-1.25; I2 = 0%; low certainty in the evidence) or clinical respiratory illness (OR 1.49; 95% CI: 0.98-2.28; I2 = 78%; very low certainty in the evidence). Only one trial evaluated coronaviruses separately and found no difference between the two groups (P = .49). LIMITATIONS: Indirectness and imprecision of available evidence. CONCLUSIONS: Low certainty evidence suggests that medical masks and N95 respirators offer similar protection against viral respiratory infection including coronavirus in healthcare workers during non-aerosol-generating care. Preservation of N95 respirators for high-risk, aerosol-generating procedures in this pandemic should be considered when in short supply.\n",
      "ABSTRACT Background The pandemic of COVID-19 is growing, and a shortage of masks and respirators has been reported globally. Policies of health organizations for healthcare workers are inconsistent, with a change in policy in the US for universal face mask use. The aim of this study was to review the evidence around the efficacy of masks and respirators for healthcare workers, sick patients and the general public. Methods A systematic review of randomized controlled clinical trials on use of respiratory protection by healthcare workers, sick patients and community members was conducted. Articles were searched on Medline and Embase using key search terms. Results A total of 19 randomised controlled trials were included in this study – 8 in community settings, 6 in healthcare settings and 5 as source control. Most of these randomised controlled trials used different interventions and outcome measures. In the community, masks appeared to be more effective than hand hygiene alone, and both together are more protective. Randomised controlled trials in health care workers showed that respirators, if worn continually during a shift, were effective but not if worn intermittently. Medical masks were not effective, and cloth masks even less effective. When used by sick patients randomised controlled trials suggested protection of well contacts. Conclusion The study suggests that community mask use by well people could be beneficial, particularly for COVID-19, where transmission may be pre-symptomatic. The studies of masks as source control also suggest a benefit, and may be important during the COVID-19 pandemic in universal community face mask use as well as in health care settings. Trials in healthcare workers support the use of respirators continuously during a shift. This may prevent health worker infections and deaths from COVID-19, as aerosolisation in the hospital setting has been documented.\n",
      "Respiratory infections may spread through droplets, airborne particles, and aerosols from infected individuals through coughing, sneezing, and speaking. In the case of Coronavirus Disease 2019 (COVID-19), droplet spread can occur from symptomatic as well as pre-symptomatic and asymptomatic persons. The U.S. Centers for Disease Control and Prevention (CDC) has therefore recently recommended home-made cloth face coverings for use by the general public in areas of significant community-based transmission. Because medical masks and N95 respirators are in short supply, these are to be reserved for healthcare workers. There is, however, little information on the effectiveness of home-made face coverings in reducing droplet dissemination. Here, we ascertained the performance of ten different fabrics, ranging from cotton to silk, in blocking high velocity droplets, using a 3-layered commercial medical mask as a benchmark material. We also assessed their breathability and ability to soak water. We reason that the materials should be as breathable as possible, without compromising blocking efficiency, to reduce air flow through the sides of the mask since such flow would defeat the purpose of the mask. We found that most home fabrics substantially block droplets, even as a single layer. With two layers, blocking performance can reach that of surgical mask without significantly compromising breathability. Furthermore, we observed that home fabrics are hydrophilic to varying degrees, and hence soak water. In contrast, medical masks are hydrophobic, and tend to repel water. Incoming droplets are thus soaked and 'held back' by home fabrics, which might offer an as of yet untapped and understudied advantage of home-made cloth masks. Overall, our study suggests that most double-layered cloth face coverings may help reduce droplet transmission of respiratory infections.\n",
      "Background: There is paucity of evidence on the effectiveness of facemask use in COVID-19 in community settings. Objectives: We aimed to estimate the effectiveness of facemask use alone or along with hand hygiene in community settings in reducing the transmission of viral respiratory illness. Methods: We searched PubMed and Embase for randomized controlled trials on facemask use in community settings to prevent viral respiratory illnesses published up to April 25, 2020. Two independent reviewers were involved in synthesis of data. Data extraction and risk-of-bias assessment were done in a standard format from the selected studies. Outcome data for clinically diagnosed or self-reported influenza-like illness (ILI) was recorded from individual studies. Pooled effect size was estimated by random-effects model for \"facemask only versus control\" and \"facemask plus hand hygiene versus control.\" Results: Of the 465 studies from PubMed and 437 studies from Embase identified from our search, 9 studies were included in qualitative synthesis and 8 studies in quantitative synthesis. Risk of bias was assessed as low (n = 4), medium (n = 3), or high (n = 1) risk. Interventions included using a triple-layered mask alone or in combination with hand hygiene. Publication bias was not significant. There was no significant reduction in ILI either with facemask alone (n = 5, pooled effect size: -0.17; 95% confidence interval [CI]: -0.43-0.10; P = 0.23; I2 = 10.9%) or facemask with handwash (n = 6, pooled effect size: (n=6, pooled effect size: -0.09; 95% CI: -0.58 to 0.40; P = 0.71, I2 = 69.4%). Conclusion: : Existing data pooled from randomized controlled trials do not reveal a reduction in occurrence of ILI with the use of facemask alone in community settings.\n",
      "BACKGROUND: Respiratory protective devices are critical in protecting against infection in healthcare workers at high risk of novel 2019 coronavirus disease (COVID‐19); however, recommendations are conflicting and epidemiological data on their relative effectiveness against COVID‐19 are limited. PURPOSE: To compare medical masks to N95 respirators in preventing laboratory‐confirmed viral infection and respiratory illness including coronavirus specifically in healthcare workers. DATA SOURCES: MEDLINE, Embase, and CENTRAL from January 1, 2014, to March 9, 2020. Update of published search conducted from January 1, 1990, to December 9, 2014. STUDY SELECTION: Randomized controlled trials (RCTs) comparing the protective effect of medical masks to N95 respirators in healthcare workers. DATA EXTRACTION: Reviewer pair independently screened, extracted data, and assessed risk of bias and the certainty of the evidence. DATA SYNTHESIS: Four RCTs were meta‐analyzed adjusting for clustering. Compared with N95 respirators; the use of medical masks did not increase laboratory‐confirmed viral (including coronaviruses) respiratory infection (OR 1.06; 95% CI 0.90‐1.25; I (2) = 0%; low certainty in the evidence) or clinical respiratory illness (OR 1.49; 95% CI: 0.98‐2.28; I (2) = 78%; very low certainty in the evidence). Only one trial evaluated coronaviruses separately and found no difference between the two groups (P = .49). LIMITATIONS: Indirectness and imprecision of available evidence. CONCLUSIONS: Low certainty evidence suggests that medical masks and N95 respirators offer similar protection against viral respiratory infection including coronavirus in healthcare workers during non–aerosol‐generating care. Preservation of N95 respirators for high‐risk, aerosol‐generating procedures in this pandemic should be considered when in short supply.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: zb434ve3\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Abstract The infection of a novel coronavirus found in Wuhan of China (SARS-CoV-2) is rapidly spreading, and the incidence rate is increasing worldwide. Due to the lack of effective treatment options for SARS-CoV-2, various strategies are being tested in China, including drug repurposing. In this study, we used our pre-trained deep learning-based drug-target interaction model called Molecule Transformer-Drug Target Interaction (MT-DTI) to identify commercially available drugs that could act on viral proteins of SARS-CoV-2. The result showed that atazanavir, an antiretroviral medication used to treat and prevent the human immunodeficiency virus (HIV), is the best chemical compound, showing an inhibitory potency with Kd of 94.94 nM against the SARS-CoV-2 3C-like proteinase, followed by remdesivir (113.13 nM), efavirenz (199.17 nM), ritonavir (204.05 nM), and dolutegravir (336.91 nM). Interestingly, lopinavir, ritonavir, and darunavir are all designed to target viral proteinases. However, in our prediction, they may also bind to the replication complex components of SARS-CoV-2 with an inhibitory potency with Kd < 1,000 nM. In addition, we also found that several antiviral agents, such as Kaletra (lopinavir/ritonavir), could be used for the treatment of SARS-CoV-2. Overall, we suggest that the list of antiviral drugs identified by the MT-DTI model should be considered, when establishing effective treatment strategies for SARS-CoV-2.\n",
      "The infection of a novel coronavirus found in Wuhan of China (2019-nCoV) is rapidly spreading, and the incidence rate is increasing worldwide. Due to the lack of effective treatment options for 2019-nCoV, various strategies are being tested in China, including drug repurposing. In this study, we used our pretrained deep learning-based drug-target interaction model called Molecule Transformer-Drug Target Interaction (MT-DTI) to identify commercially available drugs that could act on viral proteins of 2019-nCoV. The result showed that atazanavir, an antiretroviral medication used to treat and prevent the human immunodeficiency virus (HIV), is the best chemical compound, showing a inhibitory potency with Kd of 94.94 nM against the 2019-nCoV 3C-like proteinase, followed by efavirenz (199.17 nM), ritonavir (204.05 nM), and dolutegravir (336.91 nM). Interestingly, lopinavir, ritonavir, and darunavir are all designed to target viral proteinases. However, in our prediction, they may also bind to the replication complex components of 2019-nCoV with an inhibitory potency with Kd < 1000 nM. In addition, we also found that several antiviral agents, such as Kaletra, could be used for the treatment of 2019-nCoV, although there is no real-world evidence supporting the prediction. Overall, we suggest that the list of antiviral drugs identified by the MT-DTI model should be considered, when establishing effective treatment strategies for 2019-nCoV.\n",
      "BACKGROUND: The rapidly enlarging COVID-19 pandemic caused by the novel SARS-corona virus-2 is a global public health emergency of an unprecedented level. Unfortunately no treatment therapy or vaccine is yet available to counter the SARS-CoV-2 infection, which substantiates the need to expand research efforts in this direction. The indispensable function of the main protease in virus replication makes this enzyme a promising target for inhibitors screening and drug discovery to treat novel coronavirus infection. The recently concluded α-ketoamide ligand-bound X-ray crystal structure of SARS-CoV-2 M(pro) (PDB ID: 6Y2F) from Zhang et al. has revealed the potential inhibitor binding mechanism and the molecular determinants responsible for substrate binding. METHODS: For the study, we have targeted the SARS-CoV-2 M(pro) for the screening of FDA approved antiviral drugs and carried out molecular docking based virtual screening. Further molecular dynamic simulation studies of the top three selected drugs carried out to investigated for their binding affinity and stability in the SARS-CoV-2 M(pro) active site. The phylogenetic analysis was also performed to know the relatedness between the SARS-CoV-2 genomes isolated from different countries. RESULTS: The phylogenetic analysis of the SARS-CoV-2 genome reveals that the virus is closely related to the Bat-SL-CoV and does not exhibit any divergence at the genomic level. Molecular docking studies revealed that among the 77 drugs, screened top ten drugs shows good binding affinities, whereas the top three drugs: Lopinavir-Ritonavir, Tipranavir, and Raltegravir were undergone for molecular dynamics simulation studies for their conformational stability in the active site of the SARS-CoV-2 M(pro) protein. CONCLUSIONS: In the present study among the library of FDA approved antiviral drugs, the top three inhibitors Lopinavir-Ritonavir, Tipranavir, and Raltegravir show the best molecular interaction with the main protease of SARS-CoV-2. However, the in-vitro efficacy of the drug molecules screened in this study further needs to be corroborated by carrying out a biochemical and structural investigation.\n",
      "[Image: see text] Since the emergence of a novel coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) was first reported from Wuhan, China, neither a specific vaccine nor an antiviral drug against SARS-CoV-2 has become available. However, a combination of two HIV-1 protease inhibitors, lopinavir and ritonavir, has been found to be effective against SARS-CoV, and both drugs could bind well to the SARS-CoV 3C-like protease (SARS-CoV 3CL(pro)). In this work, molecular complexation between each inhibitor and SARS-CoV-2 3CL(pro) was studied using all-atom molecular dynamics simulations, free energy calculations, and pair interaction energy analyses based on MM/PB(GB)SA and FMO-MP2/PCM/6-31G* methods. Both anti-HIV drugs interacted well with the residues at the active site of SARS-CoV-2 3CL(pro). Ritonavir showed a somewhat higher number atomic contacts, a somewhat higher binding efficiency, and a somewhat higher number of key binding residues compared to lopinavir, which correspond with the slightly lower water accessibility at the 3CL(pro) active site. In addition, only ritonavir could interact with the oxyanion hole residues N142 and G143 via the formation of two hydrogen bonds. The interactions in terms of electrostatics, dispersion, and charge transfer played an important role in the drug binding. The obtained results demonstrated how repurposed anti-HIV drugs could be used to combat COVID-19.\n",
      "Background The outbreak of the novel coronavirus disease COVID 19, caused by the SARS-CoV-2 virus has spread rapidly around the globe during the past 3 months. As the virus infected cases and mortality rate of this disease is increasing exponentially, scientists and researchers all over the world are relentlessly working to understand this new virus along with possible treatment regimens by discovering active therapeutic agents and vaccines. So, there is an urgent requirement of new and effective medications that can treat the disease caused by SARS CoV 2. Methods and findings We perform the study of drugs that are already available in the market and being used for other diseases to accelerate clinical recovery, in other words repurposing of existing drugs. The vast complexity in drug design and protocols regarding clinical trials often prohibit developing various new drug combinations for this epidemic disease in a limited time. Recently, remarkable improvements in computational power coupled with advancements in Machine Learning (ML) technology have been utilized to revolutionize the drug development process. Consequently, a detailed study using ML for the repurposing of therapeutic agents is urgently required. Here, we report the ML model based on the Naive Bayes algorithm, which has an accuracy of around 73% to predict the drugs that could be used for the treatment of COVID-19. Our study predicts around ten FDA approved commercial drugs that can be used for repurposing. Among all, we suggest that the antiretroviral drug Atazanavir (DrugBank ID DB01072) would probably be one of the most effective drugs based on the selected criterions. Conclusions Our study can help clinical scientists in being more selective in identifying and testing the therapeutic agents for COVID 19 treatment. The ML based approach for drug discovery as reported here can be a futuristic smart drug designing strategy for community applications.\n",
      "BACKGROUND: The rapidly enlarging COVID-19 pandemic caused by the novel SARS-corona virus-2 is a global public health emergency of an unprecedented level. Unfortunately no treatment therapy or vaccine is yet available to counter the SARS-CoV-2 infection, which substantiates the need to expand research efforts in this direction. The indispensable function of the main protease in virus replication makes this enzyme a promising target for inhibitors screening and drug discovery to treat novel coronavirus infection. The recently concluded &#945;-ketoamide ligand-bound X-ray crystal structure of SARS-CoV-2 Mpro (PDB ID: 6Y2F) from Zhang et al. has revealed the potential inhibitor binding mechanism and the molecular determinants responsible for substrate binding. METHODS: For the study, we have targeted the SARS-CoV-2 Mpro for the screening of FDA approved antiviral drugs and carried out molecular docking based virtual screening. Further molecular dynamic simulation studies of the top three selected drugs carried out to investigated for their binding affinity and stability in the SARS-CoV-2 Mpro active site. The phylogenetic analysis was also performed to know the relatedness between the SARS-CoV-2 genomes isolated from different countries. RESULTS: The phylogenetic analysis of the SARS-CoV-2 genome reveals that the virus is closely related to the Bat-SL-CoV and does not exhibit any divergence at the genomic level. Molecular docking studies revealed that among the 77 drugs, screened top ten drugs shows good binding affinities, whereas the top three drugs: Lopinavir-Ritonavir, Tipranavir, and Raltegravir were undergone for molecular dynamics simulation studies for their conformational stability in the active site of the SARS-CoV-2 Mpro protein. CONCLUSIONS: In the present study among the library of FDA approved antiviral drugs, the top three inhibitors Lopinavir-Ritonavir, Tipranavir, and Raltegravir show the best molecular interaction with the main protease of SARS-CoV-2. However, the in-vitro efficacy of the drug molecules screened in this study further needs to be corroborated by carrying out a biochemical and structural investigation.\n",
      "The outbreak of novel coronavirus pneumonia (COVID-19) caused thousands of deaths worldwide, and the number of total infections is still rising. However, the development of effective vaccine for this novel virus would take a few months. Thus it is urgent to identify some potentially effective old drugs that can be used immediately. Fortunately, some compounds that can inhibit coronavirus in vitro have been reported. In this study, the coronavirus-specific dataset was used to fine-tune our pre-trained multi-task deep model. Next we used the re-trained model to select available commercial drugs against targeted proteins of SARS-CoV-2. The results show that abacavir, a powerful nucleoside analog reverse transcriptase inhibitor used to treat HIV, is predicted to have high binding affinity with several proteins of SARS-CoV-2. Almitrine mesylate and roflumilast which are used for respiratory diseases such as chronic obstructive pulmonary disease are also predicted to have inhibitory effect. Overall, ten drugs are listed as potential inhibitors and the important sites for these binding by our model are exhibited. We hope these results would be useful in the fight against SARS-CoV-2.\n",
      "Since the emergence of a novel coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) was first reported from Wuhan, China, neither a specific vaccine nor an antiviral drug against SARS-CoV-2 has become available. However, a combination of two HIV-1 protease inhibitors, lopinavir and ritonavir, has been found to be effective against SARS-CoV, and both drugs could bind well to the SARS-CoV 3C-like protease (SARS-CoV 3CLpro). In this work, molecular complexation between each inhibitor and SARS-CoV-2 3CLpro was studied using all-atom molecular dynamics simulations, free energy calculations, and pair interaction energy analyses based on MM/PB(GB)SA and FMO-MP2/PCM/6-31G* methods. Both anti-HIV drugs interacted well with the residues at the active site of SARS-CoV-2 3CLpro. Ritonavir showed a somewhat higher number atomic contacts, a somewhat higher binding efficiency, and a somewhat higher number of key binding residues compared to lopinavir, which correspond with the slightly lower water accessibility at the 3CLpro active site. In addition, only ritonavir could interact with the oxyanion hole residues N142 and G143 via the formation of two hydrogen bonds. The interactions in terms of electrostatics, dispersion, and charge transfer played an important role in the drug binding. The obtained results demonstrated how repurposed anti-HIV drugs could be used to combat COVID-19.\n",
      "Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is already responsible for far more deaths than previous pathogenic coronaviruses (CoVs) from 2002 and 2012. The identification of clinically approved drugs to be repurposed to combat 2019 CoV disease (COVID-19) would allow the rapid implementation of potentially life-saving procedures. The major protease (Mpro) of SARS-CoV-2 is considered a promising target, based on previous results from related CoVs with lopinavir (LPV), an HIV protease inhibitor. However, limited evidence exists for other clinically approved antiretroviral protease inhibitors, such as atazanavir (ATV). ATV is of high interest because of its bioavailability within the respiratory tract. Our results show that ATV could dock in the active site of SARS-CoV-2 Mpro, with greater strength than LPV. ATV blocked Mpro activity. We confirmed that ATV inhibits SARS-CoV-2 replication, alone or in combination with ritonavir (RTV) in Vero cells, human pulmonary epithelial cell line and primary monocytes, impairing virus-induced enhancement of IL-6 and TNF-α levels. Together, our data strongly suggest that ATV and ATV/RTV should be considered among the candidate repurposed drugs undergoing clinical trials in the fight against COVID-19.\n",
      "BACKGROUND: The recent global pandemic has placed a high priority on identifying drugs to prevent or lessen clinical infection of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), caused by Coronavirus disease-2019 (COVID-19). METHODS: We applied two computational approaches to identify potential therapeutics. First, we sought to identify existing FDA approved drugs that could block coronaviruses from entering cells by binding to ACE2 or TMPRSS2 using a high-throughput AI-based binding affinity prediction platform. Second, we sought to identify FDA approved drugs that could attenuate the gene expression patterns induced by coronaviruses, using our Disease Cancelling Technology (DCT) platform. RESULTS: Top results for ACE2 binding iincluded several ACE inhibitors, a beta-lactam antibiotic, two antiviral agents (Fosamprenavir and Emricasan) and glutathione. The platform also assessed specificity for ACE2 over ACE1, important for avoiding counterregulatory effects. Further studies are needed to weigh the benefit of blocking virus entry against potential counterregulatory effects and possible protective effects of ACE2. However, the data herein suggest readily available drugs that warrant experimental evaluation to assess potential benefit. DCT was run on an animal model of SARS-CoV, and ranked compounds by their ability to induce gene expression signals that counteract disease-associated signals. Top hits included Vitamin E, ruxolitinib, and glutamine. Glutathione and its precursor glutamine were highly ranked by two independent methods, suggesting both warrant further investigation for potential benefit against SARS-CoV-2. CONCLUSIONS: While these findings are not yet ready for clinical translation, this report highlights the potential use of two bioinformatics technologies to rapidly discover existing therapeutic agents that warrant further investigation for established and emerging disease processes.\n",
      "\n",
      "\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "for res in ret_suc:\n",
    "    try:\n",
    "        gt_docs = get_matching_queries(res[\"_id\"])\n",
    "    except:\n",
    "        continue\n",
    "    print(\"ID:\", res[\"_id\"])\n",
    "    print(\"\\nGT queries\\n-----------\")\n",
    "    for doc in gt_docs:\n",
    "        print(doc)\n",
    "    print(\"\\nRetrieved queries\\n-----------\")\n",
    "    for ret in res[\"retrieval\"]:\n",
    "        print(ret[\"document\"])\n",
    "    print(\"\\n\\n\\n\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "id": "e7421ed3-fd86-4062-821c-0869c643cfeb",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ID: sqrn6kjy\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the initial symptoms of Covid-19?\n",
      "How does the coronavirus differ from seasonal flu?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what is the origin of COVID-19\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "what causes death from Covid-19?\n",
      "has social distancing had an impact on slowing the spread of COVID-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: qvz63m93\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "what is the origin of COVID-19\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "what causes death from Covid-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: iudq5jdu\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the mortality rates overall and in specific populations\n",
      "what is known about those infected with Covid-19 but are asymptomatic?\n",
      "what are the initial symptoms of Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what is the origin of COVID-19\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "what causes death from Covid-19?\n",
      "what are the initial symptoms of Covid-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 703febui\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 023h20vk\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What vaccine candidates are being tested for Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: g4qak0bu\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "are there serological tests that detect antibodies to coronavirus?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: rirzes0m\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "Does Vitamin D impact COVID-19 prevention and treatment?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "are there any clinical trials available for the coronavirus\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "Does Vitamin D impact COVID-19 prevention and treatment?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: ywaefpe8\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "are there any clinical trials available for the coronavirus\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "are there any clinical trials available for the coronavirus\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "what are the guidelines for triaging patients infected with coronavirus?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 2qugltxu\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "what are the initial symptoms of Covid-19?\n",
      "What vaccine candidates are being tested for Covid-19?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: xxkagj6e\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "is remdesivir an effective treatment for COVID-19\n",
      "What vaccine candidates are being tested for Covid-19?\n",
      "Does Vitamin D impact COVID-19 prevention and treatment?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: nz02frdm\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what types of rapid testing for Covid-19 have been developed?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what are the guidelines for triaging patients infected with coronavirus?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what are best practices in hospitals and at home in maintaining quarantine?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "what is known about those infected with Covid-19 but are asymptomatic?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "how has lack of testing availability led to underreporting of true incidence of Covid-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: j59tm40d\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the transmission routes of coronavirus?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "are there serological tests that detect antibodies to coronavirus?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what evidence is there related to COVID-19 super spreaders\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what is the origin of COVID-19\n",
      "how has lack of testing availability led to underreporting of true incidence of Covid-19?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: rjqanuk9\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the transmission routes of coronavirus?\n",
      "how long can the coronavirus live outside the body\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what evidence is there related to COVID-19 super spreaders\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what are the transmission routes of coronavirus?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "has social distancing had an impact on slowing the spread of COVID-19?\n",
      "what is known about those infected with Covid-19 but are asymptomatic?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "how long does coronavirus remain stable  on surfaces?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: l54hk7go\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what causes death from Covid-19?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: rzr8qjw8\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "how long can the coronavirus live outside the body\n",
      "what are best practices in hospitals and at home in maintaining quarantine?\n",
      "what are the initial symptoms of Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what are the transmission routes of coronavirus?\n",
      "How does the coronavirus differ from seasonal flu?\n",
      "what are the initial symptoms of Covid-19?\n",
      "how long can the coronavirus live outside the body\n",
      "how long does coronavirus remain stable  on surfaces?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what is known about those infected with Covid-19 but are asymptomatic?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what are the guidelines for triaging patients infected with coronavirus?\n",
      "are there serological tests that detect antibodies to coronavirus?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: m2c5bvuj\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the best masks for preventing infection by Covid-19?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what is the origin of COVID-19\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 4cbjbo7t\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what types of rapid testing for Covid-19 have been developed?\n",
      "are there serological tests that detect antibodies to coronavirus?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "are there serological tests that detect antibodies to coronavirus?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: ncl2p7kg\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "are there any clinical trials available for the coronavirus\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: dtwstwbe\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 3h1o0oz3\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "what is the origin of COVID-19\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: dqour5jr\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 69toerzi\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "What vaccine candidates are being tested for Covid-19?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "are there any clinical trials available for the coronavirus\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: lnys6iuu\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "what is the origin of COVID-19\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what is the origin of COVID-19\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: jujrazsr\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "what are the transmission routes of coronavirus?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "what is the origin of COVID-19\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 3aqh6zxf\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what causes death from Covid-19?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what causes death from Covid-19?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: cn3bpmwj\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "Does Vitamin D impact COVID-19 prevention and treatment?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: wrrybwda\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "what is known about those infected with Covid-19 but are asymptomatic?\n",
      "What are the longer-term complications of those who recover from COVID-19?\n",
      "What are the impacts of COVID-19 among African-Americans that differ from the rest of the U.S. population?\n",
      "what are the health outcomes for children who contract COVID-19?\n",
      "what are the benefits and risks of re-opening schools in the midst of the COVID-19 pandemic?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 9xmeyrtu\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "What are the longer-term complications of those who recover from COVID-19?\n",
      "what is known about those infected with Covid-19 but are asymptomatic?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "what kinds of complications related to COVID-19 are associated with hypertension?\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: bsgqsdbx\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what are the initial symptoms of Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "how has lack of testing availability led to underreporting of true incidence of Covid-19?\n",
      "what are the initial symptoms of Covid-19?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: y29kct7v\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "are there any clinical trials available for the coronavirus\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "are there any clinical trials available for the coronavirus\n",
      "what is the origin of COVID-19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 48ay8yl3\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: sqrn6kjy\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the initial symptoms of Covid-19?\n",
      "How does the coronavirus differ from seasonal flu?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Abstract Coronavirus disease 2019 (COVID-19) is caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and represents a potentially fatal disease of great global public health importance. As of March 26, 2020, the outbreak of COVID-19 has resulted in 462,801 confirmed cases and 20,839 deaths globally, which is more than those caused by SARS and Middle East respiratory syndrome (MERS) in 2003 and 2013, respectively. The epidemic has posed considerable challenges worldwide. Under a strict mechanism of massive prevention and control, China has seen a rapid decrease in new cases of coronavirus; however, the global situation remains serious. Additionally, the origin of COVID-19 has not been determined and no specific antiviral treatment or vaccine is currently available. Based on the published data, this review systematically discusses the etiology, epidemiology, clinical characteristics, and current intervention measures related to COVID-19 in the hope that it may provide a reference for future studies and aid in the prevention and control of the COVID-19 epidemic.\n",
      "Coronavirus disease 2019 (COVID-19) is caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and represents a potentially fatal disease of great global public health importance. As of March 26, 2020, the outbreak of COVID-19 has resulted in 462,801 confirmed cases and 20,839 deaths globally, which is more than those caused by SARS and Middle East respiratory syndrome (MERS) in 2003 and 2013, respectively. The epidemic has posed considerable challenges worldwide. Under a strict mechanism of massive prevention and control, China has seen a rapid decrease in new cases of coronavirus; however, the global situation remains serious. Additionally, the origin of COVID-19 has not been determined and no specific antiviral treatment or vaccine is currently available. Based on the published data, this review systematically discusses the etiology, epidemiology, clinical characteristics, and current intervention measures related to COVID-19 in the hope that it may provide a reference for future studies and aid in the prevention and control of the COVID-19 epidemic.\n",
      "INTRODUCTION: The Coronavirus disease-19 (COVID-19) caused by the novel beta coronavirus named severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) started in late December 2019 in Wuhan, China. Within a short span, COVID-19 was declared a global public health emergency affecting 214 countries with 5,939,234 confirmed cases and 3,67,255 deaths as of 30 May 2020. With limited knowledge about SARS-CoV-2, no approved treatment or vaccine is available till date. AREAS COVERED: We performed a review of literature on PubMed on the SARS-CoV-2 virus and COVID-19 illness including trials of preventive and therapeutic measures. This review presents the basic biology of coronaviruses, epidemiology of COVID-19, clinical presentations, investigational therapies and vaccines, infection prevention and control measures and the lessons from the present pandemic. EXPERT OPINION: The scale of the outbreak has brought the governments, health-care professionals, and scientists around the world under tremendous pressure to devise control strategies and develop novel prevention measures. While availability of vaccine for COVID-19 may take time, the disease may be contained through hand hygiene, physical distancing, travel restriction, and aggressive steps such as 'lockdown.' Clinical trials at different phases are ongoing across different countries to expedite the development of effective drugs and vaccine to overcome the pandemic.\n",
      "Coronavirus Disease 2019 (COVID-19) has become a major health problem causing severe acute respiratory illness in humans. It has spread rapidly around the globe since its first identification in Wuhan, China, in December 2019. The causative virus is called severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), and the World Health Organization (WHO) named the new epidemic disease Coronavirus Disease (COVID-19). The incidence of COVID-19 continues to increase with more than three million confirmed cases and over 244,000 deaths worldwide. There is currently no specific treatment or vaccine against COVID-19. Therefore, in the absence of pharmaceutical interventions, the implementation of precautions and hygienic measures will be essential to control and to minimize human transmission of the virus. In this review, we highlight the epidemiology, transmission, symptoms, and treatment of this disease, as well as future strategies to manage the spread of this fatal coronavirus.\n",
      "INTRODUCTION: The Coronavirus disease-19 (COVID-19) caused by the novel beta coronavirus named Severe Acute Respiratory Syndromecoronavirus-2 (SARS-CoV-2) started in late December 2019 in Wuhan, China. Within a short span, COVID-19 was declared a global public health emergency affecting 214 countries with 5,939,234 confirmed cases and 3,67,255 deaths as of 30 May, 2020. With limited knowledge about SARS-CoV-2, no approved treatment or vaccine is available till date. AREAS COVERED: We performed a review of literature on PubMed on the SARS-CoV-2 virus and COVID-19 illness including trials of preventive and therapeutic measures. This review presents the basic biology of coronaviruses, epidemiology of COVID-19, clinical presentations, investigational therapies and vaccines, infection prevention and control measures and the lessons from the present pandemic. EXPERT OPINION: The scale of the outbreak has brought the governments, healthcare professionals and scientists around the world under tremendous pressure to devise control strategies and develop novel prevention measures. While availability of vaccine for COVID-19 may take time, the disease may be contained through hand hygiene, physical distancing, travel restriction and aggressive steps such as 'lockdown'. Clinical trials at different phases are ongoing across different countries to expedite development of effective drugs and vaccine to overcome the pandemic.\n",
      "The outbreak of Coronavirus disease 2019 (COVID-19), caused by severe acute respiratory syndrome (SARS) coronavirus 2 (SARS-CoV-2), has thus far killed over 3,000 people and infected over 80,000 in China and elsewhere in the world, resulting in catastrophe for humans. Similar to its homologous virus, SARS-CoV, which caused SARS in thousands of people in 2003, SARS-CoV-2 might also be transmitted from the bats and causes similar symptoms through a similar mechanism. However, COVID-19 has lower severity and mortality than SARS but is much more transmissive and affects more elderly individuals than youth and more men than women. In response to the rapidly increasing number of publications on the emerging disease, this article attempts to provide a timely and comprehensive review of the swiftly developing research subject. We will cover the basics about the epidemiology, etiology, virology, diagnosis, treatment, prognosis, and prevention of the disease. Although many questions still require answers, we hope that this review helps in the understanding and eradication of the threatening disease.\n",
      "There have been three major global outbreaks of acute respiratory disease caused by coronavirus in the last two decades. The ongoing Coronavirus Disease 2019 (COVID-19) first emerged in Wuhan, China, is the most dangerous, which spread to 163 countries and 6 continents and caused a major public health emergency worldwide. The outbreak is caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) originated from bats, which spreads rapidly from human to human. As of 17 March 2020, there have been 179,112 confirmed cases and 7426 deaths worldwide, with a mortality rate of 4.1%. There is currently no effective treatment or approved vaccine, so isolating the source of infection and blocking the routes of transmission is important. In this article, we summarized the worldwide epidemic trend of COVID-19 and discussed its epidemiological characteristics, prevention and control measures. We hope this article could provide experience and help for global epidemic prevention and control.\n",
      "There have been three major global outbreaks of acute respiratory disease caused by coronavirus in the last two decades. The ongoing Coronavirus Disease 2019 (COVID-19) first emerged in Wuhan, China, is the most dangerous, which spread to 163 countries and 6 continents and caused a major public health emergency worldwide. The outbreak is caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) originated from bats, which spreads rapidly from human to human. As of 17 March 2020, there have been 179,112 confirmed cases and 7426 deaths worldwide, with a mortality rate of 4.1%. There is currently no effective treatment or approved vaccine, so isolating the source of infection and blocking the routes of transmission is important. In this article, we summarized the worldwide epidemic trend of COVID-19 and discussed its epidemiological characteristics, prevention and control measures. We hope this article could provide experience and help for global epidemic prevention and control.\n",
      "Abstract In early December 2019, an outbreak of coronavirus disease 2019 (COVID-19), caused by a novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), occurred in Wuhan City, Hubei Province, China. On January 30, 2020 the World Health Organization declared the outbreak as a Public Health Emergency of International Concern. As of February 14, 2020, 49,053 laboratory-confirmed and 1,381 deaths have been reported globally. Perceived risk of acquiring disease has led many governments to institute a variety of control measures. We conducted a literature review of publicly available information to summarize knowledge about the pathogen and the current epidemic. In this literature review, the causative agent, pathogenesis and immune responses, epidemiology, diagnosis, treatment and management of the disease, control and preventions strategies are all reviewed.\n",
      "Coronavirus disease 2019 (COVID-19) is a newly emerged disease that has become a global public health concern as it rapidly spread around the world. The etiologic agent responsible for this disease has been named as severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) by the International Committee on Taxonomy of Viruses as it shows similar genomic features to that of SARS-CoV which caused a pandemic in 2002. This disease first appeared in Hubei province of China and it follows human-to-human transmission but the path this virus took to set up human infection remains a mystery. By 17 April 2020, globally there have been 2,074,529 confirmed cases with 139,378 deaths because of COVID-19. SARS-CoV-2 shows several similarities with SARS?CoV, and Middle East Respiratory Syndrome Coronavirus (MERS-CoV) with its clinical presentations. This can vary from asymptomatic infection to severe disease and mortality. Real-time reverse-transcription polymerase chain reaction (rRT-PCR) screening is considered as the standard laboratory test for the diagnosis of COVID-19. There is no proven antiviral agent against SARS-CoV-2 so the treatment for COVID-19 is symptomatic, aiming for the management of the symptoms and prevention of the complications. The outbreak of COVID-19 has led to the implementation of extraordinary public health measures throughout the world. Numerous antiviral compounds used to treat other infections are being clinically researched to find possible treatment. Similarly, the traditional public health outbreak response strategy of isolation, quarantine, social distancing and community containment has been implemented in multiple countries and has played an important role in the prevention of new outbreaks. This review aims to enhance our understanding of COVID 19. Keywords: Coronavirus disease 2019; COVID-19; SARS-CoV-2; novel coronavirus 2019; severe acute respiratory syndrome-2.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: qvz63m93\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "Does SARS-CoV-2 have any subtypes, and if so what are they?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Background COVID-19 is a disease with global public health emergency that have shook the world since its’ first detection in China in December, 2019. Severe acute respiratory syndrome Coronavirus 2 (SARS-CoV-2) is the pathogen responsible behind this pandemic. The lethality of different viral strains is found to vary in different geographical locations but the molecular mechanism is yet to be known. Methods Available data of whole genome sequencing of different viral strains published by different countries were retrieved and then analysed using Multiple Sequence Alignment and Pair-wise Sequence Alignment leading to Phylogenetic tree construction. Each location and the corresponding genetic variations were screened in depth. Then the variations are analysed at protein level giving special emphasis on Non Synonymous amino acid substitutions. The fatality rates in different countries were matched against the mutation number, rarity of the nucleotide alterations and functional impact of the Non Synonymous changes at protein level, separately and in combination. Findings All the viral strains have been found to evolve from the viral strain of Taiwan (MT192759) which is 100% identical with the ancestor SARS-CoV-2 sequences of Wuhan (NC 045512.2; submitted on 5th Jan, 2020). Transition from C to T (C>T) is the most frequent mutation in this viral genome and mutations A>T, G>A, T>A are the rarest ones, found in countries with maximum fatality rate i.e Italy, Spain and Sweden. 20 Non Synonymous mutations are located in viral genome spanning Orf1ab polyprotein, Surface glycoprotein, Nucleocapsid protein etc. The functional effect on the structure and function of the protein can favourably or unfavourably interact with the host body. Interpretation The fatality outcome depends on three important factors (a) number of mutation (b) rarity of the allelic variation and (c) functional consequence of the mutation at protein level. The molecular divergence, evolved from the ancestral strain (S) lead to extremely lethal (E), lethal(L) and non lethal (N) strains with the involvement of an Intermediate strain(I).\n",
      "OBJECTIVES: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan, China, in December 2019 and has been rapidly spreading worldwide. Although the causal relationship among mutations and the features of SARS-CoV-2 such as rapid transmission, pathogenicity, and tropism, remains unclear, our results of genomic mutations in SARS-CoV-2 may help to interpret the interaction between genomic characterization in SARS-CoV-2 and infectivity with the host. METHODS: A total of 4,254 genomic sequences of SARS-CoV-2 were collected from the Global Initiative on Sharing all Influenza Data (GISAID). Multiple sequence alignment for phylogenetic analysis and comparative genomic approach for mutation analysis were conducted using Molecular Evolutionary Genetics Analysis (MEGA), and an in-house program based on Perl language, respectively. RESULTS: Phylogenetic analysis of SARS-CoV-2 strains indicated that there were 3 major clades including S, V, and G, and 2 subclades (G.1 and G.2). There were 767 types of synonymous and 1,352 types of non-synonymous mutation. ORF1a, ORF1b, S, and N genes were detected at high frequency, whereas ORF7b and E genes exhibited low frequency. In the receptor-binding domain (RBD) of the S gene, 11 non-synonymous mutations were observed in the region adjacent to the angiotensin converting enzyme 2 (ACE2) binding site. CONCLUSION: It has been reported that the rapid infectivity and transmission of SARS-CoV-2 associated with host receptor affinity are derived from several mutations in its genes. Without these genetic mutations to enhance evolutionary adaptation, species recognition, host receptor affinity, and pathogenicity, it would not survive. It is expected that our results could provide an important clue in understanding the genomic characteristics of SARS-CoV-2.\n",
      "BACKGROUND: SARS-CoV-2 is a RNA coronavirus responsible for the pandemic of the Severe Acute Respiratory Syndrome (COVID-19). RNA viruses are characterized by a high mutation rate, up to a million times higher than that of their hosts. Virus mutagenic capability depends upon several factors, including the fidelity of viral enzymes that replicate nucleic acids, as SARS-CoV-2 RNA dependent RNA polymerase (RdRp). Mutation rate drives viral evolution and genome variability, thereby enabling viruses to escape host immunity and to develop drug resistance. METHODS: We analyzed 220 genomic sequences from the GISAID database derived from patients infected by SARS-CoV-2 worldwide from December 2019 to mid-March 2020. SARS-CoV-2 reference genome was obtained from the GenBank database. Genomes alignment was performed using Clustal Omega. Mann-Whitney and Fisher-Exact tests were used to assess statistical significance. RESULTS: We characterized 8 novel recurrent mutations of SARS-CoV-2, located at positions 1397, 2891, 14408, 17746, 17857, 18060, 23403 and 28881. Mutations in 2891, 3036, 14408, 23403 and 28881 positions are predominantly observed in Europe, whereas those located at positions 17746, 17857 and 18060 are exclusively present in North America. We noticed for the first time a silent mutation in RdRp gene in England (UK) on February 9th, 2020 while a different mutation in RdRp changing its amino acid composition emerged on February 20th, 2020 in Italy (Lombardy). Viruses with RdRp mutation have a median of 3 point mutations [range: 2-5], otherwise they have a median of 1 mutation [range: 0-3] (p value < 0.001). CONCLUSIONS: These findings suggest that the virus is evolving and European, North American and Asian strains might coexist, each of them characterized by a different mutation pattern. The contribution of the mutated RdRp to this phenomenon needs to be investigated. To date, several drugs targeting RdRp enzymes are being employed for SARS-CoV-2 infection treatment. Some of them have a predicted binding moiety in a SARS-CoV-2 RdRp hydrophobic cleft, which is adjacent to the 14408 mutation we identified. Consequently, it is important to study and characterize SARS-CoV-2 RdRp mutation in order to assess possible drug-resistance viral phenotypes. It is also important to recognize whether the presence of some mutations might correlate with different SARS-CoV-2 mortality rates.\n",
      "BACKGROUND: SARS-CoV-2 is a RNA coronavirus responsible for the pandemic of the Severe Acute Respiratory Syndrome (COVID-19). RNA viruses are characterized by a high mutation rate, up to a million times higher than that of their hosts. Virus mutagenic capability depends upon several factors, including the fidelity of viral enzymes that replicate nucleic acids, as SARS-CoV-2 RNA dependent RNA polymerase (RdRp). Mutation rate drives viral evolution and genome variability, thereby enabling viruses to escape host immunity and to develop drug resistance. METHODS: We analyzed 220 genomic sequences from the GISAID database derived from patients infected by SARS-CoV-2 worldwide from December 2019 to mid-March 2020. SARS-CoV-2 reference genome was obtained from the GenBank database. Genomes alignment was performed using Clustal Omega. Mann–Whitney and Fisher-Exact tests were used to assess statistical significance. RESULTS: We characterized 8 novel recurrent mutations of SARS-CoV-2, located at positions 1397, 2891, 14408, 17746, 17857, 18060, 23403 and 28881. Mutations in 2891, 3036, 14408, 23403 and 28881 positions are predominantly observed in Europe, whereas those located at positions 17746, 17857 and 18060 are exclusively present in North America. We noticed for the first time a silent mutation in RdRp gene in England (UK) on February 9th, 2020 while a different mutation in RdRp changing its amino acid composition emerged on February 20th, 2020 in Italy (Lombardy). Viruses with RdRp mutation have a median of 3 point mutations [range: 2–5], otherwise they have a median of 1 mutation [range: 0–3] (p value < 0.001). CONCLUSIONS: These findings suggest that the virus is evolving and European, North American and Asian strains might coexist, each of them characterized by a different mutation pattern. The contribution of the mutated RdRp to this phenomenon needs to be investigated. To date, several drugs targeting RdRp enzymes are being employed for SARS-CoV-2 infection treatment. Some of them have a predicted binding moiety in a SARS-CoV-2 RdRp hydrophobic cleft, which is adjacent to the 14408 mutation we identified. Consequently, it is important to study and characterize SARS-CoV-2 RdRp mutation in order to assess possible drug-resistance viral phenotypes. It is also important to recognize whether the presence of some mutations might correlate with different SARS-CoV-2 mortality rates.\n",
      "BACKGROUND: China announced an outbreak of new coronavirus in the city of Wuhan on December 31, 2019; lash to now, the virus transmission has become pandemic worldwide. Severe cases from the Huanan Seafood Wholesale market in Wuhan were confirmed pneumonia with a novel coronavirus (2019-nCoV). Understanding the molecular mechanisms of genome selection and packaging is critical for developing antiviral strategies. Thus, we defined the correlation in 10 severe acute respiratory syndrome coronavirus (SARS-CoV2) sequences from different countries to analyze the genomic patterns of disease origin and evolution aiming for developing new control pandemic processes. METHODS: We apply genomic analysis to observe SARS-CoV2 sequences from GenBank (http://www.ncbi.nim.nih.gov/genebank/): MN 908947 (China, C1), MN985325 (USA: WA, UW), MN996527 (China, C2), MT007544 (Australia: Victoria, A1), MT027064 (USA: CA, UC), MT039890 (South Korea, K1), MT066175 (Taiwan, T1), MT066176 (Taiwan, T2), LC528232 (Japan, J1), and LC528233 (Japan, J2) for genomic sequence alignment analysis. Multiple Sequence Alignment by Clustalw (https://www.genome.jp/tools-bin/clustalw) web service is applied as our alignment tool. RESULTS: We analyzed 10 sequences from the National Center for Biotechnology Information (NCBI) database by genome alignment and found no difference in amino acid sequences within M and N proteins. There are two amino acid variances in the spike (S) protein region. One mutation found from the South Korea sequence is verified. Two possible \"L\" and \"S\" SNPs found in ORF1ab and ORF8 regions are detected. CONCLUSION: We performed genomic analysis and comparative multiple sequences of SARS-CoV2. Studies about the biological symptoms of SARS-CoV2 in clinic animals and humans will manipulate an understanding on the origin of pandemic crisis.\n",
      "BACKGROUND. The COVID-19 pandemic caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is straining health systems around the world. Although the Chinese government implemented a number of severe restrictions on people’s movement in an attempt to contain its local and international spread, the virus had already reached many areas of the world in part due to its potent transmissibility and the fact that a substantial fraction of infected individuals develop little or no symptoms at all. Following its emergence, the virus started to generate sustained transmission in neighboring countries in Asia, Western Europe, Australia, Canada and the United States, and finally in South America and Africa. As the virus continues its global spread, a clear and evidence-based understanding of properties and dynamics of the global transmission network of SARS-CoV-2 is essential to design and put in place efficient and globally coordinated interventions. METHODS. We employ molecular surveillance data of SARS-CoV-2 epidemics for inference and comprehensive analysis of its global transmission network before the pandemic declaration. Our goal was to characterize the spatial-temporal transmission pathways that led to the establishment of the pandemic. We exploited a network-based approach specifically tailored to emerging outbreak settings. Specifically, it traces the accumulation of mutations in viral genomic variants via mutation trees, which are then used to infer transmission networks, revealing an up-to-date picture of the spread of SARS-CoV-2 between and within countries and geographic regions. RESULTS AND CONCLUSIONS. The analysis suggest multiple introductions of SARS-CoV-2 into the majority of world regions by means of heterogeneous transmission pathways. The transmission network is scale-free, with a few genomic variants responsible for the majority of possible transmissions. The network structure is in line with the available temporal information represented by sample collection times and suggest the expected sampling time difference of few days between potential transmission pairs. The inferred network structural properties, transmission clusters and pathways and virus introduction routes emphasize the extent of the global epidemiological linkage and demonstrate the importance of internationally coordinated public health measures.\n",
      "Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which first occurred in Wuhan (China) in December of 2019, causes a severe acute respiratory illness with a high mortality rate, and has spread around the world. To gain an understanding of the evolution of the newly emerging SARS-CoV-2, we herein analyzed the codon usage pattern of SARS-CoV-2. For this purpose, we compared the codon usage of SARS-CoV-2 with that of other viruses belonging to the subfamily of Orthocoronavirinae. We found that SARS-CoV-2 has a high AU content that strongly influences its codon usage, which appears to be better adapted to the human host. We also studied the evolutionary pressures that influence the codon usage of five conserved coronavirus genes encoding the viral replicase, spike, envelope, membrane and nucleocapsid proteins. We found different patterns of both mutational bias and natural selection that affect the codon usage of these genes. Moreover, we show here that the two integral membrane proteins (matrix and envelope) tend to evolve slowly by accumulating nucleotide mutations on their corresponding genes. Conversely, genes encoding nucleocapsid (N), viral replicase and spike proteins (S), although they are regarded as are important targets for the development of vaccines and antiviral drugs, tend to evolve faster in comparison to the two genes mentioned above. Overall, our results suggest that the higher divergence observed for the latter three genes could represent a significant barrier in the development of antiviral therapeutics against SARS-CoV-2.\n",
      "Severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) causes coronavirus disease 2019 (COVID-19). The relentless spread and pathogenicity of the virus have become a global public health emergency. One of the striking features of this pandemic is the pronounced impact on specific regions and ethnic groups. In particular, compared with East Asia, where the virus first emerged, SARS-CoV-2 has caused high rates of morbidity and mortality in Europe. This has not been experienced in past global viral infections, such as influenza, severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) and is unique to SARS-CoV-2. For this reason, we investigated the involvement of genetic factors associated with SARS-CoV-2 infection with a focus on angiotensin-converting enzyme (ACE)-related genes, because ACE2 is a receptor for SARS-CoV-2. We found that the ACE1 II genotype frequency in a population was significantly negatively correlated with the number of SARS-CoV-2 cases. Similarly, the ACE1 II genotype was negatively correlated with the number of deaths due to SARS-CoV-2 infection. These data suggest that the ACE1 II genotype may influence the prevalence and clinical outcome of COVID-19 and serve as a predictive marker for COVID-19 risk and severity.\n",
      "Severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) causes coronavirus disease 2019 (COVID-19). The relentless spread and pathogenicity of the virus have become a global public health emergency. One of the striking features of this pandemic is the pronounced impact on specific regions and ethnic groups. In particular, compared with East Asia, where the virus first emerged, SARS-CoV-2 has caused high rates of morbidity and mortality in Europe. This has not been experienced in past global viral infections, such as influenza, severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) and is unique to SARS-CoV-2. For this reason, we investigated the involvement of genetic factors associated with SARS-CoV-2 infection with a focus on angiotensin-converting enzyme (ACE)-related genes, because ACE2 is a receptor for SARS-CoV-2. We found that the ACE1 II genotype frequency in a population was significantly negatively correlated with the number of SARS-CoV-2 cases. Similarly, the ACE1 II genotype was negatively correlated with the number of deaths due to SARS-CoV-2 infection. These data suggest that the ACE1 II genotype may influence the prevalence and clinical outcome of COVID-19 and serve as a predictive marker for COVID-19 risk and severity.\n",
      "The Coronavirus Disease-2019 (COVID-19) that started in Wuhan, China in December 2019 has spread worldwide emerging as a global pandemic. The severe respiratory pneumonia caused by the novel SARS-CoV-2 has so far claimed more than 60,000 lives and has impacted human lives worldwide. However, as the novel SARS-CoV-2 displays high transmission rates, their underlying genomic severity is required to be fully understood. We studied the complete genomes of 95 SARS-CoV-2 strains from different geographical regions worldwide to uncover the pattern of the spread of the virus. We show that there is no direct transmission pattern of the virus among neighboring countries suggesting that the outbreak is a result of travel of infected humans to different countries. We revealed unique single nucleotide polymorphisms (SNPs) in nsp13-16 (ORF1b polyprotein) and S-Protein within 10 viral isolates from the USA. These viral proteins are involved in RNA replication, indicating highly evolved viral strains circulating in the population of USA than other countries. Furthermore, we found an amino acid addition in nsp16 (mRNA cap-1 methyltransferase) of the USA isolate (MT188341) leading to shift in amino acid frame from position 2540 onwards. Through the construction of SARS-CoV-2-human interactome, we further revealed that multiple host proteins (PHB, PPP1CA, TGF-β, SOCS3, STAT3, JAK1/2, SMAD3, BCL2, CAV1 & SPECC1) are manipulated by the viral proteins (nsp2, PL-PRO, N-protein, ORF7a, M-S-ORF3a complex, nsp7-nsp8-nsp9-RdRp complex) for mediating host immune evasion. Thus, the replicative machinery of SARS-CoV-2 is fast evolving to evade host challenges which need to be considered for developing effective treatment strategies.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: iudq5jdu\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the mortality rates overall and in specific populations\n",
      "what is known about those infected with Covid-19 but are asymptomatic?\n",
      "what are the initial symptoms of Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Abstract An unprecedented outbreak of pneumonia of unknown aetiology in Wuhan City, Hubei province in China emerged in December 2019. A novel coronavirus was identified as the causative agent and was subsequently termed COVID-19 by the World Health Organization (WHO). Considered a relative of severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS), COVID-19 is caused by a betacoronavirus named SARS-CoV-2 that affects the lower respiratory tract and manifests as pneumonia in humans. Despite rigorous global containment and quarantine efforts, the incidence of COVID-19 continues to rise, with 90,870 laboratory-confirmed cases and over 3,000 deaths worldwide. In response to this global outbreak, we summarise the current state of knowledge surrounding COVID-19.\n",
      "An unprecedented outbreak of pneumonia of unknown aetiology in Wuhan City, Hubei province in China emerged in December 2019. A novel coronavirus was identified as the causative agent and was subsequently termed COVID-19 by the World Health Organization (WHO). Considered a relative of severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS), COVID-19 is caused by a betacoronavirus named SARS-CoV-2 that affects the lower respiratory tract and manifests as pneumonia in humans. Despite rigorous global containment and quarantine efforts, the incidence of COVID-19 continues to rise, with 90,870 laboratory-confirmed cases and over 3,000 deaths worldwide. In response to this global outbreak, we summarise the current state of knowledge surrounding COVID-19.\n",
      "Abstract In December 2019, the Wuhan Municipal Health and health Commission (Hubei Province, China) reported a series of cases of pneumonia of unknown aetiology. On January 7, 2020, the Chinese authorities identified as a causative agent of the outbreak a new type of virus of the Coronaviridiae family, called SARS-CoV-2. Since then, thounsands of cases have been reported with global dissemination. Infections in humans cause a broad clinical spectrum ranging from mild upper respiratory tract infection, to severe acute respiratory distress syndrome and sepsis. There is not specific treatment for SARS-CoV-2, which is why the fundamental aspects are to establish adequate prevention measures and support treatment and management of complications.\n",
      "A cluster of severe pneumonia of unknown etiology in Wuhan City, Hubei province in China emerged in December 2019. A novel coronavirus named severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) was isolated from lower respiratory tract sample as the causative agent. The current outbreak of infections with SARS-CoV-2 is termed Coronavirus Disease 2019 (COVID-19) by the World Health Organization (WHO). COVID-19 rapidly spread into at least 114 countries and killed more than 4,000 people by March 11 2020. WHO officially declared COVID-19 a pandemic on March 11, 2020. There have been two novel coronavirus outbreaks in the past two decades. The 2002-2003 outbreak of severe acute respiratory syndrome (SARS) in 2002-2003 caused by SARS-CoV had a case fatality rate of around 10% (8,098 confirmed cases and 774 death), while Middle East respiratory syndrome (MERS) caused by MERS-CoV killed 861 people out of a total 2,502 confirmed cases between 2012 and 2019. The purpose of this review is to summarize known-to-date information about SARS-CoV-2, transmission of SARS-CoV-2, and clinical features.\n",
      "A cluster of severe pneumonia of unknown etiology in Wuhan City, Hubei province in China emerged in December 2019. A novel coronavirus named severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) was isolated from lower respiratory tract sample as the causative agent. The current outbreak of infections with SARS-CoV-2 is termed Coronavirus Disease 2019 (COVID-19) by the World Health Organization (WHO). COVID-19 rapidly spread into at least 114 countries and killed more than 4,000 people by March 11 2020. WHO officially declared COVID-19 a pandemic on March 11, 2020. There have been 2 novel coronavirus outbreaks in the past 2 decades. The outbreak of severe acute respiratory syndrome (SARS) in 2002–2003 caused by SARS-CoV had a case fatality rate of around 10% (8,098 confirmed cases and 774 deaths), while Middle East respiratory syndrome (MERS) caused by MERS-CoV killed 861 people out of a total 2,502 confirmed cases between 2012 and 2019. The purpose of this review is to summarize known-to-date information about SARS-CoV-2, transmission of SARS-CoV-2, and clinical features.\n",
      "In December 2019, pneumonia of unknown cause occurred in Wuhan, Hubei Province, China. On 7 January 2020, a novel coronavirus, named as severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2), was identified in the throat swab sample of one patient. The World Health Organization (WHO) announced the epidemic disease caused by SARS-CoV-2 as coronavirus disease 2019 (COVID-19). Currently, COVID-19 has spread widely around the world, affecting more than seventy countries. China, with a huge burden of this disease, has taken strong measures to control the spread and improve the curative rate of COVID-19. In this review, we summarized the epidemiological characteristics, clinical features, diagnosis, treatment, and prognosis of COVID-19. A comprehensive understanding will help to control the disease.\n",
      "The novel coronavirus disease (COVID-19) or also known as the severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) has been recognized as the cause of respiratory infection in Wuhan, Hubei Province, China, in late December 2019. As of April 5, 2020, this epidemic had spread to worldwide with 12,03,485 confirmed cases, including 62,000 deaths. The World Health Organization has declared it a Global Public Health Crisis. Coronavirus causes respiratory illness coughing, sneezing, breathlessness, and fever including pneumonia. The disease is transmitted person to person through infected droplets. At present, the research on novel coronavirus is still in the primary stage. Based on the published study, we thoroughly summarize the history and origin, microbiology and taxonomy, mode of transmissions, target receptor, clinical features, diagnosis, prevention, and treatment about COVID-19. This short report writes in hope for providing platform to community and researcher dealings against with the novel coronavirus and providing a reference for further studies.\n",
      "INTRODUCTION In December 2019, an outbreak of pneumonia caused by a novel coronavirus occurred in Wuhan, the capital of Central China's Hubei Province and has been declared a public health emergency of international concern by the World Health Organization since January 2020. MATERIAL AND METHODS A comprehensive search using the PubMed database was carried out to summarize the latest published information about the epidemiology, definition, pathogenesis, clinical characteristics, treatment options, prognosis and prevention of coronavirus disease 2019. DISCUSSION This new strain of coronavirus, named severe acute respiratory syndrome coronavirus 2, enters human cells that express angiotensin-converting enzyme II receptors, which exist in the respiratory, gastrointestinal and genitourinary tracts and heart, causing coronavirus disease. Transmission occurs essentially through the respiratory tract and the main symptoms are fever, cough and dyspnea. Diagnosis is based on epidemiological, clinical and imaging features and confirmed by nucleic acid testing. CONCLUSION Despite intensive research, the exact origin of the virus and pathophysiology of coronavirus disease is not yet completely known, and clinically approved vaccines and drugs that target severe acute respiratory syndrome coronavirus 2 are lacking.\n",
      "Pneumonia caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection emerged in Wuhan City, Hubei Province, China in December 2019. By Feb. 11, 2020, the World Health Organization (WHO) officially named the disease resulting from infection with SARS-CoV-2 as coronavirus disease 2019 (COVID-19). COVID-19 represents a spectrum of clinical manifestations that typically include fever, dry cough, and fatigue, often with pulmonary involvement. SARS-CoV-2 is highly contagious and most individuals within the population at large are susceptible to infection. Wild animal hosts and infected patients are currently the main sources of disease which is transmitted via respiratory droplets and direct contact. Since the outbreak, the Chinese government and scientific community have acted rapidly to identify the causative agent and promptly shared the viral gene sequence, and have carried out measures to contain the epidemic. Meanwhile, recent research has revealed critical aspects of SARS-CoV-2 biology and disease pathogenesis; other studies have focused on epidemiology, clinical features, diagnosis, management, as well as drug and vaccine development. This review aims to summarize the latest research findings and to provide expert consensus. We will also share ongoing efforts and experience in China, which may provide insight on how to contain the epidemic and improve our understanding of this emerging infectious disease, together with updated guidance for prevention, control, and critical management of this pandemic.\n",
      "Abstract With the outbreak of unknown pneumonia in Wuhan, China, in December 2019, a new coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), aroused the attention of the entire world. The current outbreak of infections with SARS-CoV-2 is termed Coronavirus Disease 2019 (COVID-19). The World Health Organization declared COVID-19 in China as a Public Health Emergency of International Concern. Two other coronavirus infections—SARS in 2002-2003 and Middle East Respiratory Syndrome (MERS) in 2012—both caused severe respiratory syndrome in humans. All 3 of these emerging infectious diseases leading to a global spread are caused by β-coronaviruses. Although coronaviruses usually infect the upper or lower respiratory tract, viral shedding in plasma or serum is common. Therefore, there is still a theoretical risk of transmission of coronaviruses through the transfusion of labile blood products. Because more and more asymptomatic infections are being found among COVID-19 cases, considerations of blood safety and coronaviruses have arisen especially in endemic areas. In this review, we detail current evidence and understanding of the transmission of SARS-CoV, MERS–CoV, and SARS-CoV-2 through blood products as of February 10, 2020, and also discuss pathogen inactivation methods on coronaviruses.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 703febui\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Coronaviruses recently returned with a new one, SARS-CoV-2, related to a potentially severe respiratory disease-called the coronavirus disease (COVID-19). Research shows that the SARS-CoV-2 can be clustered with the Bat SARS-like coronavirus. Bats possess an additional, innate ability for antiviral defense, and, on the other hand, the potential to go hand-in-hand with the virus to generate variability. Besides the high potential of the novel coronavirus in compromising the respiratory system, its rapid transmission and ability to engage many hosts in severe forms of infections or immunopathological complications make it a tough opponent for the immune system. The interactions between SARS-CoV2 and the host immune system result in unleashing tremendous amounts of cytokines, and these cytokines make a storm that would determine the outcome (recovery or death) of the lungs of the patient.\n",
      "Coronaviruses recently returned with a new one, SARS-CoV-2, related to a potentially severe respiratory disease-called the coronavirus disease (COVID-19). Research shows that the SARS-CoV-2 can be clustered with the Bat SARS-like coronavirus. Bats possess an additional, innate ability for antiviral defense, and, on the other hand, the potential to go hand-in-hand with the virus to generate variability. Besides the high potential of the novel coronavirus in compromising the respiratory system, its rapid transmission and ability to engage many hosts in severe forms of infections or immunopathological complications make it a tough opponent for the immune system. The interactions between SARS-CoV2 and the host immune system result in unleashing tremendous amounts of cytokines, and these cytokines make a storm that would determine the outcome (recovery or death) of the lungs of the patient.\n",
      "In the Wuhan Province of China, in December 2019, the novel coronavirus 2019 (COVID-19) has caused a severe involvement of the lower respiratory tract leading to an acute respiratory syndrome. Subsequently, coronavirus 2 (SARS-CoV-2) provoked a pandemic which is considered a life-threatening disease. The SARS-CoV-2, a family member of betacoronaviruses, possesses single-stranded positive-sense RNA with typical structural proteins, involving the envelope, membrane, nucleocapsid and spike proteins that are responsible for the viral infectivity, and nonstructural proteins. The effectual host immune response including innate and adaptive immunity against SARS-Cov-2 seems crucial to control and resolve the viral infection. However, the severity and outcome of the COVID-19 might be associated with the excessive production of proinflammatory cytokines “cytokine storm” leading to an acute respiratory distress syndrome. Regretfully, the exact pathophysiology and treatment, especially for the severe COVID-19, is still uncertain. The results of preliminary studies have shown that immune-modulatory or immune-suppressive treatments such as hydroxychloroquine, interleukin (IL)-6 and IL-1 antagonists, commonly used in rheumatology, might be considered as treatment choices for COVID-19, particularly in severe disease. In this review, to gain better information about appropriate anti-inflammatory treatments, mostly used in rheumatology for COVID-19, we have focused the attention on the structural features of SARS-CoV-2, the host immune response against SARS-CoV-2 and its association with the cytokine storm.\n",
      "The fast-growing outbreak of 2019 novel coronaviruses (SARS-CoV-2) reached all continents except the Antarctica in merely three months. Severe SARS-CoV-2 infection (COVID-19) has a bad clinical outcome, and some reports emphasized the role of cytokine storm and dysfunctions of multiple organs. However, the etiology of severe COVID-19 has been largely unknown. Similar as SARS-CoV and MERS-CoV, SARS-CoV-2 is also thought derived from bat coronaviruses. However, it is not pathogenic for bat at all, because free DNA in cytoplasm or blood cannot bring up violent immune response in bat; but it can produce severe inflammations in human. I hypothesized that the damage induced by free DNA is a reason for severe COVID-19, which can explain many symptoms of this disease, such as cytokine storm, acute respiratory distress syndrome (ARDS) and muscus plug, acute injuries of heart, liver and kidney, and some special symptoms of COVID-19. My hypothesis will be helpful for better understand the etiology of severe COVID-19.\n",
      "In the Wuhan Province of China, in December 2019, the novel coronavirus 2019 (COVID-19) has caused a severe involvement of the lower respiratory tract leading to an acute respiratory syndrome. Subsequently, coronavirus 2 (SARS-CoV-2) provoked a pandemic which is considered a life-threatening disease. The SARS-CoV-2, a family member of betacoronaviruses, possesses single-stranded positive-sense RNA with typical structural proteins, involving the envelope, membrane, nucleocapsid and spike proteins that are responsible for the viral infectivity, and nonstructural proteins. The effectual host immune response including innate and adaptive immunity against SARS-Cov-2 seems crucial to control and resolve the viral infection. However, the severity and outcome of the COVID-19 might be associated with the excessive production of proinflammatory cytokines \"cytokine storm\" leading to an acute respiratory distress syndrome. Regretfully, the exact pathophysiology and treatment, especially for the severe COVID-19, is still uncertain. The results of preliminary studies have shown that immune-modulatory or immune-suppressive treatments such as hydroxychloroquine, interleukin (IL)-6 and IL-1 antagonists, commonly used in rheumatology, might be considered as treatment choices for COVID-19, particularly in severe disease. In this review, to gain better information about appropriate anti-inflammatory treatments, mostly used in rheumatology for COVID-19, we have focused the attention on the structural features of SARS-CoV-2, the host immune response against SARS-CoV-2 and its association with the cytokine storm.\n",
      "A novel coronavirus, SARS-CoV-2, has recently emerged in China and spread internationally, posing a health emergency to the global community. COVID-19 caused by SARS-CoV-2 is associated with an acute respiratory illness that varies from mild to the life-threatening acute respiratory distress syndrome (ARDS). The complement system is part of the innate immune arsenal against pathogens, which many viruses can evade or employ to mediate cell entry. The immunopathology and acute lung injury orchestrated through the influx of pro-inflammatory macrophages and neutrophils can be directly activated by complement components to prime an overzealous cytokine storm. The manifestations of severe COVID-19 such as the ARDS, sepsis and multiorgan failure have an established relationship with activation of the complement cascade. We have collected evidence from all the current studies we are aware of on SARS-CoV-2 immunopathogenesis and the preceding literature on SARS-CoV-1 and MERS-CoV infection linking severe COVID-19 disease directly with dysfunction of the complement pathways. This information lends support for a therapeutic anti-inflammatory strategy against complement, where a number of clinically ready potential therapeutic agents are available.\n",
      "The 2019 novel coronavirus (2019-nCoV/SARS-CoV-2) originally arose as part of a major outbreak of respiratory disease centered on Hubei province, China. It is now a global pandemic and is a major public health concern. Taxonomically, SARS-CoV-2 was shown to be a Betacoronavirus (lineage B) closely related to SARS-CoV and SARS-related bat coronaviruses, and it has been reported to share a common receptor with SARS-CoV (ACE-2). Subsequently, betacoronaviruses from pangolins were identified as close relatives to SARS-CoV-2. Here, we perform structural modeling of the SARS-CoV-2 spike glycoprotein. Our data provide support for the similar receptor utilization between SARS-CoV-2 and SARS-CoV, despite a relatively low amino acid similarity in the receptor binding module. Compared to SARS-CoV and all other coronaviruses in Betacoronavirus lineage B, we identify an extended structural loop containing basic amino acids at the interface of the receptor binding (S1) and fusion (S2) domains. We suggest this loop confers fusion activation and entry properties more in line with betacoronaviruses in lineages A and C, and be a key component in the evolution of SARS-CoV-2 with this structural loop affecting virus stability and transmission.\n",
      "Abstract The fast-growing outbreak of 2019 novel coronaviruses (SARS-CoV-2) reached all continents except the Antarctica in merely three months. Severe SARS-CoV-2 infection (COVID-19) has a bad clinical outcome, and some reports emphasized the role of cytokine storm and dysfunctions of multiple organs. However, the etiology of severe COVID-19 has been largely unknown. Similar as SARS-CoV and MERS-CoV, SARS-CoV-2 is also thought derived from bat coronaviruses. However, it is not pathogenic for bat at all, because free DNA in cytoplasm or blood cannot bring up violent immune response in bat; but it can produce severe inflammations in human. I hypothesized that the damage induced by free DNA is a reason for severe COVID-19, which can explain many symptoms of this disease, such as cytokine storm, ARDS and muscus plug, acute injuries of heart, liver and kidney, and some special symptoms of COVID-19. My hypothesis will be helpful for better understand the etiology of severe COVID-19.\n",
      "Severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) is a novel coronavirus that has caused a worldwide pandemic of the human respiratory illness COVID-19, resulting in a severe threat to public health and safety. Analysis of the genetic tree suggests that SARS-CoV-2 belongs to the same Betacoronavirus group as severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV). Although the route for viral transmission remains a mystery, SARS-CoV-2 may have originated in an animal reservoir, likely that of bat. The clinical features of COVID-19, such as fever, cough, shortness of breath, and fatigue, are similar to those of many acute respiratory infections. There is currently no specific treatment for COVID-19, but antiviral therapy combined with supportive care is the main strategy. Here, we summarize recent progress in understanding the epidemiological, virological, and clinical characteristics of COVID-19 and discuss potential targets with existing drugs for the treatment of this emerging zoonotic disease.\n",
      "A novel coronavirus, SARS-CoV-2, has recently emerged in China and spread internationally, posing a health emergency to the global community. COVID-19 caused by SARS-CoV-2 is associated with an acute respiratory illness that varies from mild to the life-threatening acute respiratory distress syndrome (ARDS). The complement system is part of the innate immune arsenal against pathogens, in which many viruses can evade or employ to mediate cell entry. The immunopathology and acute lung injury orchestrated through the influx of pro-inflammatory macrophages and neutrophils can be directly activated by complement components to prime an overzealous cytokine storm. The manifestations of severe COVID-19 such as the ARDS, sepsis and multiorgan failure have an established relationship with activation of the complement cascade. We have collected evidence from all the current studies we are aware of on SARS-CoV-2 immunopathogenesis and the preceding literature on SARS-CoV-1 and MERS-CoV infection linking severe COVID-19 disease directly with dysfunction of the complement pathways. This information lends support for a therapeutic anti-inflammatory strategy against complement, where a number of clinically ready potential therapeutic agents are available.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 023h20vk\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What vaccine candidates are being tested for Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "The recent outbreak of severe acute respiratory syndrome (SARS) coronavirus (CoV)-2 (SARS-CoV-2) causing coronavirus disease (covid19) has posed a great threat to human health. Previous outbreaks of SARS-CoV and Middle East respiratory Syndrome CoV (MERS-CoV) from the same CoV family had posed similar threat to human health and economic growth. To date, not even a single drug specific to any of these CoVs has been developed nor any anti-viral vaccine is available for the treatment of diseases caused by CoVs. Subunits present in spike glycoproteins of SARS-CoV and SARS-CoV-2 are involved in binding to human ACE2 Receptor which is the primary method of viral invasion. As it has been observed in the previous studies that there are very minor differences in the spike glycoproteins of SARS-CoV and SARS-CoV-2. SARS-CoV-2 has an additional furin cleavage site that makes it different from SARS-CoV (Walls et al., 2020). In this study, we have analyzed spike glycoproteins of SARS-CoV-2 and SARS-CoV phylogenetically and subjected them to selection pressure analysis. Selection pressure analysis has revealed some important sites in SARS-CoV-2 and SARS-CoV spike glycoproteins that might be involved in their pathogenicity. Further, we have developed a potential multi-epitope vaccine candidate against SARS-CoV-2 by analyzing its interactions with HLA-B*15:03 subtype. This vaccine consists of multiple T-helper (TH) cells, B-cells, and Cytotoxic T-cells (CTL) epitopes joined by linkers and an adjuvant to increase its immunogenicity. Conservation of selected epitopes in SARS, MERS, and human hosts, suggests that the designed vaccine could provide cross-protection. The vaccine is designed in silico by following a reverse vaccinology method acknowledging its antigenicity, immunogenicity, toxicity, and allergenicity. The vaccine candidate that we have designed as a result of this work shows promising result indicating its potential capability of simulating an immune response.\n",
      "The 2019 novel coronavirus (COVID19 / Wuhan coronavirus), officially named as Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), is a positive-sense single-stranded RNA coronavirus. SARS-CoV-2 causes the contagious COVID19 disease also known as 2019-nCoV acute respiratory disease and has led to the ongoing 2019–20 pandemic COVID19 outbreak. The effective counter measures against SARS-CoV-2 infection require the design and development of specific and effective vaccine candidate. In the present study, we have screened and shortlisted 38 CTL, 33 HTL and 12 B cell epitopes from the eleven Protein sequences of SARS-CoV-2 by utilizing different in silico tools. The screened epitopes were further validated for their binding with their respective HLA allele binders and TAP (Transporter associated with antigen processing) molecule by molecular docking. The shortlisted screened epitopes were further utilized to design novel two multi-epitope vaccines (MEVs) composed of CTL, HTL and B cell epitopes overlaps with potential to elicit humoral as well as cellular immune response against SARS-CoV-2. To enhance the immune response for our vaccine design, truncated (residues 10-153) Onchocerca volvulus activation-associated secreted protein-1 (Ov-ASP-1) has been utilized as an adjuvant at N terminal of both the MEVs. Further molecular models for both the MEVs were prepared and validated for their stable molecular interactions with Toll-Like Receptor 3 (TLR 3). The codon-optimized cDNA of both the MEVs were further analyzed for their potential of high level of expression in a human cell line. The present study is very significant in terms of molecular designing of prospective CTL and HTL vaccine against SARS-CoV-2 infection with the potential to elicit cellular as well as humoral immune response. (SARS-CoV-2), Coronavirus, Human Transporter associated with antigen processing (TAP), Toll-Like Receptor (TLR), Epitope, Immunoinformatics, Molecular Docking, Molecular dynamics simulation, Multi-epitope Vaccine Graphical abstract The designed CTL (Cytotoxic T lymphocyte) and HTL (Helper T lymphocyte) multi-epitope vaccines (MEV) against COVID19 infection. Both the CTL and HTL MEV models show a very stable and well fit conformational complex formation tendency with the Toll like receptor 3. CTL and HTL MEVs: ribbon; Toll like receptor 3: gray cartoon; Adjuvant [truncated (residues 10-153) Onchocerca volvulus activation-associated secreted protein-1]: orange ribbon regions; Epitopes: cyan ribbons regions; 6xHis Tag: magenta ribbon regions.\n",
      "Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which first occurred in Wuhan (China) in December 2019, is a novel virus that causes a severe acute respiratory disease. The virus spike glycoproteins and nucleocapsid proteins are the main targets for the development of vaccines and antiviral drugs, to control the disease spread. We herein study the structural order-disorder propensity and the rates of evolution of these two proteins to characterize their B cell and T cell epitopes, previously suggested to contribute to immune response caused by SARS-CoV-2 infections. We first analyzed the rates of evolution along the sequences of spike and nucleocapsid proteins in relation to the spatial locations of their epitopes. For this purpose, we compared orthologs from seven human coronaviruses: SARS-CoV-2, SARS-CoV, MERS-CoV, HCoV-229E, HCoV-OC43, HCoV-NL63, and HCoV-HKU1. We then focus on the local, structural order-disorder propensities of the protein regions where the SARS-CoV-2 epitopes are located. We show that the vast majority of nucleocapsid protein epitopes overlap the RNA-binding and dimerization domains and some of them are characterized by low rates of evolutions. Similarly, spike protein epitopes are preferentially located in regions that are predicted to be ordered and well-conserved, in correspondence of the heptad repeats 1 and 2. Interestingly, both the receptor-binding motif to ACE2 and the fusion peptide of spike protein are characterized by high rates of evolution, probably to overcome host immunity. In conclusion, our results provide evidence for conserved epitopes that may help to develop long-lasting, broad-spectrum SARS-CoV-2 vaccines.\n",
      "The emergence of 2019 novel coronavirus (2019-nCoV) is of global concern and might have emerged from RNA recombination among existing coronaviruses. CoV spike (S) protein which is crucial for receptor binding, membrane fusion via conformational changes, internalization of the virus, host tissue tropism and comprises crucial targets for vaccine development, remain largely uncharacterized. Therefore, the present study has been planned to determine the sequence variation, structural and antigenic divergence of S glycoprotein which may be helpful for the management of 2019-nCoV infection. The sequences of spike glycoprotein of 2019-nCoV and SARS coronavirus (SARS-CoV) were used for the comparison. The sequence variations were determined using EMBOSS Needle pairwise sequence alignment tools. The variation in glycosylation sites was predicted by NetNGlyc 1.0 and validated by N-GlyDE server. Antigenicity was predicted by NetCTL 1.2 and validated by IEDB Analysis Resource server. The structural divergence was determined by using SuperPose Version 1.0 based on cryo-EM structure of the SARS coronavirus spike glycoprotein. Our data suggests that 2019-nCoV is newly spilled coronavirus into humans in China is closely related to SARS-CoV, which has only 12.8% of difference with SARS-CoV in S protein and has 83.9% similarity in minimal receptor-binding domain with SARS-CoV. Addition of a novel glycosylation sites were observed in 2019-nCoV. In addition, antigenic analysis proposes that great antigenic differences exist between both the viral strains, but some of the epitopes were found to be similar between both the S proteins. In spite of the variation in S protein amino acid composition, we found no significant difference in their structures. Collectively, for the first time our results exhibit the emergence of human 2019-nCoV is closely related to predecessor SARS-CoV and provide the evidence that 2019-nCoV uses various novel glycosylation sites as SARS-CoV and may have a potential to become pandemic owing its antigenic discrepancy. Further, demonstration of novel Cytotoxic T lymphocyte epitopes may impart opportunities for the development of peptide based vaccine for the prevention of 2019-nCoV.\n",
      "The beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the 120 available SARS-CoV-2 sequences (as of 21 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.\n",
      "The beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the available SARS-CoV-2 sequences (as of 9 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.\n",
      "Since the first appearance of the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS- CoV-2) in China on December 2019, the world has now witnessed the emergence of the SARS- CoV-2 outbreak. Therefore, due to the high transmissibility rate of virus, there is an urgent need to design and develop vaccines against SARS-CoV-2 to prevent more cases affected by the virus. In this study, a computational approach is proposed for vaccine design against the envelope (E) protein of SARS-CoV-2, which contains a conserved sequence feature. First, we sought to gain potential B-cell and T-cell epitopes for vaccine designing against SARS-CoV-2. Second, we attempted to develop a multi-epitope vaccine. Immune targeting of such epitopes could theoretically provide defense against SARS-CoV-2. Finally, we evaluated the affinity of the vaccine to major histocompatibility complex (MHC) molecules to stimulate the immune system response to this vaccine. We also identified a collection of B-cell and T-cell epitopes derived from E proteins that correspond identically to SARS-CoV-2 E proteins. The in-silico design of our potential vaccine against E protein of SARS-CoV-2 demonstrated a high affinity to MHC molecules, and it can be a candidate to make a protection against this pandemic event.\n",
      "The 2019 novel severe respiratory syndrome coronavirus-2 (SARS-CoV-2) outbreak has caused a large number of deaths with thousands of confirmed cases worldwide. The present study followed computational approaches to identify B- and T-cell epitopes for spike glycoprotein of SARS-CoV-2 by its interactions with the human leukocyte antigen alleles. We identified twenty-four peptide stretches on the SARS-CoV-2 spike protein that are well conserved among the reported strains. The S protein structure further validated the presence of predicted peptides on the surface. Out of which twenty are surface exposed and predicted to have reasonable epitope binding efficiency. The work could be useful for understanding the immunodominant regions in the surface protein of SARS-CoV-2 and could potentially help in designing some peptide-based diagnostics.\n",
      "The recent pandemic of COVID-19, caused by SARS-CoV-2, is unarguably the most fearsome compared with the earlier outbreaks caused by other coronaviruses, SARS-CoV and MERS-CoV. Human ACE2 is now established as a receptor for the SARS-CoV-2 spike protein. Where variations in the viral spike protein, in turn, lead to the cross-species transmission of the virus, genetic variations in the host receptor ACE2 may also contribute to the susceptibility and/or resistance against the viral infection. This study aims to explore the binding of the proteins encoded by different human ACE2 allelic variants with SARS-CoV-2 spike protein. Briefly, coding variants of ACE2 corresponding to the reported binding sites for its attachment with coronavirus spike protein were selected and molecular models of these variants were constructed by homology modeling. The models were then superimposed over the native ACE2 and ACE2-spike protein complex, to observe structural changes in the ACE2 variants and their intermolecular interactions with SARS-CoV-2 spike protein, respectively. Despite strong overall structural similarities, the spatial orientation of the key interacting residues varies in the ACE2 variants compared with the wild-type molecule. Most ACE2 variants showed a similar binding affinity for SARS-CoV-2 spike protein as observed in the complex structure of wild-type ACE2 and SARS-CoV-2 spike protein. However, ACE2 alleles, rs73635825 (S19P) and rs143936283 (E329G) showed noticeable variations in their intermolecular interactions with the viral spike protein. In summary, our data provide a structural basis of potential resistance against SARS-CoV-2 infection driven by ACE2 allelic variants.\n",
      "The 2019 novel severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) outbreak has caused a large number of deaths, with thousands of confirmed cases worldwide. The present study followed computational approaches to identify B- and T-cell epitopes for the spike (S) glycoprotein of SARS-CoV-2 by its interactions with the human leukocyte antigen alleles. We identified 24 peptide stretches on the SARS-CoV-2 S protein that are well conserved among the reported strains. The S protein structure further validated the presence of predicted peptides on the surface, of which 20 are surface exposed and predicted to have reasonable epitope binding efficiency. The work could be useful for understanding the immunodominant regions in the surface protein of SARS-CoV-2 and could potentially help in designing some peptide-based diagnostics. Also, identified T-cell epitopes might be considered for incorporation in vaccine designs.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: g4qak0bu\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what is known about an mRNA vaccine for the SARS-CoV-2 virus?\n",
      "do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Antibody responses develop following SARS-CoV-2 infection, but little is known about their epitope specificities, clonality, binding affinities, epitopes, and neutralizing activity. We isolated B cells specific for the SARS-CoV-2 envelope glycoprotein spike (S) from a COVID-19-infected subject 21 days after the onset of clinical disease. 45 S-specific monoclonal antibodies were generated. They had undergone minimal somatic mutation with limited clonal expansion, and three bound the receptor-binding domain (RBD). Two antibodies neutralized SARS-CoV-2. The most potent antibody bound the RBD and prevented binding to the ACE2 receptor, while the other bound outside the RBD. Thus, most anti-S antibodies that were generated in this patient during the first weeks of COVID-19 infection were non-neutralizing and target epitopes outside the RBD. Antibodies that disrupt the SARS-CoV-2 S-ACE2 interaction can potently neutralize the virus without undergoing extensive maturation. Such antibodies have potential preventive and/or therapeutic potential and can serve as templates for vaccine design.\n",
      "Abstract Antibody responses develop following SARS-CoV-2 infection, but little is known about their epitope specificities, clonality, binding affinities, epitopes and neutralizing activity. We isolated B cells specific for the SARS-CoV-2 envelope glycoprotein spike (S) from a COVID-19-infected subject twenty-one days after the onset of clinical disease. Forty-five S-specific monoclonal antibodies were generated. They had undergone minimal somatic mutation, with limited clonal expansion and three bound the receptor binding domain (RBD). Two antibodies neutralized SARS-CoV-2. The most potent antibody bound the RBD and prevented binding to the ACE2 receptor, while the other bound outside the RBD. Thus, most anti-S antibodies that were generated in this patient during the first weeks of COVID-19 infection were non-neutralizing and target epitopes outside the RBD. Antibodies that disrupt the SARS-CoV-2 S-ACE2 interaction can potently neutralize the virus without undergoing extensive maturation. Such antibodies have potential preventive and/or therapeutic potential and can serve as templates for vaccine-design.\n",
      "B cells specific for the SARS-CoV-2 S envelope glycoprotein spike were isolated from a COVID-19-infected subject using a stabilized spike-derived ectodomain (S2P) twenty-one days post-infection. Forty-four S2P-specific monoclonal antibodies were generated, three of which bound to the receptor binding domain (RBD). The antibodies were minimally mutated from germline and were derived from different B cell lineages. Only two antibodies displayed neutralizing activity against SARS-CoV-2 pseudo-virus. The most potent antibody bound the RBD in a manner that prevented binding to the ACE2 receptor, while the other bound outside the RBD. Our study indicates that the majority of antibodies against the viral envelope spike that were generated during the first weeks of COVID-19 infection are non-neutralizing and target epitopes outside the RBD. Antibodies that disrupt the SARS-CoV-2 spike-ACE2 interaction can potently neutralize the virus without undergoing extensive maturation. Such antibodies have potential preventive/therapeutic potential and can serve as templates for vaccine-design. IN BRIEF SARS-CoV-2 infection leads to expansion of diverse B cells clones against the viral spike glycoprotein (S). The antibodies bind S with high affinity despite being minimally mutated. Thus, the development of neutralizing antibody responses by vaccination will require the activation of certain naïve B cells without requiring extensive somatic mutation. Highlights Analysis of early B cell response to SARS-CoV-2 spike protein Most antibodies target non-neutralizing epitopes Potent neutralizing mAb blocks the interaction of the S protein with ACE2 Neutralizing antibodies are minimally mutated\n",
      "Developing therapeutics against SARS-CoV-2 could be guided by the distribution of epitopes, not only on the receptor binding domain (RBD) of the Spike (S) protein, but also across the full Spike (S) protein. We isolated and characterized monoclonal antibodies (mAbs) from ten convalescent COVID-19 patients. Three mAbs showed neutralizing activities against authentic SARS-CoV-2. An mAb, named 4A8, exhibits high neutralization potency against both authentic and pseudotyped SARS-CoV-2, but does not bind the RBD. We defined the epitope of 4A8 as the N terminal domain (NTD) of the S protein by determining its cryo-EM structure in complex with the S protein to an overall resolution of 3.1 Angstrom and local resolution of 3.3 Angstrom for the 4A8-NTD interface. This points to the NTD as a promising target for therapeutic mAbs against COVID-19.\n",
      "The coronavirus induced disease 19 (COVID-19) caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has become a worldwide threat to human lives, and neutralizing antibodies present a great therapeutic potential in curing affected patients. We purified more than one thousand memory B cells specific to SARS-CoV-2 S1 or RBD (receptor binding domain) antigens from 11 convalescent COVID-19 patients, and a total of 729 naturally paired heavy and light chain fragments were obtained by single B cell cloning technology. Among these, 178 recombinant monoclonal antibodies were tested positive for antigen binding, and the top 13 binders with Kd below 0.5 nM are all RBD binders. Importantly, all these 13 antibodies could block pseudoviral entry into HEK293T cells overexpressing ACE2, with the best ones showing IC50s around 2-3 nM. We further identified 8 neutralizing antibodies against authentic virus with IC50s within 10 nM. Among these, 414-1 blocked authentic viral entry at IC50 of 1.75 nM and in combination with 105-38 could achieve IC50 as low as 0.45 nM. Meanwhile, we also found that 3 antibodies could cross-react with the SARS-CoV spike protein. Altogether, our study provided a panel of potent human neutralizing antibodies for COVID19 as therapeutics candidates for further development.\n",
      "Neutralizing antibody responses to coronaviruses mainly target the receptor-binding domain (RBD) of the trimeric spike. Here, we characterized polyclonal immunoglobulin Gs (IgGs) and Fabs from COVID-19 convalescent individuals for recognition of coronavirus spikes. Plasma IgGs differed in their focus on RBD epitopes, recognition of alpha- and beta-coronaviruses, and contributions of avidity to increased binding/neutralization of IgGs over Fabs. Using electron microscopy, we examined specificities of polyclonal plasma Fabs, revealing recognition of both S1A and RBD epitopes on SARS-CoV-2 spike. Moreover, a 3.4 Å cryo-electron microscopy (cryo-EM) structure of a neutralizing monoclonal Fab-spike complex revealed an epitope that blocks ACE2 receptor binding. Modeling based on these structures suggested different potentials for inter-spike crosslinking by IgGs on viruses, and characterized IgGs would not be affected by identified SARS-CoV-2 spike mutations. Overall, our studies structurally define a recurrent anti-SARS-CoV-2 antibody class derived from VH3-53/VH3-66 and similarity to a SARS-CoV VH3-30 antibody, providing criteria for evaluating vaccine-elicited antibodies.\n",
      "The emerging coronavirus SARS-CoV-2 pandemic presents a global health emergency in urgent need of interventions1-3. SARS-CoV-2 entry into the target cells depends on binding between the receptor-binding domain (RBD) of the viral Spike protein and the ACE2 cell receptor2,4-6. Here, we report the isolation and characterization of 206 RBD-specific monoclonal antibodies derived from single B cells of eight SARS-CoV-2 infected individuals. We identified antibodies with potent anti-SARS-CoV-2 neutralization activity that correlates with their competitive capacity with ACE2 for RBD binding. Surprisingly, neither the anti-SARS-CoV-2 antibodies nor the infected plasma cross-reacted with SARS-CoV or MERS-CoV RBDs, although substantial plasma cross-reactivity to their trimeric Spike proteins was found. Crystal structure analysis of RBD-bound antibody revealed steric hindrance that inhibits viral engagement with ACE2 and thereby blocks viral entry. These findings suggest that anti-RBD antibodies are viral species-specific inhibitors. The antibodies identified here may be candidates for the development of SARS-CoV-2 clinical interventions.\n",
      "The emerging coronavirus SARS-CoV-2 pandemic presents a global health emergency in urgent need of interventions1-3. SARS-CoV-2 entry into the target cells depends on binding between the receptor-binding domain (RBD) of the viral Spike protein and the ACE2 cell receptor2,4-6. Here, we report the isolation and characterization of 206 RBD-specific monoclonal antibodies derived from single B cells of eight SARS-CoV-2 infected individuals. We identified antibodies with potent anti-SARS-CoV-2 neutralization activity that correlates with their competitive capacity with ACE2 for RBD binding. Surprisingly, neither the anti-SARS-CoV-2 antibodies nor the infected plasma cross-reacted with SARS-CoV or MERS-CoV RBDs, although substantial plasma cross-reactivity to their trimeric Spike proteins was found. Crystal structure analysis of RBD-bound antibody revealed steric hindrance that inhibits viral engagement with ACE2 and thereby blocks viral entry. These findings suggest that anti-RBD antibodies are viral species-specific inhibitors. The antibodies identified here may be candidates for the development of SARS-CoV-2 clinical interventions.\n",
      "Neutralizing antibody responses to coronaviruses focus on the trimeric spike, with most against the receptor-binding domain (RBD). Here we characterized polyclonal IgGs and Fabs from COVID-19 convalescent individuals for recognition of coronavirus spikes. Plasma IgGs differed in their degree of focus on RBD epitopes, recognition of SARS-CoV, MERS-CoV, and mild coronaviruses, and how avidity effects contributed to increased binding/neutralization of IgGs over Fabs. Electron microscopy reconstructions of polyclonal plasma Fab-spike complexes showed recognition of both S1(A) and RBD epitopes. A 3.4Å cryo-EM structure of a neutralizing monoclonal Fab-S complex revealed an epitope that blocks ACE2 receptor-binding on “up” RBDs. Modeling suggested that IgGs targeting these sites have different potentials for inter-spike crosslinking on viruses and would not be greatly affected by identified SARS-CoV-2 spike mutations. These studies structurally define a recurrent anti-SARS-CoV-2 antibody class derived from VH3–53/VH3–66 and similarity to a SARS-CoV VH3–30 antibody, providing criteria for evaluating vaccine-elicited antibodies.\n",
      "The COVID-19 pandemic is a major threat to global health for which there are only limited medical countermeasures, and we lack a thorough understanding of mechanisms of humoral immunity1,2. From a panel of monoclonal antibodies (mAbs) targeting the spike (S) glycoprotein isolated from the B cells of infected subjects, we identified several mAbs that exhibited potent neutralizing activity with IC50 values as low as 0.9 or 15 ng/mL in pseudovirus or wild-type (wt) SARS-CoV-2 neutralization tests, respectively. The most potent mAbs fully block the receptor-binding domain of S (SRBD) from interacting with human ACE2. Competition-binding, structural, and functional studies allowed clustering of the mAbs into defined classes recognizing distinct epitopes within major antigenic sites on the SRBD. Electron microscopy studies revealed that these mAbs recognize distinct conformational states of trimeric S protein. Potent neutralizing mAbs recognizing unique sites, COV2-2196 and COV2-2130, bound simultaneously to S and synergistically neutralized authentic SARS-CoV-2 virus. In two murine models of SARS-CoV-2 infection, passive transfer of either COV2-2916 or COV2-2130 alone or a combination of both mAbs protected mice from severe weight loss and reduced viral burden and inflammation in the lung. These results identify protective epitopes on the SRBD and provide a structure-based framework for rational vaccine design and the selection of robust immunotherapeutic cocktails.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: rirzes0m\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what evidence is there for the value of hydroxychloroquine in treating Covid-19?\n",
      "Does Vitamin D impact COVID-19 prevention and treatment?\n",
      "what evidence is there for dexamethasone as a treatment for COVID-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Despite intense research there is currently no effective vaccine available against the new severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) emerged in the later 2019 and responsible for the COVID-19 pandemic. This infectious and communicable disease has become one of the major public health challenges in the world. The clinical management of COVID-19 has been limited to infection prevention and control measures associated with supportive care such as supplemental oxygen and mechanical ventilation. Meanwhile efforts to find an effective treatment to inhibit virus replication, mitigate the symptoms, increase survival and decrease mortality rate are ongoing. Several classes of drugs, many of them already in use for other diseases, are being evaluated based on the body of clinical knowledge obtained from infected patients regarding to the natural history and evolution of the infection. Herein we will provide an updated overview of the natural history and current knowledge on drugs and therapeutic agents being tested for the prevention and treatment of COVID-19. These include different classes of drugs such as antiviral agents (chloroquine, ivermectin, nitazoxanide, hydroxychloroquine, lopinavir, remdesivir, tocilizumab), supporting agents (Vitamin C, Vitamin D, azithromycin, corticosteroids) and promising investigational vaccines. Considering the controversies and excessive number of compounds being tested and reported in the literature we hope that this review can provide useful and updated consolidated information on potential drugs used to prevent, control and treat COVID-19 patients worldwide.\n",
      "The current pandemic of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has presented unprecedented challenges to the healthcare systems in almost every country around the world. Currently, there are no proven effective vaccines or therapeutic agents against the virus. Current clinical management includes infection prevention and control measures and supportive care including supplemental oxygen and mechanical ventilatory support. Evolving research and clinical data regarding the virologic SARS-CoV-2 suggest a potential list of repurposed drugs with appropriate pharmacological effects and therapeutic efficacies in treating COVID-19 patients. In this review, we will update and summarize the most common and plausible drugs for the treatment of COVID-19 patients. These drugs and therapeutic agents include antiviral agents (remdesivir, hydroxychloroquine, chloroquine, lopinavir, umifenovir, favipiravir, and oseltamivir), and supporting agents (Ascorbic acid, Azithromycin, Corticosteroids, Nitric oxide, IL-6 antagonists), among others. We hope that this review will provide useful and most updated therapeutic drugs to prevent, control, and treat COVID-19 patients until the approval of vaccines and specific drugs targeting SARS-CoV-2.\n",
      "BACKGROUND: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), termed coronavirus disease 2019 (COVID-19) by the World Health Organization, is a newly emerging zoonotic agent that emerged in China in December 2019. No specific treatment for COVID-19 is currently available. Usual palliative treatment includes maintaining hydration and nutrition and controlling fever and cough. The clinical severity and extent of transmission need to be determined, and therapeutic options need to be developed and optimized. METHODS: The present review discusses the recent repurposing of drugs for COVID-19 treatment. RESULTS: Several compounds, including remdesivir, lopinavir, ritonavir, interferon-ß, ribavirin, chloroquine/hydroxychloroquine, azithromycin, tocilizumab, and ivermectin, have emerged as promising alternatives. They block the virus from entering host cells, prevent viral replication, and attenuate exacerbation of the host's immune response. CONCLUSION: Although some evidence indicates the positive actions of different classes of compounds for the treatment of COVID-19, few clinical assays have been established to definitively demonstrate their therapeutic value in humans. Multicenter clinical studies are urgently needed to validate and standardize therapeutic regimens that involve these agents. Although science has not yet presented us with a specific drug against COVID-19, the repurposing of drugs appears to be promising in our fight against this devastating disease.\n",
      "BACKGROUND Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), termed coronavirus disease 2019 (COVID-19) by the World Health Organization, is a newly emerging zoonotic agent that emerged in China in December 2019. No specific treatment for COVID-19 is currently available. Usual palliative treatment includes maintaining hydration and nutrition and controlling fever and cough. The clinical severity and extent of transmission need to be determined, and therapeutic options need to be developed and optimized. METHODS The present review discusses the recent repurposing of drugs for COVID-19 treatment. RESULTS Several compounds, including remdesivir, lopinavir, ritonavir, interferon-β, ribavirin, chloroquine/hydroxychloroquine, azithromycin, tocilizumab, and ivermectin, have emerged as promising alternatives. They block the virus from entering host cells, prevent viral replication, and attenuate exacerbation of the host's immune response. CONCLUSION Although some evidence indicates the positive actions of different classes of compounds for the treatment of COVID-19, few clinical assays have been established to definitively demonstrate their therapeutic value in humans. Multicenter clinical studies are urgently needed to validate and standardize therapeutic regimens that involve these agents. Although science has not yet presented us with a specific drug against COVID-19, the repurposing of drugs appears to be promising in our fight against this devastating disease.\n",
      "INTRODUCTION: The coronavirus disease 2019 (COVID-19) caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has developed since December 2019. It has caused a global pandemic with more than three hundred thousand case fatalities. However, apart from supportive care by respirators, no standard medical therapy is validated. AREAS COVERED: This paper presents old drugs with potential in vitro efficacy against SARS-CoV-2. The in vitro database, adverse effects, and potential toxicities of these drugs are reviewed regarding their feasibility of clinical prescription for the treatment of patients with COVID-19. To obtain convincing recommendations, we referred to opinions from the US National Institute of Health regarding drugs repurposed for COVID-19 therapy. EXPERT OPINION: Although strong evidence of well-designed randomized controlled studies regarding COVID-19 therapy is presently lacking, remdesivir, teicoplanin, hydroxychloroquine (not in combination with azithromycin), and ivermectin might be effective antiviral drugs and are deemed promising candidates for controlling SARS-CoV-2. In addition, tocilizumab might be considered as the supplementary treatment for COVID-19 patients with cytokine release syndrome. In future, clinical trials regarding a combination of potentially effective drugs against SARS-CoV-2 need to be conducted to establish the optimal regimen for the treatment of patients with moderate-to-severe COVID-19.\n",
      "Introduction: The coronavirus disease 2019 (COVID-19) caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has developed since December 2019. It has caused a global pandemic with more than three hundred thousand case fatalities. However, apart from supportive care by respirators, no standard medical therapy is validated.Areas covered: This paper presents old drugs with potential in vitro efficacy against SARS-CoV-2. The in vitro database, adverse effects, and potential toxicities of these drugs are reviewed regarding their feasibility of clinical prescription for the treatment of patients with COVID-19. To obtain convincing recommendations, we referred to opinions from the US National Institute of Health regarding drugs repurposed for COVID-19 therapy.Expert opinion: Although strong evidence of well-designed randomized controlled studies regarding COVID-19 therapy is presently lacking, remdesivir, teicoplanin, hydroxychloroquine (not in combination with azithromycin), and ivermectin might be effective antiviral drugs and are deemed promising candidates for controlling SARS-CoV-2. In addition, tocilizumab might be considered as the supplementary treatment for COVID-19 patients with cytokine release syndrome. In future, clinical trials regarding a combination of potentially effective drugs against SARS-CoV-2 need to be conducted to establish the optimal regimen for the treatment of patients with moderate-to-severe COVID-19.\n",
      "The emerging outbreak of coronavirus disease 2019 (COVID-19) caused by the severe acute respiratory syndrome coronavirus 2 continues to spread all over the world. Agents or vaccines of proven efficacy to treat or prevent human coronavirus infection are in urgent need and are being investigated vigorously worldwide. This review summarizes the current evidence of potential therapeutic agents, such as lopinavir/ritonavir, remdesivir, favipiravir, chloroquine, hydroxychloroquine, interferon, ribavirin, tocilizumab, and sarilumab. More clinical trials are being conducted for further confirmation of the efficacy and safety of these agents in treating COVID-19.\n",
      "Severe acute respiratory syndrome coronavirus 2 (SARS-Cov-2), initially termed 2019-new CoV (2019-nCoV), is a novel coronavirus responsible for the severe respiratory illness currently ongoing worldwide from the beginning of December 2019. This beta gene virus, very close to bat coronaviruses (bat-CoV-RaTG13) and bat-SL-CoVZC45, causes a severe disease, similar to those caused by Middle East respiratory syndrome (MERS)-CoV and SARS-CoV viruses, featured by low to moderate mortality rate. Unfortunately, the antiviral drugs commonly used in clinical practice to treat viral infections, are not applicable to SARS-Cov-2 and no vaccine is available. Thus, it is extremely necessary to identify new drugs suitable for the treatment of the 2019-nCoV outbreak. Different preclinical studies conducted on other coronaviruses suggested that promising clinical outcomes for 2019-nCoV should be obtained by using alpha-interferon, chloroquine phosphate, arabinol, remdesivir, lopinavir/ritonavir, and anti-inflammatory drugs. Moreover, clinical trials with these suitable drugs should be performed on patients affected by SARS-Cov-2 to prove their efficacy and safety. Finally, a very promising therapeutic drug, tocilizumab, is discussed; it is currently used to treat patients presenting COVID-19 pneumonia. Herein, we recapitulate these experimental studies to highlight the use of antiviral drugs for the treatment of SARS-Cov-2 disease.\n",
      "Severe acute respiratory syndrome coronavirus 2 (SARS-Cov-2), initially termed 2019-new CoV (2019-nCoV), is a novel coronavirus responsible for the severe respiratory illness currently ongoing worldwide from the beginning of December 2019. This beta gene virus, very close to bat coronaviruses (bat-CoV-RaTG13) and bat-SL-CoVZC45, causes a severe disease, similar to those caused by Middle East respiratory syndrome (MERS)-CoV and SARS-CoV viruses, featured by low to moderate mortality rate. Unfortunately, the antiviral drugs commonly used in clinical practice to treat viral infections, are not applicable to SARS-Cov-2 and no vaccine is available. Thus, it is extremely necessary to identify new drugs suitable for the treatment of the 2019-nCoV outbreak. Different preclinical studies conducted on other coronaviruses suggested that promising clinical outcomes for 2019-nCoV should be obtained by using alpha-interferon, chloroquine phosphate, arabinol, remdesivir, lopinavir/ritonavir, and anti-inflammatory drugs. Moreover, clinical trials with these suitable drugs should be performed on patients affected by SARS-Cov-2 to prove their efficacy and safety. Finally, a very promising therapeutic drug, tocilizumab, is discussed; it is currently used to treat patients presenting COVID-19 pneumonia. Herein, we recapitulate these experimental studies to highlight the use of antiviral drugs for the treatment of SARS-Cov-2 disease.\n",
      "Abstract The novel severe acute respiratory syndrome coronavirus 2 is causing a worldwide pandemic that may lead to a highly morbid and potentially fatal coronavirus disease-19 (COVID-19). There is currently no drug that has been proven as an effective therapy for COVID-19. Several candidate drugs are being considered and evaluated for treatment. This includes clinically-available drugs, such as chloroquine, hydroxychloroquine, and lopinavir/ritonavir, which are being repurposed for the treatment of COVID-19. Novel experimental therapies, such as remdesivir and favipiravir, are also actively being investigated for antiviral efficacy. Clinically-available and investigational immunomodulators, such as the IL-6 inhibitors tocilizumab and sarilumab and the anti-GMCSF lenzilumab, are being tested for their anticipated effect in counteracting the pro-inflammatory cytokine environment that characterizes severe and critical COVID-19. This review article examines the evidence behind the potential use of these leading drug candidates for the treatment of COVID-19. The authors conclude, based on this review, that there is still no high-quality evidence to support any of these proposed drug therapies. The authors, therefore, encourage the enrollment of eligible patients to multiple ongoing clinical trials that assess the efficacy and safety of these candidate therapies. Until the results of controlled trials are available, none of the suggested therapeutics is clinically proven as an effective therapy for COVID-19.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: ywaefpe8\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "are there any clinical trials available for the coronavirus\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Different treatments are currently used for clinical management of SARS-CoV-2 infection, but little is known about their efficacy yet. Here we present ongoing results to compare currently available drugs for a variety of diseases to find out if they counteract SARS-CoV-2-induced cytopathic effect in vitro. Our goal is to prioritize antiviral activity to provide a solid evidence-driven rationale for forthcoming clinical trials. Since the most effective antiviral approaches are usually based on combined therapies that tackle the viral life cycle at different stages, we are also testing combinations of drugs that may be critical to reduce the emergence of resistant viruses. We will provide results as soon as they become available, so data should be interpreted with caution, clearly understanding the limitations of the in vitro model, that may not always reflect what could happen in vivo. Thus, our goal is to test the most active antivirals identified in adequate animal models infected with SARS-CoV-2, to add more information about possible in vivo efficacy. In turn, successful antivirals could be tested in clinical trials as treatments for infected patients, but also as pre-exposure prophylaxis to avoid novel infections until an effective and safe vaccine is developed.\n",
      "BACKGROUND: With the worldwide spread of SARS-CoV-2 infection, it is becoming increasingly urgent to develop a vaccine to prevent COVID-19, as well as effective drugs to treat it. METHODS: This article is based on a selective literature search in PubMed and ClinicalTrials.gov, followed by an assessment of the ongoing clinical trials that were revealed by the search. RESULTS: A number of substances have been found to prevent the reproduction of SARS-CoV-2 in vitro. These include virustatic agents that have already been approved for the treatment of other types of viral infection, as well as drugs that are currently used for entirely different purposes. High in vitro activity has been found for the nucleotide analogue remdesivir, for the antimalarial drug chloroquine, and for nitazoxanide, a drug used to treat protozoan infections. Because the virus enters human cells by way of the membrane-associated angiotensin converting enzyme 2 (ACE2), keeping the virus from docking to this receptor is a conceivable treatment approach. Transmembrane protease serine 2 (TMPRSS2) plays a role in the fusion of the virus with cells; inhibitors of this enzyme are known as well. The potential therapeutic efficacy and tolerability of these and other active substances remain to be investigated in clinical trials. At present, more than 80 trials on COVID-10 have already been registered with ClinicalTrials.gov. Some initial findings should already be available in late April 2020. CONCLUSION: Clinical trials are now indispensable in order to determine the true clinical benefits and risks of the substances that have been found to be active against SARS-CoV-2 in vitro. There is not yet any recommendation for the therapeutic use of any particular agent beyond standard supportive treatment.\n",
      "Coronavirus disease-19 (COVID-19) has reached pandemic proportions. Most of the drugs that are being tried for the treatment have not been evaluated in any randomized controlled trials. The purpose of this review was to summarize the in-vitro and in-vivo efficacy of these drugs on Severe Acute Respiratory Syndrome (SARS-CoV-2) and related viruses (SARS and Middle East Respiratory Syndrome) and evaluate their potential for re-purposing them in the management of COVID-19.\n",
      "The global pandemic of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2 or 2019-nCoV) has prompted multiple clinical trials to jumpstart search for anti-SARS-CoV-2 therapies from existing drugs, including those with reported in vitro efficacies as well as those ones that are not known to inhibit SARS-CoV-2, such as ritonavir/lopinavir and favilavir. Here we report that after screening 19 antiviral drugs that are either in clinical trials or with proposed activity against SARS-CoV-2, remdesivir was the most effective. Chloroquine only effectively protected virus-induced cytopathic effect at around 30 µM with a therapeutic index of 1.5. Our findings also suggest that velpatasvir, ledipasvir, ritonavir, litonavir, lopinavir, favilavir, sofosbuvir, danoprevir, and pocapavir do not have direct antiviral effect.\n",
      "With the ongoing SARS-CoV-2 pandemic there is an urgent need for the discovery of a treatment for the coronavirus disease (COVID-19). Drug repurposing is one of the most rapid strategies for addressing this need and numerous compounds have been selected for in vitro testing by several groups already. These have led to a growing database of molecules with in vitro activity against the virus. Machine learning models can assist drug discovery through prediction of the best compounds based on previously published data. Herein we have implemented several machine learning methods to develop predictive models from recent SARS-CoV-2 in vitro inhibition data and used them to prioritize additional FDA approved compounds for in vitro testing selected from our in-house compound library. From the compounds predicted with a Bayesian machine learning model, CPI1062 and CPI1155 showed antiviral activity in HeLa-ACE2 cell-based assays and represent potential repurposing opportunities for COVID-19. This approach can be greatly expanded to exhaustively virtually screen available molecules with predicted activity against this virus as well as a prioritization tool for SARS-CoV-2 antiviral drug discovery programs. The very latest model for SARS-CoV-2 is available at www.assaycentral.org.\n",
      "OBJECTIVES/BACKGROUND AND OBJECTIVES: Prior epidemics of high-mortality human coronaviruses, such as the acute respiratory syndrome coronavirus (SARS-CoV or SARS-1) in 2003, have driven the characterization of compounds that could be possibly active against the currently emerging novel coronavirus SARS-CoV-2 (COVID-19). Presently, no approved treatment or prophylaxis is available for COVID-19. We comment on the existing COVID-19 research methodologies in general and the published reporting. Given the media attention and claims of effectiveness, we chose chloroquine and hydroxychloroquine, in combination with azithromycin, as an area of COVID-19 research to examine. METHODS/STUDY DESIGN AND SETTING: MEDLINE and EMBASE electronic databases were searched from 2019 to present (April 3rd, 2020) using a mix of keywords such as COVID-19 and chloroquine and hydroxychloroquine. We also searched the largest clinical medicine preprint repository, medRxiv.org. RESULTS: We found 6 studies, 3 randomized control trials and 3 observational studies, focusing on chloroquine and hydroxychloroquine (with azithromycin). We critically appraised the evidence. CONCLUSION: We found that the COVID-19 research methodology is very poor in the area of chloroquine/hydroxychloroquine research. In screening the literature, we observed the same across COVID-19 research in relation to potential treatments. The reporting is very poor and sparse, and patient-important outcomes needed to discern decision-making priorities are not reported. We do understand the barriers to perform rigorous research in health care settings overwhelmed by a novel deadly disease. However, this emergency pandemic situation does not transform flawed methods and data into credible results. The adequately powered, comparative, and robust clinical research that is needed for optimal evidence-informed decision-making remains absent in COVID-19.\n",
      "In response to the current pandemic caused by the novel SARS-CoV-2, identifying and validating effective therapeutic strategies is more than ever necessary. We evaluated the in vitro antiviral activities of a shortlist of compounds, known for their cellular broad-spectrum activities, together with drugs that are currently under evaluation in clinical trials for COVID-19 patients. We report the antiviral effect of remdesivir, lopinavir, chloroquine, umifenovir, berberine and cyclosporine A in Vero E6 cells model of SARS-CoV-2 infection, with estimated 50% inhibitory concentrations of 0.99, 5.2, 1.38, 3.5, 10.6 and 3 μM, respectively. Virus-directed plus host-directed drug combinations were also investigated. We report a strong antagonism between remdesivir and berberine, in contrast with remdesivir/diltiazem, for which we describe high levels of synergy, with mean Loewe synergy scores of 12 and peak values above 50. Combination of host-directed drugs with direct acting antivirals underscore further validation in more physiological models, yet they open up interesting avenues for the treatment of COVID-19.\n",
      "An escalating pandemic by the novel SARS-CoV-2 virus is impacting global health and effective therapeutic options are urgently needed. We evaluated the in vitro antiviral effect of compounds that were previously reported to inhibit coronavirus replication and compounds that are currently under evaluation in clinical trials for SARS-CoV-2 patients. We report the antiviral effect of remdesivir, lopinavir, homorringtonine, and emetine against SARS-CoV-2 virus in Vero E6 cells with the estimated 50% effective concentration at 23.15 μM, 26.63 μM, 2.55 μM and 0.46 μM, respectively. Ribavirin or favipiravir that are currently evaluated under clinical trials showed no inhibition at 100 μM. Synergy between remdesivir and emetine was observed, and remdesivir at 6.25 μM in combination with emetine at 0.195 μM may achieve 64.9% inhibition in viral yield. Combinational therapy may help to reduce the effective concentration of compounds below the therapeutic plasma concentrations and provide better clinical benefits.\n",
      "An escalating pandemic by the novel SARS-CoV-2 virus is impacting global health and effective therapeutic options are urgently needed. We evaluated the in vitro antiviral effect of compounds that were previously reported to inhibit coronavirus replication and compounds that are currently under evaluation in clinical trials for SARS-CoV-2 patients. We report the antiviral effect of remdesivir, lopinavir, homorringtonine, and emetine against SARS-CoV-2 virus in Vero E6 cells with the estimated 50% effective concentration at 23.15 µM, 26.63 µM, 2.55 µM and 0.46 µM, respectively. Ribavirin or favipiravir that are currently evaluated under clinical trials showed no inhibition at 100 µM. Synergy between remdesivir and emetine was observed, and remdesivir at 6.25 µM in combination with emetine at 0.195 µM may achieve 64.9% inhibition in viral yield. Combinational therapy may help to reduce the effective concentration of compounds below the therapeutic plasma concentrations and provide better clinical benefits.\n",
      "The disease caused by SARS-CoV2, covid-19, rapidly spreads worldwide, causing the greatest threat to global public health in the last 100 years. This scenario has become catastrophic as there are no approved vaccines to prevent the disease, and the main measures to contain the virus transmission are confinement and social distancing. One priority strategy is based on drug repurposing by pursuing antiviral chemotherapy that can control transmission and prevent complications associated with covid-19. With this aim, we performed a high content screening assay for the discovery of anti-SARS-CoV-2 compounds. From the 65 screened compounds, we have found four drugs capable to selectively inhibit SARS-CoV-2 in vitro infection: brequinar, abiraterone acetate, neomycin, and the extract of Hedera helix. Brequinar and abiraterone acetate had higher inhibition potency against SARS-CoV-2 than neomycin and Hedera helix extract, respectively. Drugs with reported antiviral activity and in clinical trials for covid-19, chloroquine, ivermectin, and nitazoxanide, were also included in the screening, and the last two were found to be non-selective. We used a data mining approach to build drug-host molecules-biological function-disease networks to show in a holistic way how each compound is interconnected with host node molecules and virus infection, replication, inflammatory response, and cell apoptosis. In summary, the present manuscript identified four drugs with active inhibition effect on SARS-CoV-2 in vitro infection, and by network analysis, we provided new insights and starting points for the clinical evaluation and repurposing process to treat SARS-CoV-2 infection. Summary sentence Discovery of drug repurposing candidates, inhibitors of SARS-CoV-2 infection in vitro, using a phenotypic screening strategy and network analysis.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 2qugltxu\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "The first case of human transmission of SARS-CoV-2 was reported in China in December 2019. A few months later, this viral infection had spread worldwide and became a pandemic. The disease caused by SARS-CoV-2, termed COVID-19, is multifactorial and associated with both specific antiviral as well as inflammatory responses, the extent of which may determine why some individuals are asymptomatic while others develop serious complications. Here we review possible life-threating immune events that can occur during disease progression to uncover key factors behind COVID-19 severity and provide suggestions for interventions with repurposed drugs in well-controlled and randomized clinical trials. These drugs include therapeutics with potential to inhibit SARS-CoV-2 entry into host cells such as serine protease inhibitors of the cellular protease TMPS2 and drugs targeting the renin-angiotensin system; antivirals with potential to block SARS-CoV-2 replication or factors that could boost the antiviral response; monoclonal antibodies targeting pro-inflammatory cytokines that drive the hyperinflammatory response during COVID-19 progression toward the severe stage and therapeutics that could ameliorate the function of the lungs. Furthermore, in order to help make more informed decisions on the timing of the intervention with the drugs listed in this review, we have grouped these therapeutics according to the stage of COVID-19 progression that we considered most appropriate for their mechanism of action.\n",
      "Since the infectious disease caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) was reported in China during December 2019, the coronavirus disease 2019 (COVID-19) has spread on a global scale, causing the World Health Organization (WHO) to issue a warning. While novel vaccines and drugs that target SARS-CoV-2 are under development, this review provides information on therapeutics which are under clinical trials or are proposed to antagonize SARS-CoV-2. Based on the information gained from the responses to other RNA coronaviruses, including the strains that cause severe acute respiratory syndrome (SARS)-coronaviruses and Middle East respiratory syndrome (MERS), drug repurposing might be a viable strategy. Since several antiviral therapies can inhibit viral replication cycles or relieve symptoms, mechanisms unique to RNA viruses will be important for the clinical development of antivirals against SARS-CoV-2. Given that several currently marketed drugs may be efficient therapeutic agents for severe COVID-19 cases, they may be beneficial for future viral pandemics and other infections caused by RNA viruses when standard treatments are unavailable.\n",
      "BACKGROUND Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), termed coronavirus disease 2019 (COVID-19) by the World Health Organization, is a newly emerging zoonotic agent that emerged in China in December 2019. No specific treatment for COVID-19 is currently available. Usual palliative treatment includes maintaining hydration and nutrition and controlling fever and cough. The clinical severity and extent of transmission need to be determined, and therapeutic options need to be developed and optimized. METHODS The present review discusses the recent repurposing of drugs for COVID-19 treatment. RESULTS Several compounds, including remdesivir, lopinavir, ritonavir, interferon-β, ribavirin, chloroquine/hydroxychloroquine, azithromycin, tocilizumab, and ivermectin, have emerged as promising alternatives. They block the virus from entering host cells, prevent viral replication, and attenuate exacerbation of the host's immune response. CONCLUSION Although some evidence indicates the positive actions of different classes of compounds for the treatment of COVID-19, few clinical assays have been established to definitively demonstrate their therapeutic value in humans. Multicenter clinical studies are urgently needed to validate and standardize therapeutic regimens that involve these agents. Although science has not yet presented us with a specific drug against COVID-19, the repurposing of drugs appears to be promising in our fight against this devastating disease.\n",
      "BACKGROUND: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), termed coronavirus disease 2019 (COVID-19) by the World Health Organization, is a newly emerging zoonotic agent that emerged in China in December 2019. No specific treatment for COVID-19 is currently available. Usual palliative treatment includes maintaining hydration and nutrition and controlling fever and cough. The clinical severity and extent of transmission need to be determined, and therapeutic options need to be developed and optimized. METHODS: The present review discusses the recent repurposing of drugs for COVID-19 treatment. RESULTS: Several compounds, including remdesivir, lopinavir, ritonavir, interferon-ß, ribavirin, chloroquine/hydroxychloroquine, azithromycin, tocilizumab, and ivermectin, have emerged as promising alternatives. They block the virus from entering host cells, prevent viral replication, and attenuate exacerbation of the host's immune response. CONCLUSION: Although some evidence indicates the positive actions of different classes of compounds for the treatment of COVID-19, few clinical assays have been established to definitively demonstrate their therapeutic value in humans. Multicenter clinical studies are urgently needed to validate and standardize therapeutic regimens that involve these agents. Although science has not yet presented us with a specific drug against COVID-19, the repurposing of drugs appears to be promising in our fight against this devastating disease.\n",
      "In the Wuhan Province of China, in December 2019, the novel coronavirus 2019 (COVID-19) has caused a severe involvement of the lower respiratory tract leading to an acute respiratory syndrome. Subsequently, coronavirus 2 (SARS-CoV-2) provoked a pandemic which is considered a life-threatening disease. The SARS-CoV-2, a family member of betacoronaviruses, possesses single-stranded positive-sense RNA with typical structural proteins, involving the envelope, membrane, nucleocapsid and spike proteins that are responsible for the viral infectivity, and nonstructural proteins. The effectual host immune response including innate and adaptive immunity against SARS-Cov-2 seems crucial to control and resolve the viral infection. However, the severity and outcome of the COVID-19 might be associated with the excessive production of proinflammatory cytokines “cytokine storm” leading to an acute respiratory distress syndrome. Regretfully, the exact pathophysiology and treatment, especially for the severe COVID-19, is still uncertain. The results of preliminary studies have shown that immune-modulatory or immune-suppressive treatments such as hydroxychloroquine, interleukin (IL)-6 and IL-1 antagonists, commonly used in rheumatology, might be considered as treatment choices for COVID-19, particularly in severe disease. In this review, to gain better information about appropriate anti-inflammatory treatments, mostly used in rheumatology for COVID-19, we have focused the attention on the structural features of SARS-CoV-2, the host immune response against SARS-CoV-2 and its association with the cytokine storm.\n",
      "A novel coronavirus disease 2019 (COVID-19) infection caused by severe acute respiratory syndrome coronavirus 2 (SARS-Cov-2) first emerged in December 2019 in Wuhan, China, has become a global pandemic. Currently, the management of COVID-19 infection is mainly supportive. Several clinical trials worldwide are evaluating several drugs approved for other indications, as well as multiple investigational agents for the treatment and prevention of COVID-19. Here, we give a brief overview of pharmacological agents and other therapies which are under investigation as treatment options or adjunctive agents for patients infected with COVID-19 and for chemoprophylaxis for the prevention of COVID-19 infection. At the time of writing this commentary, there is no peer-reviewed published evidence from randomized clinical trials of any pharmacological agents improving outcomes in COVID-19 patients. However, it was reported that remdesivir an investigational antiviral agent hastens clinical recovery, but a study is yet to be published in peer-reviewed medical journal.\n",
      "A novel coronavirus disease 2019 (COVID-19) infection caused by severe acute respiratory syndrome coronavirus 2 (SARS-Cov-2) first emerged in December 2019 in Wuhan, China, has become a global pandemic. Currently, the management of COVID-19 infection is mainly supportive. Several clinical trials worldwide are evaluating several drugs approved for other indications, as well as multiple investigational agents for the treatment and prevention of COVID-19. Here, we give a brief overview of pharmacological agents and other therapies which are under investigation as treatment options or adjunctive agents for patients infected with COVID-19 and for chemoprophylaxis for the prevention of COVID-19 infection. At the time of writing this commentary, there is no peer-reviewed published evidence from randomized clinical trials of any pharmacological agents improving outcomes in COVID-19 patients. However, it was reported that remdesivir an investigational antiviral agent hastens clinical recovery, but a study is yet to be published in peer-reviewed medical journal.\n",
      "In the Wuhan Province of China, in December 2019, the novel coronavirus 2019 (COVID-19) has caused a severe involvement of the lower respiratory tract leading to an acute respiratory syndrome. Subsequently, coronavirus 2 (SARS-CoV-2) provoked a pandemic which is considered a life-threatening disease. The SARS-CoV-2, a family member of betacoronaviruses, possesses single-stranded positive-sense RNA with typical structural proteins, involving the envelope, membrane, nucleocapsid and spike proteins that are responsible for the viral infectivity, and nonstructural proteins. The effectual host immune response including innate and adaptive immunity against SARS-Cov-2 seems crucial to control and resolve the viral infection. However, the severity and outcome of the COVID-19 might be associated with the excessive production of proinflammatory cytokines \"cytokine storm\" leading to an acute respiratory distress syndrome. Regretfully, the exact pathophysiology and treatment, especially for the severe COVID-19, is still uncertain. The results of preliminary studies have shown that immune-modulatory or immune-suppressive treatments such as hydroxychloroquine, interleukin (IL)-6 and IL-1 antagonists, commonly used in rheumatology, might be considered as treatment choices for COVID-19, particularly in severe disease. In this review, to gain better information about appropriate anti-inflammatory treatments, mostly used in rheumatology for COVID-19, we have focused the attention on the structural features of SARS-CoV-2, the host immune response against SARS-CoV-2 and its association with the cytokine storm.\n",
      "Severe acute respiratory syndrome coronavirus 2 (SARS-CoV2) is the reason for this ongoing pandemic infection diseases termed coronavirus disease 2019 (COVID-19) that has emerged since early December 2019 in Wuhan City, Hubei Province, China. In this century, it is the worst threat to international health and the economy. After 4 months of COVID-19 outbreak, there is no certain and approved medicine against it. In this public health emergency, it makes sense to investigate the possible effects of old drugs and find drug repositioning that is efficient, economical, and riskless process. Old drugs that may be effective are from different pharmacological categories, antimalarials, anthelmintics, anti-protozoal, anti-HIVs, anti-influenza, anti-hepacivirus, antineoplastics, neutralizing antibodies, immunoglobulins, and interferons. In vitro, in vivo, or preliminary trials of these drugs in the treatment of COVID-19 have been encouraging, leading to new research projects and trials to find the best drug/s. In this review, we discuss the possible mechanisms of these drugs against COVID-19. Also, it should be mentioned that in this manuscript, we discuss preliminary rationales; however, clinical trial evidence is needed to prove them. COVID-19 therapy must be based on expert clinical experience and published literature and guidelines from major health organizations. Moreover, herein, we describe current evidence that may be changed in the future.\n",
      "INTRODUCTION: The coronavirus disease 2019 (COVID-19) caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has developed since December 2019. It has caused a global pandemic with more than three hundred thousand case fatalities. However, apart from supportive care by respirators, no standard medical therapy is validated. AREAS COVERED: This paper presents old drugs with potential in vitro efficacy against SARS-CoV-2. The in vitro database, adverse effects, and potential toxicities of these drugs are reviewed regarding their feasibility of clinical prescription for the treatment of patients with COVID-19. To obtain convincing recommendations, we referred to opinions from the US National Institute of Health regarding drugs repurposed for COVID-19 therapy. EXPERT OPINION: Although strong evidence of well-designed randomized controlled studies regarding COVID-19 therapy is presently lacking, remdesivir, teicoplanin, hydroxychloroquine (not in combination with azithromycin), and ivermectin might be effective antiviral drugs and are deemed promising candidates for controlling SARS-CoV-2. In addition, tocilizumab might be considered as the supplementary treatment for COVID-19 patients with cytokine release syndrome. In future, clinical trials regarding a combination of potentially effective drugs against SARS-CoV-2 need to be conducted to establish the optimal regimen for the treatment of patients with moderate-to-severe COVID-19.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: xxkagj6e\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Since the outbreak of the 2019 novel coronavirus disease (COVID-19), the medical research community is vigorously seeking a treatment to control the infection and save the lives of severely infected patients. The main potential candidates for the control of viruses are virally targeted agents. In this short letter, we report our calculations on the inhibitors for the SARS-CoV-2 3CL protease and the spike protein for the potential treatment of COVID-19. The results show that the most potent inhibitors of the SARS-CoV-2 3CL protease include saquinavir, tadalafil, rivaroxaban, sildenafil, dasatinib, etc. Ergotamine, amphotericin b, and vancomycin are most promising to block the interaction of the SARS-CoV-2 S-protein with human ACE-2.\n",
      "BACKGROUND: The COVID-19 has now been declared a global pandemic by the World Health Organization. There is an emergent need to search for possible medications. METHOD: Utilization of the available sequence information, homology modeling, and in slico docking a number of available medications might prove to be effective in inhibiting the SARS-CoV-2 two main drug targets, the spike glycoprotein, and the 3CL protease. RESULTS: Several compounds were determined from the in silico docking models that might prove to be effective inhibitors for SARS-CoV-2. Several antiviral medications: Zanamivir, Indinavir, Saquinavir, and Remdesivir show potential as and 3CLPRO main proteinase inhibitors and as a treatment for COVID-19. CONCLUSION: Zanamivir, Indinavir, Saquinavir, and Remdesivir are among the exciting hits on the 3CLPRO main proteinase. It is also exciting to uncover that Flavin Adenine Dinucleotide (FAD) Adeflavin, B2 deficiency medicine, and Coenzyme A, a coenzyme, may also be potentially used for the treatment of SARS-CoV-2 infections. The use of these off-label medications may be beneficial in the treatment of the COVID-19.\n",
      "The recent outbreak of the 2019 novel coronavirus disease (COVID-19) has been proved as a global threat. No particular drug or vaccine has not yet been discovered which may act specifically against severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) and causes COVID-19. For this highly infectious virus, 3CL-like main protease (3CLpro) plays a key role in the virus life cycle and can be considered as a pivotal drug target. Structure-based virtual screening of DrugBank database resulted in 20 hits against 3CLpro. Atomistic 100 ns molecular dynamics of five top hits and binding energy calculation analyses were performed for main protease-hit complexes. Among the top five hits, Nafarelin and Icatibant affirmed the binding energy (g_MMPBSA) of -712.94 kJ/mol and -851.74 kJ/mol, respectively. Based on binding energy and stability of protein-ligand complex; the present work reports these two drug-like hits against SARS-CoV-2 main protease.Communicated by Ramaswamy H. Sarma.\n",
      "The recent outbreak of the 2019 novel coronavirus disease (COVID-19) has been proved as a global threat. No particular drug or vaccine has not yet been discovered which may act specifically against severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) and causes COVID-19. For this highly infectious virus, 3CL-like main protease (3CLpro) plays a key role in the virus life cycle and can be considered as a pivotal drug target. Structure-based virtual screening of DrugBank database resulted in 20 hits against 3CLpro. Atomistic 100 ns molecular dynamics of five top hits and binding energy calculation analyses were performed for main protease-hit complexes. Among the top five hits, Nafarelin and Icatibant affirmed the binding energy (g_MMPBSA) of -712.94 kJ/mol and -851.74 kJ/mol, respectively. Based on binding energy and stability of protein-ligand complex; the present work reports these two drug-like hits against SARS-CoV-2 main protease.Communicated by Ramaswamy H. Sarma.\n",
      "Abstract Background The COVID-19 has now been declared a global pandemic by the World Health Organization. There is an emergent need to search for possible medications. Method Utilization of the available sequence information, homology modeling, and in slico docking a number of available medications might prove to be effective in inhibiting the SARS-CoV-2 two main drug targets, the spike glycoprotein, and the 3CL protease. Results Several compounds were determined from the in silico docking models that might prove to be effective inhibitors for SARS-CoV-2. Several antiviral medications: Zanamivir, Indinavir, Saquinavir, and Remdesivir show potential as and 3CLPRO main proteinase inhibitors and as a treatment for COVID-19. Conclusion Zanamivir, Indinavir, Saquinavir, and Remdesivir are among the exciting hits on the 3CLPRO main proteinase. It is also exciting to uncover that Flavin Adenine Dinucleotide (FAD) Adeflavin, B2 deficiency medicine, and Coenzyme A, a coenzyme, may also be potentially used for the treatment of SARS-CoV-2 infections. The use of these off-label medications may be beneficial in the treatment of the COVID-19.\n",
      "Direct-acting antivirals are effective tools to control viral infections. SARS-CoV-2 is a coronavirus associated with the epidemiological outbreak in late 2019. Previous reports showed that HIV-1 protease inhibitors could block SARS-CoV main protease. Based on that and using an in silico approach, we evaluated SARS-CoV-2 main protease as a target for HIV-1 protease inhibitors to reveal the structural features related to their antiviral effect. Our results showed that several HIV inhibitors such as lopinavir, ritonavir, and saquinavir produce strong interaction with the active site of SARS-CoV-2 main protease. Furthermore, broad library protease inhibitors obtained from PubChem and ZINC (www.zinc.docking.org) were evaluated. Our analysis revealed 20 compounds that could be clustered into three groups based on their chemical features. Then, these structures could serve as leading compounds to develop a series of derivatives optimizing their activity against SARS-CoV-2 and other coronaviruses. Altogether, the results presented in this work contribute to gain a deep understanding of the molecular pharmacology of SARS-CoV-2 treatment and validate the use of protease inhibitors against SARS-CoV-2.\n",
      "[Image: see text] The coronavirus disease 2019 (COVID-19) pandemic caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has infected over 7.1 million people and led to over 0.4 million deaths. Currently, there is no specific anti-SARS-CoV-2 medication. New drug discovery typically takes more than 10 years. Drug repositioning becomes one of the most feasible approaches for combating COVID-19. This work curates the largest available experimental data set for SARS-CoV-2 or SARS-CoV 3CL (main) protease inhibitors. On the basis of this data set, we develop validated machine learning models with relatively low root-mean-square error to screen 1553 FDA-approved drugs as well as another 7012 investigational or off-market drugs in DrugBank. We found that many existing drugs might be potentially potent to SARS-CoV-2. The druggability of many potent SARS-CoV-2 3CL protease inhibitors is analyzed. This work offers a foundation for further experimental studies of COVID-19 drug repositioning.\n",
      "The coronavirus disease 2019 (COVID-19) pandemic caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has infected over 7.1 million people and led to over 0.4 million deaths. Currently, there is no specific anti-SARS-CoV-2 medication. New drug discovery typically takes more than 10 years. Drug repositioning becomes one of the most feasible approaches for combating COVID-19. This work curates the largest available experimental data set for SARS-CoV-2 or SARS-CoV 3CL (main) protease inhibitors. On the basis of this data set, we develop validated machine learning models with relatively low root-mean-square error to screen 1553 FDA-approved drugs as well as another 7012 investigational or off-market drugs in DrugBank. We found that many existing drugs might be potentially potent to SARS-CoV-2. The druggability of many potent SARS-CoV-2 3CL protease inhibitors is analyzed. This work offers a foundation for further experimental studies of COVID-19 drug repositioning.\n",
      "Human infections with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) cause coronavirus disease 19 (COVID-19) and there is currently no cure. The 3C-like protease (3CLpro), a highly conserved protease indispensable for replication of coronaviruses, is a promising target for development of broad-spectrum antiviral drugs. To advance the speed of drug discovery and development, we investigated the inhibition of SARS-CoV-2 3CLpro by natural products derived from Chinese traditional medicines. Baicalin and baicalein were identified as the first non-covalent, non-peptidomimetic inhibitors of SARS-CoV-2 3CLpro and exhibited potent antiviral activities in a cell-based system. Remarkably, the binding mode of baicalein with SARS-CoV-2 3CLpro determined by X-ray protein crystallography is distinctly different from those of known inhibitors. Baicalein is perfectly ensconced in the core of the substrate-binding pocket by interacting with two catalytic residues, the crucial S1/S2 subsites and the oxyanion loop, acting as a “shield” in front of the catalytic dyad to prevent the peptide substrate approaching the active site. The simple chemical structure, unique mode of action, and potent antiviral activities in vitro, coupled with the favorable safety data from clinical trials, emphasize that baicalein provides a great opportunity for the development of critically needed anti-coronaviral drugs.\n",
      "The severe acute respiratory syndrome-associated coronavirus (SARS-CoV) 3C-like protease (3CL(pro) or M(pro)) is an attractive target for the development of anti-SARS drugs because of its crucial role in the viral life cycle. In this study, a compound database was screened by the structure-based virtual screening approach to identify initial hits as inhibitors of SARS-CoV 3CL(pro). Out of the 59,363 compounds docked, 93 were selected for the inhibition assay, and 21 showed inhibition against SARS-CoV 3CL(pro) (IC(50) <or= 30 microM), with three of them having common substructures. Furthermore, a search for analogues with common substructure in the Maybridge, ChemBridge, and SPECS_SC databases led to the identification of another 25 compounds that exhibited inhibition against SARS-CoV 3CL(pro) (IC(50) = 3-1,000 microM). These compounds, 28 in total, were subjected to 3D-QSAR studies to elucidate the pharmacophore of SARS-CoV 3CL(pro).\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: nz02frdm\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what types of rapid testing for Covid-19 have been developed?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Objectives: To report our experiences screening and managing patients with suspected or confirmed novel coronavirus (COVID-19) disease using a hospital-specific protocol. Design: Longitudinal cohort study. Setting: A 1,200 bed tertiary care teaching hospital in Chengdu, Sichuan, China. Participants: 802 adults presenting to hospital with concerns of having COVID-19, 1,246 inpatients and 2,531 hospital visitors. Interventions: Screening and management of patients using a hospital-specific protocol, which included fever triage, monitoring visitors and patients, emergency response, personnel training for healthcare team members, health education for patients and family, medical materials management, disinfection and wastes disposal protocols. Results: Between 23 January and 28 February 2020, 73 people were identified as having fever plus respiratory signs with/without a history of exposure and were tested for the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by our hospital lab using RT PCR. Forty-five of these 73 people were subsequently excluded based on one negative RT PCR result plus positive results to quick screening tests for flu or other respiratory viruses. The remaining 28 people received a second RT PCR test 24 h later. Three people were confirmed positive for COVID-19 based on two consecutive positive RT PCR tests whilst 25 people were excluded based on two consecutive negative tests. The three COVID-19 confirmed cases received non-critical care. There were no new infections of medical staff or new infections of other hospital inpatients. Conclusions: All three cases were detected as a result of vigilant monitoring of hospital visitors. Whilst screening out-patients presenting to a fever clinic remains important, monitoring visitors must not be overlooked.\n",
      "BACKGROUND: Coronavirus disease 2019 (COVID-19) has been highly epidemic in China since January 2020. Rapid detection of the causative agent, severe acute respiratory coronavirus-2 (SARS-CoV-2), is very important due to its high rate of infectivity. This study aimed to clarify the epidemiology and clinical characteristics of COVID-19 outside of Hubei province, China, and to optimize screening strategies for COVID-19 in attempts to contain spread of the virus. METHODS: This retrospective study included all confirmed cases of COVID-19 in Hunan Provincial People's Hospital (Changsha, China) between January 22 and February 15, 2020. All cases were detected using a real-time reverse transcription polymerase chain reaction assay. The epidemiology and clinical characteristic of these cases were investigated according to outcome in attempts to optimize screening strategies for COVID-19. RESULTS: There were 24 confirmed cases of COVID-19 in the fever outpatient department of Hunan Provincial People's Hospital. Three patients were asymptomatic, and 3 exhibited mild and 3 moderate disease. There was a family cluster phenomenon. CONCLUSION: Individuals with COVID-19 can be asymptomatic or exhibit mild manifestations of disease. Close monitoring and an optimized screening strategy for COVID-19 could help deter spread of the virus.\n",
      "BACKGROUND: The outbreak of COVID-19 has led to international concern. We aimed to establish an effective screening strategy in Shanghai, China, to aid early identification of patients with COVID-19. METHODS: We did a multicentre, observational cohort study in fever clinics of 25 hospitals in 16 districts of Shanghai. All patients visiting the clinics within the study period were included. A strategy for COVID-19 screening was presented and then suspected cases were monitored and analysed until they were confirmed as cases or excluded. Logistic regression was used to determine the risk factors of COVID-19. FINDINGS: We enrolled patients visiting fever clinics from Jan 17 to Feb 16, 2020. Among 53 617 patients visiting fever clinics, 1004 (1·9%) were considered as suspected cases, with 188 (0·4% of all patients, 18·7% of suspected cases) eventually diagnosed as confirmed cases. 154 patients with missing data were excluded from the analysis. Exposure history (odds ratio [OR] 4·16, 95% CI 2·74–6·33; p<0·0001), fatigue (OR 1·56, 1·01–2·41; p=0·043), white blood cell count less than 4 × 10(9) per L (OR 2·44, 1·28–4·64; p=0·0066), lymphocyte count less than 0·8 × 10(9) per L (OR 1·82, 1·00–3·31; p=0·049), ground glass opacity (OR 1·95, 1·32–2·89; p=0·0009), and having both lungs affected (OR 1·54, 1·04–2·28; p=0·032) were independent risk factors for confirmed COVID-19. INTERPRETATION: The screening strategy was effective for confirming or excluding COVID-19 during the spread of this contagious disease. Relevant independent risk factors identified in this study might be helpful for early recognition of the disease. FUNDING: National Natural Science Foundation of China.\n",
      "Abstract Background Coronavirus disease 2019 (COVID-19) has been highly epidemic in China since January 2020. Rapid detection of the causative agent, severe acute respiratory coronavirus-2 (SARS-CoV-2), is very important due to its high rate of infectivity. This study aimed to clarify the epidemiology and clinical characteristics of COVID-19 outside of Hubei province, China, and to optimize screening strategies for COVID-19 in attempts to contain spread of the virus. Methods This retrospective study included all confirmed cases of COVID-19 in Hunan Provincial People's Hospital (Changsha, China) between January 22 and February 15, 2020. All cases were detected using a real-time reverse transcription polymerase chain reaction assay. The epidemiology and clinical characteristic of these cases were investigated according to outcome in attempts to optimize screening strategies for COVID-19. Results There were 24 confirmed cases of COVID-19 in the fever outpatient department of Hunan Provincial People's Hospital. Three patients were asymptomatic, and 3 exhibited mild and 3 moderate disease. There was a family cluster phenomenon. Conclusion Individuals with COVID-19 can be asymptomatic or exhibit mild manifestations of disease. Close monitoring and an optimized screening strategy for COVID-19 could help deter spread of the virus.\n",
      "OBJECTIVE: To examine the protective effects of appropriate personal protective equipment for frontline healthcare professionals who provided care for patients with coronavirus disease 2019 (covid-19). DESIGN: Cross sectional study. SETTING: Four hospitals in Wuhan, China. PARTICIPANTS: 420 healthcare professionals (116 doctors and 304 nurses) who were deployed to Wuhan by two affiliated hospitals of Sun Yat-sen University and Nanfang Hospital of Southern Medical University for 6-8 weeks from 24 January to 7 April 2020. These study participants were provided with appropriate personal protective equipment to deliver healthcare to patients admitted to hospital with covid-19 and were involved in aerosol generating procedures. 77 healthcare professionals with no exposure history to covid-19 and 80 patients who had recovered from covid-19 were recruited to verify the accuracy of antibody testing. MAIN OUTCOME MEASURES: Covid-19 related symptoms (fever, cough, and dyspnoea) and evidence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection, defined as a positive test for virus specific nucleic acids in nasopharyngeal swabs, or a positive test for IgM or IgG antibodies in the serum samples. RESULTS: The average age of study participants was 35.8 years and 68.1% (286/420) were women. These study participants worked 4-6 hour shifts for an average of 5.4 days a week; they worked an average of 16.2 hours each week in intensive care units. All 420 study participants had direct contact with patients with covid-19 and performed at least one aerosol generating procedure. During the deployment period in Wuhan, none of the study participants reported covid-19 related symptoms. When the participants returned home, they all tested negative for SARS-CoV-2 specific nucleic acids and IgM or IgG antibodies (95% confidence interval 0.0 to 0.7%). CONCLUSION: Before a safe and effective vaccine becomes available, healthcare professionals remain susceptible to covid-19. Despite being at high risk of exposure, study participants were appropriately protected and did not contract infection or develop protective immunity against SARS-CoV-2. Healthcare systems must give priority to the procurement and distribution of personal protective equipment, and provide adequate training to healthcare professionals in its use.\n",
      "Background: The outbreak of COVID-19 has led to international concern. We aimed to establish an effective screening strategy in Shanghai, China, to aid early identification of patients with COVID-19. Methods: We did a multicentre, observational cohort study in fever clinics of 25 hospitals in 16 districts of Shanghai. All patients visiting the clinics within the study period were included. A strategy for COVID-19 screening was presented and then suspected cases were monitored and analysed until they were confirmed as cases or excluded. Logistic regression was used to determine the risk factors of COVID-19. Findings: We enrolled patients visiting fever clinics from Jan 17 to Feb 16, 2020. Among 53â617 patients visiting fever clinics, 1004 (1·9%) were considered as suspected cases, with 188 (0·4% of all patients, 18·7% of suspected cases) eventually diagnosed as confirmed cases. 154 patients with missing data were excluded from the analysis. Exposure history (odds ratio [OR] 4·16, 95% CI 2·74-6·33; p<0·0001), fatigue (OR 1·56, 1·01-2·41; p=0·043), white blood cell count less than 4â×â109 per L (OR 2·44, 1·28-4·64; p=0·0066), lymphocyte count less than 0·8â×â109 per L (OR 1·82, 1·00-3·31; p=0·049), ground glass opacity (OR 1·95, 1·32-2·89; p=0·0009), and having both lungs affected (OR 1·54, 1·04-2·28; p=0·032) were independent risk factors for confirmed COVID-19. Interpretation: The screening strategy was effective for confirming or excluding COVID-19 during the spread of this contagious disease. Relevant independent risk factors identified in this study might be helpful for early recognition of the disease. Funding: National Natural Science Foundation of China.\n",
      "Background: Since December 2019, a number of patients infected with COVID-19 (SARS-CoV-2) have been identified in Wuhan, Hubei, China. As the epidemic has spread, similar cases have also been found in other parts of mainland China and abroad. The main reason for this spread is the highly contagious nature of the virus and the fact that children can also become infected during its incubation period. This has made the virus a substantial challenge for the outpatient triage staff of children's hospitals outside the epidemic area of the Hubei Province. It is very important for the preview and triage personnel to accurately grasp the epidemiology of the virus and identify children's symptoms in the fever clinic. Methods: We performed an analysis of our early preview and triage of suspected COVID-19 in 36 children presenting at fever clinics. Two specialists either excluded suspected cases or referred cases to the isolation ward for new nucleic acid testing. Results: All 14 children who were transferred to the isolation ward had a fever, and 71.43% of them had a cough. Their nucleic acid testing results were negative. The suspected cases and excluded suspected cases had similar epidemiology history as well as complete blood count results. With reference to the diagnostic criteria in existing pediatric guidelines, we have further improved the triage screening questionnaire for children with fever in our hospital. Conclusions: According to the situation in our city and hospital, an evaluation questionnaire that is suitable for use with children in our hospital has been formulated to achieve the goals of early detection, isolation, diagnosis, and treatment. We provided an important basis for the next step in developing accurate preview and triage screening standards and appropriate guidelines for pediatric patients.\n",
      "BACKGROUND: Since December 2019, over 80,000 patients with coronavirus disease 2019 (COVID-19) have been confirmed in China. With the increasing number of recovered patients, more attention should be paid to the follow-up of these patients. METHODS: In the study, 576 patients with COVID-19 discharged from hospital in Chongqing, China from January 24, 2020, to March 10, 2020 were evaluated by viral nucleic acid tests for severe acute respiratory syndrome coronavirus 2(SARS-CoV-2) to determine if they could be released from quarantine. Among the 576 patients, 61 patients (10.6%) had positive RT-PCR test results of SARS-CoV-2. We aimed to analyze the demographics, clinical characteristics and treatment of 61 patients. RESULTS: These positive patients were characterized by older age, chronic medical illness and mild conditions. 38 (62.3%) patients who were asymptomatic without abnormalities on chest radiographs were found in the positive with COVID-19. Also, they showed positive results of stool or sputum specimens with negative results of nasal and pharyngeal swab specimens. The median duration of positive result of SARS-CoV-2 was varied from 3 days to 35 days in the patients discharged from hospital with no family member infection. CONCLUSIONS: Multi-site screening of SARS-CoV-2 including nasal and pharyngeal swabs, stool and sputum specimens could be considered to improve the diagnosis, treatment and infection control in patients with COVID-19. Our findings provide the important information and clinical evidence for the improved management of patients recovered from COVID-19.\n",
      "INTRODUCTION: COVID-19 is a newly emerging life-threatening respiratory disease caused by a newly identified coronavirus SARS-CoV-2. METHODOLOGY: We included 28 COVID-19 patients admitted to Nantong Third Hospital from January 23 to February 26, 2020. SARS-CoV-2 infection was confirmed using real-time RT-PCR. The demographic, epidemiological, clinical, laboratory parameters were obtained from each patient. RESULTS: The vast majority (71.4%) of confirmed COVID-19 patients were brought in from outside of the city, and all others had contact history with these confirmed cases. The median age of patients was 50 years old and half had underlying diseases. The most common symptoms at the onset of illness were fever (96.4%), cough (67.9%), and chilly (28.6%), and 75.0% patients had two or more symptoms. Increased erythrocyte sedimentation rate, serum ferritin and C-reactive protein levels, and reduced absolute counts of total lymphocytes and T lymphocyte subsets were observed among the patients. The vast majority (85.7%) of patients showed bilateral or unilateral pneumonia, and three symptomatic patients and one asymptomatic case did not show abnormalities in their CT image. Among the 28 admitted patients, 24 were discharged as of February 26, 2020, with an average hospital stay of 14.96 (±4.27) days, which was not significantly associated with the interval between the onset of symptoms and admission. CONCLUSIONS: In the absence of specific antiviral drugs or a vaccine, quarantine or isolation is the most effective intervention strategy for preventing the spread of the virus. Adequate supportive medical care is crucial for good prognosis of COVID-19 patients.\n",
      "IMPORTANCE: A new outbreak of pneumonia caused by severe acute respiratory syndrome coronavirus 2 in Wuhan, China, is spreading rapidly around the globe. Limited information on control in community settings is available. OBJECTIVE: To detail measures enacted within a community to prevent the spread of coronavirus disease 2019 (COVID-19) and to evaluate the spread of COVID-19 associated with implementation of the program. DESIGN, SETTING, AND PARTICIPANTS: This case series study details the implementation of prevention measures in a specific community setting among community-dwelling individuals exposed to or at risk of COVID-19 in Haiyu, Shenzhen, China, from January 23 to April 10, 2020. EXPOSURES: Community containment strategies for tracking, quarantine, and management were strictly, cooperatively, and effectively implemented by a team that included a general practitioner, a community manager, and public safety bureau officials. MAIN OUTCOMES AND MEASURES: Number of locally acquired cases with indirect links to confirmed COVID-19 cases. Diagnosis with COVID-19 was confirmed when throat swab samples tested positive for severe acute respiratory syndrome coronavirus 2 on reverse transcription–polymerase chain reaction. RESULTS: Approximately 34 686 individuals live in Haiyu, including 2382 residents aged 65 years or older. Seven individuals with COVID-19 acquired outside the community (age, 20-70 years; 3 [42%] women) were moved from quarantine to a hospital for standard isolation treatment. A total of 20 people who were asymptomatic and who had had direct contact with these individuals were closely observed by health care workers at a nearby hotel. Additionally, 800 individuals considered to be at higher risk were moved from quarantine to home isolation for 14 days. There were no locally acquired cases of COVID-19 with indirect links reported in the Haiyu community from the time that the study began on January 23, 2020, to April 10, 2020. CONCLUSIONS AND RELEVANCE: These findings suggest that cooperation among the authorities of multiple sectors allowed for the implementation of preventive measures that were associated with limited community transmission.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: j59tm40d\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the transmission routes of coronavirus?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Summary Background Elucidation of the chain of disease transmission and identification of the source of coronavirus disease 2019 (COVID-19) infections are crucial for effective disease containment. We describe an epidemiological investigation that, with use of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) serological assays, established links between three clusters of COVID-19. Methods In Singapore, active case-finding and contact tracing were undertaken for all COVID-19 cases. Diagnosis for acute disease was confirmed with RT-PCR testing. When epidemiological information suggested that people might have been nodes of disease transmission but had recovered from illness, SARS-CoV-2 IgG serology testing was used to establish past infection. Findings Three clusters of COVID-19, comprising 28 locally transmitted cases, were identified in Singapore; these clusters were from two churches (Church A and Church B) and a family gathering. The clusters in Church A and Church B were linked by an individual from Church A (A2), who transmitted SARS-CoV-2 infection to the primary case from Church B (F1) at a family gathering they both attended on Jan 25, 2020. All cases were confirmed by RT-PCR testing because they had active disease, except for A2, who at the time of testing had recovered from their illness and tested negative. This individual was eventually diagnosed with past infection by serological testing. ELISA assays showed an optical density of more than 1·4 for SARS-CoV-2 nucleoprotein and receptor binding domain antigens in titres up to 1/400, and viral neutralisation was noted in titres up to 1/320. Interpretation Development and application of a serological assay has helped to establish connections between COVID-19 clusters in Singapore. Serological testing can have a crucial role in identifying convalescent cases or people with milder disease who might have been missed by other surveillance methods. Funding National Research Foundation (Singapore), National Natural Science Foundation (China), and National Medical Research Council (Singapore).\n",
      "BACKGROUND: Elucidation of the chain of disease transmission and identification of the source of coronavirus disease 2019 (COVID-19) infections are crucial for effective disease containment. We describe an epidemiological investigation that, with use of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) serological assays, established links between three clusters of COVID-19. METHODS: In Singapore, active case-finding and contact tracing were undertaken for all COVID-19 cases. Diagnosis for acute disease was confirmed with RT-PCR testing. When epidemiological information suggested that people might have been nodes of disease transmission but had recovered from illness, SARS-CoV-2 IgG serology testing was used to establish past infection. FINDINGS: Three clusters of COVID-19, comprising 28 locally transmitted cases, were identified in Singapore; these clusters were from two churches (Church A and Church B) and a family gathering. The clusters in Church A and Church B were linked by an individual from Church A (A2), who transmitted SARS-CoV-2 infection to the primary case from Church B (F1) at a family gathering they both attended on Jan 25, 2020. All cases were confirmed by RT-PCR testing because they had active disease, except for A2, who at the time of testing had recovered from their illness and tested negative. This individual was eventually diagnosed with past infection by serological testing. ELISA assays showed an optical density of more than 1·4 for SARS-CoV-2 nucleoprotein and receptor binding domain antigens in titres up to 1/400, and viral neutralisation was noted in titres up to 1/320. INTERPRETATION: Development and application of a serological assay has helped to establish connections between COVID-19 clusters in Singapore. Serological testing can have a crucial role in identifying convalescent cases or people with milder disease who might have been missed by other surveillance methods. FUNDING: National Research Foundation (Singapore), National Natural Science Foundation (China), and National Medical Research Council (Singapore).\n",
      "BACKGROUND: Three clusters of coronavirus disease 2019 (COVID-19) linked to a tour group from China, a company conference, and a church were identified in Singapore in February, 2020. METHODS: We gathered epidemiological and clinical data from individuals with confirmed COVID-19, via interviews and inpatient medical records, and we did field investigations to assess interactions and possible modes of transmission of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Open source reports were obtained for overseas cases. We reported the median (IQR) incubation period of SARS-CoV-2. FINDINGS: As of Feb 15, 2020, 36 cases of COVID-19 were linked epidemiologically to the first three clusters of circumscribed local transmission in Singapore. 425 close contacts were quarantined. Direct or prolonged close contact was reported among affected individuals, although indirect transmission (eg, via fomites and shared food) could not be excluded. The median incubation period of SARS-CoV-2 was 4 days (IQR 3-6). The serial interval between transmission pairs ranged between 3 days and 8 days. INTERPRETATION: SARS-CoV-2 is transmissible in community settings, and local clusters of COVID-19 are expected in countries with high travel volume from China before the lockdown of Wuhan and institution of travel restrictions. Enhanced surveillance and contact tracing is essential to minimise the risk of widespread transmission in the community. FUNDING: None.\n",
      "BACKGROUND: Three clusters of coronavirus disease 2019 (COVID-19) linked to a tour group from China, a company conference, and a church were identified in Singapore in February, 2020. METHODS: We gathered epidemiological and clinical data from individuals with confirmed COVID-19, via interviews and inpatient medical records, and we did field investigations to assess interactions and possible modes of transmission of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Open source reports were obtained for overseas cases. We reported the median (IQR) incubation period of SARS-CoV-2. FINDINGS: As of Feb 15, 2020, 36 cases of COVID-19 were linked epidemiologically to the first three clusters of circumscribed local transmission in Singapore. 425 close contacts were quarantined. Direct or prolonged close contact was reported among affected individuals, although indirect transmission (eg, via fomites and shared food) could not be excluded. The median incubation period of SARS-CoV-2 was 4 days (IQR 3–6). The serial interval between transmission pairs ranged between 3 days and 8 days. INTERPRETATION: SARS-CoV-2 is transmissible in community settings, and local clusters of COVID-19 are expected in countries with high travel volume from China before the lockdown of Wuhan and institution of travel restrictions. Enhanced surveillance and contact tracing is essential to minimise the risk of widespread transmission in the community. FUNDING: None.\n",
      "BACKGROUND: Serological tests are crucial tools for assessments of SARS-CoV-2 exposure, infection and potential immunity. Their appropriate use and interpretation require accurate assay performance data. METHOD: We conducted an evaluation of 10 lateral flow assays (LFAs) and two ELISAs to detect anti-SARS-CoV-2 antibodies. The specimen set comprised 128 plasma or serum samples from 79 symptomatic SARS-CoV-2 RT-PCR-positive individuals; 108 pre-COVID-19 negative controls; and 52 recent samples from individuals who underwent respiratory viral testing but were not diagnosed with Coronavirus Disease 2019 (COVID-19). Samples were blinded and LFA results were interpreted by two independent readers, using a standardized intensity scoring system. RESULTS: Among specimens from SARS-CoV-2 RT-PCR-positive individuals, the percent seropositive increased with time interval, peaking at 81.8–100.0% in samples taken >20 days after symptom onset. Test specificity ranged from 84.3–100.0% in pre-COVID-19 specimens. Specificity was higher when weak LFA bands were considered negative, but this decreased sensitivity. IgM detection was more variable than IgG, and detection was highest when IgM and IgG results were combined. Agreement between ELISAs and LFAs ranged from 75.7–94.8%. No consistent cross-reactivity was observed. CONCLUSION: Our evaluation showed heterogeneous assay performance. Reader training is key to reliable LFA performance, and can be tailored for survey goals. Informed use of serology will require evaluations covering the full spectrum of SARS-CoV-2 infections, from asymptomatic and mild infection to severe disease, and later convalescence. Well-designed studies to elucidate the mechanisms and serological correlates of protective immunity will be crucial to guide rational clinical and public health policies.\n",
      "BACKGROUND Rapid identification of COVID-19 cases, which is crucial to outbreak containment efforts, is challenging due to the lack of pathognomonic symptoms and in settings with limited capacity for specialized nucleic acid-based reverse transcription polymerase chain reaction (PCR) testing. METHODS This retrospective case-control study involves subjects (7 to 98 years) presenting at the designated national outbreak screening centre and tertiary care hospital in Singapore for SARS-CoV-2 testing from January 26 to February 16, 2020. COVID-19 status was confirmed by PCR testing of sputum, nasopharyngeal swabs or throat swabs. Demographic, clinical, laboratory and exposure-risk variables ascertainable at presentation were analyzed to develop an algorithm for estimating the risk of COVID-19. Model development used Akaike's information criterion in a stepwise fashion to build logistic regression models, which were then translated into prediction scores. Performance was measured using receiver operating characteristics curves, adjusting for over-confidence using leave-out-one cross validation. RESULTS The study population included 788 subjects, of whom 54 (6.9%) were SARS-CoV-2 positive and 734 (93.1%) were SARS-CoV-2 negative. The median age was 34 years and 407 (51.7%) were female. Using leave-out-one cross validation, all the models incorporating clinical tests (Models 1, 2 and 3) performed well with areas under the receiver operating characteristics curve (AUC) of 0.91, 0.88 and 0.88 respectively. In comparison, Model 4 had an AUC of 0.65. CONCLUSIONS Rapidly ascertainable clinical and laboratory data could identify individuals at high risk of COVID-19 and enable prioritization of PCR-testing and containment efforts. Basic laboratory test results were crucial to prediction models.\n",
      "Background The outbreak of the recently emerged novel corona virus disease 2019 (COVID-19) poses a challenge for public health laboratories. We aimed to evaluate the diagnostic value of serological assay for SARS-CoV-2. Methods A newly-developed ELISA assay for IgM and IgG antibodies against N protein of SARS-CoV-2 were used to screen the serums of 238 admitted hospital patients with confirmed or suspected SARS-CoV-2 infection from February 6 to February 14, 2020. SARS-CoV-2 RNA was detected by real time RT-PCR on pharyngeal swab specimens. Findings Of the 238 patients, 194 (81.5%) were detected to be antibody (IgM and/or IgG) positive, which was significantly higher than the positive rate of viral RNA (64.3%). There was no difference in the positive rate of antibody between the confirmed patients (83.0%, 127/153) and the suspected patients (78.8%, 67/85) whose nucleic acid tests were negative. After the patients were defined to the different stages of disease based on the day when the test samples were collected, the analysis results showed that the antibody positive rates were very low in the first five days after initial onset of symptoms, and then rapidly increased as the disease progressed. After 10 days, the antibody positive rates jumped to above 80% from less than 50%. On the contrary, the positive rates of viral RNA kept above 60% in the first 11 days after initial onset of symptoms, and then rapidly decreased. In addition, half of the suspected patients with symptoms for 6-10 days were detected to be antibody positive. Interpretation The suspected patients were most likely infected by SARS-CoV-2. Before the 11th day after initial onset of symptoms, nucleic acid test is important for confirmation of viral infection. The combination of serological assay can greatly improve the diagnostic efficacy. After that, the diagnosis for viral infection should be majorly dependent on serological assay. Keywords. SARS-CoV-2; diagnosis; serological assay; nucleic acid test\n",
      "BACKGROUND: Rapid identification of COVID-19 cases, which is crucial to outbreak containment efforts, is challenging due to the lack of pathognomonic symptoms and in settings with limited capacity for specialized nucleic acid-based reverse transcription polymerase chain reaction (PCR) testing. METHODS: This retrospective case-control study involves subjects (7 to 98 years) presenting at the designated national outbreak screening centre and tertiary care hospital in Singapore for SARS-CoV-2 testing from January 26 to February 16, 2020. COVID-19 status was confirmed by PCR testing of sputum, nasopharyngeal swabs or throat swabs. Demographic, clinical, laboratory and exposure-risk variables ascertainable at presentation were analyzed to develop an algorithm for estimating the risk of COVID-19. Model development used Akaike's information criterion in a stepwise fashion to build logistic regression models, which were then translated into prediction scores. Performance was measured using receiver operating characteristics curves, adjusting for over-confidence using leave-out-one cross validation. RESULTS: The study population included 788 subjects, of whom 54 (6.9%) were SARS-CoV-2 positive and 734 (93.1%) were SARS-CoV-2 negative. The median age was 34 years and 407 (51.7%) were female. Using leave-out-one cross validation, all the models incorporating clinical tests (Models 1, 2 and 3) performed well with areas under the receiver operating characteristics curve (AUC) of 0.91, 0.88 and 0.88 respectively. In comparison, Model 4 had an AUC of 0.65. CONCLUSIONS: Rapidly ascertainable clinical and laboratory data could identify individuals at high risk of COVID-19 and enable prioritization of PCR-testing and containment efforts. Basic laboratory test results were crucial to prediction models.\n",
      "Abstract Background Since December 2019, a new outbreak of the coronavirus disease 2019 (COVID-19) in Wuhan (Hubei, China) and rapidly spread throughout China, however, confirmed cases are still increasing worldwide. Objectives To investigate the epidemiological history and initial clinical characteristics of 10 patients with family aggregation of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection in Western Chongqing, China. Study design Ten patients positive for SARS-CoV-2 nucleic acid detection by real time Reverse Transcription-Polymerase Chain Reaction (RT-PCR), were collected from The People's Hospital of Dazu District, Chongqing. Epidemiological data and laboratory and imaging results were collected on the first day of admission, and analyzed based on the Diagnosis and Treatment Guideline for COVID-19 (5th edition, China). Results Of the 10 cases, case A had a history of a temporary stay in Wuhan and transmitted the virus to the others through family gathering, living together, and sharing vehicles. The average age was 56.5 years (± 11.16), six patients were males, and the incubation period was 2–14 days. Dry cough was the main symptom, followed by fever and fatigue. Most patients were clinically classified as ordinary-type, with three cases being severe-type. Chest computed tomography results were nonspecific, mainly with ground-glass attenuation and/or shadow images. Extensive lesion distribution was seen in severe cases. CD4+ lymphocyte counts were 61, 180, and 348 cells/uL in severe-type patients, respectively. Notably, viral nucleic acid values in nasopharyngeal swabs were lower (19, 25, and 26) than those of ordinary-type patients, suggesting a higher viral load. Neutrophil-lymphocyte ratio (NLR) was also higher in severe-type patients Conclusions Initial examination results of lower CD4+ lymphocyte counts and RT-PCR-CT values coupled with higher NLR may indicate the severity of COVID-19 infection for these family clusters.\n",
      "SARS-Cov-2 (severe acute respiratory disease coronavirus 2), which causes Coronavirus Disease 2019 (COVID19) was first detected in China in late 2019 and has since then caused a global pandemic. While molecular assays to directly detect the viral genetic material are available for the diagnosis of acute infection, we currently lack serological assays suitable to specifically detect SARS-CoV-2 antibodies. Here we describe serological enzyme-linked immunosorbent assays (ELISA) that we developed using recombinant antigens derived from the spike protein of SARS-CoV-2. Using negative control samples representing pre-COVID 19 background immunity in the general adult population as well as samples from COVID19 patients, we demonstrate that these assays are sensitive and specific, allowing for screening and identification of COVID19 seroconverters using human plasma/serum as early as two days post COVID19 symptoms onset. Importantly, these assays do not require handling of infectious virus, can be adjusted to detect different antibody types and are amendable to scaling. Such serological assays are of critical importance to determine seroprevalence in a given population, define previous exposure and identify highly reactive human donors for the generation of convalescent serum as therapeutic. Sensitive and specific identification of coronavirus SARS-Cov-2 antibody titers may, in the future, also support screening of health care workers to identify those who are already immune and can be deployed to care for infected patients minimizing the risk of viral spread to colleagues and other patients.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: rjqanuk9\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the transmission routes of coronavirus?\n",
      "how long can the coronavirus live outside the body\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "This article analyzes the available evidence to address airborne, aerosol transmission of the SARS‐CoV‐2. We review and present three lines of evidence: case reports of transmission for asymptomatic individuals in association with studies that show that normal breathing and talking produce predominantly small droplets of the size that are subject to aerosol transport; limited empirical data that have recorded aerosolized SARS‐CoV‐2 particles that remain suspended in the air for hours and are subject to transport over distances including outside of rooms and intrabuilding, and the broader literature that further supports the importance of aerosol transmission of infectious diseases. The weight of the available evidence warrants immediate attention to address the significance of aerosols and implications for public health protection.\n",
      "This article analyzes the available evidence to address airborne, aerosol transmission of the SARS-CoV-2. We review and present three lines of evidence: case reports of transmission for asymptomatic individuals in association with studies that show that normal breathing and talking produce predominantly small droplets of the size that are subject to aerosol transport; limited empirical data that have recorded aerosolized SARS-CoV-2 particles that remain suspended in the air for hours and are subject to transport over distances including outside of rooms and intrabuilding, and the broader literature that further supports the importance of aerosol transmission of infectious diseases. The weight of the available evidence warrants immediate attention to address the significance of aerosols and implications for public health protection.\n",
      "The transmission behaviour of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is still being defined. It is likely that it is transmitted predominantly by droplets and direct contact and it is possible that there is at least opportunistic airborne transmission. In order to protect healthcare staff adequately is necessary that we establish whether aerosol generating procedures (AGPs) increase the risk of transmission of SARS-CoV-2. Where we do not have evidence relating to SARS-CoV-2, guidelines for safely conducting these procedures should consider what risk procedures would have of transmitting related pathogens. Currently there is very little evidence detailing the transmission of SARS-CoV-2 associated with any specific procedures. Regarding aerosol generating procedures and respiratory pathogens in general, there is still a large knowledge gap that will leave clinicians unsure what risk they are putting themselves in when offering these procedures. This review aimed to summarise the evidence (and gaps in evidence) around AGPs and SARS-CoV-2.\n",
      "The transmission behaviour of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is still being defined. It is likely that it is transmitted predominantly by droplets and direct contact and it is possible that there is at least opportunistic airborne transmission. In order to protect healthcare staff adequately is necessary that we establish whether aerosol generating procedures (AGPs) increase the risk of transmission of SARS-CoV-2. Where we do not have evidence relating to SARS-CoV-2, guidelines for safely conducting these procedures should consider what risk procedures would have of transmitting related pathogens. Currently there is very little evidence detailing the transmission of SARS-CoV-2 associated with any specific procedures. Regarding aerosol generating procedures and respiratory pathogens in general, there is still a large knowledge gap that will leave clinicians unsure what risk they are putting themselves in when offering these procedures. This review aimed to summarise the evidence (and gaps in evidence) around AGPs and SARS-CoV-2.\n",
      "Health threat from SARS-CoV-2 airborne infection has become a public emergency of international concern. During the ongoing coronavirus pandemic, people have been advised by the Centers for Disease Control and Prevention to maintain social distancing of at least 2 m to limit the risk of exposure to the coronavirus. We carry out a physics modeling study for SARS-CoV-2 transport in air. We show that if aerosols and droplets follow semi-ballistic emission trajectories, then their horizontal range is proportional to the particle's diameter. For standard ambient temperature and pressure conditions, the horizontal range of these aerosols remains safely below 2 m. We also show that aerosols and droplets can remain suspended for hours in the air, providing a health threat of airborne infection. The latter argues in favor of implementing additional precautions to the recommended 2~m social distancing, e.g. wearing a face mask when we are out in public.\n",
      "We provide research findings on the physics of aerosol dispersion relevant to the hypothesized aerosol transmission of SARS-CoV-2. We utilize physics-based modeling at different levels of complexity, and literature on coronaviruses, to investigate the possibility of airborne transmission. The previous literature, our 0D-3D simulations by various physics-based models, and theoretical calculations, indicate that the typical size range of speech and cough originated droplets (d<20microns) allows lingering in the air for O(1h) so that they could be inhaled. Consistent with the previous literature, numerical evidence on the rapid drying process of even large droplets, up to sizes O(100microns), into droplet nuclei/aerosols is provided. Based on the literature and the public media sources, we provide evidence that the infected individuals could have been exposed to aerosols/droplet nuclei by inhaling them in significant numbers e.g. O(100). By 3D computational fluid dynamics (CFD) simulations, we give examples on the transport and dilution of aerosols (d<20microns) over distances O(10m) in generic environments. We study susceptible and infected individuals in generic public places by Monte-Carlo modeling. The model accounts for the locally varying aerosol concentration levels which the susceptible accumulate via inhalation. The introduced concept, 'exposure time' to virus containing aerosols is proposed to complement the traditional 'safety distance' thinking. We show that the exposure time to inhale O(100) aerosols could range from O(1s) to O(1min) or even to O(1h) depending on the situation. The Monte Carlo analysis provides clear quantitative insight to the exposure time in different public indoor environments.\n",
      "Objectives While COVID-19 is known to be spread by respiratory droplets (which travel <2m horizontally), much less is known about its transmission via aerosols, which can become airborne and be widely distributed throughout room spaces. In order to quantify the risk posed by COVID-19 infectors exhaling respiratory aerosols in enclosed spaces, we undertook a computer modelling study to simulate transmission in an office building. Methods Respiratory droplet data from four published datasets were analysed to quantify the number and volume of droplets <100m diameter produced by a typical cough and speaking event (i.e. counting from 1 to 100). This was used in a stochastic model to simulate (10,000 simulations) the number of respiratory particles, originating from a COVID-19 infector, that would be inhaled in one hour by a susceptible individual practicing socially distancing in a 5 x 5 x 2.75m office space. Several scenarios were simulated that mimicked the presence of both symptomatic and asymptomatic COVID-19 infectors. Results On average, each cough and speaking event produced similar numbers of droplets <100 m diameter (median range = 955 - 1010). Computer simulations (at ventilation rate = 2AC/h) revealed that sharing the office space with a symptomatic COVID-19 infector (4 coughs per hour) for one hour resulted in the inhalation of 187.3 (median value) respiratory droplets, whereas sharing with an asymptomatic COVID-19 positive person (10 speaking events per hour) resulted in the inhalation of 482.9 droplets. Increasing the ventilation rate resulted in only modest reductions in particle numbers inhaled. Conclusions Given that live SARS-CoV-2 virions are known to be shed in high concentrations from the nasal cavity of both symptomatic and asymptomatic COVID-19 patients, the results suggest that individuals who share enclosed spaces with an infector may be at risk of contracting COVID-19 by the aerosol route, even when practicing social distancing.\n",
      "The practice of social distancing and wearing masks has been popular worldwide in combating the contraction of COVID-19. Undeniably, although such practices help control the COVID-19 pandemic to a greater extent, the complete control of viral-laden droplet and aerosol transmission by such practices is poorly understood. This review paper intends to outline the literature concerning the transmission of viral-laden droplets and aerosols in different environmental settings and demonstrates the behavior of droplets and aerosols resulted from a cough-jet of an infected person in various confined spaces. The case studies that have come out in different countries have, with prima facie evidence, manifested that the airborne transmission plays a profound role in contracting susceptible hosts. Interestingly, the nosocomial transmission by airborne SARS-CoV-2 viral-laden aerosols in healthcare facilities may be plausible. Hence, clearly defined, science-based administrative, clinical, and physical measures are of paramount importance to eradicate the COVID-19 pandemic from the world.\n",
      "Abstract We provide research findings on the physics of aerosol and droplet dispersion relevant to the hypothesized aerosol transmission of SARS-CoV-2 during the current pandemic. We utilize physics-based modeling at different levels of complexity, along with previous literature on coronaviruses, to investigate the possibility of airborne transmission. The previous literature, our 0D-3D simulations by various physics-based models, and theoretical calculations, indicate that the typical size range of speech and cough originated droplets ( d ⩽ 20 μ m ) allows lingering in the air for O ( 1 h ) so that they could be inhaled. Consistent with the previous literature, numerical evidence on the rapid drying process of even large droplets, up to sizes O ( 100 μ m ) , into droplet nuclei/aerosols is provided. Based on the literature and the public media sources, we provide evidence that the individuals, who have been tested positive on COVID-19, could have been exposed to aerosols/droplet nuclei by inhaling them in significant numbers e.g. O ( 100 ) . By 3D scale-resolving computational fluid dynamics (CFD) simulations, we give various examples on the transport and dilution of aerosols ( d ⩽ 20 μ m ) over distances O ( 10 m ) in generic environments. We study susceptible and infected individuals in generic public places by Monte-Carlo modelling. The developed model takes into account the locally varying aerosol concentration levels which the susceptible accumulate via inhalation. The introduced concept, ’exposure time’ to virus containing aerosols is proposed to complement the traditional ’safety distance’ thinking. We show that the exposure time to inhale O ( 100 ) aerosols could range from O ( 1 s ) to O ( 1 min ) or even to O ( 1 h ) depending on the situation. The Monte-Carlo simulations, along with the theory, provide clear quantitative insight to the exposure time in different public indoor environments.\n",
      "The practice of social distancing and wearing masks has been popular worldwide in combating the contraction of COVID-19. Undeniably, although such practices help control the COVID-19 pandemic to a greater extent, the complete control of virus-laden droplet and aerosol transmission by such practices is poorly understood. This review paper intends to outline the literature concerning the transmission of virus-laden droplets and aerosols in different environmental settings and demonstrates the behavior of droplets and aerosols resulted from a cough-jet of an infected person in various confined spaces. The case studies that have come out in different countries have, with prima facie evidence, manifested that the airborne transmission plays a profound role in contracting susceptible hosts. The infection propensities in confined spaces (airplane, passenger car, and healthcare center) by the transmission of droplets and aerosols under varying ventilation conditions were discussed. Interestingly, the nosocomial transmission by airborne SARS-CoV-2 virus-laden aerosols in healthcare facilities may be plausible. Hence, clearly defined, science-based administrative, clinical, and physical measures are of paramount importance to eradicate the COVID-19 pandemic from the world.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: l54hk7go\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Importance: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which causes coronavirus disease 2019 (COVID-19) has reached a pandemic level. Coronaviruses are known to affect the cardiovascular system. We review the basics of coronaviruses, with a focus on COVID-19, along with their effects on the cardiovascular system. Observations: Coronavirus disease 2019 can cause a viral pneumonia with additional extrapulmonary manifestations and complications. A large proportion of patients have underlying cardiovascular disease and/or cardiac risk factors. Factors associated with mortality include male sex, advanced age, and presence of comorbidities including hypertension, diabetes mellitus, cardiovascular diseases, and cerebrovascular diseases. Acute cardiac injury determined by elevated high-sensitivity troponin levels is commonly observed in severe cases and is strongly associated with mortality. Acute respiratory distress syndrome is also strongly associated with mortality. Conclusions and Relevance: Coronavirus disease 2019 is associated with a high inflammatory burden that can induce vascular inflammation, myocarditis, and cardiac arrhythmias. Extensive efforts are underway to find specific vaccines and antivirals against SARS-CoV-2. Meanwhile, cardiovascular risk factors and conditions should be judiciously controlled per evidence-based guidelines.\n",
      "Importance Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which causes coronavirus disease 2019 (COVID-19) has reached a pandemic level. Coronaviruses are known to affect the cardiovascular system. We review the basics of coronaviruses, with a focus on COVID-19, along with their effects on the cardiovascular system. Observations Coronavirus disease 2019 can cause a viral pneumonia with additional extrapulmonary manifestations and complications. A large proportion of patients have underlying cardiovascular disease and/or cardiac risk factors. Factors associated with mortality include male sex, advanced age, and presence of comorbidities including hypertension, diabetes mellitus, cardiovascular diseases, and cerebrovascular diseases. Acute cardiac injury determined by elevated high-sensitivity troponin levels is commonly observed in severe cases and is strongly associated with mortality. Acute respiratory distress syndrome is also strongly associated with mortality. Conclusions and Relevance Coronavirus disease 2019 is associated with a high inflammatory burden that can induce vascular inflammation, myocarditis, and cardiac arrhythmias. Extensive efforts are underway to find specific vaccines and antivirals against SARS-CoV-2. Meanwhile, cardiovascular risk factors and conditions should be judiciously controlled per evidence-based guidelines.\n",
      "INTRODUCTION: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the cause of coronavirus disease 2019 (COVID-19) which recently has been characterized as a pandemic by the World Health Organization (WHO) having killed almost 250,000 people worldwide as of May 4th 2020. Despite the fact that SARS-CoV-2 seems to predominantly affect the respiratory system leading to pneumonia and acute respiratory distress syndrome, it is now evident that it may also affect the cardiovascular system in multiple ways. EVIDENCE ACQUISITION: The current paper is a review of the most recent literature regarding SARS-CoV-2 infection and its associated main cardiovascular clinical manifestations. EVIDENCE SYNTHESIS: Cardiovascular disease represents a prevalent underlying comorbidity associated with increased mortality rates among COVID-19 affected individuals. In addition, various cardiovascular manifestations have been linked to the viral insult, including among others acute coronary syndromes, myocarditis, acute heart failure, cardiac injury, arrhythmias and acute pulmonary embolism. CONCLUSIONS: Further studies are required in order to establish the complicated association between SARS-CoV-2 infection and its effects on the cardiovascular system. Our knowledge regarding diagnostic approaches, therapeutic management and preventive measures is constantly enriched throughout an abundance of ongoing research in the respective fields.\n",
      "INTRODUCTION Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the cause of coronavirus disease 2019 (COVID-19) which recently has been characterized as a pandemic by the World Health Organization (WHO) having killed almost 250,000 people worldwide as of May 4th 2020. Despite the fact that SARS-CoV-2 seems to predominantly affect the respiratory system leading to pneumonia and acute respiratory distress syndrome, it is now evident that it may also affect the cardiovascular system in multiple ways. EVIDENCE ACQUISITION The current paper is a review of the most recent literature regarding SARS-CoV-2 infection and its associated main cardiovascular clinical manifestations. EVIDENCE SYNTHESIS Cardiovascular disease represents a prevalent underlying comorbidity associated with increased mortality rates among COVID-19 affected individuals. In addition, various cardiovascular manifestations have been linked to the viral insult, including among others acute coronary syndromes, myocarditis, acute heart failure, cardiac injury, arrhythmias and acute pulmonary embolism. CONCLUSIONS Further studies are required in order to establish the complicated association between SARS-CoV-2 infection and its effects on the cardiovascular system. Our knowledge regarding diagnostic approaches, therapeutic management and preventive measures is constantly enriched throughout an abundance of ongoing research in the respective fields.\n",
      "Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infects host cells following binding with the cell surface ACE2 receptors, thereby leading to coronavirus disease 2019 (COVID-19). SARS-CoV-2 causes viral pneumonia with additional extrapulmonary manifestations and major complications, including acute myocardial injury, arrhythmia, and shock mainly in elderly patients. Furthermore, patients with existing cardiovascular comorbidities, such as hypertension and coronary heart disease, have a worse clinical outcome following contraction of the viral illness. A striking feature of COVID-19 pandemics is the high incidence of fatalities in advanced aged patients: this might be due to the prevalence of frailty and cardiovascular disease increase with age due to endothelial dysfunction and loss of endogenous cardioprotective mechanisms. Although experimental evidence on this topic is still at its infancy, the aim of this position paper is to hypothesize and discuss more suggestive cellular and molecular mechanisms whereby SARS-CoV-2 may lead to detrimental consequences to the cardiovascular system. We will focus on aging, cytokine storm, NLRP3/inflammasome, hypoxemia, and air pollution, which is an emerging cardiovascular risk factor associated with rapid urbanization and globalization. We will finally discuss the impact of clinically available CV drugs on the clinical course of COVID-19 patients. Understanding the role played by SARS-CoV2 on the CV system is indeed mandatory to get further insights into COVID-19 pathogenesis and to design a therapeutic strategy of cardio-protection for frail patients.\n",
      "Abstract Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which causes coronavirus disease 2019 (COVID-19) has reached a pandemic level. SARS-CoV-2 infects host cells through ACE2 receptors, leading to COVID-19-related pneumonia. The rapid increase in confirmed cases makes the prevention and control of COVID-19 extremely serious. Real-time reverse transcription-PCR (RT-PCR) assays remain the molecular test of choice for the etiologic diagnosis of SARS-CoV-2 infection while radiographic findings (chest computed tomography [CT]) and antibody-based techniques are being introduced as supplemental tools. Novel virus also cause chronic damage to the cardiovascular system, and attention should be given to cardiovascular protection during treatment for COVID-19. Acute cardiac injury determined by elevated high-sensitivity troponin levels is commonly observed in severe cases and is strongly associated with mortality. This review suggests that cardiovascular comorbidities are common in patients with COVID-19 and such patients are at higher risk of morbidity and mortality. The continuation of clinically indicated ACE inhibitor and ARB medications is recommended in COVID-19. We review the basics of coronaviruses, novel molecular targets for the coronaviruses with a focus on COVID-19, along with their effects on the cardiovascular system.\n",
      "Coronavirus disease 2019 (COVID-19), caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), is now a global pandemic with the highest number of affected individuals in the modern era. Not only is the infection inflicting significant morbidity and mortality, but there has also been a significant strain to the health care system and the economy. COVID-19 typically presents as viral pneumonia, occasionally leading to acute respiratory distress syndrome (ARDS) and death. However, emerging evidence suggests that it has a significant impact on the cardiovascular (CV) system by direct myocardial damage, severe systemic inflammatory response, hypoxia, right heart strain secondary to ARDS and lung injury, and plaque rupture secondary to inflammation. Primary cardiac manifestations include acute myocarditis, myocardial infarction, arrhythmia, and abnormal clotting. Several consensus documents have been released to help manage CV disease during this pandemic. In this review, we summarize key cardiac manifestations, their management, and future implications.\n",
      "BACKGROUND: The coronavirus disease of 2019 (COVID-19) is caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). While systemic inflammation and pulmonary complications can result in significant morbidity and mortality, cardiovascular complications may also occur. OBJECTIVE: This brief report evaluates cardiovascular complications in the setting of COVID-19 infection. DISCUSSION: The current COVID-19 pandemic has resulted in over one million infected worldwide and thousands of death. The virus binds and enters through angiotensin-converting enzyme 2 (ACE2). COVID-19 can result in systemic inflammation, multiorgan dysfunction, and critical illness. The cardiovascular system is also affected, with complications including myocardial injury, myocarditis, acute myocardial infarction, heart failure, dysrhythmias, and venous thromboembolic events. Current therapies for COVID-19 may interact with cardiovascular medications. CONCLUSIONS: Emergency clinicians should be aware of these cardiovascular complications when evaluating and managing the patient with COVID-19.\n",
      "PURPOSE OF THE REVIEW: Coronavirus disease 2019 (COVID-19), a new infectious disease caused by severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2), has reached a pandemic status. Although SARSCoV-2 causes primarily respiratory problems, concurrent cardiac injury cannot be ignored since it may be an independent predictor for adverse outcomes. To resolve these issues, we aim to summarize the prevalence and its underlying mechanisms of acute cardiac injury in the setting of SARS-CoV-2 infection. RECENT FINDINGS: The main clinical manifestation of SARS-CoV-2 infection is pneumonia, cardiovascular complications have also been identified in the earliest reported cases from Wuhan, the epicenter of the outbreak. Given the SARS-CoV-2 likely uses the angiotensin-converting enzyme-2 (ACE2) receptors as its host receptor, ACE2-related signaling pathways may play a key role in mediating myocardial injury. SUMMARY: SARS-CoV-2 infection related acute cardiac injury cannot be ignored, and its underlying mechanisms remain speculated. We would suggest that health professionals investigate cardiac function as part of the routine care.\n",
      "The recent outbreak of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 has been declared a public health emergency of international concern. COVID-19 may present as acute respiratory distress syndrome in severe cases, and patients with pre-existing cardiovascular comorbidities are reported to be the most vulnerable. Notably, acute myocardial injury, determined by elevated high-sensitivity troponin levels, is commonly observed in severe cases, and is strongly associated with mortality. Therefore, understanding the effects of COVID-19 on the cardiovascular system is essential for providing comprehensive medical care for critically ill patients. In this review, we summarize the rapidly evolving data and highlight the cardiovascular considerations related to COVID-19.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: rzr8qjw8\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "how long can the coronavirus live outside the body\n",
      "what are best practices in hospitals and at home in maintaining quarantine?\n",
      "what are the initial symptoms of Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Coronaviruses are zoonotic viruses and six species of Coronaviruses are known to cause human disease such as cause common cold, severe acute respiratory syndrome and the Middle East Respiratory Syndrome. In January 2020, scientists in Wuhan, China isolated a novel coronavirus (SARS-CoV-2), responsible for an outbreak of unknown pneumonia that had not been previously reported among humans. This virus spreads from person to person, through respiratory droplets, close contact, and by touching surfaces or objects contaminated by the virus. The incubation period varies between 2 days and 14 days. Symptoms usually include fever, cough, difficulty in breathing, pneumonia, severe acute respiratory syndrome. Older age and co-morbid conditions increase the fatality. Any person with a history of travel to and from COVID-19 affected countries in the past 14 days or any person who has had close contact with a laboratory confirmed COVID-19 are suspect cases and needs evaluation. Currently no vaccine is available and treatment is mainly supportive. Measures at workplace should include- avoiding non-essential travel, identifying and isolating sick employees at the earliest, hand hygiene, respiratory hygiene, environmental hygiene and social distancing.\n",
      "The emergence and rapid spread of novel coronavirus disease (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) as a potentially fatal disease is swiftly evolving public health crises worldwide. The origin of SARS-CoV-2 infection was first reported in people exposed to wet animal market in Wuhan City, China in December 2019. It was suggested that the infection is likely to be of zoonotic origin and transmitted to human through yet unknown intermediary. As of (22/05/2020), there are around 4,995,996 confirmed cases reported by WHO with 327,821 deaths. SARS-CoV-2 infection is transmitted via inhalation or direct contact of infected people's droplets. It has an incubation period ranging from 2 to 14 days or more. The rate of spread of SARS-CoV-2 is more than partially resembled coronavirus (SARS-CoV and MERS). The symptoms are similar to influenza like, breathlessness, sore throat and fatigue therefore, infected person is isolated and administrated with effective treatments. Infection is mild in most but in elderly (>50 years) and those with cardiac and respiratory disorder, it may progress to pneumonia, acute respiratory distress syndrome, and multi organ failure. People with strong immunity or those developed herd immunity are asymptomatic. Fatality rate ranges to 3-4% on case basis. Diagnosis of SARS-CoV-2 is recommended in respiratory secretions by special molecular tests like PCR, chest scan and common laboratory diagnosis. Currently, the existing treatment is essentially supportive and role of antiviral agents is yet to be established as there is no vaccination or therapy available. This review focuses on epidemiology, symptoms, transmission, pathogenesis, ongoing available treatments and future perspectives of SARS-CoV-2.\n",
      "There is a new public health crises threatening the world with the emergence and spread of 2019 novel coronavirus (2019-nCoV) or the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). The virus originated in bats and was transmitted to humans through yet unknown intermediary animals in Wuhan, Hubei province, China in December 2019. There have been around 96,000 reported cases of coronavirus disease 2019 (COVID-2019) and 3300 reported deaths to date (05/03/2020). The disease is transmitted by inhalation or contact with infected droplets and the incubation period ranges from 2 to 14 d. The symptoms are usually fever, cough, sore throat, breathlessness, fatigue, malaise among others. The disease is mild in most people; in some (usually the elderly and those with comorbidities), it may progress to pneumonia, acute respiratory distress syndrome (ARDS) and multi organ dysfunction. Many people are asymptomatic. The case fatality rate is estimated to range from 2 to 3%. Diagnosis is by demonstration of the virus in respiratory secretions by special molecular tests. Common laboratory findings include normal/ low white cell counts with elevated C-reactive protein (CRP). The computerized tomographic chest scan is usually abnormal even in those with no symptoms or mild disease. Treatment is essentially supportive; role of antiviral agents is yet to be established. Prevention entails home isolation of suspected cases and those with mild illnesses and strict infection control measures at hospitals that include contact and droplet precautions. The virus spreads faster than its two ancestors the SARS-CoV and Middle East respiratory syndrome coronavirus (MERS-CoV), but has lower fatality. The global impact of this new epidemic is yet uncertain.\n",
      "Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a highly contagious zoonotic virus that originated from Wuhan, China, in December 2019. The resulting disease known as COVID-19 was declared as a pandemic by WHO on March 11, 2020. As of April 25, 2020, more than 2.72 million cases have been reported across 213 countries and territories, resulting in more than 187,847 deaths (https://www.who.int/emergencies/diseases/novel-coronavirus-2019). The disease spreads through aerosol droplets and has an incubation period of 1-14 days. Research addressing crucial unknowns regarding clinical severity, the extent of transmission, and treatment options is going on all over the world since the disease inception. The primary response strategy includes limiting human-to-human transmission.\n",
      "The emergence and rapid spread of coronavirus disease 2019 (COVID-19), caused by severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2), a potentially fatal disease, is swiftly leading to public health crises worldwide. The origin of SARS-CoV-2 infection was first reported in people exposed to a seafood market in Wuhan City, China in December 2019. It has been suggested that the infection is likely to be of zoonotic origin and transmitted to humans through a not-yet-known intermediary. As of 22 May 2020, the World Health Organization reported that there were approximately 4,995,996 confirmed cases and 327,821 deaths. SARS-CoV-2 is transmitted via inhalation or direct contact with droplets from infected people. It has an incubation period ranging from 2 to &#8805;14 days. The rate of spread of SARS-CoV-2 is greater than that for severe acute respiratory syndrome coronavirus and Middle East respiratory coronavirus. The symptoms are similar to influenza (i.e. breathlessness, sore throat and fatigue) and infected cases are isolated and treated. Infection is mild in most cases, but in elderly (>50 years) patients and those with cardiac and respiratory disorders, it may progress to pneumonia, acute respiratory distress syndrome and multi-organ failure. People with strong immunity or those who have developed herd immunity are asymptomatic. The fatality rate ranges from 3% to 4%. Recommended methods for diagnosis of COVID-19 are molecular tests (e.g. polymerase chain reaction) on respiratory secretions, chest scan and common laboratory diagnosis. Currently, treatment is essentially supportive, and the role of antiviral agents is yet to be established as a vaccine is not yet available. This review will focus on epidemiology, symptoms, transmission, pathogenesis, ongoing available treatments and future perspectives of SARS-CoV-2.\n",
      "The Coronavirus Disease Pandemic 2019 (COVID-19), caused by the Severe Acute Respiratory Syndrome-related Coronavirus 2 (SARS-CoV-2), started in December 2019 in China. SARS-CoV-2 is easily transmitted by droplet infection. After an incubation period of 1-14 days, COVID-19 shows a mild course in 80â% of observed cases and a severe course in 20â%, with a lethality rate of 0.3-5.8â%. Elderly people and people with underlying diseases have a higher risk of severe courses with mandatory ventilation. So far there are neither effective drugs nor vaccinations available, so only public health interventions such as physical distancing and hygiene measures on the one hand and targeted testing followed by isolation and quarantine measures on the other hand are available. China has shown that maximum use of these measures can control the epidemic. The further course and also the consequences for the global economy cannot be clearly predicted at present.\n",
      "Coronaviruses are large family-RNA viruses that belong to the order Nidovirales, family Coronaviridae, subfamily Coronavirinae. The novel COVID-19 infection, caused by a beta coronavirus called SARS-CoV-2, is a new outbreak that has been emerged in Wuhan, China in December 2019. The most common symptoms of COVID-19 are fever, cough, and dyspnea. As per the March 12, 2020, WHO report, more than 125,048 confirmed COVID-19 cases and over 4613 deaths have been identified in more than 117 countries. It is now regarded as a pandemic that seriously spread and attack the world. The primary means of transmission is person to person through droplets that occurred during coughing or sneezing, through personal contact (shaking hands), or by touching contaminated objects. So far, there is no effective therapy and vaccine available against this novel virus and therefore, only supportive care is used as the mainstay of management of patients with COVID-19. The mortality rate of COVID-19 is considerable. This work aimed to provide insight on the newly emerged COVID-19, in the hope to gain a better understanding on the general overview, epidemiology, transmission, clinical features, diagnosis, treatment, and clinical outcomes as well as the prevention and control of COVID-19.\n",
      "The Coronavirus Disease Pandemic 2019 (COVID-19), caused by the Severe Acute Respiratory Syndrome-related Coronavirus 2 (SARS-CoV-2), started in December 2019 in China. SARS-CoV-2 is easily transmitted by droplet infection. After an incubation period of 1–14 days, COVID-19 shows a mild course in 80 % of observed cases and a severe course in 20 %, with a lethality rate of 0.3–5.8 %. Elderly people and people with underlying diseases have a higher risk of severe courses with mandatory ventilation. So far there are neither effective drugs nor vaccinations available, so only public health interventions such as physical distancing and hygiene measures on the one hand and targeted testing followed by isolation and quarantine measures on the other hand are available. China has shown that maximum use of these measures can control the epidemic. The further course and also the consequences for the global economy cannot be clearly predicted at present.\n",
      "The coronavirus (CoV) epidemic that began in China in December 2019 follows earlier epidemics of severe acute respiratory syndrome CoV in China and Middle East respiratory syndrome CoV in Saudi Arabia. The full genome of the 2019 novel coronavirus (2019-nCoV) has now been shared, and data have been gathered from several case series. As of February 11, 2020, there have been 45,182 laboratory-confirmed cases, the vast majority in China, with 1115 deaths, for an overall case-fatality rate of 2.5%. Cases have been confirmed in 27 countries. On average, each patient infects 2.2 other people. Symptomatic infection appears to predominantly affect adults, with a 5-day estimated incubation period between infection and symptom onset. The most common presenting symptoms are fever, cough, dyspnea, and myalgias and/or fatigue. All cases reported to date have shown radiographic evidence of pneumonia. 2019-nCoV is diagnosed by real-time reverse transcriptase polymerase chain reaction. Treatment is largely supportive, with regimens including antiviral therapy. Corticosteroids are not routinely recommended. Hand hygiene, prompt identification and isolation of suspect patients, and appropriate use of personal protective equipment are the most reliable methods to contain the epidemic.\n",
      "The outbreak of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is an emergent public health crisis threatening the current world health establishment. The SARS-Co-2 was first identified in Wuhan, Hubei Province, China, in December 2019. There have been about 6.5 million reported cases of coronavirus disease 2019 (COVID-19) and about 350 000 reported deaths throughout the world within the last 6 months from the onset of the epidemic. The virus is primarily transmitted by inhalation or contact with infected droplets. The COVID-19 patient usually presents with fever, cough, sore throat and breathlessness. Currently, available data indicate that the majority of people with the disease have mild symptoms, while about 20% present with moderate-to-severe disease. About 5% of these may progress to pneumonia, acute respiratory distress syndrome and multi-organ dysfunction. To date, there is no recommended medical treatment, and supportive measures are a crucial part of management. The case fatality rate of SARS-CoV-2 is lower than that of its two coronavirus predecessors, that is, severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV). The full impact of this new pandemic on health, social and economic well-being of humankind is yet to be ascertained.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: m2c5bvuj\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what are the best masks for preventing infection by Covid-19?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Coronaviruses (CoVs) are RNA viruses that have become a major public health concern since the Severe Acute Respiratory Syndrome-CoV (SARS-CoV) outbreak in 2002. The continuous evolution of coronaviruses was further highlighted with the emergence of the Middle East Respiratory Syndrome-CoV (MERS-CoV) outbreak in 2012. Currently, the world is concerned about the 2019 novel CoV (SARS-CoV-2) that was initially identified in the city of Wuhan, China in December 2019. Patients presented with severe viral pneumonia and respiratory illness. The number of cases has been mounting since then. As of late February 2020, tens of thousands of cases and several thousand deaths have been reported in China alone, in addition to thousands of cases in other countries. Although the fatality rate of SARS-CoV-2 is currently lower than SARS-CoV, the virus seems to be highly contagious based on the number of infected cases to date. In this review, we discuss structure, genome organization, entry of CoVs into target cells, and provide insights into past and present outbreaks. The future of human CoV outbreaks will not only depend on how the viruses will evolve, but will also depend on how we develop efficient prevention and treatment strategies to deal with this continuous threat.\n",
      "With the outbreak of unknown pneumonia in Wuhan, China, in December 2019, a new coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), aroused the attention of the entire world. The current outbreak of infections with SARS-CoV-2 is termed Coronavirus Disease 2019 (COVID-19). The World Health Organization declared COVID-19 in China as a Public Health Emergency of International Concern. Two other coronavirus infections-SARS in 2002-2003 and Middle East Respiratory Syndrome (MERS) in 2012-both caused severe respiratory syndrome in humans. All 3 of these emerging infectious diseases leading to a global spread are caused by ß-coronaviruses. Although coronaviruses usually infect the upper or lower respiratory tract, viral shedding in plasma or serum is common. Therefore, there is still a theoretical risk of transmission of coronaviruses through the transfusion of labile blood products. Because more and more asymptomatic infections are being found among COVID-19 cases, considerations of blood safety and coronaviruses have arisen especially in endemic areas. In this review, we detail current evidence and understanding of the transmission of SARS-CoV, MERS-CoV, and SARS-CoV-2 through blood products as of February 10, 2020, and also discuss pathogen inactivation methods on coronaviruses.\n",
      "Coronaviruses are a family of RNA viruses that typically cause mild respiratory disease in humans. However, over the past 20 years, three novel/variant coronaviruses have spilled over from animals into humans and have been associated with severe respiratory illness. In late 2002, severe acute respiratory syndrome (SARS) coronavirus (CoV) emerged in China and, over the following year, went on to cause approximately 8,100 cases and 774 deaths. A decade later, a cluster of severe pneumonia cases occurred on the Arabian Peninsula, marking the beginning of the Middle East respiratory syndrome (MERS)-CoV outbreak, which has resulted in nearly 2,500 confirmed cases and 850 deaths. Now in 2020, we are in the midst of a global pandemic caused by SARS-CoV-2, which, at the time of this writing, has claimed the lives of over 83,500 people and has been confirmed in over 1,500,000 cases. These outbreaks highlight the pathogenic potential of CoVs and the importance of infection prevention and diagnostic testing to reduce the spread of infectious diseases representing a global health threat.\n",
      "Coronavirus (COVID-19) is an enveloped RNA virus that is diversely found in humans and wildlife. A total of six species have been identified to cause disease in humans. They are known to infect the neurological, respiratory, enteric, and hepatic systems. The past few decades have seen endemic outbreaks in the form of Middle East respiratory syndrome coronavirus (MERS-CoV) and severe acute respiratory syndrome related coronavirus (SARS-CoV). Yet again, we see the emergence of another outbreak due to a new strain called the SARS-CoV-2 virus. The most recent outbreak initially presented as pneumonia of unknown etiology in a cluster of patients in Wuhan, China. The epicenter of infection was linked to seafood and exotic animal wholesale markets in the city. SARS-CoV-2 is highly contagious and has resulted in a rapid pandemic of COVID-19. As the number of cases continues to rise, it is clear that these viruses pose a threat to public health. This review will introduce a general overview of coronavirus and describe the clinical features, evaluation, and treatment of COVID-19 patients. It will also provide a means to raise awareness among primary and secondary healthcare providers during the current pandemic. Furthermore, our review focuses on the most up-to-date clinical information for the effective management, prevention, and counseling of patients worldwide.\n",
      "Abstract With the outbreak of unknown pneumonia in Wuhan, China, in December 2019, a new coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), aroused the attention of the entire world. The current outbreak of infections with SARS-CoV-2 is termed Coronavirus Disease 2019 (COVID-19). The World Health Organization declared COVID-19 in China as a Public Health Emergency of International Concern. Two other coronavirus infections—SARS in 2002-2003 and Middle East Respiratory Syndrome (MERS) in 2012—both caused severe respiratory syndrome in humans. All 3 of these emerging infectious diseases leading to a global spread are caused by β-coronaviruses. Although coronaviruses usually infect the upper or lower respiratory tract, viral shedding in plasma or serum is common. Therefore, there is still a theoretical risk of transmission of coronaviruses through the transfusion of labile blood products. Because more and more asymptomatic infections are being found among COVID-19 cases, considerations of blood safety and coronaviruses have arisen especially in endemic areas. In this review, we detail current evidence and understanding of the transmission of SARS-CoV, MERS–CoV, and SARS-CoV-2 through blood products as of February 10, 2020, and also discuss pathogen inactivation methods on coronaviruses.\n",
      "Emerging coronaviruses (CoV) are constant global public health threats to society. Multiple ongoing clinical trials for vaccines and antivirals against CoVs showcase the availability of medical interventions to both prevent and treat the future emergence of highly pathogenic CoVs in human. However, given the diverse nature of CoVs and our close interactions with wild, domestic and companion animals, the next epidemic zoonotic CoV could resist the existing vaccines and antivirals developed, which are primarily focused on Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) and Middle East Respiratory Syndrome Coronavirus (MERS CoV). In late 2019, the novel CoV (SARS-CoV-2) emerged in Wuhan, China, causing global public health concern. In this review, we will summarize the key advancements of current vaccines and antivirals against SARS-CoV and MERS-CoV as well as discuss the challenge and opportunity in the current SARS-CoV-2 crisis. At the end, we advocate the development of a “plug-and-play” platform technologies that could allow quick manufacturing and administration of broad-spectrum countermeasures in an outbreak setting. We will discuss the potential of AAV-based gene therapy technology for in vivo therapeutic antibody delivery to combat SARS-CoV-2 outbreak and the future emergence of severe CoVs.\n",
      "The last two decades have witnessed two large-scale pandemics caused by coronaviruses, including severe acute respiratory syndrome (SARS) and the Middle East respiratory syndrome (MERS). At the end of 2019, another novel coronavirus, designated as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), hit Wuhan, a city in the center of China, and subsequently spread rapidly to the whole world. Latest reports revealed that more than 800 thousand people in over 200 countries are involved in the epidemic disease by SARS-CoV-2. Due to the high mortality rate and the lack of optimum therapeutics, it is crucial to understand the biological characteristics of the virus and its possible pathogenesis to respond to the SARS-CoV-2. Rapid diagnostics and effective therapeutics are also important interventions for the management of infection control. However, the rapid evolution of SARS-CoV-2 exerted tremendous challenges on its diagnostics and therapeutics. Therefore, there is an urgent need to summarize the existing research results to guide decision-making on the prioritization of resources for research and development. In this review, we focus on our current understanding of epidemiology, pathogenesis, diagnostics and therapeutics of coronavirus disease 2019 (COVID-19).\n",
      "The last two decades have witnessed two large-scale pandemics caused by coronaviruses, including severe acute respiratory syndrome (SARS) and the Middle East respiratory syndrome (MERS). At the end of 2019, another novel coronavirus, designated as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), hit Wuhan, a city in the center of China, and subsequently spread rapidly to the whole world. Latest reports revealed that more than 800 thousand people in over 200 countries are involved in the epidemic disease by SARS-CoV-2. Due to the high mortality rate and the lack of optimum therapeutics, it is crucial to understand the biological characteristics of the virus and its possible pathogenesis to respond to the SARS-CoV-2. Rapid diagnostics and effective therapeutics are also important interventions for the management of infection control. However, the rapid evolution of SARS-CoV-2 exerted tremendous challenges on its diagnostics and therapeutics. Therefore, there is an urgent need to summarize the existing research results to guide decision-making on the prioritization of resources for research and development. In this review, we focus on our current understanding of epidemiology, pathogenesis, diagnostics and therapeutics of coronavirus disease 2019 (COVID-19).\n",
      "Wuhan, a city of China, is the epicenter for the pandemic outbreak of coronavirus disease‐2019 (COVID‐19). It has become a severe public health challenge to the world and established a public health emergency of international worry. This infectious disease has pulled down the economy of almost all top developed nations. The coronaviruses (CoVs) known for various epidemics caused time to time. Infectious diseases such as severe acute respiratory syndrome (SARS) and middle east respiratory syndrome (MERS), followed by COVID‐19, are all coronaviruses led outbreaks that scourged the history of mankind. CoVs evolved themselves to more infectious, transmissible, and more pandemic with time. To prevent the spread of the SARS‐CoV‐2, many countries have ordered the complete lockdown to combat the outbreak. This paper briefly discussed the historical background of CoVs and the evolution of human coronaviruses (HCoVs), the case studies and the development of their antiviral medications. The viral infection encountered with present‐day challenges and futuristic approaches with the help of nanotechnology to minimize the spread of infectious viruses. The antiviral drugs and their clinical advances, along with herbal medicines for viral inhibition and immunity boosters, are described. Elaboration of tables related to CoVs for the compilation of the literature has been adopted for the better understanding.\n",
      "In December 2019, an outbreak of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) was reported in Wuhan, China, and it subsequently spread in many countries around the world. Many efforts have been applied to control and prevent the spread of COVID-19, and many scientific studies have been conducted in a short period of time. Here we present an overview of the viral structure, pathogenesis, diagnosis, and clinical features of COVID-19 based on the current state of knowledge, and we compare its clinical characteristics with SARS and Middle East Respiratory Syndrome (MERS). Current researches on potentially effective treatment alternatives are discussed. We hope this review can help medical workers and researchers around the world contain the current COVID-19 pandemic.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 4cbjbo7t\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what types of rapid testing for Covid-19 have been developed?\n",
      "are there serological tests that detect antibodies to coronavirus?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "BACKGROUND The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) virus and resulting COVID-19 pandemic present important diagnostic challenges. Several diagnostic strategies are available to identify current infection, rule out infection, identify people in need of care escalation, or to test for past infection and immune response. Serology tests to detect the presence of antibodies to SARS-CoV-2 aim to identify previous SARS-CoV-2 infection, and may help to confirm the presence of current infection. OBJECTIVES To assess the diagnostic accuracy of antibody tests to determine if a person presenting in the community or in primary or secondary care has SARS-CoV-2 infection, or has previously had SARS-CoV-2 infection, and the accuracy of antibody tests for use in seroprevalence surveys. SEARCH METHODS We undertook electronic searches in the Cochrane COVID-19 Study Register and the COVID-19 Living Evidence Database from the University of Bern, which is updated daily with published articles from PubMed and Embase and with preprints from medRxiv and bioRxiv. In addition, we checked repositories of COVID-19 publications. We did not apply any language restrictions. We conducted searches for this review iteration up to 27 April 2020. SELECTION CRITERIA We included test accuracy studies of any design that evaluated antibody tests (including enzyme-linked immunosorbent assays, chemiluminescence immunoassays, and lateral flow assays) in people suspected of current or previous SARS-CoV-2 infection, or where tests were used to screen for infection. We also included studies of people either known to have, or not to have SARS-CoV-2 infection. We included all reference standards to define the presence or absence of SARS-CoV-2 (including reverse transcription polymerase chain reaction tests (RT-PCR) and clinical diagnostic criteria). DATA COLLECTION AND ANALYSIS We assessed possible bias and applicability of the studies using the QUADAS-2 tool. We extracted 2x2 contingency table data and present sensitivity and specificity for each antibody (or combination of antibodies) using paired forest plots. We pooled data using random-effects logistic regression where appropriate, stratifying by time since post-symptom onset. We tabulated available data by test manufacturer. We have presented uncertainty in estimates of sensitivity and specificity using 95% confidence intervals (CIs). MAIN RESULTS We included 57 publications reporting on a total of 54 study cohorts with 15,976 samples, of which 8526 were from cases of SARS-CoV-2 infection. Studies were conducted in Asia (n = 38), Europe (n = 15), and the USA and China (n = 1). We identified data from 25 commercial tests and numerous in-house assays, a small fraction of the 279 antibody assays listed by the Foundation for Innovative Diagnostics. More than half (n = 28) of the studies included were only available as preprints. We had concerns about risk of bias and applicability. Common issues were use of multi-group designs (n = 29), inclusion of only COVID-19 cases (n = 19), lack of blinding of the index test (n = 49) and reference standard (n = 29), differential verification (n = 22), and the lack of clarity about participant numbers, characteristics and study exclusions (n = 47). Most studies (n = 44) only included people hospitalised due to suspected or confirmed COVID-19 infection. There were no studies exclusively in asymptomatic participants. Two-thirds of the studies (n = 33) defined COVID-19 cases based on RT-PCR results alone, ignoring the potential for false-negative RT-PCR results. We observed evidence of selective publication of study findings through omission of the identity of tests (n = 5). We observed substantial heterogeneity in sensitivities of IgA, IgM and IgG antibodies, or combinations thereof, for results aggregated across different time periods post-symptom onset (range 0% to 100% for all target antibodies). We thus based the main results of the review on the 38 studies that stratified results by time since symptom onset. The numbers of individuals contributing data within each study each week are small and are usually not based on tracking the same groups of patients over time. Pooled results for IgG, IgM, IgA, total antibodies and IgG/IgM all showed low sensitivity during the first week since onset of symptoms (all less than 30.1%), rising in the second week and reaching their highest values in the third week. The combination of IgG/IgM had a sensitivity of 30.1% (95% CI 21.4 to 40.7) for 1 to 7 days, 72.2% (95% CI 63.5 to 79.5) for 8 to 14 days, 91.4% (95% CI 87.0 to 94.4) for 15 to 21 days. Estimates of accuracy beyond three weeks are based on smaller sample sizes and fewer studies. For 21 to 35 days, pooled sensitivities for IgG/IgM were 96.0% (95% CI 90.6 to 98.3). There are insufficient studies to estimate sensitivity of tests beyond 35 days post-symptom onset. Summary specificities (provided in 35 studies) exceeded 98% for all target antibodies with confidence intervals no more than 2 percentage points wide. False-positive results were more common where COVID-19 had been suspected and ruled out, but numbers were small and the difference was within the range expected by chance. Assuming a prevalence of 50%, a value considered possible in healthcare workers who have suffered respiratory symptoms, we would anticipate that 43 (28 to 65) would be missed and 7 (3 to 14) would be falsely positive in 1000 people undergoing IgG/IgM testing at days 15 to 21 post-symptom onset. At a prevalence of 20%, a likely value in surveys in high-risk settings, 17 (11 to 26) would be missed per 1000 people tested and 10 (5 to 22) would be falsely positive. At a lower prevalence of 5%, a likely value in national surveys, 4 (3 to 7) would be missed per 1000 tested, and 12 (6 to 27) would be falsely positive. Analyses showed small differences in sensitivity between assay type, but methodological concerns and sparse data prevent comparisons between test brands. AUTHORS' CONCLUSIONS The sensitivity of antibody tests is too low in the first week since symptom onset to have a primary role for the diagnosis of COVID-19, but they may still have a role complementing other testing in individuals presenting later, when RT-PCR tests are negative, or are not done. Antibody tests are likely to have a useful role for detecting previous SARS-CoV-2 infection if used 15 or more days after the onset of symptoms. However, the duration of antibody rises is currently unknown, and we found very little data beyond 35 days post-symptom onset. We are therefore uncertain about the utility of these tests for seroprevalence surveys for public health management purposes. Concerns about high risk of bias and applicability make it likely that the accuracy of tests when used in clinical care will be lower than reported in the included studies. Sensitivity has mainly been evaluated in hospitalised patients, so it is unclear whether the tests are able to detect lower antibody levels likely seen with milder and asymptomatic COVID-19 disease. The design, execution and reporting of studies of the accuracy of COVID-19 tests requires considerable improvement. Studies must report data on sensitivity disaggregated by time since onset of symptoms. COVID-19-positive cases who are RT-PCR-negative should be included as well as those confirmed RT-PCR, in accordance with the World Health Organization (WHO) and China National Health Commission of the People's Republic of China (CDC) case definitions. We were only able to obtain data from a small proportion of available tests, and action is needed to ensure that all results of test evaluations are available in the public domain to prevent selective reporting. This is a fast-moving field and we plan ongoing updates of this living systematic review.\n",
      "BACKGROUND Some people with SARS-CoV-2 infection remain asymptomatic, whilst in others the infection can cause mild to moderate COVID-19 disease and COVID-19 pneumonia, leading some patients to require intensive care support and, in some cases, to death, especially in older adults. Symptoms such as fever or cough, and signs such as oxygen saturation or lung auscultation findings, are the first and most readily available diagnostic information. Such information could be used to either rule out COVID-19 disease, or select patients for further diagnostic testing. OBJECTIVES To assess the diagnostic accuracy of signs and symptoms to determine if a person presenting in primary care or to hospital outpatient settings, such as the emergency department or dedicated COVID-19 clinics, has COVID-19 disease or COVID-19 pneumonia. SEARCH METHODS On 27 April 2020, we undertook electronic searches in the Cochrane COVID-19 Study Register and the University of Bern living search database, which is updated daily with published articles from PubMed and Embase and with preprints from medRxiv and bioRxiv. In addition, we checked repositories of COVID-19 publications. We did not apply any language restrictions. SELECTION CRITERIA Studies were eligible if they included patients with suspected COVID-19 disease, or if they recruited known cases with COVID-19 disease and controls without COVID-19. Studies were eligible when they recruited patients presenting to primary care or hospital outpatient settings. Studies including patients who contracted SARS-CoV-2 infection while admitted to hospital were not eligible. The minimum eligible sample size of studies was 10 participants. All signs and symptoms were eligible for this review, including individual signs and symptoms or combinations. We accepted a range of reference standards including reverse transcription polymerase chain reaction (RT-PCR), clinical expertise, imaging, serology tests and World Health Organization (WHO) or other definitions of COVID-19. DATA COLLECTION AND ANALYSIS Pairs of review authors independently selected all studies, at both title and abstract stage and full-text stage. They resolved any disagreements by discussion with a third review author. Two review authors independently extracted data and resolved disagreements by discussion with a third review author. Two review authors independently assessed risk of bias using the QUADAS-2 checklist. Analyses were descriptive, presenting sensitivity and specificity in paired forest plots, in ROC (receiver operating characteristic) space and in dumbbell plots. We did not attempt meta-analysis due to the small number of studies, heterogeneity across studies and the high risk of bias. MAIN RESULTS We identified 16 studies including 7706 participants in total. Prevalence of COVID-19 disease varied from 5% to 38% with a median of 17%. There were no studies from primary care settings, although we did find seven studies in outpatient clinics (2172 participants), and four studies in the emergency department (1401 participants). We found data on 27 signs and symptoms, which fall into four different categories: systemic, respiratory, gastrointestinal and cardiovascular. No studies assessed combinations of different signs and symptoms and results were highly variable across studies. Most had very low sensitivity and high specificity; only six symptoms had a sensitivity of at least 50% in at least one study: cough, sore throat, fever, myalgia or arthralgia, fatigue, and headache. Of these, fever, myalgia or arthralgia, fatigue, and headache could be considered red flags (defined as having a positive likelihood ratio of at least 5) for COVID-19 as their specificity was above 90%, meaning that they substantially increase the likelihood of COVID-19 disease when present. Seven studies carried a high risk of bias for selection of participants because inclusion in the studies depended on the applicable testing and referral protocols, which included many of the signs and symptoms under study in this review. Five studies only included participants with pneumonia on imaging, suggesting that this is a highly selected population. In an additional four studies, we were unable to assess the risk for selection bias. These factors make it very difficult to determine the diagnostic properties of these signs and symptoms from the included studies. We also had concerns about the applicability of these results, since most studies included participants who were already admitted to hospital or presenting to hospital settings. This makes these findings less applicable to people presenting to primary care, who may have less severe illness and a lower prevalence of COVID-19 disease. None of the studies included any data on children, and only one focused specifically on older adults. We hope that future updates of this review will be able to provide more information about the diagnostic properties of signs and symptoms in different settings and age groups. AUTHORS' CONCLUSIONS The individual signs and symptoms included in this review appear to have very poor diagnostic properties, although this should be interpreted in the context of selection bias and heterogeneity between studies. Based on currently available data, neither absence nor presence of signs or symptoms are accurate enough to rule in or rule out disease. Prospective studies in an unselected population presenting to primary care or hospital outpatient settings, examining combinations of signs and symptoms to evaluate the syndromic presentation of COVID-19 disease, are urgently needed. Results from such studies could inform subsequent management decisions such as self-isolation or selecting patients for further diagnostic testing. We also need data on potentially more specific symptoms such as loss of sense of smell. Studies in older adults are especially important.\n",
      "BACKGROUND: Some people with SARS-CoV-2 infection remain asymptomatic, whilst in others the infection can cause mild to moderate COVID-19 disease and COVID-19 pneumonia, leading some patients to require intensive care support and, in some cases, to death, especially in older adults. Symptoms such as fever or cough, and signs such as oxygen saturation or lung auscultation findings, are the first and most readily available diagnostic information. Such information could be used to either rule out COVID-19 disease, or select patients for further diagnostic testing. OBJECTIVES: To assess the diagnostic accuracy of signs and symptoms to determine if a person presenting in primary care or to hospital outpatient settings, such as the emergency department or dedicated COVID-19 clinics, has COVID-19 disease or COVID-19 pneumonia. SEARCH METHODS: On 27 April 2020, we undertook electronic searches in the Cochrane COVID-19 Study Register and the University of Bern living search database, which is updated daily with published articles from PubMed and Embase and with preprints from medRxiv and bioRxiv. In addition, we checked repositories of COVID-19 publications. We did not apply any language restrictions. SELECTION CRITERIA: Studies were eligible if they included patients with suspected COVID-19 disease, or if they recruited known cases with COVID-19 disease and controls without COVID-19. Studies were eligible when they recruited patients presenting to primary care or hospital outpatient settings. Studies including patients who contracted SARS-CoV-2 infection while admitted to hospital were not eligible. The minimum eligible sample size of studies was 10 participants. All signs and symptoms were eligible for this review, including individual signs and symptoms or combinations. We accepted a range of reference standards including reverse transcription polymerase chain reaction (RT-PCR), clinical expertise, imaging, serology tests and World Health Organization (WHO) or other definitions of COVID-19. DATA COLLECTION AND ANALYSIS: Pairs of review authors independently selected all studies, at both title and abstract stage and full-text stage. They resolved any disagreements by discussion with a third review author. Two review authors independently extracted data and resolved disagreements by discussion with a third review author. Two review authors independently assessed risk of bias using the QUADAS-2 checklist. Analyses were descriptive, presenting sensitivity and specificity in paired forest plots, in ROC (receiver operating characteristic) space and in dumbbell plots. We did not attempt meta-analysis due to the small number of studies, heterogeneity across studies and the high risk of bias. MAIN RESULTS: We identified 16 studies including 7706 participants in total. Prevalence of COVID-19 disease varied from 5% to 38% with a median of 17%. There were no studies from primary care settings, although we did find seven studies in outpatient clinics (2172 participants), and four studies in the emergency department (1401 participants). We found data on 27 signs and symptoms, which fall into four different categories: systemic, respiratory, gastrointestinal and cardiovascular. No studies assessed combinations of different signs and symptoms and results were highly variable across studies. Most had very low sensitivity and high specificity; only six symptoms had a sensitivity of at least 50% in at least one study: cough, sore throat, fever, myalgia or arthralgia, fatigue, and headache. Of these, fever, myalgia or arthralgia, fatigue, and headache could be considered red flags (defined as having a positive likelihood ratio of at least 5) for COVID-19 as their specificity was above 90%, meaning that they substantially increase the likelihood of COVID-19 disease when present. Seven studies carried a high risk of bias for selection of participants because inclusion in the studies depended on the applicable testing and referral protocols, which included many of the signs and symptoms under study in this review. Five studies only included participants with pneumonia on imaging, suggesting that this is a highly selected population. In an additional four studies, we were unable to assess the risk for selection bias. These factors make it very difficult to determine the diagnostic properties of these signs and symptoms from the included studies. We also had concerns about the applicability of these results, since most studies included participants who were already admitted to hospital or presenting to hospital settings. This makes these findings less applicable to people presenting to primary care, who may have less severe illness and a lower prevalence of COVID-19 disease. None of the studies included any data on children, and only one focused specifically on older adults. We hope that future updates of this review will be able to provide more information about the diagnostic properties of signs and symptoms in different settings and age groups. AUTHORS' CONCLUSIONS: The individual signs and symptoms included in this review appear to have very poor diagnostic properties, although this should be interpreted in the context of selection bias and heterogeneity between studies. Based on currently available data, neither absence nor presence of signs or symptoms are accurate enough to rule in or rule out disease. Prospective studies in an unselected population presenting to primary care or hospital outpatient settings, examining combinations of signs and symptoms to evaluate the syndromic presentation of COVID-19 disease, are urgently needed. Results from such studies could inform subsequent management decisions such as self-isolation or selecting patients for further diagnostic testing. We also need data on potentially more specific symptoms such as loss of sense of smell. Studies in older adults are especially important.\n",
      "Background During the coronavirus disease 2019 (COVID-19) pandemic little information has been available about patients with mild or moderate symptoms attended and followed in the primary care setting, most of whom had an unknown status for the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection. Objectives We aim to measure the seroprevalence of antibodies against SARS-CoV-2 infection in a community sample of asymptomatic individuals and among symptomatic patients (without confirmed diagnosis) followed in a primary care setting. As a secondary objective, we estimated the proportions of symptomatic patients seeing at an emergency department (ED), hospitalized or dying, and identified the most important clinical symptoms associated with a positive infection. Methods From April 21 to April 24 2020, we selected a random sample of 600 individuals stratified by age groups, from a total population of 19,899 individuals from a community area in Barcelona (study population 1). From April 29 to May 5 2020, we also invited all the patients that had been followed by general practitioners (GPs) (study population 2). We used for both populations COVID-19 Rapid lateral flow immunoassay which qualitatively assesses the presence of patient-generated IgG and IgM in approximately 10-15 minutes. The prevalence (95% confidence intervals [CI]) of infection (past and current) was defined as the proportion of individuals with antibody seropositivity. Odds ratios (ORs) for a positive test result were estimated using logistic regression analysis. Results Three hundred and eleven asymptomatic individuals from the randomly selected sample accepted to participate in the study. The overall mean age was 43.7 years (SD 21.79, range 1-94) and 55% were women. Seventeen individuals were seropositive for IgM and/or IgG, resulting an overall prevalence of 5,47% (95% CI, 3.44-8.58). Six-hundred and thirty-four symptomatic patients were followed by GPs. The overall mean age was 46.97 years (SD 20.05, range 0-92) and 57.73% were women. Of these, 244 patients (38.49%) were seropositive for IgM and/or IgG. During the follow-up period, 27.13% of symptomatic patients attended the ED, 11.83% were hospitalized and about 2% died. Results of the multivariate logistic regression analysis showed that the OR for a positive test was significantly increased in patients who had fever (>38{degrees}C), ageusia and contact with a patient diagnosed with COVID-19. Conclusions The seroprevalence of antibodies against SARS-CoV-2 among asymptomatic individuals in the general population was lower than expected. Approximately 40% of the symptomatic patients followed by GPs during the peak months of the pandemic in Barcelona, were positive. Fever (>38{degrees}C), anosmia, ageusia and contact with a patient diagnosed with COVID-19 were associated with a positive test result.\n",
      "OBJECTIVE: To determine the diagnostic accuracy of serological tests for coronavirus disease-2019 (covid-19). DESIGN: Systematic review and meta-analysis. DATA SOURCES: Medline, bioRxiv, and medRxiv from 1 January to 30 April 2020, using subject headings or subheadings combined with text words for the concepts of covid-19 and serological tests for covid-19. ELIGIBILITY CRITERIA AND DATA ANALYSIS: Eligible studies measured sensitivity or specificity, or both of a covid-19 serological test compared with a reference standard of viral culture or reverse transcriptase polymerase chain reaction. Studies were excluded with fewer than five participants or samples. Risk of bias was assessed using quality assessment of diagnostic accuracy studies 2 (QUADAS-2). Pooled sensitivity and specificity were estimated using random effects bivariate meta-analyses. MAIN OUTCOME MEASURES: The primary outcome was overall sensitivity and specificity, stratified by method of serological testing (enzyme linked immunosorbent assays (ELISAs), lateral flow immunoassays (LFIAs), or chemiluminescent immunoassays (CLIAs)) and immunoglobulin class (IgG, IgM, or both). Secondary outcomes were stratum specific sensitivity and specificity within subgroups defined by study or participant characteristics, including time since symptom onset. RESULTS: 5016 references were identified and 40 studies included. 49 risk of bias assessments were carried out (one for each population and method evaluated). High risk of patient selection bias was found in 98% (48/49) of assessments and high or unclear risk of bias from performance or interpretation of the serological test in 73% (36/49). Only 10% (4/40) of studies included outpatients. Only two studies evaluated tests at the point of care. For each method of testing, pooled sensitivity and specificity were not associated with the immunoglobulin class measured. The pooled sensitivity of ELISAs measuring IgG or IgM was 84.3% (95% confidence interval 75.6% to 90.9%), of LFIAs was 66.0% (49.3% to 79.3%), and of CLIAs was 97.8% (46.2% to 100%). In all analyses, pooled sensitivity was lower for LFIAs, the potential point-of-care method. Pooled specificities ranged from 96.6% to 99.7%. Of the samples used for estimating specificity, 83% (10 465/12 547) were from populations tested before the epidemic or not suspected of having covid-19. Among LFIAs, pooled sensitivity of commercial kits (65.0%, 49.0% to 78.2%) was lower than that of non-commercial tests (88.2%, 83.6% to 91.3%). Heterogeneity was seen in all analyses. Sensitivity was higher at least three weeks after symptom onset (ranging from 69.9% to 98.9%) compared with within the first week (from 13.4% to 50.3%). CONCLUSION: Higher quality clinical studies assessing the diagnostic accuracy of serological tests for covid-19 are urgently needed. Currently, available evidence does not support the continued use of existing point-of-care serological tests. STUDY REGISTRATION: PROSPERO CRD42020179452.\n",
      "BACKGROUND: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the cause of coronavirus disease 2019 (COVID-19), is associated with respiratory-related disease and death. Assays to detect virus-specific antibodies are important to understand the prevalence of infection and the course of the immune response. METHODS: Quantitative measurements of plasma or serum antibodies to the nucleocapsid and spike proteins were analyzed using luciferase immunoprecipitation system assays in 100 cross-sectional or longitudinal samples from patients with SARS-CoV-2 infection. A subset of samples was tested both with and without heat inactivation. RESULTS: At >14 days after symptom onset, antibodies against SARS-CoV-2 nucleocapsid protein showed 100% sensitivity and 100% specificity, whereas antibodies to spike protein were detected with 91% sensitivity and 100% specificity. Neither antibody levels nor the rate of seropositivity were significantly reduced by heat inactivation of samples. Analysis of daily samples from 6 patients with COVID-19 showed anti-nucleocapsid and spike protein antibodies appearing between days 8 and 14 after initial symptoms. Immunocompromised patients generally had a delayed antibody response to SARS-CoV-2, compared with immunocompetent patients. CONCLUSIONS: Antibody to the nucleocapsid protein of SARS-CoV-2 is more sensitive than spike protein antibody for detecting early infection. Analyzing heat-inactivated samples with a luciferase immunoprecipitation system assay is a safe and sensitive method for detecting SARS-CoV-2 antibodies.\n",
      "Objectives SARS-CoV-2, causing COVID-19, has emerged to cause a human pandemic. Detection of SARS-CoV-2 in respiratory samples by using PCR is the standard laboratory diagnostic tool. Our aim was to perform a limited evaluation of the diagnostic performance and user-friendliness of eleven rapid tests for detection of antibodies against SARS-CoV-2. Methods All participants were tested with PCR against SARS-CoV-2 at a clinical microbiology laboratory. Comparing with results from PCR tests, we evaluated the rapid tests' performances in three arms; 1) 20 hospitalized patients with PCR-confirmed COVID-19, 2) 23 recovered outpatients with former PCR-confirmed COVID-19, and 3) 49 participants with suspected COVID-19 presenting at a primary care emergency room. Results All eleven tests detected antibodies in hospitalized COVID-19 patients, though with varying sensitivities. In former outpatients recovered from COVID-19, there were differences between tests in the immunoglobulin type G (IgG) sensitivity, with five tests having a sensitivity below 65%. In participants with suspected COVID-19 infection, the rapid tests had very low sensitivities. Most rapid tests were easy to perform and interpret. Conclusions Rapid tests were not suited as stand-alone tests to detect present infection in a Norwegian primary care emergency room population. All the rapid tests were able to detect SARS-CoV-2 antibodies, although sensitivities varied and were generally higher in the study arm of more severely affected participants. Rapid tests with high IgG sensitivity (and specificity) may be useful for confirmation of past infection. An independent evaluation should be performed in the intended population before introducing a rapid test.\n",
      "Objectives SARS-CoV-2, causing COVID-19, has emerged to cause a human pandemic. Detection of SARS-CoV-2 in respiratory samples by using PCR is the standard laboratory diagnostic tool. Our aim was to perform a limited evaluation of the diagnostic performance and user-friendliness of eleven rapid tests for detection of antibodies against SARS-CoV-2. Methods All participants were tested with PCR against SARS-CoV-2 at a clinical microbiology laboratory. Comparing with results from PCR tests, we evaluated the rapid tests' performances in three arms; 1) 20 hospitalized patients with PCR-confirmed COVID-19, 2) 23 recovered outpatients with former PCR-confirmed COVID-19, and 3) 49 participants with suspected COVID-19 presenting at a primary care emergency room. Results All eleven tests detected antibodies in hospitalized COVID-19 patients, though with varying sensitivities. In former outpatients recovered from COVID-19, there were differences between tests in the immunoglobulin type G (IgG) sensitivity, with five tests having a sensitivity below 65%. In participants with suspected COVID-19 infection, the rapid tests had very low sensitivities. Most rapid tests were easy to perform and interpret. Conclusions Rapid tests were not suited as stand-alone tests to detect present infection in a Norwegian primary care emergency room population. All the rapid tests were able to detect SARS-CoV-2 antibodies, although sensitivities varied and were generally higher in the study arm of more severely affected participants. Rapid tests with high IgG sensitivity (and specificity) may be useful for confirmation of past infection. An independent evaluation should be performed in the intended population before introducing a rapid test.\n",
      "BACKGROUND: Serological tests are crucial tools for assessments of SARS-CoV-2 exposure, infection and potential immunity. Their appropriate use and interpretation require accurate assay performance data. METHOD: We conducted an evaluation of 10 lateral flow assays (LFAs) and two ELISAs to detect anti-SARS-CoV-2 antibodies. The specimen set comprised 128 plasma or serum samples from 79 symptomatic SARS-CoV-2 RT-PCR-positive individuals; 108 pre-COVID-19 negative controls; and 52 recent samples from individuals who underwent respiratory viral testing but were not diagnosed with Coronavirus Disease 2019 (COVID-19). Samples were blinded and LFA results were interpreted by two independent readers, using a standardized intensity scoring system. RESULTS: Among specimens from SARS-CoV-2 RT-PCR-positive individuals, the percent seropositive increased with time interval, peaking at 81.8–100.0% in samples taken >20 days after symptom onset. Test specificity ranged from 84.3–100.0% in pre-COVID-19 specimens. Specificity was higher when weak LFA bands were considered negative, but this decreased sensitivity. IgM detection was more variable than IgG, and detection was highest when IgM and IgG results were combined. Agreement between ELISAs and LFAs ranged from 75.7–94.8%. No consistent cross-reactivity was observed. CONCLUSION: Our evaluation showed heterogeneous assay performance. Reader training is key to reliable LFA performance, and can be tailored for survey goals. Informed use of serology will require evaluations covering the full spectrum of SARS-CoV-2 infections, from asymptomatic and mild infection to severe disease, and later convalescence. Well-designed studies to elucidate the mechanisms and serological correlates of protective immunity will be crucial to guide rational clinical and public health policies.\n",
      "Background SARS-CoV-2 serology tests are clinically useful to document a prior SARS-CoV-2 infection in patients with no or inconclusive PCR results and suspected COVID-19 disease or sequelae. Data are urgently needed to select the assays with optimal sensitivity at acceptable specificity. Methods A comparative analysis of analytical sensitivity was performed of seven commercial SARS-CoV-2 serology assays on 171 sera from 135 subjects with PCR-confirmed SARS-CoV-2 infection, composed of 71 patients hospitalized for COVID-19 pneumonia and 64 healthcare workers with paucisymptomatic infections. The kinetics of IgA/IgM/IgG seroconversion to viral N- and S-protein epitopes were studied from 0 to 54 days after symptom onset. Specificity was verified on 57 pre-pandemic samples. Results Wantai SARS-COV-2 Ab ELISA and Orient Gene COVID-19 IgG/IgM Rapid Test achieved a superior overall sensitivity. Elecsys Anti-SARS-CoV-2 assay, EUROIMMUN Anti-SARS-CoV-2 combined IgG/IgA and EUROIMMUN Anti-SARS-CoV-2-NCP(IgG) also showed acceptable sensitivity (>95%) versus the consensus result of all assays from 10 days post symptom onset. Optimal specificity (>98%) was achieved only by Wantai SARS-COV-2 Ab ELISA, Elecsys Anti-SARS-CoV-2 assay and Innovita 2019-nCoV Ab rapid test. LIAISON SARS-CoV-2 S1/S2 IgG showed a significantly lower sensitivity as compared to all other assays. Lack of seroconversion by any test was seen in 1.4% of hospitalized and 4.7% of paucisymptomatic infections. Within 10 days from symptom onset, only the Wantai SARS-COV-2 Ab ELISA has acceptable sensitivity. Conclusions Wantai SARS-COV-2 Ab ELISA and Elecsys Anti-SARS-CoV-2 assays are suitable for sensitive and specific screening of a SARS-CoV-2 infection from 10 days after symptom onset.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: ncl2p7kg\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), was first identified in several patients who traveled to Wuhan or went to a seafood wholesale market in Wuhan. The phylogenetic tree showed that SARS-CoV-2 was 96.2% identical to bat β-coronaviruses from lineage B. Also, several studies reported that SARS-CoV-2 uses the SARS-CoV receptor, angiotensin-converting enzyme 2, for entry to target cells. Lung alveolar and small intestine are potential targets for SARS-CoV-2 due to the high expression of the angiotensin-converting enzyme 2 receptor. In this review, we focused on the zoonotic β-coronaviruses and given there is no specific drug or vaccine for coronavirus disease 2019, we reviewed the literature on the therapy options for SARS and Middle East respiratory syndrome coronavirus infection, in order to discover their possible use in the treatment of SARS-CoV-2 infections.\n",
      "Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the cause of coronavirus disease 2019 (COVID-19), which has become a public health emergency of international concern1. Angiotensin-converting enzyme 2 (ACE2) is the cell-entry receptor for severe acute respiratory syndrome coronavirus (SARS-CoV)2. Here we infected transgenic mice that express human ACE2 (hereafter, hACE2 mice) with SARS-CoV-2 and studied the pathogenicity of the virus. We observed weight loss as well as virus replication in the lungs of hACE2 mice infected with SARS-CoV-2. The typical histopathology was interstitial pneumonia with infiltration of considerable numbers of macrophages and lymphocytes into the alveolar interstitium, and the accumulation of macrophages in alveolar cavities. We observed viral antigens in bronchial epithelial cells, macrophages and alveolar epithelia. These phenomena were not found in wild-type mice infected with SARS-CoV-2. Notably, we have confirmed the pathogenicity of SARS-CoV-2 in hACE2 mice. This mouse model of SARS-CoV-2 infection will be valuable for evaluating antiviral therapeutic agents and vaccines, as well as understanding the pathogenesis of COVID-19.\n",
      "Coronavirus disease 2019 (COVID-19) is a pandemic infectious disease caused by novel severe acute respiratory syndrome coronavirus-2 (SARS CoV-2). The SARS CoV-2 is transmitted more rapidly and readily than SARS CoV. Both, SARS CoV and SARS CoV-2 via their glycosylated spike proteins recognize the human angiotensin converting enzyme-2 (ACE-2) receptor. We generated multiple sequence alignments and phylogenetic trees for representative spike proteins of SARS CoV and SARS CoV-2 from various host sources in order to analyze the specificity in SARS CoV-2 spike proteins required for causing infection in humans. Our results show that among the genomes analyzed, two sequence regions in the N-terminal domain \"MESEFR\" and \"SYLTPG\" are specific to human SARS CoV-2. In the receptor-binding domain, two sequence regions \"VGGNY\" and \"EIYQAGSTPCNGV\" and a disulfide bridge connecting 480C and 488C in the extended loop are structural determinants for the recognition of human ACE-2 receptor. The complete genome analysis of representative SARS CoVs from bat, civet, human host sources, and human SARS CoV-2 identified the bat genome (GenBank code: MN996532.1) as closest to the recent novel human SARS CoV-2 genomes. The bat SARS CoV genomes (GenBank codes: MG772933 and MG772934) are evolutionary intermediates in the mutagenesis progression toward becoming human SARS CoV-2.\n",
      "Coronavirus disease 2019 (COVID‐19) is a pandemic infectious disease caused by novel Severe Acute Respiratory Syndrome coronavirus‐2 (SARS CoV‐2). The SARS CoV‐2 is transmitted more rapidly and readily than SARS CoV. Both, SARS CoV and SARS CoV‐2 via their glycosylated spike proteins recognize the human angiotensin converting enzyme‐2 (ACE‐2) receptor. We generated multiple sequence alignments and phylogenetic trees for representative spike proteins of SARS CoV and SARS CoV‐2 from various host sources in order to analyze the specificity in SARS CoV‐2 spike proteins required for causing infection in humans. Our results show that among the genomes analysed, two sequence regions in the N‐terminal domain (NTD); \"MESEFR\" and \"SYLTPG\" are specific to human SARS CoV‐2. In the receptor binding domain (RBD), two sequence regions; \"VGGNY\" and \"EIYQAGSTPCNGV\" and a disulfide bridge connecting 480C and 488C in the extended loop are structural determinants for the recognition of human ACE‐2 receptor. The complete genome analysis of representative SARS CoVs from bat, civet, human host sources and human SARS CoV‐2 identified the bat genome (GenBank code: MN996532.1) as closest to the recent novel human SARS CoV‐2 genomes. The bat SARS CoV genomes (GenBank codes: MG772933 and MG772934) are evolutionary intermediates in the mutagenesis progression towards becoming human SARS CoV‐2. This article is protected by copyright. All rights reserved.\n",
      "Severe acute respiratory syndrome CoV-2 (SARS-CoV-2) caused the corona virus disease 2019 (COVID-19) cases in China and has become a public health emergency of international concern1. Because angiotensin-converting enzyme 2 (ACE2) is the cell entry receptor of SARS-CoV5, we used transgenic mice bearing human ACE2 and infected with SARS-CoV-2 to study the pathogenicity of the virus. Weight loss and virus replication in lung were observed in hACE2 mice infected with SARS-CoV-2. The typical histopathology was interstitial pneumonia with infiltration of significant macrophages and lymphocytes into the alveolar interstitium, and accumulation of macrophages in alveolar cavities. Viral antigens were observed in the bronchial epithelial cells, macrophages and alveolar epithelia. The phenomenon was not found in wild-type mice with SARS-CoV-2 infection. Notably, we have confirmed the pathogenicity of SARS-CoV-2 in hACE2 mice. The mouse model with SARS-CoV-2 infection will be valuable for evaluating antiviral therapeutics and vaccines as well as understanding the pathogenesis of COVID-19.\n",
      "Severe acute respiratory syndrome CoV-2 (SARS-CoV-2) caused the corona virus disease 2019 (COVID-19) cases in China and has become a public health emergency of international concern1. Because angiotensin-converting enzyme 2 (ACE2) is the cell entry receptor of SARS-CoV5, we used transgenic mice bearing human ACE2 and infected with SARS-CoV-2 to study the pathogenicity of the virus. Weight loss and virus replication in lung were observed in hACE2 mice infected with SARS-CoV-2. The typical histopathology was interstitial pneumonia with infiltration of significant macrophages and lymphocytes into the alveolar interstitium, and accumulation of macrophages in alveolar cavities. Viral antigens were observed in the bronchial epithelial cells, macrophages and alveolar epithelia. The phenomenon was not found in wild-type mice with SARS-CoV-2 infection. Notably, we have confirmed the pathogenicity of SARS-CoV-2 in hACE2 mice. The mouse model with SARS-CoV-2 infection will be valuable for evaluating antiviral therapeutics and vaccines as well as understanding the pathogenesis of COVID-19.\n",
      "The coronavirus disease-19 (COVID-19) pandemic has emerged as one of the major outbreaks to be mentioned in history in coming times. Like severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS), severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) is a respiratory virus infecting the lungs with fever, dry cough, and acute pneumonia being the major symptoms. It infects epithelial cells expressing angiotensin converting enzyme 2 (ACE2) receptor, which is crucial for viral entry. Based on evolving clinical evidence, it is now unfitting to label SARS-CoV-2 as just a respiratory virus, as lately there are various reports that substantiate its pathogenicity in other organs of the body, including brain. In this review, we discuss the epidemiology of SARS-CoV-2 in comparison to SARS and MERS along with possibilities of viral entry into central nervous system (CNS) tissues. The review provides detailed information about the virulence, epidemiology, and insights into molecular pathways involved in the infectivity of the SARS-CoV-2 virus, along with an in-depth view of current concepts about the neurological significance of the SARS-CoV-2 virus and its neuropathological competence. The review also touches upon our current understanding of placental transmission of SARS-CoV-2, an important aspect of vertical transmission. Furthermore, the review provides a current update on strategies that have been used, are being used, or are under trial for treating the disease.\n",
      "A novel coronavirus SARS-CoV-2 was identified in Wuhan, Hubei Province, China in December of 2019. According to WHO report, this new coronavirus has resulted in 76,392 confirmed infections and 2,348 deaths in China by 22 February, 2020, with additional patients being identified in a rapidly growing number internationally. SARS-CoV-2 was reported to share the same receptor, Angiotensin-converting enzyme 2 (ACE2), with SARS-CoV. Here based on the public database and the state-of-the-art single-cell RNA-Seq technique, we analyzed the ACE2 RNA expression profile in the normal human lungs. The result indicates that the ACE2 virus receptor expression is concentrated in a small population of type II alveolar cells (AT2). Surprisingly, we found that this population of ACE2-expressing AT2 also highly expressed many other genes that positively regulating viral entry, reproduction and transmission. This study provides a biological background for the epidemic investigation of the COVID-19, and could be informative for future anti-ACE2 therapeutic strategy development.\n",
      "The novel coronavirus disease 2019 (COVID-19), caused by Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2), first appeared in December 2019, in Wuhan, China and evolved into a pandemic. As Angiotensin-Converting Enzyme 2 (ACE2) is one of the potential target receptors for SARS-CoV-2 in human body, which is expressed in different tissues, multiple organs might become affected. In the initial phase of the current pandemic, a handful of post-mortem case-series revealed COVID-19-related pathological changes in various organs. Although pathological examination is not a feasible method of diagnosis, it can elucidate pathological changes, pathogenesis of the disease, and the cause of death in COVID-19 cases. Herein, we thoroughly reviewed multiple organs including lung, gastrointestinal tract, liver, kidney, skin, heart, blood, spleen, lymph nodes, brain, blood vessels, and placenta in terms of COVID-19-related pathological alterations. Also, these findings were compared with SARS and MERS infection, wherever applicable. We found a diverse range of pathological changes, some of which resemble those found in SARS and MERS.\n",
      "Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) first appeared in Wuhan, China in December 2019 resulting in the COVID-19 pandemic. It has been shown that SARS-CoV-2 spike protein shares a high similarity with SARS-CoV spike protein with a strong binding affinity to human Angiotensin Converting Enzyme 2 (ACE2) receptor. Skin tissue may be a possible specific target for the SARS-CoV-2. Preliminary reports from the COVID-19 infection have shown cutaneous manifestation, including erythematous rash, widespread urticaria, and chickenpox-like vesicles. Moreover, many studies have shown the role of ACE in the development, severity, and complications of psoriasis. Herein, we discuss a possible shared mechanism of psoriasis with SARS-CoV-2; also, we propose a therapeutic approach to balance the effect of ACE activity in these patients.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: dtwstwbe\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "will SARS-CoV2 infected people develop immunity? Is cross protection possible?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Since 2002, beta coronaviruses (CoV) have caused three zoonotic outbreaks, SARS-CoV in 2002-2003, MERS-CoV in 2012, and the newly emerged SARS-CoV-2 in late 2019. However, little is currently known about the biology of SARS-CoV-2. Here, using SARS-CoV-2 S protein pseudovirus system, we confirm that human angiotensin converting enzyme 2 (hACE2) is the receptor for SARS-CoV-2, find that SARS-CoV-2 enters 293/hACE2 cells mainly through endocytosis, that PIKfyve, TPC2, and cathepsin L are critical for entry, and that SARS-CoV-2 S protein is less stable than SARS-CoV S. Polyclonal anti-SARS S1 antibodies T62 inhibit entry of SARS-CoV S but not SARS-CoV-2 S pseudovirions. Further studies using recovered SARS and COVID-19 patients' sera show limited cross-neutralization, suggesting that recovery from one infection might not protect against the other. Our results present potential targets for development of drugs and vaccines for SARS-CoV-2.\n",
      "Since 2002, beta coronaviruses (CoV) have caused three zoonotic outbreaks, SARS-CoV in 2002–2003, MERS-CoV in 2012, and the newly emerged SARS-CoV-2 in late 2019. However, little is currently known about the biology of SARS-CoV-2. Here, using SARS-CoV-2 S protein pseudovirus system, we confirm that human angiotensin converting enzyme 2 (hACE2) is the receptor for SARS-CoV-2, find that SARS-CoV-2 enters 293/hACE2 cells mainly through endocytosis, that PIKfyve, TPC2, and cathepsin L are critical for entry, and that SARS-CoV-2 S protein is less stable than SARS-CoV S. Polyclonal anti-SARS S1 antibodies T62 inhibit entry of SARS-CoV S but not SARS-CoV-2 S pseudovirions. Further studies using recovered SARS and COVID-19 patients’ sera show limited cross-neutralization, suggesting that recovery from one infection might not protect against the other. Our results present potential targets for development of drugs and vaccines for SARS-CoV-2.\n",
      "The emergence of a novel coronavirus, SARS-CoV-2, at the end of 2019 has resulted in widespread human infections across the globe. While genetically distinct from SARS-CoV, the etiological agent that caused an outbreak of severe acute respiratory syndrome (SARS) in 2003, both coronaviruses exhibit receptor binding domain (RBD) conservation and utilize the same host cell receptor, angiotensin-converting enzyme 2 (ACE2), for virus entry. Therefore, it will be important to test the cross-reactivity of antibodies that have been previously generated against the surface spike (S) glycoprotein of SARS-CoV in order to aid research on the newly emerged SARS-CoV-2. Here, we show that an immunogenic domain in the S2 subunit of SARS-CoV S is highly conserved in multiple strains of SARS-CoV-2. Consistently, four murine monoclonal antibodies (mAbs) raised against this immunogenic SARS-CoV fragment were able to recognise the S protein of SARS-CoV-2 expressed in a mammalian cell line. Importantly, one of them (mAb 1A9) was demonstrated to detect S in SARS-CoV-2-infected cells. To our knowledge, this is the first study showing that mAbs targeting the S2 domain of SARS-CoV can cross-react with SARS-CoV-2 and this observation is consistent with the high sequence conservation in the S2 subunit. These cross-reactive mAbs may serve as tools useful for SARS-CoV-2 research as well as for the development of diagnostic assays for its associated coronavirus disease COVID-19.\n",
      "Since 2002, beta coronaviruses (CoV) have caused three zoonotic outbreaks, SARS-CoV in 2002, MERS-CoV in 2012, and the recent outbreak of SARS-CoV-2 late in 2019 (also named as COVID-19 or novel coronavirus 2019 or nCoV2019. Spike(S) protein, one of the structural proteins of this virus plays key role in receptor (ACE2) binding and thus virus entry. Thus, this protein has attracted scientists for detailed study and therapeutic targeting. As the 2019 novel coronavirus takes its course throughout the world, more and more sequence analyses are been done and genome sequences getting deposited in various databases. From India two clinical isolates have been sequenced and the full genome deposited in GenBank. We have performed sequence analyses of the spike protein of the Indian isolates and compared with that of the Wuhan, China (where the outbreak was first reported). While all the sequences of Wuhan isolates are identical, we found point mutations in the Indian isolates. Out of the two isolates one was found to harbour a mutation in its Receptor binding domain (RBD) at position 407. At this site arginine (a positively charged amino acid) was replaced by isoleucine (a hydrophobic amino acid that is also a C-beta branched amino acid). This mutation has been seen to change the secondary structure of the protein at that region and this can potentially alter receptor ding of the virus. Although this finding needs further validation and more sequencing, the information might be useful in rational drug designing and vaccine engineering.\n",
      "Since 2002, β coronaviruses (CoVs) have caused three zoonotic outbreaks, SARS-CoV in 2002, MERS-CoV in 2012, and the recent outbreak of SARS-CoV-2 late in 2019 (also named as COVID-19 or novel coronavirus 2019 or nCoV2019). Spike (S) protein, one of the structural proteins of this virus plays key role in receptor (ACE2) binding and thus virus entry. Thus, this protein has attracted scientists for detailed study and therapeutic targeting. As the nCoV2019 takes its course throughout the world, more and more sequence analyses are being done and genome sequences are being deposited in various databases. From India, two clinical isolates have been sequenced and the full genome has been deposited in GenBank. We have performed sequence analyses of the Spike protein of the Indian isolates and compared with that of the Wuhan, China (where the outbreak was first reported). While all the sequences of Wuhan isolates are identical, we found point mutations in the Indian isolates. Out of the two isolates, one was found to harbor a mutation in its receptor-binding domain (RBD) at position 407. At this site, arginine (a positively charged amino acid) was replaced by isoleucine (a hydrophobic amino acid that is also a C-β branched amino acid). This mutation has been seen to change the secondary structure of the protein at that region and this can potentially alter receptor binding of the virus. Although this finding needs further validation and more sequencing, the information might be useful in rational drug designing and vaccine engineering.\n",
      "Severe acute respiratory syndrome coronavirus (SARS-CoV) emerged in 2002 as a highly transmissible pathogenic human betacoronavirus. The viral spike glycoprotein (S) utilizes angiotensin-converting enzyme 2 (ACE2) as a host protein receptor and mediates fusion of the viral and host membranes, making S essential to viral entry into host cells and host species tropism. As SARS-CoV enters host cells, the viral S is believed to undergo a number of conformational transitions as it is cleaved by host proteases and binds to host receptors. We recently developed stabilizing mutations for coronavirus spikes that prevent the transition from the pre-fusion to post-fusion states. Here, we present cryo-EM analyses of a stabilized trimeric SARS-CoV S, as well as the trypsin-cleaved, stabilized S, and its interactions with ACE2. Neither binding to ACE2 nor cleavage by trypsin at the S1/S2 cleavage site impart large conformational changes within stabilized SARS-CoV S or expose the secondary cleavage site, S2′.\n",
      "Human angiotensin-converting enzyme 2 (ACE2) is a functional receptor for SARS coronavirus (SARS-CoV). Here we identify the SARS-CoV spike (S)-protein-binding site on ACE2. We also compare S proteins of SARS-CoV isolated during the 2002–2003 SARS outbreak and during the much less severe 2003–2004 outbreak, and from palm civets, a possible source of SARS-CoV found in humans. All three S proteins bound to and utilized palm-civet ACE2 efficiently, but the latter two S proteins utilized human ACE2 markedly less efficiently than did the S protein obtained during the earlier human outbreak. The lower affinity of these S proteins could be complemented by altering specific residues within the S-protein-binding site of human ACE2 to those of civet ACE2, or by altering S-protein residues 479 and 487 to residues conserved during the 2002–2003 outbreak. Collectively, these data describe molecular interactions important to the adaptation of SARS-CoV to human cells, and provide insight into the severity of the 2002–2003 SARS epidemic.\n",
      "Infection of receptor-bearing cells by coronaviruses is mediated by their spike (S) proteins. The coronavirus (SARS-CoV) that causes severe acute respiratory syndrome (SARS) infects cells expressing the receptor angiotensin-converting enzyme 2 (ACE2). Here we show that codon optimization of the SARS-CoV S-protein gene substantially enhanced S-protein expression. We also found that two retroviruses, simian immunodeficiency virus (SIV) and murine leukemia virus, both expressing green fluorescent protein and pseudotyped with SARS-CoV S protein or S-protein variants, efficiently infected HEK293T cells stably expressing ACE2. Infection mediated by an S-protein variant whose cytoplasmic domain had been truncated and altered to include a fragment of the cytoplasmic tail of the human immunodeficiency virus type 1 envelope glycoprotein was, in both cases, substantially more efficient than that mediated by wild-type S protein. Using S-protein-pseudotyped SIV, we found that the enzymatic activity of ACE2 made no contribution to S-protein-mediated infection. Finally, we show that a soluble and catalytically inactive form of ACE2 potently blocked infection by S-protein-pseudotyped retrovirus and by SARS-CoV. These results permit studies of SARS-CoV entry inhibitors without the use of live virus and suggest a candidate therapy for SARS.\n",
      "Severe acute respiratory syndrome CoV-2 (SARS-CoV-2) is currently causing a worldwide pandemic with high morbidity and mortality. Development of animal models that recapitulate important aspects of coronavirus disease 2019 (COVID-19) is critical for the evaluation of vaccines and antivirals, and understanding disease pathogenesis. SARS-CoV-2 has been shown to use the same entry receptor as SARS-CoV-1, human angiotensin-converting enzyme 2 (hACE2)(1-3). Due to amino acid differences between murine and hACE2, inbred mouse strains fail to support high titer viral replication of SARS-CoV-2 virus. Therefore, a number of transgenic and knock-in mouse models, as well as viral vector-mediated hACE2 delivery systems have been developed. Here we compared the K18-hACE2 transgenic model to adenovirus-mediated delivery of hACE2 to the mouse lung. We show that K18-hACE2 mice replicate virus to high titers in both the lung and brain leading to lethality. In contrast, adenovirus-mediated delivery results in viral replication to lower titers limited to the lung, and no clinical signs of infection with a challenge dose of 104 plaque forming units. The K18-hACE2 model provides a stringent model for testing the ability of vaccines and antivirals to protect against disease, whereas the adenovirus delivery system has the flexibility to be used across multiple genetic backgrounds and modified mouse strains.\n",
      "Since 2002, ß coronaviruses (CoVs) have caused three zoonotic outbreaks, SARS-CoV in 2002, MERS-CoV in 2012, and the recent outbreak of SARS-CoV-2 late in 2019 (also named as COVID-19 or novel coronavirus 2019 or nCoV2019). Spike (S) protein, one of the structural proteins of this virus plays key role in receptor (ACE2) binding and thus virus entry. Thus, this protein has attracted scientists for detailed study and therapeutic targeting. As the nCoV2019 takes its course throughout the world, more and more sequence analyses are being done and genome sequences are being deposited in various databases. From India, two clinical isolates have been sequenced and the full genome has been deposited in GenBank. We have performed sequence analyses of the Spike protein of the Indian isolates and compared with that of the Wuhan, China (where the outbreak was first reported). While all the sequences of Wuhan isolates are identical, we found point mutations in the Indian isolates. Out of the two isolates, one was found to harbor a mutation in its receptor-binding domain (RBD) at position 407. At this site, arginine (a positively charged amino acid) was replaced by isoleucine (a hydrophobic amino acid that is also a C-ß branched amino acid). This mutation has been seen to change the secondary structure of the protein at that region and this can potentially alter receptor binding of the virus. Although this finding needs further validation and more sequencing, the information might be useful in rational drug designing and vaccine engineering.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 3h1o0oz3\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "what is the origin of COVID-19\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "The novel coronavirus SARS-CoV-2 (2019-nCoV) is a member of the family coronaviridae and contains a single-stranded RNA genome with positive-polarity. To reveal the evolution mechanism of SARS-CoV-2 genome, we performed comprehensive genomic analysis with newly sequenced SARS-CoV-2 strains and 20 closely related coronavirus strains. Among 98 nucleotide mutations at 93 sites of the genome among different SARS-CoV-2 strains, 58 of them caused amino acid change, indicating a result of neutral evolution. However, the ratio of nucleotide substitutions to amino acid substitutions of spike gene (9.07) between SARS-CoV-2 WIV04 and Bat-SARSr-CoV RaTG13 was extensively higher than those from comparisons between other coronaviruses (range 1.29 - 4.81). The elevated synonymous mutations between SARS-CoV-2 and RaTG13, suggesting they underwent stronger purifying selection. Moreover, their nucleotide substitutions are enriched with T:C transition, which is consistent with the mutation signature caused by deactivity of RNA 3’-to-5’ exoribonuclease (ExoN). The codon usage was similar between SARS-CoV-2 and other strains in beta-coronavirus lineage B, suggesting it had small impact on the mutation pattern. In comparison of SARS-CoV-2 WIV04 with Bat-SARSr-CoV RaTG13, the ratios of non-synonymous to synonymous substitution rates (dN/dS) was the lowest among all performed comparisons, reconfirming the evolution of SARS-CoV-2 under stringent selective pressure. Moreover, some sites of spike protein might be subjected to positive selection. Therefore, our results will help understanding the evolutionary mechanisms contribute to viral pathogenicity and its adaptation with hosts.\n",
      "Novel coronavirus has wrecked medical and health care facilities claiming ~5% death tolls globally. All efforts to contain the pathogenesis either using inhibitory drugs or vaccines largely remained futile due to a lack of better understanding of the genomic feature of this virus. In the present study, we compared the 2019-nCoV with other coronaviruses, which indicated that bat-SARS like coronavirus could be a probable ancestor of the novel coronavirus. The protein sequence similarity of pangolin-hCoV and bat-hCoV with human coronavirus was higher as compared to their nucleotide similarity denoting the occurrence of more synonymous mutations in the genome. Phylogenetic and alignment analysis of 591 novel coronaviruses of different clades from Group I to Group V revealed several mutations and concomitant amino acid changes. Detailed investigation on nucleotide substitution unfolded 100 substitutions in the coding region of which 43 were synonymous and 57 were of non-synonymous type. The non-synonymous substitutions resulting into 57 amino acid changes were found to be distributed over different hCoV proteins with maximum on spike protein. An important diamino acid change RG to KR was observed in ORF9 protein. Additionally, several interesting features of the novel coronavirus genome have been highlighted in respect to various other human infecting viruses which may explain extreme pathogenicity, infectivity and simultaneously the reason behind failure of the antiviral therapies. Summary This study presents a comprehensive phylogenetic analysis of SARS-CoV2 isolates to understand discrete mutations that are occurring between patient samples. This analysis will provide an explanation for varying treatment efficacies of different inhibitory drugs and a future direction towards a combinatorial treatment therapies based on the kind of mutation in the viral genome.\n",
      "Novel coronavirus has wrecked medical and health care facilities claiming â¼5% death tollsglobally. All efforts to contain the pathogenesis either using inhibitory drugs or vaccines largelyremained futile due to a lack of better understanding of the genomic feature of this virus. In thepresent study, we compared the 2019-nCoV with other coronaviruses, which indicated that batSARS like coronavirus could be a probable ancestor of the novel coronavirus. The proteinsequence similarity of pangolin-hCoV and bat-hCoV with human coronavirus was higher ascompared to their nucleotide similarity denoting the occurrence of more synonymous mutationsin the genome. Phylogenetic and alignment analysis of 591 novel coronaviruses of differentclades from Group I to Group V revealed several mutations and concomitant amino acidchanges. Detailed investigation on nucleotide substitution unfolded 100 substitutions in thecoding region of which 43 were synonymous and 57 were of non-synonymous type. The nonsynonymous substitutions resulting into 57 amino acid changes were found to be distributed overdifferent hCoV proteins with maximum on spike protein. An important di-amino acid change RGto KR was observed in ORF9 protein. Additionally, several interesting features of the novelcoronavirus genome have been highlighted in respect to various other human infecting viruseswhich may explain extreme pathogenicity, infectivity and simultaneously the reason behindfailure of the antiviral therapies. SUMMARY: This study presents a comprehensive phylogenetic analysis of SARS-CoV2 isolates to understand discrete mutations that are occurring between patient samples. The analysis unravel various amino acid mutations in the viral proteins which may provide an explanation for varying treatment efficacies of different inhibitory drugs and a future direction towards a combinatorial treatment therapies based on the kind of mutation in the viral genome.\n",
      "The 2019 novel coronavirus (2019-nCoV or SARS-CoV-2) has spread more rapidly than any other betacoronavirus including SARS-CoV and MERS-CoV. However, the mechanisms responsible for infection and molecular evolution of this virus remained unclear. We collected and analyzed 120 genomic sequences of 2019-nCoV including 11 novel genomes from patients in China. Through comprehensive analysis of the available genome sequences of 2019-nCoV strains, we have tracked multiple inheritable SNPs and determined the evolution of 2019-nCoV relative to other coronaviruses. Systematic analysis of 120 genomic sequences of 2019-nCoV revealed co-circulation of two genetic subgroups with distinct SNPs markers, which can be used to trace the 2019-nCoV spreading pathways to different regions and countries. Although 2019-nCoV, human and bat SARS-CoV share high homologous in overall genome structures, they evolved into two distinct groups with different receptor entry specificities through potential recombination in the receptor binding regions. In addition, 2019-nCoV has a unique four amino acid insertion between S1 and S2 domains of the spike protein, which created a potential furin or TMPRSS2 cleavage site. Our studies provided comprehensive insights into the evolution and spread of the 2019-nCoV. Our results provided evidence suggesting that 2019-nCoV may increase its infectivity through the receptor binding domain recombination and a cleavage site insertion. Novel 2019-nCoV sequences revealed the evolution and specificity of betacoronavirus with possible mechanisms of enhanced infectivity.\n",
      "The novel human coronavirus (SARS-CoV-2) causes the coronavirus disease 2019 (COVID-19) pandemic worldwide. The increasing sequencing data have shown abundant single nucleotide variations in SARS-CoV-2 genome. However, it is difficult to quickly analyze genomic variation and screen key mutations of SARS-CoV-2. In this study, we developed a visual program, named BioAider, for quick and convenient sequence annotation and mutation analysis on multiple genome-sequencing data. Using BioAider, we conducted a comprehensive genome variation analysis on 3,240 sequences of SARS-CoV-2 genome. Herein, we detected 14 substitution hotspots within SARS-CoV-2 genome, including 10 non-synonymous and 4 synonymous ones. Among these hotspots, NSP13-Y541C was predicted to be a crucial substitution which might affect the unwinding activity of NSP13, a key protein for viral replication. Besides, we also found 3 groups of potentially linked substitution hotspots which were worth further study. In particular, we discovered a SR-rich region (aa 184-204) on the N protein of SARS-CoV-2 distinct from SARS-CoV, indicating more complex replication mechanism and unique N-M interaction of SARS-CoV-2. Interestingly, the quantity of SRXX repeat fragments in the SR-rich region well reflected the evolutionary relationship among SARS-CoV-2 and SARS-CoV-2 related animal coronaviruses, providing further evidence of its animal origin. Overall, we developed an efficient tool for rapid identification of mutations, identified substitution hotspots in SARS-CoV-2 genomes, and detected a distinctive polymorphism SR-rich region in N protein. This tool and the detected hotspots could facilitate the viral genomic study and may contribute for screening antiviral target sites.\n",
      "OBJECTIVE: To analyze the evolution and variation of SARS-CoV-2 during the epidemic starting at the end of 2019. METHODS: We downloaded the full-length genome sequence of SARS-CoV-2 from the databases of GISAID and NCBI. Using the software for bioinformatics including MEGA-X, BEAST, and TempEst, we constructed the genomic evolution tree, inferred the time evolution signal of the virus, calculated the tMRCA time of the virus and analyzed the selection pressure of the virus during evolution. RESULTS: The phylogenetic tree showed that SARS-CoV-2 belonged to the Sarbecovirus subgenus of ß Coronavirus genus together with bat coronavirus BetaCoV/bat/Yunnan/RaTG13/2013, bat-SL-CoVZC45, bat-SL-CoVZXC21 and SARS-CoV. The genomic sequences of SARS-CoV-2 isolated from the ongoing epidemic showed a weak time evolution signal with an average tMRCA time of 73 days (95% CI: 38.9-119.3 days). No positive time evolution signal was found between SARS-CoV-2 and BetaCoV/bat/Yunnan/RaTG13/2013, but the former virus had a strong positive temporal evolution relationship with bat-SL-CoVZC45 and SARS-CoV. The major cause for mutations of SARS-CoV-2 was the pressure of purification selection during the epidemic. CONCLUSIONS: SARS-CoV-2 may have emerged as early as November, 2019, originating most likely from bat-associated coronavirus. This finding may provide evidence for tracing the sources and evolution of the virus.\n",
      "OBJECTIVE To analyze the evolution and variation of SARS-CoV-2 during the epidemic starting at the end of 2019. METHODS We downloaded the full-length genome sequence of SARS-CoV-2 from the databases of GISAID and NCBI. Using the software for bioinformatics including MEGA-X, BEAST, and TempEst, we constructed the genomic evolution tree, inferred the time evolution signal of the virus, calculated the tMRCA time of the virus and analyzed the selection pressure of the virus during evolution. RESULTS The phylogenetic tree showed that SARS-CoV-2 belonged to the Sarbecovirus subgenus of β Coronavirus genus together with bat coronavirus BetaCoV/bat/Yunnan/RaTG13/2013, bat-SL-CoVZC45, bat-SL-CoVZXC21 and SARS-CoV. The genomic sequences of SARS-CoV-2 isolated from the ongoing epidemic showed a weak time evolution signal with an average tMRCA time of 73 days (95% CI: 38.9-119.3 days). No positive time evolution signal was found between SARS-CoV-2 and BetaCoV/bat/Yunnan/RaTG13/2013, but the former virus had a strong positive temporal evolution relationship with bat-SL-CoVZC45 and SARS-CoV. The major cause for mutations of SARS-CoV-2 was the pressure of purification selection during the epidemic. CONCLUSIONS SARS-CoV-2 may have emerged as early as November, 2019, originating most likely from bat-associated coronavirus. This finding may provide evidence for tracing the sources and evolution of the virus.\n",
      "The novel coronavirus severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) that recently emerged in China is thought to have a bat origin, as its closest known relative (BatCoV RaTG13) was described previously in horseshoe bats. We analyzed the selective events that accompanied the divergence of SARS-CoV-2 from BatCoV RaTG13. To this end, we applied a population genetics-phylogenetics approach, which leverages within-population variation and divergence from an outgroup. Results indicated that most sites in the viral open reading frames (ORFs) evolved under conditions of strong to moderate purifying selection. The most highly constrained sequences corresponded to some nonstructural proteins (nsps) and to the M protein. Conversely, nsp1 and accessory ORFs, particularly ORF8, had a nonnegligible proportion of codons evolving under conditions of very weak purifying selection or close to selective neutrality. Overall, limited evidence of positive selection was detected. The 6 bona fide positively selected sites were located in the N protein, in ORF8, and in nsp1. A signal of positive selection was also detected in the receptor-binding motif (RBM) of the spike protein but most likely resulted from a recombination event that involved the BatCoV RaTG13 sequence. In line with previous data, we suggest that the common ancestor of SARS-CoV-2 and BatCoV RaTG13 encoded/encodes an RBM similar to that observed in SARS-CoV-2 itself and in some pangolin viruses. It is presently unknown whether the common ancestor still exists and, if so, which animals it infects. Our data, however, indicate that divergence of SARS-CoV-2 from BatCoV RaTG13 was accompanied by limited episodes of positive selection, suggesting that the common ancestor of the two viruses was poised for human infection. IMPORTANCE Coronaviruses are dangerous zoonotic pathogens; in the last 2 decades, three coronaviruses have crossed the species barrier and caused human epidemics. One of these is the recently emerged SARS-CoV-2. We investigated how, since its divergence from a closely related bat virus, natural selection shaped the genome of SARS-CoV-2. We found that distinct coding regions in the SARS-CoV-2 genome evolved under conditions of different degrees of constraint and are consequently more or less prone to tolerate amino acid substitutions. In practical terms, the level of constraint provides indications about which proteins/protein regions are better suited as possible targets for the development of antivirals or vaccines. We also detected limited signals of positive selection in three viral ORFs. However, we warn that, in the absence of knowledge about the chain of events that determined the human spillover, these signals should not be necessarily interpreted as evidence of an adaptation to our species.\n",
      "The novel coronavirus severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) that recently emerged in China is thought to have a bat origin, as its closest known relative (BatCoV RaTG13) was described previously in horseshoe bats. We analyzed the selective events that accompanied the divergence of SARS-CoV-2 from BatCoV RaTG13. To this end, we applied a population genetics-phylogenetics approach, which leverages within-population variation and divergence from an outgroup. Results indicated that most sites in the viral open reading frames (ORFs) evolved under conditions of strong to moderate purifying selection. The most highly constrained sequences corresponded to some nonstructural proteins (nsps) and to the M protein. Conversely, nsp1 and accessory ORFs, particularly ORF8, had a nonnegligible proportion of codons evolving under conditions of very weak purifying selection or close to selective neutrality. Overall, limited evidence of positive selection was detected. The 6 bona fide positively selected sites were located in the N protein, in ORF8, and in nsp1. A signal of positive selection was also detected in the receptor-binding motif (RBM) of the spike protein but most likely resulted from a recombination event that involved the BatCoV RaTG13 sequence. In line with previous data, we suggest that the common ancestor of SARS-CoV-2 and BatCoV RaTG13 encoded/encodes an RBM similar to that observed in SARS-CoV-2 itself and in some pangolin viruses. It is presently unknown whether the common ancestor still exists and, if so, which animals it infects. Our data, however, indicate that divergence of SARS-CoV-2 from BatCoV RaTG13 was accompanied by limited episodes of positive selection, suggesting that the common ancestor of the two viruses was poised for human infection.IMPORTANCE Coronaviruses are dangerous zoonotic pathogens; in the last 2 decades, three coronaviruses have crossed the species barrier and caused human epidemics. One of these is the recently emerged SARS-CoV-2. We investigated how, since its divergence from a closely related bat virus, natural selection shaped the genome of SARS-CoV-2. We found that distinct coding regions in the SARS-CoV-2 genome evolved under conditions of different degrees of constraint and are consequently more or less prone to tolerate amino acid substitutions. In practical terms, the level of constraint provides indications about which proteins/protein regions are better suited as possible targets for the development of antivirals or vaccines. We also detected limited signals of positive selection in three viral ORFs. However, we warn that, in the absence of knowledge about the chain of events that determined the human spillover, these signals should not be necessarily interpreted as evidence of an adaptation to our species.\n",
      "The pandemic caused by the spread of SARS-CoV-2 has led to considerable interest in its evolutionary origin and genome structure. Here, we analyzed mutation patterns in 34 human SARS-CoV-2 isolates and a closely related RaTG13 isolated from Rhinolophus affinis (a horseshoe bat). We also evaluated the CpG dinucleotide contents in SARS-CoV-2 and other human and animal coronavirus genomes. Out of 1136 single nucleotide variations (~4% divergence) between human SARS-CoV-2 and bat RaTG13, 682 (60%) can be attributed to C>U and U>C substitutions, far exceeding other types of substitutions. An accumulation of C>U mutations was also observed in SARS-CoV2 variants that arose within the human population. Globally, the C>U substitutions increased the frequency of codons for hydrophobic amino acids in SARS-CoV-2 peptides, while U>C substitutions decreased it. In contrast to most other coronaviruses, both SARS-CoV-2 and RaTG13 exhibited CpG depletion in their genomes. The data suggest that C-to-U conversion mediated by C deamination played a significant role in the evolution of the SARS-CoV-2 coronavirus. We hypothesize that the high frequency C>U transitions reflect virus adaptation processes in their hosts, and that SARS-CoV-2 could have been evolving for a relatively long period in humans following the transfer from animals before spreading worldwide.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: dqour5jr\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Since 2002, beta coronaviruses (CoV) have caused three zoonotic outbreaks, SARS-CoV in 2002–2003, MERS-CoV in 2012, and the newly emerged SARS-CoV-2 in late 2019. However, little is currently known about the biology of SARS-CoV-2. Here, using SARS-CoV-2 S protein pseudovirus system, we confirm that human angiotensin converting enzyme 2 (hACE2) is the receptor for SARS-CoV-2, find that SARS-CoV-2 enters 293/hACE2 cells mainly through endocytosis, that PIKfyve, TPC2, and cathepsin L are critical for entry, and that SARS-CoV-2 S protein is less stable than SARS-CoV S. Polyclonal anti-SARS S1 antibodies T62 inhibit entry of SARS-CoV S but not SARS-CoV-2 S pseudovirions. Further studies using recovered SARS and COVID-19 patients’ sera show limited cross-neutralization, suggesting that recovery from one infection might not protect against the other. Our results present potential targets for development of drugs and vaccines for SARS-CoV-2.\n",
      "Since 2002, beta coronaviruses (CoV) have caused three zoonotic outbreaks, SARS-CoV in 2002-2003, MERS-CoV in 2012, and the newly emerged SARS-CoV-2 in late 2019. However, little is currently known about the biology of SARS-CoV-2. Here, using SARS-CoV-2 S protein pseudovirus system, we confirm that human angiotensin converting enzyme 2 (hACE2) is the receptor for SARS-CoV-2, find that SARS-CoV-2 enters 293/hACE2 cells mainly through endocytosis, that PIKfyve, TPC2, and cathepsin L are critical for entry, and that SARS-CoV-2 S protein is less stable than SARS-CoV S. Polyclonal anti-SARS S1 antibodies T62 inhibit entry of SARS-CoV S but not SARS-CoV-2 S pseudovirions. Further studies using recovered SARS and COVID-19 patients' sera show limited cross-neutralization, suggesting that recovery from one infection might not protect against the other. Our results present potential targets for development of drugs and vaccines for SARS-CoV-2.\n",
      "The emergence of a novel coronavirus, SARS-CoV-2, at the end of 2019 has resulted in widespread human infections across the globe. While genetically distinct from SARS-CoV, the etiological agent that caused an outbreak of severe acute respiratory syndrome (SARS) in 2003, both coronaviruses exhibit receptor binding domain (RBD) conservation and utilize the same host cell receptor, angiotensin-converting enzyme 2 (ACE2), for virus entry. Therefore, it will be important to test the cross-reactivity of antibodies that have been previously generated against the surface spike (S) glycoprotein of SARS-CoV in order to aid research on the newly emerged SARS-CoV-2. Here, we show that an immunogenic domain in the S2 subunit of SARS-CoV S is highly conserved in multiple strains of SARS-CoV-2. Consistently, four murine monoclonal antibodies (mAbs) raised against this immunogenic SARS-CoV fragment were able to recognise the S protein of SARS-CoV-2 expressed in a mammalian cell line. Importantly, one of them (mAb 1A9) was demonstrated to detect S in SARS-CoV-2-infected cells. To our knowledge, this is the first study showing that mAbs targeting the S2 domain of SARS-CoV can cross-react with SARS-CoV-2 and this observation is consistent with the high sequence conservation in the S2 subunit. These cross-reactive mAbs may serve as tools useful for SARS-CoV-2 research as well as for the development of diagnostic assays for its associated coronavirus disease COVID-19.\n",
      "Severe acute respiratory syndrome coronavirus (SARS-CoV) emerged in 2002 as a highly transmissible pathogenic human betacoronavirus. The viral spike glycoprotein (S) utilizes angiotensin-converting enzyme 2 (ACE2) as a host protein receptor and mediates fusion of the viral and host membranes, making S essential to viral entry into host cells and host species tropism. As SARS-CoV enters host cells, the viral S is believed to undergo a number of conformational transitions as it is cleaved by host proteases and binds to host receptors. We recently developed stabilizing mutations for coronavirus spikes that prevent the transition from the pre-fusion to post-fusion states. Here, we present cryo-EM analyses of a stabilized trimeric SARS-CoV S, as well as the trypsin-cleaved, stabilized S, and its interactions with ACE2. Neither binding to ACE2 nor cleavage by trypsin at the S1/S2 cleavage site impart large conformational changes within stabilized SARS-CoV S or expose the secondary cleavage site, S2′.\n",
      "Since 2002, beta coronaviruses (CoV) have caused three zoonotic outbreaks, SARS-CoV in 2002, MERS-CoV in 2012, and the recent outbreak of SARS-CoV-2 late in 2019 (also named as COVID-19 or novel coronavirus 2019 or nCoV2019. Spike(S) protein, one of the structural proteins of this virus plays key role in receptor (ACE2) binding and thus virus entry. Thus, this protein has attracted scientists for detailed study and therapeutic targeting. As the 2019 novel coronavirus takes its course throughout the world, more and more sequence analyses are been done and genome sequences getting deposited in various databases. From India two clinical isolates have been sequenced and the full genome deposited in GenBank. We have performed sequence analyses of the spike protein of the Indian isolates and compared with that of the Wuhan, China (where the outbreak was first reported). While all the sequences of Wuhan isolates are identical, we found point mutations in the Indian isolates. Out of the two isolates one was found to harbour a mutation in its Receptor binding domain (RBD) at position 407. At this site arginine (a positively charged amino acid) was replaced by isoleucine (a hydrophobic amino acid that is also a C-beta branched amino acid). This mutation has been seen to change the secondary structure of the protein at that region and this can potentially alter receptor ding of the virus. Although this finding needs further validation and more sequencing, the information might be useful in rational drug designing and vaccine engineering.\n",
      "Human angiotensin-converting enzyme 2 (ACE2) is a functional receptor for SARS coronavirus (SARS-CoV). Here we identify the SARS-CoV spike (S)-protein-binding site on ACE2. We also compare S proteins of SARS-CoV isolated during the 2002–2003 SARS outbreak and during the much less severe 2003–2004 outbreak, and from palm civets, a possible source of SARS-CoV found in humans. All three S proteins bound to and utilized palm-civet ACE2 efficiently, but the latter two S proteins utilized human ACE2 markedly less efficiently than did the S protein obtained during the earlier human outbreak. The lower affinity of these S proteins could be complemented by altering specific residues within the S-protein-binding site of human ACE2 to those of civet ACE2, or by altering S-protein residues 479 and 487 to residues conserved during the 2002–2003 outbreak. Collectively, these data describe molecular interactions important to the adaptation of SARS-CoV to human cells, and provide insight into the severity of the 2002–2003 SARS epidemic.\n",
      "Since 2002, β coronaviruses (CoVs) have caused three zoonotic outbreaks, SARS-CoV in 2002, MERS-CoV in 2012, and the recent outbreak of SARS-CoV-2 late in 2019 (also named as COVID-19 or novel coronavirus 2019 or nCoV2019). Spike (S) protein, one of the structural proteins of this virus plays key role in receptor (ACE2) binding and thus virus entry. Thus, this protein has attracted scientists for detailed study and therapeutic targeting. As the nCoV2019 takes its course throughout the world, more and more sequence analyses are being done and genome sequences are being deposited in various databases. From India, two clinical isolates have been sequenced and the full genome has been deposited in GenBank. We have performed sequence analyses of the Spike protein of the Indian isolates and compared with that of the Wuhan, China (where the outbreak was first reported). While all the sequences of Wuhan isolates are identical, we found point mutations in the Indian isolates. Out of the two isolates, one was found to harbor a mutation in its receptor-binding domain (RBD) at position 407. At this site, arginine (a positively charged amino acid) was replaced by isoleucine (a hydrophobic amino acid that is also a C-β branched amino acid). This mutation has been seen to change the secondary structure of the protein at that region and this can potentially alter receptor binding of the virus. Although this finding needs further validation and more sequencing, the information might be useful in rational drug designing and vaccine engineering.\n",
      "Infection of receptor-bearing cells by coronaviruses is mediated by their spike (S) proteins. The coronavirus (SARS-CoV) that causes severe acute respiratory syndrome (SARS) infects cells expressing the receptor angiotensin-converting enzyme 2 (ACE2). Here we show that codon optimization of the SARS-CoV S-protein gene substantially enhanced S-protein expression. We also found that two retroviruses, simian immunodeficiency virus (SIV) and murine leukemia virus, both expressing green fluorescent protein and pseudotyped with SARS-CoV S protein or S-protein variants, efficiently infected HEK293T cells stably expressing ACE2. Infection mediated by an S-protein variant whose cytoplasmic domain had been truncated and altered to include a fragment of the cytoplasmic tail of the human immunodeficiency virus type 1 envelope glycoprotein was, in both cases, substantially more efficient than that mediated by wild-type S protein. Using S-protein-pseudotyped SIV, we found that the enzymatic activity of ACE2 made no contribution to S-protein-mediated infection. Finally, we show that a soluble and catalytically inactive form of ACE2 potently blocked infection by S-protein-pseudotyped retrovirus and by SARS-CoV. These results permit studies of SARS-CoV entry inhibitors without the use of live virus and suggest a candidate therapy for SARS.\n",
      "Severe acute respiratory syndrome CoV-2 (SARS-CoV-2) is currently causing a worldwide pandemic with high morbidity and mortality. Development of animal models that recapitulate important aspects of coronavirus disease 2019 (COVID-19) is critical for the evaluation of vaccines and antivirals, and understanding disease pathogenesis. SARS-CoV-2 has been shown to use the same entry receptor as SARS-CoV-1, human angiotensin-converting enzyme 2 (hACE2)(1-3). Due to amino acid differences between murine and hACE2, inbred mouse strains fail to support high titer viral replication of SARS-CoV-2 virus. Therefore, a number of transgenic and knock-in mouse models, as well as viral vector-mediated hACE2 delivery systems have been developed. Here we compared the K18-hACE2 transgenic model to adenovirus-mediated delivery of hACE2 to the mouse lung. We show that K18-hACE2 mice replicate virus to high titers in both the lung and brain leading to lethality. In contrast, adenovirus-mediated delivery results in viral replication to lower titers limited to the lung, and no clinical signs of infection with a challenge dose of 104 plaque forming units. The K18-hACE2 model provides a stringent model for testing the ability of vaccines and antivirals to protect against disease, whereas the adenovirus delivery system has the flexibility to be used across multiple genetic backgrounds and modified mouse strains.\n",
      "SARS-CoV-2, the causative agent of COVID-19, emerged at the end of 2019 and by mid-June 2020, the virus has spread to at least 215 countries, caused more than 8,000,000 confirmed infections and over 450,000 deaths, and overwhelmed healthcare systems worldwide. Like SARS-CoV, which emerged in 2002 and caused a similar disease, SARS-CoV-2 is a betacoronavirus. Both viruses use human angiotensin-converting enzyme 2 (hACE2) as a receptor to enter cells. However, the SARS-CoV-2 spike (S) glycoprotein has a novel insertion that generates a putative furin cleavage signal and this has been postulated to expand the host range. Two low passage (P) strains of SARS-CoV-2 (Wash1: P4 and Munich: P1) were cultured twice in Vero-E6 cells and characterized virologically. Sanger and MinION sequencing demonstrated significant deletions in the furin cleavage signal of Wash1: P6 and minor variants in the Munich: P3 strain. Cleavage of the S glycoprotein in SARS-CoV-2-infected Vero-E6 cell lysates was inefficient even when an intact furin cleavage signal was present. Indirect immunofluorescence demonstrated the S glycoprotein reached the cell surface. Since the S protein is a major antigenic target for the development of neutralizing antibodies we investigated the development of neutralizing antibody titers in serial serum samples obtained from COVID-19 human patients. These were comparable regardless of the presence of an intact or deleted furin cleavage signal. These studies illustrate the need to characterize virus stocks meticulously prior to performing either in vitro or in vivo pathogenesis studies.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 69toerzi\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "With the current trajectory of the 2019‐nCoV outbreak unknown, public health and medicinal measures will both be needed to contain spreading of the virus and to optimize patient outcomes. Although little is known about the virus, an examination of the genome sequence shows strong homology with its better‐studied cousin, SARS‐CoV. The spike protein used for host cell infection shows key nonsynonymous mutations that might hamper the efficacy of previously developed therapeutics but remains a viable target for the development of biologics and macrocyclic peptides. Other key drug targets, including RNA‐dependent RNA polymerase and coronavirus main proteinase (3CLpro), share a strikingly high (>95 %) homology to SARS‐CoV. Herein, we suggest four potential drug candidates (an ACE2‐based peptide, remdesivir, 3CLpro‐1 and a novel vinylsulfone protease inhibitor) that could be used to treat patients suffering with the 2019‐nCoV. We also summarize previous efforts into drugging these targets and hope to help in the development of broad‐spectrum anti‐coronaviral agents for future epidemics.\n",
      "With the current trajectory of the 2019-nCoV outbreak unknown, public health and medicinal measures will both be needed to contain spreading of the virus and to optimize patient outcomes. Although little is known about the virus, an examination of the genome sequence shows strong homology with its better-studied cousin, SARS-CoV. The spike protein used for host cell infection shows key nonsynonymous mutations that might hamper the efficacy of previously developed therapeutics but remains a viable target for the development of biologics and macrocyclic peptides. Other key drug targets, including RNA-dependent RNA polymerase and coronavirus main proteinase (3CLpro), share a strikingly high (>95 %) homology to SARS-CoV. Herein, we suggest four potential drug candidates (an ACE2-based peptide, remdesivir, 3CLpro-1 and a novel vinylsulfone protease inhibitor) that could be used to treat patients suffering with the 2019-nCoV. We also summarize previous efforts into drugging these targets and hope to help in the development of broad-spectrum anti-coronaviral agents for future epidemics.\n",
      "With the current trajectory of the 2019-nCoV outbreak unknown, public health and medicinal measures will both be needed to contain spreading of the virus and to optimize patient outcomes. While little is known about the virus, an examination of the genome sequence shows strong homology with its more well-studied cousin, SARS-CoV. The spike protein used for host cell infection shows key nonsynonymous mutations which may hamper efficacy of previously developed therapeutics but remains a viable target for the development of biologics and macrocyclic peptides. Other key drug targets, including RdRp and 3CLpro, share a strikingly high (>95%) homology to SARS-CoV. Herein, we suggest 4 potential drug candidates (an ACE2-based peptide, remdesivir, 3CLpro-1 and a novel vinylsulfone protease inhibitor) that can be used to treat patients suffering with the 2019-nCoV. We also summarize previous efforts into drugging these targets and hope to help in the development of broad spectrum anti-coronaviral agents for future epidemics.\n",
      "Wuhan coronavirus, called 2019-nCoV, is a newly emerged virus that infected more than 9692 people and leads to more than 213 fatalities by January 30, 2020. Currently, there is no effective treatment for this epidemic. However, the viral protease of a coronavirus is well-known to be essential for its replication and thus is an effective drug target. Fortunately, the sequence identity of the 2019-nCoV protease and that of severe-acute respiratory syndrome virus (SARS-CoV) is as high as 96.1%. We show that the protease inhibitor binding sites of 2019-nCoV and SARS-CoV are almost identical, which means all potential anti-SARS-CoV chemotherapies are also potential 2019-nCoV drugs. Here, we report a family of potential 2019-nCoV drugs generated by a machine intelligence-based generative network complex (GNC). The potential effectiveness of treating 2019-nCoV by using some existing HIV drugs is also analyzed.\n",
      "2019 Novel Coronavirus (2019-nCoV) is a virus identified as the cause of the outbreak of pneumonia first detected in Wuhan, China. Investigations on the transmissibility, severity, and other features associated with this virus are ongoing. Currently, there is no vaccine or therapeutic antibody to prevent the infection, and more time is required to develop an effective immune strategy against the pathogen. In contrast, specific inhibitors targeting the key protease involved in replication and proliferation of the virus are the most effective means to alleviate the epidemic. The main protease of SARS-CoV is essential for the life cycle of the virus, which showed 96.1% of similarity with the main proteaseof 2019-nCoV, is considered to be an attractive target for drug development. In this study, we have identified 4 small molecular drugs with high binding capacity with SARS-CoV main protease by high-throughput screening based on the 8,000 clinical drug libraries, all these drugs have been widely used in clinical applications with guaranteed safety, which may serve as promising candidates to treat the infection of 2019-nCoV.\n",
      "The recent outbreak of coronavirus disease 2019 (COVID-19) caused by SARS-CoV-2 in December 2019 raised global health concerns. The viral 3-chymotrypsin-like cysteine protease (3CLpro) enzyme controls coronavirus replication and is essential for its life cycle. 3CLpro is a proven drug discovery target in the case of severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV). Recent studies revealed that the genome sequence of SARS-CoV-2 is very similar to that of SARS-CoV. Therefore, herein, we analysed the 3CLpro sequence, constructed its 3D homology model, and screened it against a medicinal plant library containing 32,297 potential anti-viral phytochemicals/traditional Chinese medicinal compounds. Our analyses revealed that the top nine hits might serve as potential anti- SARS-CoV-2 lead molecules for further optimisation and drug development process to combat COVID-19.\n",
      "Abstract The recent outbreak of coronavirus disease 2019 (COVID-19) caused by SARS-CoV-2 in December 2019 raised global health concerns. The viral 3-chymotrypsin-like cysteine protease (3CLpro) enzyme controls coronavirus replication and is essential for its life cycle. 3CLpro is a proven drug discovery target in the case of severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV). Recent studies revealed that the genome sequence of SARS-CoV-2 is very similar to that of SARS-CoV. Therefore, herein, we analysed the 3CLpro sequence, constructed its 3D homology model, and screened it against a medicinal plant library containing 32,297 potential anti-viral phytochemicals/traditional Chinese medicinal compounds. Our analyses revealed that the top nine hits might serve as potential anti- SARS-CoV-2 lead molecules for further optimisation and drug development process to combat COVID-19.\n",
      "The recent emergence of a novel coronavirus (SARS-CoV-2) is causing a severe global health threat characterized by severe acute respiratory syndrome (Covid-19). At the moment, there is no specific treatment for this disease, and vaccines are still under development. The structural protein Spike is essential for virus infection and has been used as the main target for vaccine and serological diagnosis test development. We analysed 2363 sequences of the Spike protein from SARS-CoV-2 isolates and identified variability in 44 amino acid residues and their worldwide distribution in all continents. We used the three-dimensional structure of the homo-trimer model to predict conformational epitopes of B-cell, and sequence of Spike protein Wuhan-Hu-1 to predict linear epitopes of T-Cytotoxic and T-Helper cells. We identified 45 epitopes with amino acid variations. Finally, we showed the distribution of mutations within the epitopes. Our findings can help researches to identify more efficient strategies for the development of vaccines, therapies, and serological diagnostic tests based on the Spike protein of Sars-Cov-2.\n",
      "The coronavirus disease 2019 (COVID-19) pandemic has focused attention on the need to develop effective therapies against the causative agent, SARS-CoV-2, and also against other pathogenic coronaviruses (CoV) that have emerged in the past or might appear in future. Researchers are therefore focusing on steps in the CoV replication cycle that may be vulnerable to inhibition by broad-spectrum or specific antiviral agents. The conserved nature of the fusion domain and mechanism across the CoV family make it a valuable target to elucidate and develop pan-CoV therapeutics. In this article, we review the role of the CoV spike protein in mediating fusion of the viral and host cell membranes, summarizing the results of research on SARS-CoV, MERS-CoV, and recent peer-reviewed studies of SARS-CoV-2, and suggest that the fusion mechanism be investigated as a potential antiviral target. We also provide a supplemental file containing background information on the biology, epidemiology, and clinical features of all human-infecting coronaviruses, along with a phylogenetic tree of these coronaviruses.\n",
      "The 2019-novel coronavirus (nCoV) is a major source of disaster in the 21th century. However, the lack of specific drugs to prevent/treat an attack is a major need at this current point of time. In this regard, we conducted a systematic review to identify major druggable targets in coronavirus (CoV). We searched PubMed and RCSB database with keywords HCoV, NCoV, corona virus, SERS-CoV, MERS-CoV, 2019-nCoV, crystal structure, X-ray crystallography structure, NMR structure, target, and drug target till Feb 3, 2020. The search identified seven major targets (spike protein, envelop protein, membrane protein, protease, nucleocapsid protein, hemagglutinin esterase, and helicase) for which drug design can be considered. There are other 16 nonstructural proteins (NSPs), which can also be considered from the drug design perspective. The major structural proteins and NSPs may serve an important role from drug design perspectives. However, the occurrence of frequent recombination events is a major deterrent factor toward the development of CoV-specific vaccines/drugs.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: lnys6iuu\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "what is the origin of COVID-19\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "The ongoing global pandemic of infection disease COVID-19 caused by the 2019 novel coronavirus (SARS-COV-2, formerly 2019-nCoV) presents critical threats to public health and the economy since it was identified in China, December 2019. The genome of SARS-CoV-2 had been sequenced and structurally annotated, yet little is known of the intrinsic organization and evolution of the genome. To this end, we present a mathematical method for the genomic spectrum, a kind of barcode, of SARS-CoV-2 and common human coronaviruses. The genomic spectrum is constructed according to the periodic distributions of nucleotides, and therefore reflects the unique characteristics of the genome. The results demonstrate that coronavirus SARS-CoV-2 exhibits dinucleotide TT islands in the non-structural proteins 3, 4, 5, and 6. Further analysis of the dinucleotide regions suggests that the dinucleotide repeats are increased during evolution and may confer the evolutionary fitness of the virus. The special dinucleotide regions in the SARS-CoV-2 genome identified in this study may become diagnostic and pharmaceutical targets in monitoring and curing the COVID-19 disease.\n",
      "Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2), which was first reported in Wuhan, China in November 2019 has developed into a pandemic since March 2020, causing substantial human casualties and economic losses. Studies on SARS-CoV-2 are being carried out at an unprecedented rate to tackle this threat. Genomics studies, in particular, are indispensable to elucidate the dynamic nature of the RNA genome of SARS-CoV-2. RNA viruses are marked by their unique ability to undergo high rates of mutation in their genome, much more frequently than their hosts, which diversifies their strengths qualifying them to elude host immune response and amplify drug resistance. In this study, we sequenced and analyzed the genomic information of the SARS-CoV-2 isolates from two infected Indian patients and explored the possible implications of point mutations in its biology. In addition to multiple point mutations, we found a remarkable similarity between relatively common mutations of 36-nucleotide deletion in ORF8 of SARS-CoV-2. Our results corroborate with the earlier reported 29-nucleotide deletion in SARS, which was frequent during the early stage of human-to-human transmission. The results will be useful to understand the biology of SARS-CoV-2 and itsattenuation for vaccine development.\n",
      "The emerging global infectious COVID-19 coronavirus disease by novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) presents critical threats to global public health and the economy since it was identified in late December 2019 in China. The virus has gone through various pathways of evolution. For understanding the evolution and transmission of SARS-CoV-2, genotyping of virus isolates is of great importance. We present an accurate method for effectively genotyping SARS-CoV-2 viruses using complete genomes. The method employs the multiple sequence alignments of the genome isolates with the SARS-CoV-2 reference genome. The SNP genotypes are then measured by Jaccard distances to track the relationship of virus isolates. The genotyping analysis of SARS-CoV-2 isolates from the globe reveals that specific multiple mutations are the predominated mutation type during the current epidemic. Our method serves a promising tool for monitoring and tracking the epidemic of pathogenic viruses in their gradual and local genetic variations. The genotyping analysis shows that the genes encoding the S proteins and RNA polymerase, RNA primase, and nucleoprotein, undergo frequent mutations. These mutations are critical for vaccine development in disease control.\n",
      "The emerging global infectious COVID-19 disease by novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) presents critical threats to global public health and the economy since it was identified in late December 2019 in China. The virus has gone through various pathways of evolution. To understand the evolution and transmission of SARS-CoV-2, genotyping of virus isolates is of great importance. This study presents an accurate method for effectively genotyping SARS-CoV-2 viruses using complete genomes. The method employs the multiple sequence alignments of the genome isolates with the SARS-CoV-2 reference genome. The single-nucleotide polymorphism (SNP) genotypes are then measured by Jaccard distances to track the relationship of virus isolates. The genotyping analysis of SARS-CoV-2 isolates from the globe reveals that specific multiple mutations are the predominated mutation type during the current epidemic. The proposed method serves an effective tool for monitoring and tracking the epidemic of pathogenic viruses in their global and local genetic variations. The genotyping analysis shows that the genes encoding the S proteins and RNA polymerase, RNA primase, and nucleoprotein, undergo frequent mutations. These mutations are critical for vaccine development in disease control.\n",
      "The 2019 novel coronavirus (renamed SARS-CoV-2, and generally referred to as the COVID-19 virus) has spread to 184 countries with over 1.5 million confirmed cases. Such major viral outbreaks demand early elucidation of taxonomic classification and origin of the virus genomic sequence, for strategic planning, containment, and treatment. This paper identifies an intrinsic COVID-19 virus genomic signature and uses it together with a machine learning-based alignment-free approach for an ultra-fast, scalable, and highly accurate classification of whole COVID-19 virus genomes. The proposed method combines supervised machine learning with digital signal processing (MLDSP) for genome analyses, augmented by a decision tree approach to the machine learning component, and a Spearman's rank correlation coefficient analysis for result validation. These tools are used to analyze a large dataset of over 5000 unique viral genomic sequences, totalling 61.8 million bp, including the 29 COVID-19 virus sequences available on January 27, 2020. Our results support a hypothesis of a bat origin and classify the COVID-19 virus as Sarbecovirus, within Betacoronavirus. Our method achieves 100% accurate classification of the COVID-19 virus sequences, and discovers the most relevant relationships among over 5000 viral genomes within a few minutes, ab initio, using raw DNA sequence data alone, and without any specialized biological knowledge, training, gene or genome annotations. This suggests that, for novel viral and pathogen genome sequences, this alignment-free whole-genome machine-learning approach can provide a reliable real-time option for taxonomic classification.\n",
      "The 2019 novel coronavirus (renamed SARS-CoV-2, and generally referred to as the COVID-19 virus) has spread to 184 countries with over 1.5 million confirmed cases. Such major viral outbreaks demand early elucidation of taxonomic classification and origin of the virus genomic sequence, for strategic planning, containment, and treatment. This paper identifies an intrinsic COVID-19 virus genomic signature and uses it together with a machine learning-based alignment-free approach for an ultra-fast, scalable, and highly accurate classification of whole COVID-19 virus genomes. The proposed method combines supervised machine learning with digital signal processing (MLDSP) for genome analyses, augmented by a decision tree approach to the machine learning component, and a Spearman’s rank correlation coefficient analysis for result validation. These tools are used to analyze a large dataset of over 5000 unique viral genomic sequences, totalling 61.8 million bp, including the 29 COVID-19 virus sequences available on January 27, 2020. Our results support a hypothesis of a bat origin and classify the COVID-19 virus as Sarbecovirus, within Betacoronavirus. Our method achieves 100% accurate classification of the COVID-19 virus sequences, and discovers the most relevant relationships among over 5000 viral genomes within a few minutes, ab initio, using raw DNA sequence data alone, and without any specialized biological knowledge, training, gene or genome annotations. This suggests that, for novel viral and pathogen genome sequences, this alignment-free whole-genome machine-learning approach can provide a reliable real-time option for taxonomic classification.\n",
      "SARS-CoV-2 is an RNA virus of the Coronaviridae family that is the causal pathogen of the ongoing Coronavirus Disease 2019 pandemic. There are currently no antiviral drugs or vaccines to treat COVID-19, and the failure to identify effective interventions can be blamed on our incomplete understanding of the nature of this virus and its host cell infection process. Here, we experimentally determined structural maps of the SARS-CoV-2 RNA genome in infected human cells and also characterized in vitro refolded RNA structures for SARS-CoV-2 and 6 other coronaviruses. Our in vivo data confirms several structural elements predicted from theoretical analysis and goes much further in revealing many previously unknown structural features that functionally impact viral translation and discontinuous transcription in cells. Importantly, we harnessed our in vivo structure data alongside a deep-learning tool and accurately predicted several dozen functionally related host cell proteins that bind to the SARS-CoV-2 RNA genome, none of which were known previously. Thus, our in vivo structural study lays a foundation for coronavirus RNA biology and indicates promising directions for the rapid development of therapeutics to treat COVID-19. HIGHLIGHTS We mapped the in vivo structure and built secondary structural models of the SARS-CoV-2 RNA genome We discovered functionally impactful structural features in the RNA genomes of multiple coronaviruses We predicted and validated host cell proteins that bind to the SARS-CoV-2 RNA genome based on our in vivo RNA structural data using a deep-learning tool\n",
      "The ongoing pandemic of the coronavirus disease 2019 (COVID-19) is an infectious disease caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV2). We have performed an integrated sequence-based analysis of SARS-CoV2 genomes from different geographical locations in order to identify its unique features absent in SARS-CoV and other related coronavirus family genomes, conferring unique infection, facilitation of transmission, virulence and immunogenic features to the virus. The phylogeny of the genomes yields some interesting results. Systematic gene level mutational analysis of the genomes has enabled us to identify several unique features of the SARS-CoV2 genome, which includes a unique mutation in the spike surface glycoprotein (A930V (24351C>T)) in the Indian SARS-CoV2, absent in other strains studied here. We have also predicted the impact of the mutations in the spike glycoprotein function and stability, using computational approach. To gain further insights into host responses to viral infection, we predict that antiviral host-miRNAs may be controlling the viral pathogenesis. Our analysis reveals nine host miRNAs which can potentially target SARS-CoV2 genes. Interestingly, the nine miRNAs do not have targets in SARS and MERS genomes. Also, hsa-miR-27b is the only unique miRNA which has a target gene in the Indian SARS-CoV2 genome. We also predicted immune epitopes in the genomes\n",
      "A new severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) associated with human to human transmission and extreme human sickness has been as of late announced from the city of Wuhan in China. Our objectives were to mutation analysis between recently reported genomes at various times and locations and to characterize the genomic structure of SARS-CoV-2 using bioinformatics programs. Information on the variation of viruses is of considerable medical and biological impacts on the prevention, diagnosis, and therapy of infectious diseases. To understand the genomic structure and variations of the SARS-CoV-2. The study analyzed 95 SARS-CoV-2 complete genome sequences available in GenBank, National MicrobiologyData Center (NMDC) and NGDC Genome Warehouse from December-2019 until 05 of April-2020. The genomic signature analysis demonstrates that a strong association between the time of sample collection, location of sample and accumulation of genetic diversity. We found 116 mutations, the three most common mutations were 8782C>T in ORF1ab gene, 28144T>C in ORF8 gene and 29095C>T in the N gene. The mutations might affect the severity and spread of the SARS-CoV-2. The finding heavily supports an intense requirement for additional prompt, inclusive investigations that combine genomic detail, epidemiological information and graph records of the clinical features of patients with COVID-19.\n",
      "Corona virus disease 2019 SARS-CoV-2 (COVID-19) is a zoonotic virus causing a variety of severe of respiratory diseases. SARS-CoV-2 is closest to SARS-CoV and MERS-CoV in structure. The highly prevalence of COVID-19 is due to the lack onset of symptoms. Our study aimed to present an overview of the virus in terms of structure, epidemiology, symptoms, treatment, and prevention. Conduct the differences of whole genome sequence and some viral proteins to determine the gap and the change alternation of nucleotides and amino acids sequences. We evaluate 11 complete genome sequence of different coronavirus using BAST and MAFFT software. We also selected 7 types of structural proteins. We were conclude that COVID-19 might be created new mutations specifically in glycoproteins hence requires caution and complete preparation by health authorities.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: jujrazsr\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Abstract The emerging global infectious COVID-19 disease by novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) presents critical threats to global public health and the economy since it was identified in late December 2019 in China. The virus has gone through various pathways of evolution. To understand the evolution and transmission of SARS-CoV-2, genotyping of virus isolates is of great importance. This study presents an accurate method for effectively genotyping SARS-CoV-2 viruses using complete genomes. The method employs the multiple sequence alignments of the genome isolates with the SARS-CoV-2 reference genome. The single-nucleotide polymorphism (SNP) genotypes are then measured by Jaccard distances to track the relationship of virus isolates. The genotyping analysis of SARS-CoV-2 isolates from the globe reveals that specific multiple mutations are the predominated mutation type during the current epidemic. The proposed method serves an effective tool for monitoring and tracking the epidemic of pathogenic viruses in their global and local genetic variations. The genotyping analysis shows that the genes encoding the S proteins and RNA polymerase, RNA primase, and nucleoprotein, undergo frequent mutations. These mutations are critical for vaccine development in disease control.\n",
      "The emerging global infectious COVID-19 coronavirus disease by novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) presents critical threats to global public health and the economy since it was identified in late December 2019 in China. The virus has gone through various pathways of evolution. For understanding the evolution and transmission of SARS-CoV-2, genotyping of virus isolates is of great importance. We present an accurate method for effectively genotyping SARS-CoV-2 viruses using complete genomes. The method employs the multiple sequence alignments of the genome isolates with the SARS-CoV-2 reference genome. The SNP genotypes are then measured by Jaccard distances to track the relationship of virus isolates. The genotyping analysis of SARS-CoV-2 isolates from the globe reveals that specific multiple mutations are the predominated mutation type during the current epidemic. Our method serves a promising tool for monitoring and tracking the epidemic of pathogenic viruses in their gradual and local genetic variations. The genotyping analysis shows that the genes encoding the S proteins and RNA polymerase, RNA primase, and nucleoprotein, undergo frequent mutations. These mutations are critical for vaccine development in disease control.\n",
      "The emerging global infectious COVID-19 disease by novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) presents critical threats to global public health and the economy since it was identified in late December 2019 in China. The virus has gone through various pathways of evolution. To understand the evolution and transmission of SARS-CoV-2, genotyping of virus isolates is of great importance. This study presents an accurate method for effectively genotyping SARS-CoV-2 viruses using complete genomes. The method employs the multiple sequence alignments of the genome isolates with the SARS-CoV-2 reference genome. The single-nucleotide polymorphism (SNP) genotypes are then measured by Jaccard distances to track the relationship of virus isolates. The genotyping analysis of SARS-CoV-2 isolates from the globe reveals that specific multiple mutations are the predominated mutation type during the current epidemic. The proposed method serves an effective tool for monitoring and tracking the epidemic of pathogenic viruses in their global and local genetic variations. The genotyping analysis shows that the genes encoding the S proteins and RNA polymerase, RNA primase, and nucleoprotein, undergo frequent mutations. These mutations are critical for vaccine development in disease control.\n",
      "Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2), which was first reported in Wuhan, China in November 2019 has developed into a pandemic since March 2020, causing substantial human casualties and economic losses. Studies on SARS-CoV-2 are being carried out at an unprecedented rate to tackle this threat. Genomics studies, in particular, are indispensable to elucidate the dynamic nature of the RNA genome of SARS-CoV-2. RNA viruses are marked by their unique ability to undergo high rates of mutation in their genome, much more frequently than their hosts, which diversifies their strengths qualifying them to elude host immune response and amplify drug resistance. In this study, we sequenced and analyzed the genomic information of the SARS-CoV-2 isolates from two infected Indian patients and explored the possible implications of point mutations in its biology. In addition to multiple point mutations, we found a remarkable similarity between relatively common mutations of 36-nucleotide deletion in ORF8 of SARS-CoV-2. Our results corroborate with the earlier reported 29-nucleotide deletion in SARS, which was frequent during the early stage of human-to-human transmission. The results will be useful to understand the biology of SARS-CoV-2 and itsattenuation for vaccine development.\n",
      "OBJECTIVES: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan, China, in December 2019 and has been rapidly spreading worldwide. Although the causal relationship among mutations and the features of SARS-CoV-2 such as rapid transmission, pathogenicity, and tropism, remains unclear, our results of genomic mutations in SARS-CoV-2 may help to interpret the interaction between genomic characterization in SARS-CoV-2 and infectivity with the host. METHODS: A total of 4,254 genomic sequences of SARS-CoV-2 were collected from the Global Initiative on Sharing all Influenza Data (GISAID). Multiple sequence alignment for phylogenetic analysis and comparative genomic approach for mutation analysis were conducted using Molecular Evolutionary Genetics Analysis (MEGA), and an in-house program based on Perl language, respectively. RESULTS: Phylogenetic analysis of SARS-CoV-2 strains indicated that there were 3 major clades including S, V, and G, and 2 subclades (G.1 and G.2). There were 767 types of synonymous and 1,352 types of non-synonymous mutation. ORF1a, ORF1b, S, and N genes were detected at high frequency, whereas ORF7b and E genes exhibited low frequency. In the receptor-binding domain (RBD) of the S gene, 11 non-synonymous mutations were observed in the region adjacent to the angiotensin converting enzyme 2 (ACE2) binding site. CONCLUSION: It has been reported that the rapid infectivity and transmission of SARS-CoV-2 associated with host receptor affinity are derived from several mutations in its genes. Without these genetic mutations to enhance evolutionary adaptation, species recognition, host receptor affinity, and pathogenicity, it would not survive. It is expected that our results could provide an important clue in understanding the genomic characteristics of SARS-CoV-2.\n",
      "Tremendous effort has been given to the development of diagnostic tests, preventive vaccines, and therapeutic medicines for coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Much of this development has been based on the reference genome collected on January 5, 2020. Based on the genotyping of 15â¯140 genome samples collected up to June 1, 2020, we report that SARS-CoV-2 has undergone 8309 single mutations which can be clustered into six subtypes. We introduce mutation ratio and mutation h-index to characterize the protein conservativeness and unveil that SARS-CoV-2 envelope protein, main protease, and endoribonuclease protein are relatively conservative, while SARS-CoV-2 nucleocapsid protein, spike protein, and papain-like protease are relatively nonconservative. In particular, we have identified mutations on 40% of nucleotides in the nucleocapsid gene in the population level, signaling potential impacts on the ongoing development of COVID-19 diagnosis, vaccines, and antibody and small-molecular drugs.\n",
      "Tremendous effort has been given to the development of diagnostic tests, preventive vaccines, and therapeutic medicines for coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Much of this development has been based on the reference genome collected on January 5, 2020. Based on the genotyping of 6156 genome samples collected up to April 24, 2020, we report that SARS-CoV-2 has had 4459 alarmingly mutations which can be clustered into five subtypes. We introduce mutation ratio and mutation $h$-index to characterize the protein conservativeness and unveil that SARS-CoV-2 envelope protein, main protease, and endoribonuclease protein are relatively conservative, while SARS-CoV-2 nucleocapsid protein, spike protein, and papain-like protease are relatively non-conservative. In particular, the nucleocapsid protein has more than half its genes changed in the past few months, signaling devastating impacts on the ongoing development of COVID-19 diagnosis, vaccines, and drugs.\n",
      "Abstract A new severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) associated with human to human transmission and extreme human sickness has been as of late announced from the city of Wuhan in China. Our objectives were to mutation analysis between recently reported genomes at various times and locations and to characterize the genomic structure of SARS-CoV-2 using bioinformatics programs. Information on the variation of viruses is of considerable medical and biological impacts on the prevention, diagnosis, and therapy of infectious diseases. To understand the genomic structure and variations of the SARS-CoV-2. The study analyzed 95 SARS-CoV-2 complete genome sequences available in GenBank, National MicrobiologyData Center (NMDC) and NGDC Genome Warehouse from December-2019 until 05 of April-2020. The genomic signature analysis demonstrates that a strong association between the time of sample collection, location of sample and accumulation of genetic diversity. We found 116 mutations, the three most common mutations were 8782C>T in ORF1ab gene, 28144T>C in ORF8 gene and 29095C>T in the N gene. The mutations might affect the severity and spread of the SARS-CoV-2. The finding heavily supports an intense requirement for additional prompt, inclusive investigations that combine genomic detail, epidemiological information and graph records of the clinical features of patients with COVID-19.\n",
      "[Image: see text] Tremendous effort has been given to the development of diagnostic tests, preventive vaccines, and therapeutic medicines for coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Much of this development has been based on the reference genome collected on January 5, 2020. Based on the genotyping of 15 140 genome samples collected up to June 1, 2020, we report that SARS-CoV-2 has undergone 8309 single mutations which can be clustered into six subtypes. We introduce mutation ratio and mutation h-index to characterize the protein conservativeness and unveil that SARS-CoV-2 envelope protein, main protease, and endoribonuclease protein are relatively conservative, while SARS-CoV-2 nucleocapsid protein, spike protein, and papain-like protease are relatively nonconservative. In particular, we have identified mutations on 40% of nucleotides in the nucleocapsid gene in the population level, signaling potential impacts on the ongoing development of COVID-19 diagnosis, vaccines, and antibody and small-molecular drugs.\n",
      "The COVID-19 pandemic is the most significant public health issue in recent history. Its causal agent, SARS-CoV-2, has evolved rapidly since its first emergence in December 2019. Mutations in the viral genome have critical impacts on the adaptation of viral strains to the local environment, and may alter the characteristics of viral transmission, disease manifestation, and the efficacy of treatment and vaccination. Using the complete sequences of 1,932 SARS-CoV-2 genomes, we examined the genomic, geographic and temporal distributions of aged, new, and frequent mutations of SARS-CoV-2, and identified six phylogenetic clusters of the strains, which also exhibit a geographic preference in different continents. Mutations in the form of single nucleotide variations (SNVs) provide a direct interpretation for the six phylogenetic clusters. Linkage disequilibrium, haplotype structure, evolutionary process, global distribution of mutations unveiled a sketch of the mutational history. Additionally, we found a positive correlation between the average mutation count and case fatality, and this correlation had strengthened with time, suggesting an important role of SNVs on disease outcomes. This study suggests that SNVs may become an important consideration in virus detection, clinical treatment, drug design, and vaccine development to avoid target shifting, and that continued isolation and sequencing is a crucial component in the fight against this pandemic. Significance Statement Mutation is the driving force of evolution for viruses like SARS-CoV-2, the causal agent of COVID-19. In this study, we discovered that the genome of SARS-CoV-2 is changing rapidly from the originally isolated form. These mutations have been spreading around the world and caused more than 2.5 million of infected cases and 170 thousands of deaths. We found that fourteen frequent mutations identified in this study can characterize the six main clusters of SARS-CoV-2 strains. In addition, we found the mutation burden is positively correlated with the fatality of COVID-19 patients. Understanding mutations in the SARS-CoV-2 genome will provide useful insight for the design of treatment and vaccination.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 3aqh6zxf\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "what causes death from Covid-19?\n",
      "are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "In this rapid commentary, a mini-review is given of the present state-of-knowledge regarding the etiology and epidemiology of the new coronavirus 2019-nCoV and the risks for developing Acute respiratory distress syndrome (ARDS). The available knowledge on the viral genomics, molecular biology and pathogenicity of viruses of the Coronaviridae family and other Nidovirales, forms a helpful template for understanding the present pandemic outbreak. However, important questions remain unanswered about the underlying mechanism causing the very high case fatality ratios (CFR) and mechanisms regarding severe reactions like ARDS, fatal cardiac and renal failures, associated with a number of important comorbidity factors. Immunological reactions to lung alveoles in particular (involving lung macrophages and alveolar epithelial cell damage) in late phase ARDS in SARS-like CoV diseases, so far may not have received enough attention. Finally a shortlist of questions for high priority further research is suggested.\n",
      "In this rapid commentary, a mini-review is given of the present state-of-knowledge regarding the etiology and epidemiology of the new coronavirus 2019-nCoV and the risks for developing Acute respiratory distress syndrome (ARDS). The available knowledge on the viral genomics, molecular biology and pathogenicity of viruses of the Coronaviridae family and other Nidovirales, forms a helpful template for understanding the present pandemic outbreak. However, important questions remain unanswered about the underlying mechanism causing the very high case fatality ratios (CFR) and mechanisms regarding severe reactions like ARDS, fatal cardiac and renal failures, associated with a number of important comorbidity factors. Immunological reactions to lung alveoles in particular (involving lung macrophages and alveolar epithelial cell damage) in late phase ARDS in SARS-like CoV diseases, so far may not have received enough attention. Finally a shortlist of questions for high priority further research is suggested.\n",
      "Coronavirus disease-2019 (COVID-19) is caused by the newly emerged virus severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and was recently declared as a pandemic by the World Health Organization. In its severe form, the disease is characterized by acute respiratory distress syndrome, and there are no targeted intervention strategies to treat or prevent it. The immune response is thought to both contribute to the pathogenesis of disease and provide protection during its resolution. Thus, understanding the immune response to SARS-CoV-2 is of the utmost importance for developing and testing vaccines and therapeutics. In this review, we discuss the earliest knowledge and hypotheses of the mechanisms of immune pathology in the lung during acute infection as well at the later stages of disease resolution, recovery, and immune memory formation.\n",
      "Coronavirus disease-2019 (COVID-19) is caused by the newly emerged virus severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and was recently declared as a pandemic by the World Health Organization. In its severe form, the disease is characterized by acute respiratory distress syndrome, and there are no targeted intervention strategies to treat or prevent it. The immune response is thought to both contribute to the pathogenesis of disease and provide protection during its resolution. Thus, understanding the immune response to SARS-CoV-2 is of the utmost importance for developing and testing vaccines and therapeutics. In this review, we discuss the earliest knowledge and hypotheses of the mechanisms of immune pathology in the lung during acute infection as well at the later stages of disease resolution, recovery, and immune memory formation.\n",
      "Abstract Coronavirus Disease 2019 (COVID-19) was first identified in China at the end of 2019. Acute respiratory distress syndrome (ARDS) represents the most common and serious complication of COVID-19. Cytokine storms are a pathophysiological feature of COVID-19 and play an important role in distinguishing hyper-inflammatory subphenotypes of ARDS. Accordingly, in this review, we focus on hyper-inflammatory host responses in ARDS that play a critical role in the differentiated development of COVID-19. Furthermore, we discuss inflammation-related indicators that have the potential to identify hyper-inflammatory subphenotypes of COVID-19, especially for those with a high risk of ARDS. Finally, we explore the possibility of improving the quality of monitoring and treatment of COVID-19 patients and in reducing the incidence of critical illness and mortality via better distinguishing hyper- and hypo-inflammatory subphenotypes of COVID-19.\n",
      "The coronavirus disease 2019 (COVID-19) pandemic, caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has affected millions of people worldwide, igniting an unprecedented effort from the scientific community to understand the biological underpinning of COVID19 pathophysiology. In this Review, we summarize the current state of knowledge of innate and adaptive immune responses elicited by SARS-CoV-2 infection and the immunological pathways that likely contribute to disease severity and death. We also discuss the rationale and clinical outcome of current therapeutic strategies as well as prospective clinical trials to prevent or treat SARS-CoV-2 infection.\n",
      "There is an urgent need for new therapeutic strategies to contain the spread of the novel coronavirus disease 2019 (COVID-19) and to curtail its most severe complications. Severely ill patients experience pathologic manifestations of acute respiratory distress syndrome (ARDS), and clinical reports demonstrate striking neutrophilia, elevated levels of multiple cytokines, and an exaggerated inflammatory response in fatal COVID-19. Mechanical respirator devices are the most widely applied therapy for ARDS in COVID-19, yet mechanical ventilation achieves strikingly poor survival. Many patients, who recover, experience impaired cognition or physical disability. In this review, we argue the need to develop therapies aimed at inhibiting neutrophil recruitment, activation, degranulation, and neutrophil extracellular trap (NET) release. Moreover, we suggest that currently available pharmacologic approaches should be tested as treatments for ARDS in COVID-19. In our view, targeting host-mediated immunopathology holds promise to alleviate progressive pathologic complications of ARDS and reduce morbidities and mortalities in severely ill patients with COVID-19.\n",
      "Current coronavirus pandemic named coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). SARS-CoV-2 is the third coronavirus outbreak during the current century after severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) coronaviruses.1 Acute respiratory distress syndrome (ARDS) is an immunopathologic event and main cause of death following COVID-19. The main mechanism of ARDS is uncontrolled systemic inflammatory response and cytokine storm following release of proinflammatory cytokines (such as interferons (IFN), interleukines (IL), tumor necrosis factor (TNF)-α) and chemokines.2-3 So, some Chinese researchers proposed or used anti-inflammatory agents in the treatment regimen of patients with COVID-19.3-4.\n",
      "Severe pneumonia which shares several of the features of acute respiratory distress syndrome (ARDS) is the main cause of morbidity and mortality in Coronavirus disease 19 (Covid-19) for which as of now there is no effective treatment. ARDS is caused and sustained by an uncontrolled inflammatory activation characterized by a massive release of cytokines (cytokine storm), diffuse lung edema, inflammatory cell infiltraton and disseminated coagulation. Macrophage and T lymphocyte dysfunction plays a central role in this syndrome. In several experimental in vitro and in vivo models, many of these pathophysiological changes are triggered by stimulation of the P2X7 receptor. We hypothesize that this receptor might be an ideal candidate to target in Covid-19-associated severe pneumonia.\n",
      "Severe pneumonia which shares several of the features of acute respiratory distress syndrome (ARDS) is the main cause of morbidity and mortality in Coronavirus disease 19 (Covid‐19) for which as of now there is no effective treatment. ARDS is caused and sustained by an uncontrolled inflammatory activation characterized by a massive release of cytokines (cytokine storm), diffuse lung edema, inflammatory cell infiltraton and disseminated coagulation. Macrophage and T lymphocyte dysfunction plays a central role in this syndrome. In several experimental in vitro and in vivo models, many of these pathophysiological changes are triggered by stimulation of the P2X7 receptor. We hypothesize that this receptor might be an ideal candidate to target in Covid‐19‐associated severe pneumonia.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: cn3bpmwj\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "what kinds of complications related to COVID-19 are associated with diabetes\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "BACKGROUND AND AIMS: The outbreak of the new coronavirus, SARS-CoV-2, causes a respiratory disease and individuals with pre-existing cardiometabolic disorders display worse prognosis through the infection course. The aim of this minireview is to present epidemiological data related to metabolic comorbidities in association with the SARS-CoV-2. METHODS: This is a narrative mini-review with Pubmed search until April 23, 2020 using the keywords COVID-19, SARS-CoV-2, treatment of coronavirus and following terms: diabetes mellitus, obesity, arterial hypertension, ACE-inhibitors, cytokine storm, immune response and vitamin D. RESULTS: Studies indicate that obese individuals are more likely to develop infections, and that adipose tissue serves as a pathogen reservoir. In diabetic individuals higher rate of inflammatory processes is seen due to constant glucose recognition by C type lectin receptors. Hypertensive individuals, usually grouped with other conditions, are treated with drugs to reduce blood pressure mostly through ACEi and ARB, that leads to increased ACE2 expression, used by SARS-CoV-2 for human's cell entry. Until now, the studies have shown that individuals with those conditions and affected by COVID-19 present an uncontrolled release of pro-inflammatory cytokines and an unbalanced immune response, leading to the cytokine storm phenomenon. Vitamin D is highlighted as a potential therapeutic target, because in addition to acting on the immune system, it plays an important role in the control of cardiometabolic diseases. CONCLUSION: Currently, since there is no proven and effective antiviral therapy for SARS-CoV-2, the efforts should focus on controlling inflammatory response and reduce the risks of associated complications.\n",
      "BACKROUND AND AIMS: According to the World Obesity Federation, \"obesity-related conditions seem to worsen the effect of Covid-19 (SARS-CoV-2)\"; additionally the Centres for Disease Control and Prevention reported that \"people with heart disease and diabetes are at higher risk of SARS-CoV-2 complications and that severe obesity poses a higher risk for severe illness\". Recent reports have shown elevated levels of cytokines due to increased inflammation in patients with SARS-CoV-2 disease. On the other hand, obesity represents a state of low-grade inflammation, with various inflammatory products directly excreted by adipose tissue. In this concise report we aimed to assess common elements of obesity and SARS-CoV-2 infection. METHODS: Pubmed search on obesity and SARS-CoV-2 infection. RESULTS: We present \"mechanistic\" obesity-related problems that aggravate SARS-CoV-2 infection as well as tentative inflammatory/metabolic links between these diseases. CONCLUSION: Obesity and SARS-CoV-2 share common elements of the inflammatory process (and possibly also metabolic disturbances), exacerbating SARS-CoV-2 infection in the obese.\n",
      "Coronavirus disease 2019 (COVID-19) is a global pandemic that is caused by a novel coronavirus, severe acute respiratory syndrome-coronavirus-2 (SARS-CoV-2). Data from several countries have demonstrated higher morbidity and mortality among individuals with chronic metabolic diseases such as diabetes mellitus (DM). In this review, we explore the contributing factors for poorer prognosis in these individuals. As a significant proportion of patients with COVID-19 also have DM, this adds another layer of complexity to their management. We explore potential interactions between anti-diabetic medications and renin-angiotensin-aldosterone-system inhibitors with COVID-19. Suggested recommendations for the use of anti-diabetic medications in COVID-19 patients with DM are provided. We also review pertinent clinical considerations in the management of diabetic ketoacidosis in the COVID-19 patient. In addition, we aim to increase the awareness of clinicians to the metabolic effects of promising drug therapies for COVID-19. Finally, we highlight the importance of timely vaccinations for patients with DM.\n",
      "Coronavirus disease 2019 (COVID‐19) is a global pandemic that is caused by a novel coronavirus, severe acute respiratory syndrome‐coronavirus‐2 (SARS‐CoV‐2). Data from several countries have demonstrated higher morbidity and mortality among individuals with chronic metabolic diseases such as diabetes mellitus (DM). In this review, we explore the contributing factors for poorer prognosis in these individuals. As a significant proportion of patients with COVID‐19 also have DM, this adds another layer of complexity to their management. We explore potential interactions between anti‐diabetic medications and renin‐angiotensin‐aldosterone‐system inhibitors with COVID‐19. Suggested recommendations for the use of anti‐diabetic medications in COVID‐19 patients with DM are provided. We also review pertinent clinical considerations in the management of diabetic ketoacidosis in the COVID‐19 patient. In addition, we aim to increase the awareness of clinicians to the metabolic effects of promising drug therapies for COVID‐19. Finally, we highlight the importance of timely vaccinations for patients with DM.\n",
      "BACKGROUND AND AIMS: Many patients with coronavirus disease 2019 (COVID-19) have comorbidities related to metabolic syndrome (MS) during the disease course. Its presence in different ethnicities and continents places MS as an important risk factor for COVID-19. Adequate understanding of the interplay between MS, COVID-19 and proposed therapies is required for optimum management of these patients. METHODS: We systematically searched the PubMed and Google Scholar databases until June 1st, 2020 and accessed the full text on COVID-19 and MS to prepare a narrative review on this topic. RESULTS: Patients with metabolic disorders like obesity, diabetes, cardiovascular and liver disease may face a higher risk of infection of COVID-19, greatly affecting the development and prognosis of the disease, being associated with significantly worse outcome in these patients. The proposed drugs that are in clinical trial for COVID-19 treatment must be carefully considered for clinical use, especially in patients with MS. CONCLUSION: MS is a risk factor influencing the progression and prognosis of COVID-2019. The drugs currently evaluated for the infection treatment are promising but need further studies to prove their efficacy and safety, due to the adverse effects may be exacerbated by combination therapy or due to viral infection. The development of a vaccine for immunization is still the best long-term solution.\n",
      "BACKGROUND: SARS-CoV-2 coronavirus infection ranges from asymptomatic through to fatal COVID-19 characterised by a \"cytokine storm\" and lung failure. Vitamin D deficiency has been postulated as a determinant of severity. OBJECTIVES: To review the evidence relevant to vitamin D and COVID-19 METHODS: Narrative review RESULTS: Regression modelling shows that more northerly countries in the Northern Hemisphere are currently (May 2020) showing relatively high COVID-19 mortality, with an estimated 4.4% increase in mortality for each 1 degree latitude north of 28 degrees North (P=0.031) after adjustment for age of population. This supports a role for ultraviolet B acting via vitamin D synthesis. Factors associated with worse COVID-19 prognosis include old age, ethnicity, male sex, obesity, diabetes and hypertension and these also associate with deficiency of vitamin D or its response. Vitamin D deficiency is also linked to severity of childhood respiratory illness. Experimentally, vitamin D increases the ratio of angiotensin converting enzyme 2 (ACE2) to ACE, thus increasing angiotensin II hydrolysis and reducing subsequent inflammatory cytokine response to pathogens and lung injury. CONCLUSIONS: Substantial evidence supports a link between vitamin D deficiency and COVID-19 severity but it is all indirect. Community-based placebo-controlled trials of vitamin D supplementation may be difficult. Further evidence could come from study of COVID-19 outcomes in large cohorts with information on prescribing data for vitamin D supplementation or assay of serum unbound 25(OH) vitamin D levels. Meanwhile vitamin D supplementation should be strongly advised for people likely to be deficient.\n",
      "As the biomedical community races to disentangle the unknowns associated with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) - the virus responsible for causing coronavirus disease (COVID-19) - the link between diminished immune function and individuals with obesity raises important questions about the possibility for greater viral pathogenicity in this population. Increased adiposity may undermine the pulmonary microenvironment wherein viral pathogenesis and immune cell trafficking could contribute to a maladaptive cycle of local inflammation and secondary injury. A further challenge to those with obesity during the current pandemic may involve vitamin D deficiency/insufficiency. In the interest of personal and public health, we caution decision/policy makers alike not to pin all hope on a proverbial 'silver bullet.' Until further breakthroughs emerge, we should remember that modifiable lifestyle factors like diet and physical activity should not be marginalized. Decades of empirical evidence supports both as key factors promoting health and wellness.\n",
      "BACKGROUND: Individuals with diabetes are at a greater risk of hospitalization and mortality resulting from viral, bacterial, and fungal infections. The coronavirus disease-2019 (COVID-19), caused by severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2), has spread quickly to more than 213 countries and claimed 395,779 lives as of June 7, 2020. Notably, in several studies, diabetes is one of the most reported comorbidities in patients with severe COVID-19. SCOPE OF REVIEW: In this review, I summarize the clinical data on the risk for infectious diseases in individuals with diabetes while highlighting the mechanisms for altered immune regulation. The focus is on coronaviruses. Based on the new clinical data obtained from COVID-19 patients, a discussion of mechanisms, such as cytokine storm, pulmonary and endothelial dysfunction, and hypercoagulation, that may render individuals with diabetes more vulnerable to COVID-19 is provided. MAJOR CONCLUSIONS: Epidemiological studies show that poorly controlled diabetes is a risk factor for various infectious diseases. Given the global burden of diabetes and the pandemic nature of coronaviruses, understanding how diabetes affects COVID-19 severity is critical to designing tailored treatments and clinical management of individuals affected by diabetes.\n",
      "Coronavirus disease 2019 (COVID-19) has become a global concern and public health issue due to its higher infection and mortality rate; particularly, the risk is very higher among the patients who have cardiovascular diseases (CVD) and/or diabetes mellitus (DM). In this review, we analyzed the recently published literature on CVD and DM associated with COVD-19 infections and highlight their association with potential mechanisms. The findings revealed that without any previous history of CVD, the COVID-19 patients have developed some CVD complications like myocardial injury, cardiomyopathy, and venous thromboembolism after being infected with the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and required for those patients an emergency clinical support to be aware to manage those complications. Though the association between DM and COVID-19-induced severe complications is still unclear, the limited data predict that different markers like interleukin (IL)-1, IL-6, C-reactive protein, and D-dimer linked with the severity of COVID-19 infection in diabetic individuals. Further studies on a large scale are urgently needed to explore the underlying mechanisms between CVD, DM, and COVID-19 for better treatment.\n",
      "The pandemic of coronavirus disease (COVID-19), a disease caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), is causing substantial morbidity and mortality. Older age and presence of diabetes mellitus, hypertension, and obesity significantly increases the risk for hospitalization and death in COVID-19 patients. In this Perspective, informed by the studies on SARS-CoV-2, Middle East respiratory syndrome (MERS-CoV), and the current literature on SARS-CoV-2, we discuss potential mechanisms by which diabetes modulates the host-viral interactions and host-immune responses. We hope to highlight gaps in knowledge that require further studies pertinent to COVID-19 in patients with diabetes.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: wrrybwda\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "are cardiac complications likely in patients with COVID-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "We summarize the cardiovascular risks associated with Covid-19 pandemic, discussing the risks for both infected and non-infected patients.\n",
      "We highlight in this short article the side-effects of COVID-19 pandemic on the management of non-COVID patients, with potential detrimental and irreversible complications. We thus propose adjusted strategies to deal with both COVID and non-COVID patients.\n",
      "Although COVID-19 primarily affects the respiratory system, cardiovascular involvement is common, especially in individuals who are severely ill.\n",
      "Although COVID-19 primarily affects the respiratory system, cardiovascular involvement is common, especially in individuals who are severely ill.\n",
      "In patients with COVID-19, myocardial injury is prevalent and is associated with an adverse prognosis and increased mortality, according to two retrospective cohort studies from China and the USA.\n",
      "Coronavirus Disease 2019 (COVID-19) has infected more than 3.0 million people worldwide and killed more than 200,000 as of April 27, 2020. In this White Paper, we address the cardiovascular co-morbidities of COVID-19 infection; the diagnosis and treatment of standard cardiovascular conditions during the pandemic; and the diagnosis and treatment of the cardiovascular consequences of COVID-19 infection. In addition, we will also address various issues related to the safety of healthcare workers and the ethical issues related to patient care in this pandemic.\n",
      "Patients with pre-existing cardiovascular disease and risk factors are more likely to experience adverse outcomes associated with the novel coronavirus disease-2019 (COVID-19). Additionally, consistent reports of cardiac injury and de novo cardiac complications, including possible myocarditis, arrhythmia, and heart failure in patients without prior cardiovascular disease or significant risk factors, are emerging, possibly due to an accentuated host immune response and cytokine release syndrome. As the spread of the virus increases exponentially, many patients will require medical care either for COVID-19 related or traditional cardiovascular issues. While the COVID-19 pandemic is dominating the attention of the healthcare system, there is an unmet need for a standardized approach to deal with COVID-19 associated and other traditional cardiovascular issues during this period. We provide consensus guidance for the management of various cardiovascular conditions during the ongoing COVID-19 pandemic with the goal of providing the best care to all patients and minimizing the risk of exposure to frontline healthcare workers.\n",
      "It is clear that existing cardiovascular disease is a major risk factor for COVID-19 and related adverse outcomes. In addition to acute respiratory syndrome, a large cohort also develop myocardial and/or vascular dysfunction, in part from inflammation and renin angiotensin system activation with increased sympathetic outflow, cardiac arrhythmias, ischemia, heart failure, and thromboembolic complications that portend poor COVID-19 outcomes. We summarize here recent information for hospitalists/internists on the front-line of this pandemic regarding its cardiovascular impacts and management and need for cardiovascular consultation.\n",
      "The COVID-19 pandemic has emerged as a serious global threat causing a large number of fatalities and putting enormous strain on the health care resources across the world. This has resulted in preferentially triaging the coronavirus infected patients and placing others, especially cardiovascular patients at increased risk for adverse complications. The effective management of cardiac patients in the hospital environment during this COVID-19 pandemic has emerged as a real challenge. We try to address this issue and also highlight the interplay between COVID-19 and cardiovascular diseases. We hereby review the available literature and emerging guidelines about cardiovascular implications related to COVID-19 which will have a bearing on the patient care, health care professionals and cardiac centres.\n",
      "The COVID-19 pandemic has emerged as a serious global threat causing a large number of fatalities and putting enormous strain on the health care resources across the world. This has resulted in preferentially triaging the coronavirus infected patients and placing others, especially cardiovascular patients at increased risk for adverse complications. The effective management of cardiac patients in the hospital environment during this COVID-19 pandemic has emerged as a real challenge. We try to address this issue and also highlight the interplay between COVID-19 and cardiovascular diseases. We hereby review the available literature and emerging guidelines about cardiovascular implications related to COVID-19 which will have a bearing on the patient care, health care professionals and cardiac centres.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 9xmeyrtu\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "What is the mechanism of cytokine storm syndrome on the COVID-19?\n",
      "What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Background: The dynamic changes of lymphocyte subsets and cytokines profiles of patients with novel coronavirus disease (COVID-19) and their correlation with the disease severity remain unclear. Method: Peripheral blood samples were longitudinally collected from 40 confirmed COVID-19 patients and examined for lymphocyte subsets by flow cytometry and cytokine profiles by specific immunoassays. Findings: Of the 40 COVID-19 patients enrolled, 13 severe cases showed significant and sustained decreases in lymphocyte counts but increases in neutrophil counts than 27 mild cases. Further analysis demonstrated significant decreases in the counts of T cells, especially CD8 + T cells, as well as increases in IL-6, IL-10, IL-2 and IFN-γ levels in the peripheral blood in the severe cases compared to those in the mild cases. T cell counts and cytokine levels in severe COVID-19 patients who survived the disease gradually recovered at later time points to levels that were comparable to those of the mild cases. Moreover, the neutrophil-to-CD8+ T cell ratio (N8R) were identified as the most powerful prognostic factor affecting the prognosis for severe COVID-19. Conclusion: The degree of lymphopenia and a proinflammatory cytokine storm is higher in severe COVID-19 patients than in mild cases, and is associated with the disease severity. N8R may serve as a useful prognostic factor for early identification of severe COVID-19 cases.\n",
      "BACKGROUND: The dynamic changes of lymphocyte subsets and cytokines profiles of patients with novel coronavirus disease (COVID-19) and their correlation with the disease severity remain unclear. METHODS: Peripheral blood samples were longitudinally collected from 40 confirmed COVID-19 patients and examined for lymphocyte subsets by flow cytometry and cytokine profiles by specific immunoassays. FINDINGS: Of the 40 COVID-19 patients enrolled, 13 severe cases showed significant and sustained decreases in lymphocyte counts [0·6 (0·6-0·8)] but increases in neutrophil counts [4·7 (3·6-5·8)] than 27 mild cases [1.1 (0·8-1·4); 2·0 (1·5-2·9)]. Further analysis demonstrated significant decreases in the counts of T cells, especially CD8+ T cells, as well as increases in IL-6, IL-10, IL-2 and IFN-Î³ levels in the peripheral blood in the severe cases compared to those in the mild cases. T cell counts and cytokine levels in severe COVID-19 patients who survived the disease gradually recovered at later time points to levels that were comparable to those of the mild cases. Moreover, the neutrophil-to-lymphocyte ratio (NLR) (AUC=0·93) and neutrophil-to-CD8+ T cell ratio (N8R) (AUC =0·94) were identified as powerful prognostic factors affecting the prognosis for severe COVID-19. INTERPRETATION: The degree of lymphopenia and a proinflammatory cytokine storm is higher in severe COVID-19 patients than in mild cases, and is associated with the disease severity. N8R and NLR may serve as a useful prognostic factor for early identification of severe COVID-19 cases. FUNDING: The National Natural Science Foundation of China, the National Science and Technology Major Project, the Health Commission of Hubei Province, Huazhong University of Science and Technology, and the Medical Faculty of the University of Duisburg-Essen and Stiftung Universitaetsmedizin, Hospital Essen, Germany.\n",
      "Abstract Background The dynamic changes of lymphocyte subsets and cytokines profiles of patients with novel coronavirus disease (COVID-19) and their correlation with the disease severity remain unclear. Methods Peripheral blood samples were longitudinally collected from 40 confirmed COVID-19 patients and examined for lymphocyte subsets by flow cytometry and cytokine profiles by specific immunoassays. Findings Of the 40 COVID-19 patients enrolled, 13 severe cases showed significant and sustained decreases in lymphocyte counts [0•6 (0•6-0•8)] but increases in neutrophil counts [4•7 (3•6-5•8)] than 27 mild cases [1.1 (0•8-1•4); 2•0 (1•5-2•9)]. Further analysis demonstrated significant decreases in the counts of T cells, especially CD8+ T cells, as well as increases in IL-6, IL-10, IL-2 and IFN-γ levels in the peripheral blood in the severe cases compared to those in the mild cases. T cell counts and cytokine levels in severe COVID-19 patients who survived the disease gradually recovered at later time points to levels that were comparable to those of the mild cases. Moreover, the neutrophil-to-lymphocyte ratio (NLR) (AUC=0•93) and neutrophil-to-CD8+ T cell ratio (N8R) (AUC =0•94) were identified as powerful prognostic factors affecting the prognosis for severe COVID-19. Interpretation The degree of lymphopenia and a proinflammatory cytokine storm is higher in severe COVID-19 patients than in mild cases, and is associated with the disease severity. N8R and NLR may serve as a useful prognostic factor for early identification of severe COVID-19 cases. Funding The National Natural Science Foundation of China, the National Science and Technology Major Project, the Health Commission of Hubei Province, Huazhong University of Science and Technology, and the Medical Faculty of the University Hospital Essen, Germany.\n",
      "INTRODUCTION: Coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection has been widely spread. We aim to investigate the dynamic changes of peripheral blood lymphocytes subsets in adult patients with COVID-19. METHODS: Electronic medical records including demographics, clinical manifestation, comorbidities, laboratory data, and radiological materials of 435 hospitalized COVID-19 patients, with confirmed result of SARS-CoV-2 viral infection, were extracted and retrospectively analyzed. The counts of lymphocyte subsets at each week after the onset of the illness were compared with other weeks of illness and with those from normal individuals. RESULTS: The various lymphocyte subsets (CD3 +, CD4 +, CD8 +, CD19 +and CD16/56 +) were below the normal ranges at the first week after the onset of the illness, reaching a nadir during the second week, which increased gradually in the third week and returned to normal levels in the fifth week, but was still lower than the healthy controls. The CD3+, CD4+ and CD8+ counts were significantly lower in those with severe disease compared to those with non-severe disease, and in those patients that died compared to those who recovered. DISCUSSION: our research indicates that the levels of peripheral blood lymphocyte subsets (CD3 +, CD4 +, and CD8 +) are associated with disease progression, severity, and prognosis in patients with COVID-19. Dynamic monitoring of human immune function is one of the indicators for evaluating the severity and prognosis of COVID-19 patients, and it is helpful to formulate appropriate treatment strategies.\n",
      "This study evaluated the significance of lymphocyte subset detection in peripheral blood in the diagnosis and prognosis of coronavirus disease 2019 (COVID-19). Our results revealed that CD3+ T cells, CD4+ T cells, CD8+ T cells, and natural killer cells were significantly decreased in patients with COVID-19. These patients had a relatively slight decrease in CD4+ T cells but a severe decrease in CD8+ T cells. The significantly elevated CD4/CD8 ratio was observed in COVID-19 patients. T-cell subset counts were related to the severity and prognosis of COVID-19, suggesting that the counts of CD8+ T and CD4+ T cells can be used as diagnostic markers of COVID-19 and predictors of disease severity.\n",
      "This study evaluated the significance of lymphocyte subsets detection in peripheral blood in the diagnosis and prognosis of coronavirus disease 2019 (COVID-19). Our results revealed that CD3+T, CD4+T, CD8+T cells and NK cells were significantly decreased in COVID-19 patients. COVID-19 patients had a relatively slight decrease in CD4+T cells but a severe decrease of CD8+T cells. The significantly elevated CD4/CD8 ratio was observed in COVID-19 patients. T cell subset counts were related to the severity and prognosis of COVID-19. The counts of CD8+T and CD4+T cells can be used as diagnostic markers of COVID-19 and predictors of disease severity.\n",
      "BACKGROUND: In December 2019, novel coronavirus (SARS-CoV-2) pneumonia (COVID-19) was reported in Wuhan and has since rapidly spread throughout China. We aimed to clarify the characteristics and clinical significance of peripheral lymphocyte subset alteration in COVID-19. METHODS: The levels of peripheral lymphocyte subsets were measured by flow cytometry in 60 hospitalized COVID-19 patients before and after treatment, and their association with clinical characteristics and treatment efficacy was analyzed. RESULTS: Total lymphocytes, CD4(+) T cells, CD8(+) T cells, B cells, and natural killer (NK) cells decreased in COVID-19 patients, and severe cases had a lower level than mild cases. The subsets showed a significant association with inflammatory status in COVID-19, especially CD8(+) T cells and CD4(+)/CD8(+) ratio. After treatment, 37 patients (67%) showed clinical response, with an increase in CD8(+) T cells and B cells. No significant change in any subset was detected in nonresponsive cases. In multivariate analysis, posttreatment decrease in CD8(+) T cells and B cells and increase in CD4(+)/CD8(+) ratio were indicated as independent predictors of poor efficacy. CONCLUSIONS: Peripheral lymphocyte subset alteration was associated with clinical characteristics and treatment efficacy of COVID-19. CD8(+) T cells tended to be an independent predictor for COVID-19 severity and treatment efficacy.\n",
      "Severe COVID-19 patients showed worse clinical outcomes compared to mild and moderate patients. However, effective indicators are still lacking to predict the severity of the disease. In the present study, we retrospectively analyzed the clinical and laboratory data of 16 COVID-19 patients and found that the absolute counts of three T-cells (CD3+, CD4+, and CD8+) were significantly lower in the moderate and severe patients than those in mild patients and were significantly lower in severe patients than in moderate patients on admission. With the recovery of the COVID-19, serum levels of inflammatory biomarkers (CRP, PCT, and IL6) of moderate and severe patients gradually decreased. In contrast, the counts of lymphocytes and their subsets including CD3+, CD4+, and CD8+ T cells gradually increased in severe patients, and eventually showed comparable levels with moderate patients. ROC analysis showed that the counts of CD3+, CD4+, and CD8+ T-cells with AUC > 0.9 have potential values for predicting the severity of COVID-19 patients. In conclusion, the reduction of CD3+, CD4+, and CD8+ T-cells is related to the severity of COVID-19 and dynamic detection of these T-lymphocyte subsets may help predict the outcome of the patients.\n",
      "A reduced peripheral blood absolute lymphocyte count with an elevated neutrophil count has been a consistent observation in hospitalized COVID-19 patients. In this brief meta-analysis, the reduction of lymphocyte subset counts in COVID-19 patients was investigated across 20 peer-reviewed studies meeting criteria for reporting lymphocyte subset counts and COVID-19 disease severity. CD4+ T cell, CD8+ T cell, B cell, NK cell and total lymphocyte cell counts all showed statistically significant reduction in patients with severe/critical COVID-19 disease compared to mild/moderate disease. T cell subsets showed the largest standardized magnitude of change. In some studies, multivariate analysis has shown that CD4 and/or CD8 T cells counts are independently predictive of patient outcomes.\n",
      "BACKGROUND: We observe changes of the main lymphocyte subsets (CD16(+)CD56、CD19、CD3、CD4、and CD8) in COVID-19-infected patients and explore whether the changes are associated with disease severity. METHODS: One-hundred fifty-four cases of COVID-19-infected patients were selected and divided into 3 groups (moderate group, severe group and critical group). The flow cytometry assay was performed to examine the numbers of lymphocyte subsets. RESULTS: CD3(+), CD4(+) and CD8+ T lymphocyte subsets were decreased in COVID-19-infected patients. Compared with the moderate group and the sever group, CD3(+), CD4(+) and CD8(+) T cells in the critical group decreased greatly (P < 0.001, P= 0.005 or P = 0.001). CONCLUSIONS: Reduced CD3(+), CD4(+), CD8(+) T lymphocyte counts may reflect the severity of the COVID-19. Monitoring T cell changes has important implications for the diagnosis and treatment of severe patients who may become critically ill.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: bsgqsdbx\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which biomarkers predict the severe clinical course of 2019-nCOV infection?\n",
      "what are the initial symptoms of Covid-19?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Background Since late December, 2019, an outbreak of pneumonia cases caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan, and continued to spread throughout China and across the globe. To date, few data on immunologic features of Coronavirus Disease 2019 (COVID-19) have been reported. Methods In this single-centre retrospective study, a total of 21 patients with pneumonia who were laboratory-confirmed to be infected with SARS-CoV-2 in Wuhan Tongji hospital were included from Dec 19, 2019 to Jan 27, 2020. The immunologic characteristics as well as their clinical, laboratory, radiological features were compared between 11 severe cases and 10 moderate cases. Results Of the 21 patients with COVID-19, only 4 (19%) had a history of exposure to the Huanan seafood market. 7 (33.3%) patients had underlying conditions. The average age of severe and moderate cases was 63.9 and 51.4 years, 10 (90.9%) severe cases and 7 (70.0%) moderate cases were male. Common clinical manifestations including fever (100%, 100%), cough (70%, 90%), fatigue (100%, 70%) and myalgia (50%, 30%) in severe cases and moderate cases. PaO2/FiO2 ratio was significantly lower in severe cases (122.9) than moderate cases (366.2). Lymphocyte counts were significantly lower in severe cases (7000 million/L) than moderate cases (11000 million/L). Alanine aminotransferase, lactate dehydrogenase levels, high-sensitivity C-reactive protein and ferritin were significantly higher in severe cases (41.4 U/L, 567.2 U/L, 135.2 mg/L and 1734.4 ug/L) than moderate cases (17.6 U/L, 234.4 U/L, 51.4 mg/L and 880.2 ug /L). IL-2R, TNF-α and IL-10 concentrations on admission were significantly higher in severe cases (1202.4 pg/mL, 10.9 pg/mL and 10.9 pg/mL) than moderate cases (441.7 pg/mL, 7.5 pg/mL and 6.6 pg/mL). Absolute number of total T lymphocytes, CD4+T cells and CD8+T cells decreased in nearly all the patients, and were significantly lower in severe cases (332.5, 185.6 and 124.3 million/L) than moderate cases (676.5, 359.2 and 272.0 million/L). The expressions of IFN-γ by CD4+T cells tended to be lower in severe cases (14.6%) than moderate cases (23.6%). Conclusion The SARS-CoV-2 infection may affect primarily T lymphocytes, particularly CD4+T cells, resulting in significant decrease in number as well as IFN-γ production, which may be associated with disease severity. Together with clinical characteristics, early immunologic indicators including diminished T lymphocytes and elevated cytokines may serve as potential markers for prognosis in COVID-19.\n",
      "BACKGROUND Since December 2019, an outbreak of Coronavirus Disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan, and is now becoming a global threat. We aimed to delineate and compare the immunologic features of severe and moderate COVID-19. METHODS In this retrospective study, the clinical and immunologic characteristics of 21 patients (17 male and 4 female) with COVID-19 were analyzed. These patients were classified as severe (11 cases) and moderate (10 cases) according to the Guidelines released by the National Health Commission of China. RESULTS The median age of severe and moderate cases was 61.0 and 52.0 years, respectively. Common clinical manifestations included fever, cough and fatigue. Compared to moderate cases, severe cases more frequently had dyspnea, lymphopenia, and hypoalbuminemia, with higher levels of alanine aminotransferase, lactate dehydrogenase, C-reactive protein, ferritin and D-dimer as well as markedly higher levels of IL-2R, IL-6, IL-10, and TNF-α. Absolute number of T lymphocytes, CD4+T and CD8+T cells decreased in nearly all the patients, and were markedly lower in severe cases (294.0, 177.5 and 89.0 × 106/L) than moderate cases (640.5, 381.5 and 254.0 × 106/L). The expressions of IFN-γ by CD4+T cells tended to be lower in severe cases (14.1%) than moderate cases (22.8%). CONCLUSION The SARS-CoV-2 infection may affect primarily T lymphocytes particularly CD4+T and CD8+ T cells, resulting in decrease in numbers as well as IFN-γ production. These potential immunological markers may be of importance due to their correlation with disease severity in COVID-19.\n",
      "BACKGROUNDSince December 2019, an outbreak of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan, and is now becoming a global threat. We aimed to delineate and compare the immunological features of severe and moderate COVID-19.METHODSIn this retrospective study, the clinical and immunological characteristics of 21 patients (17 male and 4 female) with COVID-19 were analyzed. These patients were classified as severe (11 cases) and moderate (10 cases) according to the guidelines released by the National Health Commission of China.RESULTSThe median age of severe and moderate cases was 61.0 and 52.0 years, respectively. Common clinical manifestations included fever, cough, and fatigue. Compared with moderate cases, severe cases more frequently had dyspnea, lymphopenia, and hypoalbuminemia, with higher levels of alanine aminotransferase, lactate dehydrogenase, C-reactive protein, ferritin, and D-dimer as well as markedly higher levels of IL-2R, IL-6, IL-10, and TNF-&#945;. Absolute numbers of T lymphocytes, CD4+ T cells, and CD8+ T cells decreased in nearly all the patients, and were markedly lower in severe cases (294.0, 177.5, and 89.0 × 106/L, respectively) than moderate cases (640.5, 381.5, and 254.0 × 106/L, respectively). The expression of IFN-Î³ by CD4+ T cells tended to be lower in severe cases (14.1%) than in moderate cases (22.8%).CONCLUSIONThe SARS-CoV-2 infection may affect primarily T lymphocytes, particularly CD4+ and CD8+ T cells, resulting in a decrease in numbers as well as IFN-Î³ production by CD4+ T cells. These potential immunological markers may be of importance because of their correlation with disease severity in COVID-19.TRIAL REGISTRATIONThis is a retrospective observational study without a trial registration number.FUNDINGThis work is funded by grants from Tongji Hospital for the Pilot Scheme Project, and partly supported by the Chinese National Thirteenth Five Years Project in Science and Technology for Infectious Disease (2017ZX10202201).\n",
      "BACKGROUND: In late December 2019, an outbreak of acute respiratory illness, coronavirus disease 2019 (COVID-19), emerged in Wuhan, China. We aimed to study the epidemiology, clinical features and short-term outcomes of patients with COVID-19 in Wuhan, China. METHODS: We performed a single center, retrospective case series study in 221 patients with laboratory confirmed SARS-CoV-2 pneumonia at a university hospital, including 55 severe patients and 166 non-severe patients, from January 2, 2020 to February 10, 2020. RESULTS: Of the 221 patients with COVID-19, the median age was 55.0 years and 48.9% were male and only 8 (3.6%) patients had a history of exposure to the Huanan Seafood Market. Compared to the non-severe pneumonia patients, the median age of the severe patients was significantly older, and they were more likely to have chronic comorbidities. Most common symptoms in severe patients were high fever, anorexia and dyspnea. On admission, 33.0% patients showed leukopenia and 73.8% showed lymphopenia. In addition, the severe patients suffered a higher rate of co-infections with bacteria or fungus and they were more likely to developing complications. As of February 15, 2020, 19.0% patients had been discharged and 5.4% patients died. 80% of severe cases received ICU (intensive care unit) care, and 52.3% of them transferred to the general wards due to relieved symptoms, and the mortality rate of severe patients in ICU was 20.5%. CONCLUSIONS: Patients with elder age, chronic comorbidities, blood leukocyte/lymphocyte count, procalcitonin level, co-infection and severe complications might increase the risk of poor clinical outcomes.\n",
      "Rationale: In late December 2019, an outbreak of acute respiratory illness, now officially named as COVID-19, or coronavirus disease 2019, emerged in Wuhan, China, now spreading across the whole country and world. More data were needed to understand the clinical characteristics of the disease. Objectives: To study the epidemiology, clinical features and outcomes of patients with COVID-19. Methods: we performed a single center, retrospective case series study in 221 patients with laboratory confirmed SARS-CoV-2 pneumonia at a university hospital. Measurements and Main Results: The median age was 55.0 years and 48.9% were male and only 8 (3.6%) patients had a history of exposure to the Huanan Seafood Market. Compared to the non-severe pneumonia patients, the median age of the severe patients was significantly older, and they were more likely to have chronic comorbidities. Most common symptoms in severe patients were high fever, anorexia and dyspnea. On admission, 33.0% patients showed leukopenia and 73.8% showed lymphopenia. In addition, the severe patients suffered a higher rate of co-infections with bacteria or fungus and they were more likely to developing complications. As of February 15, 2020, 19.0% patients had been discharged and 5.4% patients died. 80% of severe cases received ICU care, and 52.3% of them transferred to the general wards due to relieved symptoms, and the mortality rate of severe patients in ICU was 20.5%. Conclusions: The COVID-19 epidemic spreads rapidly by human-to-human transmission. Patients with elder age, chronic comorbidities, blood leukocyte/lymphocyte count, procalcitonin level, co-infection and severe complications might increase the risk of poor clinical outcomes. Keywords: coronavirus disease 2019; clinical features; outcomes; severe patients\n",
      "Abstract Background In late December 2019, an outbreak of acute respiratory illness, coronavirus disease 2019 (COVID-19), emerged in Wuhan, China. We aimed to study the epidemiology, clinical features and short-term outcomes of patients with COVID-19 in Wuhan, China. Methods We performed a single center, retrospective case series study in 221 patients with laboratory confirmed SARS-CoV-2 pneumonia at a university hospital, including 55 severe patients and 166 non-severe patients, from January 2, 2020 to February 10, 2020. Results Of the 221 patients with COVID-19, the median age was 55.0 years and 48.9% were male and only 8 (3.6%) patients had a history of exposure to the Huanan Seafood Market. Compared to the non-severe pneumonia patients, the median age of the severe patients was significantly older, and they were more likely to have chronic comorbidities. Most common symptoms in severe patients were high fever, anorexia and dyspnea. On admission, 33.0% patients showed leukopenia and 73.8% showed lymphopenia. In addition, the severe patients suffered a higher rate of co-infections with bacteria or fungus and they were more likely to developing complications. As of February 15, 2020, 19.0% patients had been discharged and 5.4% patients died. 80% of severe cases received ICU (intensive care unit) care, and 52.3% of them transferred to the general wards due to relieved symptoms, and the mortality rate of severe patients in ICU was 20.5%. Conclusions Patients with elder age, chronic comorbidities, blood leukocyte/lymphocyte count, procalcitonin level, co-infection and severe complications might increase the risk of poor clinical outcomes.\n",
      "Background In December 2019, novel coronavirus (SARS-CoV-2) infected pneumonia occurred in Wuhan, China. The number of cases has increased rapidly but information on the clinical characteristics of SARS-CoV-2 pneumonia without comorbidities compared to normal controls in Chinese Han population is limited. Our objective is to describe the epidemiological and clinical characteristics of SARS-CoV-2 pneumonia without comorbidities compared to normal controls in the Chinese Han population. Methods Retrospective, multi-center case series of the 69 consecutive hospitalized patients with confirmed SARS-CoV-2 pneumonia, from February 7 to February 28, 2020; final date of follow-up was February 29, 2020. Results The study population included 69 hospitalized patients with confirmed SARS-CoV-2 pneumonia without comorbidities and 14,117 normal controls. 50.7% patients were male and 49.3% were female; 1.5% patients were asymptomatic cases, 63.8% patients were mild cases, and 36.2% patients were severe or critical cases. Compared with mild patients (n = 44), severe or critical patients (n = 25) were significantly older (median age, 67 years [IQR, 58-79] vs. 49 years [IQR, 36-60]; P < 0.01). Fever was present in 98.6% of the patients. The second most common symptom was cough (62.3%), fatigue (58.0%), sputum (39.1%), and headache (33.3%). The median incubation period was 4 days (IQR, 2 to 7). Leukocyte count was 74.1% of normal controls and lymphocyte count was 45.9% of normal controls. The phenomenon of lymphocyte depletion (PLD) observed in severe or critical cases in 100%. Levels of lactate dehydrogenase, D-dimer, procalcitonin, and interleukin-6 were showed significant differences between mild and severe or critical cases. Chest computed tomographic scans showed bilateral patchy patterns (49.3%), local patchy shadowing (29.0%), and ground glass opacity (21.7%). 7.3% patients were diagnosed ARDS, 7.3% patients were diagnosed acute cardiac injury (troponin I >28 pg/mL) and 4.4% patients were diagnosed fungal infections or shock. 4.3% patients have been discharged; 1.5% patient had died; 1.5% patient had recovery. Conclusions In this multicenter case series of 69 patients without comorbidities, the full spectrum of asymptomatic, mild, severe, and critical cases is described. 50.7% patients were male and 49.3% were female; 1.5% patients were asymptomatic cases, 63.8% patients were mild cases, and 36.2% patients were severe or critical cases. 4.3% patients have been discharged; 1.5% patient had died; 1.5% patient had recovery. Among the 25 patients with severe or critical disease, 12.0% patients were underwent non-invasive mechanical ventilation, 8.0% patients underwent invasive mechanical ventilation, and 4.0% patients died.\n",
      "Background: The coronavirus disease 2019 (COVID-19) pneumonia outbreak began in Wuhan and pandemics tend to occur. Although SARS-CoV-2-specific immunoglobulins have been detected in serum of COVID-19 patients, their dynamics and association with outcomes have not been characterized. Methods: A total of 116 hospitalized patients with confirmed COVID-19 pneumonia and SARS-CoV-2-specific immunoglobulins tested in Tongji hospital were retrospectively investigated. Clinical, laboratory, radiological characteristics and outcomes data were compared between mild-moderate group and died group. Further, a paired case-control study was conducted where each deceased case was matched to three mild-moderate patients of similar age. Findings: Among 116 subjects included, 101 mild-moderate patients survived and 15 cases died. SARS-CoV-2-specific IgM levels peaked in forth week after onset of COVID-19 pneumonia, while serum IgG levels increased over 8 weeks. Serum IgM levels were higher in deceased patients than mild-moderate patients (P = 0.024), but not IgG. Serum IgM levels were negatively correlated with clinical outcome, eosinophil count and albumin levels (r = -0.269, P = 0.003; r = -0.188, P = 0.043; and r = -0.198, P = 0.033, resp.). The area under the ROC curve (AUC) for IgM antibody was 0.681 (95% CI: 0.517-0.845, P = 0.024). In case-control study paired by age, serum IgM was higher in deceased patients than mild-moderate patients (P = 0.019), positively correlated with leucocyte count (r = 0.260, P = 0.045), while negatively correlated with clinical outcome and albumin levels (r = -0.337, P = 0.008; r = -0.265, P = 0.041). AUC for IgM levels was 0.704 (95% CI: 0.534-0.873, P = 0.019). Interpretation: These results indicate that dynamics of SARS-CoV-2 specific IgM and IgG antibodies was similar with that of SARS-CoV, while elevated serum IgM levels indicate poor outcome in patients with COVID-19 pneumonia.\n",
      "BACKGROUND The Coronavirus Disease-2019 (COVID-19), infected by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has caused a severe outbreak in China. The host immunity of COVID-19 patients is unknown. METHODS The routine laboratory tests and host immunity in COVID-19 patients with different severity of illness were compared after patient admission. RESULTS A total of 65 SARS-CoV-2-positive patients were classified as mild (n=30), severe (n=20), and extremely severe (n=15) illness. Many routine laboratory tests such as ferritin, lactate dehydrogenase and D-dimer were increased in severe and extremely severe patients. The absolute numbers of CD4+ T cells, CD8+ T cells and B cells were all gradually decreased with increased severity of illness. The activation markers such as HLA-DR and CD45RO expressed on CD4+ and CD8+ T cells were increased in severe and extremely severe patients compared with mild patients. The co-stimulatory molecule CD28 had opposite results. The percentage of natural regulatory T cells was decreased in extremely severe patients. The percentage of IFN-γ producing CD8+ T cells was increased in both severe and extremely severe patients compared with mild patients. The percentage of IFN-γ producing CD4+ T cells was increased in extremely severe patients. The IL-2R, IL-6, and IL-10 were all increased in extremely severe patients. The activation of DC and B cells was decreased in extremely severe patients. CONCLUSIONS The number and function of T cells are inconsistent in COVID-19 patients. The hyperfunction of CD4+ and CD8+ T cells is associated with the pathogenesis of extremely severe SARS-CoV-2 infection.\n",
      "IMPORTANCE: The epidemiologic and clinical characteristics of pediatric patients with coronavirus disease 2019 (COVID-19) have been reported, but information on immune features associated with disease severity is scarce. OBJECTIVE: To delineate and compare the immunologic features of mild and moderate COVID-19 in pediatric patients. DESIGN, SETTING, AND PARTICIPANTS: This single-center case series included 157 pediatric patients admitted to Wuhan Children’s Hospital with laboratory-confirmed severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Data were collected from January 25 to April 18, 2020. EXPOSURES: Documented SARS-CoV-2 infection. MAIN OUTCOMES AND MEASURES: Clinical and immunologic characteristics were collected and analyzed. Outcomes were observed until April 18, 2020. RESULTS: Of the 157 pediatric patients with COVID-19, 60 (38.2%) had mild clinical type with pneumonia, 88 (56.1%) had moderate cases, 6 (3.8%) had severe cases, and 3 (1.9%) were critically ill. The 148 children with mild or moderate disease had a median (interquartile range [IQR]) age of 84 (18-123) months, and 88 (59.5%) were girls. The most common laboratory abnormalities were increased levels of alanine aminotransferase (ALT) (median [IQR], 16.0 [12.0-26.0] U/L), aspartate aminotransferase (AST) (median [IQR], 30.0 [23.0-41.8] U/L), creatine kinase MB (CK-MB) activity (median [IQR], 24.0 [18.0-34.0] U/L), and lactate dehydrogenase (LDH) (median [IQR], 243.0 [203.0-297.0] U/L), which are associated with liver and myocardial injury. Compared with mild cases, levels of inflammatory cytokines including interleukin 6, tumor necrosis factor α, and interferon γ were unchanged, whereas the level of immune suppressive interleukin 10 was markedly increased in moderate cases compared with mild cases (median [IQR], 3.96 [3.34-5.29] pg/mL vs 3.58 [3.10-4.36] pg/mL; P = .048). There was no statistically significant difference in absolute number of lymphocytes (including T cells and B cells) between mild and moderate cases, but moderate cases were associated with a decrease in neutrophil levels compared with mild cases (median [IQR], 2310/μL [1680/μL-3510/μL] vs 3120/μL [2040/μL-4170/μL]; P = .01). Immunoglobin G and the neutrophil to lymphocyte ratio were negatively associated with biochemical indices related to liver and myocardial injury (immunoglobulin G, ALT: r, −0.3579; AST: r, −0.5280; CK-MB activity: r, −0.4786; LDH: r, −0.4984; and neutrophil to lymphocyte ratio, ALT: r, −0.1893; AST: r, −0.3912; CK-MB activity: r, −0.3428; LDH: r, −0.3234), while counts of lymphocytes, CD4(+) T cells, and interleukin 10 showed positive associations (lymphocytes, ALT: r, 0.2055; AST: r, 0.3615; CK-MB activity: r, 0.338; LDH: r, 0.3309; CD4(+) T cells, AST: r, 0.4701; CK-MB activity: r, 0.4151; LDH: r, 0.4418; interleukin 10, ALT: r, 0.2595; AST: r, 0.3386; CK-MB activity: r, 0.3948; LDH: r, 0.3794). CONCLUSIONS AND RELEVANCE: In this case series, systemic inflammation rarely occurred in pediatric patients with COVID-19, in contrast with the lymphopenia and aggravated inflammatory responses frequently observed in adults with COVID-19. Gaining a deeper understanding of the role of neutrophils, CD4(+) T cells, and B cells in the pathogenesis of SARS-CoV-2 infection could be important for the clinical management of COVID-19.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: y29kct7v\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?\n",
      "are there any clinical trials available for the coronavirus\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "SARS-CoV-2 is a newly emerging human infectious coronavirus that causes COVID-19, which has been recognized as a pandemic by the World Health Organization (WHO) on March 11(th). There is still no vaccine or definitive treatment for this virus because its pathogenesis and proliferation pathways are still unknown. Therefore, in this article, new potential COVID-19 therapies are briefly reviewed.\n",
      "SARS-CoV-2 is a newly emerging human infectious coronavirus that causes COVID-19, which has been recognized as a pandemic by the World Health Organization (WHO) on March 11th. There is still no vaccine or definitive treatment for this virus because its pathogenesis and proliferation pathways are still unknown. Therefore, in this article, new potential COVID-19 therapies are briefly reviewed.\n",
      "SARS-CoV-2 is a newly emerging human infectious coronavirus that causes COVID-19, which has been recognized as a pandemic by the World Health Organization (WHO) on March 11(th) There is still no vaccine or definitive treatment for this virus because its pathogenesis and proliferation pathways are still unknown Therefore, in this article, new potential COVID-19 therapies are briefly reviewed\n",
      "Covid-19 disease is caused by SARS-CoV-2, a virus belonging to the coronavirus family. Covid-19 is so new that there is currently no specific vaccine or treatment. Clinical trials are currently underway. In vitro tests are also being conducted to assess the efficacy of chloroquine and hydroxychloroquine for the treatment of this epidemic, which is considered a pandemic by the WHO. We note that the content of this review is dated. The information it contains is subject to change and modification as the epidemic progresses.\n",
      "The current COVID-19 pandemic is caused by the novel coronavirus SARS-CoV-2. The virus causes severe respiratory symptoms which manifest disproportionately in the elderly. Currently, there are over 6.5 million cases and 380,000 deaths reported. Given the current severity of the outbreak, there is a great need for antiviral therapies and vaccines to treat and prevent COVID-19. In this review, we provide an overview of SARS-CoV-2 and discuss the emerging therapies and vaccines that show promise in combating COVID-19. We also highlight potential viral targets that could be exploited by researchers and drug manufacturers.\n",
      "The current COVID-19 pandemic is caused by the novel coronavirus SARS-CoV-2. The virus causes severe respiratory symptoms which manifest disproportionately in the elderly. Currently, there are over 6.5 million cases and 380,000 deaths reported. Given the current severity of the outbreak, there is a great need for antiviral therapies and vaccines to treat and prevent COVID-19. In this review, we provide an overview of SARS-CoV-2 and discuss the emerging therapies and vaccines that show promise in combating COVID-19. We also highlight potential viral targets that could be exploited by researchers and drug manufacturers.\n",
      "[Image: see text] Recently, a novel coronavirus initially designated 2019-nCoV but now termed SARS-CoV-2 has emerged and raised global concerns due to its virulence. SARS-CoV-2 is the etiological agent of “coronavirus disease 2019”, abbreviated to COVID-19, which despite only being identified at the very end of 2019, has now been classified as a pandemic by the World Health Organization (WHO). At this time, no specific prophylactic or postexposure therapy for COVID-19 are currently available. Viral entry is the first step in the SARS-CoV-2 lifecycle and is mediated by the trimeric spike protein. Being the first stage in infection, entry of SARS-CoV-2 into host cells is an extremely attractive therapeutic intervention point. Within this review, we highlight therapeutic intervention strategies for anti-SARS-CoV, MERS-CoV, and other coronaviruses and speculate upon future directions for SARS-CoV-2 entry inhibitor designs.\n",
      "Recently, a novel coronavirus initially designated 2019-nCoV but now termed SARS-CoV-2 has emerged and raised global concerns due to its virulence. SARS-CoV-2 is the etiological agent of \"coronavirus disease 2019\", abbreviated to COVID-19, which despite only being identified at the very end of 2019, has now been classified as a pandemic by the World Health Organization (WHO). At this time, no specific prophylactic or postexposure therapy for COVID-19 are currently available. Viral entry is the first step in the SARS-CoV-2 lifecycle and is mediated by the trimeric spike protein. Being the first stage in infection, entry of SARS-CoV-2 into host cells is an extremely attractive therapeutic intervention point. Within this review, we highlight therapeutic intervention strategies for anti-SARS-CoV, MERS-CoV, and other coronaviruses and speculate upon future directions for SARS-CoV-2 entry inhibitor designs.\n",
      "COVID-19, the disease caused by SARS-CoV-2, is a highly contagious disease. The World Health Organization has declared the ongoing outbreak to be a global public health emergency. Currently, the research on SARS-CoV-2 is in its primary stages. Based on current published evidence, this review systematically summarizes the epidemiology, clinical characteristics, diagnosis, treatment and prevention of COVID-19. It is hoped that this review will help the public to recognize and deal with SARS-CoV-2, and provide a reference for future studies.\n",
      "The epidemic of pneumonia (COVID-19) caused by novel coronavirus (SARS-CoV-2) infection has been listed as a public health emergency of international concern by the World Health Organization (WHO), and its harm degree is defined as a global “pandemic”. At present, the efforts of various countries focus on the rapid diagnosis and isolation of patients, as well as to find a treatment that can combat the most serious impact of the disease. The number of reported COVID-19 virus infections is still increasing. Unfortunately, no drugs or vaccines have been approved for the treatment of human coronaviruses, but there is an urgent need for in-depth research on emerging human infectious coronaviruses. Clarification transmission routes and pathogenic mechanisms, and identification of potential drug treatment targets will promote the development of effective prevention and treatment measures. In the absence of confirmed effective treatments, due to public health emergencies, it is essential to study the possible effects of existing approved antivirals drugs or Chinese herbal medicines for SARS-CoV-2. This review summarizes the epidemiological characteristics, pathogenesis, virus structure and targeting strategies of COVID-19. Meanwhile, this review also focus on the re-purposing of clinically approved drugs and Chinese herbal medicines that may be used to treat COVID-19 and provide new ideas for the discovery of small molecular compounds with potential therapeutic effects on novel COVID-19.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "ID: 48ay8yl3\n",
      "\n",
      "GT queries\n",
      "-----------\n",
      "What is the protein structure of the SARS-CoV-2 spike?\n",
      "\n",
      "Retrieved queries\n",
      "-----------\n",
      "Severe acute respiratory syndrome (SARS) is a newly emerging infectious disease caused by a novel coronavirus, SARS-coronavirus (SARS-CoV). The SARS-CoV spike (S) protein is composed of two subunits; the S1 subunit contains a receptor-binding domain that engages with the host cell receptor angiotensin-converting enzyme 2 and the S2 subunit mediates fusion between the viral and host cell membranes. The S protein plays key parts in the induction of neutralizing-antibody and T-cell responses, as well as protective immunity, during infection with SARS-CoV. In this Review, we highlight recent advances in the development of vaccines and therapeutics based on the S protein.\n",
      "Severe acute respiratory syndrome (SARS) is a novel infectious disease caused by SARS coronavirus (SARS-CoV). Although SARS appears to have been successfully contained, there is still a risk for its reemergence due to sporadic laboratory accidents or the presence of a natural reservoir for SARS-CoV-like virus. Therefore, the development of effective vaccines against SARS-CoV continues to be the current focus of SARS research. This review will first describe the rationale for developing safe and effective SARS vaccines, followed by elucidating viral antigens that could be used as potential vaccine components. After comparing current vaccine categories against SARS, this article will demonstrate the advantages of subunit vaccines, describe the current situation of developing subunit vaccines, and point out the possibility for further improvement of subunit SARS vaccines. This suggests that recombinant protein/peptide-based subunit vaccines containing the spike protein, especially the receptor-bind domain of SARS-CoV, could be developed as safe and effective SARS vaccines.\n",
      "Severe acute respiratory syndrome (SARS) is a newly emerged infectious disease that caused pandemic spread in 2003. The etiological agent of SARS is a novel coronavirus (SARS-CoV). The coronaviral surface spike protein S is a type I transmembrane glycoprotein that mediates initial host binding via the cell surface receptor angiotensin-converting enzyme 2 (ACE2), as well as the subsequent membrane fusion events required for cell entry. Here we report the crystal structure of the S1 receptor binding domain (RBD) in complex with a neutralizing antibody, 80R, at 2.3 A resolution, as well as the structure of the uncomplexed S1 RBD at 2.2 A resolution. We show that the 80R-binding epitope on the S1 RBD overlaps very closely with the ACE2-binding site, providing a rationale for the strong binding and broad neutralizing ability of the antibody. We provide a structural basis for the differential effects of certain mutations in the spike protein on 80R versus ACE2 binding, including escape mutants, which should facilitate the design of immunotherapeutics to treat a future SARS outbreak. We further show that the RBD of S1 forms dimers via an extensive interface that is disrupted in receptor- and antibody-bound crystal structures, and we propose a role for the dimer in virus stability and infectivity.\n",
      "Abstract Severe acute respiratory syndrome (SARS) is an emerging infectious disease associated with a novel coronavirus and causing worldwide outbreaks. SARS coronavirus (SARS-CoV) is an enveloped RNA virus, which contains several structural proteins. Among these proteins, spike (S) protein is responsible for binding to specific cellular receptors and is a major antigenic determinant, which induces neutralizing antibody. In order to analyze the antigenicity and receptor-binding ability of SARS-CoV S protein, we expressed the S protein in Escherichia coli using a pET expression vector. After the isopropyl-β-d-thiogalactoside induction, S protein was expressed in the soluble form and purified by nickel-affinity chromatography to homogeneity. The amount of S protein recovered was 0.2–0.3mg/100ml bacterial culture. The S protein was recognized by sera from SARS patients by ELISA and Western blot, which indicated that recombinant S protein retained its antigenicity. By biotinylated ELISA and Western blot using biotin-labeled S protein as the probe, we identified 130-kDa and 140-kDa proteins in Vero cells that might be the cellular receptors responsible for SARS-CoV infection. Taken together, these results suggested that recombinant S protein exhibited the antigenicity and receptor-binding ability, and it could be a good candidate for further developing SARS vaccine and anti-SARS therapy.\n",
      "A novel human coronavirus (CoV) has been identified as the etiological agent that caused the severe acute respiratory syndrome (SARS) outbreak in 2003. The spike (S) protein of this virus is a type I surface glycoprotein that mediates binding of the virus to the host receptor and the subsequent fusion between the viral and host membranes. Because of its critical role in viral entry, the S protein is an important target for the development of anti‐SARS CoV therapeutics and prophylactics. This article reviews the structure and function of the SARS CoV S protein in the context of its role in virus entry. Topics that are discussed include: the interaction between the S1 domain of the SARS spike protein and the cellular receptor, angiotensin converting enzyme 2 (ACE2), and the structural features of the ectodomain of ACE2; the antigenic determinants presented by the S protein and the nature of neutralizing monoclonal antibodies that are elicited in vivo; the structure of the 4,3‐hydrophobic heptad repeats HR1 and HR2 of the S2 domain and their interaction to form a six‐helical bundle during the final stages of fusion. Opportunities for the design and development of anti‐SARS agents based on the inhibition of receptor binding, the therapeutic uses of S‐directed monoclonal antibodies and inhibitors of HR1–HR2 complex formation are presented. © 2006 Wiley Periodicals, Inc. Med Res Rev, 26, No. 4, 414–433, 2006\n",
      "Severe acute respiratory syndrome (SARS) is an emerging infectious disease caused by a new coronavirus (SARS-CoV). Recent studies suggest that SARS-CoV is zoonotic and may have a broad host range besides humans. Although the global outbreak of SARS has been contained, there are serious concerns over its re-emergence and bioterrorism potential. As a part of preparedness, development of a safe and effective vaccine is one of the highest priorities in fighting SARS. A number of candidate vaccines, using a variety of approaches, are under development. The first vaccine tested in clinical trial is made from the inactivated form of SARS-CoV. Several live attenuated, genetically engineered or vector vaccines encoding the SARS-CoV spike (S) protein have been in pre-clinical studies. These vaccine candidates are effective in terms of eliciting protective immunity in the vaccinated animals. However, caution should be taken with the safety of whole virus or full-length S protein-based immunogens in humans because they may induce harmful immune or inflammatory responses. We propose to use the receptor-binding domain (RBD) of SARS-CoV S protein (residues 318--510) for developing a safe and effective subunit SARS vaccine, as it is not only a functional domain that mediates virus-receptor binding but also a major neutralization determinant of SARSCoV. It has been demonstrated that the RBD of SARS-CoV S protein contains multiple conformational epitopes capable of inducing highly potent neutralizing antibody responses and protective immunity.\n",
      "Severe acute respiratory syndrome coronavirus (SARS-CoV) emerged in 2002 as a highly transmissible pathogenic human betacoronavirus. The viral spike glycoprotein (S) utilizes angiotensin-converting enzyme 2 (ACE2) as a host protein receptor and mediates fusion of the viral and host membranes, making S essential to viral entry into host cells and host species tropism. As SARS-CoV enters host cells, the viral S is believed to undergo a number of conformational transitions as it is cleaved by host proteases and binds to host receptors. We recently developed stabilizing mutations for coronavirus spikes that prevent the transition from the pre-fusion to post-fusion states. Here, we present cryo-EM analyses of a stabilized trimeric SARS-CoV S, as well as the trypsin-cleaved, stabilized S, and its interactions with ACE2. Neither binding to ACE2 nor cleavage by trypsin at the S1/S2 cleavage site impart large conformational changes within stabilized SARS-CoV S or expose the secondary cleavage site, S2′.\n",
      "The severe acute respiratory syndrome (SARS) is a newly emerging human infectious disease caused by the severe acute respiratory syndrome coronavirus (SARS-CoV). The spike (S) protein of SARS-CoV is a major virion structural protein. It plays an important role in the interaction with receptors and neutralizing antibodies. In this study, the S1 domain of the spike protein and three truncated fragments were expressed by fusion with GST in a pGEX-6p-1 vector. Western blot results demonstrated that the 510-672 fragment of the S1 domain is a linear epitope dominant region. To map the antigenic epitope of this linear epitope dominant region, a set of 16 partially overlapping fragments spanning the fragment were fused with GST and expressed. Four antigenic epitopes S1C3 (539-559), S1C4 (548-567), S1C7/8 (583-606), and S1C10/11 (607-630) were identified. Immunization of mice with each of the four antigenic epitope-fused proteins revealed that all four proteins could elicit spike protein specific antisera. All of them were able to bind to the surface domain of the whole spike protein expressed by recombinant baculovirus in insect cells. Identification of antigenic epitopes of the spike protein of SARS-CoV may provide the basis for the development of immunity-based prophylactic, therapeutic, and diagnostic clinical techniques for the severe acute respiratory syndrome.\n",
      "Severe acute respiratory syndrome (SARS) is a life-threatening disease caused by a newly identified coronavirus (CoV), SARS-CoV. The spike (S) glycoprotein of CoV is the major structural protein responsible for induction of host immune response and virus neutralization by antibodies. Hence, knowledge of neutralization determinants on the S protein is helpful for designing protective vaccines. To analyze the antigenic structure of the SARS-CoV S2 domain, the carboxyl-terminal half of the S protein, we first used sera from convalescent SARS patients to test the antigenicity of 12 overlapping fragments spanning the entire S2 and identified two antigenic determinants (Leu 803 to Ala 828 and Pro 1061 to Ser 1093). To determine whether neutralizing antibodies can be elicited by these two determinants, we immunized animals and found that both of them could induce the S2-specific antisera. In some animals, however, only one determinant (Leu 803 to Ala 828) was able to induce the antisera with the binding ability to the native S protein and the neutralizing activity to the SARS-CoV pseudovirus. This determinant is highly conserved across different SARS-CoV isolates. Identification of a conserved antigenic determinant on the S2 domain of the SARS-CoV S protein, which has the potential for inducing neutralizing antibodies, has implications in the development of effective vaccines against SARS-CoV.\n",
      "Enveloped viruses have evolved membrane glycoproteins (GPs) that mediate entry into host cells. These proteins are important targets for antiviral therapies and vaccines. Several efforts to understand and combat infection by severe acute respiratory syndrome coronavirus (SARS-CoV) have therefore focused on the viral GP, known as spike (S). In a short period of time, important aspects of SARS-CoV S-protein function were unraveled. The identification of angiotensin-converting enzyme 2 (ACE2) as a receptor for SARS-CoV provided an insight into viral tropism and pathogenesis, whereas mapping of functional domains in the S-protein enabled inhibitors to be generated. Vaccines designed on the basis of SARS-CoV S-protein were shown to be effective in animals and consequently are attractive candidates for vaccine trials in humans. Here, we discuss how SARS-CoV S facilitates viral entry into target cells and illustrate current approaches that are used to inhibit this process.\n",
      "\n",
      "\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "for res in ret_fail:\n",
    "    try:\n",
    "        gt_docs = get_matching_queries(res[\"_id\"])\n",
    "    except:\n",
    "        continue\n",
    "    print(\"ID:\", res[\"_id\"])\n",
    "    print(\"\\nGT queries\\n-----------\")\n",
    "    for doc in gt_docs:\n",
    "        print(doc)\n",
    "    print(\"\\nRetrieved queries\\n-----------\")\n",
    "    for ret in res[\"retrieval\"]:\n",
    "        print(ret[\"document\"])\n",
    "    print(\"\\n\\n\\n\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "adc2d402-03c9-4fd4-b9d6-2b33f97db62a",
   "metadata": {},
   "source": [
    "# NDCG metric for \"Retrieved Query\" vs \"Gold Query\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "id": "75eed346-9c51-469b-b53f-675d89257ef3",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.45178197064989517\n"
     ]
    }
   ],
   "source": [
    "# True positive documents\n",
    "accs = []\n",
    "for res in ret_suc:\n",
    "    try:\n",
    "        gt_docs = get_matching_queries(res[\"_id\"])\n",
    "    except:\n",
    "        continue\n",
    "    acc = 0 \n",
    "    for ret in res[\"retrieval\"]:\n",
    "        if ret[\"document\"] in gt_docs:\n",
    "            acc += 1\n",
    "    accs.append(acc / len(gt_docs))\n",
    "print(sum(accs) / len(accs))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "id": "feda13e7-1905-412c-a9f5-2a3433283153",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.3817204301075268\n"
     ]
    }
   ],
   "source": [
    "# True negative documents\n",
    "accs = []\n",
    "for res in ret_fail:\n",
    "    try:\n",
    "        gt_docs = get_matching_queries(res[\"_id\"])\n",
    "    except:\n",
    "        continue\n",
    "    acc = 0 \n",
    "    for ret in res[\"retrieval\"]:\n",
    "        if ret[\"document\"] in gt_docs:\n",
    "            acc += 1\n",
    "    accs.append(acc / len(gt_docs))\n",
    "print(sum(accs) / len(accs))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "a6f8a137-ea57-415e-9ab6-4ed324e38ba8",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
